<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">BMC Plant Biol</span> <span class="iso-abbrev" title="iso-abbrev">BMC Plant Biol</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">BMC Plant Biology</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1471-2229</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5914031">5914031</a> </span> <span class="publisher-id" title="publisher-id">1284</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1186/s12870-018-1284-7">10.1186/s12870-018-1284-7</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Population structure, genetic diversity and downy mildew resistance among <i>Ocimum</i> species germplasm </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-1629-909X</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pyne</span> <span class="given-names" tagx="given-names" title="given-names">Robert M.</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">rpyne97@gmail.com</span> </span> <a href="#Aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Honig</span> <span class="given-names" tagx="given-names" title="given-names">Josh A.</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">josh.honig@rutgers.edu</span> </span> <a href="#Aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vaiciunas</span> <span class="given-names" tagx="given-names" title="given-names">Jennifer</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">vaiciunas@sebs.rutgers.edu</span> </span> <a href="#Aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wyenandt</span> <span class="given-names" tagx="given-names" title="given-names">Christian A.</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">wyenandt@njaes.rutgers.edu</span> </span> <a href="#Aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Simon</span> <span class="given-names" tagx="given-names" title="given-names">James E.</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">jimsimon@rutgers.edu</span> </span> <a href="#Aff1" /> </span> <span class="citation_author_institution" id="Aff1">[], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0004 1936 8796</span> <span class="institution-id" title="institution-id">grid.430387.b</span> <span class="institution" title="institution">Department of Plant Biology, </span> <span class="institution" title="institution">Rutgers, the State University of New Jersey, </span> </div> </span> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-4-4</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2018-4-4</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2018</span> </span> <span class="volume" tagx="volume" title="volume">18</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">69</span> <span class="history" title="history"> <span class="received" title="received">received: 2017-8-30</span> <span class="accepted" title="accepted">accepted: 2018-4-10</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , </span> <span class="license" title="license"> <span class="license-p" title="license-p"> <b>Open Access</b>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License ( <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver ( <a href="http://creativecommons.org/publicdomain/zero/1.0/">http://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>The basil ( <i>Ocimum</i> spp.) genus maintains a rich diversity of phenotypes and aromatic volatiles through natural and artificial outcrossing. Characterization of population structure and genetic diversity among a representative sample of this genus is severely lacking. Absence of such information has slowed breeding efforts and the development of sweet basil ( <i>Ocimum basilicum</i> L.) with resistance to the worldwide downy mildew epidemic, caused by the obligate oomycete <i>Peronospora belbahrii</i>. In an effort to improve classification of relationships 20 EST-SSR markers with species-level transferability were developed and used to resolve relationships among a diverse panel of 180 <i>Ocimum</i> spp. accessions with varying response to downy mildew. </p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>Results obtained from nested Bayesian model-based clustering, analysis of molecular variance and unweighted pair group method using arithmetic average (UPGMA) analyses were synergized to provide an updated phylogeny of the <i>Ocimum</i> genus. Three (major) and seven (sub) population (cluster) models were identified and well-supported ( <i>P</i> &amp;lt; 0.001) by PhiPT (Φ <sub>PT</sub>) values of 0.433 and 0.344, respectively. Allelic frequency among clusters supported previously developed hypotheses of allopolyploid genome structure. Evidence of cryptic population structure was demonstrated for the k1 <i>O. basilicum</i> cluster suggesting prevalence of gene flow. UPGMA analysis provided best resolution for the 36-accession, DM resistant k3 cluster with consistently strong bootstrap support. Although the k3 cluster is a rich source of DM resistance introgression of resistance into the commercially important k1 accessions is impeded by reproductive barriers as demonstrated by multiple sterile F1 hybrids. The k2 cluster located between k1 and k3, represents a source of transferrable tolerance evidenced by fertile backcross progeny. The 90-accession k1 cluster was largely susceptible to downy mildew with accession ‘MRI’ representing the only source of DM resistance. </p> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>High levels of genetic diversity support the observed phenotypic diversity among <i>Ocimum</i> spp. accessions. EST-SSRs provided a robust evaluation of molecular diversity and can be used for additional studies to increase resolution of genetic relationships in the <i>Ocimum</i> genus. Elucidation of population structure and genetic relationships among <i>Ocimum</i> spp. germplasm provide the foundation for improved DM resistance breeding strategies and more rapid response to future disease outbreaks. </p> </div> <div class="electronicsupplementarymaterial" title="sec"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <p>The online version of this article (10.1186/s12870-018-1284-7) contains supplementary material, which is available to authorized users. </p> </div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd"> <i>Ocimum</i> spp. </div> <div class="kwd" title="kwd">
Downy mildew resistance</div> <div class="kwd" title="kwd">
EST-SSRs</div> <div class="kwd" title="kwd">
Population structure</div> <div class="kwd" title="kwd">
Genetic diversity</div> <div class="kwd" title="kwd">
Polyploid</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <span class="institution" title="institution">USDA-Special Crops Research Initiative</span> </div> <span class="award-id" tagx="award-id" title="award-id">2011-51181-30646</span> </div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100006031</span> <span class="institution" title="institution">United States - Israel Binational Agricultural Research and Development Fund</span> </div> </div> <span class="award-id" tagx="award-id" title="award-id">US-4947-16R</span> </div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <span class="institution" title="institution">USDA - National Institute of Food and Agriculture</span> </div> <span class="award-id" tagx="award-id" title="award-id">2016-68004-24931</span> </div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2018" /> </div> </div> </div> <div class="body" title="body"> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>The genus <i>Ocimum</i> is estimated to contain approximately 64 herbaceous annual and perennial plant species [ <a href="#CR1">1</a>]. The primary center of diversity is described as being in Tropical Africa [ <a href="#CR2">2</a>], while a secondary center exists in Tropical Asia and a tertiary center exists in the New World tropics [ <a href="#CR2">2</a>, <a href="#CR3">3</a>]. A number of species are cultivated and processed throughout the world for application in medicinal, dietary supplement, essential oil, food flavoring and culinary industries [ <a href="#CR4">4</a>]. <i>Ocimum basilicum</i> L., sweet basil, is among the most economically important culinary herbs in the United States, Europe and Israel [ <a href="#CR5">5</a>]. Plant breeding efforts have largely targeted sweet basil for improved disease resistance [ <a href="#CR6">6</a>], chilling tolerance [ <a href="#CR7">7</a>] and novel volatile profiles (chemotypes) [ <a href="#CR8">8</a>]. </p> <p> <i>Ocimum</i> spp. demonstrate relatively high rates of outcrossing in the presence of pollinators [ <a href="#CR9">9</a>], while readily self-pollinating in their absence. The flexibility of this reproductive system has been exploited by natural and artificial selection, likely serving as a catalyst for morphological and volatile diversification of <i>Ocimum</i> spp. [ <a href="#CR4">4</a>]. In addition, cytological investigation suggests this genus has also undergone extensive genome augmentation resulting in ploidy variation. While the x = 12 basic chromosome number is considered stable in tetraploid (2n = 4× = 48) <i>O. basilicum</i>, different ploidies have been reported across other <i>Ocimum</i> species [ <a href="#CR10">10</a>]. Dual basic chromosome numbers of 8 and 12 have been proposed for two major ‘Basilicum’ and ‘Sanctum’ groups, respectively [ <a href="#CR11">11</a>]. More recently, studies involving nuclear DNA content support the hypothesis of x = 12 as the genus-wide basic chromosome number with some cases of aneuploidy [ <a href="#CR12">12</a>, <a href="#CR13">13</a>]. This explanation accounts for dysploidy, which has proven to be a persistent phenomenon in plant evolutionary biology [ <a href="#CR14">14</a>]. The 386 Mbp <i>O. tenuiflorum (</i>syn. <i>O. sanctum</i>) genome was reported to be a diploid (2n = 2× = 16) supporting the existence of an x = 8 basic chromosome number and suggesting <i>O. tenuiflorum</i> is a diploid species [ <a href="#CR15">15</a>]. Thus, competing theories of major genome structure remain prevalent in the <i>Ocimum</i> genus. </p> <p>Examination of chromosome pairing behavior suggests allopolyploidy likely explains the polyploid genome structure in basil [ <a href="#CR10">10</a>, <a href="#CR11">11</a>]. For instance, karyological investigation of <i>O. basilicum</i> and <i>O. basilicum</i> x <i>O. americanum</i> F <sub>1</sub> hybrids found no evidence of tetravalent and little trivalent formation (4-8% of pollen mother cells) demonstrating high levels of preferential pairing within sub-genomes [ <a href="#CR11">11</a>]. Relatively high levels of inter-specific hybridization [ <a href="#CR9">9</a>] suggest allopolyploid formation to be more likely than autopolyploidy [ <a href="#CR16">16</a>]. This hypothesis is also supported by genotype data from simple sequence repeat (SSR) and single nucleotide polymorphism (SNP) markers in the <i>O. basilicum</i> MRI x SB22 F <sub>2</sub> mapping family demonstrated that disomic segregation providing molecular evidence for an allotetraploid (AABB) <i>O. basilicum</i> [ <a href="#CR17">17</a>]. </p> <p>In addition to variation in ploidy [ <a href="#CR10">10</a>, <a href="#CR18">18</a>], a variety of factors including unknown geographic origin and phenotypic diversity complicate classification of relationships among <i>O. basilicum</i> and the greater <i>Ocimum</i> genus. Taxonomical incongruencies are also widespread throughout the literature and among seed distributors. Furthermore, absence of standardized descriptors or voucher specimens for particular cultivars has been particularly problematic in classifying <i>O. basilicum</i> [ <a href="#CR1">1</a>]. Species of economic salience such as <i>O. x citriodorum</i>, <i>O. americanum</i> and <i>O. basilicum</i> var <i>. citriodorum</i> are cited interchangeably throughout the literature making comparison across studies challenging [ <a href="#CR19">19</a>]. Despite attempts to standardize nomenclature [ <a href="#CR19">19</a>, <a href="#CR20">20</a>] the species epithets continue to be used redundantly. </p> <p>Morphological characters and volatile composition (chemotype) have been used to estimate genetic diversity in the <i>Ocimum</i> genus [ <a href="#CR21">21</a>– <a href="#CR24">24</a>]. Classification based on phenotypic characters is problematic in that homologous similarities may be difficult or impossible to discriminate from traits resulting from convergent evolution [ <a href="#CR25">25</a>]. Reliance upon analyses using these metrics may be particularly damaging in the context of plant breeding, which relies heavily upon genetic relationships among germplasm to make informed decisions with regard to selection and crossing decisions [ <a href="#CR26">26</a>]. Nevertheless, the most comprehensive study of <i>Ocimum</i> to date remains that of Paton et al., which relied exclusively upon morphological characters [ <a href="#CR1">1</a>]. Results of this study delineated three major divisions in the genus or subgenera corresponding into <i>Ocimum</i>, <i>Nautochilus</i> and <i>Gymnocium</i> [ <a href="#CR1">1</a>]. Subgenera <i>Ocimum</i> is further divided into three sections including <i>Gratissima</i>, <i>Hiantia</i>, and <i>Ocimum</i>, where <i>O. basilicum</i> and <i>O. americanum</i> was placed [ <a href="#CR1">1</a>]. </p> <p>Molecular phylogenetic approaches have employed RAPD and/or AFLP markers to determine relationships among nine [ <a href="#CR27">27</a>], twelve [ <a href="#CR28">28</a>], twenty-two [ <a href="#CR18">18</a>], twenty-eight [ <a href="#CR12">12</a>] and thirty seven [ <a href="#CR2">2</a>] <i>Ocimum</i> spp. accessions. DNA markers from three plastid regions were used by Paton et al. to classify 12 <i>Ocimum</i> spp. accessions within the Lamiaceae family [ <a href="#CR3">3</a>]. More recently, inter-simple sequence repeat (ISSR) markers were found to have higher polymorphism information content (PIC) and greater resolving power than chloroplast DNA (cpDNA) <i>psbA-trnH</i> markers following cluster analysis of 12 accessions representing 8 <i>Ocimum</i> spp. [ <a href="#CR29">29</a>]. Despite being the preferred marker systems for phylogenetic analyses, neither SSR nor SNP markers have been used to classify relationships among <i>Ocimum</i> spp. accessions. SSRs are considered advantageous because they are multi-allelic, polymorphic, reproducible, ubiquitous throughout the genome and amenable to high-throughput genotyping with multiplexed PCR [ <a href="#CR30">30</a>]. Expressed sequence tag (EST) derived SSRs (EST-SSRs) are particularly desirable for genetic diversity and mapping studies due to their location in coding regions and transferability across related germplasm [ <a href="#CR31">31</a>, <a href="#CR32">32</a>]. The National Center for Biotechnology Information (NCBI) <i>O. basilicum</i> EST library was recently used to develop polymorphic SSRs and genotype the MRI x SB22 mapping population [ <a href="#CR17">17</a>]. </p> <p>The deficit of available sweet basil genetic and genomic resources relative to other cultivated plant species has been highlighted in recent years by a worldwide downy mildew (DM) epidemic caused by the oomycete pathogen <i>Peronospora belbahrii</i> [ <a href="#CR33">33</a>, <a href="#CR34">34</a>]. As demonstrated by extensive breeding across plant species affected by downy mildew, deployment of genetic resistance is an essential management strategy for effective disease control [ <a href="#CR35">35</a>– <a href="#CR37">37</a>]. Recent reports of Mefenoxam resistant <i>P. belbahrii</i> isolates in Israel [ <a href="#CR38">38</a>] and Italy [ <a href="#CR39">39</a>, <a href="#CR40">40</a>] exacerbate the need for resistant cultivars. However, development of resistant sweet basil varieties has been dramatically slowed by a severe lack of knowledge regarding genetic similarity between elite <i>O. basilicum</i> and exotic <i>Ocimum</i> spp. germplasm rich in sources of resistance [ <a href="#CR41">41</a>– <a href="#CR43">43</a>]. This has rendered attempted introgression of resistant loci into sweet basil accessions subject to rudimentary trial and error by laborious cross-pollinations between elite and exotic breeding lines. Among 27 F <sub>1</sub> hybrids evaluated by Ben-Naim et al., those considered ‘highly resistant’ were completely sterile [ <a href="#CR41">41</a>]. </p> <p>Population structure is among the analyses most needed to better elucidate relationships among <i>Ocimum</i> spp. Model-based clustering is a powerful analysis method frequently used to infer association of individuals into distinct populations from multilocus data sets [ <a href="#CR44">44</a>]. Association of germplasm into distinct populations provides essential a priori information needed to avoid issues of sterility while monitoring potential narrowing of the genetic pool among breeding populations. Furthermore, detection of cryptic population structure is an important prerequisite to association mapping (AM) of alleles with traits of interest (i.e. disease resistance) by avoiding false positive associations [ <a href="#CR44">44</a>, <a href="#CR45">45</a>]. Identification of population structure and estimation of genetic diversity among <i>Ocimum</i> spp. is needed to provide a more robust breeding response to future biotic or abiotic stresses. </p> <p>In this study no designations of species/relationships are considered to provide an unbiased update of basil phylogenetics and first insights into population structure. We hypothesized that SSR markers developed from the available NCBI EST database could be employed to increase resolution of genetic relationships among a diverse panel of 180 <i>Ocimum</i> spp. accessions by population structure and genetic diversity analyses. Evaluation of these accessions for response to downy mildew provides insight into the distribution of downy mildew resistance within the <i>Ocimum</i> genus and has important implications for disease resistance breeding strategies. </p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="responsetoperonosporabelbahrii" title="sec"> <div class="title" tagx="title" title="title">
Response to <i>Peronospora belbahrii</i> </div> <p>Significant differences ( <i>P</i> &amp;lt; 0.001) were detected among 180 genotypes evaluated for response to DM (Additional file <a href="#MOESM1">1</a>). Eighty-two accessions (46%) demonstrated a mean DS &amp;lt; 1.0 (considered resistant) of which 33 (18%) exhibited no sporulation (DS = 0). A large proportion (17%) of the accessions with mean DS &amp;lt; 1.0 were Rutgers breeding lines with MRI-conferred DM resistance (DM <sub>MRI</sub>). DM susceptibility was widespread, however, with 71 accessions (40%) having DS ≥ 3.0 (considered highly susceptible). The remaining 26 accessions (15%) were in an intermediate range of DS between 1 (10%) and 3 (50%) representing levels of DM tolerance. A single accession, PI 511865, segregated for response to DM indicating heterozygosity among loci controlling resistance in this accession. </p> </div> <div class="est-ssrpolymorphism" title="sec"> <div class="title" tagx="title" title="title">
EST-SSR polymorphism</div> <p>The 20 EST-SSR markers used in this study were pre-screened for unambiguous PCR amplification and demonstrated a low percentage of null genotypes. A list of DNA sequence and melting temperatures (T <sub>m</sub>) associated with EST-SSR markers can be found in Additional file <a href="#MOESM2">2</a>. A minimum of 90% and an average of 96% of individuals were successfully genotyped across all EST-SSR markers. A total of 269 unique alleles were amplified across the 180 <i>Ocimum</i> spp. panel and 2 outgroup <i>Nepeta cateria</i> accessions (CN3 and CR9). The average PIC across 20 EST-SSR markers was 0.14, ranging from 0.01 to 0.48 and 6 to 24 alleles per marker (locus) (Table <a href="#Tab1">1</a>). The average number of alleles/locus across the full panel was 1.86 (Table <a href="#Tab2">2</a>). <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  </div></p><p>Description of 20 EST-SSR markers used to classify 180-accession panel of <i>Ocimum</i> spp. </p>  <table> <thead> <tr> <th>MarkerID</th> <th>Genbank/Contg ID</th> <th>Repeat Motif</th> <th>Alleles</th> <th>PIC</th> <th>Product Size</th> <th>LG</th> <th>Position (cM)</th> </tr> </thead> <tbody> <tr> <td>OBNJR2sg33</td> <td>DY333933</td> <td>(AC)16</td> <td>19</td> <td>0.01-0.45</td> <td>271-299</td> <td>14</td> <td>82.7</td> </tr> <tr> <td>OBNJR2cn29</td> <td>Contig1138</td> <td>(AC)16</td> <td>11</td> <td>0.02-0.45</td> <td>246-448</td> <td>13</td> <td>54.6</td> </tr> <tr> <td>OBNJR2sg04</td> <td>DY343638</td> <td>(GA)17</td> <td>27</td> <td>0.01-0.48</td> <td>268-357</td> <td>13</td> <td>40.2</td> </tr> <tr> <td>OBNJR2sg30</td> <td>DY336727</td> <td>(AG)22</td> <td>20</td> <td>0.01-0.42</td> <td>241-279</td> <td /> <td /> </tr> <tr> <td>OBNJR3sg124</td> <td>DY331703</td> <td>(GCC)6</td> <td>11</td> <td>0.01-0.40</td> <td>161-327</td> <td /> <td /> </tr> <tr> <td>OBNJR3sg19</td> <td>DY343509</td> <td>(TCA)6</td> <td>6</td> <td>0.01-0.46</td> <td>195-432</td> <td>6</td> <td>18.9</td> </tr> <tr> <td>OBNJR3cn298</td> <td>Contig2510</td> <td>(CTA)6</td> <td>9</td> <td>0.01-0.42</td> <td>298-322</td> <td /> <td /> </tr> <tr> <td>OBNJR3cn359</td> <td>Contig2911</td> <td>(GGC)6</td> <td>11</td> <td>0.01-0.31</td> <td>161-292</td> <td /> <td /> </tr> <tr> <td>OBNJR3cn362</td> <td>Contig2969</td> <td>(TGA)6</td> <td>7</td> <td>0.02-0.45</td> <td>225-243</td> <td>3</td> <td>78.0</td> </tr> <tr> <td>OBNJR3sg155</td> <td>DY325572</td> <td>(GTT)7</td> <td>8</td> <td>0.02-0.40</td> <td>188-263</td> <td /> <td /> </tr> <tr> <td>OBNJR3sg168</td> <td>DY323726</td> <td>(GAA)7</td> <td>11</td> <td>0.01-0.44</td> <td>297-390</td> <td /> <td /> </tr> <tr> <td>OBNJR3sg113</td> <td>DY335879</td> <td>(CCT)7</td> <td>12</td> <td>0.01-0.32</td> <td>313-392</td> <td /> <td /> </tr> <tr> <td>OBNJR3cn56</td> <td>Contig582</td> <td>(AGG)7</td> <td>10</td> <td>0.01-0.45</td> <td>169-240</td> <td>3</td> <td>84.7</td> </tr> <tr> <td>OBNJR3cn74</td> <td>Contig715</td> <td>(CAG)7</td> <td>12</td> <td>0.01-0.43</td> <td>211-244</td> <td /> <td /> </tr> <tr> <td>OBNJR3sg145</td> <td>DY328393</td> <td>(GCT)8</td> <td>15</td> <td>0.01-0.45</td> <td>271-313</td> <td /> <td /> </tr> <tr> <td>OBNJR3cn03</td> <td>Contig100</td> <td>(CCA)10</td> <td>15</td> <td>0.01-0.44</td> <td>198-262</td> <td /> <td /> </tr> <tr> <td>OBNJR3cn210</td> <td>Contig1890</td> <td>(AAG)11</td> <td>14</td> <td>0.01-0.43</td> <td>266-302</td> <td /> <td /> </tr> <tr> <td>OBNJR3cn240</td> <td>Contig2142</td> <td>(ATA)16</td> <td>24</td> <td>0.01-0.37</td> <td>254-370</td> <td>19</td> <td>88.7</td> </tr> <tr> <td>OBNJR3sg13</td> <td>DY344184</td> <td>(ACA)10</td> <td>14</td> <td>0.01-0.36</td> <td>271-312</td> <td /> <td /> </tr> <tr> <td>OBNJR4cn17</td> <td>Contig2461</td> <td>(AAAT)5</td> <td>13</td> <td>0.01-0.41</td> <td>164-365</td> <td /> <td /> </tr> </tbody> </table>  <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 2</span>  <p>Summary of allele distribution among clusters resulting from primary and secondary (nested) model-based cluster analyses </p>  <table> <thead> <tr> <th>Cluster</th> <th>Sample size</th> <th>Total alleles</th> <th>Average alleles per locus</th> </tr> </thead> <tbody> <tr> <td>k1</td> <td>90</td> <td>3081</td> <td>1.75</td> </tr> <tr> <td>k2</td> <td>16</td> <td>709</td> <td>2.23</td> </tr> <tr> <td>k3</td> <td>36</td> <td>907</td> <td>1.34</td> </tr> <tr> <td>Admixture</td> <td>38</td> <td>1812</td> <td>2.58</td> </tr> <tr> <td>k1.1</td> <td>51</td> <td>1760</td> <td>1.76</td> </tr> <tr> <td>k1.2</td> <td>28</td> <td>953</td> <td>1.74</td> </tr> <tr> <td>k2.1</td> <td>8</td> <td>363</td> <td>2.27</td> </tr> <tr> <td>k2.2</td> <td>7</td> <td>301</td> <td>2.18</td> </tr> <tr> <td>k3.1</td> <td>7</td> <td>189</td> <td>1.41</td> </tr> <tr> <td>k3.2</td> <td>18</td> <td>452</td> <td>1.33</td> </tr> <tr> <td>k3.3</td> <td>11</td> <td>266</td> <td>1.30</td> </tr> <tr> <td>Admixture</td> <td>50</td> <td>2225</td> <td>2.49</td> </tr> <tr> <td>Overall</td> <td>180</td> <td>6509</td> <td>1.86</td> </tr> </tbody> </table> </div> <p /> </div> <div class="populationstructureandallelefrequency" title="sec"> <div class="title" tagx="title" title="title">
Population structure and allele frequency</div> <p>Primary model-based clustering (population structure) analysis provided unambiguous evidence (Additional file <a href="#MOESM3">3</a>) for three major clusters k1 ( <i>n</i> = 90), k2 ( <i>n</i> = 16) and k3 ( <i>n</i> = 36) (Fig. <a href="#Fig1">1a</a>). The remaining 38 accessions were considered admixed due to <i>qI</i> &amp;lt; 0.70 and/or genotypes of bi-parental origin from different clusters (Additional file <a href="#MOESM1">1</a>). The average number of alleles per locus among clusters differed with values of 1.75, 2.23 and 1.34 corresponding to k1, k2 and k3, respectively (Table <a href="#Tab2">2</a>). Furthermore, clear differences were observed in the distribution of alleles per locus among clusters derived from primary population structure analysis. The majority of loci in the k1 cluster had 2 alleles (57%), nearly twice the 33% observed for single-allele loci. A more even distribution was observed in the k2 cluster with 22%, 37% and 36% for one, two and three alleles per locus. This rate of tri-allelic loci is in stark contrast to the &amp;lt; 1% in k1 and k3 clusters and suggests three homozygous, homeologous loci among a proportion of markers in k2 cluster. A majority of loci in k3 cluster were mono-allelic, twice that of the 30% bi-allelic loci. Finally, the number of alleles for loci of admixed accessions was relatively evenly distributed from 1 to 4 ranging from 18% to 32%. Five alleles were observed in an additional 4.3% of admixed loci. A total of 1812 alleles and an average of 2.58 alleles per locus (highest among all clusters) were detected for the 38 admixed accessions indicating heterozygosity. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 1</span> </a> </div>   <p>Primary and nested population structure for 180-accession panel of <i>Ocimum</i> species. Primary and secondary (nested) model-based clustering analysis using Structure ver 2.2.3 software for panels of <i>Ocimum</i> spp. accessions using 20 EST-SSR markers. <b>a</b> Major clusters ( <i>K</i> = 3) (top histogram) and sub-clusters ( <i>K</i> = 7) (bottom histogram) derived from primary and nested clustering iterations. Ten accessions were admixed and nested population structure could not be inferred due to unknown parentage of admixed primary cluster membership (white bars). <b>b</b> Major clusters (K = 1) (top histogram) and sub-clusters ( <i>K</i> = 2,5,7) (bottom three histograms). ΔK statistic values for K = 2-10 (right). Accessions are ordered according to the unweighted pair group method using arithmetic average (UPGMA) clustering </p>   </div> <p /> <p>Secondary (nested) population structure analysis provided evidence for seven clusters ( <i>K</i> = 7) among 142 non-admixed accessions (Fig. <a href="#Fig1">1a</a>). Two ( <i>K</i> = 2) and three ( <i>K</i> = 3) sub-clusters within k2 (k2.1 and k2.2) and k3 (k3.1, k3.2 and k3.3.), respectively, were strongly supported by the ΔK statistic (Additional file <a href="#MOESM4">4</a>). Greatest support was provided for two ( <i>K</i> = 2) sub-clusters, k1.1 and k1.2, among the 90-accession k1 primary cluster containing commercial sweet basil accessions. However, evidence was also provided for five ( <i>K</i> = 5) and seven ( <i>K</i> = 7) sub-clusters (Fig. <a href="#Fig1">1b</a>), suggesting cryptic population structure extant within this most economically important grouping. Nested population structure could not be determined for 9 accessions with admixed cluster membership from k1 and k3. Among these accessions, <i>qI</i> for either cluster (k1 or k3) failed to exceed 0.7 and could not be assigned due to unknown parentage. An additional 12 accessions had <i>qI</i> &amp;lt; 0.70 (Additional file <a href="#MOESM1">1</a>) after being subjected to the nested population structure analysis and were considered admixed. </p> <p>The average (Table <a href="#Tab2">2</a>) and distribution of the alleles per locus for sub-clusters was similar to the major primary clusters of k1 and k2. The range of alleles per locus among k1, k2 and k3 sub-clusters were 1.74-1.76, 2.18-2.27 and 1.30-1.41, respectively, indicating a high level of consistency with regard to allelic diversity. Interestingly, a relatively high percentage (11.2%) of null alleles occurred among accessions of sub-cluster k3.2. </p> </div> <div class="molecularvariance" title="sec"> <div class="title" tagx="title" title="title">
Molecular variance</div> <p>Analysis of molecular variance (AMOVA) was performed using 142 and 130 non-admixed accessions determined by primary and nested model-based cluster analysis, respectively. Inter-population differentiation was supported at the <i>P</i> &amp;lt; 0.001 level for primary and nested population structure with Φ <sub>PT</sub> values of 0.433 and 0.344, respectively (Table <a href="#Tab3">3</a>). The molecular variance due to differences within populations for nested clustering was 66% as compared to 57% from primary clustering. All pairwise Φ <sub>PT</sub> values for ( <i>K</i> = 3) clusters were significantly different ( <i>P</i> &amp;lt; 0.001) (Table <a href="#Tab4">4</a>). The majority of pairwise Φ <sub>PT</sub> values were highly significant ( <i>P</i> &amp;lt; 0.001) for the seven ( <i>K</i> = 7) clusters derived from nested analysis (Table <a href="#Tab5">5</a>). However, k3.1, k3.2 and k3.3 failed to demonstrate evidence for population differentiation based on AMOVA results (Table <a href="#Tab5">5</a>). These three sub-clusters contributed 51% of the total within population sum of squares (SS <sub>WP</sub>) while representing only 5.4% of the accessions included in the AMOVA (data not shown). Thus, high within population genetic variation coupled with a lower sample size may have lowered resolution for differentiation of these sub-clusters. <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 3</span>  </div></p><p>Analyses of molecular variance (AMOVA) for non-admixed panel of <i>Ocimum</i> spp. using 20 EST-SSR markers among clusters resulting from primary and secondary (nested) model-based cluster analyses </p>  <table> <thead> <tr> <th>Clustering Iteration</th> <th>Source</th> <th>df <sup>a</sup> </th> <th>Sum of squares</th> <th>Mean square</th> <th>Estimated σ <sup>2</sup> </th> <th>Molecular σ <sup>2</sup> </th> </tr> </thead> <tbody> <tr> <td>Primary <sup>b</sup> </td> <td>Among <sup>d</sup> </td> <td>2</td> <td>997.3</td> <td>498.6</td> <td>13.0</td> <td>43%</td> </tr> <tr> <td>Within <sup>e</sup> </td> <td>139</td> <td>2367.4</td> <td>17.0</td> <td>17.0</td> <td>57%</td> </tr> <tr> <td>Total</td> <td>141</td> <td>3364.7</td> <td /> <td>30.0</td> <td>100%</td> </tr> <tr> <td>Nested <sup>c</sup> </td> <td>Among</td> <td>6</td> <td>996.9</td> <td>166.1</td> <td>9.0</td> <td>34%</td> </tr> <tr> <td>Within</td> <td>123</td> <td>2108.7</td> <td>17.1</td> <td>17.1</td> <td>66%</td> </tr> <tr> <td>Total</td> <td>129</td> <td>3105.7</td> <td /> <td>26.1</td> <td>100%</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p> <sup>a</sup>Degrees of freedom </p> <p> <sup>b</sup>Primary model-based clustering iteration ( <i>K = 3</i>). Φ <sub>PT</sub> = 0.433; <i>p</i> &amp;lt; 0.001 </p> <p> <sup>c</sup>Nested model-based clustering iteration ( <i>K = 7</i>). Φ <sub>PT</sub> = 0.344; <i>p</i> &amp;lt; 0.001 </p> <p> <sup>d</sup> Among clusters </p> <p> <sup>e</sup>Within clusters </p> </div>  <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 4</span>  <p>Pairwise Φ <sub>PT</sub> estimates for clusters resulting from the primary model-based cluster analyses </p>  <table> <thead> <tr> <th>Cluster</th> <th>k1</th> <th>k2</th> <th>k3</th> </tr> </thead> <tbody> <tr> <td>k1</td> <td>0.000</td> <td>–</td> <td>–</td> </tr> <tr> <td>k2</td> <td>0.576</td> <td>0.000</td> <td>–</td> </tr> <tr> <td>k3</td> <td>0.392</td> <td>0.356</td> <td>0.000</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>All Φ <sub>PT</sub> estimates were highly significant ( <i>p</i> &amp;lt; 0.001) </p> </div> </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 5</span>  <p>Pairwise Φ <sub>PT</sub> estimates for resulting from the secondary model-based cluster analyses </p>  <table> <thead> <tr> <th>Sub-Cluster</th> <th>k1.1</th> <th>k1.2</th> <th>k2.1</th> <th>k2.2</th> <th>k3.1</th> <th>k3.2</th> <th>k3.3</th> </tr> </thead> <tbody> <tr> <td>k1.1</td> <td>0.000</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> </tr> <tr> <td>k1.2</td> <td>0.057</td> <td>0.000</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> </tr> <tr> <td>k2.1</td> <td>0.572</td> <td>0.600</td> <td>0.000</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> </tr> <tr> <td>k2.2</td> <td>0.539</td> <td>0.560</td> <td>0.042 <sup>NS</sup> </td> <td>0.000</td> <td>–</td> <td>–</td> <td>–</td> </tr> <tr> <td>k3.1</td> <td>0.376</td> <td>0.389</td> <td>0.354</td> <td>0.243 <sup>*</sup> </td> <td>0.000</td> <td>–</td> <td>–</td> </tr> <tr> <td>k3.2</td> <td>0.418</td> <td>0.406</td> <td>0.361</td> <td>0.289</td> <td>0.000 <sup>NS</sup> </td> <td>0.000</td> <td>–</td> </tr> <tr> <td>k3.3</td> <td>0.433</td> <td>0.420</td> <td>0.339</td> <td>0.276</td> <td>0.034 <sup>NS</sup> </td> <td>0.021 <sup>NS</sup> </td> <td>0.000</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p> <sup>NS</sup>not significantly different <i>p</i> &amp;gt; 0.05, <sup>*</sup> <i>p</i> &amp;lt; 0.05; all other Φ <sub>PT</sub> estimates were highly significant ( <i>p</i> &amp;lt; 0.001) </p> </div> </div> <p /> </div> <div class="characterizationofgeneticrelationships" title="sec"> <div class="title" tagx="title" title="title">
Characterization of genetic relationships</div> <p>A cophenetic correlation value of <i>r</i> = 0.934 indicated strong goodness of fit [ <a href="#CR46">46</a>] between the genetic similarity matrix used and the UPGMA dendrogram generated. Furthermore, the UPGMA dendrogram is in agreement with the placement of accessions for all major clusters ( <i>K</i> = 3) determined by population structure analysis (Fig. <a href="#Fig2">2a</a>- <a href="#Fig2">b</a>). The k1 cluster demonstrated generally lower genetic diversity as evidenced by high Jaccard similarity coefficient values (Fig. <a href="#Fig2">2a</a>). Importantly, accessions within the k1 cluster largely grouped consistent with known pedigree information available for 61% of accessions. The k2 cluster divides, according to nested population structure, into two clades of comparable genetic diversity. Although delineation of k3 into three sub-clusters was not supported by AMOVA, high bootstrap support values provide evidence of distinct groupings with high statistical backing (Fig. <a href="#Fig2">2b</a>). UPGMA grouping among k1 accessions did not demonstrate the same level of bootstrap support, however, this was not unexpected considering the ambiguity of nested model-based cluster analysis results. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 2</span> </a> </div>   <p>Primary population structure and UPGMA clustering for 180-accession panel of <i>Ocimum</i> species. Unweighted pair group method using arithmetic average (UPGMA) dendrogram aligned with primary model-based clustering of <i>Ocimum</i> spp. 180-accession panel and 2 outgroup accessions (1 and 2) using 20 EST-SSR markers. Genetic distance was calculated using the Jaccard Similarity Coefficient (x-axis) and bootstrap support values are the result of 1000 permutations with support value greater than 0.500 shown. Three clusters ( <i>K</i>) were inferred using Structure ver 2.2.3 and membership to each cluster is represented by proportion of yellow, orange or blue colors within horizontal lines corresponding to each accession. Numbers right of membership histograms correspond to accessions. Response to downy mildew is indicated resistant (R) = DS &amp;lt; 1.0, intermediate (I) = 1.0 &amp;lt; DS &amp;lt; 3.0 or susceptible (S) = DS &amp;gt; 3.0. Accessions with response R* share a single source of resistance conferred by accession MRI. The figure is divided into two segments <b>a</b> (left) and <b>b</b> (right) </p>   </div> <p /> <p>The k1 cluster roots to a single major node that can be further divided into three distinct groups (Fig. <a href="#Fig2">2a</a>). This k1 cluster includes a commercial sweet basil clade that begins with ‘Caesar’ (Accession 3) and ends with ‘Lettuce Leaf Heirloom’ (Accession 31). Included are all accessions classified as <i>O. basilicum</i> such as the commercially important variety ‘Nufar’ (acc. 19) originally developed for FOB resistance and widely cultivated worldwide. In addition, all chilling (acc. 8-16) and FOB (acc. 13, 20-23, 27) tolerant Rutgers breeding lines are included in this group (Fig. <a href="#Fig2">2a</a>). A second major k1 clade includes two lineages RUMS (acc. 33-39) and RU4S (acc. 40-46, 48-59) (Fig. <a href="#Fig2">2a</a>) derived from breeding for DM <sub>MRI</sub> resistance. The resistant genotypeMRI (acc. 47) is centrally located among accessions in these lineages. USDA-GRIN accession PI 197442 (acc. 60) and all remaining k1 accessions (Fig. <a href="#Fig2">2a</a>) vary substantially in phenotype and aromatic volatile composition (data not published). Accessions in this k1 clade are primarily cultivated for specialty or ornamental markets with the exception of ‘Eleonora’ (acc. 69), a variety marketed for commercial fresh market consumption. </p> <p>A k1 x k2 admixed, hybrid clade is supported by UPGMA analysis and can be observed at the top of Fig. <a href="#Fig2">2b</a>. This clade includes F <sub>1</sub>, F <sub>2</sub> or F <sub>2:3</sub> progeny with low or no fertility from a k1 x k2 bi-parental cross. PI 172996 (acc. 116) and ‘Sweet Dani’ (acc. 115) comprise all k2 parents with the exception of RU_SB22XLIME_F1 (acc. 97) and PI_652060XRU_SB17_F1 (acc. 98). </p> <p>The k2 cluster corresponds to a major clade, which is further divided into two groups (Fig. <a href="#Fig2">2b</a>). Nested population structure analysis (Fig. <a href="#Fig1">1a</a>) and bootstrap support values (0.504 and 0.744) (Fig. <a href="#Fig2">2b</a>) are both in agreement with this sub-division. Accessions in this cluster are described by their sources as <i>O. basilicum</i>, <i>O. citriodorum</i> or <i>O. americanum</i> (Additional file <a href="#MOESM1">1</a>). Three accessions cluster separately from all other k2 accessions and constitute a k2 x k3 hybrid grouping. Another k1 x k3 admixed clade is well-supported (0.938) and includes three F <sub>1</sub> hybrids derived from a cross between DM resistant k3 parent ‘Spice’ (acc. 139) and three different k1 accessions. Two additional USDA-GRIN accessions, PI 414201 (acc. 140) and PI 414203 (acc. 141), are included in this clade and parentage is unknown. </p> <p>Although the division of k3 into sub-clusters k3.1, k3.2 and k3.3 suggested by nested population structure analysis was not supported by the pairwise Φ <sub>PT</sub> estimates (Table <a href="#Tab5">5</a>), UPGMA provides evidence of clear delineations within this major cluster (Fig. <a href="#Fig2">2b</a>). The k3.1 cluster includes 7 phenotypically indistinguishable accessions sourced from commercial seed companies and the USDA-GRIN. This cluster is highly supported (1.00) and evidently an autonomous population (Fig. <a href="#Fig1">1a</a>; Fig. <a href="#Fig2">2b</a>). Another well-supported clade (0.997) is the k3.2 cluster (Fig. <a href="#Fig1">1a</a>; Fig. <a href="#Fig2">2b</a>), which includes 11 accessions beginning with PI 500952 (acc. 175) through PI 500942 (acc. 171). UPGMA analysis suggests this cluster represents a rather substantial juncture from the rest of the panel as it and subsequent clades are considered more basal than even the two <i>Nepeta cataria</i> outgroup accessions CN3 and CR9 according to UPGMA clustering. The k3.2 cluster is phenotypically homogenous and labeled <i>O. x africanum</i> according to the USDA-GRIN system. The k3.3 cluster includes basal most clades of the UPGMA dendrogram and the highest levels of genetic diversity based on Jaccard Similarity Coefficient genetic distance estimates shown as branch lengths in Fig. <a href="#Fig2">2b</a>. All accessions within this clade, supported by a 0.509 bootstrap value, are classified <i>O. gratissimum</i> according to their sources (Additional file <a href="#MOESM1">1</a>). Within this clade bootstrap support values of 0.875, 0.977 and 0.897 support grouping of three more closely related pairs of accessions (Fig. <a href="#Fig2">2b</a>). PI 511865 (acc. 182) is labeled <i>O. selloi</i> and groups (0.980) with the phenotypically similar ‘Green Pepper Basil’ (acc. 181) (Fig. <a href="#Fig2">2b</a>). <i>O. gratissimum</i> and <i>O. tenuiflorum</i> accessions were used as outgroups in a previous AFLP-based assessment of basil phylogenetics [ <a href="#CR12">12</a>], but did not include <i>O. selloi</i> found to be more basal in this study. </p> <p>A number of additional accessions have membership to the k3.3 sub-cluster yet are more distally located from the basal end of the dendrogram (Fig. <a href="#Fig1">1a</a>). This includes phenotypically similar PI 652052 (acc. 145) and ‘Camphor’ (acc. 149); the former is classified <i>O. x africanum</i> by the USDA-GRIN, but more likely <i>O. kilimandsharicum</i> based on morphology and as described by Ben-Naim et al. [41]. These accessions group within a clade supported by a bootstrap support value of 0.907 (Fig. <a href="#Fig2">2</a>). A second, well-supported (0.867) clade included those with inferred ancestry to k3.3 (Fig. <a href="#Fig1">1a</a>). These accessions are classified <i>O. tenuiflorum</i> by commercial and USDA-GRIN sources (Additional file <a href="#MOESM1">1</a>). Strong bootstrap support and phenotypic homogeneity among these two sets of accessions indicate UPGMA best resolves these k3 relationships. </p> </div> <div class="geographicdistributionamongclusters" title="sec"> <div class="title" tagx="title" title="title">
Geographic distribution among clusters</div> <p>In contrast to the greater panel, information regarding origin is available for the majority of k3 accessions (Additional file <a href="#MOESM1">1</a>). All accessions in the k3.2 cluster were collected in Zambia indicating this grouping may be native to south-central Africa. Accessions in k3.1 and k3.3 are far less geographically congruent. Sub-cluster k3.3 includes diverse New and Old World geographic collections ranging from South America (PI 511865; Uruguay) to South East Asia (PI 652055; Sri Lanka) to Sub-Saharan Africa (PI 652064; Tanzania). The collection locations of four known accessions in k2.1 are split between Turkey (PI 172996; PI 172998) and Iran (PI 253157; 296391) suggesting a Middle-Eastern grouping. The remaining k2.1 accessions are cultivars with unknown parentage developed for a volatile composition high in monoterpenoids geranial and neral (i.e. citral). The k2.2 sub-cluster includes PI 652060 and PI 652061 collected in Pakistan and India, respectively, however, geographic origin of the remaining five accessions are unknown. The k1 cluster is comprised of largely commercial and Rutgers breeding lines in which parentage or geographical origin of parents is unknown. Ten of 12 USDA-GRIN accessions were collected in Turkey, Iran or Macedonia. </p> </div> <div class="distributionofdmresistance" title="sec"> <div class="title" tagx="title" title="title">
Distribution of DM resistance</div> <p>Two unique sources of DM resistance (mean DS &amp;lt; 1.0), MRI (acc. 47) and Kivumbisi Lime x RUSB_17 F1 (acc. 83), were identified in the major cluster k1. The former accession has been used to develop DM <sub>MRI</sub> resistant breeding lines, while the latter is a sterile F <sub>1</sub> hybrid with DM resistance conferred from the admixed accession 153 ('Kivumbisi Lime'). Apart from these two sources of resistance, the k1 cluster was largely DM susceptible with a mean DS of 3.6. Seven accessions demonstrated an intermediate response to DM (1.0 &amp;lt; DS &amp;lt; 3.0) and may be incorporated into breeding strategies targeting durable resistance [ <a href="#CR26">26</a>]. </p> <p>The frequency of DM intermediate and resistant accessions increased with genetic distance from the k1 cluster. Intermediate resistance was observed among the majority of k2 accessions reflected by the 2.27 mean DS for this cluster. Interestingly, the k2.1 mean DS of 1.5 was lower than for k2.2 (mean DS = 3.4) for which the lowest DS was 2.9 (Fig. <a href="#Fig3">3</a>). Major cluster k3 is the primary source of DM resistance (mean DS = 0.36) with 29 resistant, 4 intermediate and one susceptible genotype. The highly supported (bootstrap = 0.867) clade of four accessions 156 - 159 exhibited hypersensitive response (HR). This response was not completely effective in preventing sporulation for these accessions such as 159 (PI 652057) (DS = 2.57) (Additional file <a href="#MOESM1">1</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 3</span> </a> </div>   <p>Downy mildew response phenotype distribution among <i>Ocimum</i> species clusters. Distribution of disease severity among sub-clusters derived from secondary (nested) model-based clustering analysis using Structure ver 2.2.3 software </p>   </div> <p /> <p>Fourteen F <sub>1</sub> hybrid accessions developed from DM resistant parents demonstrated resistance with the exception of SB17 x 172996 F1 (acc.100) with DS = 2.75. These results are in agreement with previous evaluation of F <sub>1</sub> progeny providing corroborating evidence of dominant gene action [ <a href="#CR41">41</a>]. Three k1 x k3 DM resistant F <sub>1</sub> hybrids developed from ‘Spice’ (acc.133), ‘Kivumbisi Lime’ (acc. 153) and ‘Camphor’ (acc. 149) exhibit introgressed k3 resistance but were completely sterile. Hybrids from k1 accessions crossed with DM resistant and intermediate k2.1 accessions PI 172996 (acc. 116) and ‘Sweet Dani’ (acc. 115) demonstrated low fertility that could be overcome, however, an intermediate and susceptible response was observed for the respective F1 progeny (Additional file <a href="#MOESM1">1</a>). Thus, unlike k3 accessions resistance did not appear to be under dominant gene control and requires further investigation to characterize the mechanism of DM response. </p> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <div class="populationstructureandgeneticdiversity" title="sec"> <div class="title" tagx="title" title="title">
Population structure and genetic diversity</div> <p>Model-based and UPGMA clustering analyses provide evidence of three distinct delineations and 15.6% admixture among the 180 <i>Ocimum</i> spp. accession panel. Nested population structure analysis provides additional evidence for seven sub-populations. Greatest support (Δ <i>K</i>) was provided for two distinct populations ( <i>K</i> = 2) nested within the most economically important <i>O. basilicum</i> k1 cluster. However, the possibility of five and seven sub-populations could not be discounted (Fig. <a href="#Fig1">1b</a>). Cryptic population structure in the k1 cluster may be attributed to extensive interbreeding, which is particularly prevalent in the <i>O. basilicum</i> species [ <a href="#CR47">47</a>]. High levels of admixture were also observed for the K = 5 and K = 7 population models (Fig. <a href="#Fig1">1b</a>), further supporting the exchange of genomic content potentially as a function of natural and purposeful outcrossing. This is most evident in the k1.2 cluster, where both admixture (Fig. <a href="#Fig1">1b</a>) and genetic distance increase (Fig. <a href="#Fig2">2a</a>) proceeding towards the basal end of the dendrogram containing a large proportion of cultivars and USDA-GRIN accessions (Additional file <a href="#MOESM1">1</a>). These results indicate plant breeding and natural outcrossing have maintained some level of genetic diversity within this economically salient cluster of accessions. </p> <p>The k1 cluster was the largest in this study with 90 accessions, yet best evidence was provided for only two sub-clusters, which was also the case for the 16-accessions in k2. Furthermore, the 36-accession k3 cluster resolved into three sub-clusters and may potentially contain additional populations based on interpretation of UPGMA analysis (Fig. <a href="#Fig2">2b</a>). While these results do not necessarily suggest a bottleneck in k1, the k2 and k3 clusters clearly exhibit far greater genetic diversity. Introgression of accessions from the k2 and k3 clusters would be advantageous in broadening the k1 ( <i>O. basilicum</i>) gene pool. </p> </div> <div class="taxonomicaldiscrepancies" title="sec"> <div class="title" tagx="title" title="title">
Taxonomical discrepancies</div> <p>Results of this study find genetic distance and population inference is often not correlated with reported accession collection location, phenotype or even species epithet. Conflicting and redundant nomenclature in the literature, the USDA-GRIN system and commercial seed sources have confounded accurate species-level assignment to important <i>Ocimum</i> spp. accessions [ <a href="#CR24">24</a>]. Phenotype appears to be somewhat predictive of genotype in clusters k3.1 and k3.2, however, leaf shape, habit, flower morphology and volatile composition are clearly heterogeneous among the remaining sub-clusters. For instance, k2.1 accessions 115 ('Sweet Dani') and 116 (PI 172996) group within a single clade (0.553), however, Sweet Dani exhibits no anthocyanins and a volatile composition that is 68% citral [ <a href="#CR8">8</a>], while PI 172996 exhibits leaf and stem anthocyanins and a volatile composition 91% methyl chavicol [USDA-GRIN]. Meanwhile, k1.2 cluster accesions 77 ('Queenette') and 78 ('Sweet Thai') contain stem anthocyanins and high methyl chavicol comparable to PI 172996. Although phenotype-based classification may provide some predictive measure of genetic relatedness, applications such as plant breeding require highly accurate and precise understanding of relations among accessions in order to construct effective selection strategies. </p> </div> <div class="ploidyandreproductivebarriers" title="sec"> <div class="title" tagx="title" title="title">
Ploidy and reproductive barriers</div> <p>Interesting differences in the mean and distribution of alleles per locus were observed among major and sub-clusters in this study. The majority of non-admixed accessions included in the panel are considered highly inbred and therefore homozygous across loci. Given this general assumption, some inferences can be made with regard to ploidy. A majority bi-allelic loci among k1 accessions suggests this cluster is an allotetraploid with two homozygous loci representing two sub-genomes. This system has been previously observed for the well-characterized allotetraploid genomes of <i>Brassica napus</i> [ <a href="#CR32">32</a>] and <i>Gossypium hirsutum</i> [ <a href="#CR48">48</a>]. One and two alleles per locus were observed in 33 and 57% of loci in the k1 cluster, respectively. These results are similar to the distribution of alleles per locus in a 240 EST-SSR marker survey using inbred k1 genotypes SB22 and MRI, in which a polymorphic subset also demonstrated disomic inheritance [ <a href="#CR17">17</a>]. </p> <p>Among accessions in the k2 cluster, a 36% rate of three alleles per locus provides evidence of three homeologous loci, suggesting an allohexaploid genome. Previous investigations of cytology and nuclei acid content for the species <i>O. africanum</i> (syn. <i>O. citriodorum</i>) [ <a href="#CR20">20</a>] suggested an allohexaploid (2n = 6× = 72) genome structure [ <a href="#CR11">11</a>, <a href="#CR13">13</a>, <a href="#CR18">18</a>, <a href="#CR19">19</a>]. Allelic distribution and cluster analysis in this study both support the hypothesis of an allohexaploid <i>O. africanum</i> k2 cluster and a need for nomenclature to be standardized across these accessions. Confusingly, USDA-GRIN accessions in the strongly supported (0.997) sub-cluster (k3.2) are labeled as <i>O. xafricanum</i>, illustrating the continued <i>Ocimum</i> spp. taxonomic issues and a need for revision of nomenclature. Hybridization of k1 and k2 accessions was previously characterized as an interspecific crosses with low fertility [ <a href="#CR11">11</a>, <a href="#CR19">19</a>]. All 17 k1 x k2 F <sub>1</sub>, F <sub>2</sub> and F <sub>2:3</sub> accessions in this study demonstrated low fertility. This is in agreement with a previous report of low fertility of the PI 172996 (acc. 116) x ‘Peri’ F <sub>1</sub> hybrid [ <a href="#CR41">41</a>]. In this study <i>O. basilicum</i> accession ‘Peri’ is replaced with SB17 (acc. 20) and used as recurrent parent to generate the RU172S BC <sub>1</sub> progeny (Fig. <a href="#Fig2">2a</a>). Three RU172S BC <sub>1</sub> individuals demonstrated increased fertility relative to the F <sub>1</sub>, suggesting restoration of reproductive viability may be correlated with increased k1 membership and providing a platform for development of near isogenic lines (NILs). Thus, introgression of traits such as disease resistance from k2 accessions to commercial sweet basil is possible through backcross breeding. </p> <p>A majority (60%) of single allele loci among k3 accessions was nearly two- and three-fold greater than k1 and k2, respectively. One of two scenarios are likely to explain the predominance of single-allele EST-SSRs among k3 accessions. First, it is possible that a divergent sub-genome is shared among k3 accessions, but does not have adequate homology for transferability of SSR markers derived from <i>O. basilicum</i> EST database. This would suggest a system similar to triplicated <i>Brassica</i> genomes in which <i>B. napus</i> AACC and <i>B. juncea</i> AABB have only the A genomes in common [ <a href="#CR49">49</a>]. A second possibility is the complete absence of a sub-genome, which would suggest k3 as a diploid cluster. A diploid genome hypothesis was previously proposed for k3 <i>O. gratissimum</i> and <i>O. tenuiforum</i> accessions [ <a href="#CR11">11</a>, <a href="#CR15">15</a>], which have consistently lower 2C-values [ <a href="#CR12">12</a>, <a href="#CR13">13</a>]. Furthermore, the recently sequenced genome of <i>O. tenuiflorum</i> genotype ‘CIM Ayu’ is described as a diploid (2n = 2× = 16) with a 386 Mbp genome [ <a href="#CR15">15</a>], more than ten-fold less than tetraploid <i>O. basilicum</i> based on 2C estimates [ <a href="#CR12">12</a>, <a href="#CR13">13</a>]. Koroch et al. [13] reported a 2C-value for k3.3 accession 160 (PI 652066, <i>O. campechianum</i>) significantly lower ( <i>p</i> &amp;lt; 0.001) than k3.1 accession 143 (PI 652059, <i>O. tenuiflorum</i>). Accession 182 (PI 511865) was the most basal accession in this study (Fig. <a href="#Fig2">2b</a>), yet is reported to have a 2C-value comparable to PI 652059 (acc. 143) [ <a href="#CR13">13</a>]. Thus, comparison of previously reported nucleic acid content with allelic distribution, genetic distance estimates and cluster analyses in this study suggest a complex <i>Ocimum</i> spp. genome evolution with multiple chromosomal accumulation and/or deletion events. </p> <p>Three F <sub>1</sub> progeny of k3.1 accession 139 ('Spice') hybridized with <i>O. basilicum</i> k1.1 accessions 22 (RUSB_09), 6 ('DiGenova') and 47 (MRI) form a well-supported (0.938) clade with admixed accessions 135 (PI 414201) and 136 (PI 414203) (Fig. <a href="#Fig2">2b</a>). Sterility among these progeny suggest a major reproductive barrier between commercially important <i>O. basilicum</i> k1.1 accessions and highly supported (1.00) k3 clade. Ben-Naim et al. reported F <sub>1</sub> sterility among progeny of DM resistant accessions 161 (PI 500945) and 168 (PI 500950) hybridized with <i>O. basilicum</i> accession ‘Peri’ [ <a href="#CR41">41</a>]. These accessions are found in the k2.2 sub-cluster suggesting the k1 x k3 F1 sterility barrier extends to this more basal clade (Fig. <a href="#Fig2">2b</a>). Surprising k1 <i>qI</i> among a number of non-hybrid k3 accessions 149-155 (Fig. <a href="#Fig2">2b</a>) suggests the possibility of recently shared ancestry between these clusters. However, sterility among Kivumbisi Lime x SB17 progeny demonstrates high k1.1 <i>qI</i> does not correspond to fertility. Viable F <sub>1</sub> progeny could not be obtained from cross-pollination of k1.1 and k3.3 accessions. This is consistent with at least one previous report of attempted <i>O. basilicum</i> x <i>O. tenuiflorum</i> hybridization [ <a href="#CR47">47</a>]. </p> <p>Although evidence of varying ploidy levels is a suspected cause of reproductive barriers among wide crosses [ <a href="#CR41">41</a>] the exact mechanism remains unclear. Comparative genomic analysis of cultivated and wild <i>Solanum</i> spp. demonstrate seed and pollen infertility can be controlled by a small number of loci with shared evolutionary history [ <a href="#CR50">50</a>]. Further investigation is needed to determine the cause of infertility among <i>Ocimum</i> spp. </p> </div> <div class="distributionofdmresistanceandbreeding" title="sec"> <div class="title" tagx="title" title="title">
Distribution of DM resistance and breeding</div> <p>Forty-three unique sources of genetic resistance to DM are identified from greenhouse screening in this study. These resistant accessions are heavily concentrated in the k3 cluster with 29 from 7 species according to the sources from which they were obtained (Additional file <a href="#MOESM1">1</a>). An additional 9 resistant accessions had majority k3 <i>qI</i> membership and UPGMA placement among k3 accessions (Fig. <a href="#Fig2">2b</a>). 116 (PI 172996), 117 (PI 172998) and 119 (PI 296391) were first identified as having little or no disease incidence by Pyne et al. 2013. These accessions demonstrated minimal leaf sporulation in this study (DS ≤ 0.63) and were determined to be in a population (k2) distinct from commercial sweet basil (k1). MRI (acc. 47) represents the only source of DM resistance (DM <sub>MRI</sub>) identified in the <i>O. basilicum</i> k1 cluster. Recently discovered QTL <i>dm11.1</i> demonstrated dominant gene action conferred by MRI (acc.47), but detection of additional minor QTL explained additional phenotypic variance and a somewhat complex mechanism for highest levels of resistance [ <a href="#CR17">17</a>]. </p> <p>These accessions represent candidates for field confirmation of resistance to <i>P. belbahrii</i> and, ultimately, DM resistance breeding. Ben-Naim et al. found DM response in greenhouse and field experiments to be largely correlated. Response of accessions common across this and previous greenhouse studies [ <a href="#CR41">41</a>, <a href="#CR42">42</a>] is generally consistent with minor differences. Differential disease response of genotypes across studies may be due to a number of confounding effects including environment, inoculum concentration and rating system [ <a href="#CR51">51</a>]. Another possibility of greater consequence to disease control is the potential occurrence of pathogen evolution resulting in races exhibiting differential host virulence. Proliferation of races is common among the most economically devastating downy mildews such as <i>Pseudoperonospora cubensis</i> [ <a href="#CR52">52</a>], <i>Bremia lactuca</i> [ <a href="#CR37">37</a>] and <i>Hyaloperonospora brassicae</i> [ <a href="#CR53">53</a>]. Phylogenetic characterization of <i>P. belbahrii</i> isolates is needed to determine the range of genetic diversity and identify possible extant pathogen races. </p> <p>Candidacy of accessions for DM resistance breeding is contingent upon sexual compatibility and reproductive capacity with commercial sweet basil found in sub-cluster k1.1 (Fig. <a href="#Fig2">2a</a>). DM resistance from k1.1 accession 47 (MRI) is the only documented characterization [ <a href="#CR17">17</a>, <a href="#CR54">54</a>] and introgression of genetic resistance beyond F <sub>1</sub> progeny. Partial sterility observed for k1 x k2 crosses appears to be surmountable by backcrossing as demonstrated by the ‘RU172S17’ BC <sub>1</sub> accessions 89-92 (Fig. <a href="#Fig2">2a</a>). This represents a potential strategy for introduction of k2 genomic DNA associated with important traits such as intermediate DM resistance (Table <a href="#Tab1">1</a>) and FOB resistance available in many of these accessions. An intermediate DM response (DS = 2.75) for the SB17 x PI 172996 F1 hybrid was significantly ( <i>P</i> &amp;lt; 0.05) higher than resistant parent PI 172996 and closer to the mid-parent DS of 2.18 suggesting resistance may be quantitative. The heritability of DM resistance from k2 accessions such as PI 172996 is unclear and requires further investigation. </p> <p>The k3 cluster represents a rich source of genetic resistance. Furthermore, consistent non-significant differences in disease response for F <sub>1</sub> progeny and their resistant parent indicate dominant gene action is widespread among these candidates [ <a href="#CR41">41</a>]. Qualitative and, in particular, single dominant gene control of DM is well documented across plant species [ <a href="#CR37">37</a>, <a href="#CR55">55</a>, <a href="#CR56">56</a>]. HR in <i>O. tenuiflorum</i> accessions 156-159 indicates effector triggered immunity [ <a href="#CR57">57</a>], a response distinct from the remaining k3 accessions in which no signs or symptoms were observed. It remains unclear whether these accessions have evolved unique resistance genes, but nonhost resistance is more likely among distant relatives of susceptible k1 accessions. Introgression of nonhost resistance from wild lettuce relatives <i>Lactuca serriola</i> and <i>Lactuca saligna</i> to commercial <i>L. sativa</i> is being used to build durable DM resistance [ <a href="#CR37">37</a>, <a href="#CR58">58</a>]. Further investigation is needed to identify redundancy of resistant genes, especially among closely related accessions. </p> </div> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>Results of this study provide a robust characterization of population structure, genetic diversity and response to DM among <i>Ocimum</i> spp. EST-SSRs provide a useful tool for increased resolution of this genus through continued germplasm collection and genotyping. Although evidence is concurrent for allotetraploid and allohexaploid genomes among economically important k1 and k2 clusters, respectively, the highly diverse and DM resistance-rich k3 cluster remains complicated by discrepancies in reports of genome size. Characterization of subgenomes as shared or distinct among allopolyploid species and identification of a diploid progenitor(s) are needed to elucidate genome structure. </p> <p>As breeding among specialty crops becomes increasingly prevalent, more sophisticated tools will be required. This trend is evidenced in <i>Ocimum</i> spp. by recent development of a first draft genome [ <a href="#CR15">15</a>], de novo meta-transcriptomics [ <a href="#CR59">59</a>, <a href="#CR60">60</a>] and genetic/QTL mapping [ <a href="#CR17">17</a>]. Determination of major and cryptic population structure as well as phylogenetic classification among DM resistant candidates provides another important resource for accelerated genetic improvement. </p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="plantmaterial" title="sec"> <div class="title" tagx="title" title="title">
Plant material</div> <p>A 180-accession panel of <i>Ocimum</i> spp. was selected to provide a genus-wide representation of genetic diversity, while targeting germplasm developed primarily for improved performance in important agronomic traits at Rutgers University (Additional file <a href="#MOESM1">1</a>). Eight chilling ("RUCB") and 7 <i>Fusarium oxysporum</i> f.sp. <i>basilici</i> (FOB) ("RUSB") tolerant <i>O. basilicum</i> inbred lines were selected. Twenty-nine breeding lines were included from two lineages ("RU4S" and "RUMS") selected for DM resistance. Eighteen F <sub>1</sub> hybrids were developed from hybridization of 9 <i>Ocimum</i> spp. accessions of varying DM response with 4 DM-susceptible <i>O. basilicum</i> inbred lines. All cross-pollinations were performed according to Pyne et al. [ <a href="#CR54">54</a>]. Seven F <sub>2</sub> selections and a single F <sub>2:3</sub> individual were included. Four first-generation backcross lines selected for FOB resistance ("RU172S") were also included. Remaining accessions were obtained from either commercial seed companies or the USDA-GRIN repository (Aimes, IA), ultimately representing 10 species according to these sources. Finally, two catnip ( <i>Nepeta cataria</i>) accessions (CN3 and CR9) were included as outgroup species. </p> </div> <div class="diseaserating(phenotypicevaluation)" title="sec"> <div class="title" tagx="title" title="title">
Disease rating (phenotypic evaluation)</div> <p>Seed from commercial, USDA-GRIN and inbred accessions were planted in Fafard Growing Mix 2 (Sun Gro Horticulture, Agawam, MA) and germinated under intermittent misting. Hybrid and non-flowering accessions (‘Pesto Perpetuo’, ‘GCB’ and ‘Pezou’) were cloned by vegetative cutting from mother plants under intermittent misting. The <i>P. belbahrii</i> isolate collected at the Rutgers Agricultural Research and Extension Center (RAREC) in 2013 and maintained on susceptible check variety ‘DiGenova’ was used in this study. Inoculum was prepared and applied to first true leaf sets according to Pyne et al. [ <a href="#CR61">61</a>]. Eight seedlings (individuals) per each accession were inoculated and disease ratings were performed 10 days post inoculation (DPI) corresponding to the interval at which disease severity plateaus under these greenhouse conditions. Individual plants were assigned a disease severity score using an ordered categorical scale in which 0 = no sporulation, 1 = 1-10%, 2 = 11-25%, 3 = 26-50%, 4 = 51-100% [ <a href="#CR54">54</a>]. A disease severity score (DS) was assigned to each accession from the average of 8 individuals for two, repeated experiments. </p> </div> <div class="est-ssrgenotyping" title="sec"> <div class="title" tagx="title" title="title">
EST-SSR genotyping</div> <p>Young leaves were harvested from axillary nodes of each accession in this study, frozen and then ground using mortar and pestle. Genomic DNA (gDNA) was extracted from ~ 80 mg of ground leaf tissue for all accessions using the E.N.Z.A. SP Plant DNA Kit (Omega BioTek, Norcross, GA). </p> <p>A set of 240 EST-SSR markers were previously developed and used to genotype the accessions MRI and SB22 [ <a href="#CR17">17</a>]. Preliminary experiments identified 20 di-, tri- and tetranucleotide repeat markers demonstrating reliable, unambiguous amplification across replicated PCR evaluations in a diverse subset of <i>Ocimum</i> spp. accessions. Given the putative allopolyploid genome structure of these species, markers were considered to be amplified in shared sub-genomes and, thus, orthologous. Seven of the 20 selected markers were previously mapped to a sweet basil linkage map developed from MRI x SB22 F <sub>2</sub> mapping population [ <a href="#CR17">17</a>]. All primers were synthesized by Integrated DNA technologies (Coralville, IA). Multiplexed PCR was performed using conditions and fluorescent dyes (FAM, NED, PET and VIC) previously described [ <a href="#CR17">17</a>]. Fragments generated from PCR were separated by capillary electrophoresis using an ABI 3500xL Genetic Analyzer (Life Technologies Corporation, Carlsbad, CA). Genemapper 4.1 (Applied Biosystems) software was used to determine fragment size for alleles across all accessions and exported for downstream analysis. </p> <p>Polyploidy is a confounding factor in population genetic studies due to the complication of allelic configuration among genotypes. In allopolyploid accessions of a single ploidy amplification of loci in independent sub-genomes provides a platform for conversion to a diploid SSR genotyping system [ <a href="#CR48">48</a>, <a href="#CR62">62</a>]. Accurate, multi-allelic (codominant) genotype assignment of complex germplasm panels with multiple ploidy levels in the absence of subgenome assignment exceeds current data analysis capabilities. Genotype data in this study was therefore scored as binary (presence = 1; absence/null = 0) as suggested by Honig et al. [ <a href="#CR63">63</a>]. SSR markers absent of any alleles (primer site mutation) or having many (&amp;gt; 6) alleles (non-specific primer binding) were considered null and coded ‘0’. Genotypes were coded ‘-1’ in the case of missing data. Polymorphic information content (PIC) was calculated for individual alleles using the formula 2P <sub>i</sub>Q <sub>i</sub>, where P <sub>i</sub> is the frequency of presence and Q <sub>i</sub> is the frequency of absence for a given allele [ <a href="#CR63">63</a>]. The binary genotype matrix for all accessions and SSR markers used in this study can be found in Additional file <a href="#MOESM5">5</a>. </p> </div> <div class="phenotypedataanalysis" title="sec"> <div class="title" tagx="title" title="title">
Phenotype data analysis</div> <p>Analysis of variance (ANOVA) was performed using R software to determine whether the effect of genotype was significant with respect to the response variable, DS. The agricolae package in R was used to perform DS mean separation using Tukey’s honest significant difference (HSD) test with α = 0.05. All accessions were included in this analysis with the exception of PI 511865, which was found to segregate and, thus, could not be treated as a single genotype. </p> </div> <div class="populationstructureanalysis" title="sec"> <div class="title" tagx="title" title="title">
Population structure analysis</div> <p>To determine population structure, a nested Bayesian model-based clustering approach described by [ <a href="#CR64">64</a>] was implemented with Structure ver 2.3.4 software [ <a href="#CR44">44</a>] to infer assignment of individuals to <i>K</i> clusters and sub-clusters. This approach has been effective in identifying important major and cryptic population structure in various plant species including apple [ <a href="#CR65">65</a>], cotton [ <a href="#CR48">48</a>] and rice [ <a href="#CR66">66</a>]. All simulations were performed using the ‘admixture model’ with ‘correlated allele frequencies’. The algorithm was executed with parameters set to a burnin of 100,000 followed by 200,000 Markov Chain Monte Carlo (MCMC) repetitions with 10 replicated runs per <i>K</i>. Initial (primary) Structure analysis was performed for <i>K</i> = 1 – 15 using all 180 accessions to determine major delineations in overall population structure. Evanno’s method [ <a href="#CR67">67</a>] was employed to provide an estimate of the most likely number of clusters using Structure Harvester software v0.6.94 [ <a href="#CR68">68</a>] to determine the natural logarithm probability Pr(X|K) and ad hoc ΔK statistic (mean(|L"( <i>K</i>)|) / sd(L( <i>K</i>)) from membership coefficient ( <i>qI</i>) matrices. Following selection of the optimal <i>K</i> for all replicated runs, the FULLSEARCH algorithm of CLUMPP software [ <a href="#CR69">69</a>] was used to identify the optimal permuted order of <i>qI</i> matrices with the greatest pairwise similarity. The average of this permutation provided the best <i>qI</i> matrix representation, which was visualized using DISTRUCT software [ <a href="#CR70">70</a>]. </p> <p>Determinations of the <i>qI</i> threshold at which to infer cluster assignment vary by study, but range from <i>qI</i> = 0.5 [ <a href="#CR71">71</a>] – 0.8 [ <a href="#CR65">65</a>]. Inclusion of wide crosses (F <sub>1</sub>, F <sub>2</sub> and BC <sub>1</sub>) in this study provided a measure of high admixture, which informed the selection of 0.70 as an appropriate <i>qI</i> threshold. Thus, accessions were assigned to the cluster for which <i>qI</i> &amp;gt; 0.70, while those with a maximum <i>qI</i> less than 0.70 were considered admixed. In addition, hybrid (F <sub>1</sub>, F <sub>2</sub>, F <sub>2:3</sub>) and first backcross (BC <sub>1</sub>) accessions with parentage assigned to two different clusters were considered admixed. </p> <p>Nested (secondary) Structure analysis was performed separately for non-admixed accessions of each cluster ascertained from the primary analysis. Algorithm parameters were identical to that of the primary analysis but included <i>K</i> = 1-10. Hybrid and backcross accessions were then assigned to sub-clusters by replacing the primary <i>qI</i> with secondary <i>qI</i> determined for each parent. Accessions for which parentage was unknown were excluded from secondary Structure analysis. Assignment to sub-clusters was repeated using the same admixture criteria as in the primary analysis. </p> </div> <div class="amova" title="sec"> <div class="title" tagx="title" title="title">
AMOVA</div> <p>The integrity of clusters derived from population structure was further investigated by performing an analysis of molecular variance (AMOVA) using GenAlEx 6.501 software [ <a href="#CR72">72</a>]. A Nei [ <a href="#CR73">73</a>] genetic distance matrix was first calculated using the binary genotype matrix previously described for the model-based clustering analyses. Two separate AMOVA were performed to partition within and among population estimated and molecular variance (σ <sup>2</sup>) components. Statistical significance of pairwise genetic distance among (i) primary and (ii) secondary clustering iterations were used to calculate the pairwise population PhiPT (Φ <sub>PT</sub>) test statistic, which provides a measure of interpopulation genetic diversity with intra-individual (heterozygosity) variation suppressed. The Φ <sub>PT</sub> statistic was calculated for all pairwise cluster combinations, which were determined significantly different ( <i>p</i> &amp;lt; 0.05) by 999 random permutations of the data. </p> </div> <div class="upgma" title="sec"> <div class="title" tagx="title" title="title">
UPGMA</div> <p>To investigate genetic relationships among accessions in the full panel, unweighted pair group method using arithmetic average (UPGMA) clustering was performed with Numerical Taxonomy System (NTSYSpc) ver 2.21q software (Exeter Software, Setauket, New York, USA) [ <a href="#CR46">46</a>]. Genotype data from outgroup accessions CN3 and CR9 ( <i>Nepata cataria</i>) were added to the same binary genotype matrix used for population structure and AMOVA analyses resulting in a total of 182 accessions. A genetic similarity matrix was generated using the Jaccard similarity coefficient method [ <a href="#CR74">74</a>] in the NTSYSpc SIMQUAL module. Cluster analysis was then performed by UPGMA in the SAHN module and the output was visualized as a dendrogram with the TREE module. A Mantel test was performed with 999 test permutations using the MXCOMP module to determine goodness of fit between the genetic similarity matrix and the UPGMA dendrogram converted to cophenetic values with the COPH module. Finally, the original binary genotype matrix was resampled 1000 times using the RESAMPLE module and the results were used as input for the CONSENS module to calculate bootstrap values using the majority rule method and a minimum support value of 0.500. </p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Additional files</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"> <a href="12870_2018_1284_MOESM1_ESM.pdf">LINK</a> <span class="label" tagx="label" title="label">Additional file 1:</span>  <p>Description of 180-accession panel of <i>Ocimum</i> spp., cluster membership and response to downy mildew ( <i>Peronospora belbahrii</i>) reported as disease severity. (PDF 115 kb) </p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="12870_2018_1284_MOESM2_ESM.pdf">LINK</a> <span class="label" tagx="label" title="label">Additional file 2:</span>  <p>Primer sequences and melting temperatures (T <sub>m</sub>) for the EST-SSRs used in this study. (PDF 52 kb) </p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="12870_2018_1284_MOESM3_ESM.pdf">LINK</a> <span class="label" tagx="label" title="label">Additional file 3:</span>  <p>Structure Harvester output for primary model-based clustering using Structure ver 2.3.4 software. Plot of Δ <i>K</i> (left) and LnP(D) (right) for <i>K</i> = 1 – 15. (PDF 126 kb) </p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="12870_2018_1284_MOESM4_ESM.pdf">LINK</a> <span class="label" tagx="label" title="label">Additional file 4:</span>  <p>Structure Harvester output for secondary (nested) model-based clustering using Structure ver 2.3.4 software. Plots of Δ <i>K</i> (left) and LnP(D) (right) for <i>K</i> = 1 – 10. Three nested analyses shown for k1 (top), k2 (middle) and k3 (bottom) primary clusters. (PDF 215 kb) </p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="12870_2018_1284_MOESM5_ESM.xlsx">LINK</a> <span class="label" tagx="label" title="label">Additional file 5:</span>  <p>Genotype data. Binary matrix of accessions (rows) and alleles (columns). (XLSX 186 kb)</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="glossary" title="glossary"> <div class="title" tagx="title" title="title">
Abbreviations</div> <div class="def-list" title="def-list"> <div class="def-item" title="def-item"> <div class="term" title="term">
AM</div> <div class="def" title="def"> <p>Association mapping</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
AMOVA</div> <div class="def" title="def"> <p>Analysis of molecular variance</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
ANOVA</div> <div class="def" title="def"> <p>Analysis of variance</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
BC</div> <div class="def" title="def"> <p>Backcross generation</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
cDNA</div> <div class="def" title="def"> <p>Chloroplast DNA</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
cM</div> <div class="def" title="def"> <p>CentiMorgan</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
DM</div> <div class="def" title="def"> <p>Downy mildew</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
DM <sub>MRI</sub> </div> <div class="def" title="def"> <p>MRI-conferred downy mildew resistance</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
DPI</div> <div class="def" title="def"> <p>Days post inoculation</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
DS</div> <div class="def" title="def"> <p>Disease severity</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
EST</div> <div class="def" title="def"> <p>Expressed sequence tag (EST)</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
FOB</div> <div class="def" title="def"> <p> <i>Fusarium oxysporum</i> f.sp. <i>basilica</i> </p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
gDNA</div> <div class="def" title="def"> <p>Genomic DNA</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
HR</div> <div class="def" title="def"> <p>Hypersensitive response</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
HSD</div> <div class="def" title="def"> <p>Tukey’s honest significant difference</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
ISSR</div> <div class="def" title="def"> <p>Inter-simple repeat markers</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
MCMC</div> <div class="def" title="def"> <p>Markov Chain Monte Carlo</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
NCBI</div> <div class="def" title="def"> <p>National Center for Biotechnology Information</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
NIL</div> <div class="def" title="def"> <p>Near isogenice lines</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
NTSYSpc</div> <div class="def" title="def"> <p>Numerical Taxonomy System</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PIC</div> <div class="def" title="def"> <p>Polymorphism information content</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SNP</div> <div class="def" title="def"> <p>Single nucleotide polymorphism</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SSR</div> <div class="def" title="def"> <p>Simple sequence repeat</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SS <sub>WP</sub> </div> <div class="def" title="def"> <p>Within population sum of squares</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
T <sub>m</sub> </div> <div class="def" title="def"> <p>Melting temperature</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
UPGMA</div> <div class="def" title="def"> <p>Unweighted pair group method using arithmetic average</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
USDA-GRIN</div> <div class="def" title="def"> <p>United States Department of Agriculture National Genetic Resources Program</p> </div> </div> </div> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p> <b>Electronic supplementary material</b> </p> <p>The online version of this article (10.1186/s12870-018-1284-7) contains supplementary material, which is available to authorized users. </p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We wish to thank Dr. Todd Wehner (North Carolina State University) and Dr. Stacy Bonos (Rutgers University) for their review and suggestions with regard to this manuscript. We thank the New Jersey Agricultural Experiment Station (NJAES) and the New Use Agriculture and Natural Plant Products Program (NUANNP) for their expertise and support of this research. </p> <div class="funding" title="sec"> <div class="title" tagx="title" title="title">
Funding</div> <p>Funds in support of this research were provided from multiple sources. A United States United Department of Agriculture (USDA) Special Crops Research Initiative Award No. 2011-51181-30646 funded a graduate assistantship/doctoral degree for RMP. This USDA-SCRI grant in combination with a USDA National Institute of Food and Agriculture (NIFA) Award No. 2016-68004-24931 provided funds for acquisition of commercial plant varieties, greenhouse space, DNA extractions and genotyping work. A Binational Agricultural Research &amp;amp; Development (BARD) Award No. US-4947-16R and NIFA HATCH projects NJAES 12131 and 1005685 provided additional funding for genotyping work and greenhouse facilities, respectively. </p> </div> <div class="availabilityofdataandmaterials" title="sec"> <div class="title" tagx="title" title="title">
Availability of data and materials</div> <p>A table of all genotypes and SSR allele calls used for statistical analysis in this study is submitted as Additional file <a href="#MOESM5">5</a>. </p> </div> </div> <div class="title" tagx="title" title="title">
Authors’ contributions</div> <p>Design of experimentation was performed by RMP, JES and CAW. CAW provided expertise in downy mildew screening including inoculation procedures and plant response phenotyping. RMP performed DNA extraction, genotyping and greenhouse downy mildew response phenotyping. JV coordinated laboratory work and provided expertise in SSR allele calling to ensure accurate DNA fragment length data. RMP performed data analyses with supervision JH. Interpretation of data was performed by RMP and JH. CAW, JH and JES supervised the writing and revision of this manuscript. JES and JH coordinated resources and personnel necessary to complete this research. All authors read and approved the final manuscript. </p> <div class="ethicsapprovalandconsenttoparticipate" title="sec"> <div class="title" tagx="title" title="title">
Ethics approval and consent to participate</div> <p>Not applicable.</p> </div> <div class="competinginterests" title="sec"> <div class="title" tagx="title" title="title">
Competing interests</div> <p>The authors declare that they have no competing interests.</p> </div> <div class="publisher’snote" title="sec"> <div class="title" tagx="title" title="title">
Publisher’s Note</div> <p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="CR1" /> <span class="label" tagx="label" title="label">1.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paton</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harley</span> <span class="given-names" tagx="given-names" title="given-names">MR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harley</span> <span class="given-names" tagx="given-names" title="given-names">MM</span> </span> </span> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hiltunen</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Holm</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title"> <i>Ocimum</i>: an overview of classification and relationships </span> <span class="source" tagx="source" title="source">Basil: the genus Ocimum</span> <span class="year" tagx="year" title="year">1999</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Amersterdam</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Harwood Academic Publishers</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">38</span> </span> </li> <li tag="ref"> <a name="CR2" /> <span class="label" tagx="label" title="label">2.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vieira</span> <span class="given-names" tagx="given-names" title="given-names">RF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goldsbrough</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Simon</span> <span class="given-names" tagx="given-names" title="given-names">JE</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Genetic diversity of basil ( <i>Ocimum</i> spp.) based on RAPD markers </span> <span class="source" tagx="source" title="source">J Am Soc Hortic Sci</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">128</span> <span class="fpage" tagx="fpage" title="fpage">94</span> <span class="lpage" tagx="lpage" title="lpage">99</span> </span> </li> <li tag="ref"> <a name="CR3" /> <span class="label" tagx="label" title="label">3.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paton</span> <span class="given-names" tagx="given-names" title="given-names">AJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Springate</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suddee</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Otieno</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grayer</span> <span class="given-names" tagx="given-names" title="given-names">RJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harley</span> <span class="given-names" tagx="given-names" title="given-names">MM</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Phylogeny and evolution of basils and allies (Ocimeae, Labiatae) based on three plastid DNA regions </span> <span class="source" tagx="source" title="source">Mol Phylogenet Evol</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">31</span> <span class="fpage" tagx="fpage" title="fpage">277</span> <span class="lpage" tagx="lpage" title="lpage">299</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ympev.2003.08.002">10.1016/j.ympev.2003.08.002</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15019625">15019625</a> </span> </span> </li> <li tag="ref"> <a name="CR4" /> <span class="label" tagx="label" title="label">4.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Simon</span> <span class="given-names" tagx="given-names" title="given-names">JE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Quinn</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Murray</span> <span class="given-names" tagx="given-names" title="given-names">JG</span> </span> </span> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Janick</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Simon</span> <span class="given-names" tagx="given-names" title="given-names">JE</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Basil: a source of essential oils</span> <span class="source" tagx="source" title="source">Advances in new crops</span> <span class="year" tagx="year" title="year">1990</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Portland</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Timber Press</span> <span class="fpage" tagx="fpage" title="fpage">484</span> <span class="lpage" tagx="lpage" title="lpage">489</span> </span> </li> <li tag="ref"> <a name="CR5" /> <span class="label" tagx="label" title="label">5.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Putievsky E, Galambosi B. Production systems of sweet basil. In: Hiltunen R, Holm Y, editors. Basil: the genus Ocimum. Amsterdam: Harwood Academic Publishers; 1999. p. 39–61. </span> </li> <li tag="ref"> <a name="CR6" /> <span class="label" tagx="label" title="label">6.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chaimovitsh</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dudai</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Putievesky</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ashri</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Inheritance of resistance to fusarium wilt in sweet basil</span> <span class="source" tagx="source" title="source">Phytopathology</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">90</span> <span class="fpage" tagx="fpage" title="fpage">58</span> <span class="lpage" tagx="lpage" title="lpage">60</span> </span> </li> <li tag="ref"> <a name="CR7" /> <span class="label" tagx="label" title="label">7.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Ribeiro P, Simon JE. Breeding sweet basil for chilling tolerance: ASHS; 2007. p. 302–5.</span> </li> <li tag="ref"> <a name="CR8" /> <span class="label" tagx="label" title="label">8.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morales</span> <span class="given-names" tagx="given-names" title="given-names">MR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Simon</span> <span class="given-names" tagx="given-names" title="given-names">JE</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">"Sweet Dani": a new culinary and ornamental basil</span> <span class="source" tagx="source" title="source">Hortscience</span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">1</span> <span class="fpage" tagx="fpage" title="fpage">148</span> <span class="lpage" tagx="lpage" title="lpage">149</span> </span> </li> <li tag="ref"> <a name="CR9" /> <span class="label" tagx="label" title="label">9.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nation</span> <span class="given-names" tagx="given-names" title="given-names">RG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Janick</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Simon</span> <span class="given-names" tagx="given-names" title="given-names">JE</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Estimation of outcrossing in basil</span> <span class="source" tagx="source" title="source">Hortic Sci</span> <span class="year" tagx="year" title="year">1992</span> <span class="volume" tagx="volume" title="volume">27</span> <span class="fpage" tagx="fpage" title="fpage">1221</span> <span class="lpage" tagx="lpage" title="lpage">1222</span> </span> </li> <li tag="ref"> <a name="CR10" /> <span class="label" tagx="label" title="label">10.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khosla</span> <span class="given-names" tagx="given-names" title="given-names">MK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tawi</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="collab'">collab: Group S</span> </span> <span class="mixed-article-title" title="mixed-article-title">Karyomorphological studies in genus Ocimum</span> <span class="source" tagx="source" title="source">Cytologia (Tokyo)</span> <span class="year" tagx="year" title="year">1985</span> <span class="volume" tagx="volume" title="volume">50</span> <span class="fpage" tagx="fpage" title="fpage">253</span> <span class="lpage" tagx="lpage" title="lpage">263</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1508/cytologia.50.253">10.1508/cytologia.50.253</a> </span> </span> </li> <li tag="ref"> <a name="CR11" /> <span class="label" tagx="label" title="label">11.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pushpangandan</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sobti</span> <span class="given-names" tagx="given-names" title="given-names">SN</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Cytogentical studies in the genus <i>Ocimum</i>. I. Origin of <i>O. americanum</i>, cytotaxonomical and experimental proof </span> <span class="source" tagx="source" title="source">Cytologia</span> <span class="year" tagx="year" title="year">1982</span> <span class="volume" tagx="volume" title="volume">47</span> <span class="fpage" tagx="fpage" title="fpage">575</span> <span class="lpage" tagx="lpage" title="lpage">583</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1508/cytologia.47.575">10.1508/cytologia.47.575</a> </span> </span> </li> <li tag="ref"> <a name="CR12" /> <span class="label" tagx="label" title="label">12.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carović-Stanko</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liber</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Besendorfer</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Javornik</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bohanec</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kolak</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Genetic relations among basil taxa (O <i>cimum</i> L.) based on molecular markers, nuclear DNA content, and chromosome number </span> <span class="source" tagx="source" title="source">Plant Syst Evol</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">285</span> <span class="fpage" tagx="fpage" title="fpage">13</span> <span class="lpage" tagx="lpage" title="lpage">22</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00606-009-0251-z">10.1007/s00606-009-0251-z</a> </span> </span> </li> <li tag="ref"> <a name="CR13" /> <span class="label" tagx="label" title="label">13.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koroch</span> <span class="given-names" tagx="given-names" title="given-names">AR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Michael</span> <span class="given-names" tagx="given-names" title="given-names">TP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dudai</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Simon</span> <span class="given-names" tagx="given-names" title="given-names">JE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Belanger</span> <span class="given-names" tagx="given-names" title="given-names">FC</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Estimation of nuclear DNA content of cultivated <i>Ocimum</i> species by using flow cytometry </span> <span class="source" tagx="source" title="source">Isr J Plant Sci</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">58</span> <span class="fpage" tagx="fpage" title="fpage">183</span> <span class="lpage" tagx="lpage" title="lpage">189</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1560/IJPS.59.3-4.183">10.1560/IJPS.59.3-4.183</a> </span> </span> </li> <li tag="ref"> <a name="CR14" /> <span class="label" tagx="label" title="label">14.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Escudero</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martín-Bravo</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mayrose</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fernández-Mazuecos</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fiz-Palacios</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hipp</span> <span class="given-names" tagx="given-names" title="given-names">AL</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Karyotypic changes through dysploidy persist longer over evolutionary time than polyploid changes </span> <span class="source" tagx="source" title="source">PLoS One</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">e85266</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0085266">10.1371/journal.pone.0085266</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24416374">24416374</a> </span> </span> </li> <li tag="ref"> <a name="CR15" /> <span class="label" tagx="label" title="label">15.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rastogi</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kalra</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gupta</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khan</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lal</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tripathi</span> <span class="given-names" tagx="given-names" title="given-names">AK</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Unravelling the genome of holy basil: an "incomparable" "elixir of life" of traditional Indian medicine </span> <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">413</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s12864-015-1640-z">10.1186/s12864-015-1640-z</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26017011">26017011</a> </span> </span> </li> <li tag="ref"> <a name="CR16" /> <span class="label" tagx="label" title="label">16.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ramsey</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schemske</span> <span class="given-names" tagx="given-names" title="given-names">DW</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Pathways, mechanisms, and rates of Polyploid formation in flowering plants</span> <span class="source" tagx="source" title="source">Annu Rev Ecol Syst</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">29</span> <span class="fpage" tagx="fpage" title="fpage">467</span> <span class="lpage" tagx="lpage" title="lpage">501</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1146/annurev.ecolsys.29.1.467">10.1146/annurev.ecolsys.29.1.467</a> </span> </span> </li> <li tag="ref"> <a name="CR17" /> <span class="label" tagx="label" title="label">17.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Pyne R, Honig J, Vaiciunas J, Koroch AR, Wyenandt CA, Bonos SA, et al. A first linkage map and downy mildew resistance QTL discovery for sweet basil ( <i>Ocimum basilicum</i>) facilitated by double digestion restriction site associated DNA sequencing (ddRADseq). PLoS One. 2017; 10.1371/journalpone.0184319. </span> </li> <li tag="ref"> <a name="CR18" /> <span class="label" tagx="label" title="label">18.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carović-Stanko</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liber</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Politeo</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Strikić</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kolak</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Milos</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Molecular and chemical characterization of the most widespread <i>Ocimum</i> species </span> <span class="source" tagx="source" title="source">Plant Syst Evol</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">294</span> <span class="fpage" tagx="fpage" title="fpage">253</span> <span class="lpage" tagx="lpage" title="lpage">262</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00606-011-0471-x">10.1007/s00606-011-0471-x</a> </span> </span> </li> <li tag="ref"> <a name="CR19" /> <span class="label" tagx="label" title="label">19.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paton</span> <span class="given-names" tagx="given-names" title="given-names">AJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Putievesky</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Taxonomic problems and cytotaxonomic relationships between and within varieties of <i>Ocimum basilicum</i> and related species (Labiateae) </span> <span class="source" tagx="source" title="source">Kew Bull</span> <span class="year" tagx="year" title="year">1996</span> <span class="volume" tagx="volume" title="volume">51</span> <span class="fpage" tagx="fpage" title="fpage">509</span> <span class="lpage" tagx="lpage" title="lpage">524</span> <span class="pub-id"> <a href="https://dx.doi.org/10.2307/4117026">10.2307/4117026</a> </span> </span> </li> <li tag="ref"> <a name="CR20" /> <span class="label" tagx="label" title="label">20.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Suddee S, Paton AJ, Parnell JAN. Taxonomic revision of tribe Ocimeae Dumort. (Lamiaceae) in continental South East Asia III. Ociminae. Kew Bull. 2005;60:3–75. </span> </li> <li tag="ref"> <a name="CR21" /> <span class="label" tagx="label" title="label">21.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carović-Stanko</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Šalinović</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grdǐa</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liber</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kolak</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Satovic</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Efficiency of morphological trait descriptors in discrimination of <i>Ocimum basilicum</i> L. accessions </span> <span class="source" tagx="source" title="source">Plant Biosyst</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">145</span> <span class="fpage" tagx="fpage" title="fpage">298</span> <span class="lpage" tagx="lpage" title="lpage">305</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1080/11263504.2011.558677">10.1080/11263504.2011.558677</a> </span> </span> </li> <li tag="ref"> <a name="CR22" /> <span class="label" tagx="label" title="label">22.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grayer</span> <span class="given-names" tagx="given-names" title="given-names">RJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kite</span> <span class="given-names" tagx="given-names" title="given-names">GC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goldstone</span> <span class="given-names" tagx="given-names" title="given-names">FJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bryan</span> <span class="given-names" tagx="given-names" title="given-names">SE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paton</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Putievsky</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Infraspecific taxonomy and essential oil chemotypes in sweet basil, <i>Ocimum basilicum</i> </span> <span class="source" tagx="source" title="source">Phytochemistry</span> <span class="year" tagx="year" title="year">1996</span> <span class="volume" tagx="volume" title="volume">43</span> <span class="fpage" tagx="fpage" title="fpage">1033</span> <span class="lpage" tagx="lpage" title="lpage">1039</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0031-9422(96)00429-3">10.1016/S0031-9422(96)00429-3</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8987875">8987875</a> </span> </span> </li> <li tag="ref"> <a name="CR23" /> <span class="label" tagx="label" title="label">23.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vieira</span> <span class="given-names" tagx="given-names" title="given-names">RF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Simon</span> <span class="given-names" tagx="given-names" title="given-names">JE</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Chemical characterization of basil (Ocimum spp.) based on volatile oils</span> <span class="source" tagx="source" title="source">Flavour Fragr J</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">21</span> <span class="fpage" tagx="fpage" title="fpage">214</span> <span class="lpage" tagx="lpage" title="lpage">221</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/ffj.1513">10.1002/ffj.1513</a> </span> </span> </li> <li tag="ref"> <a name="CR24" /> <span class="label" tagx="label" title="label">24.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paton</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harley</span> <span class="given-names" tagx="given-names" title="given-names">MR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harley</span> <span class="given-names" tagx="given-names" title="given-names">MM</span> </span> </span> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hiltunen</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Holm</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title"> <i>Ocimum</i>: an overview of classification and relationships </span> <span class="source" tagx="source" title="source">Basil: the genus <i>Ocimum</i> </span> <span class="year" tagx="year" title="year">1996</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Amsterdam</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Hardwood Academic</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">38</span> </span> </li> <li tag="ref"> <a name="CR25" /> <span class="label" tagx="label" title="label">25.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Simpson MG. Plant systematics. 2nd ed. Amsterdam: Elsevier/Academic Press; 2010.</span> </li> <li tag="ref"> <a name="CR26" /> <span class="label" tagx="label" title="label">26.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fehr</span> <span class="given-names" tagx="given-names" title="given-names">WR</span> </span> </span> <span class="source" tagx="source" title="source">Principles of cultivar development</span> <span class="year" tagx="year" title="year">1987</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">MacMillan Publishing Company</span> </span> </li> <li tag="ref"> <a name="CR27" /> <span class="label" tagx="label" title="label">27.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Labra</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miele</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ledda</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grassi</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mazzei</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sala</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Morphological characterization, essential oil composition and DNA genotyping of <i>Ocimum basilicum</i> L. cultivars </span> <span class="source" tagx="source" title="source">Plant Sci</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">167</span> <span class="fpage" tagx="fpage" title="fpage">725</span> <span class="lpage" tagx="lpage" title="lpage">731</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.plantsci.2004.04.026">10.1016/j.plantsci.2004.04.026</a> </span> </span> </li> <li tag="ref"> <a name="CR28" /> <span class="label" tagx="label" title="label">28.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vieira</span> <span class="given-names" tagx="given-names" title="given-names">RF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grayer</span> <span class="given-names" tagx="given-names" title="given-names">RJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paton</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Simon</span> <span class="given-names" tagx="given-names" title="given-names">JE</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Genetic diversity of <i>Ocimum gratissimum</i> L. based on volatile oil constituents, flavonoids and RAPD markers </span> <span class="source" tagx="source" title="source">Biochem Syst Ecol</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">29</span> <span class="fpage" tagx="fpage" title="fpage">287</span> <span class="lpage" tagx="lpage" title="lpage">304</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0305-1978(00)00062-4">10.1016/S0305-1978(00)00062-4</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11152947">11152947</a> </span> </span> </li> <li tag="ref"> <a name="CR29" /> <span class="label" tagx="label" title="label">29.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mishra</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baskaran</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shukla</span> <span class="given-names" tagx="given-names" title="given-names">AK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shasany</span> <span class="given-names" tagx="given-names" title="given-names">AK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sundaresan</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Higher efficiency of ISSR markers over plastid <i>psbA-trnH</i> region in resolving taxonomical status of genus <i>Ocimum</i> L </span> <span class="source" tagx="source" title="source">Ecol Evol</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">7671</span> <span class="lpage" tagx="lpage" title="lpage">7682</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/ece3.2483">10.1002/ece3.2483</a> </span> </span> </li> <li tag="ref"> <a name="CR30" /> <span class="label" tagx="label" title="label">30.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kalia</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rai</span> <span class="given-names" tagx="given-names" title="given-names">MK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kalia</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singh</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dhawan</span> <span class="given-names" tagx="given-names" title="given-names">AK</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Microsatellite markers: an overview of the recent progress in plants</span> <span class="source" tagx="source" title="source">Euphytica</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">177</span> <span class="fpage" tagx="fpage" title="fpage">309</span> <span class="lpage" tagx="lpage" title="lpage">334</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s10681-010-0286-9">10.1007/s10681-010-0286-9</a> </span> </span> </li> <li tag="ref"> <a name="CR31" /> <span class="label" tagx="label" title="label">31.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ramchiary</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nguyen</span> <span class="given-names" tagx="given-names" title="given-names">VD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hong</span> <span class="given-names" tagx="given-names" title="given-names">CP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dhandapani</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Choi</span> <span class="given-names" tagx="given-names" title="given-names">SR</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Genic microsatellite markers in <i>brassica rapa</i>: development, characterization, mapping, and their utility in other cultivated and wild brassica relatives </span> <span class="source" tagx="source" title="source">DNA Res</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">18</span> <span class="fpage" tagx="fpage" title="fpage">305</span> <span class="lpage" tagx="lpage" title="lpage">320</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/dnares/dsr017">10.1093/dnares/dsr017</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21768136">21768136</a> </span> </span> </li> <li tag="ref"> <a name="CR32" /> <span class="label" tagx="label" title="label">32.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Younas</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Development of a core set of single-locus SSR markers for allotetraploid rapeseed ( <i>Brassica napus</i> L.) </span> <span class="source" tagx="source" title="source">Theor Appl Genet</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">126</span> <span class="fpage" tagx="fpage" title="fpage">937</span> <span class="lpage" tagx="lpage" title="lpage">947</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00122-012-2027-z">10.1007/s00122-012-2027-z</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23238763">23238763</a> </span> </span> </li> <li tag="ref"> <a name="CR33" /> <span class="label" tagx="label" title="label">33.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Belbahri</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Calmin</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pawlowski</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lefort</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Phylogenetic analysis and real time PCR detection of a presumbably undescribed <i>Peronospora</i> species on sweet basil and sage </span> <span class="source" tagx="source" title="source">Mycol Res</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">109</span> <span class="fpage" tagx="fpage" title="fpage">1276</span> <span class="lpage" tagx="lpage" title="lpage">1287</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1017/S0953756205003928">10.1017/S0953756205003928</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16279421">16279421</a> </span> </span> </li> <li tag="ref"> <a name="CR34" /> <span class="label" tagx="label" title="label">34.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wyenandt</span> <span class="given-names" tagx="given-names" title="given-names">CA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Simon</span> <span class="given-names" tagx="given-names" title="given-names">JE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pyne</span> <span class="given-names" tagx="given-names" title="given-names">RM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Homa</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McGrath</span> <span class="given-names" tagx="given-names" title="given-names">MT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Basil downy mildew ( <i>Peronospora belbahrii</i>): discoveries and challenges relative to its control </span> <span class="source" tagx="source" title="source">Phytopathology</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">105</span> <span class="fpage" tagx="fpage" title="fpage">885</span> <span class="lpage" tagx="lpage" title="lpage">894</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1094/PHYTO-02-15-0032-FI">10.1094/PHYTO-02-15-0032-FI</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25894318">25894318</a> </span> </span> </li> <li tag="ref"> <a name="CR35" /> <span class="label" tagx="label" title="label">35.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cohen</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van den Langenberg</span> <span class="given-names" tagx="given-names" title="given-names">KM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wehner</span> <span class="given-names" tagx="given-names" title="given-names">TC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ojiambo</span> <span class="given-names" tagx="given-names" title="given-names">PS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hausbeck</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Quesada-Ocampo</span> <span class="given-names" tagx="given-names" title="given-names">LM</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Resurgence of <i>Pseudoperonospora cubensis</i>: the causal agent of cucurbit downy mildew </span> <span class="source" tagx="source" title="source">Phytopathology</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">105</span> <span class="fpage" tagx="fpage" title="fpage">998</span> <span class="lpage" tagx="lpage" title="lpage">1012</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1094/PHYTO-11-14-0334-FI">10.1094/PHYTO-11-14-0334-FI</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25844827">25844827</a> </span> </span> </li> <li tag="ref"> <a name="CR36" /> <span class="label" tagx="label" title="label">36.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Call</span> <span class="given-names" tagx="given-names" title="given-names">AD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Criswell</span> <span class="given-names" tagx="given-names" title="given-names">AD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wehner</span> <span class="given-names" tagx="given-names" title="given-names">TC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ando</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grumet</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Resistance of cucumber cultivars to a new strain of cucurbit downy mildew</span> <span class="source" tagx="source" title="source">Hortscience</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">47</span> <span class="fpage" tagx="fpage" title="fpage">171</span> <span class="lpage" tagx="lpage" title="lpage">178</span> </span> </li> <li tag="ref"> <a name="CR37" /> <span class="label" tagx="label" title="label">37.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Parra</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maisonneuve</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lebeda</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schut</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Christopoulou</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jeuken</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Rationalization of genes for resistance to <i>Bremia lactucae</i> in lettuce </span> <span class="source" tagx="source" title="source">Euphytica</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">210</span> <span class="fpage" tagx="fpage" title="fpage">309</span> <span class="lpage" tagx="lpage" title="lpage">326</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s10681-016-1687-1">10.1007/s10681-016-1687-1</a> </span> </span> </li> <li tag="ref"> <a name="CR38" /> <span class="label" tagx="label" title="label">38.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cohen</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vaknin</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ben-Naim</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rubin</span> <span class="given-names" tagx="given-names" title="given-names">AE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Galperin</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Silverman</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">First report of the occurrence and resistance to Mefenoxam of <i>Peronospora belbahrii</i>, causal agent of downy mildew of basil ( <i>Ocimum basilicum</i>) in Israel </span> <span class="source" tagx="source" title="source">Plant Dis Sci Soc</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">97</span> <span class="fpage" tagx="fpage" title="fpage">692</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1094/PDIS-12-12-1126-PDN">10.1094/PDIS-12-12-1126-PDN</a> </span> </span> </li> <li tag="ref"> <a name="CR39" /> <span class="label" tagx="label" title="label">39.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Pintore I, Gilardi G, Gullino ML, Garibaldi A. Detection of mefenoxam-resistant strains of <i>Peronospora belbahrii</i>, the causal agent of basil downy mildew, transmitted through infected seeds. Phytoparasitica. 2016; 10.1007/s12600-016-0538-x. </span> </li> <li tag="ref"> <a name="CR40" /> <span class="label" tagx="label" title="label">40.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Collina M, Merighi M, Turan C, Pirondi A, Minuto G, Brunelli A. First report of resistance of <i>Peronospora belbahrii</i>, causal agent of downy mildew of basil, to Mefenoxam in Italy. Plant Dis. 2016;100:1787. </span> </li> <li tag="ref"> <a name="CR41" /> <span class="label" tagx="label" title="label">41.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ben-Naim</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Falach</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cohen</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Resistance against basil downy mildew in <i>Ocimum</i> species </span> <span class="source" tagx="source" title="source">Phytopathology</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">105</span> <span class="fpage" tagx="fpage" title="fpage">778</span> <span class="lpage" tagx="lpage" title="lpage">785</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1094/PHYTO-11-14-0295-R">10.1094/PHYTO-11-14-0295-R</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25844828">25844828</a> </span> </span> </li> <li tag="ref"> <a name="CR42" /> <span class="label" tagx="label" title="label">42.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Djalali Farahani-Kofoet</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Romer</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grosch</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Selecting basil genotypes with resistance against downy mildew</span> <span class="source" tagx="source" title="source">Sci Hortic</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">179</span> <span class="fpage" tagx="fpage" title="fpage">248</span> <span class="lpage" tagx="lpage" title="lpage">255</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.scienta.2014.09.036">10.1016/j.scienta.2014.09.036</a> </span> </span> </li> <li tag="ref"> <a name="CR43" /> <span class="label" tagx="label" title="label">43.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wyenandt</span> <span class="given-names" tagx="given-names" title="given-names">CA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Simon</span> <span class="given-names" tagx="given-names" title="given-names">JE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McGrath</span> <span class="given-names" tagx="given-names" title="given-names">MT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ward</span> <span class="given-names" tagx="given-names" title="given-names">DL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Susceptibility of basil cultivars and breeding lines to downy mildew ( <i>Peronospora belbahrii</i>) </span> <span class="source" tagx="source" title="source">Hortscience</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">45</span> <span class="fpage" tagx="fpage" title="fpage">1416</span> <span class="lpage" tagx="lpage" title="lpage">1419</span> </span> </li> <li tag="ref"> <a name="CR44" /> <span class="label" tagx="label" title="label">44.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pritchard</span> <span class="given-names" tagx="given-names" title="given-names">JK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stephens</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Donnelly</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Inference of population structure using multilocus genotype data</span> <span class="source" tagx="source" title="source">Genetics</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">155</span> <span class="fpage" tagx="fpage" title="fpage">945</span> <span class="lpage" tagx="lpage" title="lpage">959</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10835412">10835412</a> </span> </span> </li> <li tag="ref"> <a name="CR45" /> <span class="label" tagx="label" title="label">45.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gore</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buckler</span> <span class="given-names" tagx="given-names" title="given-names">ES</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Status and prospects of association mapping in plants</span> <span class="source" tagx="source" title="source">Plant Genome</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">1</span> <span class="fpage" tagx="fpage" title="fpage">5</span> <span class="lpage" tagx="lpage" title="lpage">20</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3835/plantgenome2008.02.0089">10.3835/plantgenome2008.02.0089</a> </span> </span> </li> <li tag="ref"> <a name="CR46" /> <span class="label" tagx="label" title="label">46.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rohlf</span> <span class="given-names" tagx="given-names" title="given-names">FJ</span> </span> </span> <span class="source" tagx="source" title="source">NTSYSpc: numerical taxonomy system</span> <span class="year" tagx="year" title="year">2008</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Setauket</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Exeter Publishing, Ltd.</span> </span> </li> <li tag="ref"> <a name="CR47" /> <span class="label" tagx="label" title="label">47.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Putievesky</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paton</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lewinsohn</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ravid</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haimovich</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Katzir</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Crossability and relationship between <i>Ocimum basilicum</i> L </span> <span class="source" tagx="source" title="source">J Herbs Spices Med Plants</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">11</span> <span class="lpage" tagx="lpage" title="lpage">24</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1300/J044v06n03_02">10.1300/J044v06n03_02</a> </span> </span> </li> <li tag="ref"> <a name="CR48" /> <span class="label" tagx="label" title="label">48.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Genetic structure, linkage disequilibrium and association mapping of verticillium wilt resistance in elite cotton ( <i>Gossypium hirsutum</i> L.) germplasm population </span> <span class="source" tagx="source" title="source">PLoS One</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">e86308</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0086308">10.1371/journal.pone.0086308</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24466016">24466016</a> </span> </span> </li> <li tag="ref"> <a name="CR49" /> <span class="label" tagx="label" title="label">49.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Cheng F, Wu J, Wang X. Genome triplication drove the diversification of <i>Brassica</i> plants. Horticultural Research. 2014;1:14024. 10.1038/hortres.2014.24. </span> </li> <li tag="ref"> <a name="CR50" /> <span class="label" tagx="label" title="label">50.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Moyle LC, Nakazato T. Comparative genetics of hybrid incompatibility: sterility in two Solanum species crosses. Genetics. 2008;1453:1437–53. </span> </li> <li tag="ref"> <a name="CR51" /> <span class="label" tagx="label" title="label">51.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Agrios</span> <span class="given-names" tagx="given-names" title="given-names">GN</span> </span> </span> <span class="source" tagx="source" title="source">Plant pathology</span> <span class="year" tagx="year" title="year">2005</span> <span class="edition" tagx="edition" title="edition">5</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Burlington</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Elsevier Academic Press</span> </span> </li> <li tag="ref"> <a name="CR52" /> <span class="label" tagx="label" title="label">52.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Ojiambo PS, Gent DH, Quesada-Ocampo LM, Hausbeck MK, Holmes GJ. Epidemiology and population biology of <i>Pseudoperonospora cubensis</i>: a model system for Management of Downy Mildews. Annu Rev Phytopathol. 2015;53:223–46. </span> </li> <li tag="ref"> <a name="CR53" /> <span class="label" tagx="label" title="label">53.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vicente</span> <span class="given-names" tagx="given-names" title="given-names">JG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gunn</span> <span class="given-names" tagx="given-names" title="given-names">ND</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bailey</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pink</span> <span class="given-names" tagx="given-names" title="given-names">DAC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Holub</span> <span class="given-names" tagx="given-names" title="given-names">EB</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Genetics of resistance to downy mildew in <i>Brassica oleracea</i> and breeding towards durable disease control for UK vegetable production </span> <span class="source" tagx="source" title="source">Plant Pathol</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">61</span> <span class="fpage" tagx="fpage" title="fpage">600</span> <span class="lpage" tagx="lpage" title="lpage">609</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-3059.2011.02539.x">10.1111/j.1365-3059.2011.02539.x</a> </span> </span> </li> <li tag="ref"> <a name="CR54" /> <span class="label" tagx="label" title="label">54.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pyne</span> <span class="given-names" tagx="given-names" title="given-names">RM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koroch</span> <span class="given-names" tagx="given-names" title="given-names">AR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wyenandt</span> <span class="given-names" tagx="given-names" title="given-names">CA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Simon</span> <span class="given-names" tagx="given-names" title="given-names">JE</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Inheritance of resistance to downy mildew in sweet basil</span> <span class="source" tagx="source" title="source">J Am Soc Hortic Sci</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">140</span> <span class="fpage" tagx="fpage" title="fpage">396</span> <span class="lpage" tagx="lpage" title="lpage">403</span> </span> </li> <li tag="ref"> <a name="CR55" /> <span class="label" tagx="label" title="label">55.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Irish</span> <span class="given-names" tagx="given-names" title="given-names">BM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Correll</span> <span class="given-names" tagx="given-names" title="given-names">JC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feng</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bentley</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">de Los Reyes</span> <span class="given-names" tagx="given-names" title="given-names">BG</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Characterization of a resistance locus ( <i>Pfs-1</i>) to the spinach downy mildew pathogen ( <i>Peronospora farinosa</i> f. sp. <i>spinaciae</i>) and development of a molecular marker linked to Pfs-1 </span> <span class="source" tagx="source" title="source">Phytopathology</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">98</span> <span class="fpage" tagx="fpage" title="fpage">894</span> <span class="lpage" tagx="lpage" title="lpage">900</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1094/PHYTO-98-8-0894">10.1094/PHYTO-98-8-0894</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18943207">18943207</a> </span> </span> </li> <li tag="ref"> <a name="CR56" /> <span class="label" tagx="label" title="label">56.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Epinat</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pitrat</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="source" tagx="source" title="source">Inheritance of resistance of three lines of muskmelon ( <i>Cucumis melo</i>) to downy mildew ( <i>Pseudoperonospora cubensis</i>) </span> <span class="year" tagx="year" title="year">1989</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Charleston</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Proc Cucurbitaceae: Evaluation and Enhancement of Cucurbit Germplasm</span> <span class="fpage" tagx="fpage" title="fpage">133</span> <span class="lpage" tagx="lpage" title="lpage">135</span> </span> </li> <li tag="ref"> <a name="CR57" /> <span class="label" tagx="label" title="label">57.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jones</span> <span class="given-names" tagx="given-names" title="given-names">JDG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dangl</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The plant immune system</span> <span class="source" tagx="source" title="source">Nature</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">444</span> <span class="fpage" tagx="fpage" title="fpage">323</span> <span class="lpage" tagx="lpage" title="lpage">329</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nature05286">10.1038/nature05286</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17108957">17108957</a> </span> </span> </li> <li tag="ref"> <a name="CR58" /> <span class="label" tagx="label" title="label">58.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">NW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lindhout</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Niks</span> <span class="given-names" tagx="given-names" title="given-names">RE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jeuken</span> <span class="given-names" tagx="given-names" title="given-names">MJW</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Genetic dissection of <i>Lactuca saligna</i> nonhost resistance to downy mildew at various lettuce developmental stages </span> <span class="source" tagx="source" title="source">Plant Pathol</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">58</span> <span class="fpage" tagx="fpage" title="fpage">923</span> <span class="lpage" tagx="lpage" title="lpage">932</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-3059.2009.02066.x">10.1111/j.1365-3059.2009.02066.x</a> </span> </span> </li> <li tag="ref"> <a name="CR59" /> <span class="label" tagx="label" title="label">59.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Guo L, Allen KS, Deiulio GA, Zhang Y, Madeiras AM, Wick RL, et al. A de-novo-assembly-based data analysis pipeline for plant obligate parasite metatranscriptomic studies. Front Plant Sci. 2016;7:925. 10.3389/FPLS.2016.00925. </span> </li> <li tag="ref"> <a name="CR60" /> <span class="label" tagx="label" title="label">60.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rastogi</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meena</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bhattacharya</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ghosh</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shukla</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sangwan</span> <span class="given-names" tagx="given-names" title="given-names">NS</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">De novo sequencing and comparative analysis of holy and sweet basil transcriptomes</span> <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">15</span> <span class="fpage" tagx="fpage" title="fpage">588</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-15-588">10.1186/1471-2164-15-588</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25015319">25015319</a> </span> </span> </li> <li tag="ref"> <a name="CR61" /> <span class="label" tagx="label" title="label">61.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pyne</span> <span class="given-names" tagx="given-names" title="given-names">RM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koroch</span> <span class="given-names" tagx="given-names" title="given-names">AR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wyenandt</span> <span class="given-names" tagx="given-names" title="given-names">CA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Simon</span> <span class="given-names" tagx="given-names" title="given-names">JE</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A rapid screening approach to identify resistance to basil downy mildew ( <i>Peronospora belbahrii</i>) </span> <span class="source" tagx="source" title="source">Hortscience</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">49</span> <span class="fpage" tagx="fpage" title="fpage">1041</span> <span class="lpage" tagx="lpage" title="lpage">1045</span> </span> </li> <li tag="ref"> <a name="CR62" /> <span class="label" tagx="label" title="label">62.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">van Dijk</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Noordijk</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dubos</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bink</span> <span class="given-names" tagx="given-names" title="given-names">MC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meulenbroek</span> <span class="given-names" tagx="given-names" title="given-names">BJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Visser</span> <span class="given-names" tagx="given-names" title="given-names">RG</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Microsatellite allele dose and configuration establishment (MADCE): an integrated approach for genetic studies in allopolyploids </span> <span class="source" tagx="source" title="source">BMC Plant Biol</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="fpage" tagx="fpage" title="fpage">25</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2229-12-25">10.1186/1471-2229-12-25</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22340438">22340438</a> </span> </span> </li> <li tag="ref"> <a name="CR63" /> <span class="label" tagx="label" title="label">63.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Honig</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kubik</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Averello</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vaiciunas</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meyer</span> <span class="given-names" tagx="given-names" title="given-names">WA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bonos</span> <span class="given-names" tagx="given-names" title="given-names">SA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Classification of bentgrass ( <i>Agrostis</i>) cultivars and accessions based on microsatellite (SSR) markers </span> <span class="source" tagx="source" title="source">Genet Resour Crop Evol</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">63</span> <span class="fpage" tagx="fpage" title="fpage">1139</span> <span class="lpage" tagx="lpage" title="lpage">1160</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s10722-015-0307-6">10.1007/s10722-015-0307-6</a> </span> </span> </li> <li tag="ref"> <a name="CR64" /> <span class="label" tagx="label" title="label">64.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vähä</span> <span class="given-names" tagx="given-names" title="given-names">JP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Erkinaro</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Niemelä</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Primmer</span> <span class="given-names" tagx="given-names" title="given-names">CR</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Life-history and habitat features influence the within-river genetic structure of Atlantic salmon </span> <span class="source" tagx="source" title="source">Mol Ecol</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">2638</span> <span class="lpage" tagx="lpage" title="lpage">2654</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-294X.2007.03329.x">10.1111/j.1365-294X.2007.03329.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17594436">17594436</a> </span> </span> </li> <li tag="ref"> <a name="CR65" /> <span class="label" tagx="label" title="label">65.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Urrestarazu</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Denancé</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ravon</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guyader</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guisnel</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feugey</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Analysis of the genetic diversity and structure across a wide range of germplasm reveals prominent gene flow in apple at the European level </span> <span class="source" tagx="source" title="source">BMC Plant Biol</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">130</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s12870-016-0818-0">10.1186/s12870-016-0818-0</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27277533">27277533</a> </span> </span> </li> <li tag="ref"> <a name="CR66" /> <span class="label" tagx="label" title="label">66.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Nachimuthu VV, Muthurajan R, Duraialaguraja S, Sivakami R, Pandian BA, Ponniah G, et al. Analysis of population structure and genetic diversity in rice germplasm using SSR markers: an initiative towards association mapping of agronomic traits in Oryza Sativa. Rice (N Y). 2015;8:30. 10.1186/s12284-015-0062-5. </span> </li> <li tag="ref"> <a name="CR67" /> <span class="label" tagx="label" title="label">67.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Evanno G, Regnaut S, Goudet J. Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study. Mol Ecol. 2005;14:2611–20. </span> </li> <li tag="ref"> <a name="CR68" /> <span class="label" tagx="label" title="label">68.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Earl DA, Bridgett M. STRUCTURE HARVESTER: a website and program for visualizing STRUCTURE output and implementing the Evanno method. Conserv Genet Resour. 2012;4:359–61 </span> </li> <li tag="ref"> <a name="CR69" /> <span class="label" tagx="label" title="label">69.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jakobsson</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rosenberg</span> <span class="given-names" tagx="given-names" title="given-names">NA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">CLUMPP: a cluster matching and permutation program for dealing with label switching and multimodality in analysis of population structure </span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">23</span> <span class="fpage" tagx="fpage" title="fpage">1801</span> <span class="lpage" tagx="lpage" title="lpage">1806</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/btm233">10.1093/bioinformatics/btm233</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17485429">17485429</a> </span> </span> </li> <li tag="ref"> <a name="CR70" /> <span class="label" tagx="label" title="label">70.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Rosenberg NA. DISTRUCT: a program for the graphical display of population structure. Mol Ecol Resour. 2004;4:137–8. </span> </li> <li tag="ref"> <a name="CR71" /> <span class="label" tagx="label" title="label">71.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Taranto</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">D’Agostino</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Greco</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cardi</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tripodi</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Genome-wide SNP discovery and population structure analysis in pepper ( <i>Capsicum annuum</i>) using genotyping by sequencing </span> <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">17</span> <span class="fpage" tagx="fpage" title="fpage">943</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s12864-016-3297-7">10.1186/s12864-016-3297-7</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27871227">27871227</a> </span> </span> </li> <li tag="ref"> <a name="CR72" /> <span class="label" tagx="label" title="label">72.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peakall</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smouse</span> <span class="given-names" tagx="given-names" title="given-names">PE</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">GenAlEx 6.5: genetic analysis in excel. Population genetic software for teaching and research — an update </span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">28</span> <span class="fpage" tagx="fpage" title="fpage">2537</span> <span class="lpage" tagx="lpage" title="lpage">2539</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/bts460">10.1093/bioinformatics/bts460</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22820204">22820204</a> </span> </span> </li> <li tag="ref"> <a name="CR73" /> <span class="label" tagx="label" title="label">73.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nei</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Genetic distance between populations</span> <span class="source" tagx="source" title="source">Am Soc Nat</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">106</span> <span class="fpage" tagx="fpage" title="fpage">283</span> <span class="lpage" tagx="lpage" title="lpage">292</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1086/282771">10.1086/282771</a> </span> </span> </li> <li tag="ref"> <a name="CR74" /> <span class="label" tagx="label" title="label">74.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jaccard</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Nouvelles recherches Sur la distribution florale</span> <span class="source" tagx="source" title="source">Bull la Soc Vaudoise des Sci Nat</span> <span class="year" tagx="year" title="year">1908</span> <span class="volume" tagx="volume" title="volume">44</span> <span class="fpage" tagx="fpage" title="fpage">223</span> <span class="lpage" tagx="lpage" title="lpage">270</span> </span> </li> </ul> </div> </div>  </body></html>