<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Front Plant Sci</span> <span class="iso-abbrev" title="iso-abbrev">Front Plant Sci</span> <span class="publisher-id" title="publisher-id">Front. Plant Sci.</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Frontiers in Plant Science</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1664-462X</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Frontiers Media S.A.</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5909171">5909171</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3389/fpls.2018.00418">10.3389/fpls.2018.00418</a> </span> <div class="article-categories" title="article-categories"> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Science</span>: <span class="subject" title="subject">Review</span> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
The Sequenced Angiosperm Genomes and Genome Databases</div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">Fei</span> </span> <a href="#aff1"> <sup>1</sup> </a> <a href="#fn002"> <sup>†</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/441747/overview" /> </span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dong</span> <span class="given-names" tagx="given-names" title="given-names">Wei</span> </span> <a href="#aff1"> <sup>1</sup> </a> <a href="#fn002"> <sup>†</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/504756/overview" /> </span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Jiawei</span> </span> <a href="#aff1"> <sup>1</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/542847/overview" /> </span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">Xinyue</span> </span> <a href="#aff1"> <sup>1</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/506812/overview" /> </span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">Junhao</span> </span> <a href="#aff2"> <sup>2</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/542670/overview" /> </span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Zhengjia</span> </span> <a href="#aff2"> <sup>2</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/542838/overview" /> </span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">Zhenguo</span> </span> <a href="#aff3"> <sup>3</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/531568/overview" /> </span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">Haibao</span> </span> <a href="#aff1"> <sup>1</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/61466/overview" /> </span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Liangsheng</span> </span> <a href="#aff1"> <sup>1</sup> </a> <a href="#c001"> <sup>*</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/191083/overview" /> </span> </span> </div> <span class="citation_author_institution" id="aff1">[], <sup>1</sup> <span class="institution" title="institution">State Key Laboratory of Ecological Pest Control for Fujian and Taiwan Crops, College of Life Sciences, Fujian Provincial Key Laboratory of Haixia Applied Plant Systems Biology, Ministry of Education Key Laboratory of Genetics, Breeding and Multiple Utilization of Corps, Fujian Agriculture and Forestry University </span> <span class="addr-line" title="addr-line">Fuzhou</span> <span class="country" tagx="country" title="country">China</span> </span> <span class="citation_author_institution" id="aff2">[], <sup>2</sup> <span class="institution" title="institution">State Key Laboratory of Subtropical Silviculture, School of Forestry and Biotechnology, Zhejiang Agriculture and Forestry University </span> <span class="addr-line" title="addr-line">Hangzhou</span> <span class="country" tagx="country" title="country">China</span> </span> <span class="citation_author_institution" id="aff3">[], <sup>3</sup> <span class="institution" title="institution">Department of Biology, Saint Louis University</span> <span class="addr-line" title="addr-line">St. Louis, MO</span> <span class="country" tagx="country" title="country">United States</span> </span> <div class="author-notes" title="author-notes"> <div class="fn-type-edited-by" title="edited-by"> <p>Edited by: Santosh Kumar Upadhyay, Panjab University, India</p> </div> <div class="fn-type-edited-by" title="edited-by"> <p>Reviewed by: Sumit Kumar Bag, National Botanical Research Institute (CSIR), India; Xiyin Wang, North China University of Science and Technology, China </p> </div> <div class="corresp" title="corresp">
*Correspondence: Liangsheng Zhang <span class="email" tagx="email" title="email">fafuzhang@163.com</span> </div> <div class="fn-type-other" title="other"> <p>This article was submitted to Bioinformatics and Computational Biology, a section of the journal Frontiers in Plant Science </p> </div> <div class="fn-type-other" title="other"> <p>†Co-first authors.</p> </div> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-4-4</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2018</span> </span> <span class="volume" tagx="volume" title="volume">9</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">418</span> <span class="history" title="history"> <span class="received" title="received">received: 2017-12-11</span> <span class="accepted" title="accepted">accepted: 2018-3-15</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Chen, Dong, Zhang, Guo, Chen, Wang, Lin, Tang and Zhang</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2018</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Angiosperms, the flowering plants, provide the essential resources for human life, such as food, energy, oxygen, and materials. They also promoted the evolution of human, animals, and the planet earth. Despite the numerous advances in genome reports or sequencing technologies, no review covers all the released angiosperm genomes and the genome databases for data sharing. Based on the rapid advances and innovations in the database reconstruction in the last few years, here we provide a comprehensive review for three major types of angiosperm genome databases, including databases for a single species, for a specific angiosperm clade, and for multiple angiosperm species. The scope, tools, and data of each type of databases and their features are concisely discussed. The genome databases for a single species or a clade of species are especially popular for specific group of researchers, while a timely-updated comprehensive database is more powerful for address of major scientific mysteries at the genome scale. Considering the low coverage of flowering plants in any available database, we propose construction of a comprehensive database to facilitate large-scale comparative studies of angiosperm genomes and to promote the collaborative studies of important questions in plant biology. </p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
angiosperm genomes</div> <div class="kwd" title="kwd">
genome database</div> <div class="kwd" title="kwd">
data sharing</div> <div class="kwd" title="kwd">
big data</div> <div class="kwd" title="kwd">
comparative genomics</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
National Natural Science Foundation of China <span class="named-content" title="named-content">10.13039/501100001809</span> </div> <span class="award-id" tagx="award-id" title="award-id">81502437</span> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
State Key Laboratory of Ecological Pest Control for Fujian and Taiwan Crops</div> <span class="award-id" tagx="award-id" title="award-id">SKB2017004</span> </div> </div> <div class="counts" title="counts"> <span class="fig-count" title="fig-count">fig-count: </span> <span class="table-count" title="table-count">table-count: </span> <span class="equation-count" title="equation-count">equation-count: </span> <span class="ref-count" title="ref-count">ref-count: </span> <span class="page-count" title="page-count">page-count: </span> <span class="word-count" title="word-count">word-count: </span> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>The "green lineage" or the plant kingdom comprises ~4,000 chlorophyta algae, 865 charophyta algae, 25,100 bryophytes, 1,340 lycophytes, 12,400 pteridophytes, 766 gymnosperms (Pryer et al., <a href="#B25">2002</a>), and ~350,000 angiosperms (or flowering plants, estimated by <a href="http://www.theplantlist.org">www.theplantlist.org</a>). Therefore, angiosperm is by far the most diverse group among all clades of the green lineage (Figure <a href="#F1">1A</a>). Originated from a single ancestor at about 167–199 mya (Bell et al., <a href="#B4">2010</a>), angiosperms have diverged into 8 extant clades, including Amborellales, Nymphaeales, Austrobaileyales, monocots, Magnoliids, Ceratophyllales, Chloranthales, and Eudicots (Zeng et al., <a href="#B37">2014</a>). Only one species is found in the basal branch angiosperm clade Amborellales, whereas the largest angiosperm clade eudicot contains ~262,000 species (Zeng et al., <a href="#B37">2014</a>; Figures <a href="#F1">1B,C</a>). Compared to other green lineage clades, the angiosperms play the most important roles in our human life. Our food, health, energy, materials, and environment largely depend on angiosperms. In addition, human culture is tightly linked to the utilization of angiosperms (Raskin et al., <a href="#B26">2002</a>). For example, early human cultures were shaped by the agriculture (Balick and Cox, <a href="#B3">1996</a>) including food production by rice and wheat, fruit gathering, wine fermentation, tea plantation, and flower culturing. Furthermore, angiosperms play important roles in the evolution of animal vision (Osorio and Vorobyev, <a href="#B22">2008</a>), taste (Li and Zhang, <a href="#B18">2014</a>), and olfactory sense (Niimura, <a href="#B21">2012</a>). The angiosperms also contributed greatly to the evolution of planet earth in the atmospheric cycle, water cycle, and the carbon cycle. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <p>Phylogeny and species of green plants and angiosperms. <b>(A)</b> The tree of the plant life ranging from green algae to flowering plants. <b>(B)</b> The tree of angiosperm life. <b>(C)</b> Angiosperm species tree of representative references genomes. The angiosperm species number is estimated by <a href="http://www.theplantlist.org">www.theplantlist.org</a>. Species number of other plant phylum is reported by Pryer et al. ( <a href="#B25">2002</a>). The number of Nymphaeales and other angiosperms species is summarized by Borsch et al. ( <a href="#B5">2008</a>) and Zeng et al. ( <a href="#B37">2014</a>), respectively. </p>   </div> <p>The <i>Science</i> editorial "So much more to know" raised 100 scientific questions to be answered and several of them are angiosperm-related (Hubble, <a href="#B13">2005</a>), such as: (1) How does a single somatic cell become a whole plant? (2) Why are some genomes really big and others quite compact? (3) What is all that "junk" doing in our genomes? (4) How did flowers evolve? Moreover, other important questions include (1) the origin of important innovations of the flowers and fruits, (2) the evolution of C4 and CAM photosynthesis, (3) the mechanisms of life style changes such as the epiphytes and parasites, (4) the genetic changes responsible for various ecological adaptations. Comparative genomics may hold the keys to these questions. The genomic sequences, bioinformatics tools, databases, and computing resources are essential infrastructures for comparative genomics. </p> <p>Rich information can be identified in the angiosperm genomes, which contain various elements, including the genes, repetitive elements, centromeres. All angiosperms are paleo-polyploids (Van de Peer et al., <a href="#B35">2017</a>) and some harbor sex chromosomes (Charlesworth, <a href="#B6">2016</a>). A genome database is designed to store and present all the information. With the rapid development of bioinformatics, genome database has evolved from mere data storage platform to a novel discipline. Two textbooks focused on the genome databases have been published: " <i>Bioinformatics for Beginners: Genes, Genomes, Molecular Evolution, Databases and Analytical Tools </i>" (Choudhuri, <a href="#B7">2014</a>) and " <i>Genomes, Browsers and Databases: Data-Mining Tools for Integrated Genomic Databases</i>" (Schattner, <a href="#B27">2008</a>). Furthermore, new journals focused on database have been launched and database articles becoming more popular in various journals. A journal named as <i>Database: The Journal of Biological Databases and Curation</i> was launched in 2009 centered in the biological database. The annual special issues of database published by two high-impact journals, <i>Plant and Cell Physiology</i> and <i>Nucleic Acids Research</i>, have built a good reputation and become influential in the biological research community. In addition, database articles are also frequently published in other leading journals of plant science, such as <i>The Plant Cell</i> and <i>Molecular Plant</i>. </p> <p>The main function of genome databases has evolved from data storage to online analysis, to lead the jigsaw puzzle in genome sequencing and resequencing projects. For example, Genome Database for Rosaceae (GDR) aims to host the genomes of all rosaceae species although only a few of their genomes have been sequenced. GDR integrated all the released rosaceae genomes and it is expected that more genome resequencing data would be added into the GDR. In the XIX International Botanical Congress (Shenzhen China, 2017), as a key part of the Earth BioGenome Project (EBP), a "10KP plan" was announced with an aim to sequence more than 10,000 genomes representing every major clade of plants and eukaryotic microbes. </p> <p>In this review, we dedicate to provide readers the latest advances of angiosperm genome projects and database constructions. We discussed the pros and cons of three types of genome databases. We advocate a genome database for all the sequenced angiosperms for prompting data sharing. We also suggest a suit of standards for genome database establishment to boost the development of future databases. The future challenges in facing the biological big data were also discussed. We believe this review will shed new light on the development of angiosperm genome database in the near future. </p> </div> <div class="genomeanddatabaseoverview" title="sec"> <div class="title" tagx="title" title="title">
Genome and database overview</div> <p>The first angiosperm genome database was launched in 2001 for the model plant <i>Arabidopsis thaliana</i> (Huala et al., <a href="#B12">2001</a>). Since 2001, various angiosperm genome databases have been developed synchronizing with the progress of sequencing projects of angiosperm genomes. The earliest angiosperm genome databases were designed as a repository of genome sequencing data. These databases have then evolved to serve as genome portals/hubs that integrate various genomic information, as well as web servers that provide online genomics analyses. These genome databases can be generally classified as three different types: single species database, comprehensive database, and clade-oriented database. </p> </div> <div class="thesequencedangiospermgenomes" title="sec"> <div class="title" tagx="title" title="title">
The sequenced angiosperm genomes</div> <p>As of August 31, 2017, the genomes of 236 angiosperm species have been completely sequenced. The list of sequenced angiosperm genomes and genome databases are provided in Table <a href="#T1">1</a>. The 236 species are found in 31 of the 64 angiosperm orders, thus nearly 50% of angiosperm orders have at least one genome sequenced. Most of them are plants of high economic importance or their wild relatives. More effort should be done on genome sequencing of more species that are important for study of evolutionary history of angiosperms such as magnoliids and basal angiosperms. </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  <p>A list of the public accessible plant genomes and their database construction status.</p>  <table> <thead> <tr> <th> <b>Species</b> </th> <th> <b>Order</b> </th> <th> <b>Type</b> </th> <th> <b>URL</b> </th> </tr> </thead> <tbody> <tr> <td> <i>Nymphaea colorata</i> </td> <td>Nymphaeales</td> <td>Ornamental</td> <td> <a href="http://www.angiosperms.org">www.angiosperms.org</a> </td> </tr> <tr> <td> <i>Amborella trichopoda</i> </td> <td>Amborellales</td> <td>Wild</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Elaeis guineensis</i> </td> <td>Arecales</td> <td>Economic</td> <td> <a href="https://chibba.agtec.uga.edu/duplication/">chibba.agtec.uga.edu/duplication/</a> </td> </tr> <tr> <td> <i>Phoenix dactylifera</i> </td> <td>Arecales</td> <td>Fruit</td> <td> <a href="https://pgsb.helmholtz-muenchen.de">pgsb.helmholtz-muenchen.de</a> </td> </tr> <tr> <td> <i>Elaeis oleifera</i> </td> <td>Arecales</td> <td>Wild</td> <td /> </tr> <tr> <td> <i>Brachypodium distachyon</i> </td> <td>Poales</td> <td>Economic</td> <td> <a href="https://plants.ensembl.org">plants.ensembl.org</a> </td> </tr> <tr> <td> <i>Eragrostis tef</i> </td> <td>Poales</td> <td>Economic</td> <td /> </tr> <tr> <td> <i>Eleusine coracana</i> </td> <td>Poales</td> <td>Food</td> <td /> </tr> <tr> <td> <i>Hordeum vulgare</i> </td> <td>Poales</td> <td>Food</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Oryza indica</i> </td> <td>Poales</td> <td>Food</td> <td> <a href="https://plants.ensembl.org">plants.ensembl.org</a> </td> </tr> <tr> <td> <i>Oryza sativa</i> </td> <td>Poales</td> <td>Food</td> <td> <a href="https://rice.plantbiology.msu.edu/">rice.plantbiology.msu.edu/</a> </td> </tr> <tr> <td> <i>Setaria italica</i> </td> <td>Poales</td> <td>Food</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Sorghum bicolor</i> </td> <td>Poales</td> <td>Food</td> <td> <a href="https://gramene.org/">gramene.org/</a> </td> </tr> <tr> <td> <i>Triticum aestivum</i> </td> <td>Poales</td> <td>Food</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Triticum turgidum</i> </td> <td>Poales</td> <td>Food</td> <td> <a href="https://gigadb.org">gigadb.org</a> </td> </tr> <tr> <td> <i>Zea mays</i> </td> <td>Poales</td> <td>Food</td> <td> <a href="https://plants.ensembl.org">plants.ensembl.org</a> </td> </tr> <tr> <td> <i>Secale cereale</i> </td> <td>Poales</td> <td>Food,</td> <td> <a href="https://pgsb.helmholtz-muenchen.de">pgsb.helmholtz-muenchen.de</a> </td> </tr> <tr> <td> <i>Ananas comosus</i> </td> <td>Poales</td> <td>Fruit</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Lolium perenne</i> </td> <td>Poales</td> <td>Ornamental</td> <td> <a href="https://pgsb.helmholtz-muenchen.de">pgsb.helmholtz-muenchen.de</a> </td> </tr> <tr> <td> <i>Zizania latifolia</i> </td> <td>Poales</td> <td>Vegetable</td> <td /> </tr> <tr> <td> <i>Oryza punctats</i> </td> <td>Poales</td> <td>Weed</td> <td> <a href="https://plants.ensembl.org">plants.ensembl.org</a> </td> </tr> <tr> <td> <i>Oryza rufipogon</i> </td> <td>Poales</td> <td>Weed</td> <td> <a href="https://plants.ensembl.org">plants.ensembl.org</a> </td> </tr> <tr> <td> <i>Aegilops tauschii</i> </td> <td>Poales</td> <td>Wild</td> <td> <a href="https://plants.ensembl.org">plants.ensembl.org</a> </td> </tr> <tr> <td> <i>Brachypodium stacei</i> </td> <td>Poales</td> <td>Wild</td> <td> <a href="https://genome.jgi.doe.gov">genome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Dichanthelium oligosanthes</i> </td> <td>Poales</td> <td>Wild</td> <td> <a href="https://genomevolution.org/CoGe">genomevolution.org/CoGe</a> </td> </tr> <tr> <td> <i>Hordeum pubiflorum</i> </td> <td>Poales</td> <td>Wild</td> <td /> </tr> <tr> <td> <i>Leersia perrieri</i> </td> <td>Poales</td> <td>Wild</td> <td> <a href="https://plants.ensembl.org">plants.ensembl.org</a> </td> </tr> <tr> <td> <i>Oropetium thomaeum</i> </td> <td>Poales</td> <td>Wild</td> <td> <a href="https://genomevolution.orgauth.iplantc.org">genomevolution.orgauth.iplantc.org</a> </td> </tr> <tr> <td> <i>Oryza barthii</i> </td> <td>Poales</td> <td>Wild</td> <td> <a href="https://plants.ensembl.org">plants.ensembl.org</a> </td> </tr> <tr> <td> <i>Oryza brachyantha</i> </td> <td>Poales</td> <td>Wild</td> <td> <a href="https://plants.ensembl.org">plants.ensembl.org</a> </td> </tr> <tr> <td> <i>Oryza glaberrima</i> </td> <td>Poales</td> <td>Wild</td> <td> <a href="https://plants.ensembl.org">plants.ensembl.org</a> </td> </tr> <tr> <td> <i>Oryza glumaepatula</i> </td> <td>Poales</td> <td>Wild</td> <td> <a href="https://plants.ensembl.org">plants.ensembl.org</a> </td> </tr> <tr> <td> <i>Oryza glumipatula</i> </td> <td>Poales</td> <td>Wild</td> <td> <a href="https://plants.ensembl.org">plants.ensembl.org</a> </td> </tr> <tr> <td> <i>Oryza meridionalis</i> </td> <td>Poales</td> <td>Wild</td> <td> <a href="https://plants.ensembl.org">plants.ensembl.org</a> </td> </tr> <tr> <td> <i>Oryza nivara</i> </td> <td>Poales</td> <td>Wild</td> <td> <a href="https://plants.ensembl.org">plants.ensembl.org</a> </td> </tr> <tr> <td> <i>Oryza longistaminata</i> </td> <td>Poales</td> <td>Wild</td> <td> <a href="https://plants.ensembl.org">plants.ensembl.org</a> </td> </tr> <tr> <td> <i>Panicum hallii</i> </td> <td>Poales</td> <td>Wild</td> <td> <a href="https://genomevolution.orgauth.iplantc.org">genomevolution.orgauth.iplantc.org</a> </td> </tr> <tr> <td> <i>Panicum virgatum</i> </td> <td>Poales</td> <td>Wild</td> <td> <a href="https://genome.jgi.doe.gov">genome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Setaria viridis</i> </td> <td>Poales</td> <td>Wild</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Triticum uratu</i> </td> <td>Poales</td> <td>Wild</td> <td> <a href="https://gigadb.org">gigadb.org</a> </td> </tr> <tr> <td> <i>Camptotheca acuminata</i> </td> <td>Cornales</td> <td>Ornamental, Economic</td> <td> <a href="https://datadryad.org">datadryad.org</a> </td> </tr> <tr> <td> <i>Musa balbisiana</i> </td> <td>Zingiberales</td> <td>Fruit</td> <td> <a href="https://genomevolution.org">genomevolution.org</a> </td> </tr> <tr> <td> <i>Musa itinerans</i> </td> <td>Zingiberales</td> <td>Fruit</td> <td> <a href="https://banana-genome-hub.southgreen.fr">banana-genome-hub.southgreen.fr</a> </td> </tr> <tr> <td> <i>Ensete ventricosum</i> </td> <td>Zingiberales</td> <td>Ornamental</td> <td /> </tr> <tr> <td> <i>Musa acuminata</i> </td> <td>Zingiberales</td> <td>Ornamental</td> <td> <a href="https://plants.ensembl.org">plants.ensembl.org</a> </td> </tr> <tr> <td> <i>Dendrobium catenatum</i> </td> <td>Asparagales</td> <td>Ornamental</td> <td /> </tr> <tr> <td> <i>Phalaenopsis equestris</i> </td> <td>Asparagales</td> <td>Ornamental</td> <td> <a href="https://chibba.agtec.uga.edu/duplication/">chibba.agtec.uga.edu/duplication/</a> </td> </tr> <tr> <td> <i>Xerophyta viscosa</i> </td> <td>Pandanales</td> <td>Wild</td> <td /> </tr> <tr> <td> <i>Lemna minor</i> </td> <td>Alismatales</td> <td>Economic</td> <td> <a href="https://genomevolution.org">genomevolution.org</a> </td> </tr> <tr> <td> <i>Spirodela polyrhiza</i> </td> <td>Alismatales</td> <td>Economic</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Zoysia japonica</i> </td> <td>Alismatales</td> <td>Ornamental</td> <td> <a href="https://zoysia.kazusa.or.jp">zoysia.kazusa.or.jp</a> </td> </tr> <tr> <td> <i>Zoysia matrella</i> </td> <td>Alismatales</td> <td>Ornamental</td> <td> <a href="https://zoysia.kazusa.or.jp">zoysia.kazusa.or.jp</a> </td> </tr> <tr> <td> <i>Zoysia pacifica</i> </td> <td>Alismatales</td> <td>Ornamental</td> <td> <a href="https://zoysia.kazusa.or.jp">zoysia.kazusa.or.jp</a> </td> </tr> <tr> <td> <i>Zostera marina</i> </td> <td>Alismatales</td> <td>Wild</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Macleaya cordata</i> </td> <td>Ranunculales</td> <td>Economic</td> <td /> </tr> <tr> <td> <i>Aquilegia coerulea</i> </td> <td>Ranunculales</td> <td>Wild</td> <td> <a href="https://genome.jgi.doe.gov">genome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Nelumbo nucifera</i> </td> <td>Proteales</td> <td>Ornamental</td> <td> <a href="https://chibba.agtec.uga.edu/duplication">chibba.agtec.uga.edu/duplication</a> </td> </tr> <tr> <td> <i>Trifolium pratense</i> </td> <td>Fabales</td> <td>Drink</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Lotus japonicus</i> </td> <td>Fabales</td> <td>Economic</td> <td> <a href="https://chibba.agtec.uga.edu/duplication/">chibba.agtec.uga.edu/duplication/</a> </td> </tr> <tr> <td> <i>Glycine soja</i> </td> <td>Fabales</td> <td>Food</td> <td> <a href="https://soybase.org">soybase.org</a> </td> </tr> <tr> <td> <i>Lupinus angustifolius</i> </td> <td>Fabales</td> <td>Ornamental</td> <td /> </tr> <tr> <td> <i>Cajanus cajan</i> </td> <td>Fabales</td> <td>Vegetable</td> <td> <a href="https://chibba.agtec.uga.edu/duplication/">chibba.agtec.uga.edu/duplication/</a> </td> </tr> <tr> <td> <i>Cicer arietinum</i> </td> <td>Fabales</td> <td>Vegetable</td> <td> <a href="https://chibba.agtec.uga.edu/duplication/">chibba.agtec.uga.edu/duplication/</a> </td> </tr> <tr> <td> <i>Glycine max</i> </td> <td>Fabales</td> <td>Vegetable</td> <td> <a href="https://plants.ensembl.org">plants.ensembl.org</a> </td> </tr> <tr> <td> <i>Medicago truncatula</i> </td> <td>Fabales</td> <td>Vegetable</td> <td> <a href="https://www.medicagogenome.org/">www.medicagogenome.org/</a> </td> </tr> <tr> <td> <i>Phaseolus vulgaris</i> </td> <td>Fabales</td> <td>Vegetable</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Vicia faba</i> </td> <td>Fabales</td> <td>Vegetable</td> <td /> </tr> <tr> <td> <i>Vigna angularis</i> </td> <td>Fabales</td> <td>Vegetable</td> <td> <a href="https://viggs.dna.affrc.go.jp">viggs.dna.affrc.go.jp</a> </td> </tr> <tr> <td> <i>Vigna radiata</i> </td> <td>Fabales</td> <td>Vegetable</td> <td> <a href="https://genomevolution.org/CoGe">genomevolution.org/CoGe</a> </td> </tr> <tr> <td> <i>Vigna unguiculata</i> </td> <td>Fabales</td> <td>Vegetable</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Arachis duranensis</i> </td> <td>Fabales</td> <td>Wild</td> <td> <a href="https://www.peanutbase.org/">www.peanutbase.org/</a> </td> </tr> <tr> <td> <i>Arachis ipaensis</i> </td> <td>Fabales</td> <td>Wild</td> <td> <a href="https://www.peanutbase.org/">www.peanutbase.org/</a> </td> </tr> <tr> <td> <i>Cicer reticulatum</i> </td> <td>Fabales</td> <td>Wild</td> <td> <a href="https://www.coolseasonFoodlegume.org">www.coolseasonFoodlegume.org</a> </td> </tr> <tr> <td> <i>Trifolium subterraneum</i> </td> <td>Fabales</td> <td>Wild</td> <td /> </tr> <tr> <td> <i>Betula pendula</i> </td> <td>Fabales</td> <td>Wood, Economic</td> <td> <a href="https://genomevolution.org">genomevolution.org</a> </td> </tr> <tr> <td> <i>Humulus lupulus</i> </td> <td>Rosales</td> <td>Drink</td> <td> <a href="https://hopbase.cgrb.oregonstate.edu">hopbase.cgrb.oregonstate.edu</a> </td> </tr> <tr> <td> <i>Cannabis sativa</i> </td> <td>Rosales</td> <td>Economic</td> <td> <a href="https://genome.ccbr.utoronto.ca/cgi-bin/hgGateway">genome.ccbr.utoronto.ca/cgi-bin/hgGateway</a> </td> </tr> <tr> <td> <i>Prunus avium</i> </td> <td>Rosales</td> <td>Food</td> <td> <a href="https://ftp://ftp.bioinfo.wsu.edu">ftp://ftp.bioinfo.wsu.edu</a> </td> </tr> <tr> <td> <i>Artocarpus camansi</i> </td> <td>Rosales</td> <td>Fruit</td> <td> <a href="https://datadryad.org">datadryad.org</a> </td> </tr> <tr> <td> <i>Ficus carica</i> </td> <td>Rosales</td> <td>Fruit</td> <td /> </tr> <tr> <td> <i>Fragaria nipponica</i> </td> <td>Rosales</td> <td>Fruit</td> <td /> </tr> <tr> <td> <i>Fragaria orientalis</i> </td> <td>Rosales</td> <td>Fruit</td> <td /> </tr> <tr> <td> <i>Fragaria vesca</i> </td> <td>Rosales</td> <td>Fruit</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Fragaria</i> × <i>ananassa</i> </td> <td>Rosales</td> <td>Fruit</td> <td /> </tr> <tr> <td> <i>Malus domestica</i> </td> <td>Rosales</td> <td>Fruit</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Morus notabilis</i> </td> <td>Rosales</td> <td>Fruit</td> <td> <a href="https://morus.swu.edu.cn/morusdb">morus.swu.edu.cn/morusdb</a> </td> </tr> <tr> <td> <i>Prunus persica</i> </td> <td>Rosales</td> <td>Fruit</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Pyrus bretschneideri</i> </td> <td>Rosales</td> <td>Fruit</td> <td> <a href="https://peargenome.njau.edu.cn">peargenome.njau.edu.cn</a> </td> </tr> <tr> <td> <i>Pyrus communis</i> </td> <td>Rosales</td> <td>Fruit</td> <td> <a href="https://genomevolution.org">genomevolution.org</a> </td> </tr> <tr> <td> <i>Rubus occidentalis</i> </td> <td>Rosales</td> <td>Fruit</td> <td> <a href="https://ftp://ftp.bioinfo.wsu.edu">ftp://ftp.bioinfo.wsu.edu</a> </td> </tr> <tr> <td> <i>Ziziphus jujuba</i> </td> <td>Rosales</td> <td>Fruit</td> <td> <a href="https://jujube.genomics.cn">jujube.genomics.cn</a> </td> </tr> <tr> <td> <i>Prunus mume</i> </td> <td>Rosales</td> <td>Ornamental</td> <td> <a href="https://chibba.agtec.uga.edu/duplication">chibba.agtec.uga.edu/duplication</a> </td> </tr> <tr> <td> <i>Rosa</i> × <i>damascena</i> </td> <td>Rosales</td> <td>Ornamental</td> <td /> </tr> <tr> <td> <i>Fragaria iinumae</i> </td> <td>Rosales</td> <td>Wild</td> <td /> </tr> <tr> <td> <i>Fragaria nubicola</i> </td> <td>Rosales</td> <td>Wild</td> <td /> </tr> <tr> <td> <i>Castanea mollissima</i> </td> <td>Fagales</td> <td>Fruit</td> <td> <a href="https://hardwoodgenomics.org">hardwoodgenomics.org</a> </td> </tr> <tr> <td> <i>Juglans regia</i> </td> <td>Fagales</td> <td>Fruit</td> <td /> </tr> <tr> <td> <i>Betula nana</i> </td> <td>Fagales</td> <td>Wild</td> <td /> </tr> <tr> <td> <i>Quercus lobata</i> </td> <td>Fagales</td> <td>Wood</td> <td> <a href="https://valleyoak.ucla.edu/genomesequence">valleyoak.ucla.edu/genomesequence</a> </td> </tr> <tr> <td> <i>Cucumis melo</i> </td> <td>Cucurbitales</td> <td>Fruit</td> <td> <a href="https://bioinformatics.psb.ugent.be/plaza/">bioinformatics.psb.ugent.be/plaza/</a> </td> </tr> <tr> <td> <i>Cucumis sativus</i> </td> <td>Cucurbitales</td> <td>Vegetable</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Lagenaria siceraria</i> </td> <td>Cucurbitales</td> <td>Vegetable</td> <td /> </tr> <tr> <td> <i>Momordica charantia</i> </td> <td>Cucurbitales</td> <td>Vegetable</td> <td /> </tr> <tr> <td> <i>Cephalotus follicularis</i> </td> <td>Oxalidales</td> <td>Economic</td> <td> <a href="https://genomevolution.orgauth.iplantc.org">genomevolution.orgauth.iplantc.org</a> </td> </tr> <tr> <td> <i>Hevea brasiliensis</i> </td> <td>Malpighiales</td> <td>Economic</td> <td> <a href="https://www4a.biotec.or.th/rubber">www4a.biotec.or.th/rubber</a> </td> </tr> <tr> <td> <i>Jatropha curcas</i> </td> <td>Malpighiales</td> <td>Economic</td> <td> <a href="https://www.kazusa.or.jp/jatropha">www.kazusa.or.jp/jatropha</a> </td> </tr> <tr> <td> <i>Populus tremula</i> </td> <td>Malpighiales</td> <td>Economic</td> <td> <a href="https://plantgenie.org">plantgenie.org</a> </td> </tr> <tr> <td> <i>Populus tremulax</i> </td> <td>Malpighiales</td> <td>Economic</td> <td> <a href="https://plantgenie.org">plantgenie.org</a> </td> </tr> <tr> <td> <i>Populus trichocarpa</i> </td> <td>Malpighiales</td> <td>Economic</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Ricinus communis</i> </td> <td>Malpighiales</td> <td>Economic</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Linum usitatissimum</i> </td> <td>Malpighiales</td> <td>Economic, Fiber</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Manihot esculenta</i> </td> <td>Malpighiales</td> <td>Vegetable</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Populus tremuloides</i> </td> <td>Malpighiales</td> <td>Wild</td> <td> <a href="https://plantgenie.org">plantgenie.org</a> </td> </tr> <tr> <td> <i>Populus euphratica</i> </td> <td>Malpighiales</td> <td>Wild</td> <td /> </tr> <tr> <td> <i>Salix purpurea</i> </td> <td>Malpighiales</td> <td>Wild</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Populus pruinosa</i> </td> <td>Malpighiales</td> <td>Wood</td> <td> <a href="https://gigadb.org">gigadb.org</a> </td> </tr> <tr> <td> <i>Populus deltoides</i> </td> <td>Malpighiales</td> <td>Wood</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Eucalyptus camaldulensis</i> </td> <td>Myrtales</td> <td>Economic</td> <td> <a href="https://www.kazusa.or.jp">www.kazusa.or.jp</a> </td> </tr> <tr> <td> <i>Punica granatum</i> </td> <td>Myrtales</td> <td>Food, Ornamental</td> <td /> </tr> <tr> <td> <i>Metrosideros polymorpha</i> </td> <td>Myrtales</td> <td>Wild</td> <td /> </tr> <tr> <td> <i>Eucalyptus grandis</i> </td> <td>Myrtales</td> <td>Wood</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Theobroma cacao</i> </td> <td>Malvales</td> <td>Drink</td> <td> <a href="https://www.cacaogenomedb.org/">www.cacaogenomedb.org/</a> </td> </tr> <tr> <td> <i>Gossypium raimondii</i> </td> <td>Malvales</td> <td>Economic</td> <td> <a href="https://chibba.agtec.uga.edu/duplication/">chibba.agtec.uga.edu/duplication/</a> </td> </tr> <tr> <td> <i>Corchirus olitorius</i> </td> <td>Malvales</td> <td>Fiber</td> <td /> </tr> <tr> <td> <i>Corchorus capsularis</i> </td> <td>Malvales</td> <td>Fiber</td> <td /> </tr> <tr> <td> <i>Gossypium arboreum</i> </td> <td>Malvales</td> <td>Fiber</td> <td> <a href="https://www.cottongen.org">www.cottongen.org</a> </td> </tr> <tr> <td> <i>Gossypium barbadense</i> </td> <td>Malvales</td> <td>Fiber</td> <td> <a href="https://www.cottongen.org">www.cottongen.org</a> </td> </tr> <tr> <td> <i>Gossypium hirsutum</i> </td> <td>Malvales</td> <td>Fiber</td> <td> <a href="https://cgp.genomics.org.cn">cgp.genomics.org.cn</a> </td> </tr> <tr> <td> <i>Hibiscus syriacus</i> </td> <td>Malvales</td> <td>Ornamental</td> <td> <a href="https://hibiscus.kobic.re.kr/hibiscus.en">hibiscus.kobic.re.kr/hibiscus.en</a> </td> </tr> <tr> <td> <i>Aquilaria agallochum</i> </td> <td>Malvales</td> <td>Wood</td> <td /> </tr> <tr> <td> <i>Brassica nigra</i> </td> <td>Brassicales</td> <td>Economic</td> <td> <a href="https://brassicadb.org">brassicadb.org</a> </td> </tr> <tr> <td> <i>Camelina sativa</i> </td> <td>Brassicales</td> <td>Economic</td> <td /> </tr> <tr> <td> <i>Capsella orientalis</i> </td> <td>Brassicales</td> <td>Food</td> <td> <a href="https://genomevolution.org">genomevolution.org</a> </td> </tr> <tr> <td> <i>Carica papaya</i> </td> <td>Brassicales</td> <td>Fruit</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Lepidium meyenii</i> </td> <td>Brassicales</td> <td>Economic</td> <td> <a href="https://ftp://202.203.187.112/genome/maca/">ftp://202.203.187.112/genome/maca/</a> </td> </tr> <tr> <td> <i>Tarenaya hassleriana</i> </td> <td>Brassicales</td> <td>Ornamental</td> <td> <a href="https://genomevolution.org/coge">genomevolution.org/coge</a> </td> </tr> <tr> <td> <i>Barbarea vulgaris</i> </td> <td>Brassicales</td> <td>Vegetable</td> <td> <a href="https://185.45.23.197:5080/Barbarea_data/">185.45.23.197:5080/Barbarea_data/</a> </td> </tr> <tr> <td> <i>Brassica juncea</i> </td> <td>Brassicales</td> <td>Vegetable</td> <td> <a href="https://brassicadb.org">brassicadb.org</a> </td> </tr> <tr> <td> <i>Brassica napus</i> </td> <td>Brassicales</td> <td>Vegetable</td> <td> <a href="https://gramene.org/">gramene.org/</a> </td> </tr> <tr> <td> <i>Brassica oleracea</i> </td> <td>Brassicales</td> <td>Vegetable</td> <td> <a href="https://plants.ensembl.org">plants.ensembl.org</a> </td> </tr> <tr> <td> <i>Brassica rapa</i> </td> <td>Brassicales</td> <td>Vegetable</td> <td> <a href="https://brassicadb.org/brad/">brassicadb.org/brad/</a> </td> </tr> <tr> <td> <i>Capsella bursa-pastoris</i> </td> <td>Brassicales</td> <td>Vegetable</td> <td /> </tr> <tr> <td> <i>Capsella rubella</i> </td> <td>Brassicales</td> <td>Vegetable</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Moringa oleifera</i> </td> <td>Brassicales</td> <td>Vegetable</td> <td> <a href="https://ftp://202.203.187.112/genome/lamu/">ftp://202.203.187.112/genome/lamu/</a> </td> </tr> <tr> <td> <i>Raphanus sativus</i> </td> <td>Brassicales</td> <td>Vegetable</td> <td> <a href="https://radish.kazusa.or.jp/index.html">radish.kazusa.or.jp/index.html</a> </td> </tr> <tr> <td> <i>Thlaspi arvense</i> </td> <td>Brassicales</td> <td>Vegetable</td> <td /> </tr> <tr> <td> <i>Aethionema arabicum</i> </td> <td>Brassicales</td> <td>Wild</td> <td> <a href="https://brassicadb.org">brassicadb.org</a> </td> </tr> <tr> <td> <i>Arabidopsis halleri</i> </td> <td>Brassicales</td> <td>Wild</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Arabidopsis lyrata</i> </td> <td>Brassicales</td> <td>Wild</td> <td> <a href="https://bioinformatics.psb.ugent.be/plaza/">bioinformatics.psb.ugent.be/plaza/</a> </td> </tr> <tr> <td> <i>Arabidopsis thaliana</i> </td> <td>Brassicales</td> <td>Wild</td> <td> <a href="https://www.arabidopsis.org/index.jsp">www.arabidopsis.org/index.jsp</a> </td> </tr> <tr> <td> <i>Arabis alpina</i> </td> <td>Brassicales</td> <td>Wild</td> <td> <a href="https://arabis-alpina.org">arabis-alpina.org</a> </td> </tr> <tr> <td> <i>Arabis montbretiana</i> </td> <td>Brassicales</td> <td>Wild</td> <td /> </tr> <tr> <td> <i>Arabis nordmanniana</i> </td> <td>Brassicales</td> <td>Wild</td> <td /> </tr> <tr> <td> <i>Boechera stricta</i> </td> <td>Brassicales</td> <td>Wild</td> <td> <a href="https://genome.jgi.doe.gov">genome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Capsella grandiflora</i> </td> <td>Brassicales</td> <td>Wild</td> <td> <a href="https://genomevolution.org">genomevolution.org</a> </td> </tr> <tr> <td> <i>Cardamine hirsuta</i> </td> <td>Brassicales</td> <td>Wild</td> <td> <a href="https://chi.mpipz.mpg.de">chi.mpipz.mpg.de</a> </td> </tr> <tr> <td> <i>Eutrema salsugineum</i> </td> <td>Brassicales</td> <td>Wild</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Leavenworthia alabamica</i> </td> <td>Brassicales</td> <td>Wild</td> <td> <a href="https://brassicadb.org">brassicadb.org</a> </td> </tr> <tr> <td> <i>Raphanus raphanistrum</i> </td> <td>Brassicales</td> <td>Wild</td> <td> <a href="https://genomevolution.org/coge">genomevolution.org/coge</a> </td> </tr> <tr> <td> <i>Sisymbrium irio</i> </td> <td>Brassicales</td> <td>Wild</td> <td> <a href="https://brassicadb.org">brassicadb.org</a> </td> </tr> <tr> <td> <i>Thellungiella parvula</i> </td> <td>Brassicales</td> <td>Wild</td> <td> <a href="https://thellungiella.org/data/">thellungiella.org/data/</a> </td> </tr> <tr> <td> <i>Citrus grandis</i> </td> <td>Sapindales</td> <td>Food,</td> <td> <a href="https://citrus.hzau.edu.cn">citrus.hzau.edu.cn</a> </td> </tr> <tr> <td> <i>Citrus ichangensis</i> </td> <td>Sapindales</td> <td>Food,</td> <td> <a href="https://citrus.hzau.edu.cn">citrus.hzau.edu.cn</a> </td> </tr> <tr> <td> <i>Citrus medica</i> </td> <td>Sapindales</td> <td>Food</td> <td> <a href="https://citrus.hzau.edu.cn">citrus.hzau.edu.cn</a> </td> </tr> <tr> <td> <i>Citrullus lanatus</i> </td> <td>Sapindales</td> <td>Fruit</td> <td> <a href="https://chibba.agtec.uga.edu/duplication">chibba.agtec.uga.edu/duplication</a> </td> </tr> <tr> <td> <i>Citrus clementina</i> </td> <td>Sapindales</td> <td>Fruit</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Citrus sinensis</i> </td> <td>Sapindales</td> <td>Fruit</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Dimocarpus longan</i> </td> <td>Sapindales</td> <td>Fruit</td> <td> <a href="https://gigadb.org">gigadb.org</a> </td> </tr> <tr> <td> <i>Atalantia buxifolia</i> </td> <td>Sapindales</td> <td>Wild</td> <td> <a href="https://citrus.hzau.edu.cn">citrus.hzau.edu.cn</a> </td> </tr> <tr> <td> <i>Azadirachta indica</i> </td> <td>Sapindales</td> <td>Wild</td> <td /> </tr> <tr> <td> <i>Vitis vinifera</i> </td> <td>Vitales</td> <td>Fruit</td> <td> <a href="https://plants.ensembl.org">plants.ensembl.org</a> </td> </tr> <tr> <td> <i>Vitis aestivalis</i> </td> <td>Vitales</td> <td>Fruit</td> <td /> </tr> <tr> <td> <i>Vitis cinerea</i> × <i>Vitis riparia</i> </td> <td>Vitales</td> <td>Fruit</td> <td /> </tr> <tr> <td> <i>Rhodiola crenulata</i> </td> <td>Saxifragales</td> <td>Economic</td> <td> <a href="https://gigadb.org">gigadb.org</a> </td> </tr> <tr> <td> <i>Kalanchoe fedtschenkoi</i> </td> <td>Saxifragales</td> <td>Ornamental</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Kalanchoe laxiflora</i> </td> <td>Saxifragales</td> <td>Ornamental</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Kalanchoe marnieriana</i> </td> <td>Saxifragales</td> <td>Ornamental</td> <td> <a href="https://genomevolution.org">genomevolution.org</a> </td> </tr> <tr> <td> <i>Chenopodium pallidicaule</i> </td> <td>Caryophyllales</td> <td>Food</td> <td /> </tr> <tr> <td> <i>Chenopodium quinoa</i> </td> <td>Caryophyllales</td> <td>Food</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Fagopyrum esculentum</i> </td> <td>Caryophyllales</td> <td>Food</td> <td> <a href="https://buckwheat.kazusa.or.jp">buckwheat.kazusa.or.jp</a> </td> </tr> <tr> <td> <i>Amaranthus hypochondriacus</i> </td> <td>Caryophyllales</td> <td>Ornamental</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Dianthus caryophyllus</i> </td> <td>Caryophyllales</td> <td>Ornamental</td> <td> <a href="https://carnation.kazusa.or.jp">carnation.kazusa.or.jp</a> </td> </tr> <tr> <td> <i>Drosera capensis</i> </td> <td>Caryophyllales</td> <td>Ornamental</td> <td /> </tr> <tr> <td> <i>Beta vulgaris</i> </td> <td>Caryophyllales</td> <td>Vegetable</td> <td> <a href="https://plants.ensembl.org">plants.ensembl.org</a> </td> </tr> <tr> <td> <i>Spinacia oleracea</i> </td> <td>Caryophyllales</td> <td>Vegetable</td> <td> <a href="https://genomevolution.org">genomevolution.org</a> </td> </tr> <tr> <td> <i>Amaranthus tuberculatus</i> </td> <td>Caryophyllales</td> <td>Weed</td> <td /> </tr> <tr> <td> <i>Chenopodium suecicum</i> </td> <td>Caryophyllales</td> <td>Wild</td> <td /> </tr> <tr> <td> <i>Silene latifolia subsp. alba</i> </td> <td>Caryophyllales</td> <td>Wild</td> <td /> </tr> <tr> <td> <i>Eichhornia paniculata</i> </td> <td>Commelinales</td> <td>Wild</td> <td /> </tr> <tr> <td> <i>Camellia sinensis</i> </td> <td>Ericales</td> <td>Drink</td> <td> <a href="https://www.plantkingdomgdb.com">www.plantkingdomgdb.com</a> </td> </tr> <tr> <td> <i>Actinidia chinensis</i> </td> <td>Ericales</td> <td>Fruit</td> <td> <a href="https://bdg.hfut.edu.cn">bdg.hfut.edu.cn</a> </td> </tr> <tr> <td> <i>Diospyros lotus</i> </td> <td>Ericales</td> <td>Fruit</td> <td> <a href="https://chibba.agtec.uga.edu/duplication/">chibba.agtec.uga.edu/duplication/</a> </td> </tr> <tr> <td> <i>Vaccinum macrocarpon</i> </td> <td>Ericales</td> <td>Fruit</td> <td /> </tr> <tr> <td> <i>Primula veris</i> </td> <td>Ericales</td> <td>Wild</td> <td /> </tr> <tr> <td> <i>Helianthus annuus</i> </td> <td>Asterales</td> <td>Economic</td> <td> <a href="https://www.sunflowergenome.org">www.sunflowergenome.org</a> </td> </tr> <tr> <td> <i>Silybum marianum</i> </td> <td>Asterales</td> <td>Economic</td> <td /> </tr> <tr> <td> <i>Carthamus tinctorius</i> </td> <td>Asterales</td> <td>Oil</td> <td /> </tr> <tr> <td> <i>Lactuca sativa</i> </td> <td>Asterales</td> <td>Vegetable</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Conyza canadensis</i> </td> <td>Asterales</td> <td>Weed</td> <td /> </tr> <tr> <td> <i>Panax notoginseng</i> </td> <td>Apiales</td> <td>Economic</td> <td> <a href="https://ftp://202.203.187.112/genome/sanqi">ftp://202.203.187.112/genome/sanqi</a> </td> </tr> <tr> <td> <i>Daucus carota</i> </td> <td>Apiales</td> <td>Vegetable</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Nicotiana benthamiana</i> </td> <td>Solanales</td> <td>Economic</td> <td> <a href="https://genomevolution.org">genomevolution.org</a> </td> </tr> <tr> <td> <i>Nicotiana tabacum</i> </td> <td>Solanales</td> <td>Economic</td> <td> <a href="https://solgenomics.net">solgenomics.net</a> </td> </tr> <tr> <td> <i>Ipomoea nil</i> </td> <td>Solanales</td> <td>Ornamental</td> <td /> </tr> <tr> <td> <i>Nicotiana sylvestris</i> </td> <td>Solanales</td> <td>Ornamental</td> <td> <a href="https://solgenomics.net">solgenomics.net</a> </td> </tr> <tr> <td> <i>Petunia inflata</i> </td> <td>Solanales</td> <td>Ornamental</td> <td> <a href="https://genomevolution.org">genomevolution.org</a> </td> </tr> <tr> <td> <i>Petunia integrifolia</i> </td> <td>Solanales</td> <td>Ornamental</td> <td> <a href="https://genomevolution.org">genomevolution.org</a> </td> </tr> <tr> <td> <i>Capsicum annuum</i> </td> <td>Solanales</td> <td>Vegetable</td> <td> <a href="https://chibba.agtec.uga.edu/duplication">chibba.agtec.uga.edu/duplication</a> </td> </tr> <tr> <td> <i>Solanum lycopersicum</i> </td> <td>Solanales</td> <td>Vegetable</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Solanum melongena</i> </td> <td>Solanales</td> <td>Vegetable</td> <td> <a href="https://genomevolution.org">genomevolution.org</a> </td> </tr> <tr> <td> <i>Solanum pimpinellifolium</i> </td> <td>Solanales</td> <td>Vegetable</td> <td> <a href="https://solgenomics.net">solgenomics.net</a> </td> </tr> <tr> <td> <i>Solanum tuberosum</i> </td> <td>Solanales</td> <td>Vegetable</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Ipomoea trifida</i> </td> <td>Solanales</td> <td>Wild</td> <td> <a href="https://sweetpotato-garden.kazusa.or.jp">sweetpotato-garden.kazusa.or.jp</a> </td> </tr> <tr> <td> <i>Nicotiana attenuata</i> </td> <td>Solanales</td> <td>Wild</td> <td /> </tr> <tr> <td> <i>Nicotiana otophora</i> </td> <td>Solanales</td> <td>Wild</td> <td /> </tr> <tr> <td> <i>Nicotiana tomentosiformis</i> </td> <td>Solanales</td> <td>Wild</td> <td> <a href="https://solgenomics.net">solgenomics.net</a> </td> </tr> <tr> <td> <i>Petunia axilaris</i> </td> <td>Solanales</td> <td>Wild</td> <td> <a href="https://genomevolution.org">genomevolution.org</a> </td> </tr> <tr> <td> <i>Solanum arcanum</i> </td> <td>Solanales</td> <td>Wild</td> <td /> </tr> <tr> <td> <i>Solanum commersonii</i> </td> <td>Solanales</td> <td>Wild</td> <td> <a href="https://solgenomics.net">solgenomics.net</a> </td> </tr> <tr> <td> <i>Solanum habrochaites</i> </td> <td>Solanales</td> <td>Wild</td> <td /> </tr> <tr> <td> <i>Solanum pennellii</i> </td> <td>Solanales</td> <td>Wild</td> <td> <a href="https://solgenomics.net">solgenomics.net</a> </td> </tr> <tr> <td> <i>Sesamum indicum</i> </td> <td>Lamiales</td> <td>Economic</td> <td> <a href="https://www.ocri-genomics.org/Sinbase/index.html">www.ocri-genomics.org/Sinbase/index.html</a> </td> </tr> <tr> <td> <i>Dorcoceras hygrometricum</i> </td> <td>Lamiales</td> <td>Economic</td> <td /> </tr> <tr> <td> <i>Mentha longifolia</i> </td> <td>Lamiales</td> <td>Economic</td> <td> <a href="https://langelabtools.wsu.edu/mgr/organism/Mentha/longifolia">langelabtools.wsu.edu/mgr/organism/Mentha/longifolia</a> </td> </tr> <tr> <td> <i>Ocimum tenuiflorum</i> </td> <td>Lamiales</td> <td>Economic</td> <td /> </tr> <tr> <td> <i>Pogostemon cablin</i> </td> <td>Lamiales</td> <td>Economic</td> <td> <a href="https://ftp://202.203.187.112">ftp://202.203.187.112</a> </td> </tr> <tr> <td> <i>Salvia miltiorrhiza</i> </td> <td>Lamiales</td> <td>Economic</td> <td> <a href="https://ftp://202.203.187.112">ftp://202.203.187.112</a> </td> </tr> <tr> <td> <i>Erythranthe guttata</i> </td> <td>Lamiales</td> <td>Ornamental</td> <td /> </tr> <tr> <td> <i>Fraxinus excelsior</i> </td> <td>Lamiales</td> <td>Ornamental</td> <td> <a href="https://www.ashgenome.org">www.ashgenome.org</a> </td> </tr> <tr> <td> <i>Mimulus guttatus</i> </td> <td>Lamiales</td> <td>Ornamental</td> <td> <a href="https://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a> </td> </tr> <tr> <td> <i>Antirrhinum majus</i> </td> <td>Lamiales</td> <td>Ornamental, Economic</td> <td> <a href="https://genomevolution.org">genomevolution.org</a> </td> </tr> <tr> <td> <i>Genlisea aurea</i> </td> <td>Lamiales</td> <td>Wild</td> <td /> </tr> <tr> <td> <i>Utricularia gibba</i> </td> <td>Lamiales</td> <td>Wild</td> <td> <a href="https://chibba.agtec.uga.edu/duplication/">chibba.agtec.uga.edu/duplication/</a> </td> </tr> <tr> <td> <i>Coffea canephora</i> </td> <td>Gentianales</td> <td>Drink</td> <td> <a href="https://genomevolution.org">genomevolution.org</a> </td> </tr> <tr> <td> <i>Catharanthus roseus</i> </td> <td>Gentianales</td> <td>Economic</td> <td> <a href="https://plantgenomics.msu.edu">plantgenomics.msu.edu</a> </td> </tr> <tr> <td> <i>Rhazya stricta</i> </td> <td>Gentianales</td> <td>Economic</td> <td /> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p> <i>The blank link of 58 species suggest these genome information were stored in NCBI-Genome column </i>. </p> </div> </div> <p>After the completion of the genome sequencing, an urgent issue is to share the genome data with the research community immediately after the genome release. The importance of data sharing is well recognized because it expands the impact of these valuable sequence data and promotes collaboration. A good genome database should meet two criteria: (i) integration of various types of genomic data, and (ii) providing genome analysis tools. </p> </div> <div class="genomedatabaseforasinglespecies" title="sec"> <div class="title" tagx="title" title="title">
Genome database for a single species</div> <p>Among the 236 sequenced angiosperm genomes, only a few of them have a well-constructed customized database (Table <a href="#T1">1</a>) to host its various genome information. 58 genomes are only stored at NCBI Genome without a customized database. The genome databases of model plants <i>Arabidopsis</i> and rice ( <i>Oryza sativa</i>) appear to be most well constructed. The most popular <i>Arabidopsis</i> database is the Arabidopsis Information Resource (TAIR, <a href="http://www.arabidopsis.org">www.arabidopsis.org</a>) (Garcia-Hernandez et al., <a href="#B10">2002</a>). TAIR provides updated genome sequence (currently V10) and various genomic information, including SNP, transposons, genes, gene families, gene annotations, gene names, proteins, and mutant orderings. Multiple web-integrated bioinformatics tools are also provided by TAIR. For examples, BLAST, WU-BLAST, FASTA, Gbrowse, Synteny Viewer, Seqviewer, Motif analysis, and Chromosome Map tool are powerful for visualization and comparative studies of genes and genome sequence at different scales. Pathway maps provide predicted gene interaction information, and has gained its popularity for the rapid development of metabolic and metabolomics researches. Other tools include Mapviewer, Metabolic Pathways, N-browse, Patmatch, VxInsight, Java Tree View, Bulk Data Retrieval, Gene Symbol Registry, and Textpresso Full Text. ARAPOT ( <a href="http://www.araport.org">www.araport.org</a>) is also an important <i>Arabidopsis</i> genome database that provides updated genome sequence (currently V11), various gene information and protein interaction networks. However, another <i>Arabidopsis</i> database (Schoof et al., <a href="#B28">2002</a>) is no longer accessible. Among the web-integrated bioinformatics tools, such as those in TAIR as an example, BLAST, WU-BLAST, FASTA, Gbrowse, Synteny Viewer, Seqviewer, Motif analysis, and Chromosome Map tool were developed for visualize and compare genes and genome sequence at various scales. Patmatch and Bulk Data Retrieval tools help users to fetch data from servers. Pathway maps provide predicted gene interaction information, and are becoming popular nowadays for the rapid development of metabolic and metabolomics researches. Besides, the other tools are good complementary to various purposes. </p> <p>However, unlike TAIR, most species-specific genome databases do not offer a rich collection of bioinformatic tools. For example, the pear ( <i>Pyrus bretschneideri</i>.) genome database ( <a href="http://peargenome.njau.edu.cn/">peargenome.njau.edu.cn</a>) only provide data download. The ash tree ( <i>Hymenoscyphus fraxinea</i> and <i>Agrilus planipennis</i>) genome database ( <a href="http://ashgenome.org/">ashgenome.org/</a>) includes the BLAST, Jbrowse tools, and data download service. The jujube ( <i>Ziziphus jujuba</i>) genome database ( <a href="http://jujube.genomics.cn/page/species/index.jsp">jujube.genomics.cn/page/species/index.jsp</a>) has tools such as gene search, Mapview and BLAST. The limited availability of bioinformatic tools may have reduced the popularity and usability of these databases. </p> <p>Constrution a comprehensive database requires the knowledge of databases and a plethora of database and web programming languages such as Java, HTML, PHP, MySQL, Python, Perl et al., which are not the expertise of most experimental biologists. Most of none-model plant genomes are sequenced by experimental biologists, which is probably the main factor for the different levels of functionalities among species-specific genome databases. Three problems are constantly encountered for species-specific genome databases. First, these databases are often constructed by outsourcing companies, or by one of the bioinformatics graduate student/staff. The cost of such database is usually low and less time consuming. However, the content of these databases is usually rarely or never updated, probably due to the expiration of the service contract with outsourcing companies or departures of graduate students/post-doctoral scholars. For this reason, many species-specific genome database are unstable and eventually become inactivated. For instance, the databases of Mei and pineapple are no longer accessible (only accessible for several months after the release of genomes). Although accessing some databases is not convenient, a few species have more than one genome database. Second, the visual design of these databases usually does not match to those comprehensive databases. The findability and accessibility are usually limited because the users of a species-specific genome database are usually limited to those who work on the same species (Adam-Blondon et al., <a href="#B1">2016</a>). Third, with the rapid development of sequencing technology, many more genomes have been sequenced but no customized genome database was built for these genomes, although they may be economically or evolutionarily important. </p> </div> <div class="comprehensivedatabasesforvariousangiospermspecies" title="sec"> <div class="title" tagx="title" title="title">
Comprehensive databases for various angiosperm species</div> <p>With the advances of next generation sequencing (NGS) and the latest third generation sequencing platforms, angiosperms with genome sequences are rapidly accumulating. The Pacific Biosciences (PacBio) company have developed a single-molecule real-time sequencing platform, which outputs long and unbiased reads with average length &amp;gt;10Kb, greatly facilitates the assembly of large and complex angiosperm genomes. The genome sequencing of a desiccation grass <i>Oropetium thomaeum</i> (VanBuren et al., <a href="#B34">2015</a>), sunflower (Badouin et al., <a href="#B2">2017</a>), and quinoa (Jarvis et al., <a href="#B14">2017</a>) all relied on PacBio and produced high-quality genome assembly. Comparative analysis of these genomic data allows scientists to answer many important questions of plant biology. Therefore, a high demand for comprehensive genome databases is expected. Currently, several comprehensive databases that include a large collection of plant genomes have been constructed (Figure <a href="#F2">2A</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <p>Selected well-constructed genome databases for/covering angiosperms. <b>(A)</b> Comprehensive databases and their featured tools and indexed genomes. <b>(B)</b> Small-scale genome databases for specific clade of angiosperms. <b>(C)</b> A comparison of the comprehensive genome databases. PGSB, Plant Genome and Systems Biology; PlantGDB, Plant Genome Database; PGDD, Plant Genome Duplication Database; GDA, Genome Database for Angiosperms. </p>   </div> <p>Phytozome ( <a href="http://phytozome.jgi.doe.gov">phytozome.jgi.doe.gov</a>) is a large plant genomic portal sponsored by the USA Department of Energy (DOE). The current release of Phytozome (v12) hosts assembled and annotated genomes from 59 angiosperm species, as well as other green lineage species, such as algae, moss, liverworts, selaginella (Goodstein et al., <a href="#B11">2012</a>). In addition to BLAST and Gbrowse tools, Phytozome also provide Biomart which allow users to annotate plant gene families, to study the evolution of plant gene families, to display genes in the genomic context (Goodstein et al., <a href="#B11">2012</a>), which is valuable for a wide range of scientists who are interested in gene family evolution. However, considering that the genomes of 236 angiosperm species have been sequenced, &amp;lt;1 third of all sequenced angiosperms are included by Phytozome, suggesting the presence of a major gap in the availability of most angiosperm genomes at Phytozome. </p> <p>The Plant Genome Duplication Database (PGDD) ( <a href="http://chibba.agtec.uga.edu/duplication/">chibba.agtec.uga.edu/duplication/</a>) is a database currently hosting 43 angiosperm genomes, with tools to identify the intragenome and cross-genome synteny relationships. Synonymous substitutions of homologs inferred from syntenic alignments could be calculated from this database (Lee et al., <a href="#B17">2012</a>). By the synteny comparison, PGDD facilitates the identification of evolutionary analysis of gene and genome duplication (Lee et al., <a href="#B17">2012</a>). </p> <p>Ensembl is well-known for developing bioinformatics tools and annotating various eukaryotic genomes (Kersey et al., <a href="#B16">2014</a>). The Ensembl Plants ( <a href="http://plants.ensembl.org/index.html">plants.ensembl.org/index.html</a>) provide a HMMER tool for homology searches of gene family members. However, it only covers 37 angiosperm genomes and does not include genome browsers for genomic context views thereby limits its readership. </p> <p>VISTA ( <a href="http://genome.lbl.gov/vista/index.shtml">genome.lbl.gov/vista/index.shtml</a>), which includes 38 angiosperm genomes, provides comprehensive tools for analyzing multiple genomes, such as tools for alignment of multiple sequences and large genomic sequences. VISTA has been extensively used by the biomedical community (Poliakov et al., <a href="#B23">2014</a>). However, some tools are not applicable to angiosperms as they are restricted to human and mouse data. </p> <p>Other databases such as PLAZA (Proost et al., <a href="#B24">2009</a>), plantgdb (Duvick et al., <a href="#B9">2008</a>), CoGe (Lyons, <a href="#B19">2008</a>), PGSB (Spannagl et al., <a href="#B31">2016</a>) also provide valuable tools for angiosperm comparative genomics. However, none of these databases contains more than 30 plant genomes, which is less than one-eighth of total sequenced angiosperms. </p> <p>Many of these comprehensive genome databases are well-maintained and are frequently updated with release of new versions, such as PlantGDB V187, Phytozome V12, and Gramene V36. Phytozome updates roughly each year, and PGDD and CoGe update timely upon availability of new genome data. Another feature is that these databases are empowered with various tools other than the open-sourced BLAST and browsers. They offer tools for gene family copies (Phytozome, PLAZA), gene/chromosome synteny (CoGe, PGDD, phytozome, PLAZA, PGSB), protein domains (PlantGDB, Phytozome, EnsemblPlant), gene expression (Phytozome, PGSB), biomart (Phytozome, EnsemblPlants), Intermine (Phytozome), GO annotation (Phytozome, EnsemblPlants, PLAZA, PGSB), alternative splicing (EnsemblPlants, plantgdb). </p> <p>Although these comprehensive databases contain a large array of species, the largest one Phytozome only includes 59 angiosperms genomes, accounting for about ¼ of sequenced angiosperm genomes. We constructed the <b>G</b>enome <b>D</b>atabase for <b>A</b>ngiosperms (GDA, <a href="http://www.angiosperms.org">www.angiosperms.org</a>) to host all of the released angiosperms genome (Figures <a href="#F2">2B</a>, <a href="#F3">3</a>). GDA aims to updates all the recently sequenced angiosperm genomes by supplying a timeline (Figure <a href="#F3">3A</a>). Currently, all the 236 angiosperm genomes, CDSs, and proteins are provided and can be accessed via BLAST suits and download service. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <p>The proposed genome database for angiosperms (GDA, <a href="http://www.angiosperms.org/">www.angiosperms.org</a>). <b>(A)</b> The menu of the database. <b>(B,C)</b> Statistics of the sequenced genomes in all the angiosperm orders. <b>(D)</b> Statistics of the sequenced crop genomes. <b>(E)</b> Timeline for the genome updates and related hyperlinks. <b>(F)</b> Ongoing projects for specific angiosperms. <b>(G)</b> BLAST page for all the released 235 angiosperm genomes, CDS, and protein information. </p>   </div> </div> <div class="databaseforclade-orientedangiospermsspecies" title="sec"> <div class="title" tagx="title" title="title">
Database for clade-oriented angiosperms species</div> <p>The NGS techniques has significantly accelerated the decoding of genomes. For example, 10 rice species have been sequenced (Table <a href="#T1">1</a>). Comparative genomics is a powerful strategy to decode the genetic basis underlying trait evolution and the evolution of genes and genomes. We summarized the five well-constructed clade-oriented genome databases that have a clear goal (Figure <a href="#F2">2</a>). </p> <p>The current version of Gramene ( <a href="http://gramene.org/">gramene.org</a>) provides curated and integrated genomic information for plants, especially the 17 grass species. Gramene's bioinformatics platforms provide specific softwares for studying the grass traits. Gramene is an early adopter of BioMart (Smedley et al., <a href="#B30">2009</a>) and develops the GrameneMart that enables scientists to perform advanced querying, download, and online comparison of grass genomic data sources through a single portal (Tello-Ruiz et al., <a href="#B32">2016</a>). Besides the genome framework, Gramene hosts a pathway framework that integrates a plant reactome pathway, and the pathway tool platform "Cyc Pathways," allowing the fast comparison of grass-specific pathways. </p> <p>LIS ( <a href="https://legumeinfo.org/">legumeinfo.org</a>) is the genome information portal for 13 economically important legumes. LIS provides bioinformatics tools for genome and map viewers, and synteny mappings (Dash et al., <a href="#B8">2016</a>). LIS also supports the bridge between the genomic information and the crop improvement by supplying the Germplasm Resources. Likewise, Sol Genomics Network ( <a href="https://solgenomics.net/">solgenomics.net</a>) is a Solanaceae-oriented database containing genome data, genomic tools, and breeders' tools. </p> <p>Other clade-oriented genome databases host &amp;lt;10 genomes (Figure <a href="#F2">2</a>), such as the Rosaceae crop oriented GDR (Jung et al., <a href="#B15">2014</a>), poplar oriented PopGenIE (Sjödin et al., <a href="#B29">2009</a>), cool season food legume oriented CSFL (Main et al., <a href="#B20">2013</a>). These clade-oriented genome databases gather multiple species, often with economic importance from the same clade, and provide genome data as well as tools for traditional breeders. However, the integration of various genomes needs more frequent updates. The visibility of these databases is often limited to specific scientists, and will be time-consuming for plant kingdom-wide researchers to obtain these data. </p> <p>These databases are clade-oriented and distinctive in data and tools compared to other databases. They include economically important crops and their related wild species, and contain genome data that are not included in those comprehensive genome database. They usually provide breeding markers such as molecular markers, various maps, breeder's toolbox, primer design, and germplasm resources. Other genomic tools such as BLAST and synteny mapping are useful to visualize and compare the genome data in various scales. Expression visualization often provides large quantity of expression datasets for fast comparison of various genes and gene families. Furthermore, re-sequencing genomes and biological pathway are also provided in these databases. However, the visibility of these databases needs to be improved. They have been frequently accessed by researchers from the same field, but they are less well-known throughout the plant research community. We recommended that a good genome database should be engaged in the alliance, such as Sol and root, tuber and banana (RTB) crops co-sponsored workshop, sharing and co-developing bioinformatics tools. </p> </div> <div class="outlookandchallenges" title="sec"> <div class="title" tagx="title" title="title">
Outlook and challenges</div> <div class="asuitofstandardsrequired" title="sec"> <div class="title" tagx="title" title="title">
A suit of standards required</div> <p>First, the transparent operation of genome datasets or tools is required. For data changes in the database, updates need to be recorded, so that we can grasp the new information in a timely manner. This can be published by news and so on. The release of data or tools needs to be forecasted. Solgenomics ( <a href="https://solgenomics.net/">solgenomics.net</a>) serves as a good example, as it provides very detailed recent changes to the database. Phytozome also provides genome update in a very conspicuous position. </p> <p>Second, the genome databases, especially for a single species-oriented ones, require a series of minimum standard tools. Data should integrate the reference but not draft genome, CDS, protein, GFF, GO annotation, and the sequencing quality report. Tools should include the gene search, blast, browse, download. </p> <p>Third, databases should be maintained for at least 3 years. Good maintenance secures a steady population of users whereas a bad one will only narrow its academic impact. </p> </div> <div class="tripal:atoolkitforgenomedatabaseconstruction" title="sec"> <div class="title" tagx="title" title="title">
Tripal: a toolkit for genome database construction</div> <p>Tripal is a member of the Generic Model Organism Database (GMOD) organization suite of genome tools. The first official version of Tripal was released in 2009 by Stephen Ficklin and Meg Staton at the Clemson University Genomics Institute (CUGI). Tripal incoperates several features: (1) Chado database and related modules for data storage and search; (2) Community-developed modules to help fasten site construction; (3) Provide an out-of-the-box setup for a genomics site for those who simply want to put new genome assemblies and annotations online; (4) Provide Application Programming Interfaces (APIs) for complete customization such that more advanced displays, look-and-feel, and new functionality can be supported; (5) Sites can be customized as desired or using theme packages from drupal. During the last several years, Tripal has been implemented in 12 genome projects, including both single genome centered and multiple genome databases: banana, cacao, citrus, cool season food legume, cotton, curcurbit, rosaceae, Vaccinium, hardwood genomics project, legume information system, Medicago truncatula, and peanut (Figure <a href="#F4">4</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <p>The tripal toolkit for genome database construction. <b>(A)</b> The Tripal is a biological application of Drupal. <b>(B)</b> core modules of Tripal. <b>(C)</b> carious modules developed during the last several years. <b>(D)</b> The successful applications of tripal in 12 angiosperm plant projects. </p>   </div> </div> <div class="thedatabasefacilitatestheimplementationofthetorontoagreement" title="sec"> <div class="title" tagx="title" title="title">
The database facilitates the implementation of the toronto agreement</div> <p>The papers that reporting genome data usually include (i) genetic and functional changes of the genome and various components such as genes, repetitive elements, and (ii) molecular mechanisms of important traits. However, much information is still not being studied and reported, so the sharing of genomic data raised more concerns. Toronto agreement was proposed in 2009 (Toronto International Data Release Workshop Authors, <a href="#B33">2009</a>), aimed to share the scientific data such as the genome sequences before the publication. Data sharing promotes collaboration and contributes to the efficient use of data. Unfortunately, rapid sharing of genome data is still an area that need improvement. </p> <p>There are a variety of technical means through database to fulfill the Toronto agreement: (i) the establishment of data access thresholds, such as a detailed disclaimer, or the registration system to identify the academic institutions (e-mail address ending with .edu) to share information only to the academic staff; (ii) only provide BLAST and/or Jbrowse and other tools, and DO not provide (it depends and could be optional) download data for data sharing. If only BLAST tools are provided, providers only need to contribute the protein and CDS without the whole genome. Jbrowse could provide genetic information without protein sequence. The provision of data and data sharing on the integrated database is not a nonexistent behavior. The earlier exposed to the public, the earlier intellectual property is committed, and the more efficient it promotes scientific collaborations. At present, due to the requirements by DOE, more genomic data in Phytozome have been released prior to the publication of related genome paper, such as the genome of kalanchoe, monkey flower, and so on. However, the Toronto agreement is yet to be fully implemented and needs in-depth practice. </p> <p>Because of the value of angiosperms, the large number of genomic data has attracted many scientists and still brings us great challenges: the immediacy, integrity and analytical ability of the data. We provide on GDA database the timeline to update each of the recently sequenced plants (Figure <a href="#F3">3</a>). At present, 1,001 <i>Arabidopsis</i> strains (Weigel and Mott, <a href="#B36">2009</a>), 2,489 millet varieties ( <a href="https://db.cngb.org/millet/">db.cngb.org/millet/</a>), and 3,000 rice genomes have completed genome sequencing (The 3,000 rice genomes project). The 10K orchid genome project (J-J Project, <a href="http://sinicaorchid.gzit.net/">sinicaorchid.gzit.net</a>) has been put forward. All these projects made a huge challenge to the current database. </p> <p>The current database for the processing of such large-scale data also lacks large-capacity computing devices and bioinformatics tools. Visualization of large data also poses a major challenge. The variety of data quality also requires an evaluation system to ensure that low-quality data is filtered to speed up the analysis. In general, the plant genome database will become a new biological branch. The supercomputing equipment, bioinformatics algorithms, and tool development need to be introduced and upgraded. In addition, a user-developer interactive than user-friendly interface is required. Overall, the upgrade of the angiosperm database will greatly enhance our understanding of important issues related to angiosperms and greatly promote the crop breeding process. </p> </div> </div> <div class="concludingremarks" title="sec"> <div class="title" tagx="title" title="title">
Concluding remarks</div> <p>The genome data of flowering plants are rapidly accumulating in quantity and complexity. With the big data concept more and more popular in solving big questions, there will be a strong demand to integrate all related data. What's more, bioinformatics tools are usually developed and built firstly in comprehensive or large databases because they attract more researchers. We reviewed and compared the pros and cons on the data, tools, special highlights from three types of genome databases that are mostly used. We also proposed that a comprehensive genome database to host the genomes of all released angiosperms to accelerate the research of major scientific questions at the genome scale. </p> </div> <div class="authorcontributions" title="sec"> <div class="title" tagx="title" title="title">
Author contributions</div> <p>LZ: designed the research; FC, WD, and LZ: collected and analyzed the data; FC, WD, JZ, XG, JC, ZW, ZL, HT, and LZ: wrote, revised, and approved the manuscript. </p> <div class="conflictofintereststatement" title="sec"> <div class="title" tagx="title" title="title">
Conflict of interest statement</div> <p>The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest. </p> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>This work was supported by the National Key Research and Development Program of China (2016YFD0100305) and National Natural Science Foundation of China (81502437), and a start-up fund from Fujian Agriculture and Forestry University to LZ, and FC is supported by a grant from State Key Laboratory of Ecological Pest Control for Fujian and Taiwan Crops (SKB2017004). </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="B1" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Adam-Blondon</span> <span class="given-names" tagx="given-names" title="given-names">A.-F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alaux</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pommier</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cantu</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheng</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cramer</span> <span class="given-names" tagx="given-names" title="given-names">G. R.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Towards an open grapevine information system</span>. <span class="source" tagx="source" title="source">Hort. Res.</span> <span class="volume" tagx="volume" title="volume">3</span>: <span class="fpage" tagx="fpage" title="fpage">16056</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/hortres.2016.56">10.1038/hortres.2016.56</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27917288">27917288</a> </span> </span> </li> <li tag="ref"> <a name="B2" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Badouin</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gouzy</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grassa</span> <span class="given-names" tagx="given-names" title="given-names">C. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Murat</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Staton</span> <span class="given-names" tagx="given-names" title="given-names">S. E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cottret</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">The sunflower genome provides insights into oil metabolism, flowering and Asterid evolution </span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">546</span>, <span class="fpage" tagx="fpage" title="fpage">148</span>– <span class="lpage" tagx="lpage" title="lpage">152</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nature22380">10.1038/nature22380</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28538728">28538728</a> </span> </span> </li> <li tag="ref"> <a name="B3" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Balick</span> <span class="given-names" tagx="given-names" title="given-names">M. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cox</span> <span class="given-names" tagx="given-names" title="given-names">P. A.</span> </span> </span> ( <span class="year" tagx="year" title="year">1996</span>). <span class="source" tagx="source" title="source">Plants, People, and Culture: the Science of Ethnobotany</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York, NY</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Scientific American Library</span>. </span> </li> <li tag="ref"> <a name="B4" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bell</span> <span class="given-names" tagx="given-names" title="given-names">C. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Soltis</span> <span class="given-names" tagx="given-names" title="given-names">D. E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Soltis</span> <span class="given-names" tagx="given-names" title="given-names">P. S.</span> </span> </span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">The age and diversification of the angiosperms re-revisited</span>. <span class="source" tagx="source" title="source">Am. J. Bot.</span> <span class="volume" tagx="volume" title="volume">97</span>, <span class="fpage" tagx="fpage" title="fpage">1296</span>– <span class="lpage" tagx="lpage" title="lpage">1303</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.3732/ajb.0900346">10.3732/ajb.0900346</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21616882">21616882</a> </span> </span> </li> <li tag="ref"> <a name="B5" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Borsch</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Löhne</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wiersema</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Phylogeny and evolutionary patterns in Nymphaeales: integrating genes, genomes and morphology </span>. <span class="source" tagx="source" title="source">Taxon</span> <span class="volume" tagx="volume" title="volume">57</span>, <span class="fpage" tagx="fpage" title="fpage">1052</span>– <span class="lpage" tagx="lpage" title="lpage">1081</span>. Available online at: <a href="http://www.ingentaconnect.com/content/iapt/tax/2008/00000057/00000004/art00004">http://www.ingentaconnect.com/content/iapt/tax/2008/00000057/00000004/art00004</a> </span> </li> <li tag="ref"> <a name="B6" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Charlesworth</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Plant sex chromosomes</span>. <span class="source" tagx="source" title="source">Annu. Rev. Plant Biol.</span> <span class="volume" tagx="volume" title="volume">67</span>, <span class="fpage" tagx="fpage" title="fpage">2.1</span>– <span class="lpage" tagx="lpage" title="lpage">2.24</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1146/annurev-arplant-043015-111911">10.1146/annurev-arplant-043015-111911</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26653795">26653795</a> </span> </span> </li> <li tag="ref"> <a name="B7" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Choudhuri</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="source" tagx="source" title="source">Bioinformatics for Beginners: Genes, Genomes, Molecular Evolution, Databases and Analytical Tools </span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York, NY</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Academic Press</span>. </span> </li> <li tag="ref"> <a name="B8" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dash</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Campbell</span> <span class="given-names" tagx="given-names" title="given-names">J. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cannon</span> <span class="given-names" tagx="given-names" title="given-names">E. K. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cleary</span> <span class="given-names" tagx="given-names" title="given-names">A. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kalberer</span> <span class="given-names" tagx="given-names" title="given-names">S. R.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Legume information system ( <a href="https://legumeinfo.org/">legumeinfo.org</a>): a key component of a set of federated data resources for the legume family </span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="volume" tagx="volume" title="volume">44</span>, <span class="fpage" tagx="fpage" title="fpage">D1181</span>– <span class="lpage" tagx="lpage" title="lpage">D1188</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkv1159">10.1093/nar/gkv1159</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26546515">26546515</a> </span> </span> </li> <li tag="ref"> <a name="B9" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Duvick</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fu</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Muppirala</span> <span class="given-names" tagx="given-names" title="given-names">U.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sabharwal</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wilkerson</span> <span class="given-names" tagx="given-names" title="given-names">M. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lawrence</span> <span class="given-names" tagx="given-names" title="given-names">C. J.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">PlantGDB: a resource for comparative plant genomics</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="volume" tagx="volume" title="volume">36</span>, <span class="fpage" tagx="fpage" title="fpage">959</span>– <span class="lpage" tagx="lpage" title="lpage">965</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkm1041">10.1093/nar/gkm1041</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18063570">18063570</a> </span> </span> </li> <li tag="ref"> <a name="B10" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garcia-Hernandez</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Berardini</span> <span class="given-names" tagx="given-names" title="given-names">T. Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Crist</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Doyle</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huala</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">TAIR: a resource for integrated Arabidopsis data</span>. <span class="source" tagx="source" title="source">Funct. Integr. Genomics</span> <span class="volume" tagx="volume" title="volume">2</span>, <span class="fpage" tagx="fpage" title="fpage">239</span>– <span class="lpage" tagx="lpage" title="lpage">253</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s10142-002-0077-z">10.1007/s10142-002-0077-z</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12444417">12444417</a> </span> </span> </li> <li tag="ref"> <a name="B11" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goodstein</span> <span class="given-names" tagx="given-names" title="given-names">D. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shu</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Howson</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Neupane</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hayes</span> <span class="given-names" tagx="given-names" title="given-names">R. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fazo</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Phytozome: a comparative platform for green plant genomics</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="volume" tagx="volume" title="volume">40</span>, <span class="fpage" tagx="fpage" title="fpage">1178</span>– <span class="lpage" tagx="lpage" title="lpage">1186</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkr944">10.1093/nar/gkr944</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22110026">22110026</a> </span> </span> </li> <li tag="ref"> <a name="B12" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huala</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dickerman</span> <span class="given-names" tagx="given-names" title="given-names">A. W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garcia-hernandez</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weems</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reiser</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lafond</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2001</span>). <span class="mixed-article-title" title="mixed-article-title">The Arabidopsis Information Resource (TAIR): a comprehensive database and web-based information retrieval, analysis, and visualization system for a model plant </span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="volume" tagx="volume" title="volume">29</span>, <span class="fpage" tagx="fpage" title="fpage">102</span>– <span class="lpage" tagx="lpage" title="lpage">105</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/29.1.102">10.1093/nar/29.1.102</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11125061">11125061</a> </span> </span> </li> <li tag="ref"> <a name="B13" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hubble</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> </span> ( <span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">So much more to know</span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">309</span>, <span class="fpage" tagx="fpage" title="fpage">78</span>– <span class="lpage" tagx="lpage" title="lpage">102</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.309.5731.78b">10.1126/science.309.5731.78b</a> </span> </span> </li> <li tag="ref"> <a name="B14" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jarvis</span> <span class="given-names" tagx="given-names" title="given-names">D. E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ho</span> <span class="given-names" tagx="given-names" title="given-names">Y. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lightfoot</span> <span class="given-names" tagx="given-names" title="given-names">D. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmöckel</span> <span class="given-names" tagx="given-names" title="given-names">S. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Borm</span> <span class="given-names" tagx="given-names" title="given-names">T. J. A.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">The genome of Chenopodium quinoa</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">542</span>, <span class="fpage" tagx="fpage" title="fpage">307</span>– <span class="lpage" tagx="lpage" title="lpage">312</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nature21370">10.1038/nature21370</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28178233">28178233</a> </span> </span> </li> <li tag="ref"> <a name="B15" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jung</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ficklin</span> <span class="given-names" tagx="given-names" title="given-names">S. P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheng</span> <span class="given-names" tagx="given-names" title="given-names">C. H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blenda</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zheng</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">The Genome Database for Rosaceae (GDR): year 10 update</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="volume" tagx="volume" title="volume">42</span>, <span class="fpage" tagx="fpage" title="fpage">1237</span>– <span class="lpage" tagx="lpage" title="lpage">1244</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkt1012">10.1093/nar/gkt1012</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24225320">24225320</a> </span> </span> </li> <li tag="ref"> <a name="B16" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kersey</span> <span class="given-names" tagx="given-names" title="given-names">P. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Allen</span> <span class="given-names" tagx="given-names" title="given-names">J. E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Christensen</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Davis</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Falin</span> <span class="given-names" tagx="given-names" title="given-names">L. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grabmueller</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Ensembl Genomes 2013: scaling up access to genome-wide data</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="volume" tagx="volume" title="volume">42</span>, <span class="fpage" tagx="fpage" title="fpage">546</span>– <span class="lpage" tagx="lpage" title="lpage">552</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkt979">10.1093/nar/gkt979</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24163254">24163254</a> </span> </span> </li> <li tag="ref"> <a name="B17" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paterson</span> <span class="given-names" tagx="given-names" title="given-names">A. H.</span> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">PGDD: a database of gene and genome duplication in plants</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="volume" tagx="volume" title="volume">41</span>, <span class="fpage" tagx="fpage" title="fpage">D1152</span>– <span class="lpage" tagx="lpage" title="lpage">D1158</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gks1104">10.1093/nar/gks1104</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23180799">23180799</a> </span> </span> </li> <li tag="ref"> <a name="B18" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Diet shapes the evolution of the vertebrate bitter taste receptor gene repertoire</span>. <span class="source" tagx="source" title="source">Mol. Biol. Evol.</span> <span class="volume" tagx="volume" title="volume">31</span>, <span class="fpage" tagx="fpage" title="fpage">303</span>– <span class="lpage" tagx="lpage" title="lpage">309</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/molbev/mst219">10.1093/molbev/mst219</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24202612">24202612</a> </span> </span> </li> <li tag="ref"> <a name="B19" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lyons</span> <span class="given-names" tagx="given-names" title="given-names">E. H.</span> </span> </span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="source" tagx="source" title="source">CoGe, a New Kind of Comparative Genomics Platform: Insights Into the Evolution of Plant Genomes </span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Ann Arbor, MI</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Proquest, Umi Dissertation Publishing</span>. </span> </li> <li tag="ref"> <a name="B20" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Main</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheng</span> <span class="given-names" tagx="given-names" title="given-names">C.-H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ficklin</span> <span class="given-names" tagx="given-names" title="given-names">S. P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jung</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zheng</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coyne</span> <span class="given-names" tagx="given-names" title="given-names">C. J.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">The cool season food legume database: an integrated resource for basic, translational and applied research </span>, in <span class="source" tagx="source" title="source">Plant and Animal Genome XXI Conference</span> ( <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Diego, CA</span>). </span> </li> <li tag="ref"> <a name="B21" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Niimura</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Olfactory receptor multigene family in vertebrates: from the viewpoint of evolutionary genomics </span>. <span class="source" tagx="source" title="source">Curr. Genomics</span> <span class="volume" tagx="volume" title="volume">13</span>, <span class="fpage" tagx="fpage" title="fpage">103</span>– <span class="lpage" tagx="lpage" title="lpage">114</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.2174/138920212799860706">10.2174/138920212799860706</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23024602">23024602</a> </span> </span> </li> <li tag="ref"> <a name="B22" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Osorio</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vorobyev</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">A review of the evolution of animal colour vision and visual communication signals</span>. <span class="source" tagx="source" title="source">Vis. Res.</span> <span class="volume" tagx="volume" title="volume">48</span>, <span class="fpage" tagx="fpage" title="fpage">2042</span>– <span class="lpage" tagx="lpage" title="lpage">2051</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.visres.2008.06.018">10.1016/j.visres.2008.06.018</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18627773">18627773</a> </span> </span> </li> <li tag="ref"> <a name="B23" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Poliakov</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Foong</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brudno</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dubchak</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">GenomeVISTA — an integrated software package for whole-genome alignment and visualization</span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">30</span>, <span class="fpage" tagx="fpage" title="fpage">2654</span>– <span class="lpage" tagx="lpage" title="lpage">2655</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/btu355">10.1093/bioinformatics/btu355</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24860159">24860159</a> </span> </span> </li> <li tag="ref"> <a name="B24" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Proost</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van Bel</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sterck</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Billiau</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van Parys</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van De Peer</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">PLAZA : A comparative genomics resource to study gene and genome evolution in Plants</span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">21</span>, <span class="fpage" tagx="fpage" title="fpage">3718</span>– <span class="lpage" tagx="lpage" title="lpage">3731</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.109.071506">10.1105/tpc.109.071506</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20040540">20040540</a> </span> </span> </li> <li tag="ref"> <a name="B25" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pryer</span> <span class="given-names" tagx="given-names" title="given-names">K. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schneider</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zimmer</span> <span class="given-names" tagx="given-names" title="given-names">E. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Banks</span> <span class="given-names" tagx="given-names" title="given-names">J. A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">Deciding among green plants for whole genome studies</span>. <span class="source" tagx="source" title="source">Trends Plant Sci.</span> <span class="volume" tagx="volume" title="volume">1385</span>, <span class="fpage" tagx="fpage" title="fpage">550</span>– <span class="lpage" tagx="lpage" title="lpage">554</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S1360-1385(02)02375-0">10.1016/S1360-1385(02)02375-0</a> </span> </span> </li> <li tag="ref"> <a name="B26" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raskin</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ribnicky</span> <span class="given-names" tagx="given-names" title="given-names">D. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Komarnytsky</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ilic</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Poulev</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Borisjuk</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">Plants and human health in the twenty-first century</span>. <span class="source" tagx="source" title="source">Trends Biotechnol.</span> <span class="volume" tagx="volume" title="volume">20</span>, <span class="fpage" tagx="fpage" title="fpage">522</span>– <span class="lpage" tagx="lpage" title="lpage">531</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0167-7799(02)02080-2">10.1016/S0167-7799(02)02080-2</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12443874">12443874</a> </span> </span> </li> <li tag="ref"> <a name="B27" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schattner</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> </span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="source" tagx="source" title="source">Genomes, Browsers and Databases: Data-Mining Tools for Integrated Genomic Databases</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Cambridge</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Cambridge University Press</span>. </span> </li> <li tag="ref"> <a name="B28" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schoof</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zaccaria</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gundlach</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lemcke</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rudd</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kolesov</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">MIPS Arabidopsis thaliana Database (MAtDB): an integrated biological knowledge resource based on the first complete plant genome </span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="volume" tagx="volume" title="volume">30</span>, <span class="fpage" tagx="fpage" title="fpage">91</span>– <span class="lpage" tagx="lpage" title="lpage">93</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/30.1.91">10.1093/nar/30.1.91</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11752263">11752263</a> </span> </span> </li> <li tag="ref"> <a name="B29" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sjödin</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Street</span> <span class="given-names" tagx="given-names" title="given-names">N. R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sandberg</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gustafsson</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jansson</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">The Populus Genome Integrative Explorer (PopGenIE): a new resource for exploring the <i>Populus</i> genome </span>. <span class="source" tagx="source" title="source">New Phytol.</span> <span class="volume" tagx="volume" title="volume">182</span>, <span class="fpage" tagx="fpage" title="fpage">1013</span>– <span class="lpage" tagx="lpage" title="lpage">1025</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1469-8137.2009.02807.x">10.1111/j.1469-8137.2009.02807.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19383103">19383103</a> </span> </span> </li> <li tag="ref"> <a name="B30" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smedley</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haider</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ballester</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Holland</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">London</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thorisson</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">BioMart-biological queries made easy</span>. <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">22</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-10-22">10.1186/1471-2164-10-22</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19144180">19144180</a> </span> </span> </li> <li tag="ref"> <a name="B31" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Spannagl</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nussbaumer</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bader</span> <span class="given-names" tagx="given-names" title="given-names">K. C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martis</span> <span class="given-names" tagx="given-names" title="given-names">M. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seidel</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kugler</span> <span class="given-names" tagx="given-names" title="given-names">K. G.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">PGSB PlantsDB : updates to the database framework for comparative plant genome research</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="volume" tagx="volume" title="volume">44</span>, <span class="fpage" tagx="fpage" title="fpage">1141</span>– <span class="lpage" tagx="lpage" title="lpage">1147</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkv1130">10.1093/nar/gkv1130</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26527721">26527721</a> </span> </span> </li> <li tag="ref"> <a name="B32" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tello-Ruiz</span> <span class="given-names" tagx="given-names" title="given-names">M. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stein</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wei</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Preece</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Olson</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Naithani</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Gramene 2016: comparative plant genomics and pathway resources</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="volume" tagx="volume" title="volume">44</span>, <span class="fpage" tagx="fpage" title="fpage">D1133</span>– <span class="lpage" tagx="lpage" title="lpage">D1140</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkv1179">10.1093/nar/gkv1179</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26553803">26553803</a> </span> </span> </li> <li tag="ref"> <a name="B33" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="collab'">collab: Toronto, International Data Release Workshop Authors</span> </span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Prepublication data sharing</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">461</span>, <span class="fpage" tagx="fpage" title="fpage">168</span>– <span class="lpage" tagx="lpage" title="lpage">170</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/461168a">10.1038/461168a</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19741685">19741685</a> </span> </span> </li> <li tag="ref"> <a name="B34" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">VanBuren</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bryant</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Edger</span> <span class="given-names" tagx="given-names" title="given-names">P. P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burgess</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Challabathula</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Single-molecule sequencing of the desiccation-tolerant grass <i>Oropetium thomaeum</i> </span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">527</span>, <span class="fpage" tagx="fpage" title="fpage">508</span>– <span class="lpage" tagx="lpage" title="lpage">511</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nature15714">10.1038/nature15714</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26560029">26560029</a> </span> </span> </li> <li tag="ref"> <a name="B35" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van de Peer</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mizrachi</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marchal</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> </span> ( <span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">The evolutionary significance of polyploidy</span>. <span class="source" tagx="source" title="source">Nat. Rev. Genet.</span> <span class="volume" tagx="volume" title="volume">18</span>, <span class="fpage" tagx="fpage" title="fpage">411</span>– <span class="lpage" tagx="lpage" title="lpage">424</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nrg.2017.26">10.1038/nrg.2017.26</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28502977">28502977</a> </span> </span> </li> <li tag="ref"> <a name="B36" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weigel</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mott</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">The 1001 genomes project for Arabidopsis thaliana</span>. <span class="source" tagx="source" title="source">Genome Biol.</span> <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">107</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1186/gb-2009-10-5-107">10.1186/gb-2009-10-5-107</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19519932">19519932</a> </span> </span> </li> <li tag="ref"> <a name="B37" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zeng</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Q.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kong</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Resolution of deep angiosperm phylogeny using conserved nuclear genes and estimates of early divergence times </span>. <span class="source" tagx="source" title="source">Nat. Commun.</span> <span class="volume" tagx="volume" title="volume">5</span>, <span class="fpage" tagx="fpage" title="fpage">4956</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/ncomms5956">10.1038/ncomms5956</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25249442">25249442</a> </span> </span> </li> </ul> </div> </div>  </body></html>