<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Ecol Evol</span> <span class="iso-abbrev" title="iso-abbrev">Ecol Evol</span> <span class="doi" title="doi">10.1002/(ISSN)2045-7758</span> <span class="publisher-id" title="publisher-id">ECE3</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Ecology and Evolution</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">2045-7758</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">John Wiley and Sons Inc.</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Hoboken</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6093152">6093152</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1002/ece3.2483">10.1002/ece3.2483</a> </span> <span class="publisher-id" title="publisher-id">ECE32483</span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Original Research</span>: <span class="subject" title="subject">Original Research</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Higher efficiency of <span class="styled-content" tagx="styled-content" title="styled-content">ISSR</span> markers over plastid <i>psbA‐trnH</i> region in resolving taxonomical status of genus <i>Ocimum</i> L. </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar</span> <span class="given-names" tagx="given-names" title="given-names">Amit</span> </span> <a href="#ece32483-aff-0001"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mishra</span> <span class="given-names" tagx="given-names" title="given-names">Priyanka</span> </span> <a href="#ece32483-aff-0001"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baskaran</span> <span class="given-names" tagx="given-names" title="given-names">Kuppusamy</span> </span> <a href="#ece32483-aff-0001"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shukla</span> <span class="given-names" tagx="given-names" title="given-names">Ashutosh K.</span> </span> <a href="#ece32483-aff-0002"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shasany</span> <span class="given-names" tagx="given-names" title="given-names">Ajit K.</span> </span> <a href="#ece32483-aff-0002"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sundaresan</span> <span class="given-names" tagx="given-names" title="given-names">Velusamy</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">vsundaresan@cimap.res.in</span> </span> <a href="#ece32483-aff-0001"> <sup>1</sup> </a> </span> </div> <span class="citation_author_institution" id="ece32483-aff-0001">[1], <span class="named-content" title="named-content">Department of Plant Biology and Systematics</span> <span class="institution" title="institution">CSIR – Central Institute of Medicinal and Aromatic Plants, Research Centre</span> <span class="named-content" title="named-content">Bengaluru</span> <span class="country" tagx="country" title="country">India</span> </span> <span class="citation_author_institution" id="ece32483-aff-0002">[2], <span class="named-content" title="named-content">Biotechnology Division</span> <span class="institution" title="institution">CSIR – Central Institute of Medicinal and Aromatic Plants</span> <span class="named-content" title="named-content">Lucknow</span> <span class="country" tagx="country" title="country">India</span> </span> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"> <span class="label" tagx="label" title="label">*</span> <b>Correspondence</b> <br /><br /> Velusamy Sundaresan, Department of Plant Biology and Systematics, CSIR – Central Institute of Medicinal and Aromatic Plants, Research Centre, Bengaluru, India. <br /><br /> Email: <span class="email" tagx="email" title="email">vsundaresan@cimap.res.in</span> <br /><br /> </div> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2016-10-10</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2016-11</span> </span> <span class="volume" tagx="volume" title="volume">6</span> <span class="issue" tagx="issue" title="issue">21</span> <span class="issue-id" tagx="issue-id" title="issue-id">10.1002/ece3.2016.6.issue-21</span> <span class="fpage" tagx="fpage" title="fpage">7671</span> <span class="lpage" tagx="lpage" title="lpage">7682</span> <span class="history" title="history"> <span class="received" title="received">received: 2016-2-16</span> <span class="rev-recd" title="rev-recd">rev-recd: 2016-8-19</span> <span class="accepted" title="accepted">accepted: 2016-8-22</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open access article under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited. </span> </span> </div> <a class="self-uri" href="file:ECE3-6-7671.pdf" title="self-uri">file:ECE3-6-7671.pdf</a> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p>High level of morphological as well as chemical variability exists within the genus <i>Ocimum</i>, and its taxonomy and phylogenetic relationships are still doubtful. For evaluating interspecific genetic relationships among the <i>Ocimum</i> species, genotyping with intersimple sequence repeat ( <span class="styled-content" tagx="styled-content" title="styled-content">ISSR</span>) markers and sequence analyses of noncoding <i>psbA‐trnH</i> intergenic region belonging to chloroplast <span class="styled-content" tagx="styled-content" title="styled-content">DNA</span> were carried out. Although <span class="styled-content" tagx="styled-content" title="styled-content">ISSR</span> markers are highly efficient and reproducible, they have not been used extensively in phylogenetic studies. The use of the plastidial barcode candidate was expected to provide more variable and informative insight into evolutionary rates, and was thus employed as a phylogenetic marker to assess interspecific relationships. This study revealed that the <span class="styled-content" tagx="styled-content" title="styled-content">ISSR</span> markers were more efficient than <i>psbA‐trnH</i> sequences in resolving the current status of <i>Ocimum</i> L. genus. Distance‐ and character‐based methodological approaches applied on the molecular data with biparental and maternal inheritance were used for deducing the phylogenetic relationships among <i>Ocimum</i> species. Average polymorphic information content (0.344) and resolving power (6.285) depicted through <span class="styled-content" tagx="styled-content" title="styled-content">ISSR</span> markers proved to be efficient in discriminating the studied species of <i>Ocimum</i>. The primers used in this study revealed 99.585% polymorphism across the species demonstrating the polymorphic nature of <span class="styled-content" tagx="styled-content" title="styled-content">ISSR</span> markers. </p> </div> <div class="kwd-group"> <div class="kwd" title="kwd"> <span class="styled-content" tagx="styled-content" title="styled-content">ISSR</span> </div> <div class="kwd" title="kwd">
Lamiaceae</div> <div class="kwd" title="kwd"> <i>Ocimum</i> spp. </div> <div class="kwd" title="kwd">
Phylogeny</div> <div class="kwd" title="kwd"> <i>psbA‐trnH</i> </div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Council of Scientific and Industrial Research</div> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
XIIth FYP project ChemBio</div> <span class="award-id" tagx="award-id" title="award-id">BSC‐0203</span> </div> </div> <div class="counts" title="counts"> <span class="fig-count" title="fig-count">fig-count: </span> <span class="table-count" title="table-count">table-count: </span> <span class="page-count" title="page-count">page-count: </span> <span class="word-count" title="word-count">word-count: </span> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="source-schema-version-number" value="2.0" /> <meta name="component-id" value="ece32483" /> <meta name="cover-date" value="November 2016" /> <meta name="details-of-publishers-convertor" value="Converter:WILEY_ML3GV2_TO_NLMPMC version:version=5.4.3 mode:remove_FC converted:18.07.2018" /> </div> </div> <p> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Kumar</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Mishra</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Baskaran</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Shukla</span>, <span class="given-names" tagx="given-names" title="given-names">A. K.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Shasany</span>, <span class="given-names" tagx="given-names" title="given-names">A. K.</span> </span>, and <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Sundaresan</span>, <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> ( <span class="year" tagx="year" title="year">2016</span>), <span class="mixed-article-title" title="mixed-article-title">Higher efficiency of <span class="styled-content" tagx="styled-content" title="styled-content">ISSR</span> markers over plastid <i>psbA‐trnH</i> region in resolving taxonomical status of genus <i>Ocimum</i> L. </span> <span class="source" tagx="source" title="source">Ecology and Evolution</span>, <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">7671</span>– <span class="lpage" tagx="lpage" title="lpage">7682</span>. doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1002/ece3.2483">10.1002/ece3.2483</a> </span> </span> </p> <div class="fn-group" title="fn-group"> <div class="fn-type-funding" title="funding"> <p> <b>Funding Information</b> </p> <p>Council of Scientific and Industrial Research (CSIR), New Delhi, XIIth FYP project ChemBio (BSC‐ 0203). </p> </div> <div class="fn-type-" title=""> <p>CIMAP Communication No.: CIMAP/PUB/2016/13.</p> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <span class="label" tagx="label" title="label">1</span> <div class="title" tagx="title" title="title">
Introduction</div> <p>The genus <i>Ocimum</i> L., belonging to family Lamiaceae (sixth largest family), collectively called as basil is an important economic and medicinal herb, widely distributed in tropical, subtropical, and warm temperate regions of the world (Paton, Harley, &amp;amp; Harley, <a href="#ece32483-bib-0031">1999</a>). Due to substantial taxonomical complexity caused by polyploidy and possibility of inter‐ and intraspecific hybridization within the genus <i>Ocimum</i>, estimates of the number of species vary from 30 (Paton, <a href="#ece32483-bib-0030">1992</a>) to 160 (Pushpangadan &amp;amp; Bradu, <a href="#ece32483-bib-0035">1995</a>). To add further to this complexity, there are varieties having similar chemical compositions within the species that do not differ significantly in morphology (Verma et al., <a href="#ece32483-bib-0049">2016</a>). Carvoic‐Stanko et al. ( <a href="#ece32483-bib-0006">2010</a>) reported that basic chromosome number in <i>Ocimum</i> spp. is <i>x</i> = 12. <i>O. basilicum</i> and <i>O. americanum</i> are reported to be tetraploid (2 <i>n</i> = 4 <i>x</i> = 48) and hexaploid (2 <i>n</i> = 6 <i>x</i> = 72), respectively. <i>O. tenuiflorum</i> is a perennial shrub with a basic chromosome number of <i>x</i> = 9, while <i>O. gratissimum</i> has a basic chromosome number <i>x</i> = 10. <i>O. tenuiflorum</i> L., <i>O. basilicum</i> L., <i>O. gratissimum</i> L., <i>O. kilimandscharicum</i> Gurke, <i>O. americanum</i> L., and <i>O. africanum</i> Lour. are important species belonging to the genus <i>Ocimum</i> that grow in different parts of the world and are known for their diverse medicinal properties and as sources of important essential oils (Carvoic‐Stanko et al., <a href="#ece32483-bib-0006">2010</a>; Prakash &amp;amp; Gupta, <a href="#ece32483-bib-0033">2005</a>). In India, nine species are found, of which three are exotic (Balyan &amp;amp; Pushpangadan, <a href="#ece32483-bib-0004">1988</a>). <i>O. adscendens</i> is a lesser known species with few studies carried out on it (Asthana, Tripathi, &amp;amp; Dixit, <a href="#ece32483-bib-0003">1986</a>; Verma et al., <a href="#ece32483-bib-0049">2016</a>). </p> <p> <i>Ocimum</i> spp. have been used in folk medicine due to their diverse biological activities, such as mosquito repellency, antimicrobial, insecticidal, antiseptic, carminative, stimulant, antispasmodic, antioxidant, and antipyretic properties (Padalia, Verma, Chauhan, &amp;amp; Chanotiya, <a href="#ece32483-bib-0028">2013</a>). <i>O. tenuiflorum</i> has been well documented in Ayurveda for its therapeutic potential as a <i>Kapha</i>‐balancing plant (Shukla et al., <a href="#ece32483-bib-0042">2013</a>) and possesses antioxidant and antimicrobial activities due to its phenolic and aromatic compounds (Verma, Padalia, Chauhan, &amp;amp; Thul, <a href="#ece32483-bib-0050">2013</a>). The oil components of <i>Ocimum</i> have been found to be produced by two different biochemical pathways, viz. shikimic acid pathway (phenylpropanoids) and mevalonic acid pathway (terpenes). Essential oil consisting of monoterpenes, sesquiterpenes, and phenylpropanoids reported from <i>Ocimum</i> spp. includes linalool, linalyl acetate, geraniol, citral, camphor, eugenol, methyl eugenol, methyl chavicol, methyl cinnamate, thymol, safrole, which are of immense value and have been used principally in the food and cosmetic industries (Balyan &amp;amp; Pushpangadan, <a href="#ece32483-bib-0004">1988</a>; Vina &amp;amp; Murillo, <a href="#ece32483-bib-0052">2003</a>). </p> <p>The genus <i>Ocimum</i> is very variable and possesses a wide range of intra‐ and interspecific genetic diversity. According to previous reports on <i>Ocimum</i>, the genus is grouped into three sections: <i>Ocimum</i> ( <i>Ocymodon</i> Benth.), <i>Hierocymum</i> Benth., and <i>Gymnocymum</i> Benth. Section <i>Ocimum</i> was again divided into subsections: <i>Ocimum</i>, <i> Gratissima</i>, and <i>Hiantia</i> Benth (Paton et al., <a href="#ece32483-bib-0031">1999</a>). Further, Pushpangadan divided the whole taxa into two groups: the Basilicum group, containing herbaceous annuals/perennials with black, ellipsoid, strongly mucilaginous seeds, and the Sanctum group, consisting of perennial shrubs with brown, globose, nonmucilaginous, or weakly mucilaginous seeds. According to Pushpangadan's classification, the Basilicum group contains only section <i>Ocimum</i> subsection <i>Ocimum</i> (Carvoic‐Stanko et al., <a href="#ece32483-bib-0006">2010</a>). However, Pushpangadan's infrageneric classification does not act in accordance with the <i>International Code of Botanical Nomenclature</i>. Several studies throughout worldwide related to cytotaxonomical studies, taxonomical status and geographic distribution, classification and relationships, genetic diversity and phylogenetic study, and chemical characterization (Balyan &amp;amp; Pushpangadan, <a href="#ece32483-bib-0004">1988</a>; Carvoic‐Stanko et al., <a href="#ece32483-bib-0007">2011</a>; Paton et al., <a href="#ece32483-bib-0031">1999</a>; Sobti &amp;amp; Pushpangadan, <a href="#ece32483-bib-0044">1979</a>; Verma et al., <a href="#ece32483-bib-0049">2016</a>; Vieira &amp;amp; Simon, <a href="#ece32483-bib-0051">2000</a>) reported ample extent of genetic variability within the genus <i>Ocimum,</i> making it taxonomically and phylogenetically convoluted. </p> <p>Inter simple sequence repeat (ISSR) markers (Zietkiewicz, Rafalski, &amp;amp; Labuda, <a href="#ece32483-bib-0055">1994</a>) are dominant genetic markers generated through polymerase chain reaction (PCR) amplification of genomic DNA using a single primer that amplifies the regions between adjacent, inversely oriented simple sequence repeats, provided the primer binding sites are located within an amplifiable range. The evolutionary rate of change within microsatellites is considerably higher as compared to most other types of DNA, which increases the likelihood of polymorphism in these sequences. However, ISSR markers are present in both nuclear and organellar genomes and are responsible for the generation of highly polymorphic loci. ISSR markers have frequently been used in the characterization of germplasm collections and in the screening of genetic diversity within species (Kumar, Mishra, Singh, &amp;amp; Sundaresan, <a href="#ece32483-bib-0019">2014</a>; Mishra et al., <a href="#ece32483-bib-0021">2015</a>), cultivar identification (Chowdhury, Vandenberg, &amp;amp; Warkentin, <a href="#ece32483-bib-0009">2002</a>), molecular mapping, genomic instability assessment, but they are rarely used for phylogeny reconstruction at the species level. The ISSR technique is less expensive comparatively to RFLPs and AFLPs, giving higher reproducibility than RAPDs. Individuals of the same species usually show a few to no difference between their ISSR patterns, whereas closely related taxa, that is, subspecies and species, give a specific banding profile that can be used for deducing phylogenetic relationship. </p> <p>Chloroplast DNA (cpDNA) is the smallest as compared to mitochondria and nuclear genome, and during the course of evolution, it is assumed to be conserved in terms of nucleotide substitution with very little rearrangements, which makes the molecule best suited to be used in resolving phylogenetic relationships especially at higher levels of evolution (Zurawski &amp;amp; Clegg, <a href="#ece32483-bib-0056">1987</a>). cpDNA sequences, particularly the noncoding regions, were in demand as source of markers because they are evolving faster as they have fewer functional constraints than gene‐coding regions in terms of nucleotide substitution rates (Kimura, <a href="#ece32483-bib-0018">1983</a>). Also, this allows comparisons between seed gene flow and pollen gene flow as well as the identification of hybridization events when compared to nuclear, biparentally inherited DNA, demonstrating genealogical structure (Dong et al., <a href="#ece32483-bib-0011">2013</a>). Thus, noncoding cpDNA sequences are expected to provide more variable and informative characters for phylogenetic studies of evolutionary rates. The <i>psbA–trnH</i> intergenic spacer, an evolutionarily plastid region, is employed as a phylogenetic marker to assess interspecific relationships. </p> <p> <i>Ocimum</i> species are similar in apparent vegetative morphology and often debated upon with regard to their nomenclature and taxonomical positioning. The nomenclature of <i>Ocimum</i> species and varieties is complicated and often confusing. Besides, different physicochemical properties have been observed in the oils extracted from morphologically identical plants (single phenotype) (Singh et al., <a href="#ece32483-bib-0043">2004</a>). This study on <i>Ocimum</i> species from India was carried out with the following objectives: (1) comparative phylogenetic analysis using ISSR markers and plastid DNA ( <i>psbA‐trnH</i>) sequences and (2) evaluation of the efficiency of ISSR markers vis‐à‐vis chloroplast noncoding <i>psbA‐trnH</i> region in elucidating the phylogenetic relationship among medicinally/agro‐economically important species of <i>Ocimum</i>. </p> </div> <div class="materialsandmethods" title="sec"> <span class="label" tagx="label" title="label">2</span> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="plantmaterial" title="sec"> <span class="label" tagx="label" title="label">2.1</span> <div class="title" tagx="title" title="title">
Plant material</div> <p>Seeds from seven different species of <i>Ocimum</i> (few of them represented by different varieties) were collected from different parts of India (Figure <a href="#ece32483-fig-0001">1</a>; Table <a href="#ece32483-tbl-0001">1</a>). Seeds were grown in the ChemBio genebank at CSIR‐CIMAP Research Centre, Bengaluru, India. The different <i>Ocimum</i> species included were <i>O. tenuiflorum</i> L. <i>, O. basilicum</i> L. <i>, O. africanum</i> Lour, <i>O. americanum</i> L. <i>, O. kilimandscharicum</i> Gurke <i>, O. gratissimum</i> L. <i>, O. viride</i> Willd., and <i>O. adscendens</i> Willd. Along with the <i>Ocimum</i> species, an outgroup taxon <i>Pogostemon cablin</i> L. was chosen because of its known distant relationship. According to the plant list, <i>O. viride</i> is synonymous to <i>O. gratissimum</i> ( <a href="http://www.theplantlist.org/">http://www.theplantlist.org/</a>), but looks morphologically different. All the voucher specimens of the plant accessions used were deposited in the herbarium at CSIR‐Central Institute of Medicinal and Aromatic Plants, Research Centre, Bengaluru, India, located at 12°58′N latitude and 77°35′E longitude for future reference. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <p>Different <i>Ocimum</i> species from India. (A) <i>Ocimum basilicum</i>, (B) <i>Ocimum americanum</i>, (C) <i>Ocimum kilimandscharicum</i>, (D) <i>Ocimum africanum</i>, (E) <i>Ocimum tenuiflorum</i> var. Ram tulsi, (F) <i>Ocimum adscendens</i>, (G) <i>Ocimum tenuiflorum</i> var. Krishna tulsi, (H) <i>Ocimum gratissimum</i>, (I) <i>Ocimum viride</i> </p>   </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  <p>Geographic location of <i>Ocimum</i> seed collections </p>  <table> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <thead> <tr> <th>S. no.</th> <th>Code</th> <th>Species</th> <th>Place of collection</th> <th>State</th> <th>Latitude</th> <th>Longitude</th> </tr> </thead> <tbody> <tr> <td>1</td> <td>V1</td> <td> <i>Ocimum tenuiflorum</i> var. Ram tulsi </td> <td>Venkateswarapuram</td> <td>Tamil Nadu</td> <td>N08°55′</td> <td>E77°34′</td> </tr> <tr> <td>2</td> <td>V2</td> <td> <i>Ocimum tenuiflorum</i> var. Krishna tulsi </td> <td>Rasipuram</td> <td>Tamil Nadu</td> <td>N11°28′</td> <td>E78°09′</td> </tr> <tr> <td>3</td> <td>V3</td> <td> <i>Ocimum basilicum</i> </td> <td>Hilsa</td> <td>Bihar</td> <td>N25°20′</td> <td>E85°17′</td> </tr> <tr> <td>4</td> <td>V4</td> <td> <i>Ocimum africanum</i> </td> <td>Kanyakumari</td> <td>Tamil Nadu</td> <td>N08°04′</td> <td>E77°32′</td> </tr> <tr> <td>5</td> <td>V5</td> <td> <i>Ocimum basilicum</i> var. purpurascens </td> <td>Vasudevanallur</td> <td>Tamil Nadu</td> <td>N09°14′</td> <td>E77°24′</td> </tr> <tr> <td>6</td> <td>V6</td> <td> <i>Ocimum americanum</i> </td> <td>Vellur</td> <td>Tamil Nadu</td> <td>N11°06′</td> <td>E78°00′</td> </tr> <tr> <td>7</td> <td>V7</td> <td> <i>Ocimum kilimandscharicum</i> </td> <td>Yercaud</td> <td>Tamil Nadu</td> <td>N11°46′</td> <td>E78°12′</td> </tr> <tr> <td>8</td> <td>V8</td> <td> <i>Ocimum gratissimum</i> </td> <td>Yercaud</td> <td>Tamil Nadu</td> <td>N11°46′</td> <td>E78°12′</td> </tr> <tr> <td>9</td> <td>V9</td> <td> <i>Ocimum adscendens</i> </td> <td>Rajapalayam</td> <td>Tamil Nadu</td> <td>N09°28′</td> <td>E77°33′</td> </tr> <tr> <td>10</td> <td>V10</td> <td> <i>Ocimum viride</i> </td> <td>Jammu Tawi</td> <td>Jammu &amp;amp; Kashmir</td> <td>N32°43′</td> <td>E74°50′</td> </tr> <tr> <td>11</td> <td>V11</td> <td> <i>Pogostemon cablin</i> </td> <td>Bangalore</td> <td>Karnataka</td> <td>N12°58′</td> <td>E77°35′</td> </tr> <tr> <td>12</td> <td>V12</td> <td> <i>Ocimum tenuiflorum</i> var. CIM‐Ayu </td> <td>Lucknow</td> <td>Uttar Pradesh</td> <td>N26°53′</td> <td>E80°58′</td> </tr> <tr> <td>13</td> <td>V13</td> <td> <i>Ocimum tenuiflorum</i> var. CIM‐Angana </td> <td>Lucknow</td> <td>Uttar Pradesh</td> <td>N26°53′</td> <td>E80°58′</td> </tr> <tr> <td>14</td> <td>V14</td> <td> <i>Ocimum basilicum</i> var. CIM‐Saumya </td> <td>Lucknow</td> <td>Uttar Pradesh</td> <td>N26°53′</td> <td>E80°58′</td> </tr> </tbody> </table> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">John Wiley &amp;amp; Sons, Ltd</span>, </span> </div> </div> </div> <div class="dnaisolation" title="sec"> <span class="label" tagx="label" title="label">2.2</span> <div class="title" tagx="title" title="title">
DNA isolation</div> <p>Fresh plant leaves were collected, and genomic DNA was isolated using the protocol described by Khanuja, Shasany, Darokar, and Kumar ( <a href="#ece32483-bib-0016">1999</a>). The purity and concentration of isolated genomic DNA were quantified spectrophotometrically (NanoDrop ND‐1000, Nanodrop Technologies, Wilmington, Delaware, USA) by measuring absorbance at 260 nm and 280 nm for the OD <sub>260</sub>/OD <sub>280</sub> ratio. Its integrity was checked by visualizing it under UV light after electrophoresis on 0.8% agarose gel. Stock DNA was diluted to make a required working solution of 25 ng/μl. </p> </div> <div class="issr‐pcramplification" title="sec"> <span class="label" tagx="label" title="label">2.3</span> <div class="title" tagx="title" title="title">
ISSR‐PCR amplification</div> <p>Twenty ISSR primers were procured from the University of British Columbia (UBC set No. 9). PCR reaction was prepared in a total volume of 25 μl and contained 1 × Taq DNA polymerase buffer (with MgCl <sub>2</sub>) (GeNei™), 50 ng genomic DNA, 1 unit of Taq DNA polymerase (GeNei™), 200 μmol/l of each dNTP (dATP: dTTP: dCTP: dGTP in 1:1:1:1 parts) (GeNei™), and 0.5 μmol/l primer. Amplifications were carried out using a thermal cycler (iCycler™ Thermal Cycler, Bio‐Rad, USA) using the following parameters: initial denaturation at 94°C for 5 min; 45 cycles of denaturation at 94°C for 1 min, annealing at 48–60°C (Table <a href="#ece32483-tbl-0002">2</a>) for 1 min, extension at 72°C for 2 min; final extension at 72°C for 7 min. Amplicons were electrophoresed on 1.2% agarose gel alongside a 2‐Log DNA ladder (New England Biolabs Inc.). Gel photographs were taken by means of Bio‐Rad Universal Hood II gel documentation system (Bio‐Rad laboratories Inc.). </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 2</span>  <p>Marker parameters calculated for each ISSR primer used with <i>Ocimum</i> spp </p>  <table> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <thead> <tr> <th>S. No.</th> <th>Primer</th> <th>Sequence (5′–3′)</th> <th>GC%</th> <th> <i>T</i> [°C] </th> <th>TB</th> <th>PB</th> <th>PPB (%)</th> <th>PIC</th> <th>RP</th> </tr> </thead> <tbody> <tr> <td>1</td> <td>UBC 807</td> <td>(AG) <sub>8</sub>T </td> <td>47</td> <td>52</td> <td>12</td> <td>12</td> <td>100</td> <td>0.374</td> <td>6.571</td> </tr> <tr> <td>2</td> <td>UBC 809</td> <td>(AG) <sub>8</sub>G </td> <td>53</td> <td>60</td> <td>14</td> <td>14</td> <td>100</td> <td>0.333</td> <td>6.571</td> </tr> <tr> <td>3</td> <td>UBC 810</td> <td>(GA) <sub>8</sub>T </td> <td>47</td> <td>52</td> <td>16</td> <td>16</td> <td>100</td> <td>0.325</td> <td>7.857</td> </tr> <tr> <td>4</td> <td>UBC 823</td> <td>(TC) <sub>8</sub>C </td> <td>53</td> <td>54</td> <td>14</td> <td>14</td> <td>100</td> <td>0.341</td> <td>7.142</td> </tr> <tr> <td>5</td> <td>UBC 825</td> <td>(AC) <sub>8</sub>T </td> <td>47</td> <td>60</td> <td>10</td> <td>10</td> <td>100</td> <td>0.307</td> <td>4.428</td> </tr> <tr> <td>6</td> <td>UBC 826</td> <td>(AC) <sub>8</sub>C </td> <td>53</td> <td>52.5</td> <td>6</td> <td>6</td> <td>100</td> <td>0.229</td> <td>1.857</td> </tr> <tr> <td>7</td> <td>UBC 828</td> <td>(TG) <sub>8</sub>A </td> <td>47</td> <td>52</td> <td>12</td> <td>12</td> <td>100</td> <td>0.363</td> <td>6.285</td> </tr> <tr> <td>8</td> <td>UBC 834</td> <td>(AG) <sub>8</sub>YT </td> <td>47</td> <td>52.5</td> <td>14</td> <td>14</td> <td>100</td> <td>0.382</td> <td>8.714</td> </tr> <tr> <td>9</td> <td>UBC 835</td> <td>(AG) <sub>8</sub>YC </td> <td>53</td> <td>52</td> <td>14</td> <td>14</td> <td>100</td> <td>0.365</td> <td>8.142</td> </tr> <tr> <td>10</td> <td>UBC 841</td> <td>(GA) <sub>8</sub>YC </td> <td>53</td> <td>52</td> <td>8</td> <td>8</td> <td>100</td> <td>0.395</td> <td>4.857</td> </tr> <tr> <td>11</td> <td>UBC 843</td> <td>(CT) <sub>8</sub>RA </td> <td>47</td> <td>53</td> <td>6</td> <td>6</td> <td>100</td> <td>0.299</td> <td>2.571</td> </tr> <tr> <td>12</td> <td>UBC 844</td> <td>(CT) <sub>8</sub>RC </td> <td>53</td> <td>54</td> <td>10</td> <td>10</td> <td>100</td> <td>0.357</td> <td>5.999</td> </tr> <tr> <td>13</td> <td>UBC 845</td> <td>(CT) <sub>8</sub>RG </td> <td>53</td> <td>53</td> <td>13</td> <td>13</td> <td>100</td> <td>0.342</td> <td>6.571</td> </tr> <tr> <td>14</td> <td>UBC 848</td> <td>(CA) <sub>8</sub>RG </td> <td>53</td> <td>52.5</td> <td>13</td> <td>13</td> <td>100</td> <td>0.368</td> <td>7.142</td> </tr> <tr> <td>15</td> <td>UBC 855</td> <td>(AC) <sub>8</sub>YT </td> <td>47</td> <td>53</td> <td>8</td> <td>8</td> <td>100</td> <td>0.376</td> <td>4.428</td> </tr> <tr> <td>16</td> <td>UBC 856</td> <td>(AC) <sub>8</sub>YA </td> <td>47</td> <td>53</td> <td>16</td> <td>16</td> <td>100</td> <td>0.343</td> <td>7.999</td> </tr> <tr> <td>17</td> <td>UBC 862</td> <td>(AGC) <sub>6</sub> </td> <td>67</td> <td>57.2</td> <td>13</td> <td>12</td> <td>92.30</td> <td>0.328</td> <td>6.285</td> </tr> <tr> <td>18</td> <td>UBC 866</td> <td>(CTC) <sub>6</sub> </td> <td>67</td> <td>57.2</td> <td>14</td> <td>14</td> <td>100</td> <td>0.338</td> <td>7.142</td> </tr> <tr> <td>19</td> <td>UBC 876</td> <td>(GATA) <sub>2</sub>(GACA) <sub>2</sub> </td> <td>38</td> <td>48</td> <td>13</td> <td>13</td> <td>100</td> <td>0.397</td> <td>8.285</td> </tr> <tr> <td>20</td> <td>UBC 881</td> <td>(GGGTG) <sub>3</sub> </td> <td>80</td> <td>58.7</td> <td>15</td> <td>15</td> <td>100</td> <td>0.327</td> <td>6.857</td> </tr> <tr> <td> <b>Total</b> </td> <td /> <td /> <td /> <td /> <td> <b>241</b> </td> <td> <b>240</b> </td> <td /> <td /> <td /> </tr> <tr> <td /> <td>Avg./Primer</td> <td /> <td /> <td /> <td> <b>12.05</b> </td> <td> <b>12.00</b> </td> <td> <b>99.585</b> </td> <td> <b>0.344</b> </td> <td> <b>6.285</b> </td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p> <i>T</i> [°C], annealing temperature; R, (A, G); Y, (C, T); TB, total band; PB, polymorphic band; PPB (%), percentage polymorphic band (%); PIC, polymorphic information content; RP, resolving power of primer. </p> </div> </div> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">John Wiley &amp;amp; Sons, Ltd</span>, </span> </div> </div> </div> <div class="pcramplificationandsequencingofpsba‐trnh(cpdna)" title="sec"> <span class="label" tagx="label" title="label">2.4</span> <div class="title" tagx="title" title="title">
PCR amplification and sequencing of <i>psbA‐trnH</i> (cpDNA) </div> <p>Noncoding <i>psbA‐trnH</i> region of cpDNA was amplified using universal primers fwd‐PA: 5′‐GTT ATG CAT GAA CGT AAT GCT C‐3′ and rev‐TH: 5′‐CGC GCA TGG TGG ATT CAC AAT CC‐3′ (Newmaster &amp;amp; Ragupathy, <a href="#ece32483-bib-0027">2009</a>; Zheng, Jiang, Wu, Wang, &amp;amp; Huang, <a href="#ece32483-bib-0054">2014</a>). PCR was run in a manner similar to that of ISSR‐PCR with the following cycling parameters: an initial denaturation at 94°C for 5 min; 30 cycles of denaturation at 94°C for 1 min, annealing at 55°C for 1 min, extension at 72°C for 90 s; final extension at 72°C for 7 min. PCR products were visualized on a 1.2% agarose gel and purified using QIAquick PCR purification kit (QIAGEN) prior to sequencing in both directions on an ABI 3130XL automated sequencer (Applied Biosystems) using both the primers. Sequences generated were deposited at the GenBank (NCBI) (Table <a href="#ece32483-tbl-0003">3</a>). </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 3</span>  <p>Characteristics of <i>Ocimum</i> spp. sequences (of noncoding <i>psbA‐trnH</i> region) deposited in GenBank (NCBI) </p>  <table> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <thead> <tr> <th>S. no.</th> <th>Species</th> <th>GenBank accession no.</th> <th>Length</th> <th>GC (%)</th> </tr> </thead> <tbody> <tr> <td>1</td> <td> <i>Ocimum tenuiflorum</i> var. Ram tulsi </td> <td>Forward</td> <td> <a href="KT338799.1">KT338799.1</a> </td> <td>328</td> <td>28.04</td> </tr> <tr> <td>Reverse</td> <td> <a href="KU666421.1">KU666421.1</a> </td> <td>372</td> <td>29.30</td> </tr> <tr> <td>2</td> <td> <i>Ocimum tenuiflorum</i> var. Krishna tulsi </td> <td>Forward</td> <td> <a href="KT338800.1">KT338800.1</a> </td> <td>328</td> <td>28.04</td> </tr> <tr> <td>Reverse</td> <td> <a href="KU666422.1">KU666422.1</a> </td> <td>321</td> <td>28.97</td> </tr> <tr> <td>3</td> <td> <i>Ocimum basilicum</i> </td> <td>Forward</td> <td> <a href="KT338801.1">KT338801.1</a> </td> <td>323</td> <td>27.55</td> </tr> <tr> <td>Reverse</td> <td> <a href="KU666423.1">KU666423.1</a> </td> <td>368</td> <td>29.61</td> </tr> <tr> <td>4</td> <td> <i>Ocimum africanum</i> </td> <td>Forward</td> <td> <a href="KT338802.1">KT338802.1</a> </td> <td>338</td> <td>28.69</td> </tr> <tr> <td>Reverse</td> <td> <a href="KU707910.1">KU707910.1</a> </td> <td>321</td> <td>26.79</td> </tr> <tr> <td>5</td> <td> <i>Ocimum basilicum</i> var. purpurascens </td> <td>Forward</td> <td> <a href="KT338803.1">KT338803.1</a> </td> <td>323</td> <td>27.24</td> </tr> <tr> <td>Reverse</td> <td> <a href="KX096039.1">KX096039.1</a> </td> <td>310</td> <td>30.32</td> </tr> <tr> <td>6</td> <td> <i>Ocimum americanum</i> </td> <td>Forward</td> <td> <a href="KT338804.1">KT338804.1</a> </td> <td>343</td> <td>28.86</td> </tr> <tr> <td>Reverse</td> <td> <a href="KU707911.1">KU707911.1</a> </td> <td>284</td> <td>26.40</td> </tr> <tr> <td>7</td> <td> <i>Ocimum kilimandscharicum</i> </td> <td>Forward</td> <td> <a href="KT338805.1">KT338805.1</a> </td> <td>324</td> <td>28.39</td> </tr> <tr> <td>Reverse</td> <td> <a href="KU707912.1">KU707912.1</a> </td> <td>292</td> <td>26.02</td> </tr> <tr> <td>8</td> <td> <i>Ocimum gratissimum</i> </td> <td>Forward</td> <td> <a href="KT338806.1">KT338806.1</a> </td> <td>320</td> <td>29.68</td> </tr> <tr> <td>Reverse</td> <td> <a href="KX013103.1">KX013103.1</a> </td> <td>368</td> <td>29.61</td> </tr> <tr> <td>9</td> <td> <i>Ocimum adscendens</i> </td> <td>Forward</td> <td> <a href="KT338807.1">KT338807.1</a> </td> <td>328</td> <td>28.65</td> </tr> <tr> <td>Reverse</td> <td> <a href="KX013104.1">KX013104.1</a> </td> <td>373</td> <td>29.22</td> </tr> <tr> <td>10</td> <td> <i>Ocimum viride</i> </td> <td>Forward</td> <td> <a href="KT338808.1">KT338808.1</a> </td> <td>320</td> <td>28.75</td> </tr> <tr> <td>Reverse</td> <td> <a href="KX013105.1">KX013105.1</a> </td> <td>358</td> <td>30.72</td> </tr> <tr> <td>11</td> <td> <i>Pogostemon cablin</i> </td> <td>Forward</td> <td> <a href="KT338809.1">KT338809.1</a> </td> <td>187</td> <td>29.41</td> </tr> <tr> <td>Reverse</td> <td> <a href="KX013106.1">KX013106.1</a> </td> <td>271</td> <td>30.62</td> </tr> <tr> <td>12</td> <td> <i>Ocimum tenuiflorum</i> var. CIM‐Ayu </td> <td>Forward</td> <td> <a href="KT338810.1">KT338810.1</a> </td> <td>328</td> <td>28.04</td> </tr> <tr> <td>Reverse</td> <td> <a href="KX096040.1">KX096040.1</a> </td> <td>373</td> <td>29.49</td> </tr> <tr> <td>13</td> <td> <i>Ocimum tenuiflorum</i> var. CIM‐Angana </td> <td>Forward</td> <td> <a href="KT338811.1">KT338811.1</a> </td> <td>328</td> <td>28.65</td> </tr> <tr> <td>Reverse</td> <td> <a href="KX096041.1">KX096041.1</a> </td> <td>372</td> <td>30.37</td> </tr> <tr> <td>14</td> <td> <i>Ocimum basilicum</i> var. CIM‐Saumya </td> <td>Forward</td> <td> <a href="KT338812.1">KT338812.1</a> </td> <td>343</td> <td>28.86</td> </tr> <tr> <td>Reverse</td> <td> <a href="KX096042.1">KX096042.1</a> </td> <td>344</td> <td>28.19</td> </tr> </tbody> </table> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">John Wiley &amp;amp; Sons, Ltd</span>, </span> </div> </div> </div> <div class="dataanalysis" title="sec"> <span class="label" tagx="label" title="label">2.5</span> <div class="title" tagx="title" title="title">
Data analysis</div> <p>Amplified ISSR products were manually scored for band presence (1) or absence (0) for each accession, and a binary data matrix was constructed. Primer banding characteristics such as the number of total bands (TB), the number of polymorphic bands (PB), and the percentage of polymorphic bands (PPB) were obtained. Polymorphic information content (PIC) analysis is useful for selection of the most appropriate marker for genetic mapping and phylogenetic analysis. It provides an estimate of the discriminating power of the marker based on the number of noticeable alleles and their distribution (Mishra et al., <a href="#ece32483-bib-0021">2015</a>; Powell et al., <a href="#ece32483-bib-0032">1996</a>). Resolving power (RP) of a marker is yet another parameter for checking the efficiency of discrimination potential of any marker system (Kumar et al., <a href="#ece32483-bib-0019">2014</a>; Prevost &amp;amp; Wilkinson, <a href="#ece32483-bib-0034">1999</a>). </p> <p>To check the genetic relatedness among the <i>Ocimum</i> spp., data matrix of ISSR markers was converted into genetic similarity matrix using Jaccard coefficient (Jaccard, <a href="#ece32483-bib-0014">1908</a>) in SPSS 17.0 (SPSS Inc.) and NTSYS‐PC 2.02j (Rohlf, <a href="#ece32483-bib-0038">1998</a>). Bootstrap analysis (Felsenstein, <a href="#ece32483-bib-0012">1985</a>) using Nei's ( <a href="#ece32483-bib-0026">1973</a>) genetic distance search with 1,000 replicates was performed to obtain the confidence of the neighbor‐joining tree using the software TREECON (Van De Peer &amp;amp; Wachter, <a href="#ece32483-bib-0048">1994</a>). Further, to highlight the resolving power of the ordination, principal component analysis (PCA) was performed based on similarity coefficient of data realized from ISSR average similarity indices using SPSS statistics 17.0 software (SPSS Inc.). </p> </div> <div class="sequencealignmentandphylogeneticanalysis" title="sec"> <span class="label" tagx="label" title="label">2.6</span> <div class="title" tagx="title" title="title">
Sequence alignment and phylogenetic analysis</div> <p>Forward and reverse electropherograms were superimposed, visually checked, and edited using the BioEdit software ( <a href="http://www.mbio.ncsu.edu/BioEdit/bioedit.html">http://www.mbio.ncsu.edu/BioEdit/bioedit.html</a>). Sequence identity was confirmed by comparison with sequences deposited in public databases, using the BlastN search algorithm ( <a href="http://www.ncbi.nlm.nih.gov/BlastN">http://www.ncbi.nlm.nih.gov/BlastN</a>). Multiple sequence alignment of <i>psbA‐trnH</i> region sequences was performed using clustalW. </p> <p>Neighbor‐joining (NJ) phylogenetic trees were constructed using Kimura 2‐parameter (K2P) model, and reliability of the trees was tested with the 1000 replicates of nonparametric bootstrap (Felsenstein, <a href="#ece32483-bib-0012">1985</a>) for estimating evolutionary distances using software TREECON (Van De Peer &amp;amp; Wachter, <a href="#ece32483-bib-0048">1994</a>). Molecular phylogeny trees were reconstructed for character support using maximum parsimony (MP) and maximum likelihood (ML) through phylogeny program PAUP*4.0b10 (Swofford, <a href="#ece32483-bib-0045">2002</a>). For MP, all character transformations were weighted equally, and gaps were treated as missing data. Tree Bisection and Reconnection (TBR) branch swapping option, addition sequence option "random," and MAXTREES set to auto increase, and 100 heuristic searches per replicate were applied. For ML, best‐fit nucleotide substitution model was tested to facilitate comparisons between 56 alternative models in PAUP*4.0b10. Heuristic search was conducted under the general time‐reversible (GTR) and gamma distributed with invariant sites (G + I) model with TBR branch swapping algorithm on a starting tree built under the parsimony criterion and bootstrapped with 1,000 replicates. Estimated substitution rate matrix: AC = 1.4792878; AG = 2.155157; AT = 0.54992304; CG = 1.2440845; CT = 1.0431192; GT = 1.000000. Gamma shape parameter = 0.678. Assumed proportion of invariable sites = 0.2897. MP and ML trees were visualized and saved using FigTree 1.4.2 (Rambaut, <a href="#ece32483-bib-0037">2014</a>). </p> </div> </div> <div class="results" title="sec"> <span class="label" tagx="label" title="label">3</span> <div class="title" tagx="title" title="title">
Results</div> <div class="performanceofissrmarkers" title="sec"> <span class="label" tagx="label" title="label">3.1</span> <div class="title" tagx="title" title="title">
Performance of ISSR markers</div> <p>The GC percentages of the primers used in this study were within the range 38%–80% (with eight primers having 47% GC content and another eight having 53% GC content). Eleven of the primers were 18 bp long, seven were 17 bp long, and one each was 16 and 15 bp long. All the used dinucleotide primers were anchored at their 3′‐end with one or two degenerate nucleotide(s), to increase the specificity and polymorphism. Twenty ISSR primers used in this study yielded 241 loci ranging from 0.2 to 1.5 kb, out of which 240 loci were polymorphic in nature representing 99.585% polymorphism. The number of loci reproduced by these arbitrary primers was found to range from 6 to 16 of different sizes with an average frequency of 12.05 loci per ISSR primer. The primers UBC 810 and UBC 856 that gave the maximum number of amplicons (16) were the most informative, while the primers UBC 826 and UBC 843 that gave the minimum number of products (6) were least informative. A total of 1,040 clear DNA fragments were scored from all <i>Ocimum</i> species and its varieties including outgroup taxa <i>P. cablin</i>. Two hundred and forty variable loci showed fragment frequency range of 0.071–0.928. Of these, 113 loci (47.08%) showed a frequency __0.25, whereas 99 (41.25%) of the loci showed a frequency range of ≥0.25 to ≤0.5, and 28 (11.67%) showed a frequency &amp;gt;0.5. The ISSR loci were 100% polymorphic at species level for each primer except UBC 862 (Table <a href="#ece32483-tbl-0002">2</a>). In this study, the highest PIC value of 0.397 was observed for primer UBC 876, and the lowest PIC value of 0.229 was observed for primer UBC 826, with an overall average PIC value of 0.344 per primer. The highest RP value was observed with the primer UBC 834 (8.714) and the lowest with the primer UBC 826 (1.857) with an average RP of 6.285 per primer. </p> </div> <div class="clusteranalysisbasedontheissr‐derivedgenotyping" title="sec"> <span class="label" tagx="label" title="label">3.2</span> <div class="title" tagx="title" title="title"> <i>Cluster analysis based on the ISSR</i>‐ <i>derived genotyping</i> </div> <p>Fingerprinting dataset composed of 241 ISSR loci with 1040 DNA fragments was selected for banding pattern similarity analysis based on Jaccard coefficient. The pairwise genetic distance based on the similarity matrix ranged from 0.098 ( <i>O. tenuiflorum</i> var. Ram tulsi and <i>O. tenuiflorum</i> var. CIM‐Ayu) to 0.888 ( <i>O. kilimandscharicum</i> and <i>O. gratissimum</i>) with an average pairwise genetic distance of 0.493. Genetic relationships were determined by neighbor‐joining tree based on the Nei's genetic distance obtained from the ISSR profiles and showed two major clusters with 51% bootstrap value, one of "Basilicum" group and other of "Sanctum" group, while <i>P. cablin</i> was placed as an outgroup as expected. "Gratissimum" group was bifurcated from "Basilicum" and "Sanctum" groups with 100% bootstrap value. "Basilicum" group consisted of <i>O. americanum</i>, <i> O. kilimandscharicum</i>, <i> O. africanum</i>, and <i>O. basilicum</i> with its varieties. "Sanctum" group consisted of <i>O. adscendens</i> and <i>O. tenuiflorum</i> with its varieties. The phylogram depicted in Figure <a href="#ece32483-fig-0002">2</a> clearly shows that <i>O. gratissimum</i> and <i>O. viride</i> diverge earlier than the "Basilicum" and "Sanctum" groups. Principal component analysis (PCA) derived on the basis of ISSR data is illustrated in Figure <a href="#ece32483-fig-0003">3</a>. The first two principal component axes accounted for 43.01% and 22.12% data variance, respectively, representing 65.17% of data variance cumulatively, which is sufficient to resolve the taxa into distinct groups. The third (9.13%) and fourth (6.72%) axes of the data variance did not significantly improve the resolution of the taxa. Groupings with the PCA analysis are in agreement with the NJ tree as revealed by the TREECON software. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <p>Neighbor‐joining bootstrap tree resulting from <span class="styled-content" tagx="styled-content" title="styled-content">ISSR</span> dataset. Bootstrap value less than 50% is omitted </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <p>Principal component analysis ( <span class="styled-content" tagx="styled-content" title="styled-content">PCA</span>) of <i>Ocimum</i> <span class="styled-content" tagx="styled-content" title="styled-content">ISSR</span> data. These cumulatively account for 65.17% (43.05% and 22.12%, respectively) of the data variance, the third axis (not shown) representing 9.13%; V1–V14 represent different plant genotypes as listed in Table <a href="#ece32483-tbl-0001">1</a> </p>   </div> </div> <div class="psba‐trnhsequencedataanalysis" title="sec"> <span class="label" tagx="label" title="label">3.3</span> <div class="title" tagx="title" title="title"> <i>psbA‐trnH</i> sequence data analysis </div> <p>Chloroplast <i>psbA‐trnH</i> intergenic spacers were successfully amplified in all <i>Ocimum</i> species and <i>P. cablin</i>. Forward sequences of <i>psbA‐trnH</i> region were observed in the range of 320 to 343 bp for <i>Ocimum</i> species and 187 bp for <i>P. cablin</i>. The average GC content in the sequences was low (28.49%). The aligned sequence length of <i>psbA‐trnH</i> was 357 bp, harbored 310 (86.83%) characters constant, 29 (8.12%) variable sites but parsimony‐uninformative and 18 (5.04%) parsimony‐informative sites. Nucleotidic divergence (p‐distance), including the outgroup, ranged between 0.00 and 0.155, with an average of 0.025. For <i>Ocimum</i> species, the maximum pairwise p‐distance of 0.039 was observed with <i>O. adscendens</i> and <i>O. basilicum</i> and with <i>O. adscendens</i> and <i>O. basilicum</i> var. purpurascens. </p> <p>Dataset of <i>psbA‐trnH</i> sequences was used to reconstruct NJ, MP, and ML trees for 14 genotypes comprising of eight <i>Ocimum</i> species (few of them represented by different varieties) and also including an outgroup <i>P. cablin</i>. The topology of all the NJ (Figure <a href="#ece32483-fig-0004">4</a>), MP (Figure <a href="#ece32483-fig-0005">5</a>), and ML (Figure <a href="#ece32483-fig-0006">6</a>) trees is almost congruent and is supported by bootstrap value of more than 50%. Topology of MP and ML tree constructed through the program PAUP*4.0b10 is similar with different bootstrap value. In the MP analysis, after bootstrapping the single most parsimonious tree was retained with a tree length of 54, a consistency index (CI) of 0.926, a retention index (RI) of 0.934, and a rescaled consistency index (RC) of 0.865. A single best tree with likelihood –ln L = 736.44021 was found in ML analyses. The trees show that "Gratissimum" group is sister to "Basilicum" group with support value of NJ &amp;lt;50%, MP 74%, and ML 54%. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <p>Neighbor‐joining bootstrap tree using Kimura 2‐parameter algorithm. Phylogram resulting from <i>psbA‐trnH</i> nucleotide sequence data </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 5</span> </a> </div>   <p>Equally weighted parsimony tree of <i>psbA‐trnH</i> sequence data for <i>Ocimum</i> species. Single consensus parsimonious trees (tree length = 54, <span class="styled-content" tagx="styled-content" title="styled-content">CI</span> = 0.926, <span class="styled-content" tagx="styled-content" title="styled-content">RI</span> = 0.934); values above branches are bootstrap value of 1,000 replicates; branch colors represent the support (violet = maximum; red = minimum) </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 6</span> </a> </div>   <p>Reconstruction of a phylogenetic tree from <i>psbA‐trnH</i> sequence dataset by maximum likelihood using <span class="styled-content" tagx="styled-content" title="styled-content">GTR</span> + G + I model of substitution. Bootstrap support values of 1,000 replicates are given above the branches. Branch colors represent the support (violet = maximum; red = minimum) </p>   </div> </div> </div> <div class="discussion" title="sec"> <span class="label" tagx="label" title="label">4</span> <div class="title" tagx="title" title="title">
Discussion</div> <p>Several groups have been working worldwide on <i>Ocimum</i> species targeting different aspects and using distinct tools. Some of these are as follows: taxonomical status and geographic distribution (Balyan &amp;amp; Pushpangadan, <a href="#ece32483-bib-0004">1988</a>); cytotaxonomical studies (Sobti &amp;amp; Pushpangadan, <a href="#ece32483-bib-0044">1979</a>); classification and relationships (Paton et al., <a href="#ece32483-bib-0031">1999</a>); genetic diversity and phylogenetic study using different molecular markers (Lal, Mistry, Thaker, Shah, &amp;amp; Vaidya, <a href="#ece32483-bib-0020">2012</a>; Patel, Fougat, Kumar, Mistry, &amp;amp; Kumar, <a href="#ece32483-bib-0029">2014</a>; Singh et al., <a href="#ece32483-bib-0043">2004</a>); chemical characterization (Carvoic‐Stanko et al., <a href="#ece32483-bib-0007">2011</a>; Verma et al., <a href="#ece32483-bib-0049">2016</a>; Vieira &amp;amp; Simon, <a href="#ece32483-bib-0051">2000</a>); compositional diversity in essential oils (Verma et al., <a href="#ece32483-bib-0050">2013</a>); genetic relations based on molecular markers, nuclear DNA content, and chromosome number (Carvoic‐Stanko et al., <a href="#ece32483-bib-0006">2010</a>); taxonomical distribution, medicinal properties, and drug development potentiality (Nahak, Mishra, &amp;amp; Sahu, <a href="#ece32483-bib-0025">2011</a>); nucleotide‐based validation by evaluating three candidate barcodes of the chloroplast region (Christina &amp;amp; Annamalai, <a href="#ece32483-bib-0010">2014</a>). However, no report is found on comparative phylogenetic study in <i>Ocimum</i> using PCR (ISSR)‐based and sequence (cpDNA)‐based analyses. To the best of our knowledge, this is the first report on the comparative phylogenetic study of <i>Ocimum</i> species through ISSR markers and plastid DNA region. The phylogenetic relationships of <i>Ocimum</i> species were inferred using distance (NJ)‐based approach applied on ISSR genomic data as well as both distance‐ and character‐based (MP and ML) methodological approaches applied on <i>psbA‐trnH</i> region sequence data. </p> <p>In the present study, 14 genotypes belonging to seven species of <i>Ocimum</i> (few of them represented by different varieties) and also including one outgroup <i>P. cablin</i> were studied using ISSR markers. The selection of ISSR markers was based on their relative technical simplicity, higher level of polymorphism detected, robustness, cost effectiveness (less time consuming and less expensive), easily adaptability to any plant species, and capability to target sequences that are abundant throughout the eukaryotic genome. The polymorphism level of 99.585% observed through ISSR analysis in the present study was in line with the earlier studies carried out by Patel et al. ( <a href="#ece32483-bib-0029">2014</a>), whereby ISSR analysis of five <i>Ocimum</i> species showed 98.17% polymorphism. Chen et al. ( <a href="#ece32483-bib-0008">2013</a>) found 97% polymorphism in four <i>Ocimum</i> species, and Lal et al. ( <a href="#ece32483-bib-0020">2012</a>) found 100% polymorphism in six species of <i>Ocimum</i>. High polymorphism of ISSR over other marker systems was also reported in many previous studies, such as in <i>Ocimum</i> (Chen et al., <a href="#ece32483-bib-0008">2013</a>; Lal et al., <a href="#ece32483-bib-0020">2012</a>), <i>Allium</i> (Mukherjee et al., <a href="#ece32483-bib-0022">2013</a>), and <i>Arachis</i> (Raina et al., <a href="#ece32483-bib-0036">2001</a>). The average PIC value of 0.344 was observed in this study, which was closer to 0.5, the maximum value of PIC for any dominant marker (Nagy et al., <a href="#ece32483-bib-0024">2012</a>). Although ISSR amplifies the regions between two microsatellites, the average PIC of ISSR marker systems was reasonably higher. The high PIC value obtained for the ISSRs reflects the efficiency of the marker to simultaneously analyze a large number of bands, rather than the level of polymorphism detected (Powell et al., <a href="#ece32483-bib-0032">1996</a>). High levels of polymorphism and PIC found in the present work showed that ISSR markers are suitable for differentiating the genus <i>Ocimum</i> and its closely related varieties. Prevost and Wilkinson ( <a href="#ece32483-bib-0034">1999</a>) reported that resolving power (RP) of a primer correlates strongly with its ability to distinguish between genotypes. In other words, RP provides quantitative data allowing comparisons between primers, including those that are able to distinguish all genotypes under examination. It can also be used to predict the performance of groups of primers. In the present study, average RP value of 6.285 proved the efficiency of ISSR markers to discriminate among the <i>Ocimum</i> species. </p> <p>ISSR analyses based on the distance‐based (NJ) method grouped the <i>Ocimum</i> spp. in three groups: "Basilicum," "Sanctum," and "Gratissimum" groups. This grouping is explained by the earlier studies carried out by Paton et al. ( <a href="#ece32483-bib-0031">1999</a>) based on morphological characters, which grouped <i>Ocimum</i> into sections and subsections. Sect. " <i>Ocimum</i>" subsect. " <i>Ocimum</i>" comprised of "Basilicum" group, and sect. " <i>Ocimum</i>" subsect. " <i>Gratissima</i>" comprised of "Gratissimum" group, while <i>O. tenuiflorum</i> is placed in section " <i>Hierocymum</i>" subsect. " <i>Foliosa</i>." Moreover, <i>O. gratissimum</i> (2 <i>n</i> = 40) and <i>O. tenuiflorum</i> (2 <i>n</i> = 36) have a smaller number of chromosomes compared to <i>O. basilicum</i> (2 <i>n</i> = 48) and <i>O. americanum</i> (2 <i>n</i> = 72) (Carvoic‐Stanko et al., <a href="#ece32483-bib-0007">2011</a>). Carvoic‐Stanko et al. ( <a href="#ece32483-bib-0006">2010</a>) also reported the same type of grouping based on molecular markers, nuclear DNA content, and chromosome number. Our studies clearly demonstrated that morphologically dissimilar species <i>O. gratissimum</i> and <i>O. viride</i> form a single group. "Gratissimum" group is bifurcated earlier with 100% support value from "Basilicum" and "Sanctum" group in ISSR–NJ tree, which is supported by the study carried out by Carvoic‐Stanko et al. ( <a href="#ece32483-bib-0007">2011</a>). According to the results presented, <i>O. kilimandscharicum</i> grouped together with <i>O. basilicum</i>, <i> O. africanum</i>, and <i>O. americanum</i>, which is again in agreement with the study done by Khosla ( <a href="#ece32483-bib-0017">1995</a>). Paton ( <a href="#ece32483-bib-0030">1992</a>) placed <i>O. kilimandscharicum, O. basilicum</i>, and <i>O. americanum</i> in section " <i>Ocimum</i>" subsection " <i>Ocimum</i>," which also backed the same grouping pattern. <i>O. kilimandscharicum</i> was found to group with <i>O. basilicum</i> in an earlier rapid amplification of polymorphic DNA (RAPD)‐based study also (Shasany, Shukla, &amp;amp; Khanuja, <a href="#ece32483-bib-0041">2007</a>). </p> <p>Multivariate approach was used to complement the information retrieved from the cluster analysis methods because it is more informative regarding distances among major groups (Taran, Zhang, Warkentin, Tullu, &amp;amp; Vanderberg, <a href="#ece32483-bib-0046">2005</a>). PCA analysis used in the present study provides complementary information to the cluster analysis, as it allows a graphical presentation of <i>Ocimum</i> species in a scatter plot. The PCA of <i>Ocimum</i> species and an outgroup <i>P. cablin</i> explained 74.3% of total variation by the first three principal components. Grouping in PCA based on ISSR marker dataset corresponded well to the ISSR–NJ phylogenetic tree. </p> <p>On the other hand, distance‐based (NJ) and character‐based (MP and ML) trees depicted through <i>psbA‐trnH</i> sequence dataset grouped the <i>Ocimum</i> species into two major clades. Clade I indicated that the "Gratissimum" group was sister to "Basilicum" group with reliable bootstrap values (NJ &amp;lt;50%, MP 74%, ML 54%). Clade II was comprised of <i>O. tenuiflorum</i> with its varieties and <i>O. adscendens</i>. The CI (0.926) and RI (0.934) of the <i>psbA‐trnH</i> consensus maximum‐parsimony tree are high, suggesting that the phylogenetic reconstruction is fairly reliable. Unlike the present grouping, many workers reported grouping of "Gratissimum" group with "Sanctum" group on the basis of morphological, cytological, and oil characters (Khosla, <a href="#ece32483-bib-0017">1995</a>; Sobti &amp;amp; Pushpangadan, <a href="#ece32483-bib-0044">1979</a>) or genetic similarity defined through RAPD (Singh et al., <a href="#ece32483-bib-0043">2004</a>). On the basis of pollen morphology, Harley, Paton, Harley, and Cade ( <a href="#ece32483-bib-0013">1992</a>) placed <i>O. gratissimum</i> and <i>O. tenuiflorum</i> in same group, while <i>O. basilicum</i>, <i> O. americanum</i>, and <i>O. kilimandscharicum</i> together in another group. On the other hand, <i>O. gratissimum</i> outgrouped in the <i>O. basilicum</i> cluster in a RAPD‐based analysis carried out by Shasany et al. ( <a href="#ece32483-bib-0041">2007</a>). </p> <p>By contrast, in all the phylogenetic trees, grouping of <i>Ocimum</i> spp. based on ISSR and <i>psbA–trnH</i> sequence dataset differs largely. Neighbor‐joining analyses based on the ISSRs dataset placed "Gratissimum" group away from both "Sanctum" and "Basilicum" groups, while the distance‐ and character‐based analyses on the <i>psbA‐trnH</i> sequence dataset placed "Gratissimum" group with "Basilicum" group. The best explanation for this grouping is that the ISSRs (biparentally inherited) amplified from both nuclear and organellar regions, while <i>psbA‐trnH</i> region amplified from cpDNA region (maternally inherited). The size of the nucleome in comparison with the plastome and the chondriome is large. Hence, the origin of the ISSR bands is likely to be nuclear and can serve as control for the groups found with the plastid region. Due to haploid and uniparental inheritance, cpDNA reveals only half of the parentage of plants with hybridization/introgression or polyploidy and may incorrectly categorize plants into a clade of one of the two parents. Therefore, phylogenetic trees based on cpDNA may be discordant (Treutlein, Smith, van Wyk, &amp;amp; Wink, <a href="#ece32483-bib-0047">2003</a>). The present report indicated the high efficiency of ISSRs markers in resolving the taxonomy of <i>Ocimum</i>. Interestingly, the earlier study by Christina and Annamalai ( <a href="#ece32483-bib-0010">2014</a>) proved that <i>psbA–trnH</i> region was a more suitable barcode as compared to the other tested cpDNA regions ( <i>matK</i>, <i> rbcL</i>) for <i>Ocimum</i> species identification, but not for differentiating their varieties. They also indicated that the study of deletions in the sequences could prove helpful in species identification. Moreover, study carried out by Bhamra, Heinrich, Howard, Johnson, and Slater ( <a href="#ece32483-bib-0005">2015</a>) in <i>Ocimum</i> illustrated <i>psbA‐trnH</i> region to be more efficient in species identification than nuclear ribosomal internal transcribed spacer ( <i>ITS</i>). </p> <p>ISSR technique is a PCR‐based method with advantages of high sensitivity to low levels of genetic variation, low‐cost, and high efficiency as compared with other DNA genotyping techniques, thus making it a very useful molecular tool for studying population genetics on a wide range of plant species, as well as for identifying species, or cultivars (Wang, Feng, Lu, Shi, &amp;amp; Liu, <a href="#ece32483-bib-0053">2009</a>). The present phylogeny study on <i>Ocimum</i> based on ISSR markers strongly supported genetic divergence and also correlated to some extent with the morphological level. </p> <p>To date, various molecular marker techniques have been developed and are used in phylogenetic investigations. Although ISSR markers are highly efficient and reproducible, yet they have not been used extensively in phylogenetic studies. Many phylogeny studies carried out through ISSRs as in case of <i>Vigna</i> (Ajibade, Weeden, &amp;amp; Chite, <a href="#ece32483-bib-0001">2000</a>), Asian‐cultivated rice <i>Oryza sativa</i> (Joshi, Gupta, Aggarwal, Ranjekar, &amp;amp; Brar, <a href="#ece32483-bib-0015">2000</a>), wheat (Nagaoka &amp;amp; Ogihara, <a href="#ece32483-bib-0023">1997</a>), and finger millet (Salimath, Oliveira, Godwin, &amp;amp; Bennetzen, <a href="#ece32483-bib-0040">1995</a>) have been found to be very effective. This study provided valuable information for the potential application of ISSR markers in phylogenetic investigation, and there is boundless scope to use this powerful technique in resolving intra‐ or interspecific status in many genera or in deciding the uniqueness of different genera within a family. Along with the vast applications of ISSR markers, unique alleles produced in the present study for <i>Ocimum</i> can be converted in codominant sequence characterized amplified regions (SCARs) to develop species‐specific diagnostic markers for a fast and effective diagnosis of this important medicinal material. </p> </div> <div class="conflictofinterest" title="sec"> <div class="title" tagx="title" title="title">
Conflict of Interest</div> <p>The authors declare that they have no competing interests.</p> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <span class="label" tagx="label" title="label">5</span> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>The authors wish to express their sincere thanks to Director, CSIR‐CIMAP, Lucknow, for providing the laboratory facilities required to carry out this research. The financial support from the Council of Scientific and Industrial Research (CSIR), New Delhi, through the XIIth FYP project ChemBio (BSC‐0203) is gratefully acknowledged. </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="ece32483-bib-0001" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Ajibade</span>, <span class="given-names" tagx="given-names" title="given-names">S. R.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Weeden</span>, <span class="given-names" tagx="given-names" title="given-names">N. F.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Chite</span>, <span class="given-names" tagx="given-names" title="given-names">S. M.</span> </span> ( <span class="year" tagx="year" title="year">2000</span>). <span class="mixed-article-title" title="mixed-article-title">Inter‐simple sequence repeat analysis of genetic relationships in the genus <i>Vigna</i> </span>. <span class="source" tagx="source" title="source">Euphytica</span>, <span class="volume" tagx="volume" title="volume">111</span>, <span class="fpage" tagx="fpage" title="fpage">47</span>– <span class="lpage" tagx="lpage" title="lpage">55</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0003" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Asthana</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Tripathi</span>, <span class="given-names" tagx="given-names" title="given-names">N. N.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Dixit</span>, <span class="given-names" tagx="given-names" title="given-names">S. N.</span> </span> ( <span class="year" tagx="year" title="year">1986</span>). <span class="mixed-article-title" title="mixed-article-title">Fungitoxic and phytotoxic studies with essential oil of <i>Ocimum adscendens</i> </span>. <span class="source" tagx="source" title="source">Journal of Phytopathology</span>, <span class="volume" tagx="volume" title="volume">117</span>, <span class="fpage" tagx="fpage" title="fpage">152</span>– <span class="lpage" tagx="lpage" title="lpage">159</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0004" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Balyan</span>, <span class="given-names" tagx="given-names" title="given-names">S. S.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Pushpangadan</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> ( <span class="year" tagx="year" title="year">1988</span>). <span class="mixed-article-title" title="mixed-article-title">A study on the taxonomical status and geographic distribution of genus <i>Ocimum</i> </span>. <span class="source" tagx="source" title="source">Perfumes and Flavours Association of India journal</span>, <span class="volume" tagx="volume" title="volume">2</span>, <span class="fpage" tagx="fpage" title="fpage">13</span>– <span class="lpage" tagx="lpage" title="lpage">19</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0005" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Bhamra</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Heinrich</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Howard</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Johnson</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Slater</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">DNA authentication of tulsi ( <i>Ocimum tenuiflorum</i>) using the nuclear ribosomal internal transcribed spacer ( <i>ITS</i>) and the chloroplast intergenic spacer <i>trnH‐psbA</i> </span>. In: (Proceedings) 63rd International Congress and Annual Meeting of the Society for Medicinal Plant and Natural Product Research (GA) (pp. 1498). Georg Thieme Verlag KG. </span> </li> <li tag="ref"> <a name="ece32483-bib-0006" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Carvoic‐Stanko</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Liber</span>, <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Besendorfer</span>, <span class="given-names" tagx="given-names" title="given-names">V.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Javornik</span>, <span class="given-names" tagx="given-names" title="given-names">B.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Bohanec</span>, <span class="given-names" tagx="given-names" title="given-names">B.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Kolak</span>, <span class="given-names" tagx="given-names" title="given-names">I.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Satovic</span>, <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Genetic relations among basil taxa ( <i>Ocimum</i> L.) based on molecular markers, nuclear DNA content, and chromosome number </span>. <span class="source" tagx="source" title="source">Plant Systematics and Evolution</span>, <span class="volume" tagx="volume" title="volume">285</span>, <span class="fpage" tagx="fpage" title="fpage">13</span>– <span class="lpage" tagx="lpage" title="lpage">22</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0007" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Carvoic‐Stanko</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Liber</span>, <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Politeo</span>, <span class="given-names" tagx="given-names" title="given-names">O.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Strikic</span>, <span class="given-names" tagx="given-names" title="given-names">F.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Kolak</span>, <span class="given-names" tagx="given-names" title="given-names">I.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Milos</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Satovic</span>, <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Molecular and chemical characterization of the most widespread <i>Ocimum</i> species </span>. <span class="source" tagx="source" title="source">Plant Systematics and Evolution</span>, <span class="volume" tagx="volume" title="volume">294</span>, <span class="fpage" tagx="fpage" title="fpage">253</span>– <span class="lpage" tagx="lpage" title="lpage">262</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0008" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Chen</span>, <span class="given-names" tagx="given-names" title="given-names">S. Y.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Dai</span>, <span class="given-names" tagx="given-names" title="given-names">T. X.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Chang</span>, <span class="given-names" tagx="given-names" title="given-names">Y. T.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Wang</span>, <span class="given-names" tagx="given-names" title="given-names">S. S.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Ou</span>, <span class="given-names" tagx="given-names" title="given-names">S. L.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Chuang</span>, <span class="given-names" tagx="given-names" title="given-names">W. L.</span> </span>, … <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Ku</span>, <span class="given-names" tagx="given-names" title="given-names">H. M.</span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Genetic diversity among <i>Ocimum</i> species based on ISSR, RAPD and SRAP markers </span>. <span class="source" tagx="source" title="source">Australian Journal of Crop Science</span>, <span class="volume" tagx="volume" title="volume">7</span>, <span class="fpage" tagx="fpage" title="fpage">1463</span>– <span class="lpage" tagx="lpage" title="lpage">1471</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0009" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Chowdhury</span>, <span class="given-names" tagx="given-names" title="given-names">M. A.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Vandenberg</span>, <span class="given-names" tagx="given-names" title="given-names">B.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Warkentin</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">Cultivar identification and genetic relationship among selected breeding lines and cultivars in chickpea ( <i>Cicer arietinum</i> L.) </span>. <span class="source" tagx="source" title="source">Euphytica</span>, <span class="volume" tagx="volume" title="volume">127</span>, <span class="fpage" tagx="fpage" title="fpage">317</span>– <span class="lpage" tagx="lpage" title="lpage">325</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0010" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Christina</span>, <span class="given-names" tagx="given-names" title="given-names">V. L. P.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Annamalai</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Nucleotide based validation of <i>Ocimum</i> species by evaluating three candidate barcodes of the chloroplast region </span>. <span class="source" tagx="source" title="source">Molecular Ecology Resources</span>, <span class="volume" tagx="volume" title="volume">14</span>, <span class="fpage" tagx="fpage" title="fpage">60</span>– <span class="lpage" tagx="lpage" title="lpage">68</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24164957">24164957</a> </span> </span> </li> <li tag="ref"> <a name="ece32483-bib-0011" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Dong</span>, <span class="given-names" tagx="given-names" title="given-names">Z. Z.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Fan</span>, <span class="given-names" tagx="given-names" title="given-names">X.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Sha</span>, <span class="given-names" tagx="given-names" title="given-names">L. N.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Zeng</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Wang</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Chen</span>, <span class="given-names" tagx="given-names" title="given-names">Q.</span> </span>, … <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Zhou</span>, <span class="given-names" tagx="given-names" title="given-names">Y. H.</span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Phylogeny and molecular evolution of the <i>rbcL</i> gene of St genome in <i>Elymus sensu</i> lato (Poaceae: Triticeae) </span>. <span class="source" tagx="source" title="source">Biochemical Systematics and Ecology</span>, <span class="volume" tagx="volume" title="volume">50</span>, <span class="fpage" tagx="fpage" title="fpage">322</span>– <span class="lpage" tagx="lpage" title="lpage">330</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0012" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Felsenstein</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> ( <span class="year" tagx="year" title="year">1985</span>). <span class="mixed-article-title" title="mixed-article-title">Confidence limits on phylogenies: An approach using the bootstrap</span>. <span class="source" tagx="source" title="source">Evolution</span>, <span class="volume" tagx="volume" title="volume">39</span>, <span class="fpage" tagx="fpage" title="fpage">783</span>– <span class="lpage" tagx="lpage" title="lpage">791</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28561359">28561359</a> </span> </span> </li> <li tag="ref"> <a name="ece32483-bib-0013" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Harley</span>, <span class="given-names" tagx="given-names" title="given-names">M. M.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Paton</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Harley</span>, <span class="given-names" tagx="given-names" title="given-names">R. M.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Cade</span>, <span class="given-names" tagx="given-names" title="given-names">P. G.</span> </span> ( <span class="year" tagx="year" title="year">1992</span>). <span class="mixed-article-title" title="mixed-article-title">Pollen morphological studies in tribe Ocimeae (Nepetoideae: Labiatae): I. <i>Ocimum</i> L. <i>Grana</i> </span>, <span class="volume" tagx="volume" title="volume">31</span>, <span class="fpage" tagx="fpage" title="fpage">161</span>– <span class="lpage" tagx="lpage" title="lpage">176</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0014" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Jaccard</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> ( <span class="year" tagx="year" title="year">1908</span>). <span class="mixed-article-title" title="mixed-article-title">Nouvelle recherches sur La distribution florale</span>. <span class="source" tagx="source" title="source">Bulletin de la Société vaudoise des sciences naturelles</span>, <span class="volume" tagx="volume" title="volume">44</span>, <span class="fpage" tagx="fpage" title="fpage">223</span>– <span class="lpage" tagx="lpage" title="lpage">270</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0015" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Joshi</span>, <span class="given-names" tagx="given-names" title="given-names">S. P.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Gupta</span>, <span class="given-names" tagx="given-names" title="given-names">V. S.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Aggarwal</span>, <span class="given-names" tagx="given-names" title="given-names">R. K.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Ranjekar</span>, <span class="given-names" tagx="given-names" title="given-names">P. K.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Brar</span>, <span class="given-names" tagx="given-names" title="given-names">D. S.</span> </span> ( <span class="year" tagx="year" title="year">2000</span>). <span class="mixed-article-title" title="mixed-article-title">Genetic diversity and phylogenetic relationship as revealed by inter‐simple sequence repeat (ISSR) polymorphism in the genus <i>Oryza</i> </span>. <span class="source" tagx="source" title="source">Theoretical and Applied Genetics</span>, <span class="volume" tagx="volume" title="volume">100</span>, <span class="fpage" tagx="fpage" title="fpage">1311</span>– <span class="lpage" tagx="lpage" title="lpage">1320</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0016" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Khanuja</span>, <span class="given-names" tagx="given-names" title="given-names">S. P. S.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Shasany</span>, <span class="given-names" tagx="given-names" title="given-names">A. K.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Darokar</span>, <span class="given-names" tagx="given-names" title="given-names">M. P.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Kumar</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> ( <span class="year" tagx="year" title="year">1999</span>). <span class="mixed-article-title" title="mixed-article-title">Rapid isolation of DNA from dry and fresh samples of plants producing large amounts of secondary metabolites and essential oils </span>. <span class="source" tagx="source" title="source">Plant Molecular Biology Reporter</span>, <span class="volume" tagx="volume" title="volume">17</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">7</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0017" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Khosla</span>, <span class="given-names" tagx="given-names" title="given-names">M. K.</span> </span> ( <span class="year" tagx="year" title="year">1995</span>). <span class="mixed-article-title" title="mixed-article-title">Study of inter‐relationship, phylogeny and evolutionary tendencies in genus <i>Ocimum</i> </span>. <span class="source" tagx="source" title="source">The Indian Journal of Genetics and Plant Breeding</span>, <span class="volume" tagx="volume" title="volume">55</span>, <span class="fpage" tagx="fpage" title="fpage">71</span>– <span class="lpage" tagx="lpage" title="lpage">83</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0018" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Kimura</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> ( <span class="year" tagx="year" title="year">1983</span>). <span class="source" tagx="source" title="source">The neutral theory of molecular evolution</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Cambridge, UK</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Cambridge University Press</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0019" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Kumar</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Mishra</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Singh</span>, <span class="given-names" tagx="given-names" title="given-names">S. C.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Sundaresan</span>, <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Efficiency of ISSR and RAPD markers in genetic divergence analysis and conservation management of <i>Justicia adhatoda</i> L., a medicinal plant </span>. <span class="source" tagx="source" title="source">Plant Systematics and Evolution</span>, <span class="volume" tagx="volume" title="volume">300</span>, <span class="fpage" tagx="fpage" title="fpage">1409</span>– <span class="lpage" tagx="lpage" title="lpage">1420</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0020" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Lal</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Mistry</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Thaker</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Shah</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Vaidya</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Genetic diversity assessment in six medicinally important species of <i>Ocimum</i> from central Gujarat (India) utilizing RAPD, ISSR and SSR markers </span>. <span class="source" tagx="source" title="source">International Journal of Advanced Biological and Biomedical Research</span>, <span class="volume" tagx="volume" title="volume">2</span>, <span class="fpage" tagx="fpage" title="fpage">279</span>– <span class="lpage" tagx="lpage" title="lpage">288</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0021" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Mishra</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Kumar</span>, <span class="given-names" tagx="given-names" title="given-names">L. D.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Kumar</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Gokul</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Ravikumar</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Shukla</span>, <span class="given-names" tagx="given-names" title="given-names">A. K.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Sundaresan</span>, <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Population dynamics and conservation implications of <i>Decalepis arayalpathra</i> (J. Joseph and V. Chandras.) Venter., a steno endemic species of Western Ghats, India </span>. <span class="source" tagx="source" title="source">Applied Biochemistry and Biotechnology</span>, <span class="volume" tagx="volume" title="volume">176</span>, <span class="fpage" tagx="fpage" title="fpage">1413</span>– <span class="lpage" tagx="lpage" title="lpage">1430</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25969156">25969156</a> </span> </span> </li> <li tag="ref"> <a name="ece32483-bib-0022" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Mukherjee</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Sikdar</span>, <span class="given-names" tagx="given-names" title="given-names">B.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Ghosh</span>, <span class="given-names" tagx="given-names" title="given-names">B.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Banerjee</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Ghosh</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Bhattacharya</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Roy</span>, <span class="given-names" tagx="given-names" title="given-names">S. C.</span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">RAPD and ISSR analysis of some economically important species, varieties, and cultivars of the genus <i>Allium</i> (Alliaceae) </span>. <span class="source" tagx="source" title="source">Turkish Journal of Botany</span>, <span class="volume" tagx="volume" title="volume">37</span>, <span class="fpage" tagx="fpage" title="fpage">605</span>– <span class="lpage" tagx="lpage" title="lpage">618</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0023" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Nagaoka</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Ogihara</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> ( <span class="year" tagx="year" title="year">1997</span>). <span class="mixed-article-title" title="mixed-article-title">Applicability of inter‐simple sequence repeat polymorphisms in wheat for use as DNA markers in comparison to RFLP and RAPD markers </span>. <span class="source" tagx="source" title="source">Theoretical and Applied Genetics</span>, <span class="volume" tagx="volume" title="volume">94</span>, <span class="fpage" tagx="fpage" title="fpage">597</span>– <span class="lpage" tagx="lpage" title="lpage">602</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0024" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Nagy</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Poczai</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Cernak</span>, <span class="given-names" tagx="given-names" title="given-names">I.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Gorji</span>, <span class="given-names" tagx="given-names" title="given-names">A. M.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Hegedus</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Taller</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">PICcalc: An online program to calculate polymorphic information content for molecular genetic studies </span>. <span class="source" tagx="source" title="source">Biochemical Genetics</span>, <span class="volume" tagx="volume" title="volume">50</span>, <span class="fpage" tagx="fpage" title="fpage">670</span>– <span class="lpage" tagx="lpage" title="lpage">672</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22573137">22573137</a> </span> </span> </li> <li tag="ref"> <a name="ece32483-bib-0025" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Nahak</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Mishra</span>, <span class="given-names" tagx="given-names" title="given-names">R. C.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Sahu</span>, <span class="given-names" tagx="given-names" title="given-names">R. K.</span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Taxonomic distribution, medicinal properties and drug development potentiality of <i>Ocimum</i> (Tulsi) </span>. <span class="source" tagx="source" title="source">Drug Invention Today</span>, <span class="volume" tagx="volume" title="volume">3</span>, <span class="fpage" tagx="fpage" title="fpage">95</span>– <span class="lpage" tagx="lpage" title="lpage">113</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0026" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Nei</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> ( <span class="year" tagx="year" title="year">1973</span>). <span class="mixed-article-title" title="mixed-article-title">Analysis of gene diversity in subdivided populations</span>. <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences</span>, <span class="volume" tagx="volume" title="volume">70</span>, <span class="fpage" tagx="fpage" title="fpage">3321</span>– <span class="lpage" tagx="lpage" title="lpage">3323</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0027" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Newmaster</span>, <span class="given-names" tagx="given-names" title="given-names">S. G.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Ragupathy</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Testing plant barcoding in a sister species complex of pantropical <i>Acacia</i> (Mimosoideae, Fabaceae) </span>. <span class="source" tagx="source" title="source">Molecular Ecology Resources</span>, <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">172</span>– <span class="lpage" tagx="lpage" title="lpage">180</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21564976">21564976</a> </span> </span> </li> <li tag="ref"> <a name="ece32483-bib-0028" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Padalia</span>, <span class="given-names" tagx="given-names" title="given-names">R. C.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Verma</span>, <span class="given-names" tagx="given-names" title="given-names">R. S.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Chauhan</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Chanotiya</span>, <span class="given-names" tagx="given-names" title="given-names">C. S.</span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Changes in aroma profiles of 11 Indian <i>Ocimum</i> taxa during plant ontogeny </span>. <span class="source" tagx="source" title="source">Acta Physiologiae Plantarum</span>, <span class="volume" tagx="volume" title="volume">35</span>, <span class="fpage" tagx="fpage" title="fpage">2567</span>– <span class="lpage" tagx="lpage" title="lpage">2587</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0029" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Patel</span>, <span class="given-names" tagx="given-names" title="given-names">H. K.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Fougat</span>, <span class="given-names" tagx="given-names" title="given-names">R. S.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Kumar</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Mistry</span>, <span class="given-names" tagx="given-names" title="given-names">J. G.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Kumar</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Detection of genetic variation in <i>Ocimum</i> species using RAPD and ISSR markers </span>. <span class="source" tagx="source" title="source">3 Biotech</span>, <span class="volume" tagx="volume" title="volume">5</span>, <span class="fpage" tagx="fpage" title="fpage">697</span>– <span class="lpage" tagx="lpage" title="lpage">707</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0030" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Paton</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> ( <span class="year" tagx="year" title="year">1992</span>). <span class="mixed-article-title" title="mixed-article-title">A synopsis of <i>Ocimum</i> L. (Labiatae) in Africa </span>. <span class="source" tagx="source" title="source">Kew Bulletin</span>, <span class="volume" tagx="volume" title="volume">47</span>, <span class="fpage" tagx="fpage" title="fpage">405</span>– <span class="lpage" tagx="lpage" title="lpage">437</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0031" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Paton</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Harley</span>, <span class="given-names" tagx="given-names" title="given-names">M. R.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Harley</span>, <span class="given-names" tagx="given-names" title="given-names">M. M.</span> </span> ( <span class="year" tagx="year" title="year">1999</span>). <span class="chapter-title" tagx="chapter-title" title="chapter-title"> <i>Ocimum</i>: An overview of classification and relationships </span> In <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hiltunen</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span>, &amp;amp; <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Holm</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> </span> (Eds.), <span class="source" tagx="source" title="source">Basil: The genus Ocimum</span> (pp. <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">38</span>). <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Amsterdam</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Harwood</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0032" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Powell</span>, <span class="given-names" tagx="given-names" title="given-names">W.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Morgante</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Andre</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Hanafey</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Vogel</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Tingey</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Rafalski</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> ( <span class="year" tagx="year" title="year">1996</span>). <span class="mixed-article-title" title="mixed-article-title">A comparison of RFLP, RAPD, AFLP and SSR markers for germplasm analysis</span>. <span class="source" tagx="source" title="source">Molecular Breeding</span>, <span class="volume" tagx="volume" title="volume">2</span>, <span class="fpage" tagx="fpage" title="fpage">225</span>– <span class="lpage" tagx="lpage" title="lpage">238</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0033" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Prakash</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Gupta</span>, <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> ( <span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Therapeutic uses of <i>Ocimum sanctum</i> Linn. (Tulsi) with a note on eugenol and its pharmacological actions: A short review </span>. <span class="source" tagx="source" title="source">Indian Journal of Physiology and Pharmacology</span>, <span class="volume" tagx="volume" title="volume">49</span>, <span class="fpage" tagx="fpage" title="fpage">125</span>– <span class="lpage" tagx="lpage" title="lpage">131</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16170979">16170979</a> </span> </span> </li> <li tag="ref"> <a name="ece32483-bib-0034" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Prevost</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Wilkinson</span>, <span class="given-names" tagx="given-names" title="given-names">M. J.</span> </span> ( <span class="year" tagx="year" title="year">1999</span>). <span class="mixed-article-title" title="mixed-article-title">A new system of comparing PCR primers applied to ISSR fingerprinting of potato cultivars</span>. <span class="source" tagx="source" title="source">Theoretical and Applied Genetics</span>, <span class="volume" tagx="volume" title="volume">98</span>, <span class="fpage" tagx="fpage" title="fpage">107</span>– <span class="lpage" tagx="lpage" title="lpage">112</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0035" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Pushpangadan</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Bradu</span>, <span class="given-names" tagx="given-names" title="given-names">B. L.</span> </span> ( <span class="year" tagx="year" title="year">1995</span>). <span class="chapter-title" tagx="chapter-title" title="chapter-title">Medicinal and aromatic plants</span> In <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chadha</span> <span class="given-names" tagx="given-names" title="given-names">K. L.</span> </span> </span>, &amp;amp; <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gupta</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> (Eds.), <span class="source" tagx="source" title="source">Advances in horticulture</span>, Vol. <span class="volume" tagx="volume" title="volume">11</span> (pp. <span class="fpage" tagx="fpage" title="fpage">627</span>– <span class="lpage" tagx="lpage" title="lpage">657</span>). <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New Delhi</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Malhotra Publishing House</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0036" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Raina</span>, <span class="given-names" tagx="given-names" title="given-names">S. N.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Rani</span>, <span class="given-names" tagx="given-names" title="given-names">V.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Kojima</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Ogihara</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Singh</span>, <span class="given-names" tagx="given-names" title="given-names">K. P.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Devarumath</span>, <span class="given-names" tagx="given-names" title="given-names">R. M.</span> </span> ( <span class="year" tagx="year" title="year">2001</span>). <span class="mixed-article-title" title="mixed-article-title">RAPD and ISSR fingerprints as useful genetic markers for analysis of genetic diversity, varietal identification, and phylogenetic relationships in peanut ( <i>Arachis hypogaea</i>) cultivars and wild species </span>. <span class="source" tagx="source" title="source">Genome</span>, <span class="volume" tagx="volume" title="volume">44</span>, <span class="fpage" tagx="fpage" title="fpage">763</span>– <span class="lpage" tagx="lpage" title="lpage">777</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11681599">11681599</a> </span> </span> </li> <li tag="ref"> <a name="ece32483-bib-0037" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Rambaut</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="source" tagx="source" title="source">FigTree</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Edinburgh</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Institute of Evolutionary Biology, Univ Edinburgh</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0038" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Rohlf</span>, <span class="given-names" tagx="given-names" title="given-names">F. J.</span> </span> ( <span class="year" tagx="year" title="year">1998</span>). <span class="source" tagx="source" title="source">NTSYS‐PC: Numerical taxonomy and multivariate analysis system. Version 2.02</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Setauket, New York</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Exeter software</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0040" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Salimath</span>, <span class="given-names" tagx="given-names" title="given-names">S. S.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Oliveira</span>, <span class="given-names" tagx="given-names" title="given-names">A. C.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Godwin</span>, <span class="given-names" tagx="given-names" title="given-names">I. D.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Bennetzen</span>, <span class="given-names" tagx="given-names" title="given-names">J. L.</span> </span> ( <span class="year" tagx="year" title="year">1995</span>). <span class="mixed-article-title" title="mixed-article-title">Assessment of genome origins and genetic diversity in the genus <i>Eleusine</i> with DNA markers </span>. <span class="source" tagx="source" title="source">Genome</span>, <span class="volume" tagx="volume" title="volume">38</span>, <span class="fpage" tagx="fpage" title="fpage">757</span>– <span class="lpage" tagx="lpage" title="lpage">763</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7672607">7672607</a> </span> </span> </li> <li tag="ref"> <a name="ece32483-bib-0041" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Shasany</span>, <span class="given-names" tagx="given-names" title="given-names">A. K.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Shukla</span>, <span class="given-names" tagx="given-names" title="given-names">A. K.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Khanuja</span>, <span class="given-names" tagx="given-names" title="given-names">S. P. S.</span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="chapter-title" tagx="chapter-title" title="chapter-title">Medicinal and aromatic plants</span> In <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kole</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> </span> (Ed.), <span class="source" tagx="source" title="source">Genome mapping and molecular breeding in plants</span>, Vol. <span class="volume" tagx="volume" title="volume">6</span> (pp. <span class="fpage" tagx="fpage" title="fpage">175</span>– <span class="lpage" tagx="lpage" title="lpage">196</span>)., <span class="series" tagx="series" title="series">Technical Crops</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Berlin Heidelberg</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer‐Verlag</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0042" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Shukla</span>, <span class="given-names" tagx="given-names" title="given-names">A. K.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Mall</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Rai</span>, <span class="given-names" tagx="given-names" title="given-names">S. K.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Singh</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Nair</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Parashar</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span> </span>, … <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Khanuja</span>, <span class="given-names" tagx="given-names" title="given-names">S. P. S.</span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">A transcriptomic approach for exploring the molecular basis for <i>dosha</i>‐balancing property‐based classification of plants in Ayurveda </span>. <span class="source" tagx="source" title="source">Molecular Biology Reports</span>, <span class="volume" tagx="volume" title="volume">40</span>, <span class="fpage" tagx="fpage" title="fpage">3255</span>– <span class="lpage" tagx="lpage" title="lpage">3262</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23322031">23322031</a> </span> </span> </li> <li tag="ref"> <a name="ece32483-bib-0043" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Singh</span>, <span class="given-names" tagx="given-names" title="given-names">A. P.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Dwivedi</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Bharti</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Srivastava</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Singh</span>, <span class="given-names" tagx="given-names" title="given-names">V.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Khanuja</span>, <span class="given-names" tagx="given-names" title="given-names">S. P. S.</span> </span> ( <span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Phylogenetic relationships as in <i>Ocimum</i> revealed by RAPD markers </span>. <span class="source" tagx="source" title="source">Euphytica</span>, <span class="volume" tagx="volume" title="volume">136</span>, <span class="fpage" tagx="fpage" title="fpage">11</span>– <span class="lpage" tagx="lpage" title="lpage">20</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0044" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Sobti</span>, <span class="given-names" tagx="given-names" title="given-names">S. N.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Pushpangadan</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> ( <span class="year" tagx="year" title="year">1979</span>). <span class="chapter-title" tagx="chapter-title" title="chapter-title">Cytotaxonomical studies in the genus <i>Ocimum</i> </span> In <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bir</span> <span class="given-names" tagx="given-names" title="given-names">S. S.</span> </span> </span> (Ed.), <span class="source" tagx="source" title="source">Taxonomy, cytogenetics and cytotaxonomy of plants</span> (pp. <span class="fpage" tagx="fpage" title="fpage">373</span>– <span class="lpage" tagx="lpage" title="lpage">377</span>). <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New Delhi</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Kalyani</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0045" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Swofford</span>, <span class="given-names" tagx="given-names" title="given-names">D. L.</span> </span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="source" tagx="source" title="source">PAUP—phylogenetic analysis using parsimony</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Sunderland, MA</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Sinauer Associates</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0046" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Taran</span>, <span class="given-names" tagx="given-names" title="given-names">B.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Zhang</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Warkentin</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Tullu</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Vanderberg</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> ( <span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Genetic diversity among varieties and wild species accessions of pea ( <i>Pisum sativum</i> L.) based on molecular markers, and morphological and physiological characters </span>. <span class="source" tagx="source" title="source">Genome</span>, <span class="volume" tagx="volume" title="volume">48</span>, <span class="fpage" tagx="fpage" title="fpage">257</span>– <span class="lpage" tagx="lpage" title="lpage">272</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15838548">15838548</a> </span> </span> </li> <li tag="ref"> <a name="ece32483-bib-0047" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Treutlein</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Smith</span>, <span class="given-names" tagx="given-names" title="given-names">G. F.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">van Wyk</span>, <span class="given-names" tagx="given-names" title="given-names">B. E.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Wink</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> ( <span class="year" tagx="year" title="year">2003</span>). <span class="mixed-article-title" title="mixed-article-title">Phylogenetic relationships in Asphodelaceae (subfamily Alooideae) inferred from chloroplast DNA sequences ( <i>rbcL</i>, <i>matK</i>) and from genomic fingerprinting (ISSR) </span>. <span class="source" tagx="source" title="source">Taxon</span>, <span class="volume" tagx="volume" title="volume">52</span>, <span class="fpage" tagx="fpage" title="fpage">193</span>– <span class="lpage" tagx="lpage" title="lpage">207</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0048" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Van De Peer</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Wachter</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> ( <span class="year" tagx="year" title="year">1994</span>). <span class="mixed-article-title" title="mixed-article-title">TREECON for windows: A software package for the construction and drawing of evolutionary trees for the Microsoft Windows environment </span>. <span class="source" tagx="source" title="source">Computer Applications in the Biosciences</span>, <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">569</span>– <span class="lpage" tagx="lpage" title="lpage">570</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7828077">7828077</a> </span> </span> </li> <li tag="ref"> <a name="ece32483-bib-0049" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Verma</span>, <span class="given-names" tagx="given-names" title="given-names">R. S.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Kumar</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Mishra</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Baskaran</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Padalia</span>, <span class="given-names" tagx="given-names" title="given-names">R. C.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Sundaresan</span>, <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Essential oil composition of four <i>Ocimum</i> spp. from the Peninsular India </span>. <span class="source" tagx="source" title="source">Journal of Essential Oil Research</span>, <span class="volume" tagx="volume" title="volume">28</span>, <span class="fpage" tagx="fpage" title="fpage">35</span>– <span class="lpage" tagx="lpage" title="lpage">41</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0050" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Verma</span>, <span class="given-names" tagx="given-names" title="given-names">R. S.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Padalia</span>, <span class="given-names" tagx="given-names" title="given-names">R. C.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Chauhan</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Thul</span>, <span class="given-names" tagx="given-names" title="given-names">S. T.</span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Exploring compositional diversity in the essential oils of 34 <i>Ocimum</i> taxa from Indian flora </span>. <span class="source" tagx="source" title="source">Industrial Crops and Products</span>, <span class="volume" tagx="volume" title="volume">45</span>, <span class="fpage" tagx="fpage" title="fpage">7</span>– <span class="lpage" tagx="lpage" title="lpage">19</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0051" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Vieira</span>, <span class="given-names" tagx="given-names" title="given-names">R. F.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Simon</span>, <span class="given-names" tagx="given-names" title="given-names">J. E.</span> </span> ( <span class="year" tagx="year" title="year">2000</span>). <span class="mixed-article-title" title="mixed-article-title">Chemical characterization of basil ( <i>Ocimum</i> spp.) found in the markets and used in traditional medicine in Brazil </span>. <span class="source" tagx="source" title="source">Economic Botany</span>, <span class="volume" tagx="volume" title="volume">54</span>, <span class="fpage" tagx="fpage" title="fpage">207</span>– <span class="lpage" tagx="lpage" title="lpage">216</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0052" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Vina</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Murillo</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> ( <span class="year" tagx="year" title="year">2003</span>). <span class="mixed-article-title" title="mixed-article-title">Essential oil composition from twelve varieties of basil ( <i>Ocimum</i> spp.) grown in Colombia </span>. <span class="source" tagx="source" title="source">Journal of the Brazilian Chemical Society</span>, <span class="volume" tagx="volume" title="volume">14</span>, <span class="fpage" tagx="fpage" title="fpage">744</span>– <span class="lpage" tagx="lpage" title="lpage">749</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0053" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Wang</span>, <span class="given-names" tagx="given-names" title="given-names">H. Z.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Feng</span>, <span class="given-names" tagx="given-names" title="given-names">S. G.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Lu</span>, <span class="given-names" tagx="given-names" title="given-names">J. J.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Shi</span>, <span class="given-names" tagx="given-names" title="given-names">N. N.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Liu</span>, <span class="given-names" tagx="given-names" title="given-names">J. J.</span> </span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Phylogenetic study and molecular identification of 31 <i>Dendrobium</i> species using inter‐simple sequence repeat (ISSR) markers </span>. <span class="source" tagx="source" title="source">Scientia Horticulturae</span>, <span class="volume" tagx="volume" title="volume">122</span>, <span class="fpage" tagx="fpage" title="fpage">440</span>– <span class="lpage" tagx="lpage" title="lpage">447</span>. </span> </li> <li tag="ref"> <a name="ece32483-bib-0054" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Zheng</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Jiang</span>, <span class="given-names" tagx="given-names" title="given-names">X.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Wu</span>, <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Wang</span>, <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Huang</span>, <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Chemical and genetic discrimination of <i>Cistanches Herba</i> based on UPLC‐QTOF/MS and DNA barcoding </span>. <span class="source" tagx="source" title="source">PLoS ONE</span>, <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">e98061</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24854031">24854031</a> </span> </span> </li> <li tag="ref"> <a name="ece32483-bib-0055" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Zietkiewicz</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>, <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Rafalski</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Labuda</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> ( <span class="year" tagx="year" title="year">1994</span>). <span class="mixed-article-title" title="mixed-article-title">Genome fingerprinting by simple sequence repeat (SSR)‐anchored polymerase chain reaction amplification </span>. <span class="source" tagx="source" title="source">Genomics</span>, <span class="volume" tagx="volume" title="volume">20</span>, <span class="fpage" tagx="fpage" title="fpage">176</span>– <span class="lpage" tagx="lpage" title="lpage">183</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8020964">8020964</a> </span> </span> </li> <li tag="ref"> <a name="ece32483-bib-0056" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Zurawski</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span> </span>, &amp;amp; <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Clegg</span>, <span class="given-names" tagx="given-names" title="given-names">M. T.</span> </span> ( <span class="year" tagx="year" title="year">1987</span>). <span class="mixed-article-title" title="mixed-article-title">Evolution of higher plant chloroplast DNA encoded genes: Implications for structure–function and phylogenetic studies </span>. <span class="source" tagx="source" title="source">Annual Review of Plant Physiology</span>, <span class="volume" tagx="volume" title="volume">38</span>, <span class="fpage" tagx="fpage" title="fpage">391</span>– <span class="lpage" tagx="lpage" title="lpage">418</span>. </span> </li> </ul> </div> </div>  </body></html>