<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Bioinformation</span> <span class="publisher-id" title="publisher-id">Bioinformation</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Bioinformation</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">0973-2063</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Biomedical Informatics</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/3218416">3218416</a> </span> <span class="publisher-id" title="publisher-id">97320630007222</span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Hypothesis</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Computer aided screening and evaluation of herbal therapeutics against MRSA infections</div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Skariyachan</span> <span class="given-names" tagx="given-names" title="given-names">Sinosh</span> </span> <a href="#A1">1</a> <a href="#COR1">*</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Krishnan</span> <span class="given-names" tagx="given-names" title="given-names">Rao Shruti</span> </span> <a href="#A1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Siddapa</span> <span class="given-names" tagx="given-names" title="given-names">Snehapriya Bangalore</span> </span> <a href="#A1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salian</span> <span class="given-names" tagx="given-names" title="given-names">Chithra</span> </span> <a href="#A1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bora</span> <span class="given-names" tagx="given-names" title="given-names">Prerana</span> </span> <a href="#A1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sebastian</span> <span class="given-names" tagx="given-names" title="given-names">Denoj</span> </span> <a href="#A2">2</a> </span> <span class="citation_author_institution" id="A1">[1], </span> <span class="citation_author_institution" id="A2">[2], </span> </div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"> <span class="label" tagx="label" title="label">*</span>Sinosh Skariyachan: <span class="email" tagx="email" title="email">sinoshskariya@gmail.com;</span> </div> </div> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2011</span> </span> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2011-10-10</span> </span> <span class="volume" tagx="volume" title="volume">7</span> <span class="issue" tagx="issue" title="issue">5</span> <span class="fpage" tagx="fpage" title="fpage">222</span> <span class="lpage" tagx="lpage" title="lpage">233</span> <span class="history" title="history"> <span class="received" title="received">received: 2011-10-10</span> <span class="accepted" title="accepted">accepted: 2011-10-11</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2011</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open-access article, which permits unrestricted use, distribution, and reproduction in any medium, for non-commercial purposes, provided the original author and source are credited. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Methicillin resistant <i>Staphylococcus aureus</i> (MRSA), a pathogenic bacterium that causes life threatening outbreaks such as community-onset and nosocomial infections has emerged as ‘superbug’. The organism developed resistance to all classes of antibiotics including the best known Vancomycin (VRSA). Hence, there is a need to develop new therapeutic agents. This study mainly evaluates the potential use of botanicals against MRSA infections. Computer aided design is an initial platform to screen novel inhibitors and the data finds applications in drug development. The drug-likeness and efficiency of various herbal compounds were screened by ADMET and docking studies. The virulent factor of most of the MRSA associated infections are Penicillin Binding Protein 2A (PBP2A) and Panton-Valentine Leukocidin (PVL). Hence, native structures of these proteins (PDB: 1VQQ and 1T5R) were used as the drug targets. The docking studies revealed that the active component of <i>Aloe vera</i>, β-sitosterol (3S, 8S, 9S, 10R, 13R, 14S, 17R) ­17­ [(2R, 5R)-5-ethyl-6-methylheptan-2-yl] -10, 13-dimethyl 2, 3, 4, 7, 8, 9, 11, 12, 14, 15, 16, 17- dodecahydro-1H-cyclopenta [a] phenanthren-3-ol) showed best binding energies of -7.40 kcal/mol and ­6.34 kcal/mol for PBP2A and PVL toxin, respectively. Similarly, Meliantriol (1S-1-[ (2R, 3R, 5R)-5-hydroxy-3-[(3S, 5R, 9R, 10R, 13S, 14S, 17S)-3-hydroxy 4, 4, 10, 13, 14-pentamethyl-2, 3, 5, 6, 9, 11, 12, 15, 16, 17-decahydro-1H-cyclopenta[a] phenanthren-17-yl] oxolan-2-yl] -2- methylpropane-1, 2 diol), active compound in <i>Azadirachta indica</i> (Neem) showed the binding energies of ­6.02 kcal/mol for PBP2A and ­8.94 for PVL toxin. Similar studies were conducted with selected herbal compound based on pharmacokinetic properties. All in silico data tested <i>in vitro</i> concluded that herbal extracts of <i>Aloe-vera</i>, Neem, Guava ( <i>Psidium guajava</i>), Pomegranate ( <i>Punica granatum </i>) and tea ( <i>Camellia sinensis</i>) can be used as therapeutics against MRSA infections. </p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
Superbug</div> <div class="kwd" title="kwd">
MRSA</div> <div class="kwd" title="kwd">
PBP2A</div> <div class="kwd" title="kwd">
PVL toxin</div> <div class="kwd" title="kwd">
docking</div> <div class="kwd" title="kwd">
β-sitosterol</div> <div class="kwd" title="kwd">
Meliantriol</div> </div> </div> </div> <div class="body" title="body"> <div class="background:" title="sec"> <div class="title" tagx="title" title="title">
Background:</div> <p>Methicillin resistant <i>Staphylococcus aureus</i> (MRSA) is a notorious pathogenic bacterium causing many infections and the disease control has become a serious issue worldwide [ <a href="#R01">1</a>]. The high outbreak of MRSA was observed in closed communities such as schools, prisons and sports teams and the disease has mainly transmitted from fomite to person and from person to person and so on [ <a href="#R02">2</a>]. The pathogenecity of the bacteria includes skin and soft tissue infections (SSTI), bone, joint and implant infections, pneumonia, septicemia and various toxicoses such as toxic shock syndrome, bloodstream infections, osteomyelitis, septic arthritis, and device-related infections, necrotizing fasciitis and purpura fulminans and abscesses [ <a href="#R03">3</a>]. Recent reports indicated the emergence of multidrug resistant Staphylococci against all classes of β-lactam antibiotics. The antibiotic resistance is mainly due to the expression of PC1 β- lactamase and the acquisition of mecA gene encoding a penicillin-binding protein, PBP2A [ <a href="#R04">4</a>]. The bacteria initially penetrate the host's immune system via epidermal and mucosal epithelia and the antimicrobial peptides play necessary role in the host's innate immune defense against the initial colonization of bacteria [ <a href="#R05">5</a>]. It has been studied that many strains of MRSA contain genes that encode the toxin called Panton Valentine Leukocidin (PVL), bi-component leukocidal toxins (synergohymenotropic toxins) consists of F and S components. PVL is encoded by the lukPVoperon encoding the LukF-PV and LukS-PV components (cytotoxins) that lyse leukocyte. Hence, PVL positive Staphylococcal infections exist as life threatening infections of soft tissues and bones [ <a href="#R06">6</a>]. </p> <p>Recent studies revealed that strains of MRSA developed resistance to conventional antibiotics and emerged as multidrug resistant superbugs. Hence, there is a significant demand of finding better therapeutic agents. The active substances present in many medicinal plants could be used as therapeutic alternatives for MRSA infection [ <a href="#R07">7</a>]. Computer aided method is a preliminary approach to screening novel therapeutic agents and the discipline is an emerging strategy as it reduces many complexities of drug discovery process. The screening of lead molecule with good pharmacological properties and drug likeness is a tedious task in drug development process. Computer aided method is an easy platform to search such kinds of biologically active compounds with favorable ADMET (Absorption, Distribution, Metabolism, Excretion and Toxicity) and drug-likeness properties [ <a href="#R08">8</a>]. The study of receptor-ligand interaction is a fundamental concept of rational drug design and the prediction of such interactions by computational docking has increasing importance in the field of structure based drug discovery [ <a href="#R09">9</a>]. </p> <p>The study mainly focused to screen potential herbal inhibitors against the drug targets of MRSA by computer based ADMET and docking studies. The data obtained from <i>in silico</i> studies were later used for <i>in vitro</i> studies. The study also provides a comparative account of the efficiency of herbal compounds to that of known antibiotics. </p> </div> <div class="methodology:" title="sec"> <div class="title" tagx="title" title="title">
Methodology:</div> <div class="identificationofpotentialligands:" title="sec"> <div class="title" tagx="title" title="title"> <i>Identification of potential ligands</i>: </div> <p>Since MRSA has emerged as a multidrug resistant organism the design of alternative therapies are very crucial to treat MRSA outbreaks. There are reports that many herbal based compounds have high medicinal values against many pathogens. Hence, an initial survey of 74 compounds from 13 different medicinal plants was carried out by extensive literature studies. It is possible to find out the druggish properties of herbal compounds by computer aided screening and the data will be useful to screen best lead molecules. The 3D structures of all plant based ligands are available in drug data base and the structures were retrieved from PubChem [ <a href="#R10">10</a>], KEGG [ <a href="#R11">11</a>] and Chemspider [ <a href="#R12">12</a>]. </p> </div> <div class="admetstudiesandscreeningofligands:" title="sec"> <div class="title" tagx="title" title="title"> <i>ADMET studies and screening of ligands</i>: </div> <p>The ligands with good pharmacological and druggish properties are very crucial for structure based drug discovery. The absorption, distribution, metabolism, excretion and toxicity (ADMET) are the most important part of pharmacological studies of lead molecules and these can be predicted by computational biology tools. Hence, all 74 ligands identified were tested for their drug-likeliness, ADME profile and toxicity analysis by Pre-ADMET. The ADME includes the extent and rate of absorption, distribution, metabolism and excretion. Absorption checks how the substance is entering to the blood circulation, distribution explains how substances are disseminating throughout the fluids and tissues of the body, metabolism is the irreversible transformation of parent compounds into daughter metabolites and excretion explains how the substances are excreted from the body after metabolism. Pre-ADMET uses Caco2-cell (heterogeneous human epithelial colorectal adenocarcinoma cell lines) and MDCK (Madin-Darby Canine Kidney) cell models for oral drug absorption prediction and skin permeability, and human intestinal absorption model for oral and trans-dermal drug absorption prediction. Similarly, the programme uses BBB (blood brain barrier) penetration and plasma protein binding models to predict the distribution. Pre-ADMET predicts toxicity based on the mutagenicity of AMES parameters and rodent carcinogenicity assays of rat and mouse [ <a href="#R13">13</a>]. </p> </div> <div class="selectionoftargetproteinsandmoleculardocking:" title="sec"> <div class="title" tagx="title" title="title"> <i>Selection of target proteins and molecular docking</i>: </div> <p>The receptor and ligand interaction is the key mechanism of computer aided drug design. Previous studies reported that Penicillin Binding Protein 2A (PBP2A) and Panton-Valentine Leukocidin (PVL) are the major virulent factors of MRSA infections [ <a href="#R14">14</a>, <a href="#R15">15</a>]. Hence, the native structures of the proteins are used as drug targets. The crystal structures of PBP2A (1VQQ) and PVL toxin (1T5R) were retrieved from Protein Data Bank and the coordinate files were used for the study. The selected herbal compounds were docked with target proteins by AutoDock. The program uses a Monte Carlo simulated annealing for configurational exploration using grid based molecular affinity potentials and provides bioactive conformation by energy minimization [ <a href="#R16">16</a>]. </p> </div> <div class="invitrostudies" title="sec"> <div class="title" tagx="title" title="title"> <i>In vitro</i>studies </div> <p>The data generated by computer aided studies should be tested experimentally for further confirmation. Based on the pharmacological properties studied by ADMET &amp;amp; docking studies, five botanicals were identified for <i>In vitro</i> testing. The botanicals used in the study are <i>Azadirachta indica</i> [ <a href="#R17">17</a>], <i>Aloe vera</i> [ <a href="#R18">18</a>], <i>Camellia sinensis</i> [ <a href="#R19">19</a>], <i>Punica granatum</i> [ <a href="#R20">20</a>] and <i>Psidium guajava </i> [ <a href="#R21">21</a>]. The herbals were collected from Horticulture Centre-Hulimavu, Bangalore. The leaf extracts were prepared by standard solvent extraction method and the antimicrobial activity of each extract was tested in different concentrations. The pure culture of MRSA was swabbed on Muller Hinton agar plates which have many wells. The extracts were added in different concentration to each well (well diffusion method) and the plates were incubated at 37°C for 24 hours. The antimicrobial activity was determined by measuring the zone of inhibition. The efficiency of herbal extracts was also compared with known antibiotics. The antibiotic susceptibility patterns were carried out by disc diffusion method. The sensitivity patterns of each antibiotic were confirmed by measuring the zone of inhibition around the disc and compared with standard antibiotic susceptibility chart. </p> </div> </div> <div class="discussion:" title="sec"> <div class="title" tagx="title" title="title">
Discussion:</div> <p>The solution for MRSA associated outbreaks are critical problem because the organism recently emerged as VRSA. Hence, the discovery of an alternative treatment has profound scope and significance. Drug discovery process is time consuming and multistep process which includes many preclinical and clinical trials. One of the major reasons for drug failures is the poor drug-likeness and pharmacokinetic properties of lead compounds. Computer aided method is a rapid and significant screening approach because it selects the lead molecules with good pharmacological and druggish properties. We have selected 74 active compounds from herbal origin such as leaf extracts of <i>Azadirachta indica</i> (Neem), <i>Psidium guajava </i> (Guavas), <i>Aloe vera, Ocimum tenuiflorum</i> (Tulsi), <i>Curcuma longa </i> (Turmeric), <i>Allium sativum</i> (Garlic), <i>Punica granatum</i> (Pomegranate), <i>Eucalyptus, Camellia sinensis</i> (Tea) and the oil of Syzygium aromaticum (Clove). The 3D structures of the ligands were retrieved from drug databases and most of lead molecules qualified the properties of drugs. However, few ligands were found to be better as per Lipinski's rule of 5, CMC-like rule and MDDR-like rule ( <a href="#SD1">Table 1</a>, see supplementary material). </p> <p>Another important concern in drug discovery is the pharmacokinetics properties. The ADME profiles of selected drug candidates are shown in <a href="#SD1">Table 2</a> (see supplementary material). <i>In silico</i> toxicity prediction is the final step of pharmacokinetic screening of lead molecules and 30% of drug failures are due to toxicity issues. In this study, we have found that the predicted toxicities of many herbal compounds were suitable for further studies. Meliantriol from <i>Azadirachta indica</i>, β-sitosterol from <i>Alo vera</i>, ursolic acid and lupeol from <i>Ocimum tenuiflorum </i>, brevifolin and ellagic acid from <i>Punica granatum</i>, corilagin from <i>Camellia sinensis</i>, and many more showed better pharmacokinetic and drug-likeness properties ( <a href="#SD1">Table 3</a>, see supplementary material). Hence, these lead molecules were identified as the best ligands for docking studies. All these herbal extracts were reported to have high medicinal potential against many pathogenic bacteria and we have tested efficiency of the same against MRSA by computer aided approach and <i>in vitro </i> studies. </p> <p>The key proteins involved in MRSA infections are PBP2A and PVL toxins. Hence, these proteins have been selected as probable drug targets. The crystal structure of Penicillin Binding Protein 2A (PDBID: 1VQQ) was retrieved from the PDB. The protein consists of two chains and three penicillin binding protein domains. Chain A is the key domain consists of 646 amino acids and constitutes 33% helices (31 helices; 214 residues) and 26% beta sheet (34 strands; 169 residues) which are important for the functional activity. The crystal structure of Panton-Valentine Leucocidin (PDBID: 1T5R) was also retrieved from PDB and it has 8 chains. All chains have 284 amino acid residues and constitute 3% helices (3 helices; 11 residues) and 59% beta sheet (22 strands; 168 residues). The interactions between selected inhibitors and target proteins were studied by docking simulations. Among antibiotics, Vancomycin, best drug currently available against MRSA, showed the best binding energy value of ­8.91 kcal/mol. However, recent reports indicate that the organism developed resistance against Vancomycin and emerged as Vancomycin resistant <i>Staphlococcus areus</i> (VRSA). The binding efficiencies of herbal active compounds also tested against the target proteins by molecular docking. The docking scores and binding energies of all selected ligands with both the targets were shown in <a href="#SD1">Table 4</a> (see supplementary material). As the best, Meliantriol, active inhibitor present in Neem, showed binding energies of ­6.02 kcal/mol with PBP2A and ­8.94 kcal/mol with PVL toxin. Similarly, β-sitosterol, an antimicrobial agent present in medicinal aloe, showed binding energies of ­7.40 kcal/mol and ­6.34 kcal/mol for PBP2A and PVL toxin respectively ( <a href="#F1">Figure 1</a>). The docking studies reveled that herbal compounds are better inhibitors than chemical drugs; Meliantriol and similar compounds are the potential lead molecules than Vancomycin. As per computer aided studies plant derived inhibitors have high therapeutic value and it should be evaluated through <i>in vitro </i> studies. </p> <p>Thus, the study clearly indicates that herbal compounds have good therapeutic value against MRSA than antibiotics. </p> <p>Since the data generated by computer aided method has just a hypothetical solution it should be tested <i>in vitro</i> also. We have studied and compared the efficiency of antibiotics and selected herbal extracts (The active compounds of those extracts found to be effective by docking studies) against pure culture of MRSA. The method employed was disc diffusion and well diffusion techniques. Our study revealed that the organism resistant to wide range of antibiotics such as Moxifloxacin, Cefatoxime, Rifampicin, Oxacillin, Co-Trimoxazole, Chloramphenicol Ciprofloxacin, Ceftazidime, Nalidixic acid, Tetracycline, Streptomycin, Erythromycin, Ampicillin, Norfloxacin and Methicillin. The organism is susceptible to only few antibiotics, Vancomycin is the best one. Moreover, recent studies on Vancomycin resistant <i>Staphylococcus areus</i> (VRSA) indicated that antibiotic treatments are challenging and pointing out ‘the end of antibiotic era’ is very near in future. The susceptibility pattern of various antibiotics against MRSA is shown in <a href="#SD1">Table 5</a> (see supplementary material). Nonetheless, interestingly, as confirmed by the computer aided approach, the leaf extracts of Neem, Tea, Guava and Pomegranate showed better antimicrobial properties at a concentration of 100 µl, 100 µl, 80 µl and 200 µl respectively, which implies the potential of herbal extract against MRSA ( <a href="#F2">Figure 2</a>). The computer aided method is useful to screen best ligands from wide range of compounds which will reduce the complexities of drug discovery process. Subsequently, the study needs more clinical testing using laboratory animals and further <i>in vivo</i> studies should be conducted. The study would provide all initial inputs to design novel therapeutics against MRSA. </p> </div> <div class="conclusion:" title="sec"> <div class="title" tagx="title" title="title">
Conclusion:</div> <p>Present studies revealed that the mortality rate of MRSA infection is very high and the organism is resistant to Vancomycin (VRSA), the best chemotherapeutic agent available against MRSA. This study concluded that computer aided screening is an effective alternative for identification of novel remedies when all antibiotics seems to have failed. Several natural herbal compounds are screened and their effectiveness against MRSA is compared with known antibiotics. Computer aided ADME and toxicity prediction favored to screen best ligands with good pharmacological activities. It was found that herbal compounds such as Melantriol, β- Sitosterol and similar compounds have good inhibitory activity against Penicillin Binding Protein 2A and Panton-Valentine Leukocidin toxin of MRSA, key virulent factors of MRSA infections. The <i>in silico</i> data have been tested <i>in vitro</i> also, as suggested by computer aided method, herbal extracts of neem, medicinal aloe, tea, pomegranate and guava were identified as potential therapeutics against MRSA. The study concluded that, though the organism emerged as a multidrug resistant "superbug" there is high scope and application of herbal remedies towards MRSA infections. </p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary material</div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Data 1</div>  <div class="media" title="media"> <a href="97320630007222S1.pdf">LINK</a> </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>The authors thankfully acknowledge the R &amp;amp; D Centre of Life Sciences and Engineering, Dayananda Sagar Institutions for providing all necessary facilities and also grateful to Dr. P.S Rao, Vice President (R &amp;amp; D), Life Sciences and Engineering and Dr. G S Jagannatha Rao, Senior Professor and Head, Department of Biotechnology, Dayananda Sagar College of Engineering for their constant support and encouragement throughout the study. </p> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-other" title="other"> <p> <b>Citation:</b>Skariyachan <i>et al</i>, Bioinformation 7(5): 222-233 (2011) </p> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="R01" /> <span class="label" tagx="label" title="label">1</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">MJ</span> <span class="given-names" tagx="given-names" title="given-names">Kuehnert</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="source" tagx="source" title="source">Emerg Infect Dis</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">868</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15963281">15963281</a> </span> </span> </li> <li tag="ref"> <a name="R02" /> <span class="label" tagx="label" title="label">2</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">MZ</span> <span class="given-names" tagx="given-names" title="given-names">David</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="source" tagx="source" title="source">PLoS One</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">e18217</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21533238">21533238</a> </span> </span> </li> <li tag="ref"> <a name="R03" /> <span class="label" tagx="label" title="label">3</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">S</span> <span class="given-names" tagx="given-names" title="given-names">Monecke</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="source" tagx="source" title="source">PLoS One</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">e17936</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21494333">21494333</a> </span> </span> </li> <li tag="ref"> <a name="R04" /> <span class="label" tagx="label" title="label">4</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">LI</span> <span class="given-names" tagx="given-names" title="given-names">Llarrull </span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="source" tagx="source" title="source">Antimicrob Agents Chemother</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">53</span> <span class="fpage" tagx="fpage" title="fpage">4051</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19470504">19470504</a> </span> </span> </li> <li tag="ref"> <a name="R05" /> <span class="label" tagx="label" title="label">5</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">K</span> <span class="given-names" tagx="given-names" title="given-names">Ouhara</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="source" tagx="source" title="source">J Antimicrob Chemother</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">61</span> <span class="fpage" tagx="fpage" title="fpage">1266</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18367458">18367458</a> </span> </span> </li> <li tag="ref"> <a name="R06" /> <span class="label" tagx="label" title="label">6</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">BA</span> <span class="given-names" tagx="given-names" title="given-names">Diep</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">M</span> <span class="given-names" tagx="given-names" title="given-names">Otto</span> </span> </span> <span class="source" tagx="source" title="source">Trends Microbiol</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">361</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18585915">18585915</a> </span> </span> </li> <li tag="ref"> <a name="R07" /> <span class="label" tagx="label" title="label">7</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">MT</span> <span class="given-names" tagx="given-names" title="given-names">Chomnawang</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="source" tagx="source" title="source">Fitoterapia</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">80</span> <span class="fpage" tagx="fpage" title="fpage">102</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19022354">19022354</a> </span> </span> </li> <li tag="ref"> <a name="R08" /> <span class="label" tagx="label" title="label">8</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">V</span> <span class="given-names" tagx="given-names" title="given-names">Khanna</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">S</span> <span class="given-names" tagx="given-names" title="given-names">Ranganathan</span> </span> </span> <span class="source" tagx="source" title="source">BMC Bioinformatics</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">S10</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19958509">19958509</a> </span> </span> </li> <li tag="ref"> <a name="R09" /> <span class="label" tagx="label" title="label">9</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">S</span> <span class="given-names" tagx="given-names" title="given-names">Lyskov</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">JJ</span> <span class="given-names" tagx="given-names" title="given-names">Gray</span> </span> </span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">36</span> <span class="fpage" tagx="fpage" title="fpage">W233</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18442991">18442991</a> </span> </span> </li> <li tag="ref"> <a name="R10" /> <span class="label" tagx="label" title="label">10</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Y</span> <span class="given-names" tagx="given-names" title="given-names">Wang</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">38</span> <span class="fpage" tagx="fpage" title="fpage">D255</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19933261">19933261</a> </span> </span> </li> <li tag="ref"> <a name="R11" /> <span class="label" tagx="label" title="label">11</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">M</span> <span class="given-names" tagx="given-names" title="given-names">Kanehisa</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">38</span> <span class="fpage" tagx="fpage" title="fpage">D355</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19880382">19880382</a> </span> </span> </li> <li tag="ref"> <a name="R12" /> <span class="label" tagx="label" title="label">12</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">J</span> <span class="given-names" tagx="given-names" title="given-names">Anthony</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="source" tagx="source" title="source">J Cheminform</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="fpage" tagx="fpage" title="fpage">O16</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2867124">2867124</a> </span> </span> </li> <li tag="ref"> <a name="R13" /> <span class="label" tagx="label" title="label">13</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">A</span> <span class="given-names" tagx="given-names" title="given-names">Seal </span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="source" tagx="source" title="source">Bioinformation</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">430</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21423889">21423889</a> </span> </span> </li> <li tag="ref"> <a name="R14" /> <span class="label" tagx="label" title="label">14</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">D</span> <span class="given-names" tagx="given-names" title="given-names">Lim</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">NC</span> <span class="given-names" tagx="given-names" title="given-names">Strynadka</span> </span> </span> <span class="source" tagx="source" title="source">Nat Struct Biol</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">870</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12389036">12389036</a> </span> </span> </li> <li tag="ref"> <a name="R15" /> <span class="label" tagx="label" title="label">15</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">V</span> <span class="given-names" tagx="given-names" title="given-names">Guillet</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="source" tagx="source" title="source">J Biol Chem</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">279</span> <span class="fpage" tagx="fpage" title="fpage">41028</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15262988">15262988</a> </span> </span> </li> <li tag="ref"> <a name="R16" /> <span class="label" tagx="label" title="label">16</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">GM</span> <span class="given-names" tagx="given-names" title="given-names">Morris </span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="source" tagx="source" title="source">J Comput Aided Mol Des</span> <span class="year" tagx="year" title="year">1996</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">293</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8877701">8877701</a> </span> </span> </li> <li tag="ref"> <a name="R17" /> <span class="label" tagx="label" title="label">17</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">PG</span> <span class="given-names" tagx="given-names" title="given-names">Kareru</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="source" tagx="source" title="source">Afr J Tradit Complement Altern Med</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">214</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/3025622">3025622</a> </span> </span> </li> <li tag="ref"> <a name="R18" /> <span class="label" tagx="label" title="label">18</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">F</span> <span class="given-names" tagx="given-names" title="given-names">Habeeb</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="source" tagx="source" title="source">Methods</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">42</span> <span class="fpage" tagx="fpage" title="fpage">315</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17560318">17560318</a> </span> </span> </li> <li tag="ref"> <a name="R19" /> <span class="label" tagx="label" title="label">19</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">YS</span> <span class="given-names" tagx="given-names" title="given-names">Cho</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="source" tagx="source" title="source">Curr Microbiol</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">57</span> <span class="fpage" tagx="fpage" title="fpage">542</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18781360">18781360</a> </span> </span> </li> <li tag="ref"> <a name="R20" /> <span class="label" tagx="label" title="label">20</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">F</span> <span class="given-names" tagx="given-names" title="given-names">Aqil</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="source" tagx="source" title="source">J Basic Microbiol</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">45</span> <span class="fpage" tagx="fpage" title="fpage">106</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15812867">15812867</a> </span> </span> </li> <li tag="ref"> <a name="R21" /> <span class="label" tagx="label" title="label">21</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">AM</span> <span class="given-names" tagx="given-names" title="given-names">Metwally</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="source" tagx="source" title="source">Pharmacogn Mag</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">212</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20931082">20931082</a> </span> </span> </li> </ul> </div> </div>  </body></html>