<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">BMC Plant Biol</span> <span class="iso-abbrev" title="iso-abbrev">BMC Plant Biol</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">BMC Plant Biology</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1471-2229</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/4552454">4552454</a> </span> <span class="publisher-id" title="publisher-id">562</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1186/s12870-015-0562-x">10.1186/s12870-015-0562-x</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Genome sequencing of herb Tulsi ( <i>Ocimum tenuiflorum</i>) unravels key genes behind its strong medicinal properties </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Upadhyay</span> <span class="given-names" tagx="given-names" title="given-names">Atul K.</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">atulk@ncbs.res.in</span> </span> <a href="#Aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chacko</span> <span class="given-names" tagx="given-names" title="given-names">Anita R.</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">anicko19@gmail.com</span> </span> <a href="#Aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gandhimathi</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">gandhimathi@ncbs.res.in</span> </span> <a href="#Aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ghosh</span> <span class="given-names" tagx="given-names" title="given-names">Pritha</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">prithag@ncbs.res.in</span> </span> <a href="#Aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harini</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">harinik@ncbs.res.in</span> </span> <a href="#Aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Joseph</span> <span class="given-names" tagx="given-names" title="given-names">Agnel P.</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">agnel-praveen.joseph@stfc.ac.uk</span> </span> <a href="#Aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Joshi</span> <span class="given-names" tagx="given-names" title="given-names">Adwait G.</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">adwait@ncbs.res.in</span> </span> <a href="#Aff1" /> <a href="#Aff3" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Karpe</span> <span class="given-names" tagx="given-names" title="given-names">Snehal D.</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">snehaldk@ncbs.res.in</span> </span> <a href="#Aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kaushik</span> <span class="given-names" tagx="given-names" title="given-names">Swati</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">Swati.kaushik15@gmail.com</span> </span> <a href="#Aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kuravadi</span> <span class="given-names" tagx="given-names" title="given-names">Nagesh</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">alwaysnagesh@gmail.com</span> </span> <a href="#Aff2" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lingu</span> <span class="given-names" tagx="given-names" title="given-names">Chandana S</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">chandana@ccamp.res.in</span> </span> <a href="#Aff2" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mahita</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">mahitaj@ncbs.res.in</span> </span> <a href="#Aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Malarini</span> <span class="given-names" tagx="given-names" title="given-names">Ramya</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">ramyaml.nggl@gmail.com</span> </span> <a href="#Aff2" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Malhotra</span> <span class="given-names" tagx="given-names" title="given-names">Sony</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">sonymalh2@gmail.com</span> </span> <a href="#Aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Malini</span> <span class="given-names" tagx="given-names" title="given-names">Manoharan</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">malinim@ncbs.res.in</span> </span> <a href="#Aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mathew</span> <span class="given-names" tagx="given-names" title="given-names">Oommen K.</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">oommenkm@ncbs.res.in</span> </span> <a href="#Aff1" /> <a href="#Aff4" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mutt</span> <span class="given-names" tagx="given-names" title="given-names">Eshita</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">eshi.mutt@gmail.com</span> </span> <a href="#Aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Naika</span> <span class="given-names" tagx="given-names" title="given-names">Mahantesha</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">mahantesha@ncbs.res.in</span> </span> <a href="#Aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nitish</span> <span class="given-names" tagx="given-names" title="given-names">Sathyanarayanan</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">nitishs@ncbs.res.in</span> </span> <a href="#Aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pasha</span> <span class="given-names" tagx="given-names" title="given-names">Shaik Naseer</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">nasir@ncbs.res.in</span> </span> <a href="#Aff1" /> <a href="#Aff3" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raghavender</span> <span class="given-names" tagx="given-names" title="given-names">Upadhyayula S.</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">usraghu2012@gmail.com</span> </span> <a href="#Aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rajamani</span> <span class="given-names" tagx="given-names" title="given-names">Anantharamanan</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">ananthu.bioinfo@gmail.com</span> </span> <a href="#Aff2" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shilpa</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">shilpas@ccamp.res.in</span> </span> <a href="#Aff2" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shingate</span> <span class="given-names" tagx="given-names" title="given-names">Prashant N.</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">prashants@ncbs.res.in</span> </span> <a href="#Aff1" /> <a href="#Aff3" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singh</span> <span class="given-names" tagx="given-names" title="given-names">Heikham Russiachand</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">russiachand@gmail.com</span> </span> <a href="#Aff2" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sukhwal</span> <span class="given-names" tagx="given-names" title="given-names">Anshul</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">anshul@ncbs.res.in</span> </span> <a href="#Aff1" /> <a href="#Aff4" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sunitha</span> <span class="given-names" tagx="given-names" title="given-names">Margaret S.</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">margarets@ncbs.res.in</span> </span> <a href="#Aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sumathi</span> <span class="given-names" tagx="given-names" title="given-names">Manojkumar</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">manojkumarsumathi@gmail.com</span> </span> <a href="#Aff2" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ramaswamy</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">ramas@instem.res.in</span> </span> <a href="#Aff2" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gowda</span> <span class="given-names" tagx="given-names" title="given-names">Malali</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">malalig@ccamp.res.in</span> </span> <a href="#Aff2" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sowdhamini</span> <span class="given-names" tagx="given-names" title="given-names">Ramanathan</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">mini@ncbs.res.in</span> <span class="email" tagx="email" title="email">minirs@gmail.com</span> </span> <a href="#Aff1" /> </span> <span class="citation_author_institution" id="Aff1">[], </span> <span class="citation_author_institution" id="Aff2">[], </span> <span class="citation_author_institution" id="Aff3">[], </span> <span class="citation_author_institution" id="Aff4">[], </span> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2015-8-8</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2015-8-8</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2015</span> </span> <span class="volume" tagx="volume" title="volume">15</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">212</span> <span class="history" title="history"> <span class="received" title="received">received: 2015-3-10</span> <span class="accepted" title="accepted">accepted: 2015-6-24</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , </span> <span class="license" title="license"> <span class="license-p" title="license-p"> <b>Open Access</b> This article is distributed under the terms of the Creative Commons Attribution 4.0 International License ( <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver ( <a href="http://creativecommons.org/publicdomain/zero/1.0/">http://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>Krishna Tulsi, a member of Lamiaceae family, is a herb well known for its spiritual, religious and medicinal importance in India. The common name of this plant is ‘Tulsi’ (or ‘Tulasi’ or ‘Thulasi’) and is considered sacred by Hindus. We present the draft genome of <i>Ocimum tenuiflurum</i> L (subtype Krishna Tulsi) in this report. The paired-end and mate-pair sequence libraries were generated for the whole genome sequenced with the Illumina Hiseq 1000, resulting in an assembled genome of 374 Mb, with a genome coverage of 61 % (612 Mb estimated genome size). We have also studied transcriptomes (RNA-Seq) of two subtypes of <i>O. tenuiflorum,</i> Krishna and Rama Tulsi and report the relative expression of genes in both the varieties. </p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>The pathways leading to the production of medicinally-important specialized metabolites have been studied in detail, in relation to similar pathways in <i>Arabidopsis thaliana</i> and other plants. Expression levels of anthocyanin biosynthesis-related genes in leaf samples of Krishna Tulsi were observed to be relatively high, explaining the purple colouration of Krishna Tulsi leaves. The expression of six important genes identified from genome data were validated by performing q-RT-PCR in different tissues of five different species, which shows the high extent of urosolic acid-producing genes in young leaves of the Rama subtype <i>.</i> In addition, the presence of eugenol and ursolic acid, implied as potential drugs in the cure of many diseases including cancer was confirmed using mass spectrometry. </p> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>The availability of the whole genome of <i>O.tenuiflorum</i> and our sequence analysis suggests that small amino acid changes at the functional sites of genes involved in metabolite synthesis pathways confer special medicinal properties to this herb. </p> </div> <div class="electronicsupplementarymaterial" title="sec"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <p>The online version of this article (doi:10.1186/s12870-015-0562-x) contains supplementary material, which is available to authorized users. </p> </div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd"> <i>O. tenuiflorum</i> </div> <div class="kwd" title="kwd">
Basil</div> <div class="kwd" title="kwd">
Thulasi</div> <div class="kwd" title="kwd">
Genome</div> <div class="kwd" title="kwd">
Transcriptome</div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2015" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Background</div> <p>Plants of the genus <i>Ocimum</i> belong to the family Lamiaceae (Order Lamiales) and are widely distributed in the tropical, sub-tropical and warm temperate regions of the world [ <a href="#CR1">1</a>]. These plants are known to produce essential oils comprising of a number of aromatic compounds and Tulsi is rightly known as the "Queen of Herbs" for this reason. In India, these plants are mostly grown at homes for worship and as offerings in temples. Among plants with medicinal value, those belonging to the genus <i>Ocimum</i> are very important aromatic herbs or shrubs. </p> <p>The genus <i>Ocimum</i> is highly variable and possesses wide genetic diversity at intra and inter-species levels. Nine species of <i>Ocimum</i> viz., <i>O. teniuflorum</i> L., <i>O. basilicum</i> L., <i>O. gratissimum</i> L., <i>O. kilimandscharicum, O. micranthum</i> L. <i>, O. campechianum</i> L., <i>O. americanum</i> L., <i>O. minimum</i> L., and <i>O. citriodorum</i> L., are found in India, three of which ( <i>O. americanum</i> L., <i>O. minimum</i> L., and <i>O. citriodorum</i> L.) are exotic [ <a href="#CR2">2</a>]. It is difficult to distinguish all these species on the basis of leaf morphology alone (Fig. <a href="#Fig1">1</a>). The metabolites (essential oils) of genus <i>Ocimum</i> have been reported to possess antioxidant and antifungal properties and to cure many diseases including bronchitis in Ayurveda, an Indian system of medicine [ <a href="#CR3">3</a>]. Plants produce specialized metabolites as part of their defense mechanisms and these metabolites have significant medicinal properties that cure several human diseases. They can be isolated from various parts of the plant, including leaves, flowers, roots, bark, seeds and stem [ <a href="#CR4">4</a>]. Pharmacological screening and the systematic study of the chemical constituents of plant metabolites provide a basis for developing new drugs. Some of the important metabolites reported from <i>Ocimum</i> species include linalool, linalyl, geraniol, citral, camphor, eugenol, methyleugenol, methyl chavicol, methyl cinnamate, thymol, safrol, taxol, urosolic acid etc. [ <a href="#CR4">4</a>]. These metabolites are of immense value in the pharmaceutical, perfume and cosmetic industries. Metabolites derived from <i>Ocimum</i> species have been found to contain many medicinally relevant properties including anti-cancer, antioxidant, antifungal and anti-inflammatory virtues, and are also recommended for the treatment of malaria, bronchitis, diarrhea, dysentery, etc. [ <a href="#CR5">5</a>]. Essential oils produced as specialized metabolites found in leaves, seeds, flowers and roots of <i>Ocimum</i> species are used in pharmaceutics and many systems of traditional Indian medicine [ <a href="#CR3">3</a>, <a href="#CR4">4</a>]. Genome and transcriptome sequencing of medicinal plants serve as a robust tool for gene discovery and downstream biochemical pathway discovery of medicinally important metabolites [ <a href="#CR6">6</a>]. Recently, an abundance of transcripts for biosynthesis of terpenoids in <i>O. sanctum</i> and of phenylpropanoids in <i>O. basilicum</i> [ <a href="#CR7">7</a>] was reported during an attempt to compare transcriptomes of the two species of <i>Ocimum</i>. Despite its important role in traditional Indian medicine and its impressive arsenal of bioactive compounds, our understanding of Krishna Tulsi biology is limited. In this paper, we present the draft genome sequence of the non-model plant <i>O. tenuiflorum</i> (subtype Krishna), along with transcriptomes of two subtypes, Krishna and Rama Tulsi from leaf samples. We have identified a large set of genes involved in the production of specialized metabolites of medicinal interest such as apigenin, luteolin, rosmarinic acid pathway, eugenol, and ursolic acid. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 1</span> </a> </div>   <p>Plant and leaf morphology of five <i>Ocimum</i> species prevalent in India viz., <i>O. tenuiflorum</i> subtype Krishna, <i>O. tenuiflorum</i> subtype Rama, <i>O. gratissimum</i>, <i>O. sacharicum</i>, <i>O. kilmand.</i> Leaf morphologies are quite different for the five species </p>   </div> <p /> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="genomesequencingandassemblyofthenon-modelplanto.tenuiflorumsubtypekrishna" title="sec"> <div class="title" tagx="title" title="title">
Genome sequencing and assembly of the non-model plant <i>O. tenuiflorum</i> subtype Krishna </div> <p>The paired-end (PE; 2x100-bp) and mate-paired (MP; 2x50-bp) DNA libraries were generated for Krishna Tulsi subtype using Illumina protocols. In total we obtained 373 million reads of PE and 166 million reads of MP data for Krishna Tulsi. Low quality (LQ) sequence reads were trimmed (Additional file <a href="#MOESM1">1</a>: Figure S1 and Additional file <a href="#MOESM2">2</a>: Figure S2) and reads with quality scores of less than Q30 were removed. The good quality reads were used for de-novo genome assembly. Median insert size of PE data was 335 (with a median absolute deviation of 21), whereas median insert size of MP data was 2473 (with a median absolute deviation of 704). K-mer 43 was opted as the best assembly from the statistical analysis of different k-mers. We obtained a maximum scaffold length of 184.7 Kb (Table <a href="#Tab1">1</a>) with an N50 length of 27.1 Kb. This assembly gives rise to a total of 78,224 scaffolds including equal to or more than 100 bp. The current draft assembly of Krishna Tulsi genome is 374.8 Mb in length. The genomic content of Krishna Tulsi is 0.72 pg/2C which is equivalent to 704.6 Mb [ <a href="#CR8">8</a>], but the estimated genome size by k-mer method is 612 Mb and 61 % of the estimated genome size was assembled. The genome size reported in the literature [ <a href="#CR8">8</a>], may be of a different cultivar. This lower genome coverage may be due to limited sequencing data (only two libraries were used in sequencing) or due to a high percentage of repeats (42.9 %). In terms of depth of sequencing, we sequenced 59× of the genome with paired-end (100 bp) and mate-pair (50 bp) libraries (since one lane can produce approximately 30Gb of data, even assuming that reads cover the entire 612 Mb of the estimated genome size). <i>Ocimum</i> species are characterized by the different basic chromosome numbers x = 8, 10, 12, or 16 [ <a href="#CR9">9</a>, <a href="#CR10">10</a>]. In case of <i>O. tenuiflorum</i> individuals with 2n = 32, 2n = 36, and 2n = 76 have been recorded and the chromosome number of <i>O. tenuiflorum</i> is observed to be 2n = 36 [ <a href="#CR8">8</a>]. <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  </div></p><p>Genome assembly results of Krishna Tulsi</p>  <table> <thead> <tr> <th>Assembly</th> <th>Status <sup>a</sup> </th> <th>Number</th> <th>N50 (bp)</th> <th>Longest seq ( <i>Kb</i>) </th> <th>Size ( <i>Mb</i>) </th> <th>% GC</th> </tr> </thead> <tbody> <tr> <td>Contigs</td> <td>All</td> <td>510359</td> <td>2562</td> <td>37.147</td> <td>362.4</td> <td>53.3</td> </tr> <tr> <td>Scaffold</td> <td>All</td> <td>78224</td> <td>27,111</td> <td>184.8</td> <td>374.8</td> <td>54.5</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p> <sup>a</sup>Contigs/scaffolds equal or more than 100 bp </p> </div>  <p /> <p>A comparative analysis of the assemblies generated using PE data alone and with both PE and MP data show that the size and quality of the genome assembled using PE data alone improved substantially with the inclusion of MP data (Additional file <a href="#MOESM3">3</a>: Figures S3 and Additional file <a href="#MOESM4">4</a>: Figure S4, Additional file <a href="#MOESM5">5</a>: Table S1 and Additional file <a href="#MOESM6">6</a>: Table S2). </p> </div> <div class="validationofdenovogenomeassembly,annotationandrepeatcontentofocimumtenuiflorumsubtypekrishnagenome" title="sec"> <div class="title" tagx="title" title="title">
Validation of <i>de novo</i> genome assembly, annotation and repeat content of <i>Ocimum tenuiflorum</i> subtype Krishna genome </div> <p>The <i>de novo</i> genome assembly was validated by mapping raw reads to the assembled genome. On an average, 74 % of reads were mapped back to the assembled genome. Almost 83.3 % of the RNA-seq reads were mapped to the assembled genome. The completeness of <i>de novo</i> genome assembly and annotations were also checked with two other approaches i.e., by using CEGMA (Core Eukaryotic Genes Mapping approach) [ <a href="#CR11">11</a>] and DEG (Database of Essential Genes) [ <a href="#CR12">12</a>] (please see Methods for details). First, we searched for essential eukaryotic genes in the <i>O. tenuiflorum</i> assembly. This resulted in the mapping of 85.1 % of complete core proteins (CEGMA) and more than 95 % including partial genes against our genome assembly (Additional file <a href="#MOESM7">7</a>: Table S3). Secondly, we searched for the predicted genes from the final assembly of essential genes recorded in the DEG database. We observed that about 89 % of essential genes were included within the assembly. These genes were also validated using Pfam domain annotation and were of comparable domain lengths as the classical members of that family (Additional file <a href="#MOESM8">8</a>: Table S4). Phylogenetic trees for highly conserved essential genes like glyceraldehyde 3-phosphate dehydrogenase (Additional file <a href="#MOESM9">9</a>: Figure S5), cytochrome P450 (Additional file <a href="#MOESM10">10</a>: Figure S6) and actin (Additional file <a href="#MOESM11">11</a>: Figure S7) from Krishna Tulsi and their respective homologues were analyzed and compared with other plant species. Krishna Tulsi genes were found to cluster with genes belonging to related species namely, <i>Solanum lycopersicum, Cucumis sativus</i> and even with distantly related <i>Arabidopsis thaliana,</i> indicating that highly conserved genes, essential to plant growth and functioning, have been detected in <i>O. tenuiflorum</i> assemblies. These trends further support the quality of the genome assembly. </p> <p>Regarding the repeat content of the genome, we identified 78224 repeat regions, with a GC content of 36.1 %, adding to 160889218 bp (160 Mb), which constituted 42.9 % of assembled genome which is 374806882 bp (374 Mb) long (Additional file <a href="#MOESM12">12</a>: Table S5). Long terminal repeats (LTRs) are found in large numbers in plant genomes (Schmidt T <b>,</b> 1999) and a similar trend is also found in the type of repeats identified in the Tulsi genome. </p> </div> <div class="genomeannotation" title="sec"> <div class="title" tagx="title" title="title">
Genome annotation</div> <p>We identified 36768 putative gene models in the initial genome draft (version 1.2) of <i>O. tenuiflorum</i> genome. At least one gene was observed in each of the 10012 scaffolds, with an average of three to four genes per scaffold. During the process of refined gene prediction, 16384 gene models were observed to have expression evidence (RNA-Seq data from leaves of Tulsi (Krishna and Rama)). A total of 19384 gene models have been identified by <i>ab initio</i> means (without any RNA or protein evidence) (Table <a href="#Tab2">2</a>). <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 2</span>  </div></p><p>Genome annotation results of Krishna Tulsi</p>  <table> <thead> <tr> <th>Annotation</th> <th>Number</th> <th>Average size (bp)</th> <th>Total length (Mb)</th> <th>% of genome</th> <th>Transcript-evidence</th> <th> <i>Ab initio</i> </th> <th>Pfam hit</th> </tr> </thead> <tbody> <tr> <td>Gene</td> <td>36768</td> <td>2421</td> <td>86.3</td> <td>23</td> <td>16384</td> <td>19384</td> <td>24,607</td> </tr> </tbody> </table>  <p /> <p>All the gene predictions, with or without RNA/protein evidences, were screened based on length (&amp;gt;100 bp). In case of sequential overlaps between different gene models, the gene models which are of longer length and with RNA or protein evidence for a given region of scaffold were preferred over the ones without any evidence. </p> <p>There are 31,020 genes with at least one homologue in NRDB and 24,607 genes which contain at least one Pfam domain. In total, 3929 unique Pfam domains were identified for all the predicted genes in Tulsi (please see URL: <a href="http://caps.ncbs.res.in/Ote">http://caps.ncbs.res.in/Ote</a> for the full list of predicted genes). A majority of domains identified were protein kinases or LRR-containing domains (Additional file <a href="#MOESM13">13</a>: Figure S8). Further comparison of Pfam results, with assembled plant genomes of similar size, reveals that the number of predicted gene models is in overall agreement in numbers as well as gene boundaries. </p> </div> <div class="orthologyoftulsigenes" title="sec"> <div class="title" tagx="title" title="title">
Orthology of Tulsi genes</div> <p>The orthology relationships were deduced between Krishna Tulsi ( <i>O. tenuiflorum</i>; Ote) and four other species viz. <i>Arabidopsis thaliana</i> (Ath), <i>Mimulus guttatus</i> (Mgu), <i>Solanum lycopersicum</i> (Sly) and <i>Oryza sativa</i> (Osa) (please see Methods for details). We observe 8370 clusters which contain a total of 89922 gene products from the five plant species (Fig. <a href="#Fig2">2a</a>). <i>M. guttatus</i> and <i>O. tenuiflorum</i> share the same order (Lamiales), but belong to different families (Phrymaceae and Lamiaceae, respectively), which was evident from the presence of the highest number of common gene families (11707) between them. This was followed by <i>Solanum lycopersicum</i> (11022), <i>Arabidopsis thaliana</i> (10206) and <i>Oryza sativa</i> (9154) as expected from taxonomic hierarchy (Fig. <a href="#Fig2">2a</a>). We found 17584 genes to be orthologous to any of the above four species. Considering all the 36768 Ote genes, 1282 groups contained only Ote Krishna Tulsi genes (3302). We obtained 16 Ote genes which lack traceable orthology to 22 other plant species and homology relationships (list of these genes is available on the database). Few of these unique Ote genes are transposons. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 2</span> </a> </div>   <p>Distribution and clustering of orthologous genes of Tulsi genome to other related plant genomes. <b>a</b>. Distribution of gene families among five plant genomes. <i>Ocimum tenuiflorum</i> (Ote - green), <i>Arabidopsis thaliana</i> (Ath – black rectangle), <i>Oryza sativa</i> (Osa – red), <i>Solanum lycopersicum</i> (Sly – blue) and <i>Mimulus guttatus</i> (Mgu – black circle). The numbers in the Venn diagram represent shared and unique gene families across these 5 species obtained by OrthoMCL. <b>b</b>. Horizontal stacked bar plot of all the genes in 23 different genomes. This figure shows ortholog group distribution in all 23 plant species including Tulsi. Each row represents a plant species - <i>Physcomitrella patens</i> (Ppa), <i>Selaginella moellendorffii</i> (Smo) <i>, Oryza sativa</i> (Osa) <i>, Setaria italic</i> (Sit) <i>, Zea mays</i> (Zma) <i>, Sorghum bicolor</i> (Sbi) <i>, Aquilegia caerulea</i> (Aca) <i>, Ocimum tenuiflorum</i> (Ote) <i>, Mimulus guttatus</i> (Mgu) <i>, Solanum lycopersicum</i> (Sly) <i>, Solanum tuberosum</i> (Stu) <i>, Vitis vinifera</i> (Vvi) <i>, Eucalyptus grandis</i> (Egr) <i>, Citrus sinensis</i> (Csi) <i>, Theobroma cacao</i> (Tca) <i>, Carica papaya</i> (Cpa) <i>, Brassica rapa</i> (Bra) <i>, Arabidopsis thaliana</i> (Ath) <i>, Fragaria vesca</i> (Fve) <i>, Prunus persica</i> (Ppe) <i>, Glycine max</i> (Gma) <i>, Medicago truncatula</i> (Mtr) <i>, Populus trichocarpa</i> (Ptr). The bar graph represents ortholog protein groups for that species subdivided into 22 categories depending on the degree of sharing with the other 22 plant species e.g., category 2 represents the number of orthologous groups that have representatives from the species of interest and from one more species out of the 23 species selected for the study </p>   </div> <p /> <p>In order to inspect in detail the distribution of the orthologous relationship of <i>Ocimum</i> genes across different species and taxonomic levels, 22 fully-sequenced plant genomes (Additional file <a href="#MOESM14">14</a>: Table S6) were considered. The orthologous groups from all 23 species were organized according to the clustering. Three hundred and thirty four clusters of genes are present across all the 23 species chosen for the study. Common genes across all species, comprising of their respective orthologous group, are plotted as a horizontal stacked bar plot (Fig. <a href="#Fig2">2b</a>). The pattern of sharing orthologous groups is quite unique to primitive plant genomes (like lycophyte and bryophyte) and monocots. However, the pattern observed in the Tulsi genome is quite similar to that of <i>M. guttatus</i> (Mgu). Interestingly, this pattern is somewhat different for two members of Solanacea, which have more genes shared only in two out of 23 genomes, perhaps due to other features such as polyploidy. </p> </div> <div class="genesinvolvedinsynthesisofspecializedmetabolitesofmedicinalvalue:comparativeanalysisbetweeno.tenuiflorum(ote,krishnatulsi)andotherplantgenomes" title="sec"> <div class="title" tagx="title" title="title">
Genes involved in synthesis of specialized metabolites of medicinal value: comparative analysis between <i>O. tenuiflorum</i> (Ote, Krishna Tulsi) and other plant genomes </div> <p>Next, we performed a restricted analysis of the genes involved in the metabolite production in Ote and the genomes of a few plant species that are either closely-related ( <i>S. lycopersicum</i>, <i>V. vinifera)</i> or well-characterised ( <i>M. truncatula</i>, and <i>A. thaliana</i>). We observed 121 (72.45 %), 130 (77.84 %), 106 (63.47 %) and 94 (56.28 %) scaffolds and contigs from the selected four representative genomes associated with 167 metabolite-related scaffolds and contigs in Ote Krishna Tulsi (Fig. <a href="#Fig3">3</a>) respectively <i>.</i> In terms of the number of orthologous genes from this selected plant genome associated with metabolite genes of Ote, we observed a similar trend of association as 601, 620, 570 and 556 genes in <i>S. lycopersicum</i>, <i>V. vinifera, M. truncatula</i>, and <i>A. thaliana</i> respectively. These numbers agree with the taxonomic phylogeny and hierarchy, suggesting that the evolution of genes involved in metabolic pathways is not a cause of recent expansions or sudden drifts. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 3</span> </a> </div>   <p>Phylogenetic representation of five selected plant genomes viz., <i>Solanum lycopercicum</i> (72.45 %), <i>Vitis vinifera</i> (77.84 %), <i>Medicago trucatula</i> (63.47 %), and <i>Arabidopsis thaliana</i> (56.28 %). The numbers indicate percentage of association of these genomes with the metabolite genes of <i>Ocimum</i> genome. These percentages agree with the taxonomic phylogeny and hierarchy, suggesting that the evolution of genes involved in metabolic pathways is not a cause of recent expansions or sudden genome drifts. The inner circle represents chromosomes from respective homolog genome. Each scaffold is organized in the middle circle and is represented in chronological order as per position on chromosomes. The line represents location of each scaffold on the respective chromosome. Colors indicate <span class="inline-graphic" title="inline-graphic" /> = &amp;lt; 2 genes, <span class="inline-graphic" title="inline-graphic" /> =2 genes, <span class="inline-graphic" title="inline-graphic" /> = &amp;gt; 2 genes, <span class="inline-graphic" title="inline-graphic" /> = Metabolite related genes. Height of orange columns in outermost circle represents amount of repeats in corresponding scaffolds </p>   </div> <p /> <p>When compared against 11,389 scaffolds (greater than 10Kb in size) from Ote, 10032, 9997, 8648 and 8277 scaffolds were found to be associated with the four reference plant genomes (Additional file <a href="#MOESM15">15</a>: Figure S9, Additional file <a href="#MOESM16">16</a>: Figure S10 and Additional file <a href="#MOESM17">17</a>: Figure S11 for three genomes and Additional file <a href="#MOESM18">18</a>: Table S7 for four genomes). Further, most of the metabolite-related scaffolds in Ote Krishna Tulsi were associated with chromosomes 1, 6, 8, and 10 of tomato (Fig. <a href="#Fig4">4</a>). In particular, gene products that are likely associated in luteolin synthesis pathway are observed to cluster in scaffolds, which are similar to nucleotide stretches in Chromosomes 3, 5, 6, 8 and 10 of the tomato genome (Fig. <a href="#Fig4">4</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 4</span> </a> </div>   <p>Circular representation of <i>O.tenuiflorum</i> metabolite related genes mapped onto chromosomes of <i>Solanum lycopersicum</i> genome. Height of orange column <span class="inline-graphic" title="inline-graphic" /> in outer circle represents amount of repeats present in respective scaffold. The inner circle represents chromosomes from Tomato genome. Inner circle of rectangles represents scaffolds, each scaffold is organized in the middle circle and is represented in chronological order as per position on chromosomes. Color of each scaffold indicates following information: <span class="inline-graphic" title="inline-graphic" /> =2 genes, <span class="inline-graphic" title="inline-graphic" /> = &amp;gt; 2 genes, <span class="inline-graphic" title="inline-graphic" /> = Metabolite related genes. Connecting line between scaffolds and chromosome represents postion of the scaffold in genome. Red color of connecting line represents presence of metabolite related genes. Scaffold Numbers are mentioned in Additional file <a href="#MOESM24">24</a>: Text A </p>   </div> <p /> </div> <div class="transcriptomedenovoassemblyofkrishnaandramatulsimatureleafsamples" title="sec"> <div class="title" tagx="title" title="title">
Transcriptome <i>de novo</i> assembly of Krishna and Rama Tulsi mature leaf samples </div> <p> <i>De novo</i> transcriptome assembly was performed for the mature leaf samples of subtype Krishna Tulsi. The best assembly resulted in 109291 contigs with N50 of 893 bp and longest sequence of 12.1 Kb. All these contigs added up to 49.5 Mb with a GC content of 42.9 %. Scaffolding of these contigs resulted in 89878 scaffolds with N50 of 1597 bp and longest sequence of 12.7 Kb. All these scaffolds added up to 56.3 Mb with a GC content of 42.9 % (Table <a href="#Tab3">3</a>). Similarly, assembly was performed for the subtype Rama Tulsi and combined reads (Krishna and Rama Tulsi) as well (Table <a href="#Tab3">3</a>). <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 3</span>  </div></p><p>Transcriptome assembly of <i>Ocimum tenuiflorum</i> subtype Krishna, Rama and combined data </p>  <table> <thead> <tr> <th /> <th>Assembly</th> <th>Status <sup>a</sup> </th> <th>Number</th> <th>N50 (bp)</th> <th>Longest seq ( <i>Kb</i>) </th> <th>Size ( <i>Mb</i>) </th> <th>% assembly</th> </tr> </thead> <tbody> <tr> <td>Krishna</td> <td>Contigs</td> <td>All</td> <td>109291</td> <td>893</td> <td>12.1</td> <td>49.5</td> <td>42.9</td> </tr> <tr> <td /> <td>Scaffold</td> <td>All</td> <td>89878</td> <td>1597</td> <td>12.7</td> <td>56.3</td> <td>42.9</td> </tr> <tr> <td>Rama</td> <td>Contigs</td> <td>All</td> <td>108701</td> <td>623</td> <td>5.6</td> <td>39.4</td> <td>43.4</td> </tr> <tr> <td /> <td>Scaffolds</td> <td>All</td> <td>102038</td> <td>815</td> <td>6.4</td> <td>40.5</td> <td>43.4</td> </tr> <tr> <td>Combined reads</td> <td>Contigs</td> <td>All</td> <td>128469</td> <td>742</td> <td>12.2</td> <td>54.4</td> <td>42.9</td> </tr> <tr> <td /> <td>Scaffolds</td> <td>All</td> <td>115164</td> <td>1073</td> <td>12.2</td> <td>57.1</td> <td>42.9</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p> <sup>a</sup>Contigs/scaffolds equal or more than 100 bp </p> </div>  <p /> </div> <div class="differentialexpressionoftranscripts" title="sec"> <div class="title" tagx="title" title="title">
Differential expression of transcripts</div> <p>The differentially expressed genes found in the transcriptomes of both the Tulsi subtypes were analysed. We observe a substantial number of genes up-regulated and down-regulated in Krishna Tulsi, compared to Rama Tulsi. Some of the highly expressed genes were also confirmed by q-RT-PCR technique in different tissue samples i.e., stems, leaves and flowers and also in five species viz. <i>O. tenuiflorum</i> subtype Krishna and Rama, <i>O. gratissimum</i>, <i>O. basilicum</i>, and <i>O. kilmand.</i> </p> <p>For a comparison, we generated a heat map of the top 50 differentially more abundant genes in Krishna Tulsi samples (Fig. <a href="#Fig5">5a</a>). Similarly, top 50 differentially more abundant genes in Rama with respect to Krishna sample were also plotted (Fig. <a href="#Fig5">5b</a>). Gamma-cadinene synthase is one of the top 50 differentially expressed transcripts with RPKM values of 577.0 and 31.7 in Krishna and Rama Tulsi samples, respectively (please see below for details). Other highly expressed transcripts in Krishna Tulsi sample are Heat shock cognate protein 80, Cellulose synthase A catalyic subunit 6 (UDP-forming), Fructose-biphosphate aldolase (chloroplatic), Phototropin-2, and Rubisco activase 1 (chloroplatic). The chalcone synthase or naringenin-chalcone synthase (CHS) is one of the enzymes important for coloration of plant parts, which is observed to be highly expressed. Abundance values of all the transcripts, along with their functional annotations by NCBI BLAST results and their corresponding Krishna Tulsi genomic scaffold, show several genes involved in the synthesis of specialized metabolites implicated to be of medicinal value (Additional file <a href="#MOESM19">19</a>: Table S8). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 5</span> </a> </div>   <p>Transcript expression of Tulsi Krishna and Rama subtypes are expressed as RPKM values. Highly significant differentially abundant RNA scaffolds/transcripts were defined to have RPKM of atleast 5 in both and the fold-change difference between two subtypes should be atleast 8 times. Only the transcripts, for which the 95 % lower-confidence-bound of more abundant subtype and 95 % upper-confidence-bound of less abundant subtype, and had at least 8 times difference, were retained. Of these differentially abundant transcripts, top-50 in Krishna and Rama subtype were plotted in the form of heat-map. <b>a</b>. Differentially more abundant transcripts in Krishna. <b>b</b>. Differentially more abundant transcripts in Rama. (please look in Additional file <a href="#MOESM24">24</a>: Text B and C for transcript IDs for a. and b) </p>   </div> <p /> <p>Dark purple coloration of the leaves and stem of subtype Krishna Tulsi is one of its characteristic phenotypes, which distinguishes it from other subtypes and species of genus <i>Ocimum.</i> Chalcone synthase (CHS) is an enzyme belonging to a family of polyketide synthases which catalyzes the initial step for flavonoid biosynthesis. Flavonoids are important plant specific metabolites that perform various functions such as pigmentation, antifungal defense <i>etc</i>. Reviewed protein sequence for CHS from UniProt (Universal Protein resource) database [ <a href="#CR13">13</a>] was employed to search against annotated protein sequences of Krishna Tulsi genome and six transcripts were obtained as possible hits. The best hit could be identified with 95 % query coverage and 99 % sequence identity. The extent of abundance of this hit (protein sequence) was checked in the leaf transcriptome of both the Tulsi subtypes viz. Krishna and Rama. Abundance (in terms of RPKM) of the six transcripts was, on an average, two times more in case of Krishna as compared to Rama (please see Fig. <a href="#Fig5">5</a>), and may be involved in the coloration phenotype of Krishna subtype plants [ <a href="#CR14">14</a>]. For further confirmation of expression of these transcripts, q-RT-PCR was performed. As expected, anthocyanin producing gene was observed to be more abundant in Krishna young leaf samples and mature leaf samples (used as control) (Fig. <a href="#Fig6">6a</a> and <a href="#Fig6">b</a>). In contrast, the chlorophyll binding protein was more abundant in Krishna mature leaf samples. In addition, we also examined the presence of gamma-cadeninene synthase gene which is responsible for aroma [ <a href="#CR15">15</a>]. This gene was found to be more abundant in Rama root sample and young leaf samples of <i>O. Saccharum</i>, but not observed in higher quantities in <i>O. kilmund</i>. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 6</span> </a> </div>   <p>Expression quantification of selected genes by q-RT-PCR method. <b>a</b>. Fold changes of genes involved in color production, obtained through q-RT PCR. Blue colour horizontal bar is for chlorophyll a-b binding protein, red to denote Gamma-cadenine synthase and green to denote Anthocyanin. Mature leaf of Krishna subtype was used as control. It can be seen that, genes responsible for color production such as Chlorophyll a-b binding protein and gene in anthacyanin pathway are down-regulated as compared to mature Krishna leaf, which corresponds to phenotypic characteristics. <b>b</b>. Fold changes of genes involved in ursolic acid biosynthetic pathway, as obtained through qRT-PCR for 5 different Tulsi subtypes. Blue colour horizontal bar is for squalene epoxidase, red to denote alpha-amyrin synthase and green to denote Cytochrome P450 monooxygenase. Mature leaf of Krishna subtype was used as control. Mature leaf of Rama subtype has high expression of genes while expression in <i>Ocimum kilmund</i> is low. Expression of these genes are uniformly high in small, developing plants. Samples are as follows: 1) <i>O. tenuiflorum</i> (Rama) - Sampling Leaf. 2) <i>O. tenuiflorum</i> (Rama) - Sampling Root. 3) <i>O. tenuiflorum</i> (Rama) - Mature Leaf. 4) <i>O. tenuiflorum</i> (Krishna) - Sampling Leaf. 5) <i>O. tenuiflorum</i> (Krishna) - Sampling Root. 6) <i>O. gratissimum</i> - Sampling Leaf. 7) <i>O. gratissimum</i> - Sampling Root. 8) <i>O. gratissimum</i> - Mature Leaf. 9) <i>O. sacharicum</i> - Sampling Leaf. 10) <i>O. sacharicum</i> - Sampling Root. 11) <i>O. sacharicum</i> - Mature Leaf. 12) <i>O. kilmund</i> - Sampling Leaf. 13) <i>O. kilmund</i> - Sampling Root. 14) <i>O. kilmund</i> - Mature Leaf </p>   </div> <p /> </div> <div class="specializedmetabolitesdetectionandvalidation" title="sec"> <div class="title" tagx="title" title="title">
Specialized metabolites detection and validation</div> <p>Nearly 30 specialized metabolites (Fig. <a href="#Fig7">7a</a>) are reported form the genus <i>Ocimum</i> which are found to have medicinal values or properties [ <a href="#CR4">4</a>]. Amongst these, 14 metabolites belonging to five basic groups were found to have complete pathway information in PlantCyc database ( <a href="http://www.plantcyc.org/">http://www.plantcyc.org/</a>) [ <a href="#CR16">16</a>] (Additional file <a href="#MOESM20">20</a>: Figure S12). Hence, genes involved in these pathways were chosen for further analysis and searched against the assembled genome of <i>O. tenuiflorum</i>. Figure <a href="#Fig7">7b</a> highlights the distribution of the genes identified in various classes of metabolites of disease relevance (i.e., these metabolites are well-known as drugs in the cure of human diseases). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 7</span> </a> </div>   <p>Number of genes involved in specialized metabolite synthesis in Tulsi genome. <b>a</b>. There are four classes of metabolites present in <i>Ocimum</i> genome viz., sesquiterpenes (52 %), flavonoids (19 %), terpenes (18 %) and phenylpropanoids (11 %). Number in the bracket is percentage of sepecialized metabolites present in the genome. 458 genes were identified as coding for enzymes involved in synthesis of specialized metabolites. <b>b</b>. Specialized metabolic pathways of disease relevance proposed in <i>Ocimum tenuiflorum.</i> Major classes of diseases investigated are indicated in different colors: anticancer <span class="inline-graphic" title="inline-graphic" />, anticancer-antioxidant <span class="inline-graphic" title="inline-graphic" />, antifungal <span class="inline-graphic" title="inline-graphic" />, antiseptic <span class="inline-graphic" title="inline-graphic" />, anti-infective <span class="inline-graphic" title="inline-graphic" />, antioxidant <span class="inline-graphic" title="inline-graphic" />, and anti-inflammatory <span class="inline-graphic" title="inline-graphic" />. The enzymes have been labelled with 5–7 letters for convenience. The numbers after the’_’ in the enzyme label represent the number of putative hits found for the given enzyme in the genomic assembly of <i>O. tenuiflorum</i>. The metabolites involved in the disease relevance and the enzymes involved in the synthesis of these metabolites are as follows: APIGENIN (Flavone-synthaseI-FSYN1, Naringenin-NADPH-oxygen-oxidoreductase-NNOOX), LUTEOLIN (Flavone-synthaseI-FSYN1, Naringenin-NADPH-oxygen-oxidoreductase-NNOOX, Flavone-3-monooxygenase-F3MON), TAXOL (Taxadiene-synthase-TSYN, Taxadiene-5-alpha-hydroxylase-T5AHYD, Taxadien-5-alpha-ol-O-acetyltransferase-T5AOOA, Taxane-10-beta-hydroxylase-T10BHYD, Taxoid-14-beta-hydroxylase-T14BHYD, 2-alpha-hydroxytaxane-2-O-benzoyltransferase-2AH2OB, 10-deacetylbaccatin-III-10-O-acetyltransferase-10D10OA, 3-N-debenzoyl-2-deoxytaxol-N-benzoyltransferase-3ND2DNB, URSOLIC ACID (ursolic-aldehyde-28-monooxygenase-UA28M, Alpha-amyrin-synthase-AASYN), OLEANOLIC ACID (Beta-amyrin-synthase-BASYN, oleanolic-aldehyde-28-monooxygenase-OA28M), SITOSTEROL (24C-methyltransferase-24CMET), ROSMARINIC ACID I (4-coumaroyl-4-hydroxyphenyllactate-3-hydroxylase-4C4H3H, Tyrosine-transaminase-TTRAN), ROMARINIC ACID II (Hydroxyphenylpyruvate-reductase-HPPRE, Tyrosine-3-monooxygenase-TTRAN), METHYL CAHVICOL (Eugenol-o-methyltransferase-EOMET), EUGENOL (Alcohol-o-acetyltransferase-AOACE, Eugenol-synthase-ESYN, Isoeugenol-synthase-ISYN), LINALOOL (Farnesyl-pyrophosphate-synthase-FPSYN, R-linool-synthase-RLSYN, S-linool-synthase-SLSYN), CARYOPHYLENE (Alpha-humulene-synthase-AHSYN, Beta-caryophyllene-synthase-BCSYN), SELINENE (Alpha-selinene-synthase-ASSYN, Beta-selinene-synthase-BSSYN), CITRAL (Geraniol-synthase-GSYN, Geraniol-dehdrogenase-GDHYD) </p>   </div> <p /> <p>A total of 458 genes were identified in Ote genome, which are either homologous or directly code for enzymes involved in the synthesis of specialized metabolites (Fig. <a href="#Fig8">8</a>) (details of gene IDs of these proteins are provided in Table <a href="#Tab4">4</a> and Additional file <a href="#MOESM21">21</a>: Table S9). Twenty eight <i>O. tenuiflorum</i> gene products were annotated as putative terpene synthases using BLAST sequence searches with E-value of 10 <sup>−4</sup> and query coverage filter of &amp;gt;75 % (Additional file <a href="#MOESM22">22</a>: Table S10). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 8</span> </a> </div>   <p>Phylogeny of terpene synthases of representative sequences of six classes from the plant kingdom along with putative Tulsi terpene synthases genes: The tree is color coded as tpsa:red, tbsb:blue, tpsc:yellow, tpsd: green, tpse: blue and tpsf:purple </p>   </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 4</span>  <p>The enzymes involved in metabolite biosynthesis were identified in the assembled genome and these genes were analyzed for their expression level in the transcriptome. The RKPM value signifies the level for expression </p>  <table> <thead> <tr> <th>Sno.</th> <th>Metabolite</th> <th>Disease implication</th> <th>Enzymes involved</th> <th>No. of hits in <i>Ocimum</i> genome </th> <th>RPKM value</th> </tr> </thead> <tbody> <tr> <td /> <td>Flavonoids</td> <td /> <td /> <td /> <td /> </tr> <tr> <td>1</td> <td>Apigenin</td> <td>Anti-cancer</td> <td>Flavone synthaseI</td> <td>16</td> <td>-</td> </tr> <tr> <td /> <td /> <td /> <td>Naringenin, NADPH oxygen oxidoreductase</td> <td>17</td> <td>-</td> </tr> <tr> <td>2</td> <td>Luteolin</td> <td>Anti-cancer</td> <td>Flavone synthaseI</td> <td>7</td> <td>34.44</td> </tr> <tr> <td /> <td /> <td /> <td>Naringenin, NADPH oxygen oxidoreductase</td> <td>9</td> <td>61.34</td> </tr> <tr> <td /> <td /> <td /> <td>Flavone 3’ monooxygenase</td> <td>5</td> <td>5.53</td> </tr> <tr> <td /> <td /> <td /> <td>Naringenin 3’ hydroxylase</td> <td>26</td> <td>104.24</td> </tr> <tr> <td /> <td /> <td /> <td>Eriodictoyl NADPH oxygen oxidoreductase</td> <td>7</td> <td>59.79</td> </tr> <tr> <td /> <td>Phenylpropanoids</td> <td /> <td /> <td /> <td /> </tr> <tr> <td>3</td> <td>Rosmarinic acid Pathway1</td> <td>Anti-cancer, anti-oxidant</td> <td>4-coumaroyl-4’-hydroxyphenyllactate 3-hydroxylase</td> <td>19</td> <td>120.52</td> </tr> <tr> <td /> <td /> <td /> <td>Tyrosine_transaminase</td> <td>2</td> <td>0.00</td> </tr> <tr> <td>4</td> <td>Rosmarinic acid Pathway 2</td> <td>Anti-cancer, anti-oxidant</td> <td>Hydroxyphenylpyruvate_reductase</td> <td>3</td> <td>45.78</td> </tr> <tr> <td /> <td /> <td /> <td>tyrosine -3-monooxygenase</td> <td>0</td> <td>35.58</td> </tr> <tr> <td>5</td> <td>Eugenol</td> <td>Anti-infective</td> <td>Alcohol_o_acetyltransferase</td> <td>10</td> <td>4.69</td> </tr> <tr> <td /> <td /> <td /> <td>Eugenol synthase</td> <td>5</td> <td>68.93</td> </tr> <tr> <td /> <td /> <td /> <td>Isoeugenol synthase</td> <td>5</td> <td>68.93</td> </tr> <tr> <td>6</td> <td>Methylchavicol</td> <td>Anti-fungal, antiparasitic, anti-oxidant</td> <td>Eugenol-o-methyltransferase</td> <td>4</td> <td>132.39</td> </tr> <tr> <td /> <td>Terpenes</td> <td /> <td /> <td /> <td /> </tr> <tr> <td>7</td> <td>Citral</td> <td>Antiseptic</td> <td>Geraniol synthase</td> <td>9</td> <td>0.00</td> </tr> <tr> <td /> <td /> <td /> <td>Geraniol_dehdrogenase</td> <td>32</td> <td>46.10</td> </tr> <tr> <td>8</td> <td>Linalool</td> <td>Anti-infective</td> <td>Farnesyl-pyrophosphate synthase</td> <td>6</td> <td>128.35</td> </tr> <tr> <td /> <td /> <td /> <td>r-linool_synthase</td> <td>15</td> <td>132.39</td> </tr> <tr> <td /> <td /> <td /> <td>s-linool_synthase</td> <td>22</td> <td>32.91</td> </tr> <tr> <td /> <td>Sesquiterpenes</td> <td /> <td /> <td /> <td /> </tr> <tr> <td>9</td> <td>Caryophylene</td> <td>Anti-inflammatory</td> <td>Alpha humulene synthase</td> <td>27</td> <td>3.92</td> </tr> <tr> <td /> <td /> <td /> <td>Beta-caryophyllene synthase</td> <td>17</td> <td>3.92</td> </tr> <tr> <td>10</td> <td>Selinene</td> <td>Anti-oxidant</td> <td>Alpha_selinene_synthase</td> <td>11</td> <td>13.31</td> </tr> <tr> <td /> <td /> <td /> <td>Beta_selinene_synthase</td> <td>13</td> <td /> </tr> <tr> <td>11</td> <td>Taxol</td> <td>Anti-cancer</td> <td>Taxadiene synthase</td> <td>3</td> <td>9.36</td> </tr> <tr> <td /> <td /> <td /> <td>Taxadiene 5-alpha hydroxylase</td> <td>33</td> <td>3.43</td> </tr> <tr> <td /> <td /> <td /> <td>Taxadien-5-alpha-ol O-acetyltransferase</td> <td>6</td> <td>26.88</td> </tr> <tr> <td /> <td /> <td /> <td>Taxane 10-beta-hydroxylase</td> <td>41</td> <td>3.43</td> </tr> <tr> <td /> <td /> <td /> <td>Taxoid 14-beta-hydroxylase</td> <td>22</td> <td>3.43</td> </tr> <tr> <td /> <td /> <td /> <td>2-alpha-hydroxytaxane 2-O-benzoyltransferase</td> <td>3</td> <td /> </tr> <tr> <td /> <td /> <td /> <td>10-deacetylbaccatin III 10-O-acetyltransferase</td> <td>3</td> <td /> </tr> <tr> <td /> <td /> <td /> <td>3’-N-debenzoyl-2’-deoxytaxol N-benzoyltransferase</td> <td>5</td> <td>4.85</td> </tr> <tr> <td>12</td> <td>Ursolic acid</td> <td>Anti-cancer</td> <td>Cytochrome P450 monooxygenase</td> <td>23</td> <td>3.43</td> </tr> <tr> <td>13</td> <td>Oleanolic acid</td> <td>Anti-cancer</td> <td>Beta-amyrin synthase</td> <td>7</td> <td>12.69</td> </tr> <tr> <td /> <td /> <td /> <td>Cytochrome P450 monooxygenase</td> <td>23</td> <td>3.43</td> </tr> <tr> <td /> <td>Sterols</td> <td /> <td /> <td /> <td /> </tr> <tr> <td>14</td> <td>Sitosterol</td> <td>Anti-cancer</td> <td>24C_methyltransferase</td> <td>1</td> <td>67.25</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p> <sup>-</sup>refers to cases where there was no significant transcript evidence </p> </div> </div> <p /> <p>Among these specialized metabolites, we focused on ursolic acid, belonging to sesquiterpenes, since it is known to have anti-inflammatory, anti-microbial, anti-tumour and anti-cancer properties. The synthesis of ursolic acid from squalene is a three-step process starting from squalene (Fig. <a href="#Fig9">9</a>). α-Amyrin is formed by concerted cyclization of squalene epoxide, while ursolic acid is eventually synthesized by the catalytic activity of multifunctional cytochrome P450. The enzymes involved are, therefore, squalene epoxidase, alpha-amyrin synthase and alpha-amyrin 2, 8 monoxygenase. Sequence search algorithms were employed to search for the three enzymes of this pathway in the Tulsi genome, starting from protein sequences for each of these enzymes from PlantCyc database as queries. The search for squalene epoxidase in Tulsi, using the sequence of this enzyme in <i>Oryza sativa japonica</i> (LOC_Os02g04710.2) as a query, gave rise to a hit (C3776143), with 50 % sequence identity covering 80 % of the query length (Additional file <a href="#MOESM23">23</a>: Figure S13). Using Amyrin synthase LUP2 from <i>A. thaliana</i> (Q8RWT0) and 13 other well-accepted alpha/beta amyrin synthases as a query, four hits were identified in the Tulsi genome (scaffold16333, scaffold20801, scaffold12312 and maker-C3776143). In classical amyrin synthases, a QW structural motif repeats six times in the entire sequence [ <a href="#CR17">17</a>, <a href="#CR18">18</a>], while there are two functional motifs, <i>viz</i>., a well conserved SDTAE [ <a href="#CR19">19</a>] motif which is believed to form the catalytic pocket and the MWCYCR [ <a href="#CR20">20</a>] motif that is shown to play a crucial role in catalysis. These motifs are observed in the four hits in Tulsi genome (Additional file <a href="#MOESM24">24</a>: Text D). Further, a phylogenetic tree was constructed using 16 query sequences and these four hits (Fig. <a href="#Fig10">10</a>). One of the Tulsi hits, (scaffold 16333_mrnal) clusters with a well-characterized alpha amyrin synthase from <i>C. roseus</i> (H2ER439) suggesting that this particular scaffold might indeed retain an alpha amyrin synthase. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 9</span> </a> </div>   <p>The synthesis of ursolic acid from squalene is a three-step process starting from squalene. A: Squalene epoxidase, B: α-amyrin synthase, C1: α-amyrin 28-monooxygenase [Multifunctional], C2: Uvaol dehydrogenase [Multifunctional] and C3: Ursolic aldehyde 28-monooxygenase. Squalene epoxidase and alpha amyrin synthase, along with alpha amyrin 28 mono-oxygenase, uvol dehydrogenase and ursolic aldehyde 28 mono-oxygenase, play important role in synthesis of ursolic acid. These three genes have been chosen for quantification of gene expression by q-RT PCR method in different tissues and species </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 10</span> </a> </div>   <p>Phylogenetic tree of sixteen amyrin query sequences and four putative amyrins from Tulsi. Tulsi hits are marked in blue clour, red ones are alpha amyrin synthase, greens are beta amyrin synthase and cyan ones are proteins from other class of amyrin. The presence of motifs and position in the phylogeny indicate that the hits obtained in <i>O. tenuiflorum</i> genome are likely to be alpha-amyrin synthases </p>   </div> <p /> <p>Interestingly, many genes involved in the synthesis of specialized metabolites of relevance in the treatment of diseases are also more abundant, as observed in the assembled transcriptome (Additional file <a href="#MOESM21">21</a>: Table S9). Similarly, genes involved in the synthesis of 16 other specialized metabolites (Additional file <a href="#MOESM25">25</a>: Table S11), are also equally interesting. However, this requires detailed understanding of the mechanism of synthesis and enzymes involved in the pathways. We analysed RNA-Seq data of two leaf samples in order to compare the genes related to important metabolite pathways and the peculiar phenotype of <i>O. tenuiflorum</i> subtype Krishna with subtype Rama Tulsi. There were 104 transcripts, whose fold change in expression was observed to be eight times more in Krishna Tulsi than in Rama Tulsi. Likewise, there were 229 transcripts whose fold change expression was eight times more in Rama Tulsi as compared to Krishna Tulsi. These are available for download at- (caps.ncbs.res.in/download/tdat_data/Supplementary_tables/Supplementary Table 8.txt). </p> <p>In the case of the multifunctional Cytochome P450 (which catalyses the last three steps in the synthesis of urosolic acid, Fig. <a href="#Fig9">9</a>), a predicted gene from scaffold2032 was obtained as a hit, when a reviewed UniProt entry F1T282 from <i>V. vinifera</i> was considered as query and searched in the Tulsi genome assembly using BLAST. This hit retains 61 % sequence identity and the alignment covers 90 % of the length of the query (alignments are shown in Additional file <a href="#MOESM23">23</a>: Figure S13). This scaffold contains a total of three predicted genes viz., Ote100020320011, Ote100020320001 (similar to UHRF1-binding protein) and Ote100020320031 (gene of interest). </p> <p>From the available transcriptome assembly, these genes, identified as involved in the synthesis of urosolic acid, were analysed for their levels of expression. The RPKM values were also high for these three genes (please see Additional file <a href="#MOESM21">21</a>: Table S9). To further validate the levels of expression of these genes, q-RT-PCR was performed using sequence-specific primers. The presence of these three enzymes is generally high in all the mature leaf samples and highest in Rama subtype (using Krishna subtype as control). Alpha-amyrin synthase is more abundant in mature leaf samples of <i>O. gratissimum</i> and <i>O. sacharicum</i> species. However, interestingly, the three enzymes are found to be more abundant in the young leaf samples of Rama subtype; in contrast, atleast one of the three genes is less in the Krishna leaf sample and in all root samples. The expression of the three genes implicated in urosolic acid synthesis is uniformly low in samples of <i>O. kilmund.</i> </p> <p>Next, to correlate gene expression and to quantify the presence of ursolic acid and eugenol, chemical profiling was performed using LC-Mass spectrometry from different tissues and samples. Eugenol and ursolic acid were observed in the highest quantities in mature leaf sample of Rama subtype and in relatively low quantities in <i>O. kilmund</i>. The amount of eugenol in the leaf sample of <i>O. tenuiflorum</i> subtype Rama (2235 ng/mg) is considerably high followed by <i>O. kilmund</i> (1472 ng/mg), <i>O. sacharicum</i> (651 ng/mg) and lowest in <i>O. gratissimum</i> (73 ng/mg). In all stem samples, the amount of eugenol is consistently low with the highest in <i>O. tenuiflorum</i> subtype Rama (24 ng/mg), <i>O. tenuiflorum</i> subtype Krishna (17 ng/mg), <i>O. kilmund</i> (15 ng/mg) and below limits of quantification in <i>O. gratissimum</i> and <i>O. sacharicum</i>. The presence of oleanolic acid is also severely reduced in stem samples of Rama subtype (2869 ng/mg) and in Krishna subtype (1088 ng/mg) in comparison to the mature leaf samples (7556 ng/mg for Rama and 4630 ng/mg for Krishna). The presence of urosolic acid is 50 % less in stem samples of Rama subtype (2883 ng/mg) when compared to the mature leaf samples (4597), whereas it is much lower in the stem samples of other species as compared to the leaf sample. The amount of ursolic acid in the stem samples of Krishna subtype (746 ng/mg) is 4.6 times less than that of the mature leaf samples (3471 ng/mg) (please see Table <a href="#Tab5">5</a>). <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 5</span>  </div></p><p>Estimation results of Eugenol and Ursolic acid in different samples</p>  <table> <thead> <tr> <th>S. No</th> <th>Sample</th> <th>Eugenol (ng/mg) Leaf Stem</th> <th>Ursolic acid (ng/mg) Leaf Stem</th> <th>Oleanolic acid (ng/mg) Leaf Stem</th> </tr> </thead> <tbody> <tr> <td>1</td> <td> <i>O. tenuiflorum</i> (Rama Tulsi) </td> <td>2235.93</td> <td>24.91</td> <td>4597.62</td> <td>2883.46</td> <td>7556.84</td> <td>2869.80</td> </tr> <tr> <td>2</td> <td> <i>O. tenuiflorum</i> (Krishna Tulsi) </td> <td>449.89</td> <td>17.16</td> <td>3471.59</td> <td>746.02</td> <td>4630.53</td> <td>1088.21</td> </tr> <tr> <td>3</td> <td> <i>O. gratissimum</i> </td> <td>73.88</td> <td>BLQ</td> <td>4391.97</td> <td>1139.15</td> <td>2584.56</td> <td>711.08</td> </tr> <tr> <td>4</td> <td> <i>O. scharicum</i> </td> <td>651.09</td> <td>BLQ</td> <td>4436.26</td> <td>1491.68</td> <td>2975.63</td> <td>1024.14</td> </tr> <tr> <td>5</td> <td> <i>O. kilmand</i> </td> <td>1472.51</td> <td>15.74</td> <td>1099.85</td> <td>191.09</td> <td>945.22</td> <td>159.37</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p> <i>• BLQ</i> Below level of quantification </p> </div>  <p /> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p> <i>O. tenuiflorum</i> subtype Krishna Tulsi is one of the non-model plants of great medicinal value, for which there has been no genomic information available till date. We have performed genome sequencing of <i>O. tenuiflorum</i> subtype Krishna of the paired-end (PE; 2x100-bp) and mate-paired (MP; 2x50-bp) DNA libraries by Illumina Hiseq 1000. The best <i>de novo</i> assembly was obtained at k-mer 43 by SOAPdenovo2, an eukaryotic <i>de novo</i> genome assembler. Repeats were identified and masked, and gene prediction and annotation was carried out using the MAKER annotation pipeline by using genomic, transcriptomics and EST data. The nearest species whose genome has been sequenced is the monkey flower ( <i>M. guttatus</i>), which shares its order Lamiales with <i>O. tenuiflorum</i> (Ote) but falls in a different family (Phrymaceae). Orthology search of Ote Krishna Tulsi genes in four genomes viz. <i>A. thaliana</i> (Ath), <i>M. guttatus</i> (Mgu), <i>S. lycopersicum</i> (Sly) and <i>O. sativa</i> (Osa) also confirmed the close relationship between Krishna Tulsi and <i>M. guttatus</i> (Mgu), in terms of the number of common gene families i.e., 578 out of 2488 total genes. When we considered all the 36,768 predicted genes from the Krishna Tulsi genome, we found that 1282 ortholog groups have <i>Ocimum-</i>only genes. These 1282 groups contain 13,306 <i>Ocimum</i> genes and hence they are referred to as paralogs by OrthoMCL. Of the remaining Ote genes, 17,584 genes were found to be orthologous to any of the other four species studied in this case. We performed an analysis of the genes involved in the metabolite production in <i>Ote</i> and the genomes of a few other related plant species. Based on the direct evidence or homology a total of 458 genes were identified in Ote genome, which are involved in coding of enzymes implied in the synthesis of specialized metabolites. Comparative analysis of transciptomes of <i>O. tenuiflorum</i> subtype Krishna and Rama was performed to detect potential differentially-regulated genes and their involvement in metabolite synthesis. On comparing both the transcriptomes, differentially expressed genes were observed with a substantial number of genes more abundant and others less abundant in either subtypes. Gamma-cadinene synthase is more abundant in Krishna sample (RPKM value 577.047) as compared to Rama sample (RPKM value 31.73). To confirm some of the more abundant genes along with Gamma-cadinene synthase, we performed q-RT-PCR in different tissue samples i.e., stem and leaves and also in five species viz. <i>O. tenuiflorum</i> subtype Krishna and Rama, <i>O. gratissimum</i>, <i>O. basilicum</i>, and <i>O. kilmand.</i> Expression of Gamma-cadinene synthase is found more in Krishna samples as compared to Rama by q-RT-PCR also. Likewise, Chalcone synthase (CHS) is an anthocyanin-producing gene, which is observed to be more abundant in young leaf samples of Krishna and mature leaf samples in transcriptome data. Subsequently, this has been confirmed by q-RT-PCR and from mass spectrometry readings of ursolic acid and eugenol from different tissue samples and from different species. </p> </div> <div class="conclusion" title="conclusion"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>We present a draft genome of <i>O. tenuiflorum</i> Krishna Tulsi subtype Krishna Tulsi. The habitat of genus <i>Ocimum</i> is tropical climate and it is wide spread over Asia, Africa, Central and South America. High RNA-seq expression values of the genes responsible for the purple coloration of the plant parts in Krishna subtype, as compared to Rama subtype, were observed. We also identified a fFew unique genes (16) of Ote, which lack any traceable orthology and homology relationships from all the 22 species used in this study. </p> <p>Krishna Tulsi is described in the Vedas and Puranas (ancient scriptures of Hindus) and has a long history of cultivation, of roughly 3000 years, and is therefore assumed to be of Indian origin [ <a href="#CR21">21</a>]. In literature, it is also referred to as the "Queen of Herbs". Major genes involved in the synthesis of medicinally important specialized metabolites in the plant could be unraveled despite limited data on sequencing and coverage [ <a href="#CR22">22</a>]. Expressions of these genes were confirmed by complementing with RNA-seq data and q-RT-PCR method. We also investigated one of the important metabolic pathways involving the production of ursolic acid in detail, by mass-spectrometry and q-RT-PCR methods. Synthesis of specialized metabolites or their precursors appear to begin in the young leaves of Tulsi. Subsequently, the mature leaves retain the medicinally relevant metabolites. <i>O. tenuiflorum</i> Rama subtype retains the high abundance of key medicinally relevant metabolites like eugenol and ursolic acid, as observed in the transcriptome, metabolite quantifications and q-RT-PCR expression values consistent with its high medicinal values. Our main emphasis was to unravel the important metabolite genes by using genomic and transcriptomic data despite limited sequencing information. </p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Methods</div> <div class="isolationofgenomicdnafromo.tenuiflorumsubtypekrishnatulsi" title="sec"> <div class="title" tagx="title" title="title">
Isolation of genomic DNA from <i>O. tenuiflorum</i> subtype Krishna Tulsi </div> <p>Young leaves of Tulsi subtype Krishna and Rama were used for genomic DNA isolation. About one gram of leaves were crushed using liquid Nitrogen and DNA extraction buffer (200 mM TrisHCL [pH-8.0], 200 mM NaCl, 25 mM EDTA and 1 % PVP) was added [ <a href="#CR23">23</a>]. The ground material along with 1/10th volume of 20 % SDS solution was incubated at 65 °C for 30 min. The tubes were centrifuged at 14,000 RPM for 10 min at room temperature to remove the debris. The supernatant was transferred into a fresh tube and treated with equal volume of phenol: chloroform: isoamyl alcohol (25:24:1) and mixed gently for 5 min. The mixture was centrifuged at 12,000 RPM for 10 min to separate the phases. The aqueous phase from the centrifuged tube was transferred to a fresh tube and DNA was precipitated with 1/5th volume of 2 M NaCl and 2 volumes of ice-cold ethanol. The DNA was pelleted by centrifugation at 12,000 RPM for 10 min. Precipitated DNA pellet was taken as a starting material for purification using the Sigma Genelute plant DNA isolation kit (G2N70, Sigma). The DNA was run on a 1 % agarose gel to assess the quality. The A260/280 ratio and quantity were determined using the nanodrop. </p> </div> <div class="genomesequencing,assemblyandannotation" title="sec"> <div class="title" tagx="title" title="title">
Genome sequencing, assembly and annotation</div> <p>Genome sequencing was performed by using Illumina HiSeq 1000 technology in the Next Generation Genomics Facility at Centre for Cellular and Molecular Platforms (C-CAMP). Genomic DNA paired-end and gel free mate-pair library preparation was performed for Krishna Tulsi using TruSeq DNA sample preparation kit (FC-121-2001) and Nextera mate-pair sample preparation kit (FC-132-1001) from Illumina ( <a href="http://www.illumina.com">www.illumina.com</a>). FASTX-Toolkit [ <a href="#CR24">24</a>] and FastQC tools [ <a href="#CR25">25</a>] were used for pre-processing of raw reads and for quality check of the reads. Genome assembly from reads of PE and MP together was done by using SOAPdenovo2, a <i>de novo</i> draft genome assembler [ <a href="#CR26">26</a>]. Preliminary assemblies were performed based on k-mers from 21 to 63 with an interval of two. Gene prediction and annotation was carried out using the MAKER annotation pipeline [ <a href="#CR27">27</a>] with predicted gene models using AUGUSTUS [ <a href="#CR28">28</a>] and <i>A. thaliana</i> genes as reference for initial prediction. The gene models were refined using homology searches against all protein sequences from <i>Viridaeplantae</i> kingdom. </p> </div> <div class="validationofgenomeassemblyandannotations" title="sec"> <div class="title" tagx="title" title="title">
Validation of genome assembly and annotations</div> <p>To validate genome assembly, we have mapped raw reads on to the <i>de novo</i> assembled genome by using REAPR (SMALT) [ <a href="#CR29">29</a>], SAMtools [ <a href="#CR30">30</a>] and Picard tools ( <a href="http://broadinstitute.github.io/picard/">http://broadinstitute.github.io/picard/</a>). Maximum and minimum insert size of 500 bp and 0 bp respectively were selected for mapping. We report an alignment pairing with best score, using standard Smith-Waterman scores. The threshold minimum score used was calculated by the formula to be: &amp;lt; Minimum score &amp;gt; = &amp;lt; world length &amp;gt; + step size – 1. Here the word length of 13 is used with a step size of 6. Estimation of the genome size of the Tulsi genome was done using the k-mer distribution analysis by Jellyfish [ <a href="#CR31">31</a>]. Essential genes implicated in the regulation, assembly and functioning of plant cells, have been identified in the Krishna Tulsi assembled genome using a two-way approach. Firstly, using CEGMA which was derived from the KOG database [ <a href="#CR32">32</a>] (for eukaryotic genomes) and core proteins in any eukaryotic genome (including ones in draft stages), essential genes were annotated. Secondly, a subset of <i>A. thaliana</i> genes were extracted from a well-characterized Database of Essential Genes (DEG) and compared against Krishna Tulsi assemblies. Validation of the extracted genes was performed by Pfam domain annotation approaches. Putative essential genes from the Krishna Tulsi dataset were further searched using BLASTP [ <a href="#CR33">33</a>] against the NCBI (NR) database and closely-related homologues were aligned and phylogenetic tree constructed. </p> </div> <div class="repeatidentification" title="sec"> <div class="title" tagx="title" title="title">
Repeat identification</div> <p>Repeat elements in the assembled genome were identified using RepeatScout (version 1.0.5) [ <a href="#CR34">34</a>] and RepeatMasker (version 4.0.3) [ <a href="#CR35">35</a>]. The library of <i>ab initio</i> repeats generated by RepeatScout was classified into known repeat classes using the RepeatClassifier module of RepeatScout (Additional file <a href="#MOESM12">12</a>: Table S5). The RepBase library of RepeatMasker and the non-redundant library of <i>ab-initio</i> classified repeats were then used to mask the repeat elements in the assembled genome. The repeat-masked genome assembly was then used for genome annotation. </p> </div> <div class="genomeannotation" title="sec"> <div class="title" tagx="title" title="title">
Genome annotation</div> <p>The repeat-masked assembled genome of Krishna Tulsi was processed through the MAKER annotation pipeline [ <a href="#CR27">27</a>]. AUGUSTUS [ <a href="#CR28">28</a>] was used for gene prediction, trained on <i>A. thaliana</i> gene models. RNA-seq data obtained from leaf samples was used as EST evidence to refine the gene models. Initial gene models of protein sequences belonging to <i>Viridaeplantae</i> kingdom, obtained from the NCBI database, were used as protein evidence for refining gene prediction. Both EST and protein evidence were prepared using EXONERATE [ <a href="#CR36">36</a>] and used for gene prediction refinement through AUGUSTUS. All the protein sequences of these gene models were subjected to validation based on identification of homologues through BLASTP search against NRDB at E-value cutoff of 10 <sup>−3</sup>. Pfam release 27 was consulted for all domain predictions with an E-value cutoff of 10 <sup>−5</sup> using HMMER3 package [ <a href="#CR37">37</a>]. </p> </div> <div class="orthologydetection" title="sec"> <div class="title" tagx="title" title="title">
Orthology detection</div> <p>All the predicted gene models from Krishna Tulsi were used with OrthoMCL tool [ <a href="#CR38">38</a>] to identify clusters between selected species of <i>A. thaliana</i> (Ath), <i>O. sativa</i> (Osa), <i>S. lycopersicum</i> (Sly), <i>M. guttatus</i> (Mgu). In order to inspect distribution of the orthologous relationship of <i>Ocimum</i> genes across different species and taxonomic levels, ProteinOrtho tool [ <a href="#CR39">39</a>] was implemented on Krishna Tulsi (Ote) gene models along with 22 different species: <i>Aquilegia caerulea</i> (Aca), <i>Glycine max</i> (Gma), <i>Setaria italic</i> (Sit), <i>Mimulus guttatus</i> (Mgu), <i>Solanum lycopersicum</i> (Sly), <i>Arabidopsis thaliana</i> (Ath), <i>Medicago truncatula</i> (Mtr), <i>Selaginella moellendorffii</i> (Smo), <i>Brassica rapa</i> (Bra), <i>Oryza sativa</i> (Osa), <i>Solanum tuberosum</i> (Stu), <i>Carica papaya</i> (Cpa), <i>Physcomitrella patens</i> (Ppa), <i>Theobroma cacao</i> (Tca), <i>Camellia sinensis</i> (Csi), <i>Prunus persica</i> (Ppe), <i>Vitis vinifera</i> (Vvi), <i>Eucalyptus grandis</i> (Egr), <i>Populus trichocarpa</i> (Ptr), <i>Zea mays</i> (Zma), <i>Fragaria vesca</i> (Fve), <i>Sorghum bicolor</i> (Sbi). All the complete proteome sets were obtained from Phytozome resource [ <a href="#CR40">40</a>]. Phylogenetic tree reconstruction was carried out using ‘RbcS’ (Rubisco small subunit) coding sequences from all 23 species. CLUSTALW [ <a href="#CR41">41</a>] and Phylip package [ <a href="#CR42">42</a>] were employed for multiple sequence alignment (MSA) and subsequent clustering using Neighbor Joining (NJ) method, respectively. Distant homology relationships were verified through PSI-BLAST [ <a href="#CR33">33</a>] at different set of E-value cutoffs. Gene products for which we were unable to establish any homology or orthology relationships, but consisted of a Pfam domain, were referred to as unique genes specific to Ote. </p> </div> <div class="comparativeanalysisbetweenkrishnatulsiandotherplantgenomes" title="sec"> <div class="title" tagx="title" title="title">
Comparative analysis between Krishna Tulsi and other plant genomes</div> <p>The most recent version of whole genome sequences of <i>S. lycopersicum, V. vinefera, M. tranculata</i> and <i>A. thaliana</i> were downloaded from NCBI ( <a href="ftp://ftp.ncbi.nlm.nih.gov/genomes/">ftp://ftp.ncbi.nlm.nih.gov/genomes/</a>). BLAT [ <a href="#CR43">43</a>] was employed for sequence searches using <i>S. lycopersicum, V. vinefera, M. tranculata</i> and <i>A. thaliana</i> genomes against two sets of Tulsi genome data: one containing 11389 scaffolds (which are greater than 10000 bp) and another containing 167 scaffolds and contigs with metabolite-related genes (identified earlier on the Krishna Tulsi genome). The figures were prepared using in-house software written for this purpose. </p> </div> <div class="isolationofrnafromtulsisubtypes,krishnaandrama,andrna-seqlibrarypreparation" title="sec"> <div class="title" tagx="title" title="title">
Isolation of RNA from Tulsi subtypes, Krishna and Rama, and RNA-seq library preparation</div> <p>RNA isolation was carried out with 100 mg of the leaf tissue (Rama and Krishna) using the Sigma Spectrum Plant Total RNA Kit (STRN50, Sigma). DNA contamination was removed by DNAse treatment using DNA-free™ kit (AM1906, Ambion). The DNase free RNA quality was determined using the Agilent Bioanalyzer. The RNA Integrity Number (RIN) values of all the samples were greater than 6. The A260/280 ratio and the quantity were determined using the nanodrop. RNA-seq library preparation was done with 1 μg of total RNA following the TruSeq RNA sample preparation from Illumina (RS-122-2001). </p> </div> <div class="transcriptomesequencingandassembly" title="sec"> <div class="title" tagx="title" title="title">
Transcriptome sequencing and assembly</div> <p>We assembled all the mRNA reads having HQ scores of all the bases more than 20, of Krishna and Rama subtype separately and also by combining the reads from both of these subtypes by using SOAPdenovo-trans [ <a href="#CR26">26</a>] at different K-mers starting from 19 to 63 at an interval of two. An insert size of 350 was used for the assembly of transcriptomes. RNA-seq reads were mapped to the assembled genome by Tophat2 [ <a href="#CR44">44</a>], which uses Bowtie2 [ <a href="#CR45">45</a>] as a mapping tool. We used a minimum and maximum intron length of 50 and 500000 bp respectively. Maximum multi hits (parameter that dictates the number of alignments to the reference for a given read) was assigned as 20 and transcriptome max hits (maximum number of mappings allowed for a read, when aligned to the transcriptome) of 60 was used. </p> </div> <div class="transcriptdifferentialexpressioncomparison" title="sec"> <div class="title" tagx="title" title="title">
Transcript differential expression comparison</div> <p>To quantify expression in terms of reads per kilo base per million (RPKM), non-redundant combined assembled transcript sequences (at 90 % sequence similarity by CD-hit EST [ <a href="#CR46">46</a>]) were taken as reference. This non-redundant transcriptome was used as the reference transcriptome to calculate differential expression of transcripts in both the samples [ <a href="#CR6">6</a>, <a href="#CR47">47</a>]. The reads of RNA-seq experiments from Krishna and Rama subtypes were mapped back on to the reference transcriptome by using SeqMap (version – 1.0.12) [ <a href="#CR48">48</a>] and RPKM values were determined by using rSeq: RNA-seq analyzer (version 0.1.1) [ <a href="#CR49">49</a>]. </p> </div> <div class="specializedmetabolitesdetectionandvalidation" title="sec"> <div class="title" tagx="title" title="title">
Specialized metabolites detection and validation</div> <p>The dataset obtained after gene prediction on the assembled genome was employed to search for enzymes involved in secondary metabolite production. There are 14 metabolites (flavonoids (2), phenylpropanoids (4), terpenes (2), sesquiterpenes (5) and sterols (1)), which are reported to be present in <i>Ocimum</i> and have known pathway information in PlantCyc ( <a href="http://www.plantcyc.org/">http://www.plantcyc.org/</a>) [ <a href="#CR16">16</a>]. Reviewed entries from the UniProt database and all the known sequences of the enzymes from other species possessing these enzymes were used as queries to search in the full dataset of scaffolds and contigs, using PSI-BLAST at E-value of 10 <sup>−5</sup> and three iterations. The protein hits obtained in our dataset were further subjected to validation using a query coverage filter of 75 %. </p> <p>In order to study the expression of genes involved in the synthesis of specialized metabolite (s), the assembled transcriptome of both <i>Ocimum</i> species were searched, employing the reviewed entry corresponding to each enzyme in the UniProt database. These searches were performed using TBLASTN at an E-value of 10 <sup>−3</sup>, and the best hit in our dataset was selected based on the least E-value. If the reviewed entry for any of the enzyme was not present, unreviewed entries from PlantCyc database were employed. </p> </div> <div class="quantificationofeugenolandursolicacidusinguhplc-ms/srmmethod" title="sec"> <div class="title" tagx="title" title="title">
Quantification of eugenol and ursolic acid using UHPLC-MS/SRM method</div> <p>A Vantage TSQ triple stage quadrupole mass spectrometer (Thermo Fisher Scientific, San Jose, CA, USA) equipped with a heated electro spray ionization (HESI) source was used for the analysis of eugenol and an APCI probe was used for the ursolic acid analysis. The mass spectrometer was interfaced with an Agilent 1290 infinity UHPLC system (Agilent Technologies India Pvt. Ltd., India) equipped with a column oven (set at 40 °C), auto sampler and a thermo-controller (set at 4 °C). The needle was washed from outside with acetonitrile (0.1 % formic acid) before every injection to avoid any potential carry-over problems. Separations were performed using a shim-pack XR-ODSIII column (2 × 150 mm, 2 μm). For Eugenol: Mobile phase A was water (10 mM Ammonium acetate) containing 0.1 % formic acid, and mobile phase B was acetonitrile containing 0.1 % formic acid. For Ursolic acid: Mobile phase A was water (10 mM Ammonium acetate), and mobile phase B was acetonitrile: methanol (3:1). Injections of 10 μL were performed using flow through a needle </p><ul> <li> <span class="label" tagx="label" title="label">(A)</span> <p>Eugenol:</p> </li> </ul> <p /> <p>Eugenol was quantified after derivatizing with pyridine sulfonyl chloride using estrone-d4 as an internal standard. Methanol was used to extract eugenol from fresh leaves (2 mg/mL) and dried stem powder (20 mg/ml). Briefly 10 μL of extract and 10 μL of internal standard (from 2.5 μg/mL) were added into 200 μL of buffer [acetone: NaHCO3 (1:1)]. To this 10 μL of pyridine sulfonyl chloride (10 mg/mL) was added and incubated at 60 °C for 15 min. After incubation the derivative was extracted with 800 μL of MTBE and the organic layer was dried and reconstituted in 50 μL of methanol followed by 10 μL injection for the analysis. A gradient (0–2 mins:30 %B, 2–5 mins:30–90 %B, 5–7 mins:90–100 %B, 7–10 mins:100 %B, 10–10.1 mins:100–30 %B, 10.1–15 mins:30) was then initiated at a flow rate of 200 μL/min. Operating conditions were as follows: spray voltage, 3000 V; ion transfer capillary temperature, 270 °C; source temperature 100 °C; sheath gas 20, auxiliary gas 5 (arbitrary units); collision gas, argon; S-lens voltage was optimized for individual metabolites; scan time of 50 millisec/transition; and ion polarity positive. A standard curve was constructed from 0.078 to 5ngon column to quantify eugenol. The SRM transition used for the analysis of eugenol is (306.1 → 79) and for estrone-d4 (416.3 → 274.1). </p><ul> <li> <span class="label" tagx="label" title="label">(B)</span> <p>Ursolic Acid:</p> </li> </ul> <p /> <p>Ursolic acid was quantified using estrone-d4 as an internal standard. A brief extraction was done from 2 mg/mL of dry powder using 1 mL of methanol (sonication-3 min, centrifugation −5 min). The extract was further diluted to 0.2 mg/mL in methanol. From this extract 10 μL was added along with 10 μL of internal standard (0.1 ug/mL) to 30 μL of methanol and 10 μL was injected for the analysis. A gradient (0–2 mins:20 %B, 2–8 mins:20–100 %B, 8–14.5 mins:100 %B, 14.5–14.6 mins:100–20 %B, 14.6–20 mins:20 %B) was then initiated at a flow rate of 200 μL/min. Operating conditions were as follows: Discharge current 4 μA; ion transfer capillary temperature, 270 °C; source temperature 300 °C; sheath gas 20, auxiliary gas 5 (arbitrary units); collision gas, argon; S-lens voltage was optimized for individual metabolites; scan time of 50 millisec/transition; and ion polarity positive. A standard curve was constructed from 0.034 to 2.5 ng on column to quantify ursolic acid. The same standard curve was used for the analysis of oleanolic acid. The SRM transition used for the analysis of both ursolic and oleanolic acid is (439.4 → 119) and for estrone-d4 (275.3 → 257.1). </p> </div> <div class="availabilityofsupportingdatasection" title="sec"> <div class="title" tagx="title" title="title">
Availability of supporting data section</div> <p>Information on the genes identified in Tulsi, along with the scaffold numbers, are provided in <a href="http://caps.ncbs.res.in/Ote">http://caps.ncbs.res.in/Ote</a>. </p> <p>BioProject : PRJNA251328</p> <p>SRA id : SRP051184</p> <p>Accession number of <i>O. tenuiflorum</i>: JQCZ00000000 </p> <p>Also please see DOI for supporting data: <a href="https://mynotebook.labarchives.com/share/National%2520Centre%2520for%2520Biological%2520Sciences/MTkuNXw2MjMwNC8xNS9UcmVlTm9kZS80MjAwNTk4MTM5fDQ5LjU">https://mynotebook.labarchives.com/share/National%2520Centre%2520for%2520Biological%2520Sciences/MTkuNXw2MjMwNC8xNS9UcmVlTm9kZS80MjAwNTk4MTM5fDQ5LjU</a>= </p> <p>Data available from the Dryad Digital Repository: <a href="http://dx.doi.org/10.5061/dryad.6f1r2">http://dx.doi.org/10.5061/dryad.6f1r2</a> </p> </div> </div> </div> <div class="back" title="back"> <div class="app-group" title="app-group"> <div class="app" title="app"> <div class="additionalfiles" title="sec"> <div class="title" tagx="title" title="title">
Additional files</div> <p> </p><div class="media" title="media"> <a href="12870_2015_562_MOESM1_ESM.png">LINK</a> <span class="label" tagx="label" title="label">Additional file 1: Figure S1.</span>  <p>Per base sequence quality of R1 reads of PE sequences used in final genome assembly.</p>  </div> <div class="media" title="media"> <a href="12870_2015_562_MOESM2_ESM.png">LINK</a> <span class="label" tagx="label" title="label">Additional file 2: Figure S2.</span>  <p>Per base sequence quality of R2 reads of PE sequences used in final genome assembly.</p>  </div> <div class="media" title="media"> <a href="12870_2015_562_MOESM3_ESM.png">LINK</a> <span class="label" tagx="label" title="label">Additional file 3: Figure S3.</span>  <p>Distribution of assembled scaffolds according to their length.</p>  </div> <div class="media" title="media"> <a href="12870_2015_562_MOESM4_ESM.png">LINK</a> <span class="label" tagx="label" title="label">Additional file 4: Figure S4.</span>  <p>Distribution of scaffold length difference between paired end and paired with mate pair end assembly. </p>  </div> <div class="media" title="media"> <a href="12870_2015_562_MOESM5_ESM.doc">LINK</a> <span class="label" tagx="label" title="label">Additional file 5: Table S1.</span>  <p>Scaffold lengths distribution in the MP + PE and PE assemblies.</p>  </div> <div class="media" title="media"> <a href="12870_2015_562_MOESM6_ESM.doc">LINK</a> <span class="label" tagx="label" title="label">Additional file 6: Table S2.</span>  <p>Statistics of scaffold length comparison from assemblies of PE and MP + PE together.</p>  </div> <div class="media" title="media"> <a href="12870_2015_562_MOESM7_ESM.doc">LINK</a> <span class="label" tagx="label" title="label">Additional file 7: Table S3.</span>  <p>Completeness of assembly and presence of essential genes by CEGMA results for <i>O. tenuiflorum</i> at two levels; (a) only in PE assembly (b) in PE + PM assembly. </p>  </div> <div class="media" title="media"> <a href="12870_2015_562_MOESM8_ESM.doc">LINK</a> <span class="label" tagx="label" title="label">Additional file 8: Table S4.</span>  <p>Presence of essential genes in <i>O.tenuiflorum</i> (Tulsi) at three levels; a) in only paired end assembly (ab-initio gene prediction), b) in paired end and mate-pair assembly’s Level 2 [evidence from RNAseq, EST and known tulsi genes], c) in paired end and mate-paired assembly’s Level 1 (gene prediction). </p>  </div> <div class="media" title="media"> <a href="12870_2015_562_MOESM9_ESM.png">LINK</a> <span class="label" tagx="label" title="label">Additional file 9: Figure S5.</span>  <p>Phylogenetic trees of essential gene, cytochrome P450 from <i>O.tenuiflorum</i> and their respective homologues. </p>  </div> <div class="media" title="media"> <a href="12870_2015_562_MOESM10_ESM.png">LINK</a> <span class="label" tagx="label" title="label">Additional file 10: Figure S6.</span>  <p>NJ tree for glyceraldehydes phosphate dehydrogenase protein in <i>O. tenuiflorum</i> (Tulsi, marked in red) and its nearest homologues. </p>  </div> <div class="media" title="media"> <a href="12870_2015_562_MOESM11_ESM.png">LINK</a> <span class="label" tagx="label" title="label">Additional file 11: Figure S7.</span>  <p>Phylogenetic trees of essential genes, actin from <i>O.tenuiflorum</i> and their respective homologues. </p>  </div> <div class="media" title="media"> <a href="12870_2015_562_MOESM12_ESM.doc">LINK</a> <span class="label" tagx="label" title="label">Additional file 12: Table S5.</span>  <p>Repeat elements identified in Tulsi genome assembly and classified in different groups of repeats. </p>  </div> <div class="media" title="media"> <a href="12870_2015_562_MOESM13_ESM.tif">LINK</a> <span class="label" tagx="label" title="label">Additional file 13: Figure S8.</span>  <p>Pie chart of distribution of protein domains (Pfam) of all the predicted genes in <i>O. tenuiflorum</i> subtype Krishna genome. </p>  </div> <div class="media" title="media"> <a href="12870_2015_562_MOESM14_ESM.xls">LINK</a> <span class="label" tagx="label" title="label">Additional file 14: Table S6.</span>  <p>List of species used for phylogeny analysis along with <i>Ocimum</i> to depict taxonomical distribution of this species in plant kingdom. </p>  </div> <div class="media" title="media"> <a href="12870_2015_562_MOESM15_ESM.png">LINK</a> <span class="label" tagx="label" title="label">Additional file 15: Figure S9.</span>  <p>Circular representation of <i>O. tenuiflorum</i> metabolite-related genes mapped onto <i>Vitis vinefera</i> plant genome. Color indicate blue = &amp;lt; 2 genes, green =2 genes, yellowgreen = &amp;gt; 2 genes, red = Metabolite related genes. Connecting line between scaffolds and chromosome represents postion of the scaffold in genome. Red color of connecting line represents presence of metabolite related genes. </p>  </div> <div class="media" title="media"> <a href="12870_2015_562_MOESM16_ESM.png">LINK</a> <span class="label" tagx="label" title="label">Additional file 16: Figure S10.</span>  <p>Circular representation of <i>O. tenuiflorum</i> metabolite-related genes mapped onto <i>Medicago tranculata</i> plant genome. Color indicate blue = &amp;lt; 2 genes, green =2 genes, yellowgreen = &amp;gt; 2 genes, red = Metabolite-related genes. Connecting line between scaffolds and chromosome represents postion of the scaffold in genome. Red color of connecting line represents presence of metabolite related genes. </p>  </div> <div class="media" title="media"> <a href="12870_2015_562_MOESM17_ESM.png">LINK</a> <span class="label" tagx="label" title="label">Additional file 17: Figure S11.</span>  <p>Circular representation of <i>O. tenuiflorum</i> metabolite-related genes mapped onto <i>Arabidopsis thaliana</i> plant genome. Color indicate blue = &amp;lt; 2 genes, green =2 genes, yellowgreen = &amp;gt; 2 genes, red = Metabolite-related genes. Connecting line between scaffolds and chromosome represents postion of the scaffold in genome. Red color of connecting line represents presence of metabolite-related genes. </p>  </div> <div class="media" title="media"> <a href="12870_2015_562_MOESM18_ESM.xls">LINK</a> <span class="label" tagx="label" title="label">Additional file 18: Table S7.</span>  <p>Associations of <i>O. tenuiflorum</i> a) scaffolds related to metabolite-related genes, b) longer length scaffolds (greater than 10Kb in size) to four different plant genomes. </p>  </div> <div class="media" title="media"> <a href="12870_2015_562_MOESM19_ESM.txt">LINK</a> <span class="label" tagx="label" title="label">Additional file 19: Table S8.</span>  <p>All the transcripts with their expression and validation at different level such as genome hit and blast hit results against non-redundant database of NCBI. </p>  </div> <div class="media" title="media"> <a href="12870_2015_562_MOESM20_ESM.pdf">LINK</a> <span class="label" tagx="label" title="label">Additional file 20: Figure S12.</span>  <p>Pathways of all the 14 important medicinal metabolites of the Tulsi genome which were studied in detail. </p>  </div> <div class="media" title="media"> <a href="12870_2015_562_MOESM21_ESM.xls">LINK</a> <span class="label" tagx="label" title="label">Additional file 21: Table S9.</span>  <p>Gene IDs involved in specialized metabolite production for each of the metabolites with known pathways. </p>  </div> <div class="media" title="media"> <a href="12870_2015_562_MOESM22_ESM.doc">LINK</a> <span class="label" tagx="label" title="label">Additional file 22: Table S10.</span>  <p>Sequences of putative terpene synthases in <i>O. tenuiflorum</i> genome. </p>  </div> <div class="media" title="media"> <a href="12870_2015_562_MOESM23_ESM.png">LINK</a> <span class="label" tagx="label" title="label">Additional file 23: Figure S13.</span>  <p> <b>a.</b> Sequence alignment of metabolite protein predicted from Scaffold 14352 from <i>Ocimum</i> and O65402 protein sequence from <i>Arabidopsis.</i> <b>b.</b> Sequence alignment of protein sequence predicted in scaffold16333 from <i>Ocimum</i> genome and Q8RWT0 protein sequence from <i>Arabidopsis.</i> <b>c.</b> Sequence alignment of protein sequence predicted in scaffold2032 from <i>Ocimum</i> genome and F1T282 protein sequence from <i>Vitis</i> proteome. </p>  </div> <div class="media" title="media"> <a href="12870_2015_562_MOESM24_ESM.doc">LINK</a> <span class="label" tagx="label" title="label">Additional file 24:</span>  <p> <b>Text A.</b> List of scaffolds as marked from START to END in Fig. <a href="#Fig4">4</a>. Text B. List of IDs of transcripts more abundant in Krishna as compared to Rama subtype (from top to bottom) marked in Fig. <a href="#Fig5">5a</a>. Text C. List of IDs of transcripts more abundant in Rama as compared to Krishna subtype (from top to bottom) as marked in Fig. <a href="#Fig5">5b</a>. Text D. Multiple sequence alignment of amyrin synthases with hits in Tulsi genome. </p>  </div> <div class="media" title="media"> <a href="12870_2015_562_MOESM25_ESM.xls">LINK</a> <span class="label" tagx="label" title="label">Additional file 25: Table S11.</span>  <p>Metabolites with unknown pathways with their disease implications. There are 15 medicinally relevant metabolites in <i>Ocimum sp.</i> with unknown pathways. </p>  </div> <p /> </div> </div> </div> <div class="glossary" title="glossary"> <div class="title" tagx="title" title="title">
Abbreviations</div> <div class="def-list" title="def-list"> <div class="def-item" title="def-item"> <div class="term" title="term">
PE</div> <div class="def" title="def"> <p>Paired end</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
MP</div> <div class="def" title="def"> <p>Mate paired</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
CEGMA</div> <div class="def" title="def"> <p>Core eukaryotic genes mapping approach</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
DEG</div> <div class="def" title="def"> <p>Database of essential genes</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
LTR</div> <div class="def" title="def"> <p>Long terminal repeats</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
Ote</div> <div class="def" title="def"> <p> <i>Ocimum tenuiflorum</i> </p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
Ath</div> <div class="def" title="def"> <p> <i>Arabidopsis thaliana</i> </p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
Mgu</div> <div class="def" title="def"> <p> <i>Mimulus guttatus</i> </p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
Sly</div> <div class="def" title="def"> <p> <i>Solanum lycopersicum</i> </p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
Osa</div> <div class="def" title="def"> <p> <i>Oryza sativa</i> (Osa) </p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SSR</div> <div class="def" title="def"> <p>Simple sequence repeats</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
CHS</div> <div class="def" title="def"> <p>chalcone synthase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
RbcS</div> <div class="def" title="def"> <p>Rubisco small subunit</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
MSA</div> <div class="def" title="def"> <p>Multiple sequence alignment</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
NJ</div> <div class="def" title="def"> <p>Neighbor joining</p> </div> </div> </div> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p> <b>Competing interests</b> </p> <p>The authors declare that they have no competing interests.</p> </div> <div class="fn-type-" title=""> <p> <b>Authors’ contributions</b> </p> <p>Conceived and designed experiments: RS, SR, MG. Data generation: NP, CS, RM, SS, MS, AR, SN, MN. Data analysis and presentation: AKU, AS, MS, KH, EM, APJ, AGJ, OKM, PNS, USR, SM, SK, AG, NK, HRS, ARC, SDK, JM, PG, MM, MSS, HRS, SN, SNP. Writing of the manuscript: AKU, AS, MS, KH, EM, APJ, AGJ, OKM, PNS, USR, SM, SK, AG, NK, HRS, ARC, SDK, JM, PG, MSS, HRS, SN, SNP. Provided resources and tools and critical reviewed manuscript: RS, MG. All authors read and approved the final manuscript. </p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We thank Ms. R. Savithri and Dr. Anna Spudich for useful discussions. We thank NCBS and InStem for infrastructure and other facilities. Sequencing was done at C-CAMP (BT/PR3481/INF/22/140/2011). We thank Mr. Kannan for metabolite confirmation done at C-CAMP. We acknowledge the financial support of NCBS scholarship to AU and RS, Glue Grant (BT/PR15352/MED/15/70/2011) funded by the Department of Biotechnology, India, to AC., Centre of Excellence Grant (BT/01/COE/09/01) funded by the Department of Biotechnology, India to AG, OKM, AS, SN, SNP and MS., University Grants Commission Scholarship, India to PG and Kothari fellowship to MM, Bridge postdoctoral fellows, NCBS and Instem India to KH, APJ, MN, PS and MSS., Indo-Japan Grant (BT/IC/ Japan(BI)/01/2010) funded by the Department of Biotechnology, India to KH., Council of Scientific and Industrial Research, India to AGJ, SDK, JM, AS and USR, Department of Biotechnology Scholarship, India to SK, EM, PS and SM, Extramural Grant (37/1606/13/EMR-II) funded by Council of Scientific and Industrial Research, India to SN., InStem to SR, Department of Biotechnology, Government of India (Ramalingaswami Fellowship Grant; BT/HRD/35/02/2006) to MG. </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="CR1" /> <span class="label" tagx="label" title="label">1.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Paton A, Harley RM, and Harley MM. Ocimum-an overview of relationships and classification. Ocimum Aromatic Plants-Industrial Profiles. Amsterdam: Harwood Academic (1999) </span> </li> <li tag="ref"> <a name="CR2" /> <span class="label" tagx="label" title="label">2.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Willis</span> <span class="given-names" tagx="given-names" title="given-names">JC</span> </span> </span> <span class="source" tagx="source" title="source">A dictionary of the flowering plants and ferns, by J. C. Willis</span> <span class="year" tagx="year" title="year">1919</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Cambridge</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">The University Press</span> </span> </li> <li tag="ref"> <a name="CR3" /> <span class="label" tagx="label" title="label">3.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Prakash</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gupta</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Therapeutic uses of Ocimum sanctum Linn (Tulsi) with a note on eugenol and its pharmacological actions: a short review </span> <span class="source" tagx="source" title="source">Indian J Physiol Pharmacol</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">49</span> <span class="fpage" tagx="fpage" title="fpage">125</span> <span class="lpage" tagx="lpage" title="lpage">31</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16170979">16170979</a> </span> </span> </li> <li tag="ref"> <a name="CR4" /> <span class="label" tagx="label" title="label">4.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Khare CP. Indian medicinal plants: an illustrated dictionary. Springer Science &amp;amp; Business Media; 2007. </span> </li> <li tag="ref"> <a name="CR5" /> <span class="label" tagx="label" title="label">5.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rao</span> <span class="given-names" tagx="given-names" title="given-names">PS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Satelli</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moridani</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jenkins</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rao</span> <span class="given-names" tagx="given-names" title="given-names">US</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Luteolin induces apoptosis in multidrug resistant cancer cells without affecting the drug transporter function: involvement of cell line-specific apoptotic mechanisms </span> <span class="source" tagx="source" title="source">Int J Cancer</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">130</span> <span class="fpage" tagx="fpage" title="fpage">2703</span> <span class="lpage" tagx="lpage" title="lpage">14</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/ijc.26308">10.1002/ijc.26308</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21792893">21792893</a> </span> </span> </li> <li tag="ref"> <a name="CR6" /> <span class="label" tagx="label" title="label">6.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Góngora-castillo</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fedewa</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yeo</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chappell</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dellapenna</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buell</span> <span class="given-names" tagx="given-names" title="given-names">CR</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Genomic approaches for interrogating the biochemistry of medicinal plant species</span> <span class="source" tagx="source" title="source">Methods Enzymol</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">517</span> <span class="fpage" tagx="fpage" title="fpage">139</span> <span class="lpage" tagx="lpage" title="lpage">159</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/B978-0-12-404634-4.00007-3">10.1016/B978-0-12-404634-4.00007-3</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23084937">23084937</a> </span> </span> </li> <li tag="ref"> <a name="CR7" /> <span class="label" tagx="label" title="label">7.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rastogi</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meena</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bhattacharya</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ghosh</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shukla</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sangwan</span> <span class="given-names" tagx="given-names" title="given-names">NS</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">De novo sequencing and comparative analysis of holy and sweet basil transcriptomes</span> <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">15</span> <span class="fpage" tagx="fpage" title="fpage">588</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-15-588">10.1186/1471-2164-15-588</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25015319">25015319</a> </span> </span> </li> <li tag="ref"> <a name="CR8" /> <span class="label" tagx="label" title="label">8.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carović-Stanko</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liber</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Besendorfer</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Javornik</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bohanec</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kolak</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Genetic relations among basil taxa (Ocimum L.) based on molecular markers, nuclear DNA content, and chromosome number </span> <span class="source" tagx="source" title="source">Plant Syst Evol</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">285</span> <span class="fpage" tagx="fpage" title="fpage">13</span> <span class="lpage" tagx="lpage" title="lpage">22</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00606-009-0251-z">10.1007/s00606-009-0251-z</a> </span> </span> </li> <li tag="ref"> <a name="CR9" /> <span class="label" tagx="label" title="label">9.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">MORTON</span> <span class="given-names" tagx="given-names" title="given-names">JK</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Cytotaxonomic studies on the West African Labiatae</span> <span class="source" tagx="source" title="source">J Linn Soc London, Bot</span> <span class="year" tagx="year" title="year">1962</span> <span class="volume" tagx="volume" title="volume">58</span> <span class="fpage" tagx="fpage" title="fpage">231</span> <span class="lpage" tagx="lpage" title="lpage">283</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1095-8339.1962.tb00896.x">10.1111/j.1095-8339.1962.tb00896.x</a> </span> </span> </li> <li tag="ref"> <a name="CR10" /> <span class="label" tagx="label" title="label">10.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Khosla M, Sobti SN. Karyomorphological Studies in Genus Ocimum II. Sanctum Group. Cytologia 50: 253-263, 1935. </span> </li> <li tag="ref"> <a name="CR11" /> <span class="label" tagx="label" title="label">11.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Parra</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bradnam</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Korf</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">CEGMA: a pipeline to accurately annotate core genes in eukaryotic genomes</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">23</span> <span class="fpage" tagx="fpage" title="fpage">1061</span> <span class="lpage" tagx="lpage" title="lpage">7</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/btm071">10.1093/bioinformatics/btm071</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17332020">17332020</a> </span> </span> </li> <li tag="ref"> <a name="CR12" /> <span class="label" tagx="label" title="label">12.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ou</span> <span class="given-names" tagx="given-names" title="given-names">H-Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">C-T</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">DEG: a database of essential genes</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">32</span> <span class="issue" tagx="issue" title="issue">Database issue</span> <span class="fpage" tagx="fpage" title="fpage">D271</span> <span class="lpage" tagx="lpage" title="lpage">2</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkh024">10.1093/nar/gkh024</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14681410">14681410</a> </span> </span> </li> <li tag="ref"> <a name="CR13" /> <span class="label" tagx="label" title="label">13.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Apweiler</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bairoch</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">CH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barker</span> <span class="given-names" tagx="given-names" title="given-names">WC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boeckmann</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ferro</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">UniProt: the Universal Protein knowledgebase</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">32</span> <span class="issue" tagx="issue" title="issue">Database issue</span> <span class="fpage" tagx="fpage" title="fpage">D115</span> <span class="lpage" tagx="lpage" title="lpage">9</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkh131">10.1093/nar/gkh131</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14681372">14681372</a> </span> </span> </li> <li tag="ref"> <a name="CR14" /> <span class="label" tagx="label" title="label">14.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ferrer</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jez</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bowman</span> <span class="given-names" tagx="given-names" title="given-names">ME</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dixon</span> <span class="given-names" tagx="given-names" title="given-names">RA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Noel</span> <span class="given-names" tagx="given-names" title="given-names">JP</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Structure of chalcone synthase and the molecular basis of plant polyketide biosynthesis</span> <span class="source" tagx="source" title="source">Nat Struct Biol</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">775</span> <span class="lpage" tagx="lpage" title="lpage">84</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/11553">10.1038/11553</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10426957">10426957</a> </span> </span> </li> <li tag="ref"> <a name="CR15" /> <span class="label" tagx="label" title="label">15.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Portnoy</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Benyamini</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bar</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harel-Beja</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gepstein</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Giovannoni</span> <span class="given-names" tagx="given-names" title="given-names">JJ</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The molecular and biochemical basis for varietal variation in sesquiterpene content in melon (Cucumis melo L.) rinds </span> <span class="source" tagx="source" title="source">Plant Mol Biol</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">66</span> <span class="fpage" tagx="fpage" title="fpage">647</span> <span class="lpage" tagx="lpage" title="lpage">61</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11103-008-9296-6">10.1007/s11103-008-9296-6</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18264780">18264780</a> </span> </span> </li> <li tag="ref"> <a name="CR16" /> <span class="label" tagx="label" title="label">16.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Plant Metabolic Network (PMN), <a href="http://www.plantcyc.org/tools/tools_overview.faces">http://www.plantcyc.org/tools/tools_overview.faces</a> on www.plantcyc.org, February 28, 2008. [http://www.plantcyc.org/about/citing_pmn.faces]. Date Accessed 05-08-2014. </span> </li> <li tag="ref"> <a name="CR17" /> <span class="label" tagx="label" title="label">17.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brendolise</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yauk</span> <span class="given-names" tagx="given-names" title="given-names">Y-K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eberhard</span> <span class="given-names" tagx="given-names" title="given-names">ED</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chagne</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Andre</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">An unusual plant triterpene synthase with predominant α-amyrin-producing activity identified by characterizing oxidosqualene cyclases from Malus × domestica </span> <span class="source" tagx="source" title="source">FEBS J</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">278</span> <span class="fpage" tagx="fpage" title="fpage">2485</span> <span class="lpage" tagx="lpage" title="lpage">99</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1742-4658.2011.08175.x">10.1111/j.1742-4658.2011.08175.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21575133">21575133</a> </span> </span> </li> <li tag="ref"> <a name="CR18" /> <span class="label" tagx="label" title="label">18.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Poralla</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hewelt</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Prestwich</span> <span class="given-names" tagx="given-names" title="given-names">GD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Abe</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reipen</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sprenger</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A specific amino acid repeat in squalene and oxidosqualene cyclases</span> <span class="source" tagx="source" title="source">Trends Biochem Sci</span> <span class="year" tagx="year" title="year">1994</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">157</span> <span class="lpage" tagx="lpage" title="lpage">8</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/0968-0004(94)90276-3">10.1016/0968-0004(94)90276-3</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8016864">8016864</a> </span> </span> </li> <li tag="ref"> <a name="CR19" /> <span class="label" tagx="label" title="label">19.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Abe</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Prestwich</span> <span class="given-names" tagx="given-names" title="given-names">GD</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Identification of the active site of vertebrate oxidosqualene cyclase</span> <span class="source" tagx="source" title="source">Lipids</span> <span class="year" tagx="year" title="year">1995</span> <span class="volume" tagx="volume" title="volume">30</span> <span class="fpage" tagx="fpage" title="fpage">231</span> <span class="lpage" tagx="lpage" title="lpage">234</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/BF02537826">10.1007/BF02537826</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7791531">7791531</a> </span> </span> </li> <li tag="ref"> <a name="CR20" /> <span class="label" tagx="label" title="label">20.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Kushiro T, Shibuya M, Masuda K, Ebizuka Y. Mutational studies on triterpene synthases: engineering lupeol synthase into -amyrin synthase. 2000:6816–6824. </span> </li> <li tag="ref"> <a name="CR21" /> <span class="label" tagx="label" title="label">21.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bast</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rani</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meena</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Chloroplast DNA phylogeography of holy basil (Ocimum tenuiflorum) in Indian subcontinent</span> <span class="source" tagx="source" title="source">ScientificWorldJournal</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">2014</span> <span class="fpage" tagx="fpage" title="fpage">847482</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1155/2014/847482">10.1155/2014/847482</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24523650">24523650</a> </span> </span> </li> <li tag="ref"> <a name="CR22" /> <span class="label" tagx="label" title="label">22.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feng</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiao</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xia</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Cassava genome from a wild ancestor to cultivated varieties</span> <span class="source" tagx="source" title="source">Nat Commun</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">5110</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/ncomms6110">10.1038/ncomms6110</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25300236">25300236</a> </span> </span> </li> <li tag="ref"> <a name="CR23" /> <span class="label" tagx="label" title="label">23.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Ghosh P, Chattopadhyay SK, Adhikari S, Saha S, Mondal S. A high throughput DNA extraction method from chemotypically heterogeneous plant species. P. 2013;2603. </span> </li> <li tag="ref"> <a name="CR24" /> <span class="label" tagx="label" title="label">24.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Lab H. FASTX toolkit. <a href="http://hannonlab.cshl.edu/fastx_toolkit/index.html">http://hannonlab.cshl.edu/fastx_toolkit/index.html</a>. </span> </li> <li tag="ref"> <a name="CR25" /> <span class="label" tagx="label" title="label">25.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Andrews S. FastQC a quality control tool for high throughput sequence data. <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>. </span> </li> <li tag="ref"> <a name="CR26" /> <span class="label" tagx="label" title="label">26.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xie</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Luo</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Patterson</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">SOAPdenovo-Trans: de novo transcriptome assembly with short RNA-Seq reads</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">30</span> <span class="fpage" tagx="fpage" title="fpage">1660</span> <span class="lpage" tagx="lpage" title="lpage">6</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/btu077">10.1093/bioinformatics/btu077</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24532719">24532719</a> </span> </span> </li> <li tag="ref"> <a name="CR27" /> <span class="label" tagx="label" title="label">27.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cantarel</span> <span class="given-names" tagx="given-names" title="given-names">BL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Korf</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Robb</span> <span class="given-names" tagx="given-names" title="given-names">SMC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Parra</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ross</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moore</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">MAKER: an easy-to-use annotation pipeline designed for emerging model organism genomes</span> <span class="source" tagx="source" title="source">Genome Res</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">18</span> <span class="fpage" tagx="fpage" title="fpage">188</span> <span class="lpage" tagx="lpage" title="lpage">96</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gr.6743907">10.1101/gr.6743907</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18025269">18025269</a> </span> </span> </li> <li tag="ref"> <a name="CR28" /> <span class="label" tagx="label" title="label">28.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stanke</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morgenstern</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">AUGUSTUS: a web server for gene prediction in eukaryotes that allows user-defined constraints </span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">33</span> <span class="issue" tagx="issue" title="issue">Web Server issue</span> <span class="fpage" tagx="fpage" title="fpage">W465</span> <span class="lpage" tagx="lpage" title="lpage">7</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gki458">10.1093/nar/gki458</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15980513">15980513</a> </span> </span> </li> <li tag="ref"> <a name="CR29" /> <span class="label" tagx="label" title="label">29.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hunt</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kikuchi</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sanders</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Newbold</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Berriman</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Otto</span> <span class="given-names" tagx="given-names" title="given-names">TD</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">REAPR: a universal tool for genome assembly evaluation</span> <span class="source" tagx="source" title="source">Genome Biol</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">R47</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/gb-2013-14-5-r47">10.1186/gb-2013-14-5-r47</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23710727">23710727</a> </span> </span> </li> <li tag="ref"> <a name="CR30" /> <span class="label" tagx="label" title="label">30.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Heng</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bob</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alec</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tim</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jue Ruan</span> <span class="given-names" tagx="given-names" title="given-names">NH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gabor</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The Sequence Alignment/Map format and SAMtools.pdf</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">2078</span> <span class="lpage" tagx="lpage" title="lpage">2079</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/btp352">10.1093/bioinformatics/btp352</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943">19505943</a> </span> </span> </li> <li tag="ref"> <a name="CR31" /> <span class="label" tagx="label" title="label">31.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marçais</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kingsford</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A fast, lock-free approach for efficient parallel counting of occurrences of k-mers</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">27</span> <span class="fpage" tagx="fpage" title="fpage">764</span> <span class="lpage" tagx="lpage" title="lpage">770</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/btr011">10.1093/bioinformatics/btr011</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21217122">21217122</a> </span> </span> </li> <li tag="ref"> <a name="CR32" /> <span class="label" tagx="label" title="label">32.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tatusov</span> <span class="given-names" tagx="given-names" title="given-names">RL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fedorova</span> <span class="given-names" tagx="given-names" title="given-names">ND</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jackson</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jacobs</span> <span class="given-names" tagx="given-names" title="given-names">AR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kiryutin</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koonin</span> <span class="given-names" tagx="given-names" title="given-names">EV</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The COG database: an updated version includes eukaryotes</span> <span class="source" tagx="source" title="source">BMC Bioinformatics</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">41</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2105-4-41">10.1186/1471-2105-4-41</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12969510">12969510</a> </span> </span> </li> <li tag="ref"> <a name="CR33" /> <span class="label" tagx="label" title="label">33.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Altschul</span> <span class="given-names" tagx="given-names" title="given-names">SF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Madden</span> <span class="given-names" tagx="given-names" title="given-names">TL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schaffer</span> <span class="given-names" tagx="given-names" title="given-names">AA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miller</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">3389</span> <span class="lpage" tagx="lpage" title="lpage">3402</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/25.17.3389">10.1093/nar/25.17.3389</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694">9254694</a> </span> </span> </li> <li tag="ref"> <a name="CR34" /> <span class="label" tagx="label" title="label">34.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Price</span> <span class="given-names" tagx="given-names" title="given-names">AL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jones</span> <span class="given-names" tagx="given-names" title="given-names">NC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pevzner</span> <span class="given-names" tagx="given-names" title="given-names">PA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">De novo identification of repeat families in large genomes</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">21</span> <span class="issue" tagx="issue" title="issue">Suppl 1</span> <span class="fpage" tagx="fpage" title="fpage">i351</span> <span class="lpage" tagx="lpage" title="lpage">8</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/bti1018">10.1093/bioinformatics/bti1018</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15961478">15961478</a> </span> </span> </li> <li tag="ref"> <a name="CR35" /> <span class="label" tagx="label" title="label">35.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">RepeatMasker open-3.0.;1996–2010. <a href="http://www.repeatmasker.org/">http://www.repeatmasker.org</a>. [ <a href="http://www.repeatmasker.org/faq.html#faq3">http://www.repeatmasker.org/faq.html#faq3</a>] </span> </li> <li tag="ref"> <a name="CR36" /> <span class="label" tagx="label" title="label">36.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Slater</span> <span class="given-names" tagx="given-names" title="given-names">GSC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Birney</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Automated generation of heuristics for biological sequence comparison</span> <span class="source" tagx="source" title="source">BMC Bioinformatics</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">31</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2105-6-31">10.1186/1471-2105-6-31</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15713233">15713233</a> </span> </span> </li> <li tag="ref"> <a name="CR37" /> <span class="label" tagx="label" title="label">37.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mistry</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Finn</span> <span class="given-names" tagx="given-names" title="given-names">RD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eddy</span> <span class="given-names" tagx="given-names" title="given-names">SR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bateman</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Punta</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Challenges in homology search: HMMER3 and convergent evolution of coiled-coil regions</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">41</span> <span class="fpage" tagx="fpage" title="fpage">e121</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkt263">10.1093/nar/gkt263</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23598997">23598997</a> </span> </span> </li> <li tag="ref"> <a name="CR38" /> <span class="label" tagx="label" title="label">38.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stoeckert</span> <span class="given-names" tagx="given-names" title="given-names">CJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Roos</span> <span class="given-names" tagx="given-names" title="given-names">DS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">OrthoMCL: identification of ortholog groups for eukaryotic genomes</span> <span class="source" tagx="source" title="source">Genome Res</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="fpage" tagx="fpage" title="fpage">2178</span> <span class="lpage" tagx="lpage" title="lpage">89</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gr.1224503">10.1101/gr.1224503</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12952885">12952885</a> </span> </span> </li> <li tag="ref"> <a name="CR39" /> <span class="label" tagx="label" title="label">39.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lechner</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Findeiss</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Steiner</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marz</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stadler</span> <span class="given-names" tagx="given-names" title="given-names">PF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Prohaska</span> <span class="given-names" tagx="given-names" title="given-names">SJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Proteinortho: detection of (co-) orthologs in large-scale analysis</span> <span class="source" tagx="source" title="source">BMC Bioinformatics</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="fpage" tagx="fpage" title="fpage">124</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2105-12-124">10.1186/1471-2105-12-124</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21526987">21526987</a> </span> </span> </li> <li tag="ref"> <a name="CR40" /> <span class="label" tagx="label" title="label">40.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goodstein</span> <span class="given-names" tagx="given-names" title="given-names">DM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Howson</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Neupane</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hayes</span> <span class="given-names" tagx="given-names" title="given-names">RD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fazo</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Phytozome: a comparative platform for green plant genomics</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">40</span> <span class="issue" tagx="issue" title="issue">Database issue</span> <span class="fpage" tagx="fpage" title="fpage">D1178</span> <span class="lpage" tagx="lpage" title="lpage">86</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkr944">10.1093/nar/gkr944</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22110026">22110026</a> </span> </span> </li> <li tag="ref"> <a name="CR41" /> <span class="label" tagx="label" title="label">41.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Larkin</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blackshields</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brown</span> <span class="given-names" tagx="given-names" title="given-names">NP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chenna</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McGettigan</span> <span class="given-names" tagx="given-names" title="given-names">PA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McWilliam</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Clustal W and Clustal X version 2.0</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">23</span> <span class="fpage" tagx="fpage" title="fpage">2947</span> <span class="lpage" tagx="lpage" title="lpage">8</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/btm404">10.1093/bioinformatics/btm404</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17846036">17846036</a> </span> </span> </li> <li tag="ref"> <a name="CR42" /> <span class="label" tagx="label" title="label">42.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Felsenstein</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">PHYLIP - phylogeny inference package (version 3.2)</span> <span class="source" tagx="source" title="source">Cladistics</span> <span class="year" tagx="year" title="year">1989</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">164</span> <span class="lpage" tagx="lpage" title="lpage">166</span> </span> </li> <li tag="ref"> <a name="CR43" /> <span class="label" tagx="label" title="label">43.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kent</span> <span class="given-names" tagx="given-names" title="given-names">WJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">BLAT---the BLAST-like alignment tool</span> <span class="source" tagx="source" title="source">Genome Res</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="fpage" tagx="fpage" title="fpage">656</span> <span class="lpage" tagx="lpage" title="lpage">664</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gr.229202.ArticlepublishedonlinebeforeMarch2002">10.1101/gr.229202.ArticlepublishedonlinebeforeMarch2002</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11932250">11932250</a> </span> </span> </li> <li tag="ref"> <a name="CR44" /> <span class="label" tagx="label" title="label">44.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pertea</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Trapnell</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pimentel</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kelley</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salzberg</span> <span class="given-names" tagx="given-names" title="given-names">SL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions </span> <span class="source" tagx="source" title="source">Genome Biol</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">R36</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/gb-2013-14-4-r36">10.1186/gb-2013-14-4-r36</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23618408">23618408</a> </span> </span> </li> <li tag="ref"> <a name="CR45" /> <span class="label" tagx="label" title="label">45.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Langmead</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salzberg</span> <span class="given-names" tagx="given-names" title="given-names">SL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Fast gapped-read alignment with Bowtie 2</span> <span class="source" tagx="source" title="source">Nat Methods</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">357</span> <span class="lpage" tagx="lpage" title="lpage">359</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nmeth.1923">10.1038/nmeth.1923</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286">22388286</a> </span> </span> </li> <li tag="ref"> <a name="CR46" /> <span class="label" tagx="label" title="label">46.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Godzik</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences </span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">22</span> <span class="fpage" tagx="fpage" title="fpage">1658</span> <span class="lpage" tagx="lpage" title="lpage">659</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/btl158">10.1093/bioinformatics/btl158</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16731699">16731699</a> </span> </span> </li> <li tag="ref"> <a name="CR47" /> <span class="label" tagx="label" title="label">47.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Góngora-Castillo</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buell</span> <span class="given-names" tagx="given-names" title="given-names">CR</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Bioinformatics challenges in de novo transcriptome assembly using short read sequences in the absence of a reference genome sequence </span> <span class="source" tagx="source" title="source">Nat Prod Rep</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">30</span> <span class="fpage" tagx="fpage" title="fpage">490</span> <span class="lpage" tagx="lpage" title="lpage">500</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1039/c3np20099j">10.1039/c3np20099j</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23377493">23377493</a> </span> </span> </li> <li tag="ref"> <a name="CR48" /> <span class="label" tagx="label" title="label">48.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wong</span> <span class="given-names" tagx="given-names" title="given-names">WH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">SeqMap: mapping massive amount of oligonucleotides to the genome</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">24</span> <span class="fpage" tagx="fpage" title="fpage">2395</span> <span class="lpage" tagx="lpage" title="lpage">6</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/btn429">10.1093/bioinformatics/btn429</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18697769">18697769</a> </span> </span> </li> <li tag="ref"> <a name="CR49" /> <span class="label" tagx="label" title="label">49.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salzman</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wong</span> <span class="given-names" tagx="given-names" title="given-names">WH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Statistical modeling of RNA-seq data</span> <span class="source" tagx="source" title="source">Stat Sci</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">26</span> <span class="fpage" tagx="fpage" title="fpage">62</span> <span class="lpage" tagx="lpage" title="lpage">83</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1214/10-STS343">10.1214/10-STS343</a> </span> </span> </li> </ul> </div> </div>  </body></html>