<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">BMC Genomics</span> <span class="iso-abbrev" title="iso-abbrev">BMC Genomics</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">BMC Genomics</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1471-2164</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/4445982">4445982</a> </span> <span class="publisher-id" title="publisher-id">1640</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1186/s12864-015-1640-z">10.1186/s12864-015-1640-z</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Unravelling the genome of Holy basil: an "incomparable" "elixir of life" of traditional Indian medicine </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rastogi</span> <span class="given-names" tagx="given-names" title="given-names">Shubhra</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">shubhra.rastogi81@gmail.com</span> </span> <a href="#Aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kalra</span> <span class="given-names" tagx="given-names" title="given-names">Alok</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">a.kalra@cimap.res.in</span> </span> <a href="#Aff2" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gupta</span> <span class="given-names" tagx="given-names" title="given-names">Vikrant</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">v.gupta@cimap.res.in</span> </span> <a href="#Aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khan</span> <span class="given-names" tagx="given-names" title="given-names">Feroz</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">f.khan@cimap.res.in</span> </span> <a href="#Aff3" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lal</span> <span class="given-names" tagx="given-names" title="given-names">Raj Kishori</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">rk.lal@cimap.res.in</span> </span> <a href="#Aff4" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tripathi</span> <span class="given-names" tagx="given-names" title="given-names">Anil Kumar</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">tripathianil@rediffmail.com</span> </span> <a href="#Aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Parameswaran</span> <span class="given-names" tagx="given-names" title="given-names">Sriram</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">sriram.p@genotypic.co.in</span> </span> <a href="#Aff5" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gopalakrishnan</span> <span class="given-names" tagx="given-names" title="given-names">Chellappa</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">chellappa@genotypic.co.in</span> </span> <a href="#Aff5" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ramaswamy</span> <span class="given-names" tagx="given-names" title="given-names">Gopalakrishna</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">gopalakrishna.r@genotypic.co.in</span> </span> <a href="#Aff5" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shasany</span> <span class="given-names" tagx="given-names" title="given-names">Ajit Kumar</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">ak.shasany@cimap.res.in</span> </span> <a href="#Aff1" /> </span> <span class="citation_author_institution" id="Aff1">[], </span> <span class="citation_author_institution" id="Aff2">[], </span> <span class="citation_author_institution" id="Aff3">[], </span> <span class="citation_author_institution" id="Aff4">[], </span> <span class="citation_author_institution" id="Aff5">[], </span> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2015-5-5</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2015-5-5</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2015</span> </span> <span class="volume" tagx="volume" title="volume">16</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">413</span> <span class="history" title="history"> <span class="received" title="received">received: 2014-12-1</span> <span class="accepted" title="accepted">accepted: 2015-5-18</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License ( <a href="http://creativecommons.org/licenses/by/4.0">http://creativecommons.org/licenses/by/4.0</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly credited. The Creative Commons Public Domain Dedication waiver ( <a href="http://creativecommons.org/publicdomain/zero/1.0/">http://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p> <i>Ocimum sanctum</i> L. ( <i>O. tenuiflorum</i>) family-Lamiaceae is an important component of Indian tradition of medicine as well as culture around the world, and hence is known as "Holy basil" in India. This plant is mentioned in the ancient texts of Ayurveda as an "elixir of life" (life saving) herb and worshipped for over 3000 years due to its healing properties. Although used in various ailments, validation of molecules for differential activities is yet to be fully analyzed, as about 80 % of the patents on this plant are on extracts or the plant parts, and mainly focussed on essential oil components. With a view to understand the full metabolic potential of this plant whole nuclear and chloroplast genomes were sequenced for the first time combining the sequence data from 4 libraries and three NGS platforms. </p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>The saturated draft assembly of the genome was about 386 Mb, along with the plastid genome of 142,245 bp, turning out to be the smallest in Lamiaceae. In addition to SSR markers, 136 proteins were identified as homologous to five important plant genomes. Pathway analysis indicated an abundance of phenylpropanoids in <i>O. sanctum</i>. Phylogenetic analysis for chloroplast proteome placed <i>Salvia miltiorrhiza</i> as the nearest neighbor. Comparison of the chemical compounds and genes availability in <i>O. sanctum</i> and <i>S. miltiorrhiza</i> indicated the potential for the discovery of new active molecules. </p> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>The genome sequence and annotation of <i>O. sanctum</i> provides new insights into the function of genes and the medicinal nature of the metabolites synthesized in this plant. This information is highly beneficial for mining biosynthetic pathways for important metabolites in related species. </p> </div> <div class="electronicsupplementarymaterial" title="sec"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <p>The online version of this article (doi:10.1186/s12864-015-1640-z) contains supplementary material, which is available to authorized users. </p> </div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
Chloroplast</div> <div class="kwd" title="kwd">
Mitochondria</div> <div class="kwd" title="kwd"> <i>Ocimum sanctum</i> L </div> <div class="kwd" title="kwd">
Secondary metabolism</div> <div class="kwd" title="kwd">
SSR’s Whole genome sequencing</div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2015" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Background</div> <p> <i>Ocimum sanctum</i> L. ( <i>O. tenuiflorum</i>) is an important sacred medicinal plant of India known as "holy basil", <i>Thulasi</i>, <i>Vishnupriya,</i> and <i>Tulsi</i> [ <a href="#CR1">1</a>] and worshipped for over more than 3000 years [ <a href="#CR2">2</a>, <a href="#CR3"> 3</a>]. This herb is popular in traditional medicine as "The Queen of Herbs," "The Incomparable One," and "The Mother Medicine of Nature" [ <a href="#CR4">4</a>]. Being legendary sacred basil (Tulsi), is recognized [ <a href="#CR5">5</a>, <a href="#CR6"> 6</a>] not only for its sanctity, but forms an indispensible part of the traditional herbal medicine of East as discussed in Ayurvedic text of Charaka Samhita as well as Unani medicinal system. It is native to India and parts of northern and eastern Africa, Hainan Island, and Taiwan, and grows wild throughout India up to an altitude of 5900 ft (1800 m) in the Himalayas [ <a href="#CR7">7</a>– <a href="#CR9">9</a>]. The leaf of the plant owes a stronger, somewhat pungent taste than other basils due to a sesquiterpenoid <i>beta-</i>caryophyllene, and a phenylpropanoid eugenol [ <a href="#CR10">10</a>]. <i>O. sanctum</i> has been suggested to possess anti-fertility, anti-cancer, anti-diabetic, anti-fungal, anti-microbial, cardioprotective, analgesic, anti-spasmodic and adaptogenic actions [ <a href="#CR6">6</a>]. The chemical composition of Tulsi is highly complex, containing many biologically active phytochemicals with variable proportions among varieties or even plants within the same field. The volatile oil of leaf [ <a href="#CR11">11</a>] contains eugenol (1-hydroxy-2-methoxy-4-allylbenzene), euginal, urosolic acid [ <a href="#CR12">12</a>], carvacrol, limatrol, caryophyllene, methyl carvicol while the seed volatile oil has fatty acids and sitosterol. In addition, the seed mucilage contains some levels of sugars and the anthocyans are present in green leaves [ <a href="#CR6">6</a>]. The leaf volatiles (terpenes and phenylpropenes) are synthesized and sequestered in glandular hairs present on the leaves, also known as peltate trichomes, which are the characteristic of lamiaceae members [ <a href="#CR13">13</a>, <a href="#CR14">14</a>]. Two types of <i>O. sanctum</i> L. are used for cultivation: (i) plants with green leaves known as Sri/ Rama Tulsi &amp;amp; (ii) plants with purple leaves known as Krishna/ Shyama Tulsi [ <a href="#CR8">8</a>]. Furthermore, the quantity of many of its constituents can be significantly altered by varying conditions used for growing; harvesting, processing and storage that are not yet well understood [ <a href="#CR15">15</a>]. All of the varieties of <i>Ocimum</i> have unique and individual chemical compositions; but their medicinal properties are not yet explored completely. Despite huge importance of <i>Ocimum</i>, very little transcriptomic and genomic data of <i>Ocimum</i> sp. is available limiting studies on important phytochemical pathways. But comparative transcriptome analysis of <i>Ocimum</i> species ( <i>O. sanctum</i> and <i>O. basilicum</i>) was recently reported [ <a href="#CR16">16</a>]. This report correlated higher digital expression of phenylpropanoid/ terpenoid pathway genes of <i>O. basilicum</i> to higher essential oil content and chromosome number ( <i>O. sanctum</i>, 2n = 16; and <i>O. basilicum</i>, 2n = 48). Also several cytochrome P450s (26) and transcription factor families (40) were identified which could be utilized to characterize genes related to secondary metabolism and its regulation. </p> <p>Hence, there was a need to know about the genome of this plant to understand its metabolic potential, diversity, regulation and evolutionary implications. Here, we report the draft nuclear genome sequence of 386 Mb and the plastid of 142,245 bp sequenced with a composite next generation sequencing technologies. On the basis of assembly, 53,480 protein coding genes were identified. Gene model prediction revealed the similarity of <i>O. sanctum</i> genome to <i>Nicotiana tabacum</i> and <i>Solanum lycopersicum,</i> all sharing same sub-class (asterid) <i>.</i> </p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results and discussion</div> <div class="genomesequencing,assemblyandvalidation" title="sec"> <div class="title" tagx="title" title="title">
Genome sequencing, assembly and validation</div> <p>A whole-genome shotgun sequencing strategy by generating long and short paired-end reads, along with long reads and mate-pair libraries was applied to assemble the 386 Mb genome sequence of <i>O. sanctum.</i> The process workflow of the same has been provided in the Fig. <a href="#Fig1">1</a>. Two libraries (long and short reads) of Illumina HiSeq2000, one library of 454 GS FLX and one mate-pair library of SOLiD 5500XL were constructed. Illumina libraries were used to generate the contigs and Illumina paired-end data along with 454 GS FLX single end data were used for contig merging as a result of which scaffolds were generated. While Illumina generated 45.37 Gb data (Additional file <a href="#MOESM1">1</a>), 454 sequencing resulted in 320.3 Mb (Additional file <a href="#MOESM2">2</a>) data and SOLiD generated 12.68 Gb (Additional file <a href="#MOESM3">3</a>) data representing approximately ~130-fold coverage of the predicted <i>O. sanctum</i> genome. With the help of the two Illumina library data (short insert and long insert) the assembly showed significant improvement in respect of N50. Long- and short-paired end reads from Illumina deep sequencing was used to assemble a total of 107,785 contigs into 22,776 scaffolds. Super-scaffolding was performed in order to merge the existing gap-closed scaffolds into super-scaffolds using relative orientation of SOLiD mate pair reads. Finally, 9059 super-scaffolds of maximum length upto 2,211,552 bp were generated (Table <a href="#Tab1">1</a>). Out of the total super-scaffolds generated, 4159 super-scaffolds are larger than or equal to 1 kb in length (Table <a href="#Tab1">1</a>). The N50 length of contigs, scaffolds and super-scaffolds was found to be 12,769 bp, 61,854 bp and 303,233 bp respectively (Table <a href="#Tab1">1</a>). The total length of the gaps in the assembled scaffolds was 26.11 Mb in a total of 3999 super-scaffolds. The total number of gaps present is 45,803 considering even the presence of a single N as one gap. The biggest gap identified was of 4906 bp in length. Mate-pair reads significantly closed ~60 % of gaps between scaffolds and five-fold change in N50 value and N90 value was also observed where N50 increased from 61,242 to 303,233 while N90 from 12,534 to 73,672. In order to validate the genome assembly, a total of 69,117 transcripts generated from our previous study [ <a href="#CR16">16</a>] were mapped to the genome data and more than 95 % transcripts (66,891) showed 100 % coverage (Additional file <a href="#MOESM4">4</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 1</span> </a> </div>   <p>Process workflow of <i>Ocimum sanctum</i> whole genome sequencing and assembly </p>   </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  <p>Assembly statistics of contigs and scaffolds generated using the three sequencing platforms Illumina HiSeq2000, 454 GS FLX and SOLiD 5500XL </p>  <table> <thead> <tr> <th>Description</th> <th>Contigs</th> <th>Scaffolds</th> <th>GapClosed scaffolds</th> <th>Super-scaffolds</th> </tr> </thead> <tbody> <tr> <td>Contigs generated</td> <td>107785</td> <td>22776</td> <td>22776</td> <td>9059</td> </tr> <tr> <td>Maximum Contig Length</td> <td>115044</td> <td>414711</td> <td>411690</td> <td>2211552</td> </tr> <tr> <td>Minimum Contig Length</td> <td>147</td> <td>200</td> <td>200</td> <td>200</td> </tr> <tr> <td>Average Contig Length</td> <td>3454</td> <td>16984</td> <td>16629</td> <td>44354</td> </tr> <tr> <td>Total Contigs Length</td> <td>372395755</td> <td>386828951</td> <td>378759759</td> <td>401803260</td> </tr> <tr> <td>Total Number of non-ATGC characters</td> <td>0</td> <td>17898452</td> <td>2665702</td> <td>26110056</td> </tr> <tr> <td>Percentage of non-ATGC characters</td> <td>0</td> <td>4.627</td> <td>0.704</td> <td>6.498</td> </tr> <tr> <td>Contigs &amp;gt; = 1 Kb</td> <td>43174</td> <td>14791</td> <td>14769</td> <td>4159</td> </tr> <tr> <td>Contigs &amp;gt; = 10 Kb</td> <td>11594</td> <td>7544</td> <td>7407</td> <td>2357</td> </tr> <tr> <td>N50 value</td> <td>12769</td> <td>61854</td> <td>61242</td> <td>303233</td> </tr> <tr> <td>N90 value</td> <td>2071</td> <td>12742</td> <td>12534</td> <td>73672</td> </tr> </tbody> </table> </div> <p /> </div> <div class="de-novoassemblyofchloroplastandmitochondriagenomedata" title="sec"> <div class="title" tagx="title" title="title"> <i>De-novo</i> assembly of chloroplast and mitochondria genome data </div> <p>The complete chloroplast (cp) genome of <i>O. sanctum</i> is 142,524 bp in length (Fig. <a href="#Fig2">2</a>). Recently, Qian et al [ <a href="#CR17">17</a>] had reported the chloroplast genome of <i>Salvia miltiorrhiza</i> to be the smallest with the exception of <i>Epifagus virginiana</i> [ <a href="#CR18">18</a>] cp genome of order lamiales. But this investigation revealed <i>O. sanctum</i> cp genome to be 8804 bp smaller than <i>S. miltiorrhiza</i> (member of the <i>Ocimum</i> family- lamiaceae) cp genome of length 151,328 bp. Hence <i>O. sanctum</i> cp genome is now reported as the smallest of the Lamiales cp genomes as it is ~8800 bp smaller than <i>E. virginiana</i> cp genome. The overall GC content of the <i>O. sanctum</i> cp genome is 36.2 %, which is similar to the other reported asterid cp genomes [ <a href="#CR17">17</a>, <a href="#CR19"> 19</a>– <a href="#CR22">22</a>]. The <i>O. sanctum</i> cp genome was found to code a total 158 genes, including 43 transfer RNA (tRNA) genes and four ribosomal RNA (rRNA) genes. The aligned reads of cp genome of <i>O. sanctum</i> to other angiosperms (referred in materials and methods section) were assembled into contigs for finally generating the scaffolds using all the Illumina data. Similar procedure was carried out for mitochondrial genome (Additional file <a href="#MOESM5">5</a> and <a href="#MOESM6">6</a>) assembly except considering <i>S. miltiorrhiza</i> as the reference mitochondial genome. A total of 48 scaffolds from 140 contigs from cp genome, and 41 scaffolds from 124 contigs (Additional file <a href="#MOESM5">5</a> and <a href="#MOESM6">6</a>) from the mitochondrial genome got generated. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 2</span> </a> </div>   <p>Gene map of the <i>Ocimum sanctum</i> chloroplast genome Genes drawn inside the circle are transcribed clockwise, and those outside are counterclockwise. Genes belonging to different functional groups are color-coded. The darker gray in the inner circle corresponds to GC content, while the lighter gray corresponds to AT content </p>   </div> <p /> </div> <div class="genomiccompositionandssrprediction" title="sec"> <div class="title" tagx="title" title="title">
Genomic composition and SSR prediction</div> <p>GC content is an important indicator of the genomic composition including evolution, gene structure (intron size and number), gene regulation and stability of DNA [ <a href="#CR23">23</a>]. Average GC content of <i>O. sanctum</i> was 38.37 %. Earlier researchers have reported that across the broad phylogenetic sweep, genome size may be correlated with intron size [ <a href="#CR24">24</a>– <a href="#CR26">26</a>], suggesting that some fraction of genome size evolution takes place within genes [ <a href="#CR27">27</a>]. While performing the annotation of gene models, taking <i>N. tabacum</i> and <i>S. lycopersicum</i> as references, it was found that the percent genes containing introns from these plants were 55.5 % and 64.5 %, respectively (Additional file <a href="#MOESM7">7</a>). It has been observed that introns and their positions are highly conserved during land plant evolution excluding conifers [ <a href="#CR28">28</a>, <a href="#CR29">29</a>]. </p> <p>Comparative studies had revealed that intron lengths and the abundance of mobile repetitive elements have a direct correlation with genome size, such that large genomes have longer introns and a higher proportion of mobile elements [ <a href="#CR30">30</a>, <a href="#CR31">31</a>]. Intron sizes in the genes of <i>O. sanctum</i> ranged from 5 bp to 8000 bp (Additional file <a href="#MOESM7">7</a>). A reason for intron size variation among organisms may be due to inherent mutational processes generating insertions and deletions [ <a href="#CR24">24</a>, <a href="#CR32">32</a>– <a href="#CR35">35</a>]. It was also reported that low distribution of recombination regions leads to increased intron size [ <a href="#CR36">36</a>, <a href="#CR37">37</a>]. </p> <p>Among different classes of molecular markers, microsatellite or simple sequence repeat (SSR) markers are the most preferred for its array of applications in plant genetics and breeding due to their multi-allelic nature, reproducibility, co-dominant inheritance with high abundance and wide genome coverage [ <a href="#CR38">38</a>, <a href="#CR39">39</a>]. A total of 4827 sequences greater than 500 bp length were examined for SSR search out of which 2612 sequences were found having SSR repeats while 2364 sequences showed the presence of more than one SSR (Fig. <a href="#Fig3">3</a>). A sum of 142,601 SSRs were predicted, with the highest being mono-repeats (85,624) and (13,389) complex SSR’s. The sequences were checked for mono-repeats occurring at-least 10 times, di-repeats occurring at-least 6 times and tri/ tetra/ penta/ hexa-repeats occurring atleast 5 times. The SSR was classified as complex when two SSR’s were present within 100 bp distance of each other. On the other side, 1,166,753 sequences of less than 500 bp were identified, only 162 SSRs highest being mono-repeats (68) followed by complex SSR’s (57) (Fig. <a href="#Fig3">3</a>). Previously, we reported SSRs from the transcriptome of <i>O. sanctum</i> [ <a href="#CR16">16</a>] but the SSRs identified were very few in comparison to the present report. Similarly, efforts were made by researchers towards the development of molecular markers in order to carry out genetic diversity studies on <i>Ocimum sp.</i> [ <a href="#CR40">40</a>– <a href="#CR42">42</a>]. But, SSRs reported from the present study with a large data set would be helpful in providing insights to the plant breeders and geneticists for evaluation of desired genotypes with varied essential oil compositions and also for further development of new species of <i>Ocimum</i>. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 3</span> </a> </div>   <p>Frequency distribution of SSRs based on motif types. p1: Mono-nucleotide repeats; p2: Di-nucleotide repeats; p3: Tri-nucleotide repeats, p4: Tetra-nucleotide repeats; p5: Penta-nucleotide repeats; p6: Hexa-nucleotide repeats; Complex: no. of SSRs involved in compound formation </p>   </div> <p /> <p>A gene density of ~30 genes per 100 kb and ~20 genes per 100 kb was observed in <i>O. sanctum</i> gene model prediction taking <i>N. tabacum</i> (tobacco) and <i>S. lycopersicum</i> (tomato), respectively as references. Since <i>O. sanctum</i> is a small genome plant, the gene density is similar to that of <i>Arabidopsis thaliana</i> i.e., upto 38 genes per 100 kb [ <a href="#CR43">43</a>]. Large genomes like barley and wheat show a gene density of about 5 genes per 23 kb [ <a href="#CR44">44</a>] as it was suggested that the larger genomes would have accumulated non-coding sequences between the single-copy genes [ <a href="#CR45">45</a>]. </p> </div> <div class="genepredictionandannotation" title="sec"> <div class="title" tagx="title" title="title">
Gene prediction and annotation</div> <p>In order to assign putative functions to the predicted genes of <i>O. sanctum</i>, they were compared against the NR (non-redundant) protein sequences of <i>Arabidopsis</i>. The associated hits were searched for their respective GO. Based on sequence homology, 85,723 protein sequences were categorized into 31 functional groups under three main categories: biological processes (BP), cellular components (CC) and molecular functions (MF) (Fig. <a href="#Fig4">4</a>, Additional file <a href="#MOESM8">8</a>). Genes were predicted from 22,776 scaffolds by mapping (BLASTP) the predicted proteins with UNIPROT with all Viridiplantae clade protein sequences. Out of 85,723 protein coding loci from 22,776 scaffolds, a total of 53,480 were annotated with UNIPROT (Additional file <a href="#MOESM9">9</a>) but only 22,270 protein coding genes were found to be unique. On observing the plant species distribution of hits to UNIPROT database, maximum hits were from the plant <i>Genlisea aurea</i> (Additional file <a href="#MOESM10">10</a>) which is one of the smallest known genome among higher plants [ <a href="#CR46">46</a>]. The number of unique protein coding genes (22,270) in the <i>O. sanctum</i> genome, was in range as reported in potato, tomato [ <a href="#CR47">47</a>], neem [ <a href="#CR48">48</a>] and grapevine [ <a href="#CR49">49</a>], having 35,004, 34,727, 20,000 and 30,434 protein coding genes, respectively. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 4</span> </a> </div>   <p>Pie-chart showing top 10 functional classes in each of the 3 categories of gene ontology classification. The three main categories are: biological process, cellular component and molecular function representing the assignment <i>O. sanctum</i> predicted proteins with BLAST matches in NR (non-redundant) protein sequences of <i>Arabidopsis</i> to each GO term </p>   </div> <p /> <p> <i>Ab initio</i> gene model prediction was performed on scaffold sequences utilizing minimal information from the nearest available species. Overall, 130,526 and 87,918 proteins were predicted using training sets of <i>Nicotiana tabacum</i> and <i>Solanum lycopersicum</i> respectively. A total of 65,935 proteins were common between the two predictions. Gene annotation of the predicted proteins with BLASTP resulted in annotation of 80,516 NR proteins. A set of 38,868 of these annotated proteins were common to the predictions from <i>N. tabacum</i> and <i>S. Lycopersicum</i>, respectively. The un-annotated predicted proteins were scanned with Pfam and another 18,940 proteins got annotated with a predicted domain signature. Database annotation of assembled scaffold sequences greater than 500 bp was carried out for matching with the EST/mRNA sequences available for <i>Ocimum</i> in the NCBI databases (Additional file <a href="#MOESM11">11</a>). A total of 23,420 EST and 52 mRNA were queried, with a match to the assembled scaffolds for 21,984 of the EST/mRNA sequences at greater than 90 % sequence identity. Also <i>Arabidopsis</i> sequences (Additional file <a href="#MOESM12">12</a>) from TAIR database and <i>N. tabacum</i> (Additional file <a href="#MOESM13">13</a>) and <i>S. lycopersicum</i> (Additional file <a href="#MOESM14">14</a>) sequences from NCBI were Blast- checked against the <i>O. sanctum</i> scaffolds with percent hitting scaffolds of 34.65 %, 4.9 % and 5.29 %, respectively. Database annotation of EST/mRNA from NCBI datasets identified the mitochondria and chloroplast expressed proteins. All of the 392 scaffolds identified were annotated to potentially map to these sequences (Additional file <a href="#MOESM15">15</a>). Out of 392 scaffolds, 270 were redundant and only 122 were non-redundant. On the basis of annotation of chloroplast and mitochondria encoded proteins against TAIR database, it was found that out of 122 non-redundant scaffolds of <i>O. sanctum</i>, 95 were chloroplastic while remaining 27 were mitochondrial. </p> </div> <div class="phylogeneticanalysis" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic analysis</div> <p> To identify the phylogenetic position of <i>O. sanctum</i> within the asterid lineage, multiple sequence alignments was performed using 63 protein-coding genes (Additional file <a href="#MOESM16">16</a>) commonly present in the 32 complete cp genomes representing 10 families within five orders of asterids including Apiaceae, Araliaceae, Asteraceae, Convolvulaceae, Gesneriaceae, Lamiaceae, Oleaceae, Pedaliaceae, Rubiaceae and Solanaceae (Additional file <a href="#MOESM17">17</a>). Two additional eudicot cp genomes, <i>Spinacia olerace</i>a and <i>Arabidopsis thaliana</i>, were set as outgroups. A phylogenetic tree was generated using maximum parsimony and maximum likelihood method (Fig. <a href="#Fig5">5</a>). Bootstrap analysis showed that there were 25 out of 31 nodes with bootstrap values &amp;gt;95 %, and 14 of these had a bootstrap value of 100 %. The tree topologies formed two major clades, euasterids I and II. The results strongly supported the position of <i>Ocimum sanctum</i> and <i>Salvia miltiorrhiza</i> with 100 % bootstrap from the same family lamiaceae as the sister of the closely related species <i>Sesamum indicum</i> and <i>Boea hygrometrica</i> in the order Lamiales. As the chloroplast genome is considered to be free from evolutionary processes, like gene duplication/ deletion, intensive evolution and pseudogene formation, which are characteristically frequent among nuclear genes, this was included in the phylogenetic analysis [ <a href="#CR50">50</a>]. Slow rate of sequence evolution in chloroplast DNA is appropriate to include cp genome in phylogenetic studies of the highly cross pollinated plant like <i>O. sanctum</i> [ <a href="#CR51">51</a>]. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 5</span> </a> </div>   <p>The phylogenetic tree of the asterid clade based on 63 protein-coding genes of chloroplast genome. Numbers above each node are bootstrap support values. <i>Spinacea oleracea</i> and <i>Arabidopsis thaliana</i> were set as outgroups </p>   </div> <p /> </div> <div class="pathwayidentification" title="sec"> <div class="title" tagx="title" title="title">
Pathway identification</div> <p>To identify the biological pathways functional in <i>O. sanctum</i> 85,723 protein sequences from scaffolds were mapped to the reference canonical pathways in KEGG taking <i>Arabidopsis thaliana</i> and <i>Oryza sativa</i> as reference organisms, out of which only 6328 proteins got predicted in KAAS (Additional file <a href="#MOESM18">18</a>). All transcripts were classified mainly under five categories: metabolism, cellular processes, genetic information processing, environmental information processing and others. Highest numbers of sequences were related to metabolism. Maximum percentage of the sequences fell under the category of phenylpropanoid biosynthesis. <i>O. sanctum</i> is good source of phenylpropene- eugenol and is one of the compounds which attributes to its medicinal property [ <a href="#CR52">52</a>, <a href="#CR53">53</a>]. Precursor molecules for phenylpropanoid biosynthesis are derived from the shikimate pathway while terpenoid biosynthesis utilizes isoprenoid precursors from cytosolic MVA (mevalonate) as well as plastidial MEP pathways (2-Cmethyl-D-erythritol 4-phosphate/1-deoxy-D-xylulose 5-phosphate/non-mevalonate pathways) [ <a href="#CR16">16</a>]. On sorting 53,480 protein coding genes of <i>O. sanctum</i> annotated from UNIPROT for phenylpropanoid (Fig. <a href="#Fig6">6</a>) and terpenoid (MEP and MVA) pathway genes (Fig. <a href="#Fig6">6</a>) it was found that the highest number of phenylpropanoid pathway genes were identified as compared to the terpenes. Since the <i>O. sanctum</i> variety used in the present study is high-yielding, eugenol-rich, oil producing variety [ <a href="#CR54">54</a>] with 83 % eugenol in the oil, it correlates with the presence of higher number of phenylpropanoid pathway genes. Interestingly, highest number transcripts of copalyl diphosphate synthase (CPS) were present among the mevalonate pathway genes (Fig. <a href="#Fig6">6</a>). This enzyme participates in gibberellin biosynthesis [ <a href="#CR55">55</a>]. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 6</span> </a> </div>   <p>Abundance of phenylpropanoid, mevalonate and non- mevalonate pathway genes as per the annotation of predicted genes against all Viridiplantae clade genes in Uniprot. [Abbreviations used- Alcohol dehydrogenase (ADH); Polyphenol oxidase (PPO); Flavonoid O-methyltransferase (FOMT); Cinnamate 4-hydroxylase (C4H); Cinnamyl alcohol dehydrogenase (CAD); Cinnamoyl-CoA reductase (CCR); Hydroxyphenylpyruvate reductase (HPPR); 4-Coumarate:coenzyme A ligase (4CL); Anthocyanidin 3-O-glucoside 5-O-glucosyltransferase (PF3R4); Chalcone synthase (CHS); Tyrosine aminotransferase (TAT); Hydroxycinnamoyl transferase (HSHCT); Rosmarinic acid synthase (RAS); Phenylalanine ammonia-lyase (PAL); Dihydroflavonol 4-reductase (DFR); Flavonoid 3’ 5’-hydroxylase (F3’5’H); UDP-glucose: flavonoid 7-O-glucosyltransferase (UFGT); Eugenol synthase 1 (EGS); <i>p</i>-Coumaroyl shikimate 3’-hydroxylase (CS3’H); <i>p</i>-Coumarate 3-hydroxylase (C3H); Alcohol acyltransferase (AAT2); Caffeoyl CoA <i>O</i>-methyltransferase (CCOMT); Chalcone isomerase (CHI); Ferulate 5-hydroxylase (F5H); Arogenate dehydrogenase (ADH); Chavicol O-methyltransferase (CVOMT); Prephenate aminotransferase (PAT); Chorismate mutase (CM); Dehydroquinate dehydratase/ shikimate dehydrogenase (DHQ-SDH); Anthocyanidin synthase (ANS); Cinnamate/p-coumarate carboxyl methyltransferase (CCMT); Caffeic acid 3- <i>O</i>-methyltransferase (COMT); 3-deoxy-D-arabino-heptulosonate 7-phosphate synthase (DAHPS); Flavanone 3-hydroxylase (F3H); Flavonoid 3’-hydroxylase (F3’H); Glutathione S-transferase (GST); 4-Hydroxyphenylpyruvate dioxygenase (HPPD); Chorismate synthase (CS); Eugenol <i>O</i>-methyltransferase (EOMT); Benzoate carboxyl methyltransferase (BAMT); Arogenate dehydratase (ADT); 3-dehydroquinate synthase (DHQS); Copalyl diphosphate synthase(CPS); Bicyclogermacrene synthase ( <i>Ov</i>-TPS4); Sesquiterpene synthase (SesquiTPS); 3-hydroxy-3-methylglutaryl-coenzyme A reductase (HMGR); (R)-limonene synthase (LS); Terpene synthase (TPS); Geraniol synthase (GES); Gamma-cadinene synthase (CDS); Secologanin synthase (SLS); Selinene synthase (SES); Multifunctional triterpene synthase (Multifunc triTPS); (+)-menthofuran synthase (MFS); Ent-kaurene synthase (KS); Lupeol synthase(LUP); Nerolidol/linalool synthase (NES/LIS-1); Alpha-zingiberene synthase (ZIS); Germacrene D synthase (GDS); 3-hydroxy-3-methylglutaryl coenzyme A synthase(HMGS); Mevalonate kinase (MVK); Mevalonate diphosphate decarboxylase (MDC); Farnesyl diphosphate synthase(FPPS); Squalene synthase (SQS); <i>Beta</i>-amyrin synthase (bAS); Mixed amyrin synthase (AS); 5-epi-aristolochene synthase (EAS); <i>Beta</i>-myrcene synthase (MYS); <i>Cis</i>-muuroladiene synthase (MxpSS1); Monoterpene synthases (MTPS); Cineole synthase (CinS2); Terpinolene synthase (TES); Valencene synthase (ValCS); 5-phosphomevalonate kinase (PMK); (-)-endo-fenchol synthase (FES); (+)-epi-alpha-bisabolol synthase ( <i>Ld</i>TPS8); Tricyclene synthase 0e23 /(E)-beta-ocimene synthase 0e23/ Terpenoid synthase 0e23) (Terpenoid synthase); Geranyl diphosphate synthase (GPPS); 1-deoxy-D-xylulose 5-phosphate synthase (DXS); 4-hydroxy-3-methylbut-2-enyl diphosphate reductase (HDR); Geranylgeranyl diphosphate synthase (GGPPS); Isopentenyl diphosphate isomerise (IDI); 1-deoxy-D-xylulose-5-phosphate reductoisomerase (DXR); 2-C-methyl-D-erythritol 4-phosphate cytidylyltransferase (MCT); 4-diphosphocytidyl-2-C-methyl-D-erythritol kinase (CMK); 2-C-methyl-D-erythritol 2,4-cyclodiphosphate synthase (MECPP); 4-hydroxy-3-methylbut-2-enyl diphosphate synthase (HDS)] </p>   </div> <p /> <p>Not only gibberellins, but a wide range of secondary metabolites, including terpenes and alkaloids, are also derived either from <i>ent</i>-copalyl pyrophosphate itself or from <i>ent</i>-kaurene or <i>ent</i>-kaurenoic acid, the next two intermediates in the metabolic pathway to gibberellins. Knowledge of these secondary metabolic pathways is very much limited as compared to gibberellin biosynthetic pathway, and is often little more than a speculation [ <a href="#CR56">56</a>]. Further functional characterization studies for copalyl diphosphate synthase may help in proving the possibility of CPS involvement in terpene and alkaloid biosynthesis. </p> </div> <div class="medicinalnatureofo.sanctum" title="sec"> <div class="title" tagx="title" title="title">
Medicinal nature of <i>O. sanctum</i> </div> <p>In this analysis <i>O. sanctum</i> cp genome was observed to be evolutionarily nearest to <i>S. miltiorrhiza</i>. In the absence of complete genome sequence data (unfinished draft genome) of <i>S. Miltiorrhiza</i>, the chloroplast genome comparison analysis was taken into account. Both the plants are used widely in two different traditional medicine systems (Indian and Chinese, respectively), and hence may be implicated for similar molecules, activities <i>vis a vis</i> the genes biosynthesizing metabolites. In addition, both plants have chromosome number described to be 2n = 16 [ <a href="#CR16">16</a>, <a href="#CR57">57</a>]. The active ingredients in <i>S. miltiorrhiza</i> are both hydrophilic (phenolic acids like rosmarinic acid, salvianolic acid B, lithospermic acid and dihydroxyphenyllactic acid) and lipophilic diterpene components (tanshinones, including structurally related tanshinone I, tanshinone IIA, cryptotanshinone, and dihydrotanshinone I) [ <a href="#CR58">58</a>]. These molecules are responsible for a wide array of activities like anti-bacterial, anti-oxidative and anti-viral to hepatoprotective activities. The chemical composition of Tulsi is highly complex, and the important are triterpene like urosolic acid (cardioprotective effect), phenolics like rosmarinic acid, apigenin, cirsimaritin, isothymusin and isothymonin (exhibiting antioxidant and anti-inflammatory activities), and important aroma components like 1, 8 cineole, linalool, methyl chavicol (estragole) and eugenol [ <a href="#CR16">16</a>]. Phenolic acid compounds production by hairy root culture have been reported in both <i>O. basilicum</i> and <i>S. miltiorrhiza</i> [ <a href="#CR59">59</a>]. In addition, the vast literature indicates phenylpropanoid derivatives in these two plants are responsible for a range of major activities. In this investigation also we could observe the dominance of phenylpropanoid pathway genes. The highest number of sequences among the mevalonate pathway genes in <i>O. sanctum</i> are observed to be homologous to copalyl diphosphate synthases (CPS), that are involved in the biosynthesis of an important bioactive diterpene tanshinone in <i>S. miltiorrhiza</i> [ <a href="#CR58">58</a>]. As <i>O. sanctum</i> is traditionally used for many aliments and the compounds of this plant are not fully investigated, the possibility exists for the discovery of tanshinone like compounds and other novel diterpenes. </p> </div> </div> <div class="conclusion" title="conclusion"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>The genome of Holy basil, assembled <i>de novo</i> in this study, presents the smallest nuclear genome in the family Lamiaceae and smallest cp genome in the order Lamiales. Phylogenetically, <i>S. miltiorrhiza</i> is most similar to <i>O. sanctum</i> with a reported genome size of approximately ~600 Mb [ <a href="#CR17">17</a>]. Although, both <i>S. miltiorrhiza</i> and <i>O. sanctum</i> predominantly produce phenylpropanoids and both have the identical diploid number of chromosome (2n = 16), the genome size of <i>O. sanctum</i> is little more than half of the genome size of <i>S. miltiorrhiza</i>. Hence, <i>O. sanctum</i> genome (386 Mb) seems to be quite compact with relatively less repeat sequences, even though it falls in the identical phylogenetic clade. In contrast to the genome sizes of the plants used in the gene model prediction like <i>Solanum lycopersicum</i> (~900 Mb) and <i>Nicotiana tabacum</i> (~4567 Mb), <i>O. sanctum</i> genome (~386 Mb) falls in the category of the plants with small genome and is just 1.5 times that of the model plant <i>Arabidopsis thaliana</i> (~135 Mb) while approximately same size as that of <i>Oryza sativa</i> (~420 Mb) [ <a href="#CR47">47</a>, <a href="#CR60">60</a>, <a href="#CR43">43</a>, <a href="#CR61">61</a>]. </p> <p>Besides the saturated genome sequence, this investigation also provides an assembled chloroplast genome, showing highest similarity to that of <i>S. miltiorrhiza</i>, an important medicinal plant of traditional Chinese medicine. Both the plants are rich in phenylpropanoids and their derivatives, and many of these are implicated for different therapeutic activities. The presence of large number of homologs of copalyl diphosphate synthases (CPS) in <i>O. sanctum</i> genome indicates the possibility of finding newer diterpenes having potential bioactivity not implicated so far. Genomic information generated in this investigation not only is an important resource for evolutionary studies it will also catalyze modern genetic research by augmenting the data available for plant comparative genomics. This will also accelerate identification of important secondary metabolite-synthesizing genes, not identified yet from this medicinal and aromatic plant. Specific pathway related genes identified or mined in this genome could be used for the production of secondary metabolites following synthetic biology approaches. Genetic markers can be developed based on these genome sequences for studies involving genetic map construction, positional cloning, strain identification and marker-assisted selection. These molecular tools and genomic resources will accelerate molecular breeding and ultimately Holy basil’s utility in medical community. </p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Methods</div> <div class="plantmaterial,dnapreparation" title="sec"> <div class="title" tagx="title" title="title">
Plant material, DNA preparation</div> <p>Leaf tissues of <i>O. sanctum</i> L. (variety CIM Ayu) were collected from the experimental farm at the CSIR-Central Institute of Medicinal and Aromatic Plants. High molecular weight genomic DNA isolated (Plant DNA extraction kit, Qiagen) from the leaves of <i>O. sanctum</i> was analyzed for its concentration and integrity. This DNA was then used for a whole-genome shotgun and mate-pair library preparation. </p> </div> <div class="librarypreparationmethods" title="sec"> <div class="title" tagx="title" title="title">
Library preparation methods</div> <div class="longandshortshotgunlibraryconstruction" title="sec"> <div class="title" tagx="title" title="title">
Long and short shot gun library construction</div> <p>Long and short insert libraries for whole genome sequencing were constructed as per Illumina TruSeq DNA library (TruSeq DNA Sample Preparation Guide, Part No. 15005180 Rev. A, Nov 2010). 2 microgram of genomic DNA was used to prepare the DNA library acoustic shearing (Covaris Inc., USA) to a fragment distribution ranging between 150 to 600 bp and purified (Agencourt Ampure XP SPRI beads, Beckman Coulter, Inc.). Fragment distribution was analyzed (high sensitivity bioanalyzer chip, Agilent Technologies), finally purified (Agencourt Ampure XP SPRI beads) and quantified (Qubit fluorometer, Invitrogen as well as a high sensitivity bioanalyzer Chip, Agilent Technologies). The library shows a peak at the range of 300-400bp for short insert and 500-600bp for long insert libraries, respectively. Finally the libraries prepared were found suitable for 100bp paired end sequencing on Illumina. </p> </div> <div class="longreads454gsflxlibrarylibraryconstruction" title="sec"> <div class="title" tagx="title" title="title">
Long reads 454 GS FLX library library construction</div> <p>454 GS FLX library was constructed according to the Roche rapid library preparation method manual (GS FLX+ Series—XL+, May 2011). Briefly ~1ug genomic DNA was fragmented (using a nebulizer), purified (Minelute PCR purification kit, Qiagen) and end-repaired followed by adapter ligation. The prepared library was validated for quality (high sensitivity bioanalyzer chip, Agilent Technologies) which showed an expected peak range of 1.4–1.8 kb. </p> </div> <div class="mate-pairlibraryconstruction" title="sec"> <div class="title" tagx="title" title="title">
Mate-pair library construction</div> <p>Mate pair libraries were generated as per the SOLiD Mate Pair Library preparation protocol. 23ug genomic DNA was sheared (ultra-sonicator, Covaris, USA) and analyzed for the size distribution (high sensitivity bioanalyzer chip, Agilent Technologies) also verified on 2 % E-gel. Next step was end-repairing of the fragments ranging from 2.5 to 3.5kb (resolved on 0.6 % agarose) followed by MPR-MPL adaptor ligation. Further, nick-translation was performed on circularized adaptor ligated DNA digested with T7 endonuclease I followed by S1 Nuclease enzymes. These products were 3’ adenylation by P1-T and P2-T, and captured using streptavidin beads (Invtirogen). Adaptor ligated sample was amplified with 18 cycles of PCR and size selected in the range of 250bp to 350bp using E-Gel (Invitrogen). </p> </div> </div> <div class="sequencingofshot-gunandmate-pairlibrariesandgenomeassembly" title="sec"> <div class="title" tagx="title" title="title">
Sequencing of shot-gun and mate-pair libraries and Genome assembly</div> <p>Long and short insert libraries, were sequence on Hiseq2000 (Illumina) using 100 base paired end chemistry. Long single end reads were generated using Roche 454 (Roche) and mate-pair libraries were run on SOLiD 5500XL (Life technologies). Illumina generated 224,617,107 paired end reads (45.37 Gb data), 454 sequencing resulted in 643,134 single end reads (320.3Mb data) while SOLiD generated 126,824,255 mate pair reads (12.68 Gb data) </p> <p>Long and short reads paired-end read data (HiSeq2000) of 449,234,214 (449 million) reads with high quality ( &amp;gt; = Q30) were assembled with Edena v3.1 [ <a href="#CR62">62</a>]. Edena was used with default parameters, i.e. minimum overlap size being 50 and coverage cutoff, 4. Total genome coverage from the long and short insert paired-end reads was ~18.25X and ~82.55X (Additional file <a href="#MOESM1">1</a>), respectively. 643,134 long single end 454 reads, processed for quality filtering with Phred score &amp;gt; =Q20 having a genome coverage of ~0.71X were then used for contig extension using SSPACE-2.03 [ <a href="#CR63">63</a>]. SSPACE was used with these parameters: (i) minimum number of overlapping bases with the seed: 45, (ii) minimum overlap required between contigs to merge adjacent contigs in a scaffold:50, (iii) minimum number of read pairs to compute scaffold: 5 and contig extension switched on (iv) minimum number of reads needed to call a base during an extension: 20 and, (v) maximum number of allowed gaps during mapping with Bowtie: 1. Scaffolds thus generated do consisted of uncalled bases (Ns). Gap filling of these inter-scaffold Ns with nucleotides was carried out using GapClosure tool [ <a href="#CR64">64</a>]. 252 million reads were generated using SOLiD showed ~30X coverage on the genome. SOLiD reads, with mean quality of Q20, and reads that have any uncalled bases (Ns) were filtered using SOPRA v1.4.6 [ <a href="#CR65">65</a>] tool. Super-scaffolding was performed in order to merge the existing gap-closed scaffolds into super-scaffolds using relative orientation of SOLiD mate pair reads. Super-scaffolding using MIP-scaffolder [ <a href="#CR66">66</a>] requires F3 and R3 reads to be mapped on preassembled scaffolds. This was achieved using SHRiMP2 [ <a href="#CR67">67</a>] tool, which aligns reads in colorspace format. </p> </div> <div class="genepredictionandannotation" title="sec"> <div class="title" tagx="title" title="title">
Gene prediction and annotation</div> <p> <i>Ab initio</i> Gene model prediction was performed on scaffold sequences greater than 500bp using gene prediction software AUGUSTUS v2.5.5 [ <a href="#CR68">68</a>]. Parameters from <i>N. tabacum</i> and <i>S. lycopersicum</i> species which share the same sub-class (asterid) with <i>Ocimum sanctum</i> were applied as training sets. Gene annotation of predicted proteins was done by matching to NCBI Non Redundant database using BLASTP (ncbi-blastv2.2.26+) [ <a href="#CR69">69</a>]. Domain prediction for unannotated proteins was performed against Pfam (release 27) HMM signatures [ <a href="#CR70">70</a>] using Pfam-A set with HMMSCAN option in HMMER 3.0 [ <a href="#CR71">71</a>] at default parameters. Further scaffold sequences greater than 500bp in length were matched for match to EST/mRNA sequences available for <i>Ocimum</i> in the NCBI databases. <i>Arabidopsis</i> sequences from TAIR database were also BLAST checked against the <i>Ocimum</i> scaffolds (greater than 500bp). <i>Nicotiana</i> and <i>Solanum</i> EST’s from NCBI database were retrieved and matched against the assembled scaffolds which had length greater than 500bp. </p> </div> <div class="comparativegenomicsandssrprediction" title="sec"> <div class="title" tagx="title" title="title">
Comparative genomics and SSR prediction</div> <p>The comparison of scaffolds with the <i>Ocimum</i> sequences was carried out using blat- Standalone BLAT v. 34x12 [ <a href="#CR72">72</a>] fast sequence search command line tool. A total of 23,368 EST and 52 mRNA were queried, with a match to the assembled scaffolds for 21,984 of the EST/mRNA sequences at greater than 90 % sequence identity. <i>Arabidopsis</i> sequences from TAIR as well as <i>N. tabacum</i> and <i>S. lycopersicum</i> EST’s from NCBI database were also blast checked against the <i>O. sanctum</i> scaffolds (greater than 500bp). Apart from the database annotation of the assembled scaffolds these were also queried for intron length, intron distribution and gene density determination using AUGUSTUS v2.5.5 [ <a href="#CR68">68</a>] with <i>N. tabacum</i> and <i>S. lycopersicum</i> as references. </p> <p>Scaffold sequences of length less than 500bp as well as greater than 500bp were separately checked for simple sequence repeats (SSRs) using MISA tool ( <a href="http://pgrc.ipk-gatersleben.de/misa/">http://pgrc.ipk-gatersleben.de/misa/</a>). The sequences were checked for mono-repeats occurring at-least 10 times, di-repeats occurring at-least 6 times and tri/tetra/penta/hexa-repeats occuring atleast 5 times. </p> </div> <div class="annotationandde-novoassemblyofchloroplastandmitochondrialgenomedata" title="sec"> <div class="title" tagx="title" title="title">
Annotation and <i>de-novo</i> assembly of chloroplast and mitochondrial genome data </div> <p>Processed short reads paired-end read data of 72,912,212 (72.91 million) reads were aligned using BOWTIE2-2.1.0 [ <a href="#CR73">73</a>] to " <i>Liquidambar formosana</i> (Accession no. KC588388.1), <i>Nandina domestica</i> (Accession no. DQ923117.1), <i>Arabidopsis thaliana</i> (Accession no. NC_000932), <i>Citrus sinensis</i> (Accession no. NC_008334), <i>Cucumis sativus</i> (Accession no. NC_007144), <i>Gossypium hirsutum</i> (Accession no. NC_007944), <i>Helianthus annuus</i> (Accession no. NC_007977), <i>Nerium oleander</i> (Accession no. KJ953906.1), <i>Oenothera biennis</i> (Accession no. NC_010361), <i>Platanus occidentalis</i> (Accession no. NC_008335), <i>Populus trichocarpa</i> (Accession no. NC_009143), <i>Spinacia oleracea</i> (Accession no. NC_002202), <i>Ximenia americana</i> (Accession no. HQ664594.1), <i>Ilex cornuta</i> (Accession no. HQ664579.1) <i>, Dillenia indica</i> (Accession no. HQ664593.1) <i>, Oxalis latifolia</i> (Accession no. HQ664602.1) <i>, Plumbago auriculata</i> (Accession no. HQ664581.1) <i>, Staphylea colchica</i> (Accession no. HQ664600.1) <i>, Lonicera japonica</i> (Accession no. HQ664582.1) <i>, Antirrhinum majus</i> (Accession no. HQ664592.1) <i>, Cornus florida</i> (Accession no. HQ664596.1) <i>, Ficus sp.</i> (Accession no. HQ664605.1) chloroplast genomes. Database annotation of EST/mRNA from NCBI datasets identified the mitochondria and chloroplast expressed proteins. These 122 scaffolds were annotated to potentially map to these sequences. The aligned reads were assembled using SPAdes-3.1.0 [ <a href="#CR74">74</a>]. The assembled contigs were scaffolded using SSPACE-2.0 using all the four libraries Illumina data. Saffolds were gapclosed using Gapcloser-1.6. OrganellarGenomeDRAW (OGDRAW) was used for generating graphical maps of plastid genomes [ <a href="#CR75">75</a>]. </p> <p>Similar procedure carried for mitochondria assembly except chloroplast genomes <i>Salvia miltiorrhiza</i> mitochondria genome used as reference and scaffolding and gapclosing was done using MIP-Scaffolder [ <a href="#CR66">66</a>] using SOLiD data. Chloroplast Scaffolds greater than 10kb were filtered, ordered and joined with 2 N’s though using <i>Salvia miltiorrhiza</i> chloroplast genome. Annotation was carried from draft genome using DOGMA tool [ <a href="#CR76">76</a>]. </p> </div> <div class="sequencedivergenceandphylogeneticanalysis" title="sec"> <div class="title" tagx="title" title="title">
Sequence divergence and phylogenetic analysis</div> <p>The 32 complete cp sequences representing the asterid lineage of angiosperms were downloaded from NCBI Organelle Genome Resources database (Additional file <a href="#MOESM17">17</a>). The 63 protein-coding gene sequences were aligned using the Clustal algorithm [ <a href="#CR77">77</a>]. For the phylogenetic analysis, a set of 63 protein-coding genes commonly present in the 31 analyzed genomes was used. Maximum parsimony (MP) and Maximum likelihood (ML) analysis was performed for the phylogenetic analysis and the tree was generated using MEGA 6.0 [ <a href="#CR78">78</a>] software. In the analysis <i>Spinacia oleracea</i> and <i>Arabidopsis thaliana</i> were set as outgroups. </p> </div> <div class="genomeannotationandpathwayidentification" title="sec"> <div class="title" tagx="title" title="title">
Genome annotation and pathway identification</div> <p>85,723 protein coding sequences were blasted against NR proteins GO (Gene Ontology) terms were assigned for each protein based on the GO terms annotated to its corresponding homologue in the NR database. Each annotated sequence may have more than one GO term, assigned either for different GO categories (Biological Process, Molecular Function and Cellular Component) or in the same category [ <a href="#CR79">79</a>]. </p> <p>Nucleotide sequences of the predicted proteins from scaffolds were retrieved (BEDTools-Version-2.13.1) [ <a href="#CR80">80</a>] and mapped to KAAS [ <a href="#CR81">81</a>] server to match pathway datasets from curated model species. Homology driven match of KO ID’s to best hits was done with default parameters. Match to model dicot and moncot plants <i>Arabidopsis</i> and <i>Oryza</i> were applied for pathway annotation. </p> </div> <div class="dataaccess" title="sec"> <div class="title" tagx="title" title="title">
Data access</div> <p>Genomic data generated by all the three platforms of <i>O. sanctum</i> whole project are available at NCBI under accession numbers SRX760129, SRR1653607 (Illumina); SRX760132, SRR1653610 (454_GS_FLX) and SRX761338, SRR1654829 (SOLiD). The data was submitted by SRA submission portal with submissionID, SUB745374 and BioProject ID, PRJNA267195. </p> </div> <div class="availabilityofsupportingdata" title="sec"> <div class="title" tagx="title" title="title">
Availability of supporting data</div> <p>The data sets supporting the results of this article are included within the article (and its additional files). </p> </div> </div> </div> <div class="back" title="back"> <div class="app-group" title="app-group"> <div class="app" title="app"> <div class="additionalfiles" title="sec"> <div class="title" tagx="title" title="title">
Additional files</div> <p> </p><div class="media" title="media"> <a href="12864_2015_1640_MOESM1_ESM.pdf">LINK</a> <span class="label" tagx="label" title="label">Additional file 1:</span>  <p> <b>Illumina libraries statistics.</b> </p>  </div> <div class="media" title="media"> <a href="12864_2015_1640_MOESM2_ESM.pdf">LINK</a> <span class="label" tagx="label" title="label">Additional file 2:</span>  <p> <b>Read length distribution chart for 454 reads.</b> </p>  </div> <div class="media" title="media"> <a href="12864_2015_1640_MOESM3_ESM.pdf">LINK</a> <span class="label" tagx="label" title="label">Additional file 3:</span>  <p> <b>SOLiD colorspace reads statistics.</b> </p>  </div> <div class="media" title="media"> <a href="12864_2015_1640_MOESM4_ESM.pdf">LINK</a> <span class="label" tagx="label" title="label">Additional file 4:</span>  <p> <b>Statistics of transcriptome and whole genome assembly of </b> <b> <i>O. sanctum.</i> </b> </p>  </div> <div class="media" title="media"> <a href="12864_2015_1640_MOESM5_ESM.pdf">LINK</a> <span class="label" tagx="label" title="label">Additional file 5:</span>  <p> <b>QC statistics of chloroplast de novo assembly at each step.</b> </p>  </div> <div class="media" title="media"> <a href="12864_2015_1640_MOESM6_ESM.pdf">LINK</a> <span class="label" tagx="label" title="label">Additional file 6:</span>  <p> <b>QC statistics of mitochondrial de novo assembly at each step.</b> </p>  </div> <div class="media" title="media"> <a href="12864_2015_1640_MOESM7_ESM.pdf">LINK</a> <span class="label" tagx="label" title="label">Additional file 7:</span>  <p> <b> <i>Ocimum sanctum</i> </b> <b>in comparison to</b> <b> <i>Nicotiana tabacum</i> </b> <b>and</b> <b> <i>Solanum lycopersicum.</i> </b> </p>  </div> <div class="media" title="media"> <a href="12864_2015_1640_MOESM8_ESM.xlsx">LINK</a> <span class="label" tagx="label" title="label">Additional file 8:</span>  <p> <b>Gene ontology classification.</b> The results are summarized in three main categories: biological process, cellular component and molecular function. </p>  </div> <div class="media" title="media"> <a href="12864_2015_1640_MOESM9_ESM.xlsx">LINK</a> <span class="label" tagx="label" title="label">Additional file 9:</span>  <p> <b>Annotation of predicted genes against all Viridiplantae clade genes in UNIPROT.</b> </p>  </div> <div class="media" title="media"> <a href="12864_2015_1640_MOESM10_ESM.pdf">LINK</a> <span class="label" tagx="label" title="label">Additional file 10:</span>  <p> <b>Top 10 Species distribution of hits to UNIPROT database.</b> </p>  </div> <div class="media" title="media"> <a href="12864_2015_1640_MOESM11_ESM.xlsx">LINK</a> <span class="label" tagx="label" title="label">Additional file 11:</span>  <p> <b> <i>Ocimum</i> </b> <b>EST blat check against</b> <b> <i>O. sanctum</i> </b> <b>scaffolds.</b> </p>  </div> <div class="media" title="media"> <a href="12864_2015_1640_MOESM12_ESM.xlsx">LINK</a> <span class="label" tagx="label" title="label">Additional file 12:</span>  <p> <b> <i>Arabidopsis</i> </b> <b>TAIR sequences blast check against</b> <b> <i>O. sanctum</i> </b> <b>scaffolds.</b> </p>  </div> <div class="media" title="media"> <a href="12864_2015_1640_MOESM13_ESM.xlsx">LINK</a> <span class="label" tagx="label" title="label">Additional file 13:</span>  <p> <b> <i>Nicotiana tabacum</i> </b> <b>EST blast check against</b> <b> <i>O. sanctum</i> </b> <b>scaffolds.</b> </p>  </div> <div class="media" title="media"> <a href="12864_2015_1640_MOESM14_ESM.xlsx">LINK</a> <span class="label" tagx="label" title="label">Additional file 14:</span>  <p> <b> <i>Solanum lycopersicum</i> </b> <b>EST blast check against</b> <b> <i>O. sanctum</i> </b> <b>scaffolds.</b> </p>  </div> <div class="media" title="media"> <a href="12864_2015_1640_MOESM15_ESM.xlsx">LINK</a> <span class="label" tagx="label" title="label">Additional file 15</span>  <p> <b>Annotation of Mitochondrial/Chloroplast scaffolds based on the TAIR database.</b> ChrC_ChrM are the chloroplast and mitochondria sequences containing scaffolds based on TAIR ID. </p>  </div> <div class="media" title="media"> <a href="12864_2015_1640_MOESM16_ESM.pdf">LINK</a> <span class="label" tagx="label" title="label">Additional file 16:</span>  <p> <b>List of 63 protein coding genes of cp genomes used in the phylogenetic analysis.</b> </p>  </div> <div class="media" title="media"> <a href="12864_2015_1640_MOESM17_ESM.pdf">LINK</a> <span class="label" tagx="label" title="label">Additional file 17:</span>  <p> <b>The list of accession numbers of the chloroplast genome sequences used in this study.</b> </p>  </div> <div class="media" title="media"> <a href="12864_2015_1640_MOESM18_ESM.xlsx">LINK</a> <span class="label" tagx="label" title="label">Additional file 18:</span>  <p> <b>Pathways associated with predicted proteins from scaffolds of</b> <b> <i>O. sanctum</i> </b> <b>as predicted by KAAS server taking Arabidopsis and rice as reference organisms.</b> </p>  </div> <p /> </div> </div> </div> <div class="glossary" title="glossary"> <div class="title" tagx="title" title="title">
Abbreviations</div> <div class="def-list" title="def-list"> <div class="def-item" title="def-item"> <div class="term" title="term">
Mb</div> <div class="def" title="def"> <p>Megabasepairs</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
bp</div> <div class="def" title="def"> <p>Basepairs</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
kb</div> <div class="def" title="def"> <p>Kilobasepairs</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SOLiD</div> <div class="def" title="def"> <p>Sequencing by oligonucleotide ligation and detection</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SSRs</div> <div class="def" title="def"> <p>Simple sequence repeats</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
cp</div> <div class="def" title="def"> <p>Chloroplast</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
GO</div> <div class="def" title="def"> <p>Gene ontology</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
BP</div> <div class="def" title="def"> <p>Biological process</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
CC</div> <div class="def" title="def"> <p>Cellular component</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
MF</div> <div class="def" title="def"> <p>Molecular function</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
UNIPROT</div> <div class="def" title="def"> <p>Universal protein resource database</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
DNA</div> <div class="def" title="def"> <p>Deoxyribonucleic acid</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
tRNA</div> <div class="def" title="def"> <p>Transfer ribonucleic acid</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
rRNA</div> <div class="def" title="def"> <p>Ribosomal ribonucleic acid</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
mRNA</div> <div class="def" title="def"> <p>Messenger ribonucleic acid</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
EST</div> <div class="def" title="def"> <p>Expressed sequence tag</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
NR</div> <div class="def" title="def"> <p>Non-redundant</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
NCBI</div> <div class="def" title="def"> <p>National Center for Biotechnology Information</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
TAIR</div> <div class="def" title="def"> <p>The Arabidopsis Information Resource</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
BLAST</div> <div class="def" title="def"> <p>Basic Local Alignment Search Tool</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
KEGG</div> <div class="def" title="def"> <p>Kyoto Encyclopedia of Genes and Genomes</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
KAAS</div> <div class="def" title="def"> <p>KEGG Automatic Annotation Server</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SRA</div> <div class="def" title="def"> <p>Sequence Read Archive</p> </div> </div> </div> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p> <b>Competing interests</b> </p> <p>The authors declare that they have no competing interests.</p> </div> <div class="fn-type-" title=""> <p> <b>Authors’ contributions</b> </p> <p>RKL developed the plant variety used in the investigation. AKS, AK, VG conceived and designed the whole genome sequencing experiments. SP, CG, GR performed the whole genome sequencing experiments. SR, SP, CG, GR, FK, VG analyzed the whole genome sequencing data. AKS, FK, AKT, SR interpreted the analyzed data. SR, AKS and AKT wrote the manuscript. All authors read and approved the final manuscript. </p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>This work was supported by CSIR- Twelfth Five Year Plan Project (BSC0107 and BSC0203). The authors express their sincere gratitude to Director, CSIR-CIMAP for his keen interest and support. We acknowledge NGBMAP for providing the plant material as well as Genotypic Technology (P) Ltd (Bangalore, India) team including Rushiraj Manchiganti, Prasad M Sarashetti, Pritam Sarkhel, Shilp Purohit, for their help during analysis and the CEO Dr. Raja C Mugasimangalam for his inputs in MS writing. There is no conflict of interest. </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="CR1" /> <span class="label" tagx="label" title="label">1.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Darrah</span> <span class="given-names" tagx="given-names" title="given-names">HH</span> </span> </span> <span class="source" tagx="source" title="source">The cultivated basils</span> <span class="year" tagx="year" title="year">1980</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Independence, MO</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Buckeye Printing Company</span> </span> </li> <li tag="ref"> <a name="CR2" /> <span class="label" tagx="label" title="label">2.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gupta</span> <span class="given-names" tagx="given-names" title="given-names">SK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Prakash</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Srivastava</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Validation of traditional claim of Tulsi. Ocimum sanctum Linn. as a medicinal plant</span> <span class="source" tagx="source" title="source">Indian J Exp Biol</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">40</span> <span class="issue" tagx="issue" title="issue">7</span> <span class="fpage" tagx="fpage" title="fpage">765</span> <span class="lpage" tagx="lpage" title="lpage">73</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12597545">12597545</a> </span> </span> </li> <li tag="ref"> <a name="CR3" /> <span class="label" tagx="label" title="label">3.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Uma Devi</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Radioprotective, anticarcinogenic and antioxidant properties of the Indian holy basil, <i>Ocimum sanctum</i> (Tulasi) </span> <span class="source" tagx="source" title="source">Indian J Exp Biol</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">39</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">185</span> <span class="lpage" tagx="lpage" title="lpage">90</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11495275">11495275</a> </span> </span> </li> <li tag="ref"> <a name="CR4" /> <span class="label" tagx="label" title="label">4.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singh</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hoette</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> </span> <span class="source" tagx="source" title="source">Tulsi: The Mother Medicine of Nature</span> <span class="year" tagx="year" title="year">2002</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Lucknow, India</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">International Institute of Herbal Medicine</span> </span> </li> <li tag="ref"> <a name="CR5" /> <span class="label" tagx="label" title="label">5.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Warrier PK: In: Indian Medicinal Plants. Edited by Longman O. New Delhi: CBS publication; 1995:168 </span> </li> <li tag="ref"> <a name="CR6" /> <span class="label" tagx="label" title="label">6.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pattanayak</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Behera</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Das</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Panda</span> <span class="given-names" tagx="given-names" title="given-names">SK</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title"> <i>Ocimum sanctum</i> Linn. A reservoir plant for therapeutic applications: An overview </span> <span class="source" tagx="source" title="source">Pharmacogn Rev</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="issue" tagx="issue" title="issue">7</span> <span class="fpage" tagx="fpage" title="fpage">95</span> <span class="lpage" tagx="lpage" title="lpage">105</span> <span class="pub-id"> <a href="https://dx.doi.org/10.4103/0973-7847.65323">10.4103/0973-7847.65323</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22228948">22228948</a> </span> </span> </li> <li tag="ref"> <a name="CR7" /> <span class="label" tagx="label" title="label">7.</span> <span class="element-citation'"> <span class="person-group'"> <span class="collab'">collab: World Health Organization</span> </span> <span class="mixed-article-title" title="mixed-article-title">Folium Ocimi Sancti</span> <span class="source" tagx="source" title="source">WHO Monographs on Selected Medicinal Plants</span> <span class="year" tagx="year" title="year">2002</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Geneva, Switzerland</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">World Health Organization</span> <span class="fpage" tagx="fpage" title="fpage">206</span> <span class="lpage" tagx="lpage" title="lpage">216</span> </span> </li> <li tag="ref"> <a name="CR8" /> <span class="label" tagx="label" title="label">8.</span> <span class="element-citation'"> <span class="person-group'"> <span class="collab'">collab: Anonymous</span> </span> <span class="mixed-article-title" title="mixed-article-title">Wealth of India</span> <span class="source" tagx="source" title="source">Publication and Information Directorate</span> <span class="year" tagx="year" title="year">1991</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New Delhi, India</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">CSIR</span> <span class="fpage" tagx="fpage" title="fpage">79</span> <span class="lpage" tagx="lpage" title="lpage">89</span> </span> </li> <li tag="ref"> <a name="CR9" /> <span class="label" tagx="label" title="label">9.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gupta</span> <span class="given-names" tagx="given-names" title="given-names">AK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tandon</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharma</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gupta</span> <span class="given-names" tagx="given-names" title="given-names">AK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tandon</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharma</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Ocimum sanctum Linn</span> <span class="source" tagx="source" title="source">Quality Standards <i>of Indian Medicinal Plants, Volume 5</i> </span> <span class="year" tagx="year" title="year">2008</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New Delhi, India</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Medicinal Plants Unit, Indian Council of Medical Research</span> <span class="fpage" tagx="fpage" title="fpage">275</span> <span class="lpage" tagx="lpage" title="lpage">84</span> </span> </li> <li tag="ref"> <a name="CR10" /> <span class="label" tagx="label" title="label">10.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bhasin</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title"> <i>Ocimum</i>- Taxonomy, medici <i>nal potentialities and economic value of essentia</i>l oil </span> <span class="source" tagx="source" title="source">Journal of Biosphere</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">1</span> <span class="fpage" tagx="fpage" title="fpage">48</span> <span class="lpage" tagx="lpage" title="lpage">50</span> </span> </li> <li tag="ref"> <a name="CR11" /> <span class="label" tagx="label" title="label">11.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kelm</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nair</span> <span class="given-names" tagx="given-names" title="given-names">MG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Strasburg</span> <span class="given-names" tagx="given-names" title="given-names">GM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">DeWitt</span> <span class="given-names" tagx="given-names" title="given-names">DL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Antioxidant and cyclooxygenase inhibitory phenolic compounds from <i>Ocimum sanctum</i> Linn </span> <span class="source" tagx="source" title="source">Phytomedicine</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">7</span> <span class="lpage" tagx="lpage" title="lpage">13</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0944-7113(00)80015-X">10.1016/S0944-7113(00)80015-X</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10782484">10782484</a> </span> </span> </li> <li tag="ref"> <a name="CR12" /> <span class="label" tagx="label" title="label">12.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shishodia</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Majumdar</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Banerjee</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aggarwal</span> <span class="given-names" tagx="given-names" title="given-names">BB</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Urosolic acid inhibits nuclear factor-kappaB activation induced by carcinogenic agents through suppression of IkappaBalpha kinase and p65 phosphorylation: Correlation with down-regulation of cyclooxygenase 2, matrix metalloproteinase 9, and cyclin D1 </span> <span class="source" tagx="source" title="source">Cancer Res</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">63</span> <span class="fpage" tagx="fpage" title="fpage">4375</span> <span class="lpage" tagx="lpage" title="lpage">4383</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12907607">12907607</a> </span> </span> </li> <li tag="ref"> <a name="CR13" /> <span class="label" tagx="label" title="label">13.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Iijima</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gang</span> <span class="given-names" tagx="given-names" title="given-names">DR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fridman</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lewinsohn</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pichersky</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Characterization of geraniol synthase from the peltate glands of sweet basil</span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">134</span> <span class="fpage" tagx="fpage" title="fpage">370</span> <span class="lpage" tagx="lpage" title="lpage">379</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.103.032946">10.1104/pp.103.032946</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14657409">14657409</a> </span> </span> </li> <li tag="ref"> <a name="CR14" /> <span class="label" tagx="label" title="label">14.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Tissier A. Trichome Specific Expression: Promoters and Their Applications. Transgenic Plants -Advances and Limitations. 2012;353–378. </span> </li> <li tag="ref"> <a name="CR15" /> <span class="label" tagx="label" title="label">15.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Rahman S, Islam R, Kamruzzaman, Alam K, Jamal AHM. <i>Ocimum sanctum</i> L.: A review of phytochemical and pharmacological profile. American Journal of Drug Discovery and Development 2011, ISSN 2150-427x / doi:10.3923/ajdd.2011. </span> </li> <li tag="ref"> <a name="CR16" /> <span class="label" tagx="label" title="label">16.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rastogi</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meena</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bhattacharya</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ghosh</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shukla</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sangwan</span> <span class="given-names" tagx="given-names" title="given-names">NS</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title"> <i>De novo</i> sequencing and comparative analysis of holy and sweet basil transcriptomes </span> <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">15</span> <span class="fpage" tagx="fpage" title="fpage">588</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-15-588">10.1186/1471-2164-15-588</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25015319">25015319</a> </span> </span> </li> <li tag="ref"> <a name="CR17" /> <span class="label" tagx="label" title="label">17.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qian</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Song</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gao</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The Complete Chloroplast Genome Sequence of the Medicinal Plant <i>Salvia miltiorrhiza</i> </span> <span class="source" tagx="source" title="source">PLoS One</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">e57607</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0057607">10.1371/journal.pone.0057607</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23460883">23460883</a> </span> </span> </li> <li tag="ref"> <a name="CR18" /> <span class="label" tagx="label" title="label">18.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wolfe</span> <span class="given-names" tagx="given-names" title="given-names">KH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morden</span> <span class="given-names" tagx="given-names" title="given-names">CW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmer</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Function and evolution of a minimal plastid genome from a nonphotosynthetic parasitic plant </span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span> <span class="year" tagx="year" title="year">1992</span> <span class="volume" tagx="volume" title="volume">89</span> <span class="issue" tagx="issue" title="issue">22</span> <span class="fpage" tagx="fpage" title="fpage">10648</span> <span class="lpage" tagx="lpage" title="lpage">10652</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.89.22.10648">10.1073/pnas.89.22.10648</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/1332054">1332054</a> </span> </span> </li> <li tag="ref"> <a name="CR19" /> <span class="label" tagx="label" title="label">19.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yi</span> <span class="given-names" tagx="given-names" title="given-names">DK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">KJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Complete chloroplast genome sequences of important oilseed crop <i>Sesamum indicum</i> L </span> <span class="source" tagx="source" title="source">PLoS One</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0035872">10.1371/journal.pone.0035872</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22606240">22606240</a> </span> </span> </li> <li tag="ref"> <a name="CR20" /> <span class="label" tagx="label" title="label">20.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mariotti</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cultrera</span> <span class="given-names" tagx="given-names" title="given-names">NG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Diez</span> <span class="given-names" tagx="given-names" title="given-names">CM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baldoni</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rubini</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Identification of new polymorphic regions and differentiation of cultivated olives (Olea europaea L.) through plastome sequence comparison </span> <span class="source" tagx="source" title="source">BMC Plant Biol</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">211</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2229-10-211">10.1186/1471-2229-10-211</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20868482">20868482</a> </span> </span> </li> <li tag="ref"> <a name="CR21" /> <span class="label" tagx="label" title="label">21.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fang</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deng</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The complete chloroplast and mitochondrial genome sequences of <i>Boea hygrometrica</i>: insights into the evolution of plant organellar genomes </span> <span class="source" tagx="source" title="source">PLoS One</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">e30531</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0030531">10.1371/journal.pone.0030531</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22291979">22291979</a> </span> </span> </li> <li tag="ref"> <a name="CR22" /> <span class="label" tagx="label" title="label">22.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">KJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">HL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Complete chloroplast genome sequences from Korean ginseng ( <i>Panax schinseng</i> Nees) and comparative analysis of sequence evolution among 17 vascular plants </span> <span class="source" tagx="source" title="source">DNA Res</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">247</span> <span class="lpage" tagx="lpage" title="lpage">261</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/dnares/11.4.247">10.1093/dnares/11.4.247</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15500250">15500250</a> </span> </span> </li> <li tag="ref"> <a name="CR23" /> <span class="label" tagx="label" title="label">23.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carels</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hatey</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jabbari</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bernardi</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Compositional properties of homologous coding sequences from plants</span> <span class="source" tagx="source" title="source">J Mol Evol</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">46</span> <span class="fpage" tagx="fpage" title="fpage">45</span> <span class="lpage" tagx="lpage" title="lpage">53</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/PL00006282">10.1007/PL00006282</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9419224">9419224</a> </span> </span> </li> <li tag="ref"> <a name="CR24" /> <span class="label" tagx="label" title="label">24.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wendel</span> <span class="given-names" tagx="given-names" title="given-names">JF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cronn</span> <span class="given-names" tagx="given-names" title="given-names">RC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alvarez</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Small</span> <span class="given-names" tagx="given-names" title="given-names">RL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Senchina</span> <span class="given-names" tagx="given-names" title="given-names">DS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Intron Size and Genome Size in Plants</span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="issue" tagx="issue" title="issue">12</span> <span class="fpage" tagx="fpage" title="fpage">2346</span> <span class="lpage" tagx="lpage" title="lpage">2352</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/oxfordjournals.molbev.a004062">10.1093/oxfordjournals.molbev.a004062</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12446829">12446829</a> </span> </span> </li> <li tag="ref"> <a name="CR25" /> <span class="label" tagx="label" title="label">25.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deutsch</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Long</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Intron-exon structure of eukaryotic model organisms</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">27</span> <span class="fpage" tagx="fpage" title="fpage">3219</span> <span class="lpage" tagx="lpage" title="lpage">3228</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/27.15.3219">10.1093/nar/27.15.3219</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10454621">10454621</a> </span> </span> </li> <li tag="ref"> <a name="CR26" /> <span class="label" tagx="label" title="label">26.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vinogradov</span> <span class="given-names" tagx="given-names" title="given-names">AE</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Intron-genome size relationship on a large evolutionary scale</span> <span class="source" tagx="source" title="source">J Mol Evol</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">49</span> <span class="fpage" tagx="fpage" title="fpage">376</span> <span class="lpage" tagx="lpage" title="lpage">384</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/PL00006561">10.1007/PL00006561</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10473779">10473779</a> </span> </span> </li> <li tag="ref"> <a name="CR27" /> <span class="label" tagx="label" title="label">27.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mclysaght</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Enright</span> <span class="given-names" tagx="given-names" title="given-names">AJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Skrabanek</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wolfe</span> <span class="given-names" tagx="given-names" title="given-names">KH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Estimation of synteny conservation and genome compaction between pufferfish (Fugu) and human </span> <span class="source" tagx="source" title="source">Yeast</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">17</span> <span class="fpage" tagx="fpage" title="fpage">22</span> <span class="lpage" tagx="lpage" title="lpage">36</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/(SICI)1097-0061(200004)17:1&lt;22::AID-YEA5&gt;3.3.CO;2-J">10.1002/(SICI)1097-0061(200004)17:1&amp;lt;22::AID-YEA5&amp;gt;3.3.CO;2-J</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10797599">10797599</a> </span> </span> </li> <li tag="ref"> <a name="CR28" /> <span class="label" tagx="label" title="label">28.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zimmer</span> <span class="given-names" tagx="given-names" title="given-names">AD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lang</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buchta</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rombauts</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nishiyama</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hasebe</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Reannotation and extended community resources for the genome of the non-seed plant <i>Physcomitrella patens</i> provide insights into the evolution of plant gene structures and functions </span> <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">498</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-14-498">10.1186/1471-2164-14-498</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23879659">23879659</a> </span> </span> </li> <li tag="ref"> <a name="CR29" /> <span class="label" tagx="label" title="label">29.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Teich</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grauvogel</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Petersen</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Intron distribution in Plantae: 500 million years of stasis during land plant evolution</span> <span class="source" tagx="source" title="source">Gene</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">394</span> <span class="fpage" tagx="fpage" title="fpage">96</span> <span class="lpage" tagx="lpage" title="lpage">104</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.gene.2007.02.011">10.1016/j.gene.2007.02.011</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17400407">17400407</a> </span> </span> </li> <li tag="ref"> <a name="CR30" /> <span class="label" tagx="label" title="label">30.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lynch</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Conery</span> <span class="given-names" tagx="given-names" title="given-names">JS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The origins of genome complexity</span> <span class="source" tagx="source" title="source">Science</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">302</span> <span class="fpage" tagx="fpage" title="fpage">1401</span> <span class="lpage" tagx="lpage" title="lpage">1404</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.1089370">10.1126/science.1089370</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14631042">14631042</a> </span> </span> </li> <li tag="ref"> <a name="CR31" /> <span class="label" tagx="label" title="label">31.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sena</span> <span class="given-names" tagx="given-names" title="given-names">JS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Giguère</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boyle</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rigault</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Birol</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zuccolo</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Evolution of gene structure in the conifer Picea glauca: a comparative analysis of the impact of intron size </span> <span class="source" tagx="source" title="source">BMC Plant Biol</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">95</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2229-14-95">10.1186/1471-2229-14-95</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24734980">24734980</a> </span> </span> </li> <li tag="ref"> <a name="CR32" /> <span class="label" tagx="label" title="label">32.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ogata</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fujibuchi</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kanehisa</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The size differences among mammalian introns are due to the accumulation of small deletions </span> <span class="source" tagx="source" title="source">FEBS Lett</span> <span class="year" tagx="year" title="year">1996</span> <span class="volume" tagx="volume" title="volume">390</span> <span class="fpage" tagx="fpage" title="fpage">99</span> <span class="lpage" tagx="lpage" title="lpage">103</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/0014-5793(96)00636-9">10.1016/0014-5793(96)00636-9</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8706839">8706839</a> </span> </span> </li> <li tag="ref"> <a name="CR33" /> <span class="label" tagx="label" title="label">33.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moriyama</span> <span class="given-names" tagx="given-names" title="given-names">EN</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Petrov</span> <span class="given-names" tagx="given-names" title="given-names">DA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hartl</span> <span class="given-names" tagx="given-names" title="given-names">DL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Genome size and intron size in Drosophila</span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">15</span> <span class="fpage" tagx="fpage" title="fpage">770</span> <span class="lpage" tagx="lpage" title="lpage">773</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/oxfordjournals.molbev.a025980">10.1093/oxfordjournals.molbev.a025980</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9615458">9615458</a> </span> </span> </li> <li tag="ref"> <a name="CR34" /> <span class="label" tagx="label" title="label">34.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Petrov</span> <span class="given-names" tagx="given-names" title="given-names">DA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Evolution of genome size: new approaches to an old problem</span> <span class="source" tagx="source" title="source">Trends Genet</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">17</span> <span class="fpage" tagx="fpage" title="fpage">23</span> <span class="lpage" tagx="lpage" title="lpage">28</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0168-9525(00)02157-0">10.1016/S0168-9525(00)02157-0</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11163918">11163918</a> </span> </span> </li> <li tag="ref"> <a name="CR35" /> <span class="label" tagx="label" title="label">35.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Petrov</span> <span class="given-names" tagx="given-names" title="given-names">DA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sangster</span> <span class="given-names" tagx="given-names" title="given-names">TA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnston</span> <span class="given-names" tagx="given-names" title="given-names">JS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hartl</span> <span class="given-names" tagx="given-names" title="given-names">DL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shaw</span> <span class="given-names" tagx="given-names" title="given-names">KL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Evidence for DNA loss as a determinant of genome size</span> <span class="source" tagx="source" title="source">Science</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">287</span> <span class="fpage" tagx="fpage" title="fpage">1060</span> <span class="lpage" tagx="lpage" title="lpage">1062</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.287.5455.1060">10.1126/science.287.5455.1060</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10669421">10669421</a> </span> </span> </li> <li tag="ref"> <a name="CR36" /> <span class="label" tagx="label" title="label">36.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lynch</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Intron evolution as a population-genetic process</span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">99</span> <span class="fpage" tagx="fpage" title="fpage">6118</span> <span class="lpage" tagx="lpage" title="lpage">6123</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.092595699">10.1073/pnas.092595699</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11983904">11983904</a> </span> </span> </li> <li tag="ref"> <a name="CR37" /> <span class="label" tagx="label" title="label">37.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Comeron</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kreitman</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The correlation between intron length and recombination in Drosophila: dynamic equilibrium between mutational and selective forces </span> <span class="source" tagx="source" title="source">Genetics</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">156</span> <span class="fpage" tagx="fpage" title="fpage">1175</span> <span class="lpage" tagx="lpage" title="lpage">1190</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11063693">11063693</a> </span> </span> </li> <li tag="ref"> <a name="CR38" /> <span class="label" tagx="label" title="label">38.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gupta</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shukla</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Roy</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sen</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharma</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">In silico SSR and FDM analysis through EST sequences in <i>Ocimum basilicum</i> </span> <span class="source" tagx="source" title="source">Plant Omics Journal</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="issue" tagx="issue" title="issue">4</span> <span class="fpage" tagx="fpage" title="fpage">121</span> <span class="lpage" tagx="lpage" title="lpage">128</span> </span> </li> <li tag="ref"> <a name="CR39" /> <span class="label" tagx="label" title="label">39.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gupta</span> <span class="given-names" tagx="given-names" title="given-names">PK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Varshney</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The development and use of microsatellite markers for genetic analysis and plant breeding with emphasis on bread wheat </span> <span class="source" tagx="source" title="source">Euphytica</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">113</span> <span class="fpage" tagx="fpage" title="fpage">163</span> <span class="lpage" tagx="lpage" title="lpage">185</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1023/A:1003910819967">10.1023/A:1003910819967</a> </span> </span> </li> <li tag="ref"> <a name="CR40" /> <span class="label" tagx="label" title="label">40.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carovic-Stanko</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liber</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Besendorfer</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Javornik</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bohanec</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kolak</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Genetic relations among basil taxa ( <i>Ocimum</i> L.) based on molecular markers, nuclear DNA content, and chromosome number </span> <span class="source" tagx="source" title="source">Plant Syst Evol</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">285</span> <span class="fpage" tagx="fpage" title="fpage">13</span> <span class="lpage" tagx="lpage" title="lpage">22</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00606-009-0251-z">10.1007/s00606-009-0251-z</a> </span> </span> </li> <li tag="ref"> <a name="CR41" /> <span class="label" tagx="label" title="label">41.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lal</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mistry</span> <span class="given-names" tagx="given-names" title="given-names">KN</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thaker</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shah</span> <span class="given-names" tagx="given-names" title="given-names">SD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vaidya</span> <span class="given-names" tagx="given-names" title="given-names">PB</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Genetic diversity assessment in six medicinally important species of <i>Ocimum</i> from central Gujarat (India) utilizing RAPD, ISSR and SSR markers </span> <span class="source" tagx="source" title="source">Int J Ad Biol Res</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">279</span> <span class="lpage" tagx="lpage" title="lpage">288</span> </span> </li> <li tag="ref"> <a name="CR42" /> <span class="label" tagx="label" title="label">42.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Mahajan V, Rather IA, Awasthi P, Anand R, Gairola S, Meena SR, et al. Development of chemical and EST-SSR markers for <i>Ocimum</i> genus. Industrial Crops and Products 2014, In Press, doi:10.1016/j.indcrop.2014.10.052. </span> </li> <li tag="ref"> <a name="CR43" /> <span class="label" tagx="label" title="label">43.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Initiative</span> <span class="given-names" tagx="given-names" title="given-names">TAG</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Analysis of the genome sequence of the flowering plant <i>Arabidopsis thaliana</i> </span> <span class="source" tagx="source" title="source">Nature</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">408</span> <span class="fpage" tagx="fpage" title="fpage">796</span> <span class="lpage" tagx="lpage" title="lpage">815</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/35048692">10.1038/35048692</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11130711">11130711</a> </span> </span> </li> <li tag="ref"> <a name="CR44" /> <span class="label" tagx="label" title="label">44.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feuillet</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Keller</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">High gene density is conserved at syntenic loci of small and large grass genomes</span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">96</span> <span class="fpage" tagx="fpage" title="fpage">8265</span> <span class="lpage" tagx="lpage" title="lpage">8270</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.96.14.8265">10.1073/pnas.96.14.8265</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10393983">10393983</a> </span> </span> </li> <li tag="ref"> <a name="CR45" /> <span class="label" tagx="label" title="label">45.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bennetzen</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">SanMiguel</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tikhonov</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Francki</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Avramova</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Grass genomes</span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">95</span> <span class="fpage" tagx="fpage" title="fpage">1975</span> <span class="lpage" tagx="lpage" title="lpage">1978</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.95.5.1975">10.1073/pnas.95.5.1975</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9482817">9482817</a> </span> </span> </li> <li tag="ref"> <a name="CR46" /> <span class="label" tagx="label" title="label">46.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leushkin</span> <span class="given-names" tagx="given-names" title="given-names">EV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sutormin</span> <span class="given-names" tagx="given-names" title="given-names">RA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nabieva</span> <span class="given-names" tagx="given-names" title="given-names">ER</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Penin</span> <span class="given-names" tagx="given-names" title="given-names">AA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kondrashov</span> <span class="given-names" tagx="given-names" title="given-names">AS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Logacheva</span> <span class="given-names" tagx="given-names" title="given-names">MD</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The miniature genome of a carnivorous plant Genlisea aurea contains a low number of genes and short non-coding sequences </span> <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">476</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-14-476">10.1186/1471-2164-14-476</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23855885">23855885</a> </span> </span> </li> <li tag="ref"> <a name="CR47" /> <span class="label" tagx="label" title="label">47.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">The Tomato Genome Consortium: The tomato genome sequence provides insights into fleshy fruit evolution. Nature 2012, 485: doi:10.1038/nature11119. </span> </li> <li tag="ref"> <a name="CR48" /> <span class="label" tagx="label" title="label">48.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Krishnan</span> <span class="given-names" tagx="given-names" title="given-names">NM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pattnaik</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jain</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gaur</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Choudhary</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vaidyanathan</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A draft of the genome and four transcriptomes of a medicinal and pesticidal angiosperm <i>Azadirachta indica</i> </span> <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="fpage" tagx="fpage" title="fpage">464</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-13-464">10.1186/1471-2164-13-464</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22958331">22958331</a> </span> </span> </li> <li tag="ref"> <a name="CR49" /> <span class="label" tagx="label" title="label">49.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">The French-Italian Public Consortium for Grapevine Genome characterization. The grapevine genome sequence suggests ancestral hexaploidization in major angiosperm phyla. Nature 2007, 449: doi:10.1038/nature06148. </span> </li> <li tag="ref"> <a name="CR50" /> <span class="label" tagx="label" title="label">50.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Clegg MT, Zurawski G. Chloroplast DNA and the Study of Plant Phylogeny: Present Status and Future Prospects. In: Soltis PS, Soltis DE, Dpyle JJ, Editors. Molecular Systematics of Plants. New York: Springer US; 1992. 1-13. </span> </li> <li tag="ref"> <a name="CR51" /> <span class="label" tagx="label" title="label">51.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmer</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Chloroplast DNA and Molecular Phylogeny</span> <span class="source" tagx="source" title="source">Bioessays</span> <span class="year" tagx="year" title="year">1985</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="issue" tagx="issue" title="issue">6</span> <span class="fpage" tagx="fpage" title="fpage">263</span> <span class="lpage" tagx="lpage" title="lpage">267</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/bies.950020607">10.1002/bies.950020607</a> </span> </span> </li> <li tag="ref"> <a name="CR52" /> <span class="label" tagx="label" title="label">52.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shukla</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kaur</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ahuja</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Tulsi the Medicinal Value</span> <span class="source" tagx="source" title="source">Online International Interdisciplinary Research Journal</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">9</span> <span class="lpage" tagx="lpage" title="lpage">14</span> </span> </li> <li tag="ref"> <a name="CR53" /> <span class="label" tagx="label" title="label">53.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Prakash</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gupta</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Therapeutic uses of <i>Ocimum sanctum</i> Linn (Tulsi) with a note on eugenol and its pharmacological actions: a short review </span> <span class="source" tagx="source" title="source">Indian J Physiol Pharmacol</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">49</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">125</span> <span class="lpage" tagx="lpage" title="lpage">131</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16170979">16170979</a> </span> </span> </li> <li tag="ref"> <a name="CR54" /> <span class="label" tagx="label" title="label">54.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lal</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khanuja</span> <span class="given-names" tagx="given-names" title="given-names">SPS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Agnihotri</span> <span class="given-names" tagx="given-names" title="given-names">AK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Misra</span> <span class="given-names" tagx="given-names" title="given-names">HO</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shasany</span> <span class="given-names" tagx="given-names" title="given-names">AK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Naqvi</span> <span class="given-names" tagx="given-names" title="given-names">AA</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">High yielding eugenol rich oil producing variety of <i>Ocimum sanctum</i> – CIM-Ayu </span> <span class="source" tagx="source" title="source">J Med Arom Plant Sci</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">746</span> <span class="lpage" tagx="lpage" title="lpage">747</span> </span> </li> <li tag="ref"> <a name="CR55" /> <span class="label" tagx="label" title="label">55.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fall</span> <span class="given-names" tagx="given-names" title="given-names">RR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">West</span> <span class="given-names" tagx="given-names" title="given-names">CA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Purification and properties of kaurene synthetase from <i>Fusarium moniliforme</i> </span> <span class="source" tagx="source" title="source">J Biol Chem</span> <span class="year" tagx="year" title="year">1971</span> <span class="volume" tagx="volume" title="volume">246</span> <span class="issue" tagx="issue" title="issue">22</span> <span class="fpage" tagx="fpage" title="fpage">6913</span> <span class="lpage" tagx="lpage" title="lpage">6928</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/4331199">4331199</a> </span> </span> </li> <li tag="ref"> <a name="CR56" /> <span class="label" tagx="label" title="label">56.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">BRENDA [ <a href="http://www.brenda-enzymes.org/enzyme.php?ecno=4.2.3.19">http://www.brenda-enzymes.org/enzyme.php?ecno=4.2.3.19</a>]. </span> </li> <li tag="ref"> <a name="CR57" /> <span class="label" tagx="label" title="label">57.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">HX</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fan</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">RW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ding</span> <span class="given-names" tagx="given-names" title="given-names">CB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">YH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Studies on chromosome numbers of <i>Salvia miltiorrhiza</i>, <i>S. flava</i> and <i>S. evansiana</i> </span> <span class="source" tagx="source" title="source">Zhongguo Zhong Yao Za Zhi</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">31</span> <span class="fpage" tagx="fpage" title="fpage">1847</span> <span class="lpage" tagx="lpage" title="lpage">1849</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17285979">17285979</a> </span> </span> </li> <li tag="ref"> <a name="CR58" /> <span class="label" tagx="label" title="label">58.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ding</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheng</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kong</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wei</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Transcriptome analysis of medicinal plant <i>Salvia miltiorrhiza</i> and identification of genes related to tanshinone biosynthesis </span> <span class="source" tagx="source" title="source">PLoS One</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="issue" tagx="issue" title="issue">11</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0080464">10.1371/journal.pone.0080464</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24260395">24260395</a> </span> </span> </li> <li tag="ref"> <a name="CR59" /> <span class="label" tagx="label" title="label">59.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hao</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ji</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shi</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feng</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Exogenous ABA and polyamines enhanced salvianolic acids contents in hairy root cultures of <i>Salvia miltiorrhiza</i> Bge. f.alba </span> <span class="source" tagx="source" title="source">Plant Omics</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">446</span> <span class="lpage" tagx="lpage" title="lpage">452</span> </span> </li> <li tag="ref"> <a name="CR60" /> <span class="label" tagx="label" title="label">60.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sierro</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Battey</span> <span class="given-names" tagx="given-names" title="given-names">JND</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ouadi</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bakaher</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bovet</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Willig</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The tobacco genome sequence and its comparison with those of tomato and potato</span> <span class="source" tagx="source" title="source">Nat Commun</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">3833</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/ncomms4833">10.1038/ncomms4833</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24807620">24807620</a> </span> </span> </li> <li tag="ref"> <a name="CR61" /> <span class="label" tagx="label" title="label">61.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wong</span> <span class="given-names" tagx="given-names" title="given-names">GK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A Draft Sequence of the Rice Genome ( <i>Oryza sativa</i> L. ssp. <i>indica</i>) </span> <span class="source" tagx="source" title="source">Science</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">296</span> <span class="issue" tagx="issue" title="issue">5565</span> <span class="fpage" tagx="fpage" title="fpage">79</span> <span class="lpage" tagx="lpage" title="lpage">92</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.1068037">10.1126/science.1068037</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11935017">11935017</a> </span> </span> </li> <li tag="ref"> <a name="CR62" /> <span class="label" tagx="label" title="label">62.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hernandez</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">François</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Farinelli</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Osterås</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schrenzel</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">De novo bacterial genome sequencing: millions of very short reads assembled on a desktop computer </span> <span class="source" tagx="source" title="source">Genome Res</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">18</span> <span class="fpage" tagx="fpage" title="fpage">802</span> <span class="lpage" tagx="lpage" title="lpage">809</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gr.072033.107">10.1101/gr.072033.107</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18332092">18332092</a> </span> </span> </li> <li tag="ref"> <a name="CR63" /> <span class="label" tagx="label" title="label">63.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boetzer</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Henkel</span> <span class="given-names" tagx="given-names" title="given-names">CV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jansen</span> <span class="given-names" tagx="given-names" title="given-names">HJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Butler</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pirovano</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">SSPACE: Scaffolding pre-assembled contigs using SSPACE</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">27</span> <span class="issue" tagx="issue" title="issue">4</span> <span class="fpage" tagx="fpage" title="fpage">578</span> <span class="lpage" tagx="lpage" title="lpage">579</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/btq683">10.1093/bioinformatics/btq683</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21149342">21149342</a> </span> </span> </li> <li tag="ref"> <a name="CR64" /> <span class="label" tagx="label" title="label">64.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Luo</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xie</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yuan</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler</span> <span class="source" tagx="source" title="source">Gigascience</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">1</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">18</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/2047-217X-1-18">10.1186/2047-217X-1-18</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23587118">23587118</a> </span> </span> </li> <li tag="ref"> <a name="CR65" /> <span class="label" tagx="label" title="label">65.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dayarian</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Michael</span> <span class="given-names" tagx="given-names" title="given-names">TP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sengupta</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">SOPRA: scaffolding algorithm for paired reads via statistical optimization</span> <span class="source" tagx="source" title="source">BMC Bioinformatics</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">345</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2105-11-345">10.1186/1471-2105-11-345</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20576136">20576136</a> </span> </span> </li> <li tag="ref"> <a name="CR66" /> <span class="label" tagx="label" title="label">66.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salmela</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mäkinen</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Välimäki</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ylinen</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ukkonen</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Fast scaffolding with small independent mixed integer programs</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">27</span> <span class="issue" tagx="issue" title="issue">23</span> <span class="fpage" tagx="fpage" title="fpage">3259</span> <span class="lpage" tagx="lpage" title="lpage">3265</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/btr562">10.1093/bioinformatics/btr562</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21998153">21998153</a> </span> </span> </li> <li tag="ref"> <a name="CR67" /> <span class="label" tagx="label" title="label">67.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">David</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dzamba</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lister</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ilie</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brudno</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">SHRiMP2: sensitive yet practical SHort Read Mapping</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">27</span> <span class="issue" tagx="issue" title="issue">7</span> <span class="fpage" tagx="fpage" title="fpage">1011</span> <span class="lpage" tagx="lpage" title="lpage">1012</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/btr046">10.1093/bioinformatics/btr046</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21278192">21278192</a> </span> </span> </li> <li tag="ref"> <a name="CR68" /> <span class="label" tagx="label" title="label">68.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stanke</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Steinkamp</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Waack</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morgenstern</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">AUGUSTUS: a web server for gene finding in eukaryotes</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">32</span> <span class="fpage" tagx="fpage" title="fpage">W309</span> <span class="lpage" tagx="lpage" title="lpage">312</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkh379">10.1093/nar/gkh379</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15215400">15215400</a> </span> </span> </li> <li tag="ref"> <a name="CR69" /> <span class="label" tagx="label" title="label">69.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Altschul</span> <span class="given-names" tagx="given-names" title="given-names">SF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Madden</span> <span class="given-names" tagx="given-names" title="given-names">TL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schäffer</span> <span class="given-names" tagx="given-names" title="given-names">AA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miller</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">17</span> <span class="fpage" tagx="fpage" title="fpage">3389</span> <span class="lpage" tagx="lpage" title="lpage">3402</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/25.17.3389">10.1093/nar/25.17.3389</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694">9254694</a> </span> </span> </li> <li tag="ref"> <a name="CR70" /> <span class="label" tagx="label" title="label">70.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Punta</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coggill</span> <span class="given-names" tagx="given-names" title="given-names">PC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eberhardt</span> <span class="given-names" tagx="given-names" title="given-names">RY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mistry</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tate</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boursnell</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The Pfam protein families database</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">40</span> <span class="fpage" tagx="fpage" title="fpage">D290</span> <span class="lpage" tagx="lpage" title="lpage">D301</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkr1065">10.1093/nar/gkr1065</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22127870">22127870</a> </span> </span> </li> <li tag="ref"> <a name="CR71" /> <span class="label" tagx="label" title="label">71.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Eddy SR. Biological sequence analysis using profile hidden Markov models (Version 3.0 March 2010). [ <a href="http://hmmer.org/">http://hmmer.org/</a>]. </span> </li> <li tag="ref"> <a name="CR72" /> <span class="label" tagx="label" title="label">72.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kent</span> <span class="given-names" tagx="given-names" title="given-names">WJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">BLAT–the BLAST-like alignment tool</span> <span class="source" tagx="source" title="source">Genome Res</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="fpage" tagx="fpage" title="fpage">656</span> <span class="lpage" tagx="lpage" title="lpage">664</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gr.229202.ArticlepublishedonlinebeforeMarch2002">10.1101/gr.229202.ArticlepublishedonlinebeforeMarch2002</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11932250">11932250</a> </span> </span> </li> <li tag="ref"> <a name="CR73" /> <span class="label" tagx="label" title="label">73.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Langmead</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salzberg</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Fast gapped-read alignment with Bowtie 2</span> <span class="source" tagx="source" title="source">Nat Methods</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">357</span> <span class="lpage" tagx="lpage" title="lpage">359</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nmeth.1923">10.1038/nmeth.1923</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286">22388286</a> </span> </span> </li> <li tag="ref"> <a name="CR74" /> <span class="label" tagx="label" title="label">74.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bankevich</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nurk</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Antipov</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gurevich</span> <span class="given-names" tagx="given-names" title="given-names">AA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dvorkin</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kulikov</span> <span class="given-names" tagx="given-names" title="given-names">AS</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">SPAdes: A New Genome Assembly Algorithm and Its Applications to Single-Cell Sequencing</span> <span class="source" tagx="source" title="source">J Comput Biol</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="issue" tagx="issue" title="issue">5</span> <span class="fpage" tagx="fpage" title="fpage">455</span> <span class="lpage" tagx="lpage" title="lpage">477</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1089/cmb.2012.0021">10.1089/cmb.2012.0021</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22506599">22506599</a> </span> </span> </li> <li tag="ref"> <a name="CR75" /> <span class="label" tagx="label" title="label">75.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lohse</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Drechsel</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bock</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">OrganellarGenomeDRAW (OGDRAW) - a tool for the easy generation of high-quality custom graphical maps of plastid and mitochondrial genomes </span> <span class="source" tagx="source" title="source">Curr Genet</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">52</span> <span class="fpage" tagx="fpage" title="fpage">267</span> <span class="lpage" tagx="lpage" title="lpage">274</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00294-007-0161-y">10.1007/s00294-007-0161-y</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17957369">17957369</a> </span> </span> </li> <li tag="ref"> <a name="CR76" /> <span class="label" tagx="label" title="label">76.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wyman</span> <span class="given-names" tagx="given-names" title="given-names">SK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jansen</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boore</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Automatic annotation of organellar genomes with DOGMA</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">20</span> <span class="issue" tagx="issue" title="issue">17</span> <span class="fpage" tagx="fpage" title="fpage">3252</span> <span class="lpage" tagx="lpage" title="lpage">3255</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/bth352">10.1093/bioinformatics/bth352</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15180927">15180927</a> </span> </span> </li> <li tag="ref"> <a name="CR77" /> <span class="label" tagx="label" title="label">77.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thompson</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Higgins</span> <span class="given-names" tagx="given-names" title="given-names">DG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gibson</span> <span class="given-names" tagx="given-names" title="given-names">TJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice </span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">1994</span> <span class="volume" tagx="volume" title="volume">22</span> <span class="fpage" tagx="fpage" title="fpage">4673</span> <span class="lpage" tagx="lpage" title="lpage">4680</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/22.22.4673">10.1093/nar/22.22.4673</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7984417">7984417</a> </span> </span> </li> <li tag="ref"> <a name="CR78" /> <span class="label" tagx="label" title="label">78.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tamura</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stecher</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peterson</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Filipski</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">MEGA6: Molecular Evolutionary Genetics Analysis Version 6.0</span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">30</span> <span class="fpage" tagx="fpage" title="fpage">2725</span> <span class="lpage" tagx="lpage" title="lpage">2729</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/molbev/mst197">10.1093/molbev/mst197</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24132122">24132122</a> </span> </span> </li> <li tag="ref"> <a name="CR79" /> <span class="label" tagx="label" title="label">79.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ashburner</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ball</span> <span class="given-names" tagx="given-names" title="given-names">CA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blake</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Botstein</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Butler</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cherry</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Gene ontology: tool for the unification of biology</span> <span class="source" tagx="source" title="source">Nat Genet</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">25</span> <span class="lpage" tagx="lpage" title="lpage">29</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/75556">10.1038/75556</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10802651">10802651</a> </span> </span> </li> <li tag="ref"> <a name="CR80" /> <span class="label" tagx="label" title="label">80.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Quinlan</span> <span class="given-names" tagx="given-names" title="given-names">AR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hall</span> <span class="given-names" tagx="given-names" title="given-names">IM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">BEDTools: a flexible suite of utilities for comparing genomic features</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">26</span> <span class="issue" tagx="issue" title="issue">6</span> <span class="fpage" tagx="fpage" title="fpage">841</span> <span class="lpage" tagx="lpage" title="lpage">842</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/btq033">10.1093/bioinformatics/btq033</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20110278">20110278</a> </span> </span> </li> <li tag="ref"> <a name="CR81" /> <span class="label" tagx="label" title="label">81.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moriya</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Itoh</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Okuda</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoshizawa</span> <span class="given-names" tagx="given-names" title="given-names">AC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kanehisa</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">KAAS: an automatic genome annotation and pathway reconstruction server</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">35</span> <span class="fpage" tagx="fpage" title="fpage">W182</span> <span class="lpage" tagx="lpage" title="lpage">W185</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkm321">10.1093/nar/gkm321</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17526522">17526522</a> </span> </span> </li> </ul> </div> </div>  </body></html>