<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Sci Rep</span> <span class="iso-abbrev" title="iso-abbrev">Sci Rep</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Scientific Reports</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">2045-2322</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5700130">5700130</a> </span> <span class="publisher-id" title="publisher-id">15478</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41598-017-15478-3">10.1038/s41598-017-15478-3</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Transcriptome and metabolite analyses reveal the complex metabolic genes involved in volatile terpenoid biosynthesis in garden sage ( <i>Salvia officinalis</i>) </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ali</span> <span class="given-names" tagx="given-names" title="given-names">Mohammed</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">Penghui</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">She</span> <span class="given-names" tagx="given-names" title="given-names">Guangbiao</span> </span> <a href="#Aff2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">Daofu</span> </span> <a href="#Aff3">3</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wan</span> <span class="given-names" tagx="given-names" title="given-names">Xiaochun</span> </span> <a href="#Aff2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">Jian</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">jzhao2@qq.com</span> </span> <a href="#Aff1">1</a> </span> <span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0004 1790 4137</span> <span class="institution-id" title="institution-id">grid.35155.37</span> <span class="institution" title="institution">National Key Laboratory of Crop Genetic Improvement, Huazhong Agricultural University, </span> </div> </span> <span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0004 1760 4804</span> <span class="institution-id" title="institution-id">grid.411389.6</span> <span class="institution" title="institution">State Key Laboratory of Tea Plant Biology and Utilization, Anhui Agricultural University, </span> </div> </span> <span class="citation_author_institution" id="Aff3">[3], </span> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2017-11-11</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2017-11-11</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2017</span> </span> <span class="volume" tagx="volume" title="volume">7</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">16074</span> <span class="history" title="history"> <span class="received" title="received">received: 2017-2-1</span> <span class="accepted" title="accepted">accepted: 2017-10-27</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , </span> <span class="license" title="license"> <span class="license-p" title="license-p"> <b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>A large number of terpenoid compounds have been extracted from different tissues of S. officinalis. However, the molecular genetic basis of terpene biosynthesis pathways is virtually unknown. In this study, approximately 6.6 Gb of raw data were generated from the transcriptome of S. officinalis leaves using Illumina HiSeq 2000 sequencing. After filtering and removing the adapter sequences from the raw data, the number of reads reached 21 million, comprising 98 million of high-quality nucleotide bases. 48,671 unigenes were assembled de novo and annotated for establishing a valid database for studying terpenoid biosynthesis. We identified 135 unigenes that are putatively involved in terpenoid metabolism, including 70 mevalonate and methyl-erythritol phosphate pathways, terpenoid backbone biosynthesis genes, and 65 terpene synthase genes. Moreover, five terpene synthase genes were studied for their functions in terpenoid biosynthesis by using transgenic tobacco; most transgenic tobacco plants expressing these terpene synthetic genes produced increased amounts of terpenoids compared with wild-type control. The combined data analyses from the transcriptome and metabolome provide new insights into our understanding of the complex metabolic genes in terpenoid-rich sage, and our study paves the way for the future metabolic engineering of the biosynthesis of useful terpene compounds in S. officinalis. </p> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2017" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Garden sage ( <i>Salvia officinalis</i> L.) belongs to the genus <i>Salvia</i>, which is one of the economically best-known genera due to its vast medicinal properties and rich aromatic oils. The genus <i>Salvia</i> (tribe Mentheae) is the largest of the Lamiaceae family, which comprises nearly 1,000 species. <i>Salvia</i> plants are widely distributed in three regions around the world but mainly exist in Central and South America (~500 species), West Asia (~200 species) and East Asia (~100 species), while the other <i>Salvia</i> species are spread throughout the world <sup> <a href="#CR1">1</a> </sup>. Most of these plants contain various medicinally active components used throughout history in folk medicine, e.g., <i>S</i>. <i>japonica</i>, <i>S</i>. <i>tuxtlensis</i>, <i>S</i>. <i>guaranitica</i>, <i>S</i>. <i>miltiorrhiza</i>, <i>S</i>. <i>chloroleuca</i>, <i>S</i>. <i>aureus</i>, <i>S</i>. <i>przewalskii</i>, <i>S</i>. <i>epidermidis</i>, <i>S</i>. <i>santolinifolia</i>, <i>S</i>. <i>hydrangea</i>, <i>S</i>. <i>tomentosa</i>, <i>S</i>. <i>isensis</i>, <i>S</i>. <i>lavandulifolia</i>, <i>S</i>. <i>glabrescens</i>, <i>S</i>. <i>nipponica</i>, <i>S</i>. <i>fruticosa</i>, <i>S</i>. <i>allagospadonopsis</i>, <i>S</i>. <i>macrochlamys</i> and <i>S</i>. <i>recognita</i>. Recently, <i>Salvia</i> species have become a valuable source for pharmaceutical research for identifying and discovering biologically active compounds <sup> <a href="#CR2">2</a> </sup>. Essential oils of <i>Salvia</i> species exhibit significant bioactivities, including antimutagenic, anticancer, antimicrobial, anti-inflammatory, choleretic, antioxidant and antimicrobial activities. <i>Salvia</i> essential oils contain more than 100 active compounds with pharmacological effects, and they can be categorized into monoterpenes, sesquiterpenes, diterpenes, and triterpenes <sup> <a href="#CR2">2</a> </sup>. During their biosynthesis, these terpenoids are sequentially built up from the isoprene unit (C5) building blocks, isopentyl diphosphates (IPP) and dimethylallyl diphosphate (DMADP). These components are condensed in a sequential manner by prenyltransferases, resulting in the formation of prenyl diphosphates, such as diphosphate (GPP), farnesyl pyrophosphate (FPP), and geranylgeranyl pyrophosphate (GGPP) <sup> <a href="#CR3">3</a> </sup>. These prenyl diphosphates are the immediate precursors for the biosynthesis of mono-, sesqui-, di- and tetraterpenes. Despite the scientific and medicinal interest in these terpenoids of <i>S</i>. <i>officinalis</i>, the genes that are involved in the biosynthesis of these compounds have not yet been fully identified or understood. Plant secondary metabolites have significant use in the food and pharmaceutical industries, such as in fine chemicals and cosmetics. The biosynthesis, regulation and metabolic engineering of useful secondary metabolites have been extensively studied <sup> <a href="#CR4">4</a> </sup>. In recent years, next-generation sequencing (NGS)-based RNA sequencing (RNA-Seq) has become a powerful tool for discovering genes that are involved in the biosynthesis of various secondary metabolite pathways in medicinal plants <sup> <a href="#CR5">5</a> </sup>. For example, the phenylpropanoid and terpenoid biosynthesis pathways in <i>Ocimum sanctum</i> and <i>Ocimum basilicum</i> <sup> <a href="#CR6">6</a> </sup>, the biosynthesis of active ingredients in <i>Salvia miltiorrhiza</i> <sup> <a href="#CR7">7</a> </sup>, the biosynthesis of carotenoids in <i>Momordica cochinchinensis</i> <sup> <a href="#CR8">8</a> </sup>, the biosynthesis of cellulose and lignin in Chinese fir ( <i>Cunninghaimia lanceolata</i>) <sup> <a href="#CR9">9</a> </sup>, and the biosynthesis of tea-specific compounds, i.e., catechins, caffeine and theanine pathways in tea ( <i>Camellia sinensis</i>) <sup> <a href="#CR10">10</a> </sup>, have been explored using NGS. Characterization of plant terpene synthases (TPSs) is typically carried out by the production of the recombinant enzymes in <i>Escherichia coli</i>. This is often difficult due to enzyme solubility and codon usage issues. Furthermore, plant terpene synthases that are localized to the plastids, such as diterpene synthases, must be abridged in a more or less experimental approach to ameliorate expression <sup> <a href="#CR11">11</a>, <a href="#CR12">12</a> </sup>. Transgenic tobacco ( <i>Nicotiana tabacum</i>) is very efficient and has been successfully used for the characterization of two diterpene genes in glandular trichomes: labdane and Z-abienol <sup> <a href="#CR13">13</a> </sup>. Here, we characterized genes that are involved in terpenoid biosynthesis in <i>S</i>. <i>officinalis</i> and determined their biological significance in <i>S</i>. <i>officinalis</i> for terpenoid production in various tissues. In this study, a transcriptome database was established for <i>S</i>. <i>officinalis</i> leaves using NGS technology to identify and to characterize genes that are related to the terpenoid biosynthesis pathway. The criteria used to achieve these objectives and to elucidate the complex metabolic pathways and genes for the understanding of terpenoid production in <i>S</i>. <i>officinalis</i> included the following: (i) transcriptome analysis of leaves using Illumina HiSeq 2000 sequencing; (ii) Gas Chromagraphy coupled Mass Spectrometry (GC-MS) analysis for three fresh plant parts (old leaves, young leaves, and stems); (iii) characterization of five terpene genes in transgenic <i>N</i>. <i>tabacum</i>; (iv) qRT-PCR of highly expressed genes that are involved in the biosynthesis of terpenoids; (v) and the combination of data from the transcriptome, qRT-PCR, and metabolome with GC-MS for revealing the functions of metabolic genes that are involved in the biosynthesis of valuable terpenoids. </p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results and Discussion</div> <div class="identificationofessentialoilcomponents" title="sec"> <div class="title" tagx="title" title="title">
Identification of essential oil components</div> <p>For GC-MS analysis, 236 bioactive phytochemical compounds were identified using n-hexane extracts from three fresh aerial parts of <i>S</i>. <i>officinalis</i>. The numbers of obtained bioactive phytochemical compounds from young leaves, old leaves and stems were 113 (89.29%), 108 (91.54%) and 82 (85.27%), respectively. The results of the qualitative and quantitative analyses of all phytochemical compounds from the essential oils are reported in (Table <a href="#Tab1">1</a> and Supplementary Table <a href="#MOESM1">S1</a>). The identified phytochemical compounds are listed based on the retention time, compound mass and percentage of peak area (Fig. <a href="#Fig1">1A,B</a>). In young leaves, the monoterpene compounds were shown as the main group (66.64%), followed by the group of sesquiterpene compounds (15.87%) and diterpene compounds (1.4%). In old leaves, the monoterpene compounds were observed to be the main group (52.7%), followed by the sesquiterpene group (15.01%) and the diterpene group (14.18%), and only one triterpene compound represented 0.16%. Sesquiterpenes form the main group of compounds (23%) found in the stems, followed by diterpenes (19.53%), monoterpenes (19.11%) and and one triterpene compound represented 0.02% (Supplementary Table <a href="#MOESM1">S1</a>). Moreover, the three hexane extracts from the different tissues for essential oils contained unique, common and major phytochemical compounds. For example, the essential oil extracts of young leaves (A) had 61 unique compounds, 35 common compounds shared with the essential oil extracts from old leaves, five common compounds shared with the essential oil extract from stems and 12 common compounds shared among all three plant parts. Furthermore, the old leaves (B) contained 57 unique compounds and four common compounds shared with the stems. On the other hand, the stems (C) contained 61 unique compounds (Fig. <a href="#Fig1">1C</a>). Regarding the major phytochemical compounds, 1,8-cineole (41.20%) was the major compound in the essential oil extracts from young leaves, followed by β-caryophyllene (9.01%), camphor (6.27%), β-pinene (6.23%), and α-terpinenyl acetate (4.23%), whereas the essential oil extracts of old leaves was characterized by 1,8-cineole (25.93%), followed by camphor (11.52%), sugiol (10.80%), β-caryophyllene (5.51%), and α-caryophyllene (3.72%). Sugiol was characterized as the major compound of stem extracts (15.89%), followed by 1,8-cineole (12.37%), β-caryophyllene (10.23%), α-caryophyllene (7.30%), and then isocaryophyllene oxide (3.24%) (Table <a href="#Tab1">1</a>). When comparing the composition of the three essential oil extracts of <i>S</i>. <i>officinalis</i>, we deduced that some common compounds exist at different levels within the parts of <i>S</i>. <i>officinalis</i> (Fig. <a href="#Fig1">1A</a>). Additionally, some of the compounds that have been found in <i>S</i>. <i>officinalis</i> were detected in other <i>Salvia</i> plant species (Table <a href="#Tab1">1</a> and Supplementary Table <a href="#MOESM1">S1</a>) <sup> <a href="#CR14">14</a>– <a href="#CR16">16</a> </sup>. Therefore, we suggest that plant parts can have a major effect on the composition of their essential oils. From these and previous GC-MS data <sup> <a href="#CR17">17</a>, <a href="#CR18">18</a> </sup>, an important question has been raised: why do the monoterpene compounds of <i>S</i>. <i>officinalis</i> mostly accumulated in young leaves? This question was difficult to answer before conducting the present work because there was a lack of information at the genetic level regarding the terpenoid biosynthetic pathway and how these compounds are synthesized in <i>S</i>. <i>officinalis</i>. <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  </div></p><p>The major chemical compositions in the essential oils of <i>S</i>. <i>officinalis</i>. </p>  <table> <thead> <tr> <th>N</th> <th>Compound name</th> <th>Retention time (min.)</th> <th>Retention time index</th> <th>Formula</th> <th>Molecular Mass (g mol <sup>−1</sup>) </th> <th>Terpene type</th> <th>Young leaf</th> <th>Old leaf</th> <th>Stem</th> <th>O.S.S</th> </tr> <tr> <th>% Peak area</th> <th>% Peak area</th> <th>% Peak area</th> </tr> </thead> <tbody> <tr> <td>1</td> <td>(−)-α-Pinene,</td> <td>5.82</td> <td>934</td> <td>C10H16</td> <td>136.23</td> <td>Mono</td> <td> <b>2.14</b> </td> <td> <b>1.96</b> </td> <td> <b>0.35</b> </td> <td>S.L, S.A, S.F, S.C</td> </tr> <tr> <td>2</td> <td>Camphene-</td> <td>6.48</td> <td>941</td> <td>C10H16</td> <td>136.23</td> <td>Mono</td> <td> <b>0.54</b> </td> <td> <b>2.17</b> </td> <td> <b>0.15</b> </td> <td>S.L, S.A, S.F, S.C</td> </tr> <tr> <td>3</td> <td>(±)-Sabinene</td> <td>7.56</td> <td>977</td> <td>C10H16</td> <td>136.23</td> <td>Mono</td> <td> <b>0.37</b> </td> <td> <b>0.16</b> </td> <td>—</td> <td>S.L, S.A</td> </tr> <tr> <td>4</td> <td>beta.-Pinene</td> <td>7.77</td> <td>980</td> <td>C10H16</td> <td>136.23</td> <td>Mono</td> <td> <b>6.23</b> </td> <td> <b>3.19</b> </td> <td> <b>1.28</b> </td> <td>S.L, S.A, S.F, S.C</td> </tr> <tr> <td>5</td> <td>1,8-cineole</td> <td>11.29</td> <td>1028</td> <td>C10H18O</td> <td>154.24</td> <td>Mono</td> <td> <b>41.2</b> </td> <td> <b>25.93</b> </td> <td> <b>12.37</b> </td> <td>S.L, S.A, S.F</td> </tr> <tr> <td>6</td> <td>p-Menth-8-en-1-ol</td> <td>15.47</td> <td>1161</td> <td>C10H18O</td> <td>154.24</td> <td>Mono</td> <td> <b>0.07</b> </td> <td> <b>0.15</b> </td> <td>—</td> <td /> </tr> <tr> <td>7</td> <td>δ-Thujone,</td> <td>15.68</td> <td>1103</td> <td>C10H16O</td> <td>152.23</td> <td>Mono</td> <td> <b>1.26</b> </td> <td> <b>0.56</b> </td> <td>—</td> <td /> </tr> <tr> <td>8</td> <td>Thujone</td> <td>16.31</td> <td>1110</td> <td>C10H16O</td> <td>152.23</td> <td>Mono</td> <td> <b>0.51</b> </td> <td> <b>0.17</b> </td> <td>—</td> <td>S.F</td> </tr> <tr> <td>9</td> <td>(−)-2-bornanone</td> <td>17.69</td> <td>1146</td> <td>C10H18O</td> <td>154.24</td> <td>Mono</td> <td>—</td> <td>—</td> <td> <b>1.62</b> </td> <td>S.L, S.A, S.F, S.C</td> </tr> <tr> <td>10</td> <td>Camphor</td> <td>17.70</td> <td>1139</td> <td>C10H16O</td> <td>152.23</td> <td>Mono</td> <td> <b>6.27</b> </td> <td> <b>11.52</b> </td> <td>—</td> <td>S.L, S.A, S.F</td> </tr> <tr> <td>11</td> <td>Terpinen-4-ol</td> <td>19.33</td> <td>1164</td> <td>C10H18O</td> <td>154.24</td> <td>Mono</td> <td> <b>0.06</b> </td> <td> <b>0.16</b> </td> <td>—</td> <td>S.L, S.A, S.C</td> </tr> <tr> <td>12</td> <td>Bornyl acetate</td> <td>23.54</td> <td>1276</td> <td>C17H24O4</td> <td>196.28</td> <td>Mono</td> <td> <b>0.24</b> </td> <td> <b>1.40</b> </td> <td> <b>0.51</b> </td> <td>S.F</td> </tr> <tr> <td>13</td> <td>α-Terpinenyl acetate</td> <td>24.88</td> <td>1341</td> <td>C12H20O2</td> <td>196.28</td> <td>Mono</td> <td> <b>4.23</b> </td> <td> <b>1.26</b> </td> <td> <b>1.52</b> </td> <td /> </tr> <tr> <td>14</td> <td>α-Gurjunene,</td> <td>27.76</td> <td>1409</td> <td>C15H24</td> <td>204.35</td> <td>Sesqui</td> <td> <b>0.03</b> </td> <td> <b>0.02</b> </td> <td>—</td> <td>S.L, S.A</td> </tr> <tr> <td>15</td> <td>Isocaryophyllene</td> <td>28.20</td> <td>1409</td> <td>C15H24</td> <td>204.35</td> <td>Sesqui</td> <td> <b>9.01</b> </td> <td> <b>5.51</b> </td> <td> <b>10.23</b> </td> <td>S.F</td> </tr> <tr> <td>16</td> <td>Beta.-copaene</td> <td>28.60</td> <td>1455</td> <td>C15H24</td> <td>204.35</td> <td>Sesqui</td> <td> <b>0.02</b> </td> <td> <b>0.01</b> </td> <td>—</td> <td /> </tr> <tr> <td>17</td> <td>α-caryophyllene</td> <td>29.41</td> <td>1442</td> <td>C15H24</td> <td>204.35</td> <td>Sesqui</td> <td> <b>2.22</b> </td> <td> <b>3.72</b> </td> <td> <b>7.30</b> </td> <td>S.L, S.A</td> </tr> <tr> <td>18</td> <td>(−)-Germacrene D,</td> <td>30.09</td> <td>1482</td> <td>C15H24</td> <td>204.35</td> <td>Sesqui</td> <td> <b>0.06</b> </td> <td> <b>0.14</b> </td> <td> <b>1.29</b> </td> <td /> </tr> <tr> <td>19</td> <td>Gamma.-Elemene</td> <td>30.73</td> <td>1433</td> <td>C15H24</td> <td>204.35</td> <td>Sesqui</td> <td> <b>0.8</b> </td> <td>—</td> <td> <b>0.36</b> </td> <td /> </tr> <tr> <td>20</td> <td>Caryophyllene oxide</td> <td>3.43</td> <td>1573</td> <td>C15H24</td> <td>204.35</td> <td>Sesqui</td> <td> <b>0.48</b> </td> <td> <b>2.13</b> </td> <td> <b>3.24</b> </td> <td>S.L, S.A, S.N</td> </tr> <tr> <td>21</td> <td>Geranyl-.alpha.-terpinene</td> <td>39.88</td> <td>2223</td> <td>C20H32</td> <td>272.46</td> <td>Diter</td> <td> <b>0.06</b> </td> <td> <b>0.03</b> </td> <td>—</td> <td>S.L, S.A, S.F</td> </tr> <tr> <td>22</td> <td>Isoaromadendrene epoxide</td> <td>40.79</td> <td>1623</td> <td>C15H24O</td> <td>220.35</td> <td>Sesqui</td> <td> <b>0.4</b> </td> <td>—</td> <td> <b>0.08</b> </td> <td>S.L, S.A</td> </tr> <tr> <td>23</td> <td>Beta.-ylangene</td> <td>41.45</td> <td>1421</td> <td>C15H24</td> <td>204.35</td> <td>Sesqui</td> <td> <b>0.96</b> </td> <td> <b>0.36</b> </td> <td>—</td> <td /> </tr> <tr> <td>24</td> <td>Labda-8(20), Biformene</td> <td>42.35</td> <td>2004</td> <td>C20H32</td> <td>272.46</td> <td>Diter</td> <td> <b>0.17</b> </td> <td> <b>0.25</b> </td> <td>—</td> <td>S.L, S.A</td> </tr> <tr> <td>25</td> <td>Kaur-16-en-18-yl acetate</td> <td>42.79</td> <td>1997</td> <td>C22H34O2</td> <td>330.50</td> <td>Diter</td> <td> <b>0.02</b> </td> <td> <b>0.06</b> </td> <td>—</td> <td /> </tr> <tr> <td>26</td> <td>β-cis-Caryophyllene</td> <td>43.82</td> <td>1425</td> <td>C15H24</td> <td>204.35</td> <td>Sesqui</td> <td> <b>0.02</b> </td> <td> <b>0.03</b> </td> <td>—</td> <td>S.F</td> </tr> <tr> <td>27</td> <td>Epimanool</td> <td>45.80</td> <td>2056</td> <td>C20H34O</td> <td>290.48</td> <td>Diter</td> <td> <b>2.07</b> </td> <td> <b>2.10</b> </td> <td> <b>167</b> </td> <td /> </tr> <tr> <td>28</td> <td>Humulane-1,6-dien-3-ol</td> <td>50.99</td> <td>1619</td> <td>C15H26O</td> <td>222.36</td> <td>Sesqui</td> <td> <b>0.87</b> </td> <td> <b>0.91</b> </td> <td>—</td> <td /> </tr> <tr> <td>29</td> <td>Ferruginol</td> <td>52.11</td> <td>2330</td> <td>C20H30O</td> <td>286.45</td> <td>Diter</td> <td> <b>0.51</b> </td> <td> <b>0.63</b> </td> <td>—</td> <td /> </tr> <tr> <td>30</td> <td>Sugiol</td> <td>53.07</td> <td>2659.9</td> <td>C20H28O2</td> <td>300.43</td> <td>Diter</td> <td> <b>0.47</b> </td> <td> <b>10.80</b> </td> <td> <b>15.89</b> </td> <td /> </tr> <tr> <td>31</td> <td>Totarol</td> <td>60.84</td> <td>2260</td> <td>C20H30O</td> <td>286.45</td> <td>Diter</td> <td>—</td> <td> <b>0.13</b> </td> <td>—</td> <td /> </tr> <tr> <td>32</td> <td>Squalene</td> <td>74.12</td> <td>2817</td> <td>C30H50</td> <td>410.71</td> <td>Diter</td> <td>—</td> <td> <b>0.16</b> </td> <td>—</td> <td /> </tr> <tr> <td>33</td> <td>2-methyloctacosane</td> <td>79.90</td> <td>2864</td> <td>C29H60</td> <td>408.78</td> <td>Diter</td> <td> <b>0.12</b> </td> <td> <b>0.23</b> </td> <td> <b>0.65</b> </td> <td /> </tr> <tr> <td /> <td>Total</td> <td /> <td /> <td /> <td /> <td /> <td> <b>89.29</b> </td> <td> <b>91.54</b> </td> <td> <b>85.27</b> </td> <td /> </tr> <tr> <td>Total Precentage of Monoterpenes</td> <td /> <td /> <td /> <td> <b>66.64</b> </td> <td> <b>52.7</b> </td> <td> <b>19.11</b> </td> <td /> </tr> <tr> <td>Total Precentage of Sesquiterpenes</td> <td /> <td /> <td /> <td> <b>15.87</b> </td> <td> <b>15.01</b> </td> <td> <b>23.12</b> </td> <td /> </tr> <tr> <td>Total Precentage of Triterpenes</td> <td /> <td /> <td /> <td /> <td> <b>0.16</b> </td> <td> <b>0.02</b> </td> <td /> </tr> <tr> <td>Total Precentage of Diterpenes</td> <td /> <td /> <td /> <td> <b>1.4</b> </td> <td> <b>14.18</b> </td> <td> <b>19.53</b> </td> <td /> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>Abbreviations: R.T: Retention Time, O.S.S: Other salvia species, SA: <i>Salvia acetabulosa</i>, S.L: <i>Salvia leriifolia</i>, S.F: <i>Salvia fruticosa</i>, S.N: <i>Salvia nemorosa</i>, S.C: <i>Salvia compressa</i>. </p> </div>  <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <p>Typical GC-MS mass spectragraphs for terpenoids from young leaf, old leaf, and stem of <i>Salvia officinalis</i>. ( <b>A</b>) GC-MS peaks of the essential oil extracts, ( <b>B</b>) Mass spectrum of GC peaks with retention time for the major compound. ( <b>C</b>) Three-Way-Venn-Diagram to show the number of unique and common compounds in the essential oil extracts from young leaf ( <b>A</b>), old leaf ( <b>B</b>), and stem ( <b>C</b>) of <i>Salvia officinalis</i>. </p>   </div> <p /> </div> <div class="illuminasequencingandthedenovoassemblyofthes.officinalisleaftranscriptome" title="sec"> <div class="title" tagx="title" title="title">
Illumina sequencing and the <i>de novo</i> assembly of the <i>S</i>. <i>officinalis</i> leaf transcriptome </div> <p>In the past few years, the Illumina sequencing platform has become a powerful method for analysing and discovering the genomes of non-model plants <sup> <a href="#CR19">19</a>, <a href="#CR20">20</a> </sup>. In this context, to generate transcriptome sequences, complementary DNA (cDNA) libraries were prepared from leaf tissues of <i>S</i>. <i>officinalis</i>, and cDNA was then sequenced using paired-end reads (PE) sequencing using an Illumina HiSeq 2000 platform. Previous reports involving Illumina sequencing reported that the use of PE sequencing showed significant improvement in the efficiency of <i>de novo</i> assembly and increased the depth of sequencing <sup> <a href="#CR9">9</a>, <a href="#CR21">21</a> </sup>. The cDNA sequencing generated 6.6 Gb of raw data from <i>S</i>. <i>officinalis</i> leaves. After filtering and removing the adapter sequences from the raw data, the number of reads was 21,487,871 (21.48 million), comprising of 98,521,170 high-quality nucleotide bases, with 95.90% Q20, 91.69% Q30 and 48.73% GC content. For further analysis, high-quality reads were selected, and the transcriptome was assembled using the Trinity program <sup> <a href="#CR22">22</a> </sup>, which produced 88,554 transcripts with an N50 length of 1,793 bp, an N90 length of 479 bp and a mean length of 1,113 bp. Moreover, 48,671 unigenes could be detected with an N50 length of 1,485 bp, an N90 length of 298 bp and a mean length of 813 bp. The distribution of the assembled transcript length ranged from 200 to &amp;gt;2,000 bases; the maximum number of transcripts (34,051 transcripts, 38.45%) ranged from 200 bp to 500 bp, followed by 22,529 transcripts (25.44%) ranging from 1,000 to 2,000 bp and then 17,658 transcripts (19.94%) ranging from 500 to 1,000 bp. On the contrary, the lowest number of transcripts (14,316 transcripts, 16.17%) was obtained for a size of more than 2,000 bp. By contrast, the assembled unigene lengths were distributed between 200 and &amp;gt;2,000 bp. The maximum number of unigenes (27,381 unigenes, 56.26%) ranged from 200 to 500 bp, followed by 8,576 unigenes (17.62%) ranging from 500 to 1,000 bp and then 8.068 unigenes (16.58%) ranging from 1000 to 2,000 bp. Finally, the lowest number of unigenes (4,646 unigenes, 9.54%) was obtained for a size of &amp;gt;2000 bp. The length distributions of the transcripts and unigenes are shown in Supplementary Table <a href="#MOESM1">S2</a> and Fig. <a href="#MOESM1">S1</a>. Our results are in good agreement with those for <i>Boehmeria nivea</i>, <i>Medicago sativa</i>, <i>C</i>. <i>Longa</i>, <i>Centella asiatica</i> and <i>Apium graveolens</i>, in which the largest number of both transcript and unigene lengths were found to range between 75 and 500 bp <sup> <a href="#CR23">23</a>, <a href="#CR24">24</a> </sup>. </p> </div> <div class="functionalannotationandclassificationofassembleds.officinalisunigenes" title="sec"> <div class="title" tagx="title" title="title">
Functional annotation and classification of assembled <i>S</i>. <i>officinalis</i> unigenes </div> <p>The total number of unigenes (48,671, 100% of all unigenes) was compared against the public dabases, including the NCBI non-redundant protein sequences (NR), the NCBI nucleotide sequences (NT), the Kyoto Encyclopedia of Genes and Genomes (KEGG), the KEGG orthology (KO), Swiss-Prot, the protein family annotation (PFAM), Gene Ontology (GO), and the euKaryotic Ortholog Groups database (KOG) annotation databases (Supplementary Table <a href="#MOESM1">S3</a> and Fig. <a href="#MOESM1">S2</a>). The annotation percentage results in this research were higher than the annotation percentages in other non-model plant studies [58% in safflower ( <i>Carthamus tinctorius</i>) and 58.01% in Chinese fir ( <i>C</i>. <i>lanceolata</i>)] <sup> <a href="#CR9">9</a>, <a href="#CR25">25</a>, <a href="#CR26">26</a> </sup>. The international standardized gene functional annotation system (GO Annotation) provides a powerful way to recognize the functions and properties of sequences that have not been characterized for an organism <sup> <a href="#CR27">27</a> </sup>. The BLAST2 GO program was used to categorize the functions of these annotated unigenes, and a total of 22,891 unigenes (47.03% of all of the assembled unigenes) were mapped to at least one GO term. Based on sequence homology, the unigene sequences from <i>S</i>. <i>officinalis</i> were categorized into 48 functional groups under three general sections: 59,883 were assigned to the biological process (BP), 43,029 were assigned to the cellular component (CC) and 29,760 were assigned to the molecular function (MF) sections. As a result, cellular process (13,933) and metabolic process (13,423) were the most enriched GO terms in the biological process (BP) section. Regarding the CC section, the cell (8,737) and cell part (8,720) were the most enriched. Within the molecular function (MF) section, binding (13,539) and catalytic activity (11,726) were highly enriched (Fig. <a href="#Fig2">2</a>). These results revealed that the main Gene Ontology (GO) classifications in the annotated unigenes were responsible for metabolism and fundamental biological regulation. These results were similar to previous results with the <i>S</i>. <i>miltiorrhiza</i> transcriptome and with the transcriptomes of <i>O</i>. <i>sanctum</i> and <i>O</i>. <i>basilicum</i> (members of the same family), which have the highest percentages of metabolic process, cellular process, cell, cell part, binding and catalytic activity <sup> <a href="#CR28">28</a>, <a href="#CR29">29</a> </sup>. Moreover, these results are in agreement with previous studies on <i>de novo</i> transcriptome assembly in the tuberous root of sweet potato, <i>de novo</i> transcriptome sequencing from <i>R</i>. <i>sativus</i> and <i>de novo</i> characterization of roots from the Chinese medicinal plant <i>P</i>. <i>cuspidatum</i> <sup> <a href="#CR26">26</a>, <a href="#CR29">29</a> </sup>. The lowest percentage of unigenes categories included channel regulator activity (56), cell junction (28) and cell killing (27). Therefore, the present work suggests that the enormous potential data that exist in the Gene Ontology (GO) classifications can be used to identify the new genes. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <p>Functional annotation and classification of assembled unigenes from <i>S</i>. <i>officinalis</i>. Gene Ontology (GO) terms are summarized in three general sections of the biological process (BP), cellular component (CC) and molecular function (MF). </p>   </div> <p /> </div> <div class="kegganalysisofs.officinalistranscriptomes" title="sec"> <div class="title" tagx="title" title="title">
KEGG analysis of <i>S</i>. <i>officinalis</i> transcriptomes </div> <p>KEGG pathway database can facilitate the understanding of the functional annotations of enzymes and the biological functions of genes regarding their networks <sup> <a href="#CR7">7</a>, <a href="#CR30">30</a> </sup>. To identify active biological functional pathways in the leaf tissues of <i>S</i>. <i>officinalis</i>, all 48,671 unigene sequences were mapped in reference to the canonical pathways of KEGG, but 9,716 (19.96%) unigene sequences could be assigned to 267 KEGG pathways. Furthermore, all transcripts were classified into five larger pathway categories, including cellular processes, environmental information processing, genetic information processing, metabolism and organismal systems (Fig. <a href="#Fig3">3</a>). The highest number of transcripts from <i>S</i>. <i>officinalis</i> was assigned to the metabolism category, followed by genetic information processing, organismal systems, and cellular processes, whereas the lowest number of transcripts was related to the category of environmental information processing. Interestingly, 608 transcripts of <i>S</i>. <i>officinalis</i> were related to the biosynthesis of various secondary metabolite pathways, which were sorted into 27 subcategories, with phenylpropanoid biosynthesis (ko00940), terpenoid backbone biosynthesis (ko00900) and carotenoid biosynthesis (ko00906) representing the largest subcategories (Supplementary Table <a href="#MOESM1">S4</a>). These results were in agreement with previous results from the transcriptomes of <i>O</i>. <i>sanctum</i> and <i>O</i>. <i>basilicum</i>, which are members of the same family, and from <i>de novo</i> transcriptome sequencing from <i>R</i>. <i>sativus</i>, the transcriptome of which had the highest percentages of phenylpropanoid biosynthesis and terpenoid backbone biosynthesis <sup> <a href="#CR6">6</a>, <a href="#CR9">9</a> </sup>. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <p>KEGG classified into five largest categories pathways includes cellular processes ( <b>A</b>), environmental information processing ( <b>B</b>), genetic information processing ( <b>C</b>), metabolism ( <b>D</b>) and organismal systems ( <b>E</b>). </p>   </div> <p /> </div> <div class="genesrelatedtothebiosynthesisofisoprenoids" title="sec"> <div class="title" tagx="title" title="title">
Genes related to the biosynthesis of isoprenoids</div> <p>Various types of terpenoids were found in the essential oil extracts of <i>S</i>. <i>officinalis</i>. The mixture contained mainly myrcene, (+)-neomenthol, 1,8-cineole, (3S)-linalool, α-humulene/β-caryophyllene, momilactone-A, gibberellin 3, gibberellin 2, ent-copalyl diphosphate, ent-kaurene, ent-kaurenoic acid, ent-isokaurene C2, gibberellin 20, and beta-amyrin. Precursor molecules for terpenoid biosynthesis are derived from the cytosolic mevalonate (MVA) and plastidial methyl-erythritol phosphate (MEP) pathways. Therefore, queries against the Lamiaceae family transcriptome libraries were applied to identify and to determine genes that encode enzymes involved in the different steps of the terpenoid biosynthesis pathway, such as Mevalonate diphosphate decarboxylase, Isopentenyl phosphate kinase, isopentenyl pyrophosphate isomerase for swithing IPP to DMAPP isomerase, GPS (geranyl pyrophosphate synthase), FPS (farnesyl pyrophosphate synthase) and GGPS (geranylgeranyl pyrophosphate synthase) <sup> <a href="#CR31">31</a>, <a href="#CR32">32</a> </sup>. Furthermore, we identified and estimated the expression levels of isoprenoid genes by using uniprot annotations against the transcriptome libraries (Table <a href="#Tab2">2</a>). From the annotation data analyses, we found many transcript genes related to isoprenoid biosynthesis from the MEP pathway with higher expression levels, including gene transcripts such as <i>SoDXS4</i>, <i>1</i>(1-deoxy-D-xylulose-5-phosphate synthase 4, 1), <i>SoDXR</i> (1-deoxy-D-xylulose-5-phosphate reductoisomerase), <i>SoMCT</i> (2-C-methyl-D-erythritol 4-phosphate cytidylyltransferase), <i>SoISPF</i> (2-C-methyl-D-erythritol 2,4-cyclodiphosphate synthase), <i>SoHDS2</i> ((E)-4-hydroxy-3-methylbut-2-enyl-diphosphate synthase 2), <i>SoHDR2</i>,3 (4-hydroxy-3-methylbut-2-enyl diphosphate reductase 2, 3) and <i>SoIDI1</i> (isopentenyl diphosphate isomerase1). Additionally, we obtained some gene transcripts that were related to isoprenoid biosynthesis from the MVA pathway with higher expression levels, such as <i>SoAACT1</i>, 4 (acetyl-CoA C-acetyltransferase 1, 4), <i>SoHMGS</i> (hydroxymethyl glutaryl-CoA synthase), <i>SoHMGR</i>4, 3, 2 (hydroxymethyl glutaryl-CoA reductase 4, 3, 2) <i>SoMVK</i> (mevalonate kinase) and <i>SoPMK</i> (phospho-mevalonate kinase). Moreover, the transcriptome dataset of <i>S</i>. <i>officinalis</i> presented other genes, such as <i>SoGPS</i>, <i>SoFPS2</i>, and <i>SoGGPSΙΙ10</i>, which are the immediate precursor of the mono-, sesqui-, and di-terpene biosynthesis pathway. The <i>So</i>GPS, <i>So</i>FPS2, and <i>So</i>GGPSΙΙ10 genes were highly abundant in leaves and had higher values of fragments per kilobase of transcripts per million mapped fragments (FPKM), which were 20.23, 281.11 and 49.23, respectively (Fig. <a href="#Fig4">4</a> and Table <a href="#Tab2">2</a>). Our results were similar to previously obtained results from the transcriptomes of <i>O</i>. <i>sanctum</i> and <i>O</i>. <i>basilicum</i>, which are members of the same family and have a higher number of transcripts for the DXS and GPPS genes related to the terpenoid biosynthesis pathway <sup> <a href="#CR6">6</a> </sup>. <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 2</span>  </div></p><p>Transcript abundance of MEP, MVA and other terpenoid backbone biosynthesis pathway genes as per the <i>S</i>. <i>officinalis</i> transcriptome data annotation. </p>  <table> <thead> <tr> <th>Pathway</th> <th>Gene name</th> <th>Kegg Entry</th> <th>Unigene ID</th> <th>EC. No.</th> <th>Gene length</th> <th>Read in leaf</th> <th>FPKM</th> </tr> </thead> <tbody> <tr> <td>MEP</td> <td> <i>SoDXS1</i> </td> <td>K01662</td> <td> <i>SO|comp28301_c0</i> </td> <td>2.2.1.7</td> <td>2113</td> <td>2308</td> <td>72.96</td> </tr> <tr> <td> <i>SoDXS2</i> </td> <td>K01662</td> <td> <i>SO|comp3312_c0</i> </td> <td>2.2.1.7</td> <td>2536</td> <td>153</td> <td>4.36</td> </tr> <tr> <td> <i>SoDXS3</i> </td> <td>K01662</td> <td> <i>SO|comp528454_c0</i> </td> <td>2.2.1.7</td> <td>329</td> <td>3</td> <td>1.38</td> </tr> <tr> <td> <i>SoDXS4</i> </td> <td>K01662</td> <td> <i>SO|comp10248_c0</i> </td> <td>2.2.1.7</td> <td>2600</td> <td>7406.34</td> <td>186.68</td> </tr> <tr> <td> <i>SoDXS5</i> </td> <td>K01662</td> <td> <i>SO|comp28480_c0</i> </td> <td>2.2.1.7</td> <td>537</td> <td>477.9</td> <td>85.25</td> </tr> <tr> <td> <i>SoDXS6</i> </td> <td>K01662</td> <td> <i>SO|comp404816_c0</i> </td> <td>2.2.1.7</td> <td>417</td> <td>5</td> <td>1.38</td> </tr> <tr> <td> <i>SoDXS7</i> </td> <td>K01662</td> <td> <i>SO|comp4108_c0</i> </td> <td>2.2.1.7</td> <td>1360</td> <td>75</td> <td>3.91</td> </tr> <tr> <td> <i>SoDXS8</i> </td> <td>K01662</td> <td> <i>SO|comp15712_c0</i> </td> <td>2.2.1.7</td> <td>961</td> <td>83</td> <td>7.45</td> </tr> <tr> <td> <i>SoDXR</i> </td> <td>K00099</td> <td> <i>SO|comp10244_c0</i> </td> <td>1.1.1.267</td> <td>2044</td> <td>4842.56</td> <td>158.81</td> </tr> <tr> <td> <i>SoMCT</i> </td> <td>K00991</td> <td> <i>SO|comp17627_c0</i> </td> <td>2.7.7.60</td> <td>1180</td> <td>373</td> <td>22.99</td> </tr> <tr> <td> <i>SoISPF</i> </td> <td>K01770</td> <td> <i>SO|comp7621_c0</i> </td> <td>4.6.1.12</td> <td>1076</td> <td>1173</td> <td>80.86</td> </tr> <tr> <td> <i>SoHDS1</i> </td> <td>K03526</td> <td> <i>SO|comp23789_c0</i> </td> <td>1.17.7.1</td> <td>382</td> <td>43</td> <td>14.11</td> </tr> <tr> <td> <i>SoHDS2</i> </td> <td>K03526</td> <td> <i>SO|comp10199_c0</i> </td> <td>1.17.7.1</td> <td>4859</td> <td>12460.35</td> <td>161.86</td> </tr> <tr> <td> <i>SoHDS3</i> </td> <td>K03526</td> <td> <i>SO|comp23789_c1</i> </td> <td>1.17.7.1</td> <td>1511</td> <td>161</td> <td>7.89</td> </tr> <tr> <td> <i>SoHDS4</i> </td> <td>K03526</td> <td> <i>SO|comp8549_c0</i> </td> <td>1.17.7.1</td> <td>425</td> <td>15</td> <td>5.26</td> </tr> <tr> <td> <i>SoHDR1</i> </td> <td>K03527</td> <td> <i>SO|comp6889_c0</i> </td> <td>1.17.1.2</td> <td>460</td> <td>599</td> <td>138.19</td> </tr> <tr> <td> <i>SoHDR2</i> </td> <td>K03527</td> <td> <i>SO|comp26756_c1</i> </td> <td>1.17.1.2</td> <td>1992</td> <td>22309.1</td> <td>755.85</td> </tr> <tr> <td> <i>SoHDR3</i> </td> <td>K03527</td> <td> <i>SO|comp24712_c0</i> </td> <td>1.17.1.2</td> <td>1515</td> <td>1249.27</td> <td>57.42</td> </tr> <tr> <td> <i>SoIDI1</i> </td> <td>K01823</td> <td> <i>SO|comp26922_c0</i> </td> <td>5.3.3.2</td> <td>1512</td> <td>6989.53</td> <td>322</td> </tr> <tr> <td> <i>SoIDI2</i> </td> <td>K01823</td> <td> <i>SO|comp20600_c0</i> </td> <td>5.3.3.2</td> <td>845</td> <td>148</td> <td>13.84</td> </tr> <tr> <td>MVA</td> <td> <i>SoAACT1</i> </td> <td>K00626</td> <td> <i>SO|comp27297_c0</i> </td> <td>2.3.1.9</td> <td>1682</td> <td>4550</td> <td>185.61</td> </tr> <tr> <td> <i>SoAACT2</i> </td> <td>K00626</td> <td> <i>SO|comp1036893_c0</i> </td> <td>2.3.1.9</td> <td>226</td> <td>2</td> <td>3.25</td> </tr> <tr> <td> <i>SoAACT3</i> </td> <td>K00626</td> <td> <i>SO|comp1105580_c0</i> </td> <td>2.3.1.9</td> <td>206</td> <td>0</td> <td>0</td> </tr> <tr> <td> <i>SoAACT4</i> </td> <td>K00626</td> <td> <i>SO|comp15111_c0</i> </td> <td>2.3.1.9</td> <td>1820</td> <td>510.61</td> <td>19.06</td> </tr> <tr> <td> <i>SoHMGS</i> </td> <td>K01641</td> <td> <i>SO|comp10117_c0</i> </td> <td>2.3.3.10</td> <td>1878</td> <td>6188.8</td> <td>223.01</td> </tr> <tr> <td> <i>SoHMGR1</i> </td> <td>K00021</td> <td> <i>SO|comp63356_c0</i> </td> <td>1.1.1.34</td> <td>1323</td> <td>69</td> <td>3.71</td> </tr> <tr> <td> <i>SoHMGR2</i> </td> <td>K00021</td> <td> <i>SO|comp26128_c0</i> </td> <td>1.1.1.34</td> <td>2000</td> <td>596</td> <td>25.17</td> </tr> <tr> <td> <i>SoHMGR3</i> </td> <td>K00021</td> <td> <i>SO|comp17290_c0</i> </td> <td>1.1.1.34</td> <td>2973</td> <td>2018.51</td> <td>44.04</td> </tr> <tr> <td> <i>SoHMGR4</i> </td> <td>K00021</td> <td> <i>SO|comp27097_c0</i> </td> <td>1.1.1.34</td> <td>2068</td> <td>6819.56</td> <td>220.78</td> </tr> <tr> <td> <i>SoHMGR5</i> </td> <td>K00021</td> <td> <i>SO|comp574496_c0</i> </td> <td>1.1.1.34</td> <td>244</td> <td>2</td> <td>2.34</td> </tr> <tr> <td> <i>SoHMGR6</i> </td> <td>K00021</td> <td> <i>SO|comp12556_c0</i> </td> <td>1.1.1.34</td> <td>262</td> <td>2</td> <td>1.79</td> </tr> <tr> <td> <i>SoHMGR7</i> </td> <td>K00021</td> <td> <i>SO|comp8357_c0</i> </td> <td>1.1.1.34</td> <td>286</td> <td>3</td> <td>2.02</td> </tr> <tr> <td> <i>SoMVK</i> </td> <td>K00869</td> <td> <i>SO|comp26218_c0</i> </td> <td>2.7.1.36</td> <td>1670</td> <td>671.21</td> <td>42.87</td> </tr> <tr> <td> <i>SoPMK</i> </td> <td>K00938</td> <td> <i>SO|comp23601_c0</i> </td> <td>2.7.4.2</td> <td>2023</td> <td>656.67</td> <td>23.66</td> </tr> <tr> <td>Monoterpene</td> <td> <i>SoGPS</i> </td> <td>K14066</td> <td> <i>SO|comp20551_c0</i> </td> <td>2.5.1.1</td> <td>1329</td> <td>511.75</td> <td>20.23</td> </tr> <tr> <td>Sesqui and Triterpene</td> <td> <i>SoFPPS1</i> </td> <td>K00787</td> <td> <i>SO|comp15527_c0</i> </td> <td>2.5.1.10</td> <td>1426</td> <td>101</td> <td>7.1</td> </tr> <tr> <td> <i>SoFPPS2</i> </td> <td>K00787</td> <td> <i>SO|comp10352_c0</i> </td> <td>2.5.1.10</td> <td>1540</td> <td>6232.1</td> <td>281.11</td> </tr> <tr> <td>Diterpene</td> <td> <i>SoGGPSΙΙ1</i> </td> <td>K13789</td> <td> <i>SO|comp10800_c1</i> </td> <td>2.5.1.29</td> <td>426</td> <td>47</td> <td>12.46</td> </tr> <tr> <td> <i>SoGGPSΙΙ2</i> </td> <td>K13789</td> <td> <i>SO|comp25415_c0</i> </td> <td>2.5.1.29</td> <td>1441</td> <td>548.03</td> <td>26.69</td> </tr> <tr> <td> <i>SoGGPSΙΙ3</i> </td> <td>K13789</td> <td> <i>SO|comp8246_c0</i> </td> <td>2.5.1.29</td> <td>1466</td> <td>1556</td> <td>74.28</td> </tr> <tr> <td> <i>SoGGPSΙΙ4</i> </td> <td>K13789</td> <td> <i>SO|comp116899_c0</i> </td> <td>2.5.1.29</td> <td>251</td> <td>3</td> <td>3.14</td> </tr> <tr> <td> <i>SoGGPSΙΙ5</i> </td> <td>K13789</td> <td> <i>SO|comp10800_c0</i> </td> <td>2.5.1.29</td> <td>827</td> <td>149</td> <td>14.33</td> </tr> <tr> <td> <i>SoGGPSΙΙ6</i> </td> <td>K13789</td> <td> <i>SO|comp60755_c0</i> </td> <td>2.5.1.29</td> <td>1035</td> <td>77</td> <td>5.57</td> </tr> <tr> <td> <i>SoGGPSΙΙ7</i> </td> <td>K13789</td> <td> <i>SO|comp107254_c0</i> </td> <td>2.5.1.29</td> <td>793</td> <td>25</td> <td>2.54</td> </tr> <tr> <td> <i>SoGGPSΙΙ8</i> </td> <td>K13789</td> <td> <i>SO|comp394168_c0</i> </td> <td>2.5.1.29</td> <td>243</td> <td>3</td> <td>3.56</td> </tr> <tr> <td> <i>SoGGPSΙΙ9</i> </td> <td>K13789</td> <td> <i>SO|comp24966_c0</i> </td> <td>2.5.1.29</td> <td>1367</td> <td>392</td> <td>20.3</td> </tr> <tr> <td> <i>SoGGPSΙΙ10</i> </td> <td>K13789</td> <td> <i>SO|comp28724_c0</i> </td> <td>2.5.1.29</td> <td>1515</td> <td>1071</td> <td>49.23</td> </tr> <tr> <td>Other Terpenoid Backbone Biosynthesis</td> <td> <i>SoFLDH</i> </td> <td>K15891</td> <td> <i>SO|comp24181_c1</i> </td> <td>1.1.1.216</td> <td>1829</td> <td>436</td> <td>16.18</td> </tr> <tr> <td> <i>SoFOLK1</i> </td> <td>K15892</td> <td> <i>SO|comp22857_c1</i> </td> <td>2.7.1.</td> <td>789</td> <td>89</td> <td>9.11</td> </tr> <tr> <td> <i>SoFOLK2</i> </td> <td>K15892</td> <td> <i>SO|comp31365_c0</i> </td> <td>2.7.1.</td> <td>1300</td> <td>605.78</td> <td>33.27</td> </tr> <tr> <td> <i>SoFOLK3</i> </td> <td>K15892</td> <td> <i>SO|comp825225_c0</i> </td> <td>2.7.1.</td> <td>205</td> <td>2</td> <td>5.34</td> </tr> <tr> <td> <i>SoFOLK4</i> </td> <td>K15892</td> <td> <i>SO|comp22857_c0</i> </td> <td>2.7.1.</td> <td>273</td> <td>27</td> <td>21.03</td> </tr> <tr> <td> <i>SoPCYOX1</i> </td> <td>K05906</td> <td> <i>SO|comp17568_c0</i> </td> <td>1.8.3.5, 1.8.3.6</td> <td>2275</td> <td>717</td> <td>21.78</td> </tr> <tr> <td> <i>SoSTE24-1</i> </td> <td>K06013</td> <td> <i>SO|comp520699_c0</i> </td> <td>3.4.24.84</td> <td>326</td> <td>4</td> <td>1.88</td> </tr> <tr> <td> <i>SoSTE24-2</i> </td> <td>K06013</td> <td> <i>SO|comp8969_c0</i> </td> <td>3.4.24.84</td> <td>1628</td> <td>659.79</td> <td>27.93</td> </tr> <tr> <td> <i>SoCHLP1</i> </td> <td>K10960</td> <td> <i>SO|comp12058_c0</i> </td> <td>1.3.1.83</td> <td>1475</td> <td>92</td> <td>4.49</td> </tr> <tr> <td> <i>SoCHLP2</i> </td> <td>K10960</td> <td> <i>SO|comp18613_c0</i> </td> <td>1.3.1.83</td> <td>2153</td> <td>20001.75</td> <td>619.4</td> </tr> <tr> <td> <i>SoFACE2</i> </td> <td>K08658</td> <td> <i>SO|comp23619_c0</i> </td> <td>3.4.22.-</td> <td>1334</td> <td>215</td> <td>13.49</td> </tr> <tr> <td> <i>SoPCME1</i> </td> <td>K15889</td> <td> <i>SO|comp7802_c0</i> </td> <td>3.1.1.-</td> <td>1819</td> <td>1899.94</td> <td>71.36</td> </tr> <tr> <td> <i>SoPCME2</i> </td> <td>K15889</td> <td> <i>SO|comp1774_c0</i> </td> <td>3.1.1.-</td> <td>332</td> <td>3</td> <td>1.35</td> </tr> <tr> <td> <i>SoPCME3</i> </td> <td>K15889</td> <td> <i>SO|comp321627_c0</i> </td> <td>3.1.1.-</td> <td>531</td> <td>8</td> <td>1.45</td> </tr> <tr> <td> <i>SoFNTB</i> </td> <td>K05954</td> <td> <i>SO|comp25369_c0</i> </td> <td>2.5.1.58</td> <td>1536</td> <td>283.6</td> <td>12.95</td> </tr> <tr> <td> <i>SoFNTA</i> </td> <td>K05955</td> <td> <i>SO|comp21306_c0</i> </td> <td>2.5.1.58</td> <td>1260</td> <td>297</td> <td>18.19</td> </tr> <tr> <td> <i>SoSPS</i> </td> <td>K05356</td> <td> <i>SO|comp10273_c0</i> </td> <td>2.5.1.84, 2.5.1.85</td> <td>1915</td> <td>6102.29</td> <td>215.16</td> </tr> <tr> <td> <i>SoDHDDS1</i> </td> <td>K11778</td> <td> <i>SO |comp24971_c0</i> </td> <td>2.5.1.87</td> <td>1475</td> <td>6128.72</td> <td>362.2</td> </tr> <tr> <td> <i>SoDHDDS2</i> </td> <td>K11778</td> <td> <i>SO |comp24827_c0</i> </td> <td>2.5.1.87</td> <td>1631</td> <td>135.49</td> <td>6.75</td> </tr> <tr> <td> <i>SoDHDDS3</i> </td> <td>K11778</td> <td> <i>SO |comp193146_c0</i> </td> <td>2.5.1.87</td> <td>368</td> <td>5</td> <td>1.77</td> </tr> <tr> <td> <i>SoDHDDS4</i> </td> <td>K11778</td> <td> <i>SO |comp26151_c1</i> </td> <td>2.5.1.87</td> <td>1619</td> <td>1109.77</td> <td>74.03</td> </tr> <tr> <td> <i>SoDHDDS5</i> </td> <td>K11778</td> <td> <i>SO |comp16046_c0</i> </td> <td>2.5.1.87</td> <td>1617</td> <td>3689.56</td> <td>179.12</td> </tr> <tr> <td> <i>SoICMT1</i> </td> <td>K00587</td> <td> <i>SO|comp11697_c0</i> </td> <td>2.1.1.100</td> <td>677</td> <td>27</td> <td>3.41</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>Abbreviations: FPKM: Fragments per Kilobase of transcripts per Million mapped fragments, <i>SoDXS</i>: 1-deoxy-D-xylulose-5-phosphate synthase, <i>SoDXR</i>:1-deoxy-D-xylulose-5-phosphate reductoi <i>So</i>merase, <i>SoMCT</i>: 2-C-methyl-D-erythritol 4-phosphate cytidylyltransferase, <i>SoISPF</i>: 2-C-methyl-D-erythritol 2,4-cyclodiphosphate synthase, <i>SoHDS</i>:(E)-4-hydroxy-3-methylbut-2-enyl-diphosphate synthase, <i>SoHDR</i>: 4-hydroxy-3-methylbut-2-enyl diphosphate reductases, <i>SoIDI</i>: i <i>So</i>pentenyl-diphosphate delta i <i>So</i>merase, <i>SoAACT</i>: acetyl-CoA C-acetyl transferase, <i>SoHMGS</i>: hydroxyl methyl glutaryl-CoA synthase, <i>SoHMGR</i>: hydroxyl methyl glutaryl-CoA reductase (NADPH), <i>SoMVK</i>: mevalonate kinase, <i>SoPMK</i>: phospho-mevalonate kinase, <i>SoGPS</i>: geranyl pyrophosphate synthase, <i>SoFPPS</i>: farnesyl pyrophosphate synthase, <i>SoGGPS</i>: geranylgeranyl pyrophosphate synthase, type II, <i>SoFLDH</i>: farne <i>So</i>l dehydrogenase, <i>SoFOLK</i>: farne <i>So</i>l kinase, <i>SoPCYOX1</i>: prenylcysteine oxidases/farnesyl cysteine lyase, <i>SoSTE24</i>: STE24 endopeptidases, <i>SoCHLP</i>: geranylgeranyl reductase, <i>SoFACE</i>: prenyl-protein peptidases, <i>SoPCME</i>: prenylcysteine alpha-carboxyl methylesterase, <i>SoFNTB</i>: protein farnesyltransferase subunit beta, <i>SoFNTA</i>: protein farnesyltransferase/geranylgeranyltransferase type-1 subunit alpha, <i>SoSPS</i>: all-trans-nonaprenyl-diphosphate synthase, <i>SoDHDDS</i>: Di trans, polycis-polyprenyl diphosphate synthase, <i>So ICMT</i>: protein-S-i <i>So</i>prenylcysteine O-methyltransferase. </p> </div>  <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <p>Representative terpenoid biosynthesis pathway with cognate heat maps for transcript levels of genes from transcriptome data with substrates and products, colored arrows connect substrates to their corresponding products. Green/red color-coded heat maps represent relative transcript levels of different terpenoid genes determined by Illumina HiSeq 2000 sequencing; red, upregulated; green, downregulated. Transcript levels data represent by FPKM: Fragments per Kilobase of transcripts per Million mapped fragments. MeV: MultiExperiment Viewer software was used to depict transcript levels. DXS: 1-deoxy-D-xylulose-5-phosphate synthase, DXR:1-deoxy-D-xylulose-5-phosphate reductoisomerase, MCT: 2-C-methyl-D-erythritol 4-phosphate cytidylyltransferase, ISPF: 2-C-methyl-D-erythritol 2,4-cyclodiphos-phate synthase, HDS:(E)-4-hydroxy-3-methylbut-2-enyl-diphosphate synthase, HDR: 4-hydroxy-3-methylbut-2-enyl diphosphate reductases, IDI: isopentenyl-diphosphate delta isomerase, AACT: acetyl-CoA C-acetyl transferase, HMGS: hydroxyl methyl glutaryl-CoA synthase, HMGR: hydroxymethyl glutaryl-CoA reductase (NADPH), MVK: mevalonate kinase, PMK: phospho-mevalonate kinase, GPPS: geranyl pyrophosphate synthase, FPPS: farnesyl pyrophosphate synthase, GGPS: geranylgeranyl pyrophosphate synthase, type II, CINO:1,8-cineole synthase, MYS: myrcene/ocimene synthase, LINA: (3S)-linalool synthase, NEOM:(+)-neomenthol dehydrogenase, SABI:(+)-sabinene synthase, TPS6:(−)-germacrene D synthase, AMS:beta-amyrin synthase, SEQ: Squalene monooxygenase, HUMS:α-humulene/β-caryophyllene synthase, GA2:gibberellin 2- -oxidase, GA20:gibberellin 20-oxidase, E-KS:ent-kaurene synthase, MAS:momilactone-A synthase, GA3:gibberellin 3-beta-dioxygenase, E-KIA: ent-isokaurene C2-hydroxylase, E-KIH:ent-kaurenoic acid hydroxylase, E-CDS: ent-copalyl diphosphate synthase. </p>   </div> <p /> </div> <div class="genesrelatedtoterpenesynthases" title="sec"> <div class="title" tagx="title" title="title">
Genes related to terpene synthases</div> <p>Plants produce various terpenoid compounds with highly diverse structures. These compounds play an important role and function in the interactions with environmental factors and in fundamental biological processes <sup> <a href="#CR32">32</a>, <a href="#CR33">33</a> </sup>. Multiple terpenoids are synthesized in plants by the expression of many TPS genes. Moreover, some TPSs have the ability to catalyse the production of multiple products. Thus, the <i>TPS</i> gene family was classified according to phylogenetic relationships into eight subfamilies (TPS a, b, c, d, e/f, g, and h), which comprise mono-, sesqui-, di- and triterpene synthases <sup> <a href="#CR34">34</a> </sup>. Therefore, the annotation of transcriptome data from <i>S</i>. <i>officinalis</i> against the Lamiaceae family and Arabidopsis revealed many terpene synthases involved in the terpenoid biosynthesis pathway, e.g., myrcene, (+)-neomenthol, 1,8-cineole, (3S)-linalool, α-humulene/β-caryophyllene, momilactone-A, gibberellin 3, gibberellin 2, ent-copalyl diphosphate, ent-kaurene, ent-kaurenoic acid, ent-isokaurene C2, gibberellin 20, beta-amyrin and squalene. From the dataset, 65 TPS unigenes were identified and determined based on sequence similarities with a TPS sequence in the canonical annotation reference database. Twenty unigenes were annotated as being involved in monoterpene biosynthesis, including myrcene/ocimene synthase, (+)-neomenthol dehydrogenase, 1,8-cineole synthase, (+)-sabinene synthase and (3S)-linalool synthase, and three other unigenes were annotated as being involved in sesquiterpene biosynthesis, including α-humulene/β-caryophyllene synthase and (−)-germacrene D synthase. Additionally, 29 unigenes were annotated as being involved in diterpene biosynthesis, including momilactone-A synthase, gibberellin 3-beta-dioxygenase, gibberellin 2-oxidase, ent-copalyl diphosphate synthase, ent-kaurene synthase, ent-kaurenoic acid hydroxylase, ent-isokaurene C2-hydroxylase and gibberellin 20-oxidase. Finally, 12 unigenes were annotated as being involved in triterpene biosynthesis, including beta-amyrin synthase, squalene monooxygenase, and farnesyl-diphosphate, but some of these 12 genes showed high abundance in leaves and higher FPKM values (Fig. <a href="#Fig4">4</a> and Table <a href="#Tab3">3</a>). The previous compounds have significant pharmacological activities, such as anticancer, anti-HIV, antiviral, anti-inflammatory and antibacterial activities. Sesquiterpenoids are similar to triterpenoids as both share the same origin and originate from farnesyl diphosphate (FDP). Triterpenoid compounds originate from the conversion of FDP into squalene by squalene synthase (SQS) and then to (S)-2,3-epoxysqualene by squalene monooxygenase (SQE)]. Subsequently, (S)-2,3-epoxysqualene is converted to beta-amyrin and camelliol C in the presence of multifunctional (S)-2,3-epoxysqualene cyclase via beta-amyrin synthase and camelliol C synthase, respectively. Similar reports about triterpenoid biosynthesis from (S)-2,3-epoxysqualene cyclases are available for <i>O</i>. <i>basilicum</i> and <i>Catharanthus roseus</i> <sup> <a href="#CR35">35</a>, <a href="#CR36">36</a> </sup>. <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 3</span>  </div></p><p>Transcript abundance of TPS genes as per the <i>S</i>. <i>officinalis</i> transcriptome. </p>  <table> <thead> <tr> <th>Terpene synthase</th> <th>Kegg Entry</th> <th>Unigene ID</th> <th>Annotation</th> <th>Length (bp)</th> <th>EC. No.</th> <th>Read in leaf</th> <th>FPKM</th> </tr> </thead> <tbody> <tr> <td>Monoterpene</td> <td>K12467</td> <td> <i>So|comp422551_c0</i> </td> <td>myrcene/ocimene synthase</td> <td>203</td> <td>4.2.3.15</td> <td>1</td> <td>2.83</td> </tr> <tr> <td>K12467</td> <td> <i>So |comp431748_c0</i> </td> <td>myrcene/ocimene synthase</td> <td>214</td> <td>4.2.3.15</td> <td>2</td> <td>4.24</td> </tr> <tr> <td>K12467</td> <td> <i>So |comp11163_c0</i> </td> <td>myrcene/ocimene synthase</td> <td>387</td> <td>4.2.3.15</td> <td>53</td> <td>16.93</td> </tr> <tr> <td>K12467</td> <td> <i>So |comp191_c0</i> </td> <td>myrcene/ocimene synthase</td> <td>280</td> <td>4.2.3.15</td> <td>3</td> <td>2.15</td> </tr> <tr> <td>K12467</td> <td> <i>So |comp189977_c0</i> </td> <td>myrcene/ocimene synthase</td> <td>458</td> <td>4.2.3.15</td> <td>12</td> <td>2.79</td> </tr> <tr> <td>K15095</td> <td> <i>So |comp8961_c0</i> </td> <td>(+)-neomenthol dehydrogenase</td> <td>256</td> <td>1.1.1.208</td> <td>2</td> <td>1.95</td> </tr> <tr> <td>K15095</td> <td> <i>So |comp210403_c0</i> </td> <td>(+)-neomenthol dehydrogenase</td> <td>433</td> <td>1.1.1.208</td> <td>5</td> <td>1.29</td> </tr> <tr> <td>K15095</td> <td> <i>So |comp972_c0</i> </td> <td>(+)-neomenthol dehydrogenase</td> <td>327</td> <td>1.1.1.208</td> <td>5</td> <td>2.33</td> </tr> <tr> <td>K15095</td> <td> <i>So |comp26078_c1</i> </td> <td>(+)-neomenthol dehydrogenase</td> <td>566</td> <td>1.1.1.208</td> <td>55</td> <td>9.04</td> </tr> <tr> <td>K15095</td> <td> <i>So |comp12329_c0</i> </td> <td>(+)-neomentholdehydrogenase</td> <td>447</td> <td>1.1.1.208</td> <td>14</td> <td>3.4</td> </tr> <tr> <td>K15095</td> <td> <i>So |comp10962_c0</i> </td> <td>(+)-neomenthol dehydrogenase</td> <td>1072</td> <td>1.1.1.208</td> <td>2170.28</td> <td>136.67</td> </tr> <tr> <td>K07385</td> <td> <i>So |comp5570_c0</i> </td> <td>1,8-cineole synthase</td> <td>354</td> <td>4.2.3.108</td> <td>20</td> <td>7.73</td> </tr> <tr> <td>K07385</td> <td> <i>So |comp184887_c0</i> </td> <td>1,8-cineole synthase</td> <td>402</td> <td>4.2.3.108</td> <td>8</td> <td>2.37</td> </tr> <tr> <td>K07385</td> <td> <i>So |comp144107_c0</i> </td> <td>1,8-cineole synthase</td> <td>295</td> <td>4.2.3.108</td> <td>4</td> <td>2.46</td> </tr> <tr> <td>K07385</td> <td> <i>So |comp53392_c0</i> </td> <td>1,8-cineole synthase</td> <td>453</td> <td>4.2.3.108</td> <td>24</td> <td>5.69</td> </tr> <tr> <td>K07385</td> <td> <i>So |comp26990_c0</i> </td> <td>1,8-cineole synthase</td> <td>2092</td> <td>4.2.3.108</td> <td>10511.52</td> <td>335.99</td> </tr> <tr> <td>K07385</td> <td> <i>So |comp14705_c0</i> </td> <td>1,8-cineole synthase</td> <td>1978</td> <td>4.2.3.108</td> <td>1339.74</td> <td>49.23</td> </tr> <tr> <td>K07385</td> <td> <i>So |comp18462_c0</i> </td> <td>(+)-sabinene synthase</td> <td>1182</td> <td>4.2.3.108</td> <td>93</td> <td>7.5</td> </tr> <tr> <td>K15086</td> <td> <i>So |comp15872_c0</i> </td> <td>(3S)-linalool synthase-1</td> <td>1763</td> <td>4.2.3.25</td> <td>95</td> <td>8.28</td> </tr> <tr> <td>K15086</td> <td> <i>So |comp6814_c0</i> </td> <td>(3S)-linalool synthase-2</td> <td>1789</td> <td>4.2.3.25</td> <td>116</td> <td>4.41</td> </tr> <tr> <td>Sesquiterpene</td> <td>K14184</td> <td> <i>So |comp101158_c0</i> </td> <td>α-humulene/β-caryophyllene synthase</td> <td>284</td> <td>4.2.3.57</td> <td>10</td> <td>6.87</td> </tr> <tr> <td>K15803</td> <td> <i>So |comp26367_c0</i> </td> <td>(−)-germacrene D synthase</td> <td>1965</td> <td>4.2.3.75</td> <td>777.46</td> <td>39.5</td> </tr> <tr> <td>Diterpene</td> <td>K13070</td> <td> <i>So |comp21612_c0</i> </td> <td>momilactone-A synthase</td> <td>1348</td> <td>1.1.1.295</td> <td>309</td> <td>18.14</td> </tr> <tr> <td>K04124</td> <td> <i>So |comp14297_c0</i> </td> <td>gibberellin 3-beta-dioxygenase</td> <td>1149</td> <td>1.14.11.15</td> <td>97</td> <td>6.17</td> </tr> <tr> <td>K04125</td> <td> <i>So |comp19885_c0</i> </td> <td>gibberellin 2- oxidase</td> <td>1257</td> <td>1.14.11.13</td> <td>319</td> <td>18.23</td> </tr> <tr> <td>K04125</td> <td> <i>So |comp436332_c0</i> </td> <td>gibberellin 2- oxidase</td> <td>217</td> <td>1.14.11.13</td> <td>1</td> <td>1.97</td> </tr> <tr> <td>K04125</td> <td> <i>So |comp324252_c0</i> </td> <td>gibberellin 2- oxidase</td> <td>226</td> <td>1.14.11.13</td> <td>2</td> <td>3.25</td> </tr> <tr> <td>K04125</td> <td> <i>So |comp747555_c0</i> </td> <td>gibberellin 2- oxidase</td> <td>212</td> <td>1.14.11.13</td> <td>1</td> <td>2.22</td> </tr> <tr> <td>K04125</td> <td> <i>So |comp5948_c0</i> </td> <td>gibberellin 2- oxidase</td> <td>274</td> <td>1.14.11.13</td> <td>3</td> <td>2.31</td> </tr> <tr> <td>K04120</td> <td> <i>So |comp2895_c0</i> </td> <td>ent-copalyl diphosphate synthase</td> <td>392</td> <td>5.5.1.13</td> <td>7</td> <td>2.18</td> </tr> <tr> <td>K04120</td> <td> <i>So |comp274743_c0</i> </td> <td>ent-copalyl diphosphate synthase</td> <td>356</td> <td>5.5.1.13</td> <td>6</td> <td>2.29</td> </tr> <tr> <td>K04120</td> <td> <i>So |comp246076_c0</i> </td> <td>ent-copalyl diphosphate synthase</td> <td>488</td> <td>5.5.1.13</td> <td>7</td> <td>1.46</td> </tr> <tr> <td>K04120</td> <td> <i>So |comp132266_c0</i> </td> <td>ent-copalyl diphosphate synthase</td> <td>656</td> <td>5.5.1.13</td> <td>18</td> <td>2.38</td> </tr> <tr> <td>K04120</td> <td> <i>So |comp15163_c0</i> </td> <td>ent-copalyl diphosphate synthase</td> <td>486</td> <td>5.5.1.13</td> <td>21</td> <td>4.45</td> </tr> <tr> <td>K04120</td> <td> <i>So |comp350819_c0</i> </td> <td>ent-copalyl diphosphate synthase</td> <td>209</td> <td>5.5.1.13</td> <td>2</td> <td>4.8</td> </tr> <tr> <td>K04120</td> <td> <i>So |comp112536_c0</i> </td> <td>ent-copalyl diphosphate synthase</td> <td>491</td> <td>5.5.1.13</td> <td>11</td> <td>2.27</td> </tr> <tr> <td>K04120</td> <td> <i>So |comp689_c0</i> </td> <td>ent-copalyl diphosphate synthase</td> <td>544</td> <td>5.5.1.13</td> <td>13</td> <td>2.27</td> </tr> <tr> <td>K04120</td> <td> <i>So |comp6575_c0</i> </td> <td>ent-copalyl diphosphate synthase</td> <td>2713</td> <td>5.5.1.13</td> <td>1174</td> <td>28.26</td> </tr> <tr> <td>K04120</td> <td> <i>So |comp2827_c0</i> </td> <td>ent-copalyl diphosphate synthase</td> <td>246</td> <td>5.5.1.13</td> <td>3</td> <td>3.39</td> </tr> <tr> <td>K04120</td> <td> <i>So |comp23218_c0</i> </td> <td>ent-copalyl diphosphate synthase</td> <td>3679</td> <td>5.5.1.13</td> <td>1267.32</td> <td>25.28</td> </tr> <tr> <td>K04121</td> <td> <i>So |comp20544_c0</i> </td> <td>ent-kaurene synthase</td> <td>1775</td> <td>4.2.3.19</td> <td>992</td> <td>38.08</td> </tr> <tr> <td>K04121</td> <td> <i>So |comp21378_c0</i> </td> <td>ent-kaurene synthase</td> <td>3023</td> <td>4.2.3.19</td> <td>760.01</td> <td>24.46</td> </tr> <tr> <td>K04121</td> <td> <i>So |comp22699_c0</i> </td> <td>ent-kaurene synthase</td> <td>2049</td> <td>4.2.3.19</td> <td>968</td> <td>41.49</td> </tr> <tr> <td>K04121</td> <td> <i>So |comp186871_c0</i> </td> <td>ent-kaurene synthase</td> <td>674</td> <td>4.2.3.19</td> <td>14</td> <td>1.78</td> </tr> <tr> <td>K04121</td> <td> <i>So |comp4251_c0</i> </td> <td>ent-kaurene synthase</td> <td>510</td> <td>4.2.3.19</td> <td>20</td> <td>3.88</td> </tr> <tr> <td>K04123</td> <td> <i>So |comp15654_c0</i> </td> <td>ent-kaurenoic acid hydroxylase</td> <td>1954</td> <td>1.14.13.79</td> <td>212</td> <td>7.43</td> </tr> <tr> <td>K04123</td> <td> <i>So |comp338203_c0</i> </td> <td>ent-kaurenoic acid hydroxylase</td> <td>257</td> <td>1.14.13.79</td> <td>2</td> <td>1.92</td> </tr> <tr> <td>K04123</td> <td> <i>So |comp404902_c0</i> </td> <td>ent-kaurenoic acid hydroxylase</td> <td>324</td> <td>1.14.13.79</td> <td>4</td> <td>1.91</td> </tr> <tr> <td>K04123</td> <td> <i>So |comp353538_c0</i> </td> <td>ent-kaurenoic acid hydroxylase</td> <td>288</td> <td>1.14.13.79</td> <td>2</td> <td>1.32</td> </tr> <tr> <td>K16083</td> <td> <i>So |comp1777_c0</i> </td> <td>ent-isokaurene C2-hydroxylase</td> <td>231</td> <td>4.2.3.103</td> <td>2</td> <td>2.94</td> </tr> <tr> <td>K05282</td> <td> <i>So |comp23119_c0</i> </td> <td>gibberellin 20-oxidase</td> <td>1354</td> <td>1.14.11.12</td> <td>997.41</td> <td>52.75</td> </tr> <tr> <td>K05282</td> <td> <i>So |comp210082_c0</i> </td> <td>gibberellin 20-oxidase</td> <td>466</td> <td>1.14.11.12</td> <td>9</td> <td>2.03</td> </tr> <tr> <td>Triterpene</td> <td>K15813</td> <td> <i>So |comp27006_c0</i> </td> <td>beta-amyrin synthase</td> <td>3060</td> <td>5.4.99.39</td> <td>14004.17</td> <td>296.25</td> </tr> <tr> <td>K15822</td> <td> <i>So |comp17362_c0</i> </td> <td>beta-amyrin synthase</td> <td>2661</td> <td>5.4.99.39</td> <td>1712.24</td> <td>42.09</td> </tr> <tr> <td>K15813</td> <td> <i>So |comp16071_c0</i> </td> <td>beta-amyrin synthase</td> <td>282</td> <td>5.4.99.39</td> <td>13</td> <td>13.23</td> </tr> <tr> <td>K00511</td> <td> <i>So |comp26984_c0</i> </td> <td>Squalene monooxygenase</td> <td>2092</td> <td>1.14.13.132</td> <td>8377.74</td> <td>267.79</td> </tr> <tr> <td>K00511</td> <td> <i>So |comp24504_c0</i> </td> <td>Squalene monooxygenase</td> <td>2141</td> <td>1.14.13.132</td> <td>600</td> <td>18.69</td> </tr> <tr> <td>K00511</td> <td> <i>So |comp1139_c0</i> </td> <td>Squalene monooxygenase</td> <td>326</td> <td>1.14.13.132</td> <td>3</td> <td>1.41</td> </tr> <tr> <td>K00511</td> <td> <i>So |comp98442_c0</i> </td> <td>Squalene monooxygenase</td> <td>263</td> <td>1.14.13.132</td> <td>3</td> <td>2.65</td> </tr> <tr> <td>K00511</td> <td> <i>So |comp14919_c0</i> </td> <td>Squalene monooxygenase</td> <td>492</td> <td>1.14.13.132</td> <td>14</td> <td>3.81</td> </tr> <tr> <td>K00511</td> <td> <i>So |comp11693_c0</i> </td> <td>Squalene monooxygenase</td> <td>211</td> <td>1.14.13.132</td> <td>1</td> <td>2.28</td> </tr> <tr> <td>K00511</td> <td> <i>So |comp12366_c0</i> </td> <td>Squalene monooxygenase</td> <td>594</td> <td>1.14.13.132</td> <td>12</td> <td>1.83</td> </tr> <tr> <td>K00801</td> <td> <i>So |comp26757_c1</i> </td> <td>farnesyl-diphosphate farnesyltransferase</td> <td>1934</td> <td>2.5.1.21</td> <td>3936.99</td> <td>139.91</td> </tr> <tr> <td>K00801</td> <td> <i>So |comp29648_c0</i> </td> <td>farnesyl-diphosphate farnesyltransferase</td> <td>443</td> <td>2.5.1.21</td> <td>228</td> <td>56.24</td> </tr> </tbody> </table>  <p /> </div> <div class="ssrdiscoveryandanalysis" title="sec"> <div class="title" tagx="title" title="title">
SSR discovery and analysis</div> <p>The Illumina HiSeq 2000 system offers the opportunity to analyse molecular markers such as simple sequence repeats (SSRs) that are related to terpenoid pathway genes. SSR molecular markers have proven to be a powerful method for understanding genetic variation. Moreover, polymorphic SSR markers are very important for the investigation of related comparative genomics, genetic diversity, evolution, linkage mapping, gene-based association studies, and relatedness. Even though SNP markers have become promising, especially for studying complex genetic traits and high-throughput mapping, SSRs provide many advantages compared with other marker systems. Hence, SSRs have become the preferable codominant molecular marker for the construction of linkage maps <sup> <a href="#CR37">37</a> </sup>. Therefore, the development of novel SSR molecular markers for <i>S</i>. <i>officinalis</i> plants could be a valuable tool for breeding studies and genetic applications. Therefore, SSR markers were identified from transcriptome sequencing data using MISA (MIcroSAtellite) ( <a href="http://pgrc.ipkgatersle-ben.de/misa/misa.html">http://pgrc.ipkgatersle-ben.de/misa/misa.html</a>). Of the 48,671 transcripts of <i>S</i>. <i>officinalis</i>, 7,439 transcripts were observed to have SSRs (Supplementary Table <a href="#MOESM1">S5</a>). The total number of SSR-containing sequences in <i>S</i>. <i>officinalis</i> was 9,149 following stringent selection criteria used to identify these SSRs. The analysis data showed that dinucleotide repeats were the most abundant motif type in <i>S</i>. <i>officinalis</i> (4,295; 44.132%), followed by mononucleotide (2,348; 24.13%), trinucleotide (2,317; 23.81%), tetranucleotide (116; 1.191%), and hexanucleotide (39; 0.4%) types, while the pentanucleotide type was the least abundant motif (34; 0.35%) (Supplementary Tables <a href="#MOESM1">S6</a> and <a href="#MOESM1">S7</a> and Fig. <a href="#MOESM1">S3</a>). Except for the absence of mononucleotides, these results were similar to the previous results obtained from the transcriptomes of <i>O</i>. <i>sanctum</i> and <i>O</i>. <i>basilicum</i> (members of the same family), which have dinucleotide repeats as the most abundant motif type, followed by tri-, tetra-, hexa- and penta nucleotide types as the least abundant motif  <sup> <a href="#CR6">6</a> </sup>. After analysing the data from mono- to hexanucleotide motifs to obtain the number of repeat units, we found that the highest repeat unit of potential SSRs was 6, which accounted for 1,999 SSRs (21.86%), followed by 10 SSRs (1,490; 16.30%), 5 (1,411; 15.43%), and 7 (1,301; 14.23%), and the smallest repeat unit of potential SSRs was ≥24 (7; 0.08) (Supplementary Table <a href="#MOESM1">S7</a>). The AG/CT dinucleotide repeat was the most prevalent motif detected in all SSRs (2,999; 30.81%) followed by A/T as a mononucleotide repeat (2,272; 23.34%). By contrast, the least abundant motif in all SSRs (4; 0.041%) was detected in (AAAAC/GTTTT/AAAAG/CTTTT/AAAAT/ATTTT/AAACC/GGTTT) as pentanucleotide repeat and in (AAAATG/ATTTTC/AAATAG/ATTTCT/AAATTC/AATTTG/AACAAT/ATTGTT) as hexanucleotide repeat. Finally, several SSR motifs were associated with many unique sequences that encode enzymes (e.g. SoDXS4, SoDXS5, SoHDR2, SoHMGS, SoHMGR3, SoFLDH, SoPCYOX1, SoFNTA, SoDHDDS1, SoDHDDS5, momilactone-A synthase, SoGGPSΙΙ7, SoGGPSΙΙ10, ent-copalyl diphosphate synthase, ent-kaurenoic acid hydroxylase, beta-amyrin synthase and squalene monooxygenase) involved in terpenoid biosynthesis (Supplementary Table <a href="#MOESM1">S8</a>). </p> </div> <div class="validationofthegeneexpressionpatternsbyquantitativert-pcr" title="sec"> <div class="title" tagx="title" title="title">
Validation of the gene expression patterns by quantitative RT-PCR</div> <p>To determinate the reliability of the Illumina HiSeq 2000 read analysis, eleven candidate genes with a higher differential expression were selected, and their expression profiles were compared within young leaf, old leaf, stem, flower and bud flower samples. Quantitative real-time (qRT) PCR was used to determine the ‘transcriptional control’, which indicates the number of mRNA copies of the enzyme that complements the end-product quantity. Therefore, the correlation between the <i>TPS</i> mRNAs with their products and the end-products showed a relationship between the chosen differentially expressed genes (DEGs), monoterpene synthase ( <i>SoGPS</i>; comp20551_c0), sesquiterpene synthase ( <i>SoFPS</i>2; comp10352_c0), diterpene synthase ( <i>SoGGPS</i>; comp25415_c0), myrcene/ocimene synthase ( <i>SoMYRS</i>; comp11163_c0) 1,8-cineole synthase ( <i>SoCINS</i>; comp26990_c0), (3S)-linalool synthase-2 ( <i>SoLINS</i>; comp6814_c0), α-humulene/β-caryophyllene synthase ( <i>SoHUMS</i>;comp101158_c0), (−)-germacrene D synthase ( <i>SoTPS6</i>; comp26367_c0), squalene monooxygenase ( <i>SoSQUS</i>; comp26984_c0), (+)-sabinene synthase ( <i>SoSABS</i>; comp18462_c0) and (+)-neomenthol dehydrogenase ( <i>SoNEOD</i>; comp10962_c0) and the terpenoid biosynthesis pathway of <i>S</i>. <i>officinalis</i>. <i>SoACTIN</i> was used as an internal reference gene (Supplementary Table <a href="#MOESM1">S9</a>). The expression patterns of the eleven selected DEGs in the young leaf, old leaf, stem, flower, and bud flower samples were examined (Fig. <a href="#Fig5">5</a>) by qRT-PCR, and the results were consistent with the results from the Illumina HiSeq 2000 read analysis. At the current stage, we may be able to answer the question which terpenoid compounds accumulate mostly in which <i>S</i>. <i>officinalis</i> tissue. From our results, we found that <i>SoGPS</i>, <i>SoFPS</i>, <i>SoMYRS</i>, and <i>SoCINS</i> genes showed the highest expression levels in young leaves, followed by old leaves, stems, flowers and bud flowers. Moreover, (+)-sabinene synthase ( <i>So</i>SABS) genes showed the highest expression levels in young leaves, followed by bud flowers, old leaves, flowers, and stems. (3S)-linalool synthase ( <i>SoLINS</i>) genes showed the highest expression levels in stems, followed by bud flowers, young leaves, old leaves, and flowers. Furthermore, diterpene synthase gene <i>SoGGPS</i> showed the highest expression levels in stems, followed by old leaves, young leaves, flowers and bud flowers. On the other hand, <i>SoTPS6</i> gene showed the highest expression levels in young leaves followed by bud flowers, old leaves, stems, and flowers. Squalene monooxygenase ( <i>SoSQUS</i>) gene showed the highest expression levels in young leaves followed by old leaves, flowers, bud flowers, and stems. Finally, a α-humulene/β-caryophyllene synthase ( <i>SoHUMS</i>) gene showed the highest expression levels in stems, followed by young leaves, old leaves, bud flowers and flowers. These results were compatible with our GC-MS analysis data, indicating that indicated that the main group of terpenes in young leaves, old leaves and stems consisted of mono- and sesquiterpenes. According to the findings of the GC-MS analysis, the major monoterpene compound in young and old leaves was 1,8-cineole (Table <a href="#Tab1">1</a>). Therefore, we suggest that young leaves are the primary site for monoterpene, sesquiterpene and 1,8-cineole synthase biosynthesis and accumulation, followed by old leaves, and then stems. These results are in agreement with those of previous studies <sup> <a href="#CR38">38</a>, <a href="#CR39">39</a> </sup> that reported that the main monoterpenes in <i>S</i>. <i>officinalis</i> and other <i>Salvia</i> plant species are formed and accumulate in very young leaf epidermal glands, as the formation of most epidermal glands and the accumulation of the monoterpenes take a very short time in young leaf tissues. Consequently, in our study we focused on young leaves in which these genes are expressed at higher levels; monoterpenes and sesquiterpenes are also formed at their highest levels in young leaves. In addition, from our study, we found a correlation between the 1,8-cineole accumulation and 1,8-cineole synthase ( <i>SoCINS</i>) expression levels in different tissues. For instance, the most abundant 1,8-cineole accumulation and highest <i>SoCINS</i> expression were in young leaves, followed by old leaves, stems, flowers and bud flowers (Table <a href="#Tab1">1</a> and Fig. <a href="#Fig1">1</a>). Our results are in line with those of previous studies <sup> <a href="#CR40">40</a>– <a href="#CR47">47</a> </sup> that reported that the monoterpene levels are thought to be mainly controlled transcriptionally producing different TPS enzymes. (+)-Neomenthol was not detected by GC-MS analysis as was expected from gene expression analysis, showing the expression of a putative neomenthol dehydrogynase gene that were detected in the Illumina HiSeq 2000 reads and qRT-PCR. This could be due to other unknown reasons <sup> <a href="#CR48">48</a> </sup>. The combination of the analysed data reads from the Illumina HiSeq 2000, qRT- PCR and the GC-MS will pave the way for understanding the complex mechanisms for controlling and regulating the diverse production of terpene compounds. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 5</span> </a> </div>   <p>Quantitative RT-PCR validation of expression of terpene synthase genes selected from the DGE analysis in <i>S</i>. <i>officinalis</i>. Total RNAs were extracted from young leaves, old leaves, stem, flower and bud flower samples and the expression of <i>SoNEOD</i>, <i>SoGPS</i>, <i>SoFPPS</i>, <i>SoGGPS</i>, <i>SoMYRS</i>, <i>SoLINS</i>, <i>SoHUMS</i>, <i>SoTPS6</i>, <i>SoSQUS</i>, <i>SoSABS</i> and <i>SoCINS</i> genes were analysed using quantitative real-time. <i>SoACTIN</i> was used as the internal reference. The values are means ± SE of three biological replicates. </p>   </div> <p /> </div> <div class="functionalcharacterizationoftpsgenesintransgenicn.tabacumleaves" title="sec"> <div class="title" tagx="title" title="title">
Functional characterization of <i>TPS</i> genes in transgenic <i>N</i>. <i>tabacum</i> leaves </div> <p>To test <i>N</i>. <i>tabacum</i> in a transgenic expression system for the production of <i>Salvia</i> terpenes, the following genes were selected from <i>S</i>. <i>officinalis</i>: (+)-neomenthol dehydrogenase ( <i>NEOD</i>), 1,8-cineole synthase ( <i>CINS</i>), (+)-sabinene synthase ( <i>SABS</i>), (3S)-linalool synthase ( <i>LINS</i>), and (−)-germacrene D synthase ( <i>TPS6</i>) encoded by <i>SoNEOD</i>, <i>SoCINS</i>, <i>SoSABS</i>, <i>SoLINS</i>, and <i>SoTPS6</i>, respectively.The stable constitutive expression of the Salvia <i>TPS</i> genes in tobacco was carried out by the infection of <i>N</i>. <i>tabacum</i> leaves using <i>A</i>. <i>tumefaciens</i> strain EHA105 carrying pB2GW7- <i>NEOD</i>, pB2GW7- <i>CINS</i>, pB2GW7- <i>SABS</i>, pB2GW7- <i>LINS</i>, and pB2GW7- <i>TPS6</i> under the control of 35S promoter. Samples of infected were collected 45 days after transgenic tobacco acclimatization (Fig. <a href="#Fig6">6A</a>). We then used semiquantitative RT-PCR to analyse the positive transgenic tobacco and assessed the expression levels of terpene genes from the different samples (Fig. <a href="#Fig6">6B</a> and Supplementary Fig. <a href="#MOESM1">S4</a>). The terpenes were extracted with hexane and analysed by GC-MS. The mono-, sesqui-, di- and triterpene peaks were clearly detected, and the type and amount of compounds represented by the percentage of peak area (% peak area). Compounds were identified by comparing the mass spectra of the compounds with mass spectra libraries. The annotation of the detected components was also confirmed by comparing them with the published references and extracts of tobacco cultivars, which produce different types and amounts of terpenes <sup> <a href="#CR49">49</a>, <a href="#CR50">50</a> </sup>. Overexpression of <i>SoNEOD</i>, <i>SoCINS</i>, <i>SoSABS</i>, <i>SoLINS</i>, and <i>SoTPS6</i> genes in tobacco plants produced different amounts of mono-, sesqui-, di-, and triterpenes and other terpenoids. Moreover, from the results shown in Table <a href="#Tab4">4</a>, Supplementary Fig. <a href="#MOESM1">5</a> and Table <a href="#MOESM1">S10</a>, we found that the transient expression of the different TPS genes from Salvia produced different types and amounts of mono-, sesqui-, di-, and triterpenes and other terpenoid compounds. We also could show a high similarity between the product patterns of <i>TPS</i> genes from Salvia with these from other plant species (Fig. <a href="#Fig7">7</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 6</span> </a> </div>   <p>Overexpression of five <i>S</i>. <i>officinalis TPS</i> genes in transgenic <i>N</i>. <i>tabacum</i>. ( <b>A</b>) Transgenic tobacco plants after adaptation to soil pots. ( <b>B</b>) Semiquantitative RT-PCR analysis of the terpene synthase gene expression. </p>   </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 4</span>  <p>The major terpenoid compositions in transgenic <i>N. tabacum</i> leaves overexpressing <i>SoNEOD, SoCINS,SoSABS, SoLINS, and SoTPS6</i>. </p>  <table> <thead> <tr> <th>N</th> <th>Compound name</th> <th>Retention time (min.)</th> <th>Retention time index</th> <th>Major fragment ions ( <i>m/z</i>) </th> <th>Formula</th> <th>Molecular Mass (g mol <sup>-1</sup>) </th> <th>Terpene Type</th> <th>% Peak area</th> </tr> <tr> <th>W.T</th> <th> <i>SoNEOD</i> </th> <th> <i>SoCINS</i> </th> <th> <i>SoSABS</i> </th> <th> <i>SoLINS</i> </th> <th> <i>SoTPS6</i> </th> </tr> </thead> <tbody> <tr> <td>1</td> <td>L-.alpha.-Terpineol</td> <td>20.607</td> <td>1144</td> <td>139,136,121,107,93,81,79</td> <td>C10H18O</td> <td>154.2493</td> <td>Mono</td> <td>—</td> <td>—</td> <td>0.05</td> <td>—</td> <td>—</td> <td>—</td> </tr> <tr> <td>2</td> <td>L-(-)-Nicotine</td> <td>26.372</td> <td>1360.5</td> <td>162,133,119,92,84,51</td> <td>C10H14N2</td> <td>162.2316</td> <td /> <td>22.49</td> <td>0.08</td> <td>—</td> <td>0.71</td> <td>—</td> <td>0.35</td> </tr> <tr> <td>3</td> <td>trans-.beta.-Ionone</td> <td>30.596</td> <td>1488</td> <td>177,159,149,135,133,121,119</td> <td>C13H20O</td> <td>192.2973</td> <td /> <td>—</td> <td>—</td> <td>0.05</td> <td>0.17</td> <td>—</td> <td>—</td> </tr> <tr> <td>4</td> <td>Topanol;Stavox</td> <td>31.146</td> <td>1514.</td> <td>220,205,177,161,145,115,105</td> <td>C15H24O</td> <td>220.3505</td> <td>Sesqui</td> <td>—</td> <td>0.04</td> <td>—</td> <td>—</td> <td>0.04</td> <td>—</td> </tr> <tr> <td>5</td> <td>cis-Carveol</td> <td>35.519</td> <td>1226</td> <td>137,134,123,119,109,105,95</td> <td>C10H16O</td> <td>152.2334</td> <td>Mono</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> <td>0.1</td> <td>—</td> </tr> <tr> <td>6</td> <td>.alpha.-Campholenal</td> <td>35.809</td> <td>1122</td> <td>137,119,108,81,67</td> <td>C10H16O</td> <td>152.2334</td> <td>Mono</td> <td>—</td> <td>0.08</td> <td>—</td> <td>0.23</td> <td>—</td> <td>—</td> </tr> <tr> <td>7</td> <td>Menthofuran;</td> <td>36.17</td> <td>1163</td> <td>150,108,79,77,39</td> <td>C10H14O</td> <td>150.2176</td> <td>Mono</td> <td>—</td> <td>—</td> <td>0.05</td> <td>—</td> <td>—</td> <td>—</td> </tr> <tr> <td>8</td> <td>α-Bulnesene</td> <td>45.066</td> <td>1506</td> <td>204,189,161,147,135,107,93</td> <td>C15H24</td> <td>204.3511</td> <td>Sesqui</td> <td>—</td> <td>—</td> <td /> <td>0.22</td> <td>—</td> <td>—</td> </tr> <tr> <td>9</td> <td>cis-9-Hexadecenal</td> <td>45.266</td> <td>1759</td> <td>220,149,138,135,124,121,111</td> <td>C16H30O</td> <td>238.4088</td> <td /> <td>—</td> <td>—</td> <td>—</td> <td>0.33</td> <td>—</td> <td>—</td> </tr> <tr> <td>10</td> <td>Triadimefon</td> <td>45.59</td> <td>1997</td> <td>208,181,128,110,85,57,41,29</td> <td>C14H16ClN3O2</td> <td>293.749</td> <td /> <td>—</td> <td>—</td> <td>0.32</td> <td>—</td> <td>—</td> <td>—</td> </tr> <tr> <td>11</td> <td>Cyclooctasiloxane, hexadecamethyl-</td> <td>45.773</td> <td>1688</td> <td>415,401,355,281,221,147,73</td> <td>C16H48O8Si8</td> <td>593.2315</td> <td /> <td>—</td> <td>0.29</td> <td>—</td> <td>—</td> <td>1.57</td> <td>1.04</td> </tr> <tr> <td>12</td> <td>7-Hexadecenal, (Z)-</td> <td>45.855</td> <td>2144</td> <td>220,135,121,111,98,82,67,55</td> <td>C16H30O</td> <td>238.4088</td> <td /> <td>—</td> <td>—</td> <td>1.7</td> <td>—</td> <td>—</td> <td>—</td> </tr> <tr> <td>13</td> <td>β-Elemol</td> <td>46.067</td> <td>1547</td> <td>204,189,161,135,121,107,93</td> <td>C15H26O</td> <td>222.3663</td> <td>Sesqui</td> <td>0.14</td> <td>2.33</td> <td>—</td> <td>1</td> <td>—</td> <td>—</td> </tr> <tr> <td>14</td> <td>(+) Ledol</td> <td>46.099</td> <td>1599</td> <td>204,189,161,122,109,69,43,41</td> <td>C15H26O</td> <td>222.3663</td> <td>Sesqui</td> <td>—</td> <td>0.2</td> <td>0.1</td> <td>—</td> <td>0.33</td> <td>—</td> </tr> <tr> <td>15</td> <td>Ledol</td> <td>46.367</td> <td>1600</td> <td>204,189,161,147,122,109,81,69</td> <td>C15H26O</td> <td>222.3663</td> <td>Sesqui</td> <td>—</td> <td>—</td> <td>0.37</td> <td>1.22</td> <td>—</td> <td>—</td> </tr> <tr> <td>16</td> <td>d-Ledol</td> <td>46.431</td> <td>1602</td> <td>204,189,161,147,133,122,119</td> <td>C15H26O</td> <td>222.3663</td> <td>Sesqui</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> <td>0.51</td> <td>—</td> </tr> <tr> <td>17</td> <td>Retinol, acetate</td> <td>46.635</td> <td>2531</td> <td>268,253,145,119,105,91,81,43</td> <td>C22H32O2</td> <td>328.4883</td> <td /> <td>0.32</td> <td>1.3</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> </tr> <tr> <td>18</td> <td>Phytol</td> <td>46.736</td> <td>2110</td> <td>278,196,179,137,123,95,72</td> <td>C20H40O</td> <td>296.5310</td> <td>Diter</td> <td>—</td> <td>—</td> <td>—</td> <td>0.74</td> <td>0.36</td> <td>5.11</td> </tr> <tr> <td>19</td> <td>cis-Phytol</td> <td>47.331</td> <td>2114</td> <td>278,193,179,151,137,123,109</td> <td>C20H40O</td> <td>296.5310</td> <td>Diter</td> <td>3.19</td> <td>6.28</td> <td>6.75</td> <td>10.84</td> <td>3.92</td> <td>—</td> </tr> <tr> <td>20</td> <td>Carveol</td> <td>48.108</td> <td>1225</td> <td>137,134,119,105,93,92,91,77,41</td> <td>C10H16O</td> <td>152.2334</td> <td>Mono</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> <td>0.27</td> <td>—</td> </tr> <tr> <td>21</td> <td>α-Limonene diepoxide</td> <td>48.755</td> <td>1294</td> <td>137,123,95,79,67</td> <td>C10H16O2</td> <td>168.2328</td> <td>Mono</td> <td>—</td> <td>—</td> <td>—</td> <td>2.34</td> <td /> <td>—</td> </tr> <tr> <td>22</td> <td>Ledol</td> <td>48.982</td> <td>1580</td> <td>204,189,161,135,133,109,107,69</td> <td>C15H26O</td> <td>222.3663</td> <td>Sesqui</td> <td>—</td> <td>—</td> <td>—</td> <td>0.28</td> <td>2.22</td> <td>—</td> </tr> <tr> <td>23</td> <td>Caryophyllene</td> <td>50.085</td> <td>1420</td> <td>204,189,161,133,120,93,79,69</td> <td>C15H24</td> <td>204.3511</td> <td>Sesqui</td> <td>30.71</td> <td>31.53</td> <td>—</td> <td>45.45</td> <td>54.04</td> <td>72.67</td> </tr> <tr> <td>24</td> <td>Isopulegol</td> <td>50.305</td> <td>1143</td> <td>154,136,121,93,81,67</td> <td>C10H18O</td> <td>154.2493</td> <td>Mono</td> <td>—</td> <td>0.12</td> <td /> <td>—</td> <td>—</td> <td>—</td> </tr> <tr> <td>25</td> <td>.alpha.-Guaiene</td> <td>50.47</td> <td>1437</td> <td>204,189,161,147,133,105,93,79</td> <td>C15H24</td> <td>204.3511</td> <td>Diter</td> <td>—</td> <td>—</td> <td>0.57</td> <td>—</td> <td>—</td> <td>—</td> </tr> <tr> <td>26</td> <td>5.beta.,7.beta.H,10.alpha.-Eudesm-11-en-1.alpha.-ol</td> <td>50.703</td> <td>1638</td> <td>222,207,161,137,125,95,81,55</td> <td>C15H26O</td> <td>222.3663</td> <td>Sesqui</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> <td>0.24</td> <td>—</td> </tr> <tr> <td>27</td> <td>Lycopene</td> <td>51.348</td> <td>3949</td> <td>457,413,347,321,269,203,177</td> <td>C40H56</td> <td>536.89</td> <td>Sesqui</td> <td>—</td> <td>—</td> <td>0.19</td> <td>0.39</td> <td>—</td> <td>—</td> </tr> <tr> <td>28</td> <td>α-Limonene diepoxide</td> <td>53.194</td> <td>1294</td> <td>137,123,95,79,67</td> <td>C10H16O2</td> <td>168.2328</td> <td>Mono</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> <td>0.15</td> <td>—</td> </tr> <tr> <td>29</td> <td>α-Elemol</td> <td>53.591</td> <td>1546</td> <td>204,189,161,121,107,93,81,59</td> <td>C15H26O</td> <td>222.3663</td> <td>Sesqui</td> <td /> <td>0.53</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> </tr> <tr> <td>30</td> <td>Triadimenol</td> <td>53.554</td> <td>2023</td> <td>168,128,112,99,70,59,43,41,29</td> <td>C14H18ClN3O2</td> <td>295.765</td> <td>Sesqui</td> <td>—</td> <td>—</td> <td>0.2</td> <td>—</td> <td>—</td> <td>—</td> </tr> <tr> <td>31</td> <td>Squalene</td> <td>53.568</td> <td>2833</td> <td>367,341,299,203,175,161,137</td> <td>C30H50</td> <td>410.7180</td> <td>Triter</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> <td>0.48</td> <td>—</td> </tr> <tr> <td>32</td> <td>Verbenol</td> <td>53.994</td> <td>1130</td> <td>137,119,94,81,59</td> <td>C10H16O</td> <td>152.2334</td> <td>Sesqui</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> <td>0.1</td> <td>—</td> </tr> <tr> <td>33</td> <td>Caryophyllene oxide</td> <td>54.241</td> <td>1546</td> <td>220,205,177,161,135,121,93,79</td> <td>C15H24O</td> <td>220.3505</td> <td>Sesqui</td> <td>—</td> <td>—</td> <td>0.09</td> <td>—</td> <td>—</td> <td>—</td> </tr> <tr> <td>34</td> <td>Globulol</td> <td>54.833</td> <td>1581</td> <td>204,149,161,109,82,69,55,43,41</td> <td>C15H26O</td> <td>222.3663</td> <td>Sesqui</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> <td>0.05</td> <td>—</td> </tr> <tr> <td>35</td> <td>Caryophyllene oxide</td> <td>58.543</td> <td>1583</td> <td>205,177,149,138,109,93,91,79</td> <td>C15H24O</td> <td>220.3505</td> <td>Sesqui</td> <td>—</td> <td>—</td> <td>0.13</td> <td>—</td> <td>—</td> <td>—</td> </tr> <tr> <td>36</td> <td>Dotriacontane</td> <td>74.491</td> <td>3200</td> <td>407,379,337,295,253,225,127</td> <td>C32H66</td> <td>450.8664</td> <td /> <td>5.48</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> </tr> <tr> <td>37</td> <td>Trans-Squalene</td> <td>74.571</td> <td>2835</td> <td>341,328,299,203,149,127,81,69</td> <td>C30H50</td> <td>410.7180</td> <td>Triter</td> <td>—</td> <td>—</td> <td>—</td> <td>0.13</td> <td>—</td> <td>1.24</td> </tr> </tbody> </table> </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 7</span> </a> </div>   <p>Phylogenetic analysis of terpenoid biosynthesis genes from <i>S</i>. <i>officinalis</i> and other plants. MEGA6 program was used for building up the tree through neighbor joining method. </p>   </div> <p /> <p>The putative functions of <i>TPS</i> genes isolated from <i>S</i>. <i>officinalis</i> were initially predicted according to the conserved motifs using the InterPro protein sequence analysis &amp;amp; classification ( <a href="http://www.ebi.ac.uk/interpro/">http://www.ebi.ac.uk/interpro/</a>) database. The SoCINO protein with a 591-aa length has an N-terminal domain (IPR001906) from 66–279 aa and a metal-binding domain (IPR005630) from 265–589 aa; inside the latter domain are two motifs: one is an RR (x) 8 W motif (RRTGGYQPTLW) starting at 57 aa, and the other one is a DDxxD motif (DDVFD) starting at 345 aa. On the other hand, the <i>So</i>LINA protein is 505 aa in length. This protein has an N-terminal domain (IPR001906) from 1–183 aa and a metal-binding domain (IPR005630) from 171–497 aa, and inside the last domain are DDxxD conserved motifs (DDIFD) starting at 250 aa. Finally, the protein sequences contained one or two of this domain belonging to the <i>TPS</i> gene family. </p> <p>Croteau and coworkers had revealed the carbocationic reaction mechanism for all monoterpene synthases by reporting that the reaction was initiated by the divalent metal ion-dependent ionization of the substrate. The resulting cationic intermediate undergoes a series of hydride shifts or other rearrangements and cyclizations until the reaction was terminated by the addition of a nucleophile or proton loss. They also illustrated this reaction mechanism by studying the native enzymes with substrate inhibitors, analogues and intermediates <sup> <a href="#CR51">51</a>, <a href="#CR52">52</a> </sup>. Moreover, Croteau <i>et al</i>. 1987 <sup> <a href="#CR53">53</a> </sup> elucidated the preliminary conversion of the geranyl cation to the tertiary linalyl cation to facilitate cyclization to a six-membered ring. Afterwards, the linalyl cation provides the cyclic α-terpinyl cation; this is an important branching point intermediate in the formation of all cyclic monoterpenes because multiple terpene products can be obtained through electrophilic attack of C1 on the C6–C7 linalyl cation double bond and from the α-terpinyl cation <sup> <a href="#CR53">53</a> </sup>. From the previous discussion, the reaction mechanisms of monoterpene synthases are highly reticulate. The individual intermediate may have multiple fates, which suggests the explanation for the ability of terpene enzymes to make various terpene products <sup> <a href="#CR54">54</a>– <a href="#CR57">57</a> </sup>. On the other hand, the carbocationic reaction mechanism that uses sesquiterpene synthase to form sesquiterpenes by catalysing farnesyl pyrophosphate (FPP) recycling is similar to the reaction mechanism by those monoterpene synthases. Moreover, the larger carbon skeleton of FPP and the presence of three double bonds instead of two suggest a rationale for increases of the structural diversity of the sesquiterpene products. Furthermore, the initial cyclization reactions for sesquiterpene synthases can be divided into two types. Type one involves cyclization of the initially formed farnesyl cation to yield 11-membered (E)-humulyl cation) rings of large size and a C2–C3 double bond (this type has no barrier to cyclization). The second type involves cyclization that proceeds after the tertiary nerolidyl cation produced from preliminary isomerization of the C2–C3 double bond. This isomerization mechanism is directly analogous to the isomerization of GPP to yield a linalyl cation in monoterpene synthesis. The nerolidyl cation is considered an intermediate in the sesquiterpene synthase mechanism <sup> <a href="#CR58">58</a>– <a href="#CR62">62</a> </sup>. </p> <p>Collectively, we can state that the ability of <i>TPS</i> genes to convert a prenyl diphosphate substrate into diverse products during different reaction cycles is one of the unique traits of this type of enzyme. As described above, this property is found in the majority of all characterized monoterpene and sesquiterpene synthases. However, some monoterpene and sesquiterpene synthases can catalyse substrates into a single product, and the proteins may have specific methods for multiple product formations. For example, γ-humulene synthase from <i>A</i>. <i>grandis</i> has two DDxxD motifs located on opposite sides and can generate 52 different sesquiterpenes. This protein is able to bind substrates with two different conformations, resulting in different sets of products <sup> <a href="#CR63">63</a> </sup>. In another example regarding the first monoterpene synthase cloned from <i>Salvia officinalis</i>, (+)-sabinene synthase produces 63% (+)-sabinene but also 21% γ-terpinene, 7.0% terpinolene, 6.5% limonene and 2.5% myrcene in <i>in vitro</i> assays <sup> <a href="#CR64">64</a> </sup>. These additional monoterpene products or their immediate metabolites are also found in the monoterpene-rich essential oil of the <i>S</i>. <i>officinalis</i> plant. </p> </div> </div> <div class="conclusion" title="conclusion"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>In this study, a large, high-quality transcriptome database was established for <i>S</i>. <i>officinalis</i> leaves using NGS technology to characterize and to identify genes that are related to terpenoid biosynthesis. Using <i>de novo</i> sequencing and analysis of the <i>S</i>. <i>officinalis</i> transcriptome data via the Illumina HiSeq 2000 system, we identified many genes that encode enzymes involved in the terpenoid biosynthesis pathway. The purpose of identifying these genes is not only to facilitate functional studies but also to develop biotechnology for improving the production of medicinal ingredients through metabolic engineering. We profiled terpenoids from three tissues of <i>S</i>. <i>officinalis</i> and used qRT-PCR to determine the correlation between the expression levels of <i>TPS</i> genes and the end-products. By combining the transcriptome and metabolome analyses with RNA-Seq or qRT-PCR with GC-MS approaches, this study paves the way for understanding the complex metabolic genes for the production of the diverse terpene compounds in garden sage. The results from our study will allow to understand the specific activities of TPSs in <i>S</i>. <i>officinalis</i> for the production of interesting compounds and to develop new technology for utilization. </p> <p>To our knowledge, this is the first study to use Illumina HiSeq 2000 paired-end sequencing technology to investigate the global transcriptome of <i>S</i>. <i>officinalis</i>. The valuable genetic resource in <i>Salvia</i> will provide the foundation for future genetic and functional genomic research on <i>S</i>. <i>officinalis</i> or closely related species. We further studied the functions of various <i>S</i>. <i>officinalis TPS</i> genes, including <i>SoNEOD</i>, <i>SoCINS</i>, <i>SoSABS</i>, <i>SoLINS</i>, and <i>SoTPS6</i>, by stably expressing these genes in <i>N</i>. <i>tabacum</i> transgenic plants. <i>SoNEOD</i>, <i>SoCINS</i>, <i>SoSABS</i>, <i>SoLINS</i>, and <i>SoTPS6</i> were functionally expressed in the leaves of <i>N</i>. <i>tabacum</i>, and these transgenes altered the levels of terpenoids, as confirmed by GC-MS analysis of extracted transgenic <i>N</i>. <i>tabacum</i> leaves. The GC-MS analysis revealed that these <i>S</i>. <i>officinalis</i> terpene synthases isolated from <i>S</i>. <i>officinalis</i> can convert a prenyl diphosphate substrate into diverse products, which is one of the unique traits of this type of enzyme. Our study provides new insights into our understanding of plant terpenoid biosynthesis and the potential for biotechnology application. </p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="plantmaterialsandtissuecollection" title="sec"> <div class="title" tagx="title" title="title">
Plant materials and tissue collection</div> <p>Seeds of <i>Salvia officinalis</i> were collected from the Egyptian Desert Gene Bank, North Sinai Research Station, Department of Plant Genetic Resources, Desert Research Center, Egypt, and grown at Huazhong Agricultural University, Wuhan, China. Different tissues were sampled from one-year-old <i>S</i>. <i>officinalis</i> plants. For RNA-Seq, three biological replicates from leaves were sampled and handled. Each replicate consisted of two young and two old leaves from the same plant. For qRT-PCR, three biological replicates were collected from the following five parts (young leaves, old leaves, stems, flowers and bud flowers). All samples were immediately frozen in liquid nitrogen and then stored at −80 °C until RNA extraction. Furthermore, another three biological replicates from the individual three fresh parts were collected for isolation of the essential oil. </p> </div> <div class="isolationofchemicalcompounds" title="sec"> <div class="title" tagx="title" title="title">
Isolation of chemical compounds</div> <p>The correct method to reduce technical variability throughout a sampling procedure is essential to stop cell metabolism and to avoid leaking of metabolites during the various preparation steps before the actual metabolite extraction. Therefore, three biological replicates from each of the three fresh parts were immediately frozen on dry ice. In the laboratory, the frozen three biological replicates from each of the three fresh part samples were homogenized in liquid nitrogen with a mortar and pestle, after which the plant material (ca. 10 g) was directly soaked in n-hexane as a solvent in Amber storage bottles, 60 ml screw-top vials with silicone/PTFE septum lids ( <a href="http://www.sigmaaldrich.com">http://www.sigmaaldrich.com</a>) were used to reduce loss of volatiles to the headspace then incubated with shaking at 37 °C and 200 rpm for 72 h. Afterward, the solvent was transferred using a glass pipette to a 10-ml glass centrifuge tube with screw-top vials with silicone/PTFE septum lids and centrifuged at 5,000 rpm for 10 minutes at 4 °C to remove plant debris. The supernatant was pipetted into glass vials with a screw cap and oil was concentrated until remaining 1.5 ml of concentrated oils under a stream of nitrogen gas with a nitrogen evaporator (Organomation) and water bath at room temperature (Toption-China-WD-12). The concentrated oils transferred to a fresh crimp vial amber glass, 1.5 ml screw-top vials with silicone/PTFE septum lids were used to reduce a loss of volatiles to the headspace. For absolute oil recovery, the remaining film crude oil in the internal surface of concentrated glass vials was dissolved in the minimum volume of n-hexane, thoroughly mixed and transferred to the same fresh crimp vial amber glass, 1.5 ml. And the crimp vial was placed on the autosampler of GC-MS system for GC-MS analysis, or each tube was covered with parafilm after closed with screw-top vials with silicone/PTFE septum lids and stored at −20 °C until GC-MS analysis. </p> </div> <div class="gc-msanalysisofessentialoilcomponents" title="sec"> <div class="title" tagx="title" title="title">
GC-MS analysis of essential oil components</div> <p>GC analysis was performed using a Shimadzu model GCMS-QP2010 Ultra (Tokyo, Japan) system. An approximately 1 µl aliquot of each sample was injected (split ratios of 15:1) into a GC-MS equipped with an HP-5 fused silica capillary column (30 m × 0.25 mm ID, 0.25 µm film thickness). Helium was used as the carrier gas at a constant flow of 1.0 ml min <sup>−1</sup>. The mass spectra were monitored between 50–450 m/z. Temperature was initially under isothermal conditions at 60 °C for 10 min. Temperature was then increased at a rate of 4 °C min <sup>−1</sup> to 220 °C, held isothermal at 220 °C for 10 min, increased by 1 °C min <sup>−1</sup> to 240 °C, held isothermal at 240 °C for 2 min, and finally held isothermal for 10 min at 350 °C. The identification of the volatile constituents were done by parallel comparison of their recorded mass spectra with the data stored in the Wiley GC/MS Library (10 <sup>th</sup> Edition) (Wiley, New York, NY, USA), and the retention time index ( <a href="http://massfinder.com/wiki/MassFinder_Analysing_your_own_data">http://massfinder.com/wiki/MassFinder_Analysing_your_own_data</a>), with the Volatile Organic Compounds (VOC) Analysis S/W software, and the NIST Library (2014 edition), The Adams Library ( <a href="http://essentialoilcomponentsbygcms.com/list-of-compounds-in-the-essential-oil-components-database/">http://essentialoilcomponentsbygcms.com/list-of-compounds-in-the-essential-oil-components-database/</a>), and the Terpenoids Library ( <a href="http://massfinder.com/wiki/Terpenoids_Library_List">http://massfinder.com/wiki/Terpenoids_Library_List</a>). The relative% amount of each component was calculated by comparing its average peak area to the total areas, as well as Retention time index. (All of the experiments were performed simultaneously three times under the same conditions for each isolation technique with total GC running time was 80 minutes. </p> </div> <div class="rnaextraction" title="sec"> <div class="title" tagx="title" title="title">
RNA extraction</div> <p>Total RNAs from the three biological leaf replicates were extracted for RNA-Seq. Moreover, total RNAs from three biological replicates from each of the plant parts (young leaves, old leaves, stems, flowers and bud flowers) were extracted for qRT-PCR. Additionally, total RNAs from three biological replicates of transgenic <i>N</i>. <i>tabacum</i> were extracted for semiquantitative RT-PCR using the TRIzol Reagent (Invitrogen, USA) and treated with DNase I (Takara). RNA quality was examined on 1% agarose gels, and the purity was analysed using a Nano-Photometer® spectrophotometer (IMPLEN, CA, USA). RNA concentration was determined using a Qubit® RNA Assay Kit in a Qubit® 2.0 Fluorometer (Life Technologies, CA, USA). RNA pools were prepared for cDNA libraries by mixing equal volumes from the three RNAs replications in one tube. </p> </div> <div class="cdnalibrarypreparationandsequencing" title="sec"> <div class="title" tagx="title" title="title">
cDNA library preparation and sequencing</div> <p>Three micrograms of RNA per sample were used for generating a sequencing library. cDNA was synthesized using an RNA Library Prep Kit for Illumina® (NEB, USA) for generated sequencing libraries according to the manufacturer’s instructions. The first strand of cDNA was synthesized in the presence of random hexamer primers and M-MuLV Reverse Transcriptase (RNase H), and the second strand of cDNA was synthesized in the presence of DNA polymerase I and RNase H. The remaining cDNA was converted into blunt ends in the presence of exonuclease/polymerase activities. After the adenylation of 3′ ends of DNA fragments, NEB Next, an adaptor with a hairpin loop structure, was ligated to prepare for hybridization. To select cDNA fragments of preferentially 150~200 bp in length, the library fragments were purified using an AMPure XP system (Beckman Coulter, Beverly, USA). Then, 3 μl of USER Enzyme (NEB, USA) was used with size-selected, adaptor-ligated cDNA at 37 °C for 15 min followed by 95 °C for 5 min. Afterwards, PCR was performed with Phusion High-Fidelity DNA polymerase, universal PCR primers and Index (X) Primer. Finally, PCR products were purified (AMPure XP system), and the library quality was assessed using an Agilent Bioanalyzer 2100 system (Agilent Technologies, CA, USA). Clustering of the index-coded samples was performed on a cBot Cluster Generation System using a TruSeq PE Cluster Kit v3-cBot-HS (Illumina) according to the manufacturer’s instructions (Novogene Experimental Department). After cluster generation, the library preparations were sequenced on an Illumina HiSeq 2000 platform, and paired-end reads were generated. </p> </div> <div class="qualitycontrol" title="sec"> <div class="title" tagx="title" title="title">
Quality control</div> <p>Raw data (raw reads) in fastq format were first processed through in-house Perl scripts. During this step, clean data (clean reads) were obtained by removing reads containing adapters, reads containing ploy-N and low-quality reads from the raw data. At the same time, Q20, Q30, GC content and sequence duplication level of the clean data were calculated. All of the downstream analyses were based on high-quality clean data. </p> </div> <div class="denovotranscriptomeassembly" title="sec"> <div class="title" tagx="title" title="title"> <i>De novo</i> transcriptome assembly </div> <p> <i>De novo</i> assembly of the processed reads was carried out using Trinity program (Version: trinityaseq_r 2012-10-05) <sup> <a href="#CR20">20</a> </sup>, with the min_kmer_cov set to 2 by default and all other parameters set to default. The Trinity method consists of three software modules, (1) Inchworm, (2) Chrysalis and (3) Butterfly, applied sequentially to process large volumes of RNA-Seq reads. In the first step, read datasets were assembled into linear contigs by the first module (Inchworm program). The minimally overlapping contigs were then clustered into sets of connected components (build graph components) by the second module (Chrysalis program), and the transcripts were then constructed from each de Bruijn graph by the third software module (Butterfly program). Finally, the transcripts were clustered by similarity of correct match length beyond 80% for longer transcripts or 90% for shorter transcripts using the multiple sequence alignment tool. </p> </div> <div class="annotationofunigenes" title="sec"> <div class="title" tagx="title" title="title">
Annotation of unigenes</div> <p>Unigenes were used as query sequences to search the annotation databases, including the NCBI non-redundant protein sequences database (NR) ( <a href="http://www.ncbi.nlm.nih.gov/">http://www.ncbi.nlm.nih.gov/</a>) and Swiss-Prot (a manually annotated and reviewed protein sequence database) ( <a href="http://www.ebi.ac.uk/uniprot/">http://www.ebi.ac.uk/uniprot/</a>), based on sequence homology to entries in the Gene Ontology (GO) database ( <a href="http://www.geneontology.org/">http://www.geneontology.org/</a>). Unigene sequences from <i>S</i>. <i>officinalis</i> were categorized into three general sections: biological process (BP), cellular component (CC) and molecular function (MF). Additionally, the unigenes were used as query sequences for searching the Kyoto Encyclopedia of Genes and Genome (KEGG) pathways database ( <a href="http://www.genome.jp/kegg/">http://www.genome.jp/kegg/</a>) and the Pfam (Protein family) database ( <a href="http://pfam.sanger.ac.uk/">http://pfam.sanger.ac.uk/</a>). </p> </div> <div class="differentialexpressionanalysis" title="sec"> <div class="title" tagx="title" title="title">
Differential expression analysis</div> <p>Expression levels of unigenes were normalized and calculated as the values of fragments per kilobase of transcripts per million mapped fragments (FPKM) during the assembly and clustering process. Differential expression analysis of unigenes was performed using the DESeq R package (1.10.1). DESeq provides statistical routines for assessing the differential gene expression in leaf tissues and assigns genes as differentially expressed when the P-value &amp;lt; 0.05. P-value results were corrected using the Benjamini and Hochberg approach for controlling the false discovery rate (FDR) <sup> <a href="#CR65">65</a> </sup>. </p> </div> <div class="quantitativereal-timepcr(qrt-pcr)analysis" title="sec"> <div class="title" tagx="title" title="title">
Quantitative real-time PCR (qRT-PCR) analysis</div> <p>Quantitative RT-PCR was performed using an IQ <sup>TM</sup> 5 Multicolor Real-Time PCR Detection System (Bio-Rad, USA) as described previously <sup> <a href="#CR66">66</a> </sup> with SYBR Green Master (ROX) (Newbio Industry, China) following the manufacturer’s instructions at a total reaction volume of 20 µl. Gene-specific primers for <i>So</i>Actin as a reference gene and for the other eleven genes ( <i>SoNEOD</i>, <i>SoGPS</i>, <i>SoFPPS</i>, <i>SoGGPS</i>, <i>SoMYRS</i>, <i>SoLINS</i>, <i>SoHUMS</i>, <i>SoTPS6</i>, <i>SoSQUS</i>, <i>SoSABS</i> and <i>SoCINS</i>) involved in the biosynthesis of terpenes were designed using the primer designing tools of IDTdna ( <a href="http://www.idtdna.com">http://www.idtdna.com</a>), as listed in Supplementary Table <a href="#MOESM1">S9</a>. The quantitative RT-PCR conditions were set as standard conditions: 95 °C for 3 min, 40 cycles of amplification (95 °C for 10 s, 60 or 58 °C for 30 s and 72 °C for 20 s), and a final extension at 65 °C for 1 min. The gene expression was normalized using <i>SoActin</i> as a reference gene. The relative expression levels were calculated by comparing the cycle thresholds (CTs) of the target genes with that of the reference gene <i>SoActin</i> using the 2 <sup>−ΔΔCt</sup> method <sup> <a href="#CR67">67</a>, <a href="#CR68">68</a> </sup>. The sizes of amplification products were 140–160 bp. The quantified data were analysed using the Bio-Rad IQ <sup>TM</sup> 5 Multicolor Real-Time Manager software. Finally, the relative expression levels of <i>SoNEOD</i>, <i>SoGPS</i>, <i>SoFPPS</i>, <i>SoGGPS</i>, <i>SoMYRS</i>, <i>SoLINS</i>, <i>SoHUMS</i>, <i>SoTPS6</i>, <i>SoSQUS</i>, <i>SoSABS</i>, and <i>SoCINS</i> were detected. All reactions were performed with three replications. </p> </div> <div class="identificationofsimplesequencerepeats(ssrs)" title="sec"> <div class="title" tagx="title" title="title">
Identification of simple sequence repeats (SSRs)</div> <p>All of the transcripts of <i>S</i>. <i>officinalis</i> were analysed with the MISA program version 1.0 ( <a href="http://pgrc.ipkgatersleben.de/misa/misa.html">http://pgrc.ipkgatersleben.de/misa/misa.html</a>) for the detection of SSR motifs that have mono- to hexanucleotide repeats. In addition, primers for each SSR were designed using Primer3 version 2.3.5 (http://primer3.sourceforge.-net/releases.php). The minimum number of SSR repeat units during analysis was ≥24 for mono- and dinucleotides and was 8, 7, 7, and 9 for tri-, tetra-, penta-, and hexanucleotide repeats, respectively. The default parameters corresponding to each unit size of the minimum number of repetitions were 1–10, 2–6, 3, 5, 4, 5, 5, 5, and 6-5 for Unigene SSR detection. </p> </div> <div class="full-lengthterpenesynthasecdnaclonesandvectors" title="sec"> <div class="title" tagx="title" title="title">
Full-length terpene synthase cDNA clones and vectors</div> <p>Full-length cDNAs for <i>SoNEOD</i>, <i>SoCINS</i>, <i>SoSABS</i>, <i>SoLINS</i> and <i>SoTPS6</i> were obtained by PCR amplification using short and long gene-specific primers (Supplementary Table <a href="#MOESM1">S11</a>) based on RNA-Seq sequence information from the transcriptome sequencing of <i>S</i>. <i>officinalis</i> leaves. Leaf cDNA was used as a template for the initial PCR amplification and performed using short primers with the KOD-Plus DNA polymerase (Novagen) under the following PCR conditions: 3 min at 94 °C followed by 10 s at 98 °C; 30 s at 60, 57, 59, 60 or 60 °C (different annealing temperatures), 1.5 min at 68 °C, and then 10 min at 68 °C. This process was repeated for 35 cycles. The cDNA was used as a template for PCR cloning using long primers with the KOD-Plus DNA polymerase for the Gateway pDONR221 vector. The amplified PCR products were purified and cloned into the Gateway entry vector pDONR221 using bp Clonase (Invitrogen, USA). The resulting pDONR221 constructs harbouring target genes were sequenced, and Gateway LR Clonase (Invitrogen, USA) was used for recombination into the destination vector pB2GW7 for tobacco transformation. All final constructs containing <i>SoNEOD</i>, <i>SoCINS</i>, <i>SoSABS</i>, <i>SoLINS</i> and <i>SoTPS6</i> were confirmed by sequencing. </p> </div> <div class="nicotianaplantgrowthconditionsandpreparationofagrobacteriumculturesforinfection" title="sec"> <div class="title" tagx="title" title="title"> <i>Nicotiana</i> plant growth conditions and preparation of <i>Agrobacterium</i> cultures for infection </div> <p>Wild-type <i>Nicotiana tabacum</i> plant seeds were grown under standard greenhouse conditions for ten days at the Wuhan Doublehelix Biology Science and Technology Company, Wuhan, Hubei, China. In addition, the constructs of pB2GW7 vectors with all inserted genes were introduced into <i>Agrobacterium tumefaciens</i> strain EHA105 by direct electroporation. Recombinant <i>A</i>. <i>tumefaciens</i> was grown for two days at 28 °C in solid LB media supplemented with 50 μg/ml each of rifampicin and spectinomycin. An individual colony of each sample was inoculated into 1.0 ml of liquid medium and grown at 28 °C under 200 rpm agitation overnight with the same media composition. After 24 h, 1.0 ml of each sample of liquid medium was transferred to a 250-ml conical flask containing 50 ml of LB media supplemented with the same compositions; the samples were grown at 28 °C in a shaker overnight until an optical density of 0.6–1.0 (OD 600) was reached. Overnight cell cultures were harvested by centrifugation at 5,000 rpm for 10 min at 4 °C, and the pellet was resuspended in the infection medium (50 ml of LB-free media + 50 μl of acetosyringone). <i>Nicotiana tabacum</i> plantlet leaves were collected from the greenhouse and sterilized by soaking in 70% ethanol for 30 s, soaking in 0.1% HgCl for 6 min, and then washing three times using autoclaved water each time for 3 min. Then, we cut the leaves into small pieces (1 cm × 1 cm) and discarded the petiole and midrib, after which the leaf pieces were soaked in Petri dishes with infection media for 10 min and stirred every 2 min. The transformation procedure was performed as described previously <sup> <a href="#CR69">69</a> </sup>. More than 15 individual transgenic tobacco lines were generated for each transgene and examined with PCR for positive transgenic lines of more than 10 lines for each transgene. The positive plants with good roots were transferred to the greenhouse for adaptation. Then, the transgenic tobacco plants were analysed for terpenoid profiling and target gene expression. </p> </div> <div class="semiquantitativert-pcranalysis" title="sec"> <div class="title" tagx="title" title="title">
Semiquantitative RT-PCR analysis</div> <p>Semiquantitative real-time PCR was performed on an Eppendorf PCR (Eppendorf Mastercycler-Nexus GSX1, POCD Scientific, Australia) system with a total reaction volume of 25 µl. A gene-specific primer for NtEF-1α ( <i>Nicotiana tabacum</i> EF-1-alpha-related GTP-binding protein) was used as a reference gene, and the other five gene-specific primers for <i>SoNEOD</i>, <i>SoCINS</i>, <i>SoSABS</i>, <i>SoLINS</i>, and <i>SoTPS6</i>, which are involved in the biosynthesis of terpenes, were designed using the primer designing tools of IDTdna ( <a href="http://www.idtdna.com/scitools/Applications/RealTimePCR/">http://www.idtdna.com/scitools/Applications/RealTimePCR/</a>); the primer sequences are listed in (Supplementary Table <a href="#MOESM1">S9</a>). The Semiquantitative RT-PCR conditions were as follows: predenaturation step at 95 °C for 4 min, 35 cycles of amplification (95 °C for 30 s, 58 or 60 °C for 30 s and 72 °C for 1 min), and a final extension step at 72 °C for 10 min. The PCR products were resolved on 1% agarose gel, and the expression levels of <i>NtEF-1α</i>, <i>SoNEOD</i>, <i>SoCINS</i>, <i>SoSABS</i>, <i>SoLINS</i>, and <i>SoTPS6</i> were detected. </p> <div class="metaboliteextractionfromtransgenictobaccoleaves" title="sec"> <div class="title" tagx="title" title="title">
Metabolite extraction from transgenic tobacco leaves</div> <p>Terpenoid compounds from non-transgenic tobacco leaves (control) and transgenic tobacco leaves containing either <i>SoNEOD</i>, <i>SoCINS</i>, <i>SoSABS SoLINS</i>, or <i>SoTPS6</i> expression constructs were extracted and isolated. For this, three leaves from each transgenic tobacco line (one leaf from each plant) were homogenized in liquid nitrogen with a mortar and pestle, after which the plant material powder was directly soaked in n-hexane as a solvent in Amber storage bottles, 60 ml screw-top vials with silicone/PTFE septum lids ( <a href="http://www.sigmaaldrich.com">http://www.sigmaaldrich.com</a>) were used to reduce loss of volatiles to the headspace then incubated with shaking at 37 °C and 200 rpm for 72 h. Afterward, the solvent was transferred using a glass pipette to a 10-ml glass centrifuge tube with screw-top vials with silicone/PTFE septum lids and centrifuged at 5,000 rpm for 10 minutes at 4 °C to remove plant debris. The supernatant was pipetted into glass vials with a screw cap and oil was concentrated until remaining 1.5 ml of concentrated oils under a stream of nitrogen gas with a nitrogen evaporator (Organomation) and water bath at room temperature (Toption-China-WD-12). The concentrated oils transferred to a fresh crimp vial amber glass, 1.5 ml screw-top vials with silicone/PTFE septum lids were used to reduce a loss of volatiles to the headspace. For absolute oil recovery, the remaining film crude oil in the internal surface of concentrated glass vials was dissolved in the minimum volume of n-hexane, thoroughly mixed and transferred to the same fresh crimp vial amber glass, 1.5 ml. And the crimp vial was placed on the autosampler of the gas chromatography mass spectrometer (GC-MS) system for GC-MS analysis, or each tube was covered with parafilm after closed with screw-top vials with silicone/PTFE septum lids and stored at −20 °C until GC-MS analysis. The same programme and standard conditions that were used for GC-MS analysis with <i>S</i>. <i>officinalis</i> essential oil components were applied. </p> </div> </div> <div class="geneaccessionnumber" title="sec"> <div class="title" tagx="title" title="title">
Gene accession number</div> <p>Gene accession numbers: Genes studied here are accessible to GenBank. Salvia officinalis geranyl-diphosphate synthase ( <i>SoGPS</i>, KY399788); farnesyl pyrophosphate synthetase ( <i>SoFPPS</i>, KY399787); (3S)-linalool synthase ( <i>SoLINS</i>, KY399786); terpene synthase 6 ( <i>SoTPS6</i>, KY399785); (−)-germacrene D synthase ( <i>SoSABS</i>, KY399783); <i>Salvia officinalis</i> 1,8-cineole synthase ( <i>SoCINS</i>, KY399782); <i>Salvia officinalis</i> geranyl diphosphate synthase 2 ( <i>SoGGPP</i>, KY486794); <i>Salvia officinalis</i> squalene monooxygenase ( <i>SoSQUS</i>, KY486795). </p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41598_2017_15478_MOESM1_ESM.pdf">LINK</a>  <p>Supplementary Tables and Figures</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p> <b>Electronic supplementary material</b> </p> <p> <b>Supplementary information</b> accompanies this paper at 10.1038/s41598-017-15478-3. </p> </div> <div class="fn-type-" title=""> <p> <b>Publisher's note:</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. </p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>This study was supported by the National Basic Research Development Program of China (973 Program, Grant 2013CB127001), and collaboration fund between Anhui Agricultural Univeristy and Huazhong Agricultural University. The authors thank Dr. Hongbo Liu for assistance for running the GC-Ms and data analysis, and thank Dr. Mohamed Amar and Dr. Wael Moussa for providing <i>S</i>. <i>officinalis</i> seeds. The authors thank Dr. Hazem Abdelnabby, Dr. Mohammed Ayaad and Dr. Mahmoud Khalil for proof-reading the manuscript. </p> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>J.Z. conceived and designed the study, M.A., P.L., G.S. performed experiments, D.C. performed tobacco transformation, J.Z. and M.A. wrote the paper. All authors discussed the results and commented on the manuscript and participated in the analysis of the data. </p> <div class="competinginterests" title="sec"> <div class="title" tagx="title" title="title">
Competing Interests</div> <p>The authors declare that they have no competing interests.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="CR1" /> <span class="label" tagx="label" title="label">1.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Alziar, G. Catalogue synonymique des Salvia L. dumonde (Lamiaceae). 5 (3–4):87–136; 6(1–2, 4):79–115, 163–204; 7(1–2):59–109; 9(2–3):413–497; 10(3–4). ( <i>I</i>.– <i>VI</i>. Biocosme Mesogeén) 33–117 (1988–1993). </span> </li> <li tag="ref"> <a name="CR2" /> <span class="label" tagx="label" title="label">2.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Atsuko</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hiroshi</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Phylogenetic relationships among subgenera, species, and varieties of Japanese Salvia L. (Lamiaceae) </span> <span class="source" tagx="source" title="source">J Plant Res.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">124</span> <span class="fpage" tagx="fpage" title="fpage">245</span> <span class="lpage" tagx="lpage" title="lpage">252</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s10265-010-0367-9">10.1007/s10265-010-0367-9</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20628783">20628783</a> </span> </span> </li> <li tag="ref"> <a name="CR3" /> <span class="label" tagx="label" title="label">3.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carretero-Paulet</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Campositionm Expression and molecular analysis of the Arabidopsis DXR gene encoding 1-Deoxy-D-xylulose-5-phosphate reductoisomerase, the first committed enzyme of the 2-C-Methyl-D-erythritol-4-phosphate pathway </span> <span class="source" tagx="source" title="source">Plant Physiol.</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">129</span> <span class="fpage" tagx="fpage" title="fpage">1581</span> <span class="lpage" tagx="lpage" title="lpage">1591</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.003798">10.1104/pp.003798</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12177470">12177470</a> </span> </span> </li> <li tag="ref"> <a name="CR4" /> <span class="label" tagx="label" title="label">4.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Davis</span> <span class="given-names" tagx="given-names" title="given-names">LC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Verpoorte</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Elicitor signal transduction leading to production of plant secondary metabolites</span> <span class="source" tagx="source" title="source">Biotechnology advances</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">23</span> <span class="fpage" tagx="fpage" title="fpage">283</span> <span class="lpage" tagx="lpage" title="lpage">333</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.biotechadv.2005.01.003">10.1016/j.biotechadv.2005.01.003</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15848039">15848039</a> </span> </span> </li> <li tag="ref"> <a name="CR5" /> <span class="label" tagx="label" title="label">5.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ward</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ponnala</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weber</span> <span class="given-names" tagx="given-names" title="given-names">CA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Strategies for transcriptome analysis in nonmodel plants</span> <span class="source" tagx="source" title="source">Am J Bot.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="fpage" tagx="fpage" title="fpage">267</span> <span class="lpage" tagx="lpage" title="lpage">276</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3732/ajb.1100334">10.3732/ajb.1100334</a> </span> </span> </li> <li tag="ref"> <a name="CR6" /> <span class="label" tagx="label" title="label">6.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shubhra</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">De novo sequencing and comparative analysis of holy and sweet basil transcriptomes</span> <span class="source" tagx="source" title="source">BMC Genomics.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">15</span> <span class="fpage" tagx="fpage" title="fpage">588</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-15-588">10.1186/1471-2164-15-588</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25015319">25015319</a> </span> </span> </li> <li tag="ref"> <a name="CR7" /> <span class="label" tagx="label" title="label">7.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hua</span> <span class="given-names" tagx="given-names" title="given-names">WP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Song</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">LJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">ZZ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">De novo transcriptome sequencing in Salvia miltiorrhiza to identify genes involved in the biosynthesis of active ingredients </span> <span class="source" tagx="source" title="source">Genomics.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">98</span> <span class="fpage" tagx="fpage" title="fpage">272</span> <span class="lpage" tagx="lpage" title="lpage">279</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ygeno.2011.03.012">10.1016/j.ygeno.2011.03.012</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21473906">21473906</a> </span> </span> </li> <li tag="ref"> <a name="CR8" /> <span class="label" tagx="label" title="label">8.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hyun</span> <span class="given-names" tagx="given-names" title="given-names">TK</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">De novo transcriptome sequencing of Momordica cochinchinensisto identify genes involved in the carotenoid biosynthesis </span> <span class="source" tagx="source" title="source">Plant Mol Biol.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">79</span> <span class="fpage" tagx="fpage" title="fpage">413</span> <span class="lpage" tagx="lpage" title="lpage">427</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11103-012-9919-9">10.1007/s11103-012-9919-9</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22580955">22580955</a> </span> </span> </li> <li tag="ref"> <a name="CR9" /> <span class="label" tagx="label" title="label">9.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">HH</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">De novo characterization of the Chinese fir (Cunninghamia lanceolata) transcriptome and analysis of candidate genes involved in cellulose and lignin biosynthesis </span> <span class="source" tagx="source" title="source">BMC Genomics.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="fpage" tagx="fpage" title="fpage">648</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-13-648">10.1186/1471-2164-13-648</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23171398">23171398</a> </span> </span> </li> <li tag="ref"> <a name="CR10" /> <span class="label" tagx="label" title="label">10.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shi</span> <span class="given-names" tagx="given-names" title="given-names">CY</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Deep sequencing of the Camellia sinens is transcriptome revealed candidate genes for major metabolic pathways of tea-specific compounds </span> <span class="source" tagx="source" title="source">BMC Genomics.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="fpage" tagx="fpage" title="fpage">131</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-12-131">10.1186/1471-2164-12-131</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21356090">21356090</a> </span> </span> </li> <li tag="ref"> <a name="CR11" /> <span class="label" tagx="label" title="label">11.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aziz</span> <span class="given-names" tagx="given-names" title="given-names">RA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hamed</span> <span class="given-names" tagx="given-names" title="given-names">FK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Abdulah</span> <span class="given-names" tagx="given-names" title="given-names">NA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Determination of the main components of the essential oil extracted from <i>Salvia fruticosa</i> by sing GC and GC-MS DAMASCUS </span> <span class="source" tagx="source" title="source">J AGR SCI.</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">24</span> <span class="fpage" tagx="fpage" title="fpage">223</span> <span class="lpage" tagx="lpage" title="lpage">236</span> </span> </li> <li tag="ref"> <a name="CR12" /> <span class="label" tagx="label" title="label">12.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nadaf</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nasrabadi</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Halimi</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">GC-MS Analysis of n-Hexane Extract from Aerial Parts of <i>Salvia nemorosa</i> </span> <span class="source" tagx="source" title="source">Middle-East Journal of Scientific Research.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">1127</span> <span class="lpage" tagx="lpage" title="lpage">1130</span> </span> </li> <li tag="ref"> <a name="CR13" /> <span class="label" tagx="label" title="label">13.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Christophe</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Characterization of two genes for the biosynthesis of the labdane diterpene Z-abienol in tobacco ( <i>Nicotiana tabacum</i>) glandular trichomes </span> <span class="source" tagx="source" title="source">The Plant Journal.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">74</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">17</span> </span> </li> <li tag="ref"> <a name="CR14" /> <span class="label" tagx="label" title="label">14.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Monica</span> <span class="given-names" tagx="given-names" title="given-names">RL</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Comparative Chemical Composition and Antiproliferative Activity of Aerial Parts of <i>Salvia leriifolia</i> Benth. And <i>Salvia acetabulosa</i> L. Essential Oils Against Human Tumor Cell <i>In Vitro</i> Models </span> <span class="source" tagx="source" title="source">J Med Food.</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="fpage" tagx="fpage" title="fpage">62</span> <span class="lpage" tagx="lpage" title="lpage">69</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1089/jmf.2009.0060">10.1089/jmf.2009.0060</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20136437">20136437</a> </span> </span> </li> <li tag="ref"> <a name="CR15" /> <span class="label" tagx="label" title="label">15.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fateme</span> <span class="given-names" tagx="given-names" title="given-names">A-M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mohammad</span> <span class="given-names" tagx="given-names" title="given-names">HF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Abdolhossein</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ali</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maryam</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Volatile Constituents of <i>Salvia compressa</i> and Logochilus macranthus, two Labiatae Herbs Growing wild in Iran. Res </span> <span class="source" tagx="source" title="source">J.Recent Sci.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="fpage" tagx="fpage" title="fpage">66</span> <span class="lpage" tagx="lpage" title="lpage">68</span> </span> </li> <li tag="ref"> <a name="CR16" /> <span class="label" tagx="label" title="label">16.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aziz</span> <span class="given-names" tagx="given-names" title="given-names">RA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hamed</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Abdulah</span> <span class="given-names" tagx="given-names" title="given-names">NA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Determination of the Main Components of the Essential Oil Extracted From <i>Salvia fruticosa</i> by sing GC and GC-MS DAMASCUS </span> <span class="source" tagx="source" title="source">J. AGR. SCI.</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">24</span> <span class="fpage" tagx="fpage" title="fpage">223</span> <span class="lpage" tagx="lpage" title="lpage">236</span> </span> </li> <li tag="ref"> <a name="CR17" /> <span class="label" tagx="label" title="label">17.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Daniel</span> <span class="given-names" tagx="given-names" title="given-names">JS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Localization of Salvinorin A and Related Compounds in Glandular Trichomes of the Psychoactive Sage </span> <span class="source" tagx="source" title="source">Salvia divinorum. Annals of Botany.</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">93</span> <span class="fpage" tagx="fpage" title="fpage">763</span> <span class="lpage" tagx="lpage" title="lpage">771</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/aob/mch089">10.1093/aob/mch089</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15087301">15087301</a> </span> </span> </li> <li tag="ref"> <a name="CR18" /> <span class="label" tagx="label" title="label">18.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rafidah</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Volatile Profiling of Aromatic Traditional Medicinal Plant, <i>Polygonum minus</i> in Different Tissues and Its Biological Activities </span> <span class="source" tagx="source" title="source">Molecules.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">19220</span> <span class="lpage" tagx="lpage" title="lpage">19242</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3390/molecules191119220">10.3390/molecules191119220</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25420073">25420073</a> </span> </span> </li> <li tag="ref"> <a name="CR19" /> <span class="label" tagx="label" title="label">19.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title"> <i>De novo</i> assembly and characterization of root transcriptome using Illumina paired-end sequencing and development of cSSR markers in sweetpotato ( <i>Ipomoea batatas</i>) </span> <span class="source" tagx="source" title="source">BMC Genomics.</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">726</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-11-726">10.1186/1471-2164-11-726</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21182800">21182800</a> </span> </span> </li> <li tag="ref"> <a name="CR20" /> <span class="label" tagx="label" title="label">20.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">HA</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">High-Throughput Sequencing and De Novo assembly of <i>Brassica oleracea</i> var. Capitata L. for transcriptome Analysis </span> <span class="source" tagx="source" title="source">PLoS One.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">e92087</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0092087">10.1371/journal.pone.0092087</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24682075">24682075</a> </span> </span> </li> <li tag="ref"> <a name="CR21" /> <span class="label" tagx="label" title="label">21.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liang</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yiu</span> <span class="given-names" tagx="given-names" title="given-names">S-M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lim</span> <span class="given-names" tagx="given-names" title="given-names">BL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title"> <i>De novo</i> assembly and characterization of <i>Camelina sativa</i> transcriptome by paired-end sequencing </span> <span class="source" tagx="source" title="source">BMC Genomics.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">146</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-14-146">10.1186/1471-2164-14-146</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23496985">23496985</a> </span> </span> </li> <li tag="ref"> <a name="CR22" /> <span class="label" tagx="label" title="label">22.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grabherr</span> <span class="given-names" tagx="given-names" title="given-names">MG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haas</span> <span class="given-names" tagx="given-names" title="given-names">BJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yassour</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Levin</span> <span class="given-names" tagx="given-names" title="given-names">JZ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thompson</span> <span class="given-names" tagx="given-names" title="given-names">DA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome</span> <span class="source" tagx="source" title="source">Nat Biotechnol.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">29</span> <span class="fpage" tagx="fpage" title="fpage">644</span> <span class="lpage" tagx="lpage" title="lpage">652</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nbt.1883">10.1038/nbt.1883</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21572440">21572440</a> </span> </span> </li> <li tag="ref"> <a name="CR23" /> <span class="label" tagx="label" title="label">23.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Annadurai</span> <span class="given-names" tagx="given-names" title="given-names">RS</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title"> <i>De Novo</i> transcriptome assembly (NGS) of Curcuma longa L. rhizome reveals novel transcripts related to anticancer and antimalarial terpenoids </span> <span class="source" tagx="source" title="source">PLoS One.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">e56217</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0056217">10.1371/journal.pone.0056217</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23468859">23468859</a> </span> </span> </li> <li tag="ref"> <a name="CR24" /> <span class="label" tagx="label" title="label">24.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">An</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Transcriptome profiling to discover putative genes associated with paraquat resistance in goosegrass ( <i>Eleusine indica L</i>.) </span> <span class="source" tagx="source" title="source">PLoS One.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">e99940</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0099940">10.1371/journal.pone.0099940</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24927422">24927422</a> </span> </span> </li> <li tag="ref"> <a name="CR25" /> <span class="label" tagx="label" title="label">25.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yan</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title"> <i>De novo</i> transcriptome sequencing of radish ( <i>Raphanus sativus</i> L.) and analysis of major genes involved in glucosinolate metabolism </span> <span class="source" tagx="source" title="source">BMC Genomics.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">836</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-14-26">10.1186/1471-2164-14-26</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24279309">24279309</a> </span> </span> </li> <li tag="ref"> <a name="CR26" /> <span class="label" tagx="label" title="label">26.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">LL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tong</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hu</span> <span class="given-names" tagx="given-names" title="given-names">SQ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The first Illumina-based de novo transcriptome sequencing and analysis of safflower flowers </span> <span class="source" tagx="source" title="source">PLoS One.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">e38653</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/annotation/e4e8ca0c-f6e8-4b32-aae1-b5f8e0c7ebc3">10.1371/annotation/e4e8ca0c-f6e8-4b32-aae1-b5f8e0c7ebc3</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22723874">22723874</a> </span> </span> </li> <li tag="ref"> <a name="CR27" /> <span class="label" tagx="label" title="label">27.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gahlan</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title"> <i>De novo</i> sequencing and characterization of <i>Picrorhiza kurrooa</i> transcriptome at two temperatures showed major transcriptome adjustments </span> <span class="source" tagx="source" title="source">BMC Genomics.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="fpage" tagx="fpage" title="fpage">126</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-13-126">10.1186/1471-2164-13-126</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22462805">22462805</a> </span> </span> </li> <li tag="ref"> <a name="CR28" /> <span class="label" tagx="label" title="label">28.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Transcriptome analysis of medicinal plant <i>Salvia miltiorrhiza</i> and identification of genes related to tanshinone biosynthesis </span> <span class="source" tagx="source" title="source">PLoS One.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">e80464</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0080464">10.1371/journal.pone.0080464</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24260395">24260395</a> </span> </span> </li> <li tag="ref"> <a name="CR29" /> <span class="label" tagx="label" title="label">29.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xie</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">De novo sequencing and a comprehensive analysis of purple sweet potato ( <i>Impomoea batatas</i> L.) transcriptome </span> <span class="source" tagx="source" title="source">Planta.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">236</span> <span class="fpage" tagx="fpage" title="fpage">101</span> <span class="lpage" tagx="lpage" title="lpage">113</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00425-012-1591-4">10.1007/s00425-012-1591-4</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22270559">22270559</a> </span> </span> </li> <li tag="ref"> <a name="CR30" /> <span class="label" tagx="label" title="label">30.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kanehisa</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goto</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">KEGG: Kyoto encyclopedia of genes and genomes</span> <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">28</span> <span class="fpage" tagx="fpage" title="fpage">27</span> <span class="lpage" tagx="lpage" title="lpage">30</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/28.1.27">10.1093/nar/28.1.27</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10592173">10592173</a> </span> </span> </li> <li tag="ref"> <a name="CR31" /> <span class="label" tagx="label" title="label">31.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Virginie</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Crystal structure of isopentenyl diphosphate:dimethylallyl diphosphate isomerase</span> <span class="source" tagx="source" title="source">The EMBO Journal.</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="issue" tagx="issue" title="issue">20(7)</span> <span class="fpage" tagx="fpage" title="fpage">1530</span> <span class="lpage" tagx="lpage" title="lpage">1537</span> </span> </li> <li tag="ref"> <a name="CR32" /> <span class="label" tagx="label" title="label">32.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dorothea</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Terpene synthases and the regulation, diversity and biological roles of terpene metabolism</span> <span class="source" tagx="source" title="source">Current Opinion in Plant Biology.</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">297</span> <span class="lpage" tagx="lpage" title="lpage">304</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.pbi.2006.03.014">10.1016/j.pbi.2006.03.014</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16600670">16600670</a> </span> </span> </li> <li tag="ref"> <a name="CR33" /> <span class="label" tagx="label" title="label">33.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Douglas</span> <span class="given-names" tagx="given-names" title="given-names">JMG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rodney</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Terpenoid Metabolism</span> <span class="source" tagx="source" title="source">The Plant Cell.</span> <span class="year" tagx="year" title="year">1995</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">1015</span> <span class="lpage" tagx="lpage" title="lpage">1026</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.7.7.1015">10.1105/tpc.7.7.1015</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7640522">7640522</a> </span> </span> </li> <li tag="ref"> <a name="CR34" /> <span class="label" tagx="label" title="label">34.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nagegowda</span> <span class="given-names" tagx="given-names" title="given-names">DA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Plant volatile terpenoid metabolism: Biosynthetic genes, transcriptional regulation and subcellular compartmentation </span> <span class="source" tagx="source" title="source">FEBS Lett.</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">584</span> <span class="fpage" tagx="fpage" title="fpage">2965</span> <span class="lpage" tagx="lpage" title="lpage">2973</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.febslet.2010.05.045">10.1016/j.febslet.2010.05.045</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20553718">20553718</a> </span> </span> </li> <li tag="ref"> <a name="CR35" /> <span class="label" tagx="label" title="label">35.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Misra, R. C., Maiti, P., Chanotiya, C. S., Shanker, K. &amp;amp; Ghosh, S. Methyl jasmonate-elicited transcriptional responses and pentacyclic triterpenoid biosynthesis in sweet basil. <i>Plant Physiol</i>. https://doi.org/10.1104/pp. 113.232884 (2014). </span> </li> <li tag="ref"> <a name="CR36" /> <span class="label" tagx="label" title="label">36.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Molecular characterization of the pentacyclic triterpenoid biosynthetic pathway in <i>Catharanthus roseus</i> </span> <span class="source" tagx="source" title="source">Planta.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">236</span> <span class="fpage" tagx="fpage" title="fpage">1571</span> <span class="lpage" tagx="lpage" title="lpage">1581</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00425-012-1712-0">10.1007/s00425-012-1712-0</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22837051">22837051</a> </span> </span> </li> <li tag="ref"> <a name="CR37" /> <span class="label" tagx="label" title="label">37.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Verma</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shah</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bhatia</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Development of an expressed gene catalogue and molecular markers from the de novo assembly of short sequence reads of the lentil ( <i>Lens culinarisMedik</i>.) transcriptome </span> <span class="source" tagx="source" title="source">Plant Biotechnol J.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">894</span> <span class="lpage" tagx="lpage" title="lpage">905</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/pbi.12082">10.1111/pbi.12082</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23759076">23759076</a> </span> </span> </li> <li tag="ref"> <a name="CR38" /> <span class="label" tagx="label" title="label">38.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sabine</span> <span class="given-names" tagx="given-names" title="given-names">G-G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Corinna</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ralf</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johannes</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Seasonal influence on gene expression of monoterpene synthases in <i>Salvia officinalis</i> (Lamiaceae) </span> <span class="source" tagx="source" title="source">J Plant Physiolol.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">169</span> <span class="fpage" tagx="fpage" title="fpage">353</span> <span class="lpage" tagx="lpage" title="lpage">359</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.jplph.2011.11.004">10.1016/j.jplph.2011.11.004</a> </span> </span> </li> <li tag="ref"> <a name="CR39" /> <span class="label" tagx="label" title="label">39.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Croteau</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Felton</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Karp</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kjonaas</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Relationship of camphor biosynthesis to leaf development in sage <i>Salvia officinalis</i> </span> <span class="source" tagx="source" title="source">Plant Physiol.</span> <span class="year" tagx="year" title="year">1981</span> <span class="volume" tagx="volume" title="volume">67</span> <span class="fpage" tagx="fpage" title="fpage">820</span> <span class="lpage" tagx="lpage" title="lpage">824</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.67.4.820">10.1104/pp.67.4.820</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16661761">16661761</a> </span> </span> </li> <li tag="ref"> <a name="CR40" /> <span class="label" tagx="label" title="label">40.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dudareva</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cseke</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blanc</span> <span class="given-names" tagx="given-names" title="given-names">VM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pichersky</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Evolution of floral scent in Clarkia: novel patterns of S-linalool synthase gene expression in the <i>C</i>. <i>breweri</i> flower </span> <span class="source" tagx="source" title="source">Plant Cell.</span> <span class="year" tagx="year" title="year">1996</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">1137</span> <span class="lpage" tagx="lpage" title="lpage">1148</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.8.7.1137">10.1105/tpc.8.7.1137</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8768373">8768373</a> </span> </span> </li> <li tag="ref"> <a name="CR41" /> <span class="label" tagx="label" title="label">41.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McConkey</span> <span class="given-names" tagx="given-names" title="given-names">ME</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gershenzon</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Croteau</span> <span class="given-names" tagx="given-names" title="given-names">RB</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Developmental regulation of monoterpene biosynthesis in the glandular trichomes of peppermint </span> <span class="source" tagx="source" title="source">Plant Physiol.</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">122</span> <span class="fpage" tagx="fpage" title="fpage">215</span> <span class="lpage" tagx="lpage" title="lpage">224</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.122.1.215">10.1104/pp.122.1.215</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10631265">10631265</a> </span> </span> </li> <li tag="ref"> <a name="CR42" /> <span class="label" tagx="label" title="label">42.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mahmoud</span> <span class="given-names" tagx="given-names" title="given-names">SS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Croteau</span> <span class="given-names" tagx="given-names" title="given-names">RB</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Menthofuran regulates essential oil biosynthesis in peppermint by controlling a downstream monoterpene reductase </span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci.</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">100</span> <span class="fpage" tagx="fpage" title="fpage">14481</span> <span class="lpage" tagx="lpage" title="lpage">14486</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.2436325100">10.1073/pnas.2436325100</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14623962">14623962</a> </span> </span> </li> <li tag="ref"> <a name="CR43" /> <span class="label" tagx="label" title="label">43.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mahmoud</span> <span class="given-names" tagx="given-names" title="given-names">SS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Williams</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Corteau</span> <span class="given-names" tagx="given-names" title="given-names">RB</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Cosuppression of limonene-3-hydroxylase in peppermint promotes accumulation of limonene in the essential oil </span> <span class="source" tagx="source" title="source">Phytochemistry.</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">65</span> <span class="fpage" tagx="fpage" title="fpage">547</span> <span class="lpage" tagx="lpage" title="lpage">554</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.phytochem.2004.01.005">10.1016/j.phytochem.2004.01.005</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15003417">15003417</a> </span> </span> </li> <li tag="ref"> <a name="CR44" /> <span class="label" tagx="label" title="label">44.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xie</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kapteyn</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gang</span> <span class="given-names" tagx="given-names" title="given-names">DR</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A systems biology investigation of the MEP/terpenoid and shikimate/phenylpropanoid pathways points to multiple levels of metabolic control in sweet basil glandular trichomes </span> <span class="source" tagx="source" title="source">Plant J.</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">54</span> <span class="fpage" tagx="fpage" title="fpage">349</span> <span class="lpage" tagx="lpage" title="lpage">361</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2008.03429.x">10.1111/j.1365-313X.2008.03429.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18248593">18248593</a> </span> </span> </li> <li tag="ref"> <a name="CR45" /> <span class="label" tagx="label" title="label">45.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lane</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boecklemann</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Woronuk</span> <span class="given-names" tagx="given-names" title="given-names">GN</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sarker</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mahmoud</span> <span class="given-names" tagx="given-names" title="given-names">SS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A genomics resource for investigating regulation of essential oil production in <i>Lavandula angustifolia</i> </span> <span class="source" tagx="source" title="source">Planta.</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">231</span> <span class="fpage" tagx="fpage" title="fpage">835</span> <span class="lpage" tagx="lpage" title="lpage">845</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00425-009-1090-4">10.1007/s00425-009-1090-4</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20043174">20043174</a> </span> </span> </li> <li tag="ref"> <a name="CR46" /> <span class="label" tagx="label" title="label">46.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmiderer</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grausgruber-Gröger</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grassi</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Steinborn</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Novak</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Influence of gibberellin and daminozide on the expression of terpene synthases in common sage ( <i>Salvia officinalis</i>) </span> <span class="source" tagx="source" title="source">J Plant Physiol.</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">167</span> <span class="fpage" tagx="fpage" title="fpage">779</span> <span class="lpage" tagx="lpage" title="lpage">786</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.jplph.2009.12.009">10.1016/j.jplph.2009.12.009</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20163890">20163890</a> </span> </span> </li> <li tag="ref"> <a name="CR47" /> <span class="label" tagx="label" title="label">47.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kampranis</span> <span class="given-names" tagx="given-names" title="given-names">SC</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Rational conversion of substrate and product specificity in a Salvia monoterpene synthase: structural insights into the evolution of terpene synthase function </span> <span class="source" tagx="source" title="source">Plant Cell.</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">1994</span> <span class="lpage" tagx="lpage" title="lpage">2005</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.106.047779">10.1105/tpc.106.047779</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17557809">17557809</a> </span> </span> </li> <li tag="ref"> <a name="CR48" /> <span class="label" tagx="label" title="label">48.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xianzhong</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hongjian</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lifen</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Donghong</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xingqian</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Extraction of essential oil from discarded tobacco leaves by solvent extraction and steam distillation, and identification of its chemical composition </span> <span class="source" tagx="source" title="source">Ind.Crops Prod.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">39</span> <span class="fpage" tagx="fpage" title="fpage">162</span> <span class="lpage" tagx="lpage" title="lpage">169</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.indcrop.2012.02.029">10.1016/j.indcrop.2012.02.029</a> </span> </span> </li> <li tag="ref"> <a name="CR49" /> <span class="label" tagx="label" title="label">49.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fumin</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liangquan</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baizhan</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hongwu</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shaomin</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Comparison of different extraction methods: steam distillation, simultaneous distillation and extraction and headspace co-distillation, used for the analysis of the volatile components in aged flue-cured tobacco leaves </span> <span class="source" tagx="source" title="source">J. Chromatogr. A.</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">1040</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">17</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.chroma.2004.03.057">10.1016/j.chroma.2004.03.057</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15248421">15248421</a> </span> </span> </li> <li tag="ref"> <a name="CR50" /> <span class="label" tagx="label" title="label">50.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Natalia</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">( <i>E</i>)-β-Ocimene and Myrcene Synthase Genes of Floral Scent Biosynthesis in Snapdragon </span> <span class="source" tagx="source" title="source">The Plant Cell.</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">15</span> <span class="fpage" tagx="fpage" title="fpage">1227</span> <span class="lpage" tagx="lpage" title="lpage">1241</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.011015">10.1105/tpc.011015</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12724546">12724546</a> </span> </span> </li> <li tag="ref"> <a name="CR51" /> <span class="label" tagx="label" title="label">51.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Diane</span> <span class="given-names" tagx="given-names" title="given-names">M M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Functional annotation, genome organization and phylogeny of the grapevine ( <i>Vitis vinifera</i>) terpene synthase gene family based on genome assembly, FLcDNA cloning, and enzyme assays </span> <span class="source" tagx="source" title="source">BMC Plant Biol.</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">226</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2229-10-226">10.1186/1471-2229-10-226</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20964856">20964856</a> </span> </span> </li> <li tag="ref"> <a name="CR52" /> <span class="label" tagx="label" title="label">52.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rodney</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mark</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Robert</span> <span class="given-names" tagx="given-names" title="given-names">CR</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Biosynthesis of monoterpenes –conversion of the acyclic precursor’s geranyl pyrophosphate and nerylpyrophosphate to the rearranged monoterpenes fenchol and fenchone by a soluble enzyme preparation from fennel ( <i>Foeniculum vulgare</i>) </span> <span class="source" tagx="source" title="source">Arch. Biochem. Biophys.</span> <span class="year" tagx="year" title="year">1980</span> <span class="volume" tagx="volume" title="volume">200</span> <span class="fpage" tagx="fpage" title="fpage">524</span> <span class="lpage" tagx="lpage" title="lpage">533</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/0003-9861(80)90384-7">10.1016/0003-9861(80)90384-7</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7436420">7436420</a> </span> </span> </li> <li tag="ref"> <a name="CR53" /> <span class="label" tagx="label" title="label">53.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rodney</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Biosynthesis and catabolism of monoterpenoids</span> <span class="source" tagx="source" title="source">Chem. Rev.</span> <span class="year" tagx="year" title="year">1987</span> <span class="volume" tagx="volume" title="volume">87</span> <span class="fpage" tagx="fpage" title="fpage">929</span> <span class="lpage" tagx="lpage" title="lpage">954</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/cr00081a004">10.1021/cr00081a004</a> </span> </span> </li> <li tag="ref"> <a name="CR54" /> <span class="label" tagx="label" title="label">54.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wise</span> <span class="given-names" tagx="given-names" title="given-names">ML</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rodney</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Comprehensive Natural Products Chemistry, Isoprenoids Including Caroteinoids and Steroids</span> <span class="source" tagx="source" title="source">Elsevier, Amsterdam.</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="fpage" tagx="fpage" title="fpage">97</span> <span class="lpage" tagx="lpage" title="lpage">135</span> </span> </li> <li tag="ref"> <a name="CR55" /> <span class="label" tagx="label" title="label">55.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lücker</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Monoterpene biosynthesis in lemon (Citrus Limon) – cDNA isolation and functional analysis of four monoterpene synthases </span> <span class="source" tagx="source" title="source">Eur. J. Biochem.</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">269</span> <span class="fpage" tagx="fpage" title="fpage">3160</span> <span class="lpage" tagx="lpage" title="lpage">3171</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1046/j.1432-1033.2002.02985.x">10.1046/j.1432-1033.2002.02985.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12084056">12084056</a> </span> </span> </li> <li tag="ref"> <a name="CR56" /> <span class="label" tagx="label" title="label">56.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takehiko</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Molecular cloning and functional characterization of four monoterpene synthase genes from Citrus unshiu Marc </span> <span class="source" tagx="source" title="source">Plant Sci.</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">166</span> <span class="fpage" tagx="fpage" title="fpage">49</span> <span class="lpage" tagx="lpage" title="lpage">58</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.plantsci.2003.07.006">10.1016/j.plantsci.2003.07.006</a> </span> </span> </li> <li tag="ref"> <a name="CR57" /> <span class="label" tagx="label" title="label">57.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huber</span> <span class="given-names" tagx="given-names" title="given-names">DP</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Characterization of four terpene synthase cDNAs from methyl jasmonateinduced Douglas-fir, Pseudotsuga menziesii </span> <span class="source" tagx="source" title="source">Phytochemistry.</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">66</span> <span class="fpage" tagx="fpage" title="fpage">1427</span> <span class="lpage" tagx="lpage" title="lpage">1439</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.phytochem.2005.04.030">10.1016/j.phytochem.2005.04.030</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15921711">15921711</a> </span> </span> </li> <li tag="ref"> <a name="CR58" /> <span class="label" tagx="label" title="label">58.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martin</span> <span class="given-names" tagx="given-names" title="given-names">DM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bohlmann</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Identification of Vitis vinifera (−)-alpha-terpineol synthase by in silico screening of full-length cDNA ESTs and functional characterization of recombinant terpene synthase </span> <span class="source" tagx="source" title="source">Phytochemistry.</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">65</span> <span class="fpage" tagx="fpage" title="fpage">1223</span> <span class="lpage" tagx="lpage" title="lpage">1229</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.phytochem.2004.03.018">10.1016/j.phytochem.2004.03.018</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15184006">15184006</a> </span> </span> </li> <li tag="ref"> <a name="CR59" /> <span class="label" tagx="label" title="label">59.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">David</span> <span class="given-names" tagx="given-names" title="given-names">EC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stephen</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pushpalatha</span> <span class="given-names" tagx="given-names" title="given-names">PNM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Trichodiene biosynthesis and the enzymatic cyclization of farnesyl pyrophosphate</span> <span class="source" tagx="source" title="source">J. Am. Chem. Soc.</span> <span class="year" tagx="year" title="year">1981</span> <span class="volume" tagx="volume" title="volume">103</span> <span class="fpage" tagx="fpage" title="fpage">2136</span> <span class="lpage" tagx="lpage" title="lpage">2138</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/ja00414a053">10.1021/ja00414a053</a> </span> </span> </li> <li tag="ref"> <a name="CR60" /> <span class="label" tagx="label" title="label">60.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">David</span> <span class="given-names" tagx="given-names" title="given-names">EC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guohan</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Trichodiene synthase – stereochemical studies of the cryptic allylic diphosphate isomerase activity using an anomalous substrate </span> <span class="source" tagx="source" title="source">J. Org. Chem.</span> <span class="year" tagx="year" title="year">1994</span> <span class="volume" tagx="volume" title="volume">59</span> <span class="fpage" tagx="fpage" title="fpage">5794</span> <span class="lpage" tagx="lpage" title="lpage">5798</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/jo00098a046">10.1021/jo00098a046</a> </span> </span> </li> <li tag="ref"> <a name="CR61" /> <span class="label" tagx="label" title="label">61.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">David</span> <span class="given-names" tagx="given-names" title="given-names">EC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Manish</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Epicubenol synthase and the stereochemistry of the enzymatic cyclization of farnesyl and nerolidyl diphosphate </span> <span class="source" tagx="source" title="source">J. Am. Chem. Soc.</span> <span class="year" tagx="year" title="year">1995</span> <span class="volume" tagx="volume" title="volume">117</span> <span class="fpage" tagx="fpage" title="fpage">5602</span> <span class="lpage" tagx="lpage" title="lpage">5603</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/ja00125a029">10.1021/ja00125a029</a> </span> </span> </li> <li tag="ref"> <a name="CR62" /> <span class="label" tagx="label" title="label">62.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Iris</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The enzymatic cyclization of nerolidyl diphosphate by delta cadinene synthase from cotton stele tissue infected with Verticillium dahlia </span> <span class="source" tagx="source" title="source">Phytochemistry.</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">47</span> <span class="fpage" tagx="fpage" title="fpage">961</span> <span class="lpage" tagx="lpage" title="lpage">967</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0031-9422(98)80054-X">10.1016/S0031-9422(98)80054-X</a> </span> </span> </li> <li tag="ref"> <a name="CR63" /> <span class="label" tagx="label" title="label">63.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Steele</span> <span class="given-names" tagx="given-names" title="given-names">CL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Crock</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bohlmann</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Croteau</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Sesquiterpene synthases from grand fir ( <i>Abies grandis</i>) – Comparison of constitutive and wound-induced activities, and cDNA isolation, characterization and bacterial expression of delta-selinene synthase and gamma-humulene synthase </span> <span class="source" tagx="source" title="source">J. Biol. Chem.</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">273</span> <span class="fpage" tagx="fpage" title="fpage">2078</span> <span class="lpage" tagx="lpage" title="lpage">2089</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1074/jbc.273.4.2078">10.1074/jbc.273.4.2078</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9442047">9442047</a> </span> </span> </li> <li tag="ref"> <a name="CR64" /> <span class="label" tagx="label" title="label">64.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wise</span> <span class="given-names" tagx="given-names" title="given-names">ML</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Savage</span> <span class="given-names" tagx="given-names" title="given-names">TJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Katahira</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Croteau</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Monoterpene synthases from common sage ( <i>Salvia officinalis</i>) – cDNA isolation, characterization, and functional expression of (+)-sabinene synthase, 1, 8-cineole synthase, and (+)- bornyl diphosphate synthase </span> <span class="source" tagx="source" title="source">J. Biol. Chem.</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">273</span> <span class="fpage" tagx="fpage" title="fpage">14891</span> <span class="lpage" tagx="lpage" title="lpage">14899</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1074/jbc.273.24.14891">10.1074/jbc.273.24.14891</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9614092">9614092</a> </span> </span> </li> <li tag="ref"> <a name="CR65" /> <span class="label" tagx="label" title="label">65.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Anders, S., Huber, W. Differential expression analysis for sequence count data. <i>Genome Biology</i>, 10.1186/gb-2010-11-10-r106 (2010). </span> </li> <li tag="ref"> <a name="CR66" /> <span class="label" tagx="label" title="label">66.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Metabolic engineering of proantho-cyanidin production by repressing the isoflavone pathways and redirecting anthocyanidin precursor flux in legume </span> <span class="source" tagx="source" title="source">Plant Biotechnol J.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="issue" tagx="issue" title="issue">7</span> <span class="fpage" tagx="fpage" title="fpage">1604</span> <span class="lpage" tagx="lpage" title="lpage">18</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/pbi.12524">10.1111/pbi.12524</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26806316">26806316</a> </span> </span> </li> <li tag="ref"> <a name="CR67" /> <span class="label" tagx="label" title="label">67.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Livak</span> <span class="given-names" tagx="given-names" title="given-names">KJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmittgen</span> <span class="given-names" tagx="given-names" title="given-names">TD</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Analysis of relative gene expression data using real-time quantitative PCR and the 2 <sup>−ΔΔCT</sup> method </span> <span class="source" tagx="source" title="source">Methods.</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">402</span> <span class="lpage" tagx="lpage" title="lpage">408</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1006/meth.2001.1262">10.1006/meth.2001.1262</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11846609">11846609</a> </span> </span> </li> <li tag="ref"> <a name="CR68" /> <span class="label" tagx="label" title="label">68.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hongmei</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Transcriptional data mining of <i>Salvia miltiorrhiza</i> in response to methyl jasmonate to examine the mechanism of bioactive compound biosynthesis and regulation </span> <span class="source" tagx="source" title="source">Physiologia Plantarum.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">152</span> <span class="fpage" tagx="fpage" title="fpage">241</span> <span class="lpage" tagx="lpage" title="lpage">255</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/ppl.12193">10.1111/ppl.12193</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24660670">24660670</a> </span> </span> </li> <li tag="ref"> <a name="CR69" /> <span class="label" tagx="label" title="label">69.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Sunjung, P. <i>Agrobacterium tumefaciens</i> –mediated transformation of tobacco ( <i>Nicotiana tabacum</i> L.) leaf disks: evaluation of the co-cultivation conditions to increase β -Glucuronidase gene activity. (Master’s dissertation). Retrieved from <a href="http://etd.lsu.edu/docs/available/etd-07052006-173930/unrestricted/Park_thesis.Pdf">http://etd.lsu.edu/docs/available/etd-07052006-173930/unrestricted/Park_thesis.Pdf</a> (2006). </span> </li> </ul> </div> </div>  </body></html>