<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Bioinformation</span> <span class="iso-abbrev" title="iso-abbrev">Bioinformation</span> <span class="publisher-id" title="publisher-id">Bioinformation</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Bioinformation</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">0973-2063</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Biomedical Informatics</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/3134781">3134781</a> </span> <span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/21769194">21769194</a> </span> <span class="publisher-id" title="publisher-id">97320630006315</span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Hypothesis</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Structure prediction and functional characterization of secondary metabolite proteins of <i>Ocimum</i> </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Roy</span> <span class="given-names" tagx="given-names" title="given-names">Sudeep</span> </span> <a href="#COR1">*</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maheshwari</span> <span class="given-names" tagx="given-names" title="given-names">Nidhi</span> </span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chauhan</span> <span class="given-names" tagx="given-names" title="given-names">Rashi</span> </span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sen</span> <span class="given-names" tagx="given-names" title="given-names">Naresh Kumar</span> </span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharma</span> <span class="given-names" tagx="given-names" title="given-names">Ashok</span> </span> </span> <span class="citation_author_institution" id="">[], </span> </div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"> <span class="label" tagx="label" title="label">*</span>Sudeep Roy: <span class="email" tagx="email" title="email">roysudeep28@gmail.com</span>Phone: +919450614880 </div> </div> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2011</span> </span> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2011-7-7</span> </span> <span class="volume" tagx="volume" title="volume">6</span> <span class="issue" tagx="issue" title="issue">8</span> <span class="fpage" tagx="fpage" title="fpage">315</span> <span class="lpage" tagx="lpage" title="lpage">319</span> <span class="history" title="history"> <span class="received" title="received">received: 2011-6-18</span> <span class="accepted" title="accepted">accepted: 2011-6-27</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2011</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open-access article, which permits unrestricted use, distribution, and reproduction in any medium,for non-commercial purposes, provided the original author and source are credited. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Various species of <i>Ocimum</i> have acquired special attention due to their medicinal properties. Different parts of the plant (root, stem, flower, leaves) are used in the treatment of a wide range of disorders from centuries. Experimental structures (X-ray and NMR) of proteins from different <i>Ocimum</i> species, are not yet available in the Protein Databank (PDB). These proteins play a key role in various metabolic pathways in <i>Ocimum</i>. 3D structures of the proteins are essential to determine most of their functions. Homology modeling approach was employed in order to derive structures for these proteins. A program meant for comparative modeling- Modeller 9v7 was utilized for the purpose. The modeled proteins were further validated by Prochek and Verify-3d and Errat servers. Amino acid composition and polarity of these proteins was determined by CLC-Protein Workbench tool. Expasy's Prot-param server and Cys_rec tool were used for physico-chemical and functional characterization of these proteins. Studies of secondary structure of these proteins were carried out by computational program, Profunc. Swiss-pdb viewer was used to visualize and analyze these homology derived structures. The structures are finally submitted in Protein Model Database, PMDB so that they become accessible to other users for further studies. </p> </div> <div class="kwd-group"> <div class="kwd" title="kwd"> <i>Ocimum</i> </div> <div class="kwd" title="kwd">
Homology modeling</div> <div class="kwd" title="kwd">
CLC protein work bench</div> <div class="kwd" title="kwd">
Secondary structure prediction</div> <div class="kwd" title="kwd">
Swiss-PDB Viewer</div> </div> </div> </div> <div class="body" title="body"> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>Botanically, basil belongs to the genus <i>Ocimum</i> of the family Lamiaceae. More than 160 species of <i>Ocimum</i> are reported from different parts of the world. Different parts (roots, stem, leaves, seeds and flowers) of <i>Ocimum</i> have been used for treatment of variety of diseases such as bronchitis, malaria, diarrhea, dysentery, skin diseases, arthritis etc. <i>Ocimum</i> sp. contains monoterpene derivatives such as camphor, limonene, thymol, citral, geraniol and linalool. A detailed analysis of protein sequences from <i>Ocimum</i>, their probable structures and mode of action are yet to be accomplished. Plants synthesize chemicals in their leaves in order to protect themselves from herbivores. One such class of defense compounds that has been used extensively by humans are members of phenylpropenoid class namely eugenol, chavicol and their derivatives. It has been reported that in basil glands, two closely related (90% identical) enzymes chavicol o-methyltransferase (CVOMT) and eugenol o-methyltransferase (EOMT) catalyze the formation of methylchavicol and methyleugenol from chavicol and eugenol respectively [ <a href="#R01">1</a>]. The enzymes are involved in aroma production in basil. From an evolutionary perspective plant and microbial PALs (phenylalanine ammonia lyase) are part of superfamily of enzymes from plants, fungi and bacteria, and are likely derived from a precursor of the widespread histidine ammonia lyase (HAL) family in the histidine degradation pathway [ <a href="#R02">2</a>]. PAL catalyses the non-oxidative deamination of phenylalanine to trans-cinnamate and directs the carbon flow from the shikimate pathway to the various branches of the general phenylpropanoid metabolism. Lipooxygenase (Fatty-acid metabolism) is one of the most widely studied enzymes found in more than 60 species of plant and animal kingdom. The enzyme catalyses the biooxygenation of polyunsaturated fatty acids (PUFA) containing a cis, cis-1, 4-pentadiene unit to form conjugated hydroperoxydienoic acids. Lipoxygenase has considerable application in food related products such as in bread making. The enzyme plays a significant role in formation of secondary metabolites in sweet basil. </p> <p>Enzymatic browning of fruits and vegetables is caused mainly by the conversion of native phenolic compounds to quinones which are then polymerized to brown, red or black pigments imparting colour to various plant parts. The enzymes responsible for catalyzing this sequence of reactions are termed as polyphenol oxidases, but are also known as tyrosinases, catecholases, cresolases and phenolases [ <a href="#R03">3</a>]. Because of deleterious effect of enzymatic browning on fruits and vegetables much work is devoted as to retard or at least delay the browning process. Polyphenol oxidase being the causative agent responsible for browning is exploited for the purpose. The enzyme is involved in iso-quinoline alkaloid biosynthesis and in biosynthesis of other secondary metabolites. In order to understand biochemical function and interaction properties of the protein at molecular level, three dimensional structure of protein is foremost requirement. However, the number of available protein sequences exceeds far behind the available three dimensional protein structures. In order to compensate this, homology modeling approach came into being. These methods are believed to be cost-effective and time-effective when compared to X-rays crystallography and NMR techniques. Computational methods make use of hidden information inside amino acid sequences in order to predict protein structure and function. In the present study, <i>In silico</i> analysis and homology modeling studies on uncharacterized proteins in different species of <i>Ocimum</i> like <i>O. basilicum</i>, <i>O. tenuiflorum</i>, <i>O. citriodorum</i>, <i>O. seloi</i>, <i>O. gratissimum</i> and <i>O. americanum</i> whose structures are not yet available in PDB have been accomplished. </p> </div> <div class="materialsandmethodology" title="sec"> <div class="title" tagx="title" title="title">
Materials and Methodology</div> <p>The amino acid sequences of secondary metabolite proteins of <i>Ocimum</i> whose structures are not yet available in RCSB Protein Databank (PDB) were retrieved from SWISSPROT, a public resource of curated protein sequences [ <a href="#R04">4</a>] and subjected to NCBI BLAST [ <a href="#R05">5</a>]. Based on high score, lower e-value and maximum sequence identity, the best template was selected which was then used as reference structure to build a 3D model. Template and target proteins considered for the study have been shown in (see <a href="#SD1">Table 1</a>). </p> <div class="modelbuildingandevaluation" title="sec"> <div class="title" tagx="title" title="title">
Model building and evaluation</div> <p>The three dimensional structures of proteins were modeled using Modeler 9v8 [ <a href="#R06">6</a>]. Quality of generated models was evaluated with PROCHECK [ <a href="#R07">7</a>] by Ramachandran plot analysis [ <a href="#R08">8</a>]. Stereochemical quality and accuracy of the selected models was further improved by subjecting it to energy minimization with the GROMOS 96 43B1 parameters set, implementation of Swiss-PDB Viewer [ <a href="#R09">9</a>]. Validation of generated models was further performed by VERIFY 3D [ <a href="#R10">10</a>] and ERRAT [ <a href="#R11">11</a>] programs. ProSA [ <a href="#R12">12</a>] was used for the analysis of Zscores and energy plots. The three dimensional structures of modeled proteins were analyzed using Deep View Swiss PDB viewer. Root Mean Square Deviation (RMSD) values were calculated between the set of targets and template protein to see how much modeled protein deviates from the template protein structure. </p> </div> <div class="computationofaminoacidcomposition" title="sec"> <div class="title" tagx="title" title="title">
Computation of amino acid composition</div> <p>Amino acid composition (see <a href="#SD1">Table 2</a>) of <i>Ocimum</i> proteins under study was calculated using CLC protein workbench tool ( <a href="www.clcbio.com/protein">www.clcbio.com/protein</a>). The tool also provides estimation of percentage of hydrophobic and hydrophilic residues present in the protein (see <a href="#SD1">Table 3</a>). </p> </div> <div class="physiochemicalcharacterization" title="sec"> <div class="title" tagx="title" title="title">
Physiochemical characterization</div> <p>For physiochemical characterization, theoretical pI (isoelectric point), molecular weight, ­R and +R (total number of positive and negative residues), EI (extinction coefficient) [ <a href="#R13">13</a>], II (instability index [ <a href="#R14">14</a>]) [ <a href="#R15">15</a>], AI (aliphatic index) and GRAVY (grand average hydropathy) [ <a href="#R16">16</a>] were computed using the Expasy's ProtParam server [ <a href="#R17">17</a>] for set of proteins ( <a href="http://us.expasy.org/tools/protparam.html">http://us.expasy.org/tools/protparam.html</a>). The results are shown in (see <a href="#SD1">Table 4</a>) </p> </div> <div class="functionalcharacterization" title="sec"> <div class="title" tagx="title" title="title">
Functional characterization</div> <p>CYS_REC ( <a href="http://sunl.softberry.com/berry.phtml?topic">http://sunl.softberry.com/berry.phtml? topic </a>) was used to locate "SS bond" between the pair of cystein residues, if present. The tool yields position of cysteins, total number of cysteins present and pattern, if present, of pairs in the protein sequence as output. All the <i>Ocimum</i> proteins under study showed absence of disulphide bonds. The results are presented in Table 5 (see <a href="#SD1">Supplementary material</a>). </p> </div> <div class="secondarystructureprediction" title="sec"> <div class="title" tagx="title" title="title">
Secondary structure prediction</div> <p>Profunc [ <a href="#R18">18</a>] was employed for calculating the secondary structural features of <i>Ocimum</i> protein sequences. The results are presented in Table 6(see <a href="#SD1">Supplementary material</a>). </p> </div> <div class="submissionofthemodeledproteinsinproteinmodeldatabase(pmdb)" title="sec"> <div class="title" tagx="title" title="title">
Submission of the modeled proteins in protein model database (PMDB)</div> <p>The models generated for various <i>Ocimum</i> proteins were successfully submitted in Protein model database, PMDB [ <a href="#R19">19</a>] without any stereochemical errors. The submitted models can be accessed via their PMIDs (see Table 7 <a href="#SD1">Supplementary material</a>). </p> </div> </div> <div class="resultsanddiscussion" title="sec"> <div class="title" tagx="title" title="title">
Results and Discussion</div> <p>As experimental structures of some of the important secondary metabolite proteins of <i>Ocimum</i> are not available, homology modeling approach was used in order to derive their structures. </p> <div class="modelbuilding,refinementandevaluation" title="sec"> <div class="title" tagx="title" title="title">
Model building, refinement and evaluation</div> <div class="procheckanalysis" title="sec"> <div class="title" tagx="title" title="title">
PROCHECK analysis</div> <p>Ramachandran plot for Chavicol O-methyltransferase (D3KYA1) has been illustrated in <a href="#F1">Figure 1</a>. Altogether more than 90% of the residues were found to be in favoured and allowed regions, which validate the quality of homology models. The overall G-factor for D3KYA1 was ­0.19. As the value is greater than the acceptable value ­0.50, this suggests that the modeled structure is acceptable. The modeled structures were also validated by other structure verification servers such as Verify 3D and Errat. Verify 3D assigned a 3D-1D score of &amp;gt;0.2 for all the modeled proteins. This implies that the models are compatible with its sequence. ERRAT showed overall quality factor of 49.62 for D3KYA1. The plot generated by Verify-3D and Errat for Chavicol omethyltransferase has been illustrated in <a href="#F2">Figure 2A &amp;amp; 2B</a>. </p> </div> <div class="prosaanalysis" title="sec"> <div class="title" tagx="title" title="title">
PROSA analysis</div> <p>The z-score for all the modeled proteins was found to be within the range of scores typically found for native proteins of similar size showing good quality of the model. Energy Plot for chavicol o-methyltransferase (D3KYA1) with chain length (257 AA) and z-score (­7.26) is presented in <a href="#F3">Figure 3A &amp;amp; 3B</a>. </p> </div> <div class="swiss-pdbvieweranalysisofpredictedmodel" title="sec"> <div class="title" tagx="title" title="title">
Swiss-PDB viewer analysis of predicted model</div> <p>Visualization and analysis of the model using Swiss-PDB reveals that there are no steric hindrances between the residues and thus modeled structures are stable. Structure-structure superimposition was done in order to calculate Root Mean Square Deviation (RMSD) between the target and template sequence. RMSD values for D3KYA1 were found to be 0.94. This implies good quality of the modeled structures. <a href="#F4">Figure 4</a> represents modeled structure of Chavicol o-methyltransferase. </p> </div> <div class="physiochemicalcharacterization" title="sec"> <div class="title" tagx="title" title="title">
Physiochemical characterization</div> <p>The physiochemical parameters viz., theoretical isoelectric point (Ip), molecular weight, total number of positive and negative residues, extinction coefficient, half-life, instability index, aliphatic index and grand average hydropathy (GRAVY) were computed using the Expasy's ProtParam tool ( <a href="#SD1">Table 4</a>). The computed pI value for A8D7D8, B2ZA17, B6VQV5, B6VQV6, D3KYA1 (pI&amp;lt;7) indicated their acidic nature, whereas pI for A8D6D7, B2ZA12, B2ZA16 (pI&amp;gt;7) revealed there basic behaviour. The computed isoelectric point (pI) will be useful for developing buffer system for purification by isoelectric focusing method. Extinction coefficient values for <i>Ocimum</i> proteins at 280 nm ranged from 1490 to 50795 M-1cm-1 for B6VQV6 and D3KYA1 indicating the presence of higher concentration of Tyr and Trp. Cys was very low in concentration in all the eight <i>Ocimum</i> proteins studied. This indicates that these proteins cannot be analyzed using UV spectral methods. On the basis of instability index Expasy's ProtParam classified the B2ZA17 (Eugenol o-methyltransferase), A8D7D8 (Lipoxygenase), B2ZA12 (Eugenol o-methyltransferase) and B2ZA16 (Eugenol o-methyltransferase) proteins as unstable (Instability index&amp;gt;40) and other <i>Ocimum</i> proteins as stable (Instability index&amp;lt;40). The aliphatic index (AI) which is defined as the relative volume of a protein occupied by aliphatic side chain is regarded as the positive factor for the increase of thermal stability of globular proteins. The very high aliphatic index of all <i>Ocimum</i> proteins infers that these proteins may be stable for a wide range of temperature. The very low GRAVY index of proteins B6VQV6 and D3KYA1 infers that these proteins could result in a better interaction with water. </p> </div> <div class="functionalcharacterization" title="sec"> <div class="title" tagx="title" title="title">
Functional characterization</div> <p>The result of primary analysis suggests that all the <i>Ocimum</i> proteins under study were hydrophobic in nature due to the presence of high non-polar residues content ( <a href="#SD1">Table 2 &amp;amp; 3</a>). As percentage of Cysteine(C) is very low in all the <i>Ocimum</i> proteins under study ( <a href="#SD1">Table 2</a>), none of these proteins have disulphide bond linkages, as indicated by CYS_REC result ( <a href="#SD1">Table 4</a>). The extensive hydrogen bonding may provide stability to these proteins in absence of disulphide bonds. Proteins B2ZA12, A8D6D7 and B6VQV5 have high percentage of methionine(M), alanine(A), leucine(L) and lysine(K). As these amino acids have high helix-forming propensities, alpha helix are dominant in these proteins. This is also evident from analysis of PROFUNC result ( <a href="#SD1">Table 6</a>). Rest of the <i>Ocimum</i> proteins had mixed secondary structures i.e. alphahelices, beta-strands and coils. All the proteins showed high percentage of glycine and proline ( <a href="#SD1">Table 2</a>). As these amino acids are common in turns, other secondary structures such as Beta turns and Gamma turns are dominant in these proteins ( <a href="#SD1">Table 6</a>). </p> </div> <div class="submissionofmodeledproteinsinpmdb" title="sec"> <div class="title" tagx="title" title="title">
Submission of modeled proteins in PMDB</div> <p>The modeled structures of proteins from various species of Ocimum were successfully deposited in Protein Model Database (PMDB). The PMDB ID for the submitted structures has been presented in (see <a href="#SD1">Table 7</a>). These 3D structures may be further used in characterizing the protein experimentally. </p> </div> </div> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>In this study proteins from various species of <i>Ocimum</i> were modeled using homology modeling approach. Different parameters such as isoelectric point, molecular weight, total number of positive and negative residues, extinction coefficient, instability index, aliphatic index and grand average hydropathy (GRAVY) were computed for these proteins in order to determine their physiochemical characteristics. All the proteins were found to be deficient in amino acid cystein, and therefore lack presence of disulphide linkages as also inferred from analysis of cys_rec result. In the absence of disulphide bond, extensive hydrogen bonding is believed to be responsible for stability of these proteins. Polarity studies using CLC protein work bench tool confirmed all the studied proteins to be hydrophobic in nature. This may be due to the presence of a large number of non-polar residues. Secondary structure studies showed that all the studied proteins contain high proportion of other secondary structures ie. Beta-turns and Gamma-turns. This is attributed to the presence of higher concentration of proline and glycine residues. The modeled structures can be accessed through protein model database PMDB via there PMID's. Homology derived models are extensively used in wide range of applications such as virtual screening, site-directed mutagenesis experiments or in rationalizing the effects of sequence variation. These structures will serve as cornerstone for functional analysis of experimentally derived crystal structures. </p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary material</div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Data 1</div>  <div class="media" title="media"> <a href="97320630006315S1.pdf">LINK</a> </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>Financial support of Department of Biotechnology (DBT) Govt of India; New Delhi under BTISnet program is gratefully acknowledged. Sudeep Roy is thankful to Council of Scientific and Industrial Research for Senior Research Fellowship. </p> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-other" title="other"> <p> <b>Citation:</b>Roy <i>et al</i>, Bioinformation 6(8): 315-319 (2011) </p> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="R01" /> <span class="label" tagx="label" title="label">1</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">T</span> <span class="given-names" tagx="given-names" title="given-names">Koeduka</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">149</span> <span class="fpage" tagx="fpage" title="fpage">384</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18987218">18987218</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18987218">18987218</a> </span> </span> </li> <li tag="ref"> <a name="R02" /> <span class="label" tagx="label" title="label">2</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">D</span> <span class="given-names" tagx="given-names" title="given-names">Rother</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="source" tagx="source" title="source">Eur J Biochem</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">269</span> <span class="fpage" tagx="fpage" title="fpage">3065</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12071972">12071972</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12071972">12071972</a> </span> </span> </li> <li tag="ref"> <a name="R03" /> <span class="label" tagx="label" title="label">3</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">AM</span> <span class="given-names" tagx="given-names" title="given-names">Mayer</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">E</span> <span class="given-names" tagx="given-names" title="given-names">Harel</span> </span> </span> <span class="source" tagx="source" title="source">Phytochemistry</span> <span class="year" tagx="year" title="year">1979</span> <span class="volume" tagx="volume" title="volume">18</span> <span class="fpage" tagx="fpage" title="fpage">193</span> </span> </li> <li tag="ref"> <a name="R04" /> <span class="label" tagx="label" title="label">4</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">A</span> <span class="given-names" tagx="given-names" title="given-names">Bairoch</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">R</span> <span class="given-names" tagx="given-names" title="given-names">Apweiler</span> </span> </span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">28</span> <span class="fpage" tagx="fpage" title="fpage">45</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10592178">10592178</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10592178">10592178</a> </span> </span> </li> <li tag="ref"> <a name="R05" /> <span class="label" tagx="label" title="label">5</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">SF</span> <span class="given-names" tagx="given-names" title="given-names">Altschul</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="source" tagx="source" title="source">J Mol Biol</span> <span class="year" tagx="year" title="year">1990</span> <span class="volume" tagx="volume" title="volume">215</span> <span class="fpage" tagx="fpage" title="fpage">403</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712">2231712</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712">2231712</a> </span> </span> </li> <li tag="ref"> <a name="R06" /> <span class="label" tagx="label" title="label">6</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">A</span> <span class="given-names" tagx="given-names" title="given-names">Sali</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">TL</span> <span class="given-names" tagx="given-names" title="given-names">Blundell</span> </span> </span> <span class="source" tagx="source" title="source">J Mol Biol</span> <span class="year" tagx="year" title="year">1993</span> <span class="volume" tagx="volume" title="volume">234</span> <span class="fpage" tagx="fpage" title="fpage">779</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8254673">8254673</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8254673">8254673</a> </span> </span> </li> <li tag="ref"> <a name="R07" /> <span class="label" tagx="label" title="label">7</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">RA</span> <span class="given-names" tagx="given-names" title="given-names">Laskowski</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="source" tagx="source" title="source">J Appl Cryst</span> <span class="year" tagx="year" title="year">1993</span> <span class="volume" tagx="volume" title="volume">26</span> <span class="fpage" tagx="fpage" title="fpage">283</span> </span> </li> <li tag="ref"> <a name="R08" /> <span class="label" tagx="label" title="label">8</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">GN</span> <span class="given-names" tagx="given-names" title="given-names">Ramachandran</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="source" tagx="source" title="source">J Mol Biol</span> <span class="year" tagx="year" title="year">1963</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">95</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/13990617">13990617</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/13990617">13990617</a> </span> </span> </li> <li tag="ref"> <a name="R09" /> <span class="label" tagx="label" title="label">9</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">W</span> <span class="given-names" tagx="given-names" title="given-names">Kaplan</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">TG</span> <span class="given-names" tagx="given-names" title="given-names">Littlejohn</span> </span> </span> <span class="source" tagx="source" title="source">Brief Bioinform</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="fpage" tagx="fpage" title="fpage">195</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11465736">11465736</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11465736">11465736</a> </span> </span> </li> <li tag="ref"> <a name="R10" /> <span class="label" tagx="label" title="label">10</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">D</span> <span class="given-names" tagx="given-names" title="given-names">Eisenberg</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="source" tagx="source" title="source">Methods Enzymol</span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">277</span> <span class="fpage" tagx="fpage" title="fpage">396</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9379925">9379925</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9379925">9379925</a> </span> </span> </li> <li tag="ref"> <a name="R11" /> <span class="label" tagx="label" title="label">11</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">VC</span> <span class="given-names" tagx="given-names" title="given-names">Colovos</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">TO</span> <span class="given-names" tagx="given-names" title="given-names">Yeates</span> </span> </span> <span class="source" tagx="source" title="source">Protein Sci</span> <span class="year" tagx="year" title="year">1993</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="fpage" tagx="fpage" title="fpage">1511</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8401235">8401235</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8401235">8401235</a> </span> </span> </li> <li tag="ref"> <a name="R12" /> <span class="label" tagx="label" title="label">12</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">MJ</span> <span class="given-names" tagx="given-names" title="given-names">Sippl</span> </span> </span> <span class="source" tagx="source" title="source">Proteins</span> <span class="year" tagx="year" title="year">1993</span> <span class="volume" tagx="volume" title="volume">17</span> <span class="fpage" tagx="fpage" title="fpage">355</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8108378">8108378</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8108378">8108378</a> </span> </span> </li> <li tag="ref"> <a name="R13" /> <span class="label" tagx="label" title="label">13</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">SC</span> <span class="given-names" tagx="given-names" title="given-names">Gill</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">PH</span> <span class="given-names" tagx="given-names" title="given-names">von Hippel</span> </span> </span> <span class="source" tagx="source" title="source">Anal Biochem</span> <span class="year" tagx="year" title="year">1989</span> <span class="volume" tagx="volume" title="volume">182</span> <span class="fpage" tagx="fpage" title="fpage">319</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2610349">2610349</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2610349">2610349</a> </span> </span> </li> <li tag="ref"> <a name="R14" /> <span class="label" tagx="label" title="label">14</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">K</span> <span class="given-names" tagx="given-names" title="given-names">Guruprasad</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="source" tagx="source" title="source">Protein Eng</span> <span class="year" tagx="year" title="year">1990</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">155</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2075190">2075190</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2075190">2075190</a> </span> </span> </li> <li tag="ref"> <a name="R15" /> <span class="label" tagx="label" title="label">15</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">A</span> <span class="given-names" tagx="given-names" title="given-names">Ikai</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="source" tagx="source" title="source">J Biochem</span> <span class="year" tagx="year" title="year">1980</span> <span class="volume" tagx="volume" title="volume">88</span> <span class="fpage" tagx="fpage" title="fpage">1895</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7462208">7462208</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7462208">7462208</a> </span> </span> </li> <li tag="ref"> <a name="R16" /> <span class="label" tagx="label" title="label">16</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">J</span> <span class="given-names" tagx="given-names" title="given-names">Kyte</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">RF</span> <span class="given-names" tagx="given-names" title="given-names">Doolittle</span> </span> </span> <span class="source" tagx="source" title="source">J Mol Biol</span> <span class="year" tagx="year" title="year">1982</span> <span class="volume" tagx="volume" title="volume">157</span> <span class="fpage" tagx="fpage" title="fpage">105</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7108955">7108955</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7108955">7108955</a> </span> </span> </li> <li tag="ref"> <a name="R17" /> <span class="label" tagx="label" title="label">17</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">MR</span> <span class="given-names" tagx="given-names" title="given-names">Wilkins</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="source" tagx="source" title="source">Methods Mol Biol</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">112</span> <span class="fpage" tagx="fpage" title="fpage">531</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10027275">10027275</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10027275">10027275</a> </span> </span> </li> <li tag="ref"> <a name="R18" /> <span class="label" tagx="label" title="label">18</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">RA</span> <span class="given-names" tagx="given-names" title="given-names">Laskowski</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">33</span> <span class="fpage" tagx="fpage" title="fpage">W89</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15980588">15980588</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15980588">15980588</a> </span> </span> </li> <li tag="ref"> <a name="R19" /> <span class="label" tagx="label" title="label">19</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">T</span> <span class="given-names" tagx="given-names" title="given-names">Castrignanò</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">34</span> <span class="fpage" tagx="fpage" title="fpage">D306</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16381873">16381873</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16381873">16381873</a> </span> </span> </li> </ul> </div> </div>  </body></html>