<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Sci Rep</span> <span class="iso-abbrev" title="iso-abbrev">Sci Rep</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Scientific Reports</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">2045-2322</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5830579">5830579</a> </span> <span class="publisher-id" title="publisher-id">22134</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41598-018-22134-x">10.1038/s41598-018-22134-x</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Bioprocessing strategies for cost-effective large-scale biogenic synthesis of nano-MgO from endophytic <i>Streptomyces coelicolor</i> strain E72 as an anti-multidrug-resistant pathogens agent </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">EL-Moslamy</span> <span class="given-names" tagx="given-names" title="given-names">Shahira H.</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">elmoslamygroup@yahoo.com</span> </span> <a href="#Aff1" /> </span> <span class="citation_author_institution" id="Aff1">[], </span> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-2-2</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2018-2-2</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2018</span> </span> <span class="volume" tagx="volume" title="volume">8</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">3820</span> <span class="history" title="history"> <span class="received" title="received">received: 2017-9-18</span> <span class="accepted" title="accepted">accepted: 2018-2-13</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , </span> <span class="license" title="license"> <span class="license-p" title="license-p"> <b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>In this report, the local nano-MgO synthesizer strain has been isolated from <i>Ocimum sanctum</i> plant and deposited in GenBank as endophytic <i>Streptomyces coelicolor</i> strain E72. Its intracellular metabolic fraction that contains 7.2 μg/μl of carbohydrate, 6.3 g/l of protein and 5.2 nmol/hr/ml of nitrate reductase used to produce multi-surface shaped nano-MgO with diameter ~25 nm. To the best of our knowledge, this is the first report using statistical nanobiotechnological strategies (Plackett -Burman, Box-Behnken and Taguchi experimental designs) to study and evaluate the endophytic <i>S</i>. <i>coelicolor</i> biomass production (123.3 g/l) and extract the highest bioactive metabolites that used for biogenic synthesis of nano-MgO (320 g/l) through exponential sucrose pulses feeding fermentation strategy after 192 hr in semi industrial scale bioreactor (7 L). Purified nano-MgO applied <i>in vitro</i> against multi-drug resistant human pathogens and the large inhibition zone recorded against <i>Shigella flexneri</i> (108 ± 10.53 mm). The average of MICs was recorded as 25 µg/ml that inhibited 90% of the pathogenic living cells and compared with 100 mg/ml ampicilin/sulbactam solution that killed 40% of the same pathogen. These results are expected to gather sufficient knowledge to discover and develop a new cheap and eco-friendly nano-MgO as an extremely strong antimicrobial agent used in biomedical applications. </p> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2018" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p>There are different human pathogenic species resist all known antibiotic classes and organic antimicrobial agents (bacteriocins and enzymes, etc), so the pharmaceutical companies and the researchers planned to emerge up from this problem by discovering and developing other antimicrobial agents <sup> <a href="#CR1">1</a>– <a href="#CR4">4</a> </sup>. Synthesized nanoparticles recognized as effective antimicrobial agents used in different biomedical applications due to their morphological structure, stability, and surface properties <sup> <a href="#CR5">5</a>– <a href="#CR10">10</a> </sup>. In recent years, the most studied nano-science publications interested in bioprocessing development for metal oxide nanoparticles that used as novel antimicrobial agents due to their ability to fight and control multi-drugs resistant microbial infections <sup> <a href="#CR2">2</a>, <a href="#CR3">3</a>, <a href="#CR5">5</a>, <a href="#CR6">6</a> </sup>. The explored nanoparticles that significantly effect on the growth of different human pathogenic species can be listed in the following sequence Ag &amp;gt; Hg &amp;gt; Cu &amp;gt; Cd &amp;gt; Cr &amp;gt; Pb &amp;gt; Co &amp;gt; Au &amp;gt; Zn &amp;gt; Fe &amp;gt; Mn &amp;gt; Mo &amp;gt; Sn <sup> <a href="#CR11">11</a>– <a href="#CR17">17</a> </sup>. Magnesium oxide nanoparticles are an attractive metal oxide material that used for bio-medical, agriculture and electrochemical applications <sup> <a href="#CR7">7</a>– <a href="#CR9">9</a> </sup>. Recently, the potential bio-applications of the bio-synthesized nano-MgO such as a curative catalyst agent, anticancer drug, anti-diabetic agent, anti-microbial agent, electrochemical biosensor, remediation agent, paints and superconductors were reported in several studies <sup> <a href="#CR5">5</a>– <a href="#CR10">10</a>, <a href="#CR14">14</a>– <a href="#CR17">17</a> </sup>. Nano-MgO was applied also as antimicrobial agent that induced the systemic resistance in plant against microbial wilt disease that caused by phytopathogenic bacteria and/or phytopathogenic fungi <sup> <a href="#CR18">18</a>– <a href="#CR20">20</a> </sup>. According to the cytotoxic profiles, there are many reports strongly highlight that the low concentration of the bio-synthesized nano-MgO are less toxic nanoparticles to human cells with time then can be applied safely in the therapeutic, agriculture and water treatment applications with lass potential hazards to the environment and humans <sup> <a href="#CR21">21</a>– <a href="#CR23">23</a> </sup>. Nano-MgO has significant characters such as long-term thermal stability at high levels, low electrical conductivity, and low toxicity compared with nano-Ag that draw the attention of many scientists on the development of those nanoparticles at industrial level by using the safe bio-processing strategy <sup> <a href="#CR18">18</a>– <a href="#CR22">22</a> </sup>. Nano-MgO can be synthesized conventionally by using chemical or physical techniques via sol-gel, hydrothermal, flame spray pyrolysis, laser vaporization, chemical gas phase deposition, combustion aerosol synthesis, aqueous wet chemical, and surfactant methods <sup> <a href="#CR21">21</a>– <a href="#CR23">23</a> </sup>. Nano-MgO that synthesized by using these complicated and expensive techniques produced hazardous residues which affected on its bio-application for biological and medical uses <sup> <a href="#CR13">13</a>– <a href="#CR17">17</a> </sup>. So, biogenic methods served as proficient alternate methods to produce nontoxic nano-MgO for biological and therapeutic applications by using plant extracts, algae, bacteria, fungi and yeasts <sup> <a href="#CR16">16</a>– <a href="#CR19">19</a> </sup>. Since the extracted safe bioactive metabolites (bio-chemicals, enzymes, vitamins, alkaloids, phenols, proteins, and polysaccharides) can be used as reducing/capping agents and/or stabilizing agents for the biogenic synthesis of nano-MgO <sup> <a href="#CR20">20</a>, <a href="#CR21">21</a> </sup>. So the fabrication of nano-MgO by using biogenic techniques has different and significant characters due to their simplicity, environmental sustainability, cost-effectiveness, reproducibility compared with physical and chemical techniques <sup> <a href="#CR22">22</a>, <a href="#CR23">23</a> </sup>. The most important factor that affecting on the biosynthesis of nanoparticles which used in biomedical applications is nanoparticles synthesizer because it should be safe and produce the highest bio-reluctant concentration by using low cost of cultivation method <sup> <a href="#CR23">23</a>– <a href="#CR25">25</a> </sup>. This factor followed by others such as reaction period, temperature, pH, and its nano-form stability. Due to all the previous characters <i>actinomycetes</i> are attractive safe microorganism used for biogenic synthesis of nanoparticles. Nowadays, there are many advanced nano-biotechnological strategies, have emerged can be used as tools to improve the production of natural bioactive metabolites and their activities to reduce the production line cost. Bad construction of the optimization technique that depends on the effect of one variable at a time leads to misinterpretation of the final results because of the omission of interaction effect among different variables such as medium parameters, biosynthesis of nanoparticles factors and so on <sup> <a href="#CR23">23</a>– <a href="#CR25">25</a> </sup>. So, statistical experimental designs are ideal promising tools that recently extensively applied and reported as proficient and effective methodologies such as Plackett - Burman design (PBD), Box - Behnken design (BBD) and Taguchi design methods (TD) that evaluate the desired natural products which used for different industrial applications as much as possible <sup> <a href="#CR23">23</a>– <a href="#CR25">25</a> </sup>. These designs depended on specific sequence steps such as screening for the most effective factors that selected for studying the interaction effect among them and calculating their optimum concentrations which affect on desirable responses <sup> <a href="#CR23">23</a>– <a href="#CR25">25</a> </sup>. </p> <p>Medicinal plants are precious safe sources of novel microbial and potent bioactive compounds diversity that used as new drug candidates (raw materials). It is important to note that these treasure troves of natural drugs are most effective with negligible side effects and investment costless available <sup> <a href="#CR26">26</a>– <a href="#CR28">28</a> </sup>. Endophyte is a microbe that colonizes in the intracellular or intercellular tissues of healthy plant and explored as an important part of micro-ecosystem <sup> <a href="#CR29">29</a>– <a href="#CR31">31</a> </sup>. Endophytic microbe’s flora that isolated from these medicinal plants has the same and/or similar bioactive metabolites, thus making them a proficient and unique source of novel safe natural compounds for medicinal applications <sup> <a href="#CR32">32</a>– <a href="#CR35">35</a> </sup>. Several studies have indicated that endophytes are distributed in roots, stems, leaves, flowers, fruits and seeds which classified into endophytic bacteria, actinomycetes and fungi <sup> <a href="#CR36">36</a>– <a href="#CR38">38</a> </sup>. In recent years, secondary metabolites of isolated endophytes used as antitumor, antimicrobial, antioxidant, and other functions that have a potential application prospect in pharmacy <sup> <a href="#CR28">28</a>, <a href="#CR31">31</a>, <a href="#CR36">36</a>– <a href="#CR38">38</a> </sup>. Until now, very few researchers have reported endophytic actinomycetes as nanoparticles biosynthesizer by using nanobiotechnological strategies <sup> <a href="#CR29">29</a>, <a href="#CR30">30</a> </sup>. So, endophytic actinomycetes have attracted attention in the search for an increasingly important source of promising potential new bioactive compounds that used as novel nanoparticles-synthesizers <sup> <a href="#CR26">26</a>– <a href="#CR28">28</a>, <a href="#CR34">34</a>, <a href="#CR35">35</a> </sup>. To the best of our knowledge, there are no reports on the biosynthesis of nano-MgO using an endophytic <i>Streptomyces</i> sp. So this study focuses on isolation of endophytic <i>Streptomyces sp</i> from <i>Ocimum sanctum</i> and using promising statistical techniques for <i>Streptomyces sp</i> cultivation and biogenic synthesis of nano-MgO that was characterized and its antimicrobial activity was evaluated against clinical multi drug resistant strains. </p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Methods</div> <div class="collectionoftheplant,surfacesterilizationandactinomycetesisolation" title="sec"> <div class="title" tagx="title" title="title">
Collection of the plant, surface sterilization and <i>Actinomycetes</i> isolation </div> <p>Healthy plants of <i>Ocimum sanctum</i> were randomly selected from Lower Egypt herbal gardens. Samples were placed in sterile polythene bag cover and brought to the laboratory in boxes. A Leaf sample was washed with tap water for many times to remove adhered soil debris and with sterilized water for three times in sterile beaker. After that the leaf sample was surface-sterilized with 70% ethyl alcohol for 5 min and subsequently with absolute ethyl alcohol for 2 min <sup> <a href="#CR32">32</a>, <a href="#CR36">36</a> </sup>. The ethanol sterilized leaves were dried on sterilized filter paper for 10 min and then were treated with 6% sodium hypochlorite with Tween 20 (0.1%) for 2 min to remove epiphytic microorganisms. The leaves were soaked in 10% NaHCO <sub>3</sub> for 20 min to disrupt the growth of the fungi <sup> <a href="#CR29">29</a>, <a href="#CR31">31</a> </sup>. At last leaves rinsed with sterilized distilled water for many times and sterility checked (the final rinsing water was spread on starch casein agar as the control plates) <sup> <a href="#CR23">23</a>– <a href="#CR25">25</a> </sup>. The surface sterilized leaves were cut with sterile scalpel blade and crushed in sterilized distilled water by using sterile pestle and mortar. By using serial dilution method 100 μl of the diluted extract was spread on starch casein agar (0.1% sodium caseinate, 1% soluble starch, 0.03% K <sub>2</sub>HPO <sub>4</sub>, 1.8% agar, pH 6.8–7.0) supplemented with 25 µg/ml cycloheximide, 50 μg/ml nalidixic acid and 50 μg/ml nystatin to suppress fungal growth and incubated at 30 °C for three weeks <sup> <a href="#CR29">29</a>, <a href="#CR31">31</a> </sup>. <i>Actinomycetes</i> growing on the media were isolated and purified. Purified isolates were stored at −80 °C in 50% (v/v) glycerol for long term use. Finally, the pure cultures can be inoculated on the broth medium for bioprocessing fermentation studies. </p> </div> <div class="screeningofisolatedactinomycetesfornano-mgobiosynthesis" title="sec"> <div class="title" tagx="title" title="title">
Screening of isolated <i>Actinomycetes</i> for nano-MgO biosynthesis </div> <p>Each isolate was grown in 100 ml of MGYP medium (0.3% malt extract, 1% glucose, 0.3% yeast extract, 0.5% peptone and the final pH was 6.8) at 30 °C for 72 hr. Balls of mycelia were separated by centrifugation (10,000 rpm) at 4 °C for 15 min and then the mycelia were washed thrice with sterile distilled water. Wet weight of the harvested biomass (20 g) was re-suspended in 100 ml sterile distilled water and incubated on a shaker (150 rpm) at 30 °C for 72 hr; then the cell-free filtrate was obtained by using ultrafiltration system. Biosynthesis reaction for nano-MgO was set up by a mix of 90 ml of precursor metal salt (1 M Mg(NO <sub>3</sub>) <sub>2</sub>·6H <sub>2</sub>O) with 10 ml of extracellular bioactive compounds (pH 5.0) at 80 °C under constant mechanically stirring for 2 hr and allowed to settle for 24 hr <sup> <a href="#CR9">9</a>, <a href="#CR39">39</a> </sup>. The upper layer was discarded carefully and the remaining pellet was centrifuged at 14,000 rpm for 25 min. This pellet was washed several times to remove the impurities <sup> <a href="#CR5">5</a>, <a href="#CR8">8</a> </sup>. Subsequently, the separated pellet was dried in an oven at 80 °C for 12 hr and then crushed to make it very fine powder by using a mortar pestle. Finally, the fine powder of nano-MgO is calcinated at 700 °C for 3 hr for the removal of impurities. Nano-MgO antibacterial activity was screened against Gram−ve and Gram + ve bacteria ( <i>Escherichia coli</i>, <i>Proteus vulgaris</i>, <i>Salmonella typhimurium</i>, <i>Shigella flexneri</i>, <i>Klebsiella pneumonia</i>, <i>Staphylococcus aureus</i>, <i>Streptococcus pneumonia</i> and <i>Bacillus cereus</i>) by using agar well diffusion method and compared with standard conventional antibiotics by using disc diffusion method <sup> <a href="#CR2">2</a>, <a href="#CR4">4</a>, <a href="#CR7">7</a>, <a href="#CR40">40</a>, <a href="#CR41">41</a> </sup>. </p> </div> <div class="physicochemicalpropertiesofpreparednano-mgo" title="sec"> <div class="title" tagx="title" title="title">
Physicochemical properties of prepared nano-MgO</div> <p>The initial characterization for prepared nano-MgO was monitored by measuring the UV–vis absorbance spectrum by using a UV-Visible Spectroscopy (Shimadzu, Tokyo, Japan) to detect the surface plasmon resonance band <sup> <a href="#CR6">6</a>, <a href="#CR8">8</a>– <a href="#CR10">10</a> </sup>. The structural properties (real estate properties) of the nano-MgO were analyzed by transmission electron microscope (JEOL JEM2100F- Japan), scanning electronmicroscopy (SEM) (JEOL JSM 6360LA, Japan), scientific particle size analyzer (PSA) and its chemical composition was demonstrated by an energy dispersive X-Ray (EDX) analyzer. The FTIR spectrums were measured using Shimadzu FTIR-8400 S, Japan, over the wavelength range 400–4000 cm <sup>−1</sup>. X-ray diffraction patterns were obtained using Schimadzu 7000 diffractometer operating with Cu Kα <sub>1</sub> radiation(λ = 0.15406 nm) generated at 30 kV and 30 mA with the scan rate of 2°/min for 2θ values between 20° and 80°. The average particle size was estimated using the Scherrer formula Eq. ( <a href="#Equ1">1</a>). </p><div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">1</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M1">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$${\bf{D}}=k{\boldsymbol{\lambda }}/{\boldsymbol{\beta }}\,\cos \,{\boldsymbol{\theta }}$$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
k</div> <div class="mi_UNKNOWN">
λ</div> <div class="mo_UNKNOWN">
/</div> <div class="mi_UNKNOWN">
β</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
cos</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
θ</div> </div> <div class="graphic_UNKNOWN" /> </div> </div>where <b>D</b> is particle diameter, <b>k</b> is a constant equals 0.9, <b>λ</b> is wavelength of X-ray source (0.1541 nm), <b>β</b> is the full width at half maximum (FWHM) and θ is the half diffraction angle. <p /> </div> <div class="molecularidentificationoftheendophyticactinomycetesisolate" title="sec"> <div class="title" tagx="title" title="title">
Molecular identification of the endophytic <i>Actinomycetes</i> isolate </div> <p>Genomic DNA was extracted by using spin-column DNeasy Blood &amp;amp; Tissue Kit according to the QIAGEN quick- prep method. The 16 S ribosomal RNA gene was amplified by using the PCR reaction mixture (25 μl) that contained 80 ng of template DNA, 10 mM each dNTP, 1.5 mM MgCl <sub>2</sub>, 10pmol of each primer (5′ AGT TTG ATC CTG GCT CAG 3′ and 5′ ACG GCT ACC TTG TTA CGA CTT 3′), and 0.5 μl of 500 U Taq DNA polymerase. PCR amplification program was carried out for 40 cycles of 95 °C for 1 min (denaturing), 52 °C for 1 min (annealing) and 72 °C for 1 min (extension). The initial denaturing temperature was 95 °C extended to 5 min; the final extension was for 10 min at 72 °C and then cooled to 4 °C <sup> <a href="#CR3">3</a>, <a href="#CR4">4</a>, <a href="#CR15">15</a>, <a href="#CR16">16</a>, <a href="#CR42">42</a> </sup>. The purified PCR product was sequenced directly according to the protocol recommended by the manufacturer (Model 3130 automated DNA sequencer, Genetic Analyzer, Applied Biosystems, Hitachi, Japan). The sequence was compared for similarity with the homologous sequences using NCBI BLAST available at <a href="http://www.ncbinlmnih.gov/">http://www.ncbinlmnih.gov/</a>. Multiple sequence alignment was carried out by using BioEdit software (Hall, 1999), and the phylogenetic tree was constructed using the neighbor-joining (NJ) method through an MEGA6 program. </p> </div> <div class="screeningoftheselectedendophyticactinomycetesbioactivemetabolitesforbiogenicsynthesisofnano-mgo" title="sec"> <div class="title" tagx="title" title="title">
Screening of the selected endophytic <i>Actinomycetes</i> bioactive metabolites for biogenic synthesis of nano-MgO </div> <p>To measure the effectiveness broth culture media that used for the highest biomass production and hence the maximum bioactive metabolites production which used as reducing/stabilizing agent for biogenic synthesis of nano-MgO the comparative evaluation strategy for cultivation media was performed in this study. The compositions of the broth culture media were selected from the previous publications <sup> <a href="#CR43">43</a>– <a href="#CR48">48</a> </sup>. Modified Czapek-Dox medium <b>(M1)</b> consisted of (g/l): Sucrose, 30.0 g; (NH <sub>4</sub>) <sub>2</sub>SO <sub>4</sub>, 2.0 g; NaNO <sub>3</sub>, 1.0 g; K <sub>2</sub>HPO <sub>4</sub>, 1.0 g; KCl, 0.5 g; MgSO <sub>4</sub>·7H <sub>2</sub>O 0.5 g and FeSO <sub>4</sub>7H <sub>2</sub>O, 0.01 g, Yeast Malt Broth <b>(M2):</b> malt extract, 6.0 g; maltose, 1.8 g; dextrose, 6.0 g; and yeast extract, 1.2 g, Starch casein agar <b>(M3):</b> Soluble starch, 10.0 g; casein, 0.3 g; KNO <sub>3</sub>, 2.0 g; NaCl, 2.0 g; K <sub>2</sub>HPO <sub>4</sub>, 2.0 g; MgSO <sub>4</sub>·7H <sub>2</sub>O, 0.05 g; CaCO <sub>3</sub>, 0.02 g; FeSO <sub>4</sub>·7H <sub>2</sub>O, 0.01 g; pH 7.0, Basal synthetic medium <b>(M4)</b>: (NH <sub>4</sub>) H <sub>2</sub>PO <sub>4</sub>, 2.0 g; Dextrose, 20.0 g; K <sub>2</sub>HPO <sub>4</sub>, 1.0 g; MgSO <sub>4</sub>·7H <sub>2</sub>O, 0.5 g; CaCl <sub>2</sub>·2H <sub>2</sub>O, 0.04 g; FeSO <sub>4</sub>·7H <sub>2</sub>O, 0.005 g, and ZnSO <sub>4</sub>·7H <sub>2</sub>O, 0.0005 g, and finally, Glucose Soyabean meal broth <b>(M5)</b> that containing: Glucose, 10.0 g; Soyabean meal, 10.0 g; NaCl, 10.0 g; CaCO <sub>3,</sub> 1.0 g and pH adjusted to 7.0. The cultivation was carried out in 250 ml shake flasks containing 100 ml of the different media separately that sterilized at 121 °C for 15 min and inoculated by 5discs of vegetative cells. The cultivation temperature was controlled at 26 ± 2 °C for 3 days with shaking in an orbital rotary shaker (150 rpm). The biomass was harvested through ultrafiltration system and washed several times with distilled water in order to remove any impurities. By using a clean spatula the pure biomass was transferred to a pre-weighed dry Whatman filter paper and then placed in an oven at 50 °C to reach a fixed weight which expressed as g/l of culture medium <sup> <a href="#CR23">23</a>– <a href="#CR25">25</a> </sup>. </p> <p>High functional bioactive compounds yielding through cell fractionations were checked (extracellularly and intracellularly). After fermentation step, the broth culture was centrifuged at 14,000 rpm for 20 min to separate the supernatant and the pellet <sup> <a href="#CR1">1</a>, <a href="#CR25">25</a>, <a href="#CR43">43</a>, <a href="#CR49">49</a> </sup>. To extract the extracellular metabolites equal volume of ethyl acetate was added to the supernatant and the mixture was kept for overnight in a rotary shaker. The filtrates were then evaporated using a rotary evaporator at 40 °C. The crude intracellular bioactive metabolite was extracted from the washed biomass (the pellet) that treated with CTAB extraction buffer (100 mM Tris-Cl (pH 8.0), 20 mM EDTA (pH 8.0), 1.4 M NaCl, 2% (w/v) cetyltrimethyl ammonium bromide and 1% PVP 40,000 (polyvinyl pyrrolidone) for 2 hr under shaking condition and finally centrifuged at 14000 rpm for 10 min <sup> <a href="#CR44">44</a>, <a href="#CR50">50</a>, <a href="#CR51">51</a> </sup>. The concentrated bioactive metabolites extracts were then transferred into clean glass screw cap tube and stored at 4 °C for further use. </p> <p>Total contained proteins in both concentrated cell fractionations were quantified by using the commercial Bio-Rad Colorimetric Protein Assays kit. Also, the total carbohydrate contents of both cell fractionations were determined by using the total carbohydrate colorimetric assay kit (Milpitas, CA 95035 USA). The nitrate reductase enzyme in both cell fractionations was checked and assayed according to <sup> <a href="#CR29">29</a>, <a href="#CR31">31</a>, <a href="#CR52">52</a> </sup>. Cell fractionations were mixed with 10 ml of assay medium (30 mM KNO <sub>3</sub> and 5% isopropanol in 0.1 M phosphate buffer of pH 7.5) and incubated at 30 °C for 1 hr in the dark condition. After incubation, nitrites formed into the assay mixture (pink color) after adding 1 ml of sulphanilamide and NEED (N-(1-naphthyl) ethylenediamine dihydrochloride) solutions <sup> <a href="#CR32">32</a>, <a href="#CR38">38</a> </sup>. The enzyme activity was estimated by using an UV–vis spectrophotometer at 440 nm and finally expressed in terms of nM of nitrite/hr/ml. Finally, nano-MgO biosynthesis reaction was set up by a mix of 90 ml of precursor metal salt (1 M Mg(NO <sub>3</sub>) <sub>2</sub>·6H <sub>2</sub>O) with 10 ml of both cell fractionations (pH 5.0) at 80 °C under constant mechanically stirring for 2 hr and allowed to settle for 24 hr <sup> <a href="#CR9">9</a>, <a href="#CR14">14</a>, <a href="#CR17">17</a>, <a href="#CR53">53</a>, <a href="#CR54">54</a> </sup>. The centrifuged pellet (at 14,000 rpm for 25 min) was washed several times dried in an oven at 80 °C for 12 hr. The fine powders of nano-MgO weight are estimated (g/l) at these comparative studies. </p> </div> <div class="cost-effectiveoptimizationstrategiesforhighbiomassproduction" title="sec"> <div class="title" tagx="title" title="title">
Cost-effective optimization strategies for high biomass production</div> <p>The traditional non-statistical optimization strategies, one-factor-at-a-time approach (OFAT), Is incompetent to detect the true optimum due to the complex interactions among various used parameters <sup> <a href="#CR23">23</a>– <a href="#CR25">25</a>, <a href="#CR41">41</a>, <a href="#CR55">55</a>– <a href="#CR57">57</a> </sup>. Recently, statistical experimental design methodologies considered as powerful tools used for the optimization of the target bioactive metabolite production. In order to it has overcome many obstacles; therefore it can solve multivariate problems to check the significance of several variables and their complex interactions. So statistical experimental designs such as Plackett-Burman design (PBD) and Box-Behnken design (BBD) were applied in this report to produce the highest cost effectiveness biomass production and reduce the fermentation processing time. SigmaPlot 12.5 and Minitab 15.0 were used for these experimental designs and subsequent analysis of the final experimental data. </p> <p>The statistical optimization strategies for biomass production were carried out in two steps by using fractional factorial method (PBD) followed by response surface method (BBD). Fractional factorial method (PBD) was adopted for screening the most significant nutritional parameters for biomass and bioactive metabolites production by using identified <i>Streptomyces</i> strain in submerged fermentation. In the experimental design, seven independent variables (Sucrose, (NH <sub>4</sub>) <sub>2</sub>SO <sub>4</sub>, NaNO <sub>3</sub>, KCl, K <sub>2</sub>HPO <sub>4</sub>, MgSO <sub>4</sub>·7H <sub>2</sub>O, and FeSO <sub>4</sub>·7H <sub>2</sub>O) were screened to pick significant factors that influence biomass production by representing them at two levels (high and low) in eight experiments. All trials were set up in 250 ml Erlenmeyer flasks containing 100 ml of the selected medium (basal medium) and carried out in triplicate to calculate the average biomass weight (the response). The significance of variables was determined by calculating the <i>p-</i>value (&amp;lt;5%) and confidence levels through standard regression analysis <sup> <a href="#CR23">23</a>– <a href="#CR25">25</a>, <a href="#CR55">55</a> </sup>. The quality of the first order polynomial model equation was applied statistically by the coefficient of determination R <sup>2</sup>, and its statistical significance was tested by an F-test. Finally, the significant variables were selected for further optimization study. The 1 <sup>st</sup> order model and main effect the response represented in Eqs ( <a href="#Equ2">2</a>) and ( <a href="#Equ3">3</a>). </p><div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">2</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M3">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$${\boldsymbol{Y}}={{\boldsymbol{\beta }}}_{{\boldsymbol{o}}}+{\boldsymbol{\Sigma }}{{\boldsymbol{\beta }}}_{{\boldsymbol{i}}}{{\boldsymbol{x}}}_{{\boldsymbol{i}}}$$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
o</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
Σ</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> <div class="graphic_UNKNOWN" /> </div> </div>where <b>Y</b> is the response (biomass production), <b>β</b> <sub> <b>0</b> </sub> is the models intercept, <b>β</b> <sub> <b>i</b> </sub> is the variable estimate, and <b>X</b> <sub> <b>i</b> </sub> represents the variable. <div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">3</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M5">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$${\bf{M}}{\bf{a}}{\bf{i}}{\bf{n}}\,{\bf{e}}{\bf{f}}{\bf{f}}{\bf{e}}{\bf{c}}{\bf{t}}=\sum \,(+{\bf{1}})/{\bf{n}}\,(+{\bf{1}})\,-\sum \,(-{\bf{1}})/{\bf{n}}\,(-{\bf{1}})$$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
M</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
n</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
∑</div> <div class="mspace_UNKNOWN" /> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
/</div> <div class="mi_UNKNOWN">
n</div> <div class="mspace_UNKNOWN" /> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
−</div> <div class="mo_UNKNOWN">
∑</div> <div class="mspace_UNKNOWN" /> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
/</div> <div class="mi_UNKNOWN">
n</div> <div class="mspace_UNKNOWN" /> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="graphic_UNKNOWN" /> </div> </div> <p /> <p>A 3-level for 3 factors BBD is a powerful most common statistical tool that widely employed for the optimization of industrial fermentation bioprocesses by investigating the interactive effect of the tested variables on product yield through a mathematical model <sup> <a href="#CR23">23</a>– <a href="#CR25">25</a>, <a href="#CR41">41</a>, <a href="#CR55">55</a> </sup>. In this report, the BBD consisted of fifteen trials and the three independent variables (Sucrose, (NH <sub>4</sub>) <sub>2</sub>SO <sub>4</sub> and K <sub>2</sub>HPO <sub>4</sub>) were studied at three different levels, (−1), (0), (+1), and the dependent variable (response) was biomass production to determine their optimum levels for maximum biomass production. Statistical analysis of this model was performed to evaluate the analysis of variance (ANOVA). The created model was applied using the coefficient results of each variable. The second-order polynomial structured represented in Eq. ( <a href="#Equ4">4</a>). </p><div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">4</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M7">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$${\bf{Y}}={{\boldsymbol{\beta }}}_{{\bf{0}}}{\boldsymbol{+}}{{\boldsymbol{\beta }}}_{{\bf{1}}}{{\bf{X}}}_{{\bf{1}}}{\boldsymbol{+}}{{\boldsymbol{\beta }}}_{{\bf{2}}}{{\bf{X}}}_{{\bf{2}}}{\boldsymbol{+}}{{\boldsymbol{\beta }}}_{{\bf{3}}}{{\bf{X}}}_{{\bf{3}}}{\boldsymbol{+}}{{\boldsymbol{\beta }}}_{{\bf{11}}}{{{\bf{X}}}_{{\bf{1}}}}^{{\bf{2}}}{\boldsymbol{+}}{{\boldsymbol{\beta }}}_{{\bf{22}}}{{{\bf{X}}}_{{\bf{2}}}}^{{\bf{2}}}{\boldsymbol{+}}{{\boldsymbol{\beta }}}_{{\bf{33}}}{{{\bf{X}}}_{{\bf{3}}}}^{{\bf{3}}}{\boldsymbol{+}}{{\boldsymbol{\beta }}}_{{\bf{12}}}{{\bf{X}}}_{{\bf{1}}}{{\bf{X}}}_{{\bf{2}}}{\boldsymbol{+}}{{\boldsymbol{\beta }}}_{{\bf{13}}}{{\bf{X}}}_{{\bf{1}}}{{\bf{X}}}_{{\bf{3}}}{\boldsymbol{+}}{{\boldsymbol{\beta }}}_{{\bf{23}}}{{\bf{X}}}_{{\bf{2}}}{{\bf{X}}}_{{\bf{3}}}$$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
3</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
3</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
11</div> </div> </div> <div class="msup_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
22</div> </div> </div> <div class="msup_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
33</div> </div> </div> <div class="msup_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
3</div> </div> </div> <div class="mn_UNKNOWN">
3</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
12</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
13</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
3</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
23</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
3</div> </div> </div> </div> <div class="graphic_UNKNOWN" /> </div> </div>where: <b>Y</b> is the response (biomass production); <b>β</b> <sub>0</sub> is the model intercept; <b>X</b> <sub> <b>1</b> </sub>, <b>X</b> <sub> <b>2</b> </sub> and <b>X</b> <sub> <b>3</b> </sub> are independent variables; <b>β</b> <sub> <b>1</b> </sub>, <b>β</b> <sub> <b>2</b> </sub> and <b>β</b> <sub> <b>3</b> </sub> are linear coefficients; <b>β</b> <sub> <b>12</b> </sub>, <b>β</b> <sub> <b>13</b> </sub> and <b>β</b> <sub> <b>23</b> </sub> are cross product coefficients; and <b>β</b> <sub> <b>11</b> </sub>, <b>β</b> <sub> <b>22</b> </sub> and <b>β</b> <sub> <b>33</b> </sub> are the quadratic coefficients. The confidence level and the accuracy of the model have been calculated by using Eqs ( <a href="#Equ5">5</a>)and ( <a href="#Equ6">6</a>). <div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">5</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M9">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$${\bf{T}}{\bf{h}}{\bf{e}}\,{\bf{c}}{\bf{o}}{\bf{n}}{\bf{f}}{\bf{i}}{\bf{d}}{\bf{e}}{\bf{n}}{\bf{c}}{\bf{e}}\,{\bf{l}}{\bf{e}}{\bf{v}}{\bf{e}}{\bf{l}} \% =100\,\ast \,(1-{\boldsymbol{P}}\,{\bf{v}}{\bf{a}}{\bf{l}}{\bf{u}}{\bf{e}})$$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
h</div> <div class="mi_UNKNOWN">
e</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
n</div> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
n</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
e</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
l</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
%</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
100</div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
∗</div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
P</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
l</div> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="graphic_UNKNOWN" /> </div> </div> <div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">6</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M11">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$${\bf{A}}{\bf{c}}{\bf{c}}{\bf{u}}{\bf{r}}{\bf{a}}{\bf{c}}{\bf{y}}\,{\bf{o}}{\bf{f}}\,{\bf{t}}{\bf{h}}{\bf{e}}\,{\bf{m}}{\bf{o}}{\bf{d}}{\bf{e}}{\bf{l}}\,=\,{{\bf{Y}}}_{{\bf{E}}{\bf{x}}{\bf{p}}{\bf{e}}{\bf{r}}{\bf{i}}{\bf{m}}{\bf{e}}{\bf{n}}{\bf{t}}}/{{\bf{Y}}}_{{\bf{C}}{\bf{a}}{\bf{l}}{\bf{c}}{\bf{u}}{\bf{l}}{\bf{a}}{\bf{t}}{\bf{e}}{\bf{d}}}\,\times \,100$$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
y</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
f</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
h</div> <div class="mi_UNKNOWN">
e</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
m</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
l</div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
=</div> <div class="mspace_UNKNOWN" /> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
m</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
n</div> <div class="mi_UNKNOWN">
t</div> <div class="mspace_UNKNOWN" /> </div> </div> </div> <div class="mo_UNKNOWN">
/</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
l</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
l</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
d</div> </div> </div> </div> </div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
×</div> <div class="mspace_UNKNOWN" /> <div class="mn_UNKNOWN">
100</div> </div> <div class="graphic_UNKNOWN" /> </div> </div>The simultaneous effects of the three most significant independent factors on each response were visualized using three-dimensional graphs generated by stat soft Sigma plot 12.5 soft ware. The value of each X <sub>1</sub>, X <sub>2</sub> and X <sub>3</sub> was further optimized to calculate the highest Y value using Microsoft excel 2007 solver. The experiments for calculating optimum value of each variable were then conducted. All experiments were carried out in triplicates and finally, the mean values were represented. <p /> </div> <div class="scalingupfermentationstrategies" title="sec"> <div class="title" tagx="title" title="title">
Scaling up fermentation strategies</div> <p>In an accurate industrial bioprocesses strategy, living microbial cells used as cell factories that produce bioactive metabolites by utilizing mathematical kinetic models which are able to develop these cell properties and/or the final production bioprocesses <sup> <a href="#CR23">23</a>– <a href="#CR25">25</a>, <a href="#CR46">46</a>, <a href="#CR51">51</a>, <a href="#CR55">55</a> </sup>. Kinetic cell factory modeling used to understand, predict, develop and assess the effects of modifying (adding and/or removing) cell cultivation components and/or cultivation conditions <sup> <a href="#CR23">23</a>– <a href="#CR25">25</a>, <a href="#CR44">44</a> </sup>. In this work, scaling up strategies performed by using semi-uncontrolled batch, controlled batch and exponential pulses fed-batch cultivation systems. From the previous steps the optimized medium were used for large scale production system. The main objective of this experiment was to set up the large scale fermentation system and estimate the microbial growth kinetics in a submerged cultivation system. </p> <div class="shake-flaskcultivationsystem" title="sec"> <div class="title" tagx="title" title="title">
Shake-flask cultivation system</div> <p>The prepared seed culture was conducted by inoculating a 250 ml Erlenmeyer flask containing 50 ml of the optimized medium with a spore suspension (prepared from a 15-day starch nitrate agar culture) and incubating at 30 °C with constant shaking (150 rpm) for 120 hr and then re-inoculated into 1 L flasks containing 500 ml of the optimized medium and cultured at the same conditions. Samples were periodically removed, centrifuged at 10,000 rpm for 15 min (at 4 °C) for measurements of biomass dry weight and for the preparation of intracellular fractions for biosynthesis of nano-MgO as described in the "Screening of isolated <i>Actinomycetes</i> for nano-MgO biosynthesis" section. </p> </div> <div class="stirred-bioreactorbatchcultivationsystem" title="sec"> <div class="title" tagx="title" title="title">
Stirred-bioreactor batch cultivation system</div> <p>Batch fermentations were conducted in a 7 L Bioflo 310 fermentor (New Brunswick Scientific, Edison, NJ, USA) at 30 °C under controlled pH, temperature and DO concentration. The bioreactor vessel that contained the statistically optimized medium was sterilized by autoclaving at 121 °C for 25 min, while sucrose solution was sterilized separately by autoclaving at 121 °C for 15 min and added aseptically to the bioreactor and then inoculated with prepared inoculums 10% (v/v). The fermentation temperature was adjusted at 30 °C and the agitation speed and aeration rate started at 150 rpm and 0.5 vvm respectively and the dissolved oxygen concentration was maintained at above 40% of air saturation by changing the agitation speed and by regulating the air supply. The pH was controlled at 6.8 by adding 5 N NaOH and/or 5 N HCl. Antifoam (silicone oil 0.5:10 v/v) at a concentration of 1:100 (v/v) in distilled water was added after 24 hr. Samples (50 ml each) were collected periodically and analyzed for biomass, sugar consumption, and nano-MgO production as described previously. </p> </div> <div class="stirred-bioreactorfed-batchcultivationsystem" title="sec"> <div class="title" tagx="title" title="title">
Stirred-bioreactor fed-batch cultivation system</div> <p>The fed batch experiment was designed for carbon source feeding to increase the bioactive metabolites by overcoming carbon limitations. Fed-batch cultivation strategy was developed and conducted with sucrose exponential pulses feeding system by using the optimized batch cultivation results. Feeding was carried out between 96 and 255 hr with the rate of 0.84 g/hr using a prepared sucrose solution (700 g/l) and keeping the dissolved oxygen (DO) value at 40% saturation. Finally, biomass dry weight, sugar consumption, and nano-MgO production were analyzed as described previously. The behavior of microbial cell growth was described kinetically by using different fermentation mode equations <sup> <a href="#CR23">23</a>– <a href="#CR25">25</a>, <a href="#CR44">44</a>, <a href="#CR59">59</a> </sup>. Yield coefficient was calculated based on the amount of sucrose consumed and the growth yield during cultivation. In the log phase, cell mass density increases exponentially with time <i>t</i> and specific growth rate <i>μ</i> (h <sup>−1</sup>) is independent of nutrient concentration. Since the growth kinetic relationship is affected by many parameters such as biomass yield coefficient ( <b> <i>Y</i> </b> <sub> <i>X/S</i> </sub>), Maximum biomass ( <b>X</b> <sub> <b>max</b> </sub>), Maximum nano-MgO production ( <b>P</b> <sub> <b>max</b> </sub>), nano-MgO production yield coefficient ( <b>Y</b> <sub> <b>P</b> </sub>) and maximum specific growth rate ( <b>µ</b> <sub> <b>max</b> </sub>). Yield coefficient was calculated by using Eq. ( <a href="#Equ7">7</a>). </p><div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">7</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M13">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$${{\boldsymbol{Y}}}_{{\boldsymbol{X}}/{\boldsymbol{S}}}=\frac{{\rm{\Delta }}{\boldsymbol{X}}}{{\rm{\Delta }}{\boldsymbol{S}}}=\frac{{\boldsymbol{X}}-{{\bf{X}}}_{0}}{{{\bf{S}}}_{0}-{\boldsymbol{S}}}$$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> <div class="mo_UNKNOWN">
/</div> <div class="mi_UNKNOWN">
S</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
X</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
S</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> <div class="mspace_UNKNOWN" /> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
S</div> </div> </div> </div> <div class="graphic_UNKNOWN" /> </div> </div>where: <b> <i>Y</i> </b> <sub> <b> <i>X/S</i> </b> </sub> Biomass yield on substrate, <b>X</b> Cell concentration, <b>X</b> <sub> <b>0</b> </sub> Initial cell concentrations, <b>S</b> Substrate, <b>S</b> <sub> <b>0</b> </sub> Initial substrate concentration. <b>X</b> and <b>X</b> <sub> <b>0</b> </sub> are biomass concentrations (g/l) at measuring time <b>t</b> and initial time <b>t</b> <sub> <b>0</b> </sub> respectively. <b>S</b> and <b>S</b> <sub> <b>0</b> </sub> are the consumed amounts of carbon source (g/l) at the same times mentioned previously. In logarithmic growth phase, cell biomass density increases exponentially with time <b>t</b> and specific growth rate <b>μ (h</b> <sup>−1</sup> <b>)</b> is independent of nutrient concentration as shown in Eq. ( <a href="#Equ8">8</a>). The most common kinetic model for cell growth was used to determine the maximum specific cell growth rate (Eq. <a href="#Equ9">9</a>). Finally, If the specific growth rate <b>μ</b> is constant; the feeding rate calculated by using Eq. ( <a href="#Equ10">10</a>). <div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">8</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M15">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\frac{{{\boldsymbol{d}}}_{{\boldsymbol{x}}}}{{{\boldsymbol{d}}}_{{\boldsymbol{t}}}}={\boldsymbol{\mu }}X,{\boldsymbol{X}}={{\boldsymbol{X}}}_{0}\,{\boldsymbol{at}}\,{\boldsymbol{t}}=0\to {\bf{l}}{\bf{n}}(\frac{{\boldsymbol{X}}}{{{\boldsymbol{X}}}_{0}})={\boldsymbol{\mu }}t,{\boldsymbol{or}}\,{\boldsymbol{X}}\,=\,{{\boldsymbol{X}}}_{0}\,{{\boldsymbol{e}}}^{{\boldsymbol{\mu }}t}$$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
μ</div> <div class="mi_UNKNOWN">
X</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
X</div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
at</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> <div class="mo_UNKNOWN">
→</div> <div class="mi_UNKNOWN">
l</div> <div class="mi_UNKNOWN">
n</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
μ</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
or</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
X</div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
=</div> <div class="mspace_UNKNOWN" /> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="mspace_UNKNOWN" /> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
μ</div> <div class="mi_UNKNOWN">
t</div> </div> </div> </div> <div class="graphic_UNKNOWN" /> </div> </div> <div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">9</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M17">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$${\boldsymbol{\mu }}={\boldsymbol{\mu }}\_{\bf{m}}{\bf{a}}{\bf{x}}/({\boldsymbol{K}}\_{\boldsymbol{S}})+{\boldsymbol{S}}$$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
μ</div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
μ</div> <div class="mo_UNKNOWN">
_</div> <div class="mi_UNKNOWN">
m</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
x</div> <div class="mo_UNKNOWN">
/</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
K</div> <div class="mo_UNKNOWN">
_</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
S</div> </div> <div class="graphic_UNKNOWN" /> </div> </div>where; <b>μ</b> specific cell growth rate (hr <sup>−1</sup>), <b>μ</b> <sub> <b>max</b> </sub> maximum specific cell growth rate (hr <sup>−1</sup>), <b>S</b> substrate concentration (g/l), <b>K</b> <sub> <b>S</b> </sub> Saturation constant (g/l) =  <b>S</b> when <b>μ = </b>1/2 <b>μ</b> <sub> <b>max</b> </sub>. <div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">10</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M19">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$${\bf{X}}{\bf{V}}={{\bf{X}}}_{0}{{\bf{V}}}_{0}\,{{\bf{e}}}^{{\boldsymbol{\mu }}{\bf{t}}},\frac{{\bf{d}}({\bf{S}}{\bf{V}})}{{\bf{d}}{\bf{t}}}=0,\frac{{\bf{d}}({\bf{S}}{\bf{V}})}{{\bf{d}}{\bf{t}}}={\bf{F}}{{\bf{S}}}_{0}-(\frac{{\boldsymbol{\mu }}{\bf{X}}{\bf{V}}}{{{\bf{Y}}}_{{\bf{X}}/{\bf{S}}}}),{\bf{F}}=\frac{{\boldsymbol{\mu }}{{\bf{X}}}_{0}{{\bf{V}}}_{0}{{\bf{e}}}^{{\boldsymbol{\mu }}{\bf{t}}}}{{{\bf{S}}}_{0}{{\bf{Y}}}_{{\bf{X}}/{\bf{S}}}}$$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> <div class="mi_UNKNOWN">
V</div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="mspace_UNKNOWN" /> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
μ</div> <div class="mi_UNKNOWN">
t</div> <div class="mspace_UNKNOWN" /> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
V</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> <div class="mo_UNKNOWN">
,</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
V</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
F</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
μ</div> <div class="mi_UNKNOWN">
X</div> <div class="mi_UNKNOWN">
V</div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> <div class="mo_UNKNOWN">
/</div> <div class="mi_UNKNOWN">
S</div> </div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
F</div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
μ</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
μ</div> <div class="mi_UNKNOWN">
t</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> <div class="mo_UNKNOWN">
/</div> <div class="mi_UNKNOWN">
S</div> </div> </div> </div> </div> </div> <div class="graphic_UNKNOWN" /> </div> </div> <p /> </div> </div> <div class="optimizationstrategyforthebiogenicsynthesisofnano-mgo" title="sec"> <div class="title" tagx="title" title="title">
Optimization strategy for the biogenic synthesis of nano-MgO</div> <p>In the present study, Robust Design (Taguchi method) steps applied in a stepwise to optimize the biosynthesis of nano-MgO reaction. To yield a valid final product Taguchi method was set up by using several stages: select the significant variables, designing the accurate matrix, data statistically data analysis and finally validation step by using the optimum levels <sup> <a href="#CR23">23</a> </sup> <sup>–</sup> <sup> <a href="#CR25">25</a>, <a href="#CR60">60</a> </sup>. This study aimed to gain the maximum nano-MgO dry weight which produced by using a different concentration of the intracellular metabolites (reducing/stabilizing agent), Mg (NO <sub>3</sub>) <sub>2</sub>.6H <sub>2</sub>O (precursor) and pH at a different temperature. In this work, L32 Taguchi orthogonal array design (2**4) was chosen for this optimization strategy (4 factors, 2 levels, and 32 runs). After conducting calculation of the average of formed nanoMgO dry weight and signal-to-noise (S/N) ratio (the larger-the-better group) for each reaction condition as designed and F test &amp;amp; ANOVA are also carried out by using the MINITAB 16 software to study the significance of all factors and their relations at specific levels. Finally, a confirmation test was conducted to validate the predicted results produced based on Taguchi’s methods approach with the actual experimental value. In this investigations, the S/N ratios (maximize) are calculated using the Eq. <a href="#Equ11">11</a>. The predicted S/N ratio with the optimal level of the design parameters can be calculated by using Eq. <a href="#Equ12">12</a>. </p><div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">11</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M21">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$${\bf{S}}/{\bf{N}}=-10\,\mathrm{log}(\frac{1}{{\bf{n}}})\sum _{{\bf{i}}={\bf{1}}}^{{\bf{n}}}\frac{1}{{{\bf{Y}}}_{{\bf{i}}}^{2}}$$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
/</div> <div class="mi_UNKNOWN">
N</div> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
−</div> <div class="mspace_UNKNOWN" /> <div class="mn_UNKNOWN">
10</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
log</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mi_UNKNOWN">
n</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mi_UNKNOWN">
n</div> </div> <div class="mfrac_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> </div> </div> <div class="graphic_UNKNOWN" /> </div> </div>where <b>n</b> is the number of observations and <b>y</b> is the observed data; the <b>S</b>/ <b>N</b> ratio is expressed using a decibel scale ( <b>dB</b>). <div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">12</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M23">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\frac{{\boldsymbol{S}}}{{{\boldsymbol{N}}}_{{\rm{Predicted}}}\,}=\frac{{\boldsymbol{S}}}{{{\rm{N}}}_{{\rm{m}}}}+\,\sum _{{\boldsymbol{i}}=1}^{{\boldsymbol{n}}}|\frac{{\boldsymbol{S}}}{{\boldsymbol{Ni}}}-\frac{{\boldsymbol{S}}}{{{\rm{N}}}_{{\rm{m}}}}|$$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Predicted</div> </div> </div> <div class="mspace_UNKNOWN" /> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mspace_UNKNOWN" /> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mi_UNKNOWN">
n</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
|</div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Ni</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> </div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
|</div> </div> </div> <div class="graphic_UNKNOWN" /> </div> </div>where <b> <i>S</i> </b> <b>/</b> <b> <i>N</i> </b> <sub> <b>m</b> </sub> is the total mean <b> <i>S</i> </b> <b>/</b> <b> <i>N</i> </b> ratio and <b> <i>S</i> </b> <b>/</b> <b> <i>N</i> </b> <sub> <b>i</b> </sub> is the mean <b> <i>S</i> </b> <b>/</b> <b> <i>N</i> </b> ratio at the optimal level; <b> <i>n</i> </b> is the number of the parameters. <p /> </div> <div class="antibacterialefficiencyofnano-mgo" title="sec"> <div class="title" tagx="title" title="title">
Antibacterial efficiency of nano-MgO</div> <p>Nutrient agar plates were prepared and swabbed with 100 μl of mature broth culture of human pathogenic bacterial strains individually. Firstly, the pathogens sensitivity was tested by using the standard antibiotics <sup> <a href="#CR58">58</a> </sup> (Amikacin 30 mcg, Ampicilin/sulbactam 40 μg, Ampicillin 10 μg, Cefotaxime 30 µg, Cefoxitin 30 µg, Cephradine 30 μg, Chloramphenicol 30 mcg, Erythromycin 15 mcg, Gentamicin 10 mcg, Nitrofurantion 300 mcg, Norfloxacin 10 mcg, Pefloxacin 5 mcg and Penicillin G 10 U) through disc diffusion method <sup> <a href="#CR23">23</a>– <a href="#CR25">25</a>, <a href="#CR43">43</a>, <a href="#CR50">50</a>, <a href="#CR59">59</a>, <a href="#CR60">60</a> </sup>. Secondly, the minimum inhibitory concentration (a concentration which completely inhibited growth) were calculated by using different concentration of pure nano-MgO solution (5, 10, 15, 20, 25, 30, 35, 40, 45, 50 and 55 µg/ml) through well diffusion method. Then the plates were incubated at 37 °C for 24–48 hr, and the inhibition zones measured in millimeter (mm) of the every well or disc. Analysis of variance (ANOVA) was used to determine whether there was a significant difference between replicates and demonstrate either a significant difference ( <i>p</i> &amp;lt; 0.05) or no significant difference ( <i>p</i> &amp;gt; 0.05) for the results obtained. To evaluate the antibacterial activity of 25 µg/ml nano-MgO solution; the comparative study was applied by using 100 mg/ml ampicilin/sulbactam solution as a standard. An inoculum size of test pathogens was added to 50 ml of nutrient broth separately and the bacterial growth adjusted to an OD of McFarland 0.5 (1 × 10 <sup>8</sup> CFU/ml). Subsequently, 50 ml of nutrient broth that supplemented with 25 µg/ml nano-MgO solution and 100 mg/ml ampicilin/sulbactam solution were added to the above flask separately. The negative control bacterial culture was set up without nanoparticles or antibiotic solutions. All the flasks were incubated on rotator shaker (200 rpm) at 37 °C. Eventually, the 1 ml of each sample was collected to determine CFU on nutrient agar plates. Notably, the average number of CFU/ml (log10) was calculated and the final results were analyzed by using Student’s‘t’ test (P-values &amp;lt;5%). Finally, a minimal bactericidal concentration (the lowest nano-MgO concentration that kills 100% of the initial bacterial cells) was calculated by bacterial counting method as described above. </p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="screeningofisolatedendophyticactinomycetesforbiogenicsynthesisofactivenano-mgoandmolecularidentification" title="sec"> <div class="title" tagx="title" title="title">
Screening of isolated endophytic <i>Actinomycetes</i> for biogenic synthesis of active nano-MgO and molecular identification </div> <p>A total of 10 isolates of endophytic actinomycetes were isolated from leaf tissues of <i>Ocimum sanctum</i>. One endophytic isolate out of these ten isolates showed the turbidity change of the cell filtrate after adding the precursor metal salt (1 M Mg(NO <sub>3</sub>) <sub>2</sub>·6H <sub>2</sub>O) as shown in Fig. <a href="#Fig1">1(C,D)</a>. This nano-MgO synthesizers isolate was coded as E72. The formation of nano-MgO was confirmed and its properties studied by using important analysis methods (UV-Vis spectroscopy, SEM, TEM, EDX, XRD, PSA, and FTIR) for identifying and characterizing these nanoparticles. A detailed characterization of the prepared nanoparticles was noted after the completed reduction process (24 hr) by an extinction of surface plasmon resonance (SPR), that observed at 448 nm in UV- vis spectrophotometer that confirmed the synthesis of nano-MgO (Fig. <a href="#Fig1">1A</a>). The average particles size found to be 25 nm that measured by using Eq. ( <a href="#Equ1">1</a>) and confirmed by particle size analyzer at both angles 90° and 11° (Fig. <a href="#Fig1">1B</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <p>( <b>A</b>) UV-visible spectroscopy analysis of the nano-MgO showing SPR peak at 448 nm, ( <b>B</b>) Nano-MgO particle size curve at two angles 11° and 90°, Digital photograph indicates biogenic synthesis nano-MgO reaction by using endophytic <i>S</i>. <i>coelicolor</i> strain E72 extract as reducing agent and Mg(NO <sub>3</sub>) <sub>2</sub>.6H <sub>2</sub>O as a precursor: ( <b>C</b>) Endophytic <i>S</i>. <i>coelicolor</i> strain E72 extract, ( <b>D</b>) fabricated nano-MgO. </p>   </div> <p /> <p>The analysis of SEM and TEM data that obtained from micrograph images confirmed the formation of spherical and ellipsoidal nano-MgO shape (Fig. <a href="#Fig2">2</a>). The EDX spectrum that showed a strong metal signal peak of magnesium (Mg) and other peaks of oxygen (O <sub>2</sub>) and a signal of carbon (C) also confirmed the synthesis of nano-MgO (Fig. <a href="#Fig3">3A</a>). The crystalline nature of the nano-MgO was clarified with X-ray diffraction analysis (Fig. <a href="#Fig3">3B</a>). From the XRD pattern, five distinct diffraction peaks were absorbed at 36°, 52°, 62°, 72° and 82° along with miller indices values (111), (200), (220), (311) and (222), respectively and no peak of other phases were found in the X-ray spectrum, arising from an impurity was observed that indicating the nano-MgO obtained through this safe biogenic synthesis method possesses an ultra-pure crystalline phase. This was in agreement with the standard diffraction pattern of a pure cubic phase of nano-MgO which is consistent with bulk MgO crystal (JCPDS Card No. 77–2364, MgO). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <p>( <b>A</b>) SEM analysis of the nano-MgO that bio-fabricated by using endophytic <i>S</i>. <i>coelicolor</i> strain E72 extract with scale 10 μm. ( <b>B</b>) and ( <b>C</b>) TEM images with scales 200 nm and 50 nm respectively. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <p>( <b>A</b>) Energy dispersive X-ray analysis of the nano-MgO that bio-fabricated by using endophytic <i>S</i>. <i>coelicolor</i> strain E72 extract and ( <b>B</b>) XRD spectrum. </p>   </div> <p /> <p>To identify the structure of the bonding chemical compound in intracellular fraction before biogenic synthesis of nano-MgO and after this process, the FTIR was measured for the nan-MgO spectrum (capping and stabilizing biomolecules) and the intracellular fraction spectrum (reducing biomolecules) as shown in Fig. <a href="#Fig4">4(A,B)</a>. Firstly, the important bands at 439–879 cm <sup>−1</sup> that observed in the nan-MgO spectrum assigned to the metal–oxygen bonds stretching vibration, which confirm the formation of nano-MgO which agreed well with the UV, EDX and X-ray spectra. Also the intensive peaks were observed in the same spectrum at 1410.84 &amp;amp; 1066.67 cm <sup>−1</sup> corresponding to the C-N stretching vibration of aliphatic amino acid, 2899 &amp;amp; 2964 cm <sup>−1</sup> corresponding to C–H stretching of methylene groups of the protein and 3348 cm <sup>−1</sup> corresponding to N–H stretching of the secondary amide of the protein. In the intracellular spectrum, the intensive peaks of the sample at 3410.26 cm <sup>−1</sup> corresponding to the C-N stretching vibration of the amide I &amp;amp; amide II bands of the proteins, 1641.48 cm <sup>−1</sup> corresponding to the stretching vibrations of C=C or the O–H bending mode and 1458.2 cm <sup>−1</sup> corresponding to the C-N stretching vibration of aromatic and aliphatic amino acid that associated with vibrational mode of H <sup>−</sup> ion bonded to Mg <sup>2+</sup> on co-ordination sites. These results confirm that the presence of protein which might have an important role in the biogenic synthesis of nano-MgO. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <p>FTIR spectrum of the nano-MgO that bio-fabricated by using endophytic <i>S</i>. <i>coelicolor</i> strain E72 extract ( <b>A</b>) and its respective intracellular fraction extract ( <b>B</b>) in the range of 500–4000 cm <sup>−1</sup>. </p>   </div> <p /> <p>In addition to the morphological characterization, the molecular identification was also carried out to confirm the identification of the promising endophytic <i>Actinomycetes</i> isolate by using 16 S rRNA gene. The Sanger’s dideoxy nucleotide sequencing of amplified 16 S rRNA gene resulted in 1502 bp nucleotide sequence. The Blastn analyses, pair wise and multiple sequence alignment revealed 98–100% identity with the sequences of <i>Streptomyces coelicolor</i> strains and is designated as <i>Streptomyces coelicolor</i> strain E72 and has been deposited in NCBI GenBank (Accession Number MF429778). Multiple sequence alignment was carried out using ClustalW2 with default parameters. Phylogenetic tree was constructed by the neighbor-joining (NJ) method with nucleotide pairwise genetic distance as shown in Fig. <a href="#Fig5">5</a>. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 5</span> </a> </div>   <p>Phylogenetic tree based on 16 S rDNA gene sequence analysis, showing the relationship of endophytic <i>S</i>. <i>coelicolor</i> strain E72 <b>(</b>MF429778) with reference strains (NCBI GenBank) which constructed by using the neighbor-joining method with the aid of MEGA 6.0 program. Sequence divergence is indicated by the scale bar. </p>   </div> <p /> <p>The antimicrobial activity of the prepared nano-MgO and some conventional antibiotics were screened and assessed against the tested clinical pathogens as shown in Table ( <a href="#Tab1">1</a>). The tested human pathogens showed varied sensitivity efficiency against tested antibiotics as well as against prepared nano-MgO. So these results concluded that, nano-MgO showed broad spectrum antibacterial activity against tested multidrug resistant human pathogens. <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  </div></p><p>Antimicrobial activity of some conventional antibiotics and the prepared MgO nano-powder that fabricated from endophytic Streptomyces sp extract against Gram-positive and Gram- negative human pathogenic bacteria. </p>  <table> <thead> <tr> <th>Antimicrobial agents</th> <th>Concentration</th> <th>Human pathogenic bacteria</th> </tr> <tr> <th>Gram-negative</th> <th>Gram-positive</th> </tr> <tr> <th> <i>Escherichia Coli</i> </th> <th> <i>Proteus vulgaris</i> </th> <th> <i>Salmonella typhimurium</i> </th> <th> <i>Shigella flexneri</i> </th> <th> <i>Klebsiella pneumonia</i> </th> <th> <i>Staphylococcus aureus</i> </th> <th> <i>Streptococcus pneumonia</i> </th> <th> <i>Bacillus cereus</i> </th> </tr> </thead> <tbody> <tr> <td>Prepared nano-MgO</td> <td>100 mg/ml</td> <td>S</td> <td>R</td> <td>S</td> <td>S</td> <td>S</td> <td>R</td> <td>S</td> <td>S</td> </tr> <tr> <td>Amikacin</td> <td>30 mcg</td> <td>R</td> <td>S</td> <td>R</td> <td>R</td> <td>R</td> <td>S</td> <td>S</td> <td>R</td> </tr> <tr> <td>Ampicilin/sulbactam</td> <td>40 mg</td> <td>R</td> <td>R</td> <td>R</td> <td>R</td> <td>R</td> <td>R</td> <td>R</td> <td>R</td> </tr> <tr> <td>Ampicillin</td> <td>10 μg</td> <td>S</td> <td>R</td> <td>R</td> <td>R</td> <td>R</td> <td>R</td> <td>S</td> <td>R</td> </tr> <tr> <td>Cefotaxime</td> <td>30 µg</td> <td>S</td> <td>R</td> <td>R</td> <td>R</td> <td>R</td> <td>R</td> <td>S</td> <td>R</td> </tr> <tr> <td>Cefoxitin</td> <td>30 µg</td> <td>R</td> <td>R</td> <td>S</td> <td>R</td> <td>R</td> <td>R</td> <td>R</td> <td>R</td> </tr> <tr> <td>Cephradine</td> <td>30 μg</td> <td>R</td> <td>R</td> <td>S</td> <td>R</td> <td>S</td> <td>R</td> <td>R</td> <td>S</td> </tr> <tr> <td>Chloramphenicol</td> <td>30 mcg</td> <td>R</td> <td>R</td> <td>S</td> <td>R</td> <td>S</td> <td>R</td> <td>R</td> <td>R</td> </tr> <tr> <td>Erythromycin</td> <td>15 mcg</td> <td>S</td> <td>R</td> <td>R</td> <td>R</td> <td>S</td> <td>R</td> <td>R</td> <td>R</td> </tr> <tr> <td>Gentamicin</td> <td>10 mcg</td> <td>S</td> <td>S</td> <td>R</td> <td>R</td> <td>R</td> <td>S</td> <td>R</td> <td>R</td> </tr> <tr> <td>Nitrofurantion</td> <td>300 mcg</td> <td>S</td> <td>S</td> <td>R</td> <td>R</td> <td>R</td> <td>S</td> <td>S</td> <td>S</td> </tr> <tr> <td>Norfloxacin</td> <td>10 mcg</td> <td>R</td> <td>R</td> <td>S</td> <td>R</td> <td>R</td> <td>S</td> <td>R</td> <td>R</td> </tr> <tr> <td>Pefloxacin</td> <td>5 mcg</td> <td>R</td> <td>R</td> <td>R</td> <td>R</td> <td>R</td> <td>R</td> <td>S</td> <td>R</td> </tr> <tr> <td>Penicillin</td> <td>10 U</td> <td>R</td> <td>R</td> <td>S</td> <td>R</td> <td>S</td> <td>R</td> <td>R</td> <td>R</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p> <b>R</b>: Resistant and <b>S</b>: Sensitive. </p> </div>  <p /> </div> <div class="screeningofthemostpotentbioactivemetabolitesforbiogenicsynthesisofnano-mgo" title="sec"> <div class="title" tagx="title" title="title">
Screening of the most potent bioactive metabolites for biogenic synthesis of nano-MgO</div> <p>The modern biotechnological strategies applied in all the following experiments to select the suitable and cheaper cultivation medium that produce the highest <i>Streptomyces coelicolor</i> strain E72 mass production and also screening the most potent extracted metabolites that used for biogenic synthesis of nano-MgO. Since, the growth medium considered as one of the most factors that affecting cell density and hence bioactive compounds production during an industrial fermentation process that finally improving nano-MgO biosynthesis. So in the present report, the screening studies including two parameters, the kind of the best media (organic, inorganic and mixed media) and location (extracellular and intracellular fractions) of the active metabolites (nitrate reductase, total protein, and total carbohydrate) that produce the highest nano-MgO dry mass weight. Among the tested media, the Modified Czapek-Dox medium ( <b>M1</b>) produced significantly higher 2.5 g/l of biomass dry weight and 10 g/l of nano-MgO dry mass weight that recorded by using intracellular fraction. The highest quantitative measurements of these metabolites were detected in intracellular fraction since the highly carbohydrate concentration (7.2 μg/μl), nitrate reductase (5.2 nmol/hr/ml) and protein concentration (6.3 g/l) were recorded as shown in Fig. <a href="#Fig6">6</a>. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 6</span> </a> </div>   <p>The location of the highest bioactive metabolites (total carbohydrates, total protein and nitrate reductase) and the highest dry weight of nano-MgO that biosynthesized by using the extracellular, and intracellular fractions extracted from endophytic <i>S</i>. <i>coelicolor</i> strain E72 that cultivated in different five media <b>(M1</b>, <b>M2</b>, <b>M3</b>, <b>M4 and M5)</b>. </p>   </div> <p /> </div> <div class="optimizationstrategiesforthehigheststreptomycescoelicolorstraine72biomassproduction" title="sec"> <div class="title" tagx="title" title="title">
Optimization strategies for the highest <i>Streptomyces coelicolor</i> strain E72 biomass production </div> <p>The microbial cultural requirement (medium components and culturing conditions) for high microbial mass weight and also bioactive compounds production is one of the major cost-effective problem facing industrial producers. So in these investigations statistical-mathematical designs such as Plackett-Burman design (PBD) that followed by Box-Behnken designs (BBD) were used for optimization of the industrial medium components to reduce biomass production costs and hence the prepared nanoparticles production cost by producing the promising bioactive compounds. Firstly, screening process was carried out by using Plackett-Burman design (PBD) to determine which variables significantly affected microbial biomass production based on the 1 <sup>st</sup> order model equation <sup> <a href="#CR23">23</a>– <a href="#CR25">25</a> </sup>. Secondly, response surface methodology was used to study the combined effect among these significant variables and also calculate their optimal levels through Box-Behnken designs (BBD) process based on second-order polynomial equation <sup> <a href="#CR23">23</a>– <a href="#CR25">25</a>, <a href="#CR55">55</a> </sup>. </p> <div class="plackett-burmandesign(pbd)" title="sec"> <div class="title" tagx="title" title="title">
Plackett-Burman design (PBD)</div> <p>Recently, application of the statistical Plackett-Burman design as a very effective screening tool for optimization of the microbial biomass production reported and discussed in many publications <sup> <a href="#CR23">23</a>– <a href="#CR25">25</a>, <a href="#CR55">55</a> </sup>. In this report, the Plackett-Burman design experiments were conducted in 8 runs to select the significant variables that affected on the final biomass dry weight. Wide variations of biomass production that reflected the importance of the statistical medium optimization step to attain higher biomass production were shown in Table ( <a href="#Tab2">2</a>). The maximum biomass production (11.05 g/l) was achieved in the run number 2, while the minimum value (3.52 g/l) was observed in the run number 8. The relationship between an independent variables and biomass production is determined by using multiple-regression model and the % confidence level were calculated (Table <a href="#Tab3">3</a>). The main effects of the examined factors on the biomass production were calculated and the normal probability plot of the residuals represented graphically in Fig. <a href="#Fig7">7</a>. The <i>P-</i>value from the ANOVA analysis for each variable was determined to analyze the relationship between the variables at 95% or higher confidence level (Fig. <a href="#Fig7">7</a> <b>)</b>. The final results revealed that, there is a statistically significant relationship among the tested variables at 99% confidence level. These results indicated that four variables from the seven different independent variable named sucrose, (NH <sub>4</sub>) <sub>2</sub>SO <sub>4</sub>, KCl, and K <sub>2</sub>HPO <sub>4</sub> affected positively on biomass production, where the three variables named NaNO <sub>3</sub>, MgSO <sub>4</sub>·7H <sub>2</sub>O, and FeSO <sub>4</sub>·7H <sub>2</sub>O affected negatively. The predicted optimum levels of the independent variables were verified and compared with the basal medium setting. The high correlation between the predicted and observed values indicated the validity of the statistical design. The final results indicated that the biomass production can be increased more than 4times when eliminating NaNO <sub>3</sub> and KCl from the optimized medium. The final composition of optimized medium consisted of (g/l) 40.0 Sucrose, 4.0 (NH <sub>4</sub>) <sub>2</sub>SO <sub>4</sub>, 3.0 K <sub>2</sub>HPO <sub>4</sub>, 0.5 MgSO <sub>4</sub>·7H <sub>2</sub>O, and 0.01 FeSO <sub>4</sub>·7H <sub>2</sub>O. Finally, the significant independent variables named Sucrose (F <sub>1</sub>), (NH <sub>4</sub>) <sub>2</sub>SO <sub>4</sub> (F <sub>2</sub>) and K <sub>2</sub>HPO <sub>4</sub> (F <sub>3</sub>) were used for further optimization step by using a Box-Behnken designs (BBD). <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 2</span>  </div></p><p>Eight-trial of Plackett–Burman experimental design used for evaluation of seven independent variables along with the endophytic <i>S</i>. <i>coelicolor</i> strain E72 biomass production. </p>  <table> <thead> <tr> <th>Trail</th> <th>Variables (g/l)</th> <th>Biomass g/l</th> <th>Predicted Biomass g/l</th> </tr> <tr> <th>Sucrose</th> <th>(NH <sub>4</sub>) <sub>2</sub>SO <sub>4</sub> </th> <th>NaNO <sub>3</sub> </th> <th>KCl</th> <th>K <sub>2</sub>HPO <sub>4</sub> </th> <th>MgSO <sub>4</sub>·7H <sub>2</sub>O </th> <th>FeSO <sub>4</sub>7H <sub>2</sub>O </th> </tr> </thead> <tbody> <tr> <td>1</td> <td>40</td> <td>0</td> <td>0</td> <td>2</td> <td>1</td> <td>1</td> <td>0.5</td> <td>4.562</td> <td>4.623</td> </tr> <tr> <td>2</td> <td>40</td> <td>4</td> <td>0</td> <td>0.5</td> <td>3</td> <td>0.5</td> <td>0.5</td> <td>11.046</td> <td>11.053</td> </tr> <tr> <td>3</td> <td>40</td> <td>4</td> <td>2</td> <td>0.5</td> <td>1</td> <td>1</td> <td>0.01</td> <td>6.564</td> <td>6.623</td> </tr> <tr> <td>4</td> <td>15</td> <td>4</td> <td>2</td> <td>2</td> <td>1</td> <td>0.5</td> <td>0.5</td> <td>3.765</td> <td>3.823</td> </tr> <tr> <td>5</td> <td>40</td> <td>0</td> <td>2</td> <td>2</td> <td>3</td> <td>0.5</td> <td>0.01</td> <td>8.758</td> <td>8.823</td> </tr> <tr> <td>6</td> <td>15</td> <td>4</td> <td>0</td> <td>2</td> <td>3</td> <td>1</td> <td>0.01</td> <td>8.451</td> <td>8.523</td> </tr> <tr> <td>7</td> <td>15</td> <td>0</td> <td>2</td> <td>0.5</td> <td>3</td> <td>1</td> <td>0.5</td> <td>4.463</td> <td>4.523</td> </tr> <tr> <td>8</td> <td>15</td> <td>0</td> <td>0</td> <td>0.5</td> <td>1</td> <td>0.5</td> <td>0.01</td> <td>3.465</td> <td>3.523</td> </tr> </tbody> </table>  <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 3</span>  <p>Regression statistics, analysis of variance (ANOVA) for Plackett-Berman experimental design results used for optimizing the endophytic <i>S</i>. <i>coelicolor</i> strain E72 biomass production. </p>  <table> <thead> <tr> <th>Variables</th> <th>Coefficients</th> <th>Standard Error</th> <th>t Stat</th> <th>P-value</th> <th>Confidence level (%)</th> </tr> </thead> <tbody> <tr> <td>Sucrose</td> <td>1.34125</td> <td>0.025251575</td> <td>53.115499</td> <td>1.47E-05*</td> <td>99.999</td> </tr> <tr> <td>(NH <sub>4</sub>) <sub>2</sub>SO <sub>4</sub> </td> <td>1.06625</td> <td>0.025251575</td> <td>42.225089</td> <td>2.92E-05*</td> <td>99.997</td> </tr> <tr> <td>NaNO <sub>3</sub> </td> <td>−0.49125</td> <td>0.025251575</td> <td>−19.454232</td> <td>0.000297*</td> <td>99.970</td> </tr> <tr> <td>KCl</td> <td>0.00875</td> <td>0.025251575</td> <td>0.346513</td> <td>0.751837</td> <td>24.816</td> </tr> <tr> <td>K <sub>2</sub>HPO <sub>4</sub> </td> <td>1.79125</td> <td>0.025251575</td> <td>70.936169</td> <td>6.17E-06*</td> <td>99.999</td> </tr> <tr> <td>MgSO <sub>4</sub>·7H <sub>2</sub>O </td> <td>−0.36625</td> <td>0.025251575</td> <td>−14.504046</td> <td>0.000711*</td> <td>99.929</td> </tr> <tr> <td>FeSO <sub>4</sub>7H <sub>2</sub>O </td> <td>−0.43375</td> <td>0.025251575</td> <td>−17.177146</td> <td>0.00043*</td> <td>99.957</td> </tr> <tr> <td /> <td> <b>df</b> </td> <td> <b>SS</b> </td> <td> <b>MS</b> </td> <td> <b>F</b> </td> <td> <b>Significance F</b> </td> </tr> <tr> <td>Regression</td> <td>7</td> <td>53.6647875</td> <td>7.6663982</td> <td>1502.88</td> <td>2.61E-05</td> </tr> <tr> <td>Residual</td> <td>3</td> <td>0.015303409</td> <td>0.0051011</td> <td /> <td /> </tr> <tr> <td>Total</td> <td>10</td> <td>53.68009091</td> <td /> <td /> <td /> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p> <b> <i>Multiple R</i> </b>, <i>0</i>. <i>99</i>, <b> <i>R Square</i> </b>, <i>0</i>. <i>99</i>, <b> <i>Adjusted R Square</i> </b>, <i>0</i>. <i>99</i>, <b> <i>Standard Error</i> </b>, <i>0</i>. <i>07</i>, *Significant values, <b>df</b>: Degree of freedom, <b>SS</b> - sum of squares, <b>MS</b>- mean square, <b>F</b>: Fishers’s function, <b>P</b>: Level of significance. </p> </div> </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 7</span> </a> </div>   <p>( <b>A</b>) Column chart shown the main effect of the variables affecting biomass production of endophytic <i>S</i>. <i>coelicolor</i> strain E72 according to the results of Plackett-Burman design. ( <b>B</b>) The Pareto chart represents the <i>P</i>-value and confidence level of different variables, ( <b>C</b>) The normal probability plot of the residuals. </p>   </div> <p /> </div> <div class="box-behnkendesigns(bbd)" title="sec"> <div class="title" tagx="title" title="title">
Box-Behnken designs (BBD)</div> <p>In this study, a total of 15 experiments according to a BBD experimental matrix and the final biomass production were performed and summarized in Table ( <a href="#Tab4">4</a>). The adequacy of the model was checked using ANOVA which was tested using Fisher’s statistical analysis as shown in Table ( <a href="#Tab5">5</a>). The Model <i>F</i>-value of 30.743 with a very low probability value ( <i>P</i> model &amp;gt;  <i>F</i> 0.00074) implies the model is significant. The analyzed data revealed that the positive coefficients for some variables and interaction effects increased final biomass production, while other negative coefficients decreased it. By using the degree of significance the analyzed results revealed that the quadratic effect of Sucrose ( <b>F</b> <sub> <b>1</b> </sub>), (NH <sub>4</sub>) <sub>2</sub>SO <sub>4</sub> ( <b>F</b> <sub> <b>2</b> </sub>) and K <sub>2</sub>HPO <sub>4</sub> ( <b>F</b> <sub> <b>3</b> </sub>) are significant model terms ( <i>P</i> value &amp;gt; 0.05) as shown in Fig. <a href="#Fig8">8</a>. Linear coefficients and interaction between three variables are significant ( <i>P</i> value &amp;gt; 0.05) indicating that there is a significant correlation between three variables that play a major role in increasing the biomass production. In order to evaluate the relationship between dependent and independent variables and to determine the highest biomass production corresponding to the optimum levels of the three tested variables; a second-order polynomial model was proposed to calculate the optimum levels of these variables and defines predicted response ( <b>Y</b>) in terms of the independent variables: </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M25">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\begin{array}{rcl}{{\bf{Y}}}_{({\rm{biomass}}{\rm{production}})} &amp;amp; = &amp;amp; 17.46+{\bf{1}}.67\ast {{\bf{F}}}_{{\bf{1}}}-{\bf{2}}.67\ast {{\bf{F}}}_{{\bf{2}}}-{\bf{0}}.{\bf{8}}\ast {{\bf{F}}}_{{\bf{3}}}\\ &amp;amp; &amp;amp; -\,{\bf{3}}.05\ast {{\bf{F}}}_{{\bf{1}}}{{\bf{F}}}_{{\bf{1}}}-{\bf{2}}.90\ast {{\bf{F}}}_{{\bf{2}}}{{\bf{F}}}_{{\bf{2}}}-{\bf{2}}.56\ast {{\bf{F}}}_{{\bf{3}}}{{\bf{F}}}_{{\bf{3}}}\\ &amp;amp; &amp;amp; -\,{\bf{2}}.50\ast {{\bf{F}}}_{{\bf{1}}}{{\bf{F}}}_{{\bf{2}}}-{\bf{1}}.15\ast {{\bf{F}}}_{{\bf{1}}}{{\bf{F}}}_{{\bf{3}}}+{\bf{0}}.86\ast {{\bf{F}}}_{{\bf{2}}}{{\bf{F}}}_{{\bf{3}}}\end{array}$$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
biomass</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
production</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> </div> <div class="mtd_UNKNOWN"> <div class="mo_UNKNOWN">
=</div> </div> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
17</div> <div class="mo_UNKNOWN">
.</div> <div class="mn_UNKNOWN">
46</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
.</div> <div class="mn_UNKNOWN">
67</div> <div class="mo_UNKNOWN">
∗</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
.</div> <div class="mn_UNKNOWN">
67</div> <div class="mo_UNKNOWN">
∗</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
0</div> <div class="mo_UNKNOWN">
.</div> <div class="mn_UNKNOWN">
8</div> <div class="mo_UNKNOWN">
∗</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
3</div> </div> </div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN" /> <div class="mtd_UNKNOWN" /> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mspace_UNKNOWN" /> <div class="mn_UNKNOWN">
3</div> <div class="mo_UNKNOWN">
.</div> <div class="mn_UNKNOWN">
05</div> <div class="mo_UNKNOWN">
∗</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
.</div> <div class="mn_UNKNOWN">
90</div> <div class="mo_UNKNOWN">
∗</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
.</div> <div class="mn_UNKNOWN">
56</div> <div class="mo_UNKNOWN">
∗</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
3</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
3</div> </div> </div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN" /> <div class="mtd_UNKNOWN" /> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mspace_UNKNOWN" /> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
.</div> <div class="mn_UNKNOWN">
50</div> <div class="mo_UNKNOWN">
∗</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
.</div> <div class="mn_UNKNOWN">
15</div> <div class="mo_UNKNOWN">
∗</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
3</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
0</div> <div class="mo_UNKNOWN">
.</div> <div class="mn_UNKNOWN">
86</div> <div class="mo_UNKNOWN">
∗</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
3</div> </div> </div> </div> </div> </div> </div> </div> <div class="graphic_UNKNOWN" /> </div> </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 4</span>  <p>Box–Behnken experimental design matrix of three variables with actual factor levels corresponding to coded factor levels, actual values, and the mean of <i>S</i>. <i>coelicolor</i> strain E72 biomass production. </p>  <table> <thead> <tr> <th>Trial No</th> <th>Variables (g/l)</th> <th>Biomass (g/l)</th> <th>Predicted Biomass (g/l)</th> </tr> <tr> <th> <b>Sucrose (F</b> <sub> <b>1</b> </sub> <b>)</b> </th> <th> <b>(NH</b> <sub> <b>4</b> </sub> <b>)</b> <sub> <b>2</b> </sub> <b>SO</b> <sub> <b>4</b> </sub> <b>(F</b> <sub> <b>2</b> </sub> <b>)</b> </th> <th> <b>K</b> <sub> <b>2</b> </sub> <b>HPO</b> <sub> <b>4</b> </sub> <b>(F</b> <sub> <b>3</b> </sub> <b>)</b> </th> </tr> </thead> <tbody> <tr> <td> <b>1</b> </td> <td>0(50)</td> <td>0(5)</td> <td>0(4)</td> <td>17.3</td> <td>17.47</td> </tr> <tr> <td> <b>2</b> </td> <td>−1(40)</td> <td>−1(4)</td> <td>0(4)</td> <td>10.4</td> <td>10.00</td> </tr> <tr> <td> <b>3</b> </td> <td>0(50)</td> <td>1(6)</td> <td>1(5)</td> <td>9.6</td> <td>9.39</td> </tr> <tr> <td> <b>4</b> </td> <td>0(50)</td> <td>−1(4)</td> <td>1(5)</td> <td>12</td> <td>13.01</td> </tr> <tr> <td> <b>5</b> </td> <td>0(50)</td> <td>0(5)</td> <td>0(4)</td> <td>17.2</td> <td>17.47</td> </tr> <tr> <td> <b>6</b> </td> <td>1(60)</td> <td>1(6)</td> <td>0(4)</td> <td>7.6</td> <td>8.00</td> </tr> <tr> <td> <b>7</b> </td> <td>1(60)</td> <td>0(5)</td> <td>1(5)</td> <td>11.9</td> <td>11.57</td> </tr> <tr> <td> <b>8</b> </td> <td>1(60)</td> <td>0(5)</td> <td>−1(3)</td> <td>15</td> <td>15.47</td> </tr> <tr> <td> <b>9</b> </td> <td>−1(40)</td> <td>1(6)</td> <td>0(4)</td> <td>10</td> <td>10.54</td> </tr> <tr> <td> <b>10</b> </td> <td>−1(40)</td> <td>0(5)</td> <td>1(5)</td> <td>11</td> <td>10.53</td> </tr> <tr> <td> <b>11</b> </td> <td>0(50)</td> <td>1(6)</td> <td>−1(3)</td> <td>10</td> <td>9.26</td> </tr> <tr> <td> <b>12</b> </td> <td>0(50)</td> <td>−1(4)</td> <td>−1(3)</td> <td>16.4</td> <td>16.34</td> </tr> <tr> <td> <b>13</b> </td> <td>−1(40)</td> <td>0(5)</td> <td>−1(3)</td> <td>9.5</td> <td>9.83</td> </tr> <tr> <td> <b>14</b> </td> <td>0(50)</td> <td>0(5)</td> <td>0(4)</td> <td>17.9</td> <td>17.47</td> </tr> <tr> <td> <b>15</b> </td> <td>1(60)</td> <td>−1(4)</td> <td>0(4)</td> <td>18.9</td> <td>18.36</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 5</span>  <p>Regression statistics, analysis of variance (ANOVA) for Box−Behnken experimental design results used for optimizing the <i>S</i>. <i>coelicolor</i> strain E72 biomass production. </p>  <table> <thead> <tr> <th>Variables</th> <th>Coefficients</th> <th>Standard Error</th> <th>t Stat</th> <th> <i>P</i>-value </th> <th>Confidences level %</th> </tr> </thead> <tbody> <tr> <td>F <sub>1</sub> </td> <td>1.670588235</td> <td>0.29803</td> <td>5.60543</td> <td>0.002498*</td> <td>99.75</td> </tr> <tr> <td>F <sub>2</sub> </td> <td>−2.670588235</td> <td>0.29803</td> <td>−8.96079</td> <td>0.000289*</td> <td>99.97</td> </tr> <tr> <td>F <sub>3</sub> </td> <td>−0.8</td> <td>0.293742</td> <td>−2.72348</td> <td>0.041602</td> <td>95.84</td> </tr> <tr> <td>F <sub>1</sub>F <sub>1</sub> </td> <td>−3.053921569</td> <td>0.435301</td> <td>−7.01565</td> <td>0.000907*</td> <td>99.91</td> </tr> <tr> <td>F <sub>2</sub>F <sub>2</sub> </td> <td>−2.903921569</td> <td>0.435301</td> <td>−6.67106</td> <td>0.001143*</td> <td>99.89</td> </tr> <tr> <td>F <sub>3</sub>F <sub>3</sub> </td> <td>−2.562745098</td> <td>0.435301</td> <td>−5.88729</td> <td>0.002009*</td> <td>99.80</td> </tr> <tr> <td>F <sub>1</sub> F <sub>2</sub> </td> <td>−2.508823529</td> <td>0.427457</td> <td>−5.86918</td> <td>0.002037*</td> <td>99.80</td> </tr> <tr> <td>F <sub>1</sub> F <sub>3</sub> </td> <td>−1.15</td> <td>0.415414</td> <td>−2.76832</td> <td>0.039444*</td> <td>96.06</td> </tr> <tr> <td>F <sub>2</sub> F <sub>3</sub> </td> <td>0.864705882</td> <td>0.403011</td> <td>2.145616</td> <td>0.084713</td> <td>91.53</td> </tr> <tr> <td /> <td> <b>df</b> </td> <td> <b>SS</b> </td> <td> <b>MS</b> </td> <td> <b>F</b> </td> <td> <b>Significance F</b> </td> </tr> <tr> <td>Regression</td> <td>9</td> <td>190.9926</td> <td>21.2214</td> <td>30.743</td> <td>0.00074</td> </tr> <tr> <td>Residual</td> <td>5</td> <td>3.451373</td> <td>0.690275</td> <td /> <td /> </tr> <tr> <td>Total</td> <td>14</td> <td>194.444</td> <td /> <td /> <td /> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p> <b> <i>Multiple R</i> </b>, <b> <i>0</i> </b>. <b> <i>99</i> </b>, <b> <i>R Square</i> </b>, <b> <i>0</i> </b>. <b> <i>98</i> </b>, <b> <i>Adjusted R Square</i> </b> <b>, 0.95</b>, <b> <i>Standard Error</i> </b>, <b> <i>0</i> </b>. <b> <i>83</i> </b>, *Significant values, <b>df</b>: Degree of freedom, <b>SS</b>- sum of squares, <b>MS</b>- mean square. <b>F</b>: Fishers’s function, <b>P</b>: Level of significance. </p> </div> </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 8</span> </a> </div>   <p>( <b>A</b>) Pareto chart represent the <i>P</i>-value and confidence level of different variables affecting biomass production of endophytic <i>S</i>. <i>coelicolor</i> strain E72 according to the results of Box-Behnken design. ( <b>B</b>) The normal probability plot of the residuals. </p>   </div> <p /> <p>The predicted optimal levels of the variables for highest biomass production were calculated and the high-yield medium formulation consisted of (g/l) 48.2 Sucrose, 4.8 (NH <sub>4</sub>) <sub>2</sub>SO <sub>4</sub>, 3.5 K <sub>2</sub>HPO <sub>4</sub>, 0.5 MgSO <sub>4</sub>·7H <sub>2</sub>O, and 0.01 FeSO <sub>4</sub>·7H <sub>2</sub>O. The biomass yield that obtained from the experiment (18.5 g/l) was very close to the predicted one (18.9 g/l) by using the regression model, which proved the validity of the model. The verification revealed a high degree of accuracy of the model of 97.8%, indicating the model validation under the tested medium composition. </p> </div> </div> <div class="scalingupfermentationstrategiesforendophyticstreptomycescoelicolorstraine72" title="sec"> <div class="title" tagx="title" title="title">
Scaling up fermentation strategies for endophytic <i>Streptomyces coelicolor</i> strain E72 </div> <p>In a batch fermentation system(constant operation), all necessary medium components and the microbial inoculum are added at the beginning by using basic controls for pH, temperature, dissolved oxygen, and foam, and the product/s is harvested at the end of this batch run. In fed-batch fermentation system (dynamic operation), the nutrients (carbon source/inducers or whole nutrients) necessary for microbial growth and product formation are added by using one or more feeding system during the operational period and finally the biomass product or the desired product is harvested usually at the end of the operational run <sup> <a href="#CR23">23</a>– <a href="#CR25">25</a> </sup>. Large scale production of the endophytic <i>Streptomyces coelicolor</i> strain E72 biomass is a very important step for large scale biosynthesis of nano-MgO system, so in this work scaling up strategy for biomass production and hence nano-MgO production was applied and the growth kinetics studied intensively by using batch and fed-batch operation systems in both of them shake-flask and stirred tank bioreactor (BIOFLO® 310). The behavior of microbial biomass growth can be calculated and described kinetically by using different cultivation mode systems many parameters such as <b> <i>X</i> </b> <sub> <b> <i>max-Biomass</i> </b> </sub>, maximal biomass dry weight; <b> <i>X</i> </b> <sub> <b> <i>max-time</i> </b> </sub>, time of maximal biomass dry weight; <b> <i>dx/dt</i> </b>, cell growth rate; <b> <i>μ</i> </b> <sub> <b> <i>max</i> </b> </sub>, maximal specific growth rate; <b> <i>P</i> </b> <sub> <b> <i>max-vol</i> </b> </sub>, maximal volumetric nano-MgO production; <b> <i>P</i> </b> <sub> <b> <i>max-time</i> </b> </sub>, time of maximal nano-MgO production; <b> <i>Y</i> </b> <sub> <b> <i>x/s</i> </b> </sub>, biomass/glucose consumed; <b> <i>Y</i> </b> <sub> <b> <i>p/x</i> </b> </sub>, nano-MgO/biomass; and finally, <b> <i>Y</i> </b> <sub> <b> <i>p/s</i> </b> </sub>, nano-MgO/glucose consumed. </p> <div class="batchcultivationin1 lshake-flask(sf)" title="sec"> <div class="title" tagx="title" title="title">
Batch cultivation in 1 L shake-flask (SF)</div> <p>The cultivation in SF was used to detect the time required for the complete consumption of glucose and attainment of the highest biomass &amp;amp; nano-MgO concentration by studying the growth kinetics. The endophytic <i>Streptomyces coelicolor</i> strain E72 pre-culture was inculcated in 1 L shake flask (working volume 500 ml). As shown in Fig. <a href="#Fig9">9A</a>, an initial glucose concentration was used gradually and consumed after 84 hr (0.14 g/l). By using kinetics measurements for produced microbial cells, the highest biomass ( <b> <i>X</i> </b> <sub> <b> <i>max-Biomass</i> </b> </sub>) and nano-MgO ( <b> <i>P</i> </b> <sub> <b> <i>max-vol</i> </b> </sub>) production were 24.65 and 35.95 g/l respectively after 118 hr as shown in Table ( <a href="#Tab6">6</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 9</span> </a> </div>   <p>( <b>A</b>) Real time monitoring of growth patterns, glucose consumption and biosynthesized nano-MgO for endophytic <i>S</i>. <i>coelicolor</i> strain E72 that cultivated in 1 L shake flask through batch fermentation mode, ( <b>B</b>) Real time monitoring of growth patterns, glucose consumption, feeding rate, and biosynthesized nano-MgO for endophytic <i>S</i>. <i>coelicolor</i> strain E72 that cultivated in 7 L stirred tank bioreactor through exponential pulses fed-batch fermentation mode. </p>   </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 6</span>  <p>Growth conditions and Kinetic parameters for biomass producton, nano-MgO production, and glucose consumption during the endophytic <i>S</i>. <i>coelicolor</i> strain E72 cultivation in different cultivation scales and mode of operation in shake flask and in stirred tank bioreactor. </p>  <table> <thead> <tr> <th>Parameters</th> <th>Shake flask</th> <th>Batch fermentation</th> <th>Fed- Batch fermentation</th> </tr> </thead> <tbody> <tr> <td> <b>Growth conditions</b> </td> </tr> <tr> <td>Container volume (ml)</td> <td>1000</td> <td>7000</td> <td>7000</td> </tr> <tr> <td>Working Volume (ml)</td> <td>500</td> <td>2500</td> <td>5500</td> </tr> <tr> <td>pH</td> <td>Un-controlled</td> <td>6.9</td> </tr> <tr> <td>Aeration (v/v)</td> <td>Un-controlled</td> <td>Adjusted to let the dissolved O <sub>2</sub> level did not below 40% </td> </tr> <tr> <td>Agitation (RPM)</td> <td>200</td> <td>Adjusted to let the dissolved O <sub>2</sub> level did not below 40% </td> </tr> <tr> <td>Feeding rate (g/l/hr)</td> <td /> <td /> <td>0.42</td> </tr> <tr> <td>Fermentation period (hr)</td> <td>118</td> <td>96</td> <td>192</td> </tr> <tr> <td> <b>Yield coefficients</b> </td> </tr> <tr> <td>Y <sub>x/s</sub> (g/g) </td> <td>2.465</td> <td>4.36</td> <td>12.33</td> </tr> <tr> <td>Y <sub>p/x</sub> (g/g) </td> <td>0.46</td> <td>0.89</td> <td>1.10</td> </tr> <tr> <td>Y <sub>p/s</sub> (g/g) </td> <td>3.595</td> <td>3.9</td> <td>13.53</td> </tr> <tr> <td> <b>Growth and glucose consumption parameters</b> </td> </tr> <tr> <td>X <sub>max-Biomass</sub> (g/l) </td> <td>24.65</td> <td>43.6</td> <td>123.3</td> </tr> <tr> <td>X <sub>max-time</sub> (h) </td> <td>96</td> <td>96</td> <td>240</td> </tr> <tr> <td>dx/dt (g/l/h)</td> <td>0.058</td> <td>0.056</td> <td>0.057</td> </tr> <tr> <td>μ <sub>max</sub> (1/h) </td> <td>0.052</td> <td>0.32</td> <td>0.024</td> </tr> <tr> <td> <b>Nano-MgO production parameters</b> </td> </tr> <tr> <td>P <sub>max-vol</sub> (g/l) </td> <td>35.95</td> <td>39</td> <td>135.3</td> </tr> <tr> <td>P <sub>max-time</sub> (h) </td> <td>90</td> <td>90</td> <td>246</td> </tr> </tbody> </table> </div> <p /> </div> <div class="batchcultivationin7 lstirredtankbioreactor(bf)" title="sec"> <div class="title" tagx="title" title="title">
Batch cultivation in 7 L stirred tank bioreactor (BF)</div> <p>In this experiment, biomass and nano-MgO production kinetics were calculated and studied during batch cultivation in 7 L stirred tank bioreactor using the same medium composition and inoculum concentration as used in the shake flask experiment. The pre-culture inculcated in 7 L stirred tank bioreactor (working volume 2500 ml). During the batch cultivation, glucose was completely consumed after 96 hr. In this step, the growth kinetics measurements indicated that; <b> <i>µ</i> </b> <sub> <b> <i>max</i> </b> </sub> was 0.03, <b> <i>X</i> </b> <sub> <b> <i>max</i> </b> </sub> was 43.6 g/l, P <sub> <b>max</b> </sub> was 39 g/l; and incubation period was 96 hr as shown in Fig. <a href="#Fig9">9B</a> and Table <a href="#Tab6">6</a>. These results indicated that the batch fermentation data was higher than the shake flask experimental data at different stages of cell growth due to the controlled conditions in stirred tank bioreactor (agitation, airflow and pH). </p> </div> <div class="fed-batchcultivationusingexponentialpulsesfeedingstrategywithsucrosefeedingin7 lstirredtankbioreactor(fbf)" title="sec"> <div class="title" tagx="title" title="title">
Fed-batch cultivation using exponential pulses feeding strategy with sucrose feeding in 7 L stirred tank bioreactor (FBF) </div> <p>In this step, cultivation was run under the same conditions of batch culture for the first 96 hr. After that time, feeding was started at an exponential rate of 0.84 g/hr using a concentrated sucrose solution (700 g/l) as shown in Fig. <a href="#Fig9">9B</a>. The pattern of glucose consumption rate and biomass production indicated that when glucose became limited in the culture, biomass dry weight decreased immediately as function of cell death, so the feeding strategy started before this microbial growth stage. The results showed that the fed-batch culture using pulses sucrose feeding system that added exponentially produced biomass concentration ( <b> <i>X</i> </b> <sub> <b> <i>max-Biomass</i> </b> </sub>) 123.3 g/l and nano-MgO ( <b> <i>P</i> </b> <sub> <b> <i>max-vol</i> </b> </sub>) 135.3 g/l after 192 hr (Table <a href="#Tab6">6</a> <b>)</b>. The calculated yield coefficients <b> <i>Y</i> </b> <sub> <b> <i>p/x</i> </b> </sub> and <b> <i>Y</i> </b> <sub> <b> <i>p/s</i> </b> </sub> indicated that the grown cells in this culture showed the highest physiological activities depending on the highest substrate conversion for bioactive metabolites production when compared with SF and BF systems. </p> </div> </div> <div class="optimizationstrategyforthenano-mgobiogenicsynthesisreaction" title="sec"> <div class="title" tagx="title" title="title">
Optimization strategy for the nano-MgO biogenic synthesis reaction</div> <p>Genichi Taguchi has developed an attractive effective tool for optimization and development the high quality industrial production process at a low cost <sup> <a href="#CR23">23</a>– <a href="#CR25">25</a> </sup>. In this design the factor levels (the inner array) are identified in accordance with numbers 1, 2…., and so on that tested against various combinations of the noise factors in the outer array so this design called design of an orthogonal array (signal-to-noise ratio" S/N"). The <b>S/N</b> ratio is expressed using a decibel scale ( <b>dB</b>) <sup> <a href="#CR25">25</a> </sup>. In this work, the biosynthesis of nano-MgO via a simple, fast and eco- friendly method using the Taguchi design ( <b>TD</b>) was evaluated. Taguchi’s model has several arrays to be used that can be chosen based on the experiment’s requirement. Design of an orthogonal array (OA) coded as <b> <i>L</i> </b> <sub> <b> <i>n</i> </b> </sub> <b> <i>(m</i> </b> <sup> <b> <i>k</i> </b> </sup> <b>)</b> where <b> <i>n</i> </b>: number of lines, <b> <i>m</i> </b> <b>:</b> number of parameter levels, and <b> <i>k</i> </b> <b>:</b> number of parameters. In this work, thirty two experiments were conducted at different setup of four parameters coded based on <b>L</b> <sub> <b>32</b> </sub> ( <b>2</b> <sup> <b>4</b> </sup>) OA design to calculate the production of nano-MgO and their corresponding S/N ratio values that listed in Table <a href="#Tab7">7</a>. An ANOVA, the normal probability plot of the residuals and an <i>F</i>-test are used to analyze the experimental data. Moreover, in multiple linear regression analysis, there is a value of R <sup>2</sup> which is the regression coefficient where R <sup>2</sup> is 0.99 for the models, which indicate that the fit of the experimental data is satisfactory as shown in Table ( <a href="#Tab8">8</a>). Based on the ANOVA analysis of the S/N ratio value and calculation of main effect for nano-MgO biosynthesis parameters by using factor level, the highest delta is the top rank as shown in Fig. <a href="#Fig10">10</a>. So the ranked variables with actual values from the most to the less impactful parameters that affect nano-MgO biosynthesis performance was obtained with Metabolites conc. (100%), Mg (NO <sub>3</sub>) <sub>2</sub>.6H <sub>2</sub>O (1 M), Temperature (100 °C) and pH (7.0). Finally, a confirmation test for this design is necessary in order to verify the optimum conditions of nano-MgO biogenic synthesis design. The optimal conditions are set for all factors and a selected number of experiments are run under this defined condition. The result from the confirmation experiment (Mean: 320 ± 8.07 [g/l] and S/N Ratio: 50.92 ± 4.90 [dB]) is compared with the predicted one (Mean: 315 ± 7.07 [g/l] and S/N Ratio: 49.96 ± 1.95 [dB]) based on the variables and their levels tested so there is a very good agreement between the predicted and experimental value is observed. From these results, the nano-MgO biosynthesis can be increased (2.4 times larger than basal condition) and improved through the Taguchi method approach. <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 7</span>  </div></p><p>Taguchi’s L <sub>32</sub> (2 <sup>4</sup>) Orthogonal array design experimental setup for final optimization of nano-MgO biogenic synthesis reaction by using the endophytic <i>S</i>. <i>coelicolor</i> strain E72 intracellular extract. </p>  <table> <thead> <tr> <th>Exp</th> <th>Coded levels of independent variables</th> <th>Nano-MgO (g/l)</th> <th>S/N ratio [dB]</th> </tr> <tr> <th>Metabolites conc</th> <th>Mg(NO <sub>3</sub>) <sub>2</sub>.6H <sub>2</sub>O </th> <th>pH</th> <th>Temperature</th> <th>Actual value ± SD</th> <th>Predicted value</th> </tr> </thead> <tbody> <tr> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>120 ± 00.00</td> <td>120.72</td> <td>41.58</td> </tr> <tr> <td>2</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>120 ± 00.00</td> <td>120.72</td> <td>41.58</td> </tr> <tr> <td>3</td> <td>1</td> <td>1</td> <td>1</td> <td>2</td> <td>130 ± 00.00</td> <td>133.03</td> <td>42.28</td> </tr> <tr> <td>4</td> <td>1</td> <td>1</td> <td>1</td> <td>2</td> <td>130 ± 00.01</td> <td>133.03</td> <td>42.28</td> </tr> <tr> <td>5</td> <td>1</td> <td>1</td> <td>2</td> <td>1</td> <td>122 ± 00.00</td> <td>123.53</td> <td>41.73</td> </tr> <tr> <td>6</td> <td>1</td> <td>1</td> <td>2</td> <td>1</td> <td>122 ± 00.01</td> <td>123.53</td> <td>41.73</td> </tr> <tr> <td>7</td> <td>1</td> <td>1</td> <td>2</td> <td>2</td> <td>133 ± 00.00</td> <td>135.84</td> <td>42.48</td> </tr> <tr> <td>8</td> <td>1</td> <td>1</td> <td>2</td> <td>2</td> <td>133 ± 00.01</td> <td>135.84</td> <td>42.48</td> </tr> <tr> <td>9</td> <td>1</td> <td>2</td> <td>1</td> <td>1</td> <td>235 ± 21.21</td> <td>237.28</td> <td>47.37</td> </tr> <tr> <td>10</td> <td>1</td> <td>2</td> <td>1</td> <td>1</td> <td>235 ± 21.22</td> <td>237.28</td> <td>47.37</td> </tr> <tr> <td>11</td> <td>1</td> <td>2</td> <td>1</td> <td>2</td> <td>250 ± 00.00</td> <td>249.59</td> <td>47.96</td> </tr> <tr> <td>12</td> <td>1</td> <td>2</td> <td>1</td> <td>2</td> <td>250 ± 00.01</td> <td>249.59</td> <td>47.96</td> </tr> <tr> <td>13</td> <td>1</td> <td>2</td> <td>2</td> <td>1</td> <td>250 ± 00.00</td> <td>240.09</td> <td>47.96</td> </tr> <tr> <td>14</td> <td>1</td> <td>2</td> <td>2</td> <td>1</td> <td>250 ± 00.01</td> <td>240.09</td> <td>47.96</td> </tr> <tr> <td>15</td> <td>1</td> <td>2</td> <td>2</td> <td>2</td> <td>252.5 ± 03.53</td> <td>252.41</td> <td>48.04</td> </tr> <tr> <td>16</td> <td>1</td> <td>2</td> <td>2</td> <td>2</td> <td>252.5 ± 03.54</td> <td>252.41</td> <td>48.04</td> </tr> <tr> <td>17</td> <td>2</td> <td>1</td> <td>1</td> <td>1</td> <td>185 ± 07.07</td> <td>184.16</td> <td>45.33</td> </tr> <tr> <td>18</td> <td>2</td> <td>1</td> <td>1</td> <td>1</td> <td>185 ± 07.08</td> <td>184.16</td> <td>45.33</td> </tr> <tr> <td>19</td> <td>2</td> <td>1</td> <td>1</td> <td>2</td> <td>205 ± 07.07</td> <td>196.47</td> <td>46.23</td> </tr> <tr> <td>20</td> <td>2</td> <td>1</td> <td>1</td> <td>2</td> <td>205 ± 07.08</td> <td>196.47</td> <td>46.23</td> </tr> <tr> <td>21</td> <td>2</td> <td>1</td> <td>2</td> <td>1</td> <td>185 ± 07.07</td> <td>186.97</td> <td>45.33</td> </tr> <tr> <td>22</td> <td>2</td> <td>1</td> <td>2</td> <td>1</td> <td>185 ± 07.08</td> <td>186.97</td> <td>45.33</td> </tr> <tr> <td>23</td> <td>2</td> <td>1</td> <td>2</td> <td>2</td> <td>200 ± 00.00</td> <td>199.28</td> <td>46.02</td> </tr> <tr> <td>24</td> <td>2</td> <td>1</td> <td>2</td> <td>2</td> <td>200 ± 00.01</td> <td>199.28</td> <td>46.02</td> </tr> <tr> <td>25</td> <td>2</td> <td>2</td> <td>1</td> <td>1</td> <td>300 ± 00.00</td> <td>300.72</td> <td>49.54</td> </tr> <tr> <td>26</td> <td>2</td> <td>2</td> <td>1</td> <td>1</td> <td>300 ± 00.01</td> <td>300.72</td> <td>49.54</td> </tr> <tr> <td>27</td> <td>2</td> <td>2</td> <td>1</td> <td>2</td> <td>310 ± 00.00</td> <td>313.03</td> <td>49.83</td> </tr> <tr> <td>28</td> <td>2</td> <td>2</td> <td>1</td> <td>2</td> <td>310 ± 00.01</td> <td>313.03</td> <td>49.83</td> </tr> <tr> <td>29</td> <td>2</td> <td>2</td> <td>2</td> <td>1</td> <td>300 ± 00.00</td> <td>303.53</td> <td>49.54</td> </tr> <tr> <td>30</td> <td>2</td> <td>2</td> <td>2</td> <td>1</td> <td>300 ± 00.01</td> <td>303.53</td> <td>49.54</td> </tr> <tr> <td>31</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>315 ± 07.07</td> <td>315.84</td> <td>49.96</td> </tr> <tr> <td>32</td> <td>2</td> <td>2</td> <td>2</td> <td>2</td> <td>315 ± 07.08</td> <td>315.84</td> <td>49.96</td> </tr> <tr> <td> <b>Level</b> </td> <td> <b>(v/v)</b> </td> <td> <b>(M)</b> </td> <td /> <td> <b>(</b>°C <b>)</b> </td> <td /> </tr> <tr> <td>1</td> <td>10%</td> <td>0.5</td> <td>5</td> <td>50</td> </tr> <tr> <td>2</td> <td>100%</td> <td>1</td> <td>7</td> <td>100</td> </tr> </tbody> </table>  <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 8</span>  <p>Statistical analysis of Taguchi design showing coefficient, <i>t</i>-test values, <i>P</i>-values and confidence level (%) for variables affecting on nano-MgO biogenic synthesis reaction by using the endophytic <i>S</i>. <i>coelicolor</i> strain E72 intracellular extract. </p>  <table> <thead> <tr> <th>Variables</th> <th>Coefficients</th> <th>Standard Error</th> <th>T Stat</th> <th> <i>P</i>-value </th> <th /> </tr> </thead> <tbody> <tr> <td>Intercept</td> <td>−74.4063</td> <td>6.898715</td> <td>−10.7855</td> <td>2.74E-11</td> <td /> </tr> <tr> <td>Metabolites conc. (v/v)</td> <td>63.4375</td> <td>2.268283</td> <td>27.96718</td> <td>1.79E-21*</td> </tr> <tr> <td>Mg(NO3)2.6H2O (M)</td> <td>116.5625</td> <td>2.268283</td> <td>51.38798</td> <td>1.8E-28*</td> </tr> <tr> <td>pH</td> <td>2.8125</td> <td>2.268283</td> <td>1.239924</td> <td>0.225672</td> </tr> <tr> <td>Temperature(°C)</td> <td>12.3125</td> <td>2.268283</td> <td>5.428113</td> <td>9.66E-06*</td> </tr> <tr> <td /> <td> <b>df</b> </td> <td> <b>SS</b> </td> <td> <b>MS</b> </td> <td> <b>F</b> </td> <td> <b>Significance F</b> </td> </tr> <tr> <td>Regression</td> <td>4</td> <td>142165.1</td> <td>35541.28</td> <td>863.4723</td> <td>4.66459E-28</td> </tr> <tr> <td>Residual</td> <td>27</td> <td>1111.344</td> <td>41.16088</td> </tr> <tr> <td>Total</td> <td>31</td> <td>143276.5</td> <td /> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p> <b> <i>Multiple R</i> </b>, <i>0</i>. <i>99</i>, <b> <i>R Square</i> </b>, <i>0</i>. <i>99</i>, <b> <i>Adjusted R Square</i> </b>, <i>0</i>. <i>99</i>, <b> <i>Standard Error</i> </b>, <i>6</i>. <i>41</i>, *Significant values, <b>df</b>: Degree of freedom, <b>SS</b>- sum of squares, <b>MS</b>- mean square, <b>F</b>: Fishers’s function, <b>P</b>: Level of significance. </p> </div> </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 10</span> </a> </div>   <p>( <b>A</b>) Response graph of S/N ratio for larger-the-better analysis of nano-MgO production according to the Taguchi <i>’s</i> experimental results. The horizontal axis shows the different levels of each significant factor. The lines represent the trend of each factor with respect to different levels. ( <b>B</b>) The normal probability plot of the residuals. ( <b>C</b>) Percent distribution of each factor to the response for nano-MgO production. </p>   </div> <p /> <p>Finally, this report considered as the first report that demonstrated for using the potential of nanobiotechnological strategies to enhance the biogenic synthesis of nano-MgO by using active metabolites extracted from endophytic <i>Streptomyces coelicolor</i> via semi-industrial scale bioreactor. The experimental designs strategies and statistical inference results indicated that both biomass and nano-MgO productions were higher by using the optimized culture and the optimized biosynthesis reaction respectively compared to the corresponding uncontrolled conditions by more than 49 and 30 times, respectively through PBD, BBD, pulses feeding fermentation strategy and TD as summarized in the Fig. <a href="#Fig11">11</a>. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 11</span> </a> </div>   <p>Summary of the maximum biomass weight of endophytic <i>S</i>. <i>coelicolor</i> strain E72 and the highest nano-MgO production during the used bio-processing strategies after applying Plackett-Burman (PB), Box-Behnken designs (BB) and Taguchi <i>’s</i> experimental design (TD) and large scale batch cultivation (BF) in 1 L shake flask (SF), and exponential pulses fed-batch fermentation mode in 7 L stirred tank bioreactor (FBF). </p>   </div> <p /> </div> <div class="evaluationofantibacterialactivityinvitroforbiogenicsynthesizednano-mgo" title="sec"> <div class="title" tagx="title" title="title">
Evaluation of antibacterial activity <i>in vitro</i> for biogenic synthesized nano-MgO </div> <p>Antibacterial activity test was performed by using different concentrations of prepared nano-MgO against some human pathogenic bacteria to determine the large inhibition zone was calculated and the lowest concentration of nano-MgO that inhibits or kills 100% of the initial bacterial cells were also checked as shown in Table ( <a href="#Tab9">9</a>). The nano-MgO concentrations that formed large inhibition zones were varied from 5 to 55 μg/ml which showed the highest antibacterial activity against <i>Shigella flexneri</i> (108 mm at 25 μg/ml) followed by <i>Staphylococcus aureus</i> (102 mm at 15 μg/ml), <i>Streptococcus pneumonia</i> (89 mm at 40 μg/ml) and <i>Klebsiella pneumonia</i> (85 mm at 20 μg/ml). The lowest concentration of nano-MgO that inhibits of the initial bacterial cells was also varied in descending order, 15, 20, 25 and 40 μg/ml. The bactericidal efficiency also varied from 35 to 55 μg/ml. The comparative study performed to evaluate the final antibacterial activity of 25 μg/ml nano-MgO solution (average of MICs) and ampicilin/sulbactam solution (100 mg/ml) as a standard antibiotic with negative control (native microbial culturing) as shown in Fig. <a href="#Fig12">12</a>. By calculating the Log10 CFU/ml, the results indicated that the nano-MgO solution (25 μg/ml) inhibits 90% of the pathogenic living cells compared with ampicilin/sulbactam solution (100 mg/ml) that killed 40% of the same pathogenic living cells. Finally, these results revealed that nano-MgO showed a strong inhibition activity against anti-multidrug-resistant pathogens not only gram negative but also against gram positive. <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 9</span>  </div></p><p>Detection of the maximum inhibition zone produced from bio-application of biogenic synthesized nano-MgO as antibacterial agent against some human pathogenic bacteria <i>in vitro</i>. </p>  <table> <thead> <tr> <th>Human pathogenic bacteria</th> <th>Nano-MgO concentration (µg/ml)</th> </tr> <tr> <th>Zone of inhibition (mm)</th> <th>MIC</th> <th>MBC</th> </tr> <tr> <th>5</th> <th>10</th> <th>15</th> <th>20</th> <th>25</th> <th>30</th> <th>35</th> <th>40</th> <th>45</th> <th>50</th> <th>55</th> </tr> </thead> <tbody> <tr> <td> <i>Escherichia coli</i> </td> <td>2 ± 0.052</td> <td>15 ± 5.5</td> <td>36 ± 12.3</td> <td>42 ± 15.3</td> <td>56 ± 1.73</td> <td>36 ± 8.6</td> <td>38 ± 3.6</td> <td>20 ± 3.6</td> <td>14 ± 1.3</td> <td>10 ± 3.4</td> <td>4 ± 0.3</td> <td>25</td> <td>50</td> </tr> <tr> <td> <i>Proteus vulgaris</i> </td> <td>3 ± 0.52</td> <td>22 ± 2.3</td> <td>45 ± 2.36</td> <td>75 ± 2.5</td> <td>69 ± 2.14</td> <td>68 ± 3.6</td> <td>59 ± 3.6</td> <td>42 ± 6.3</td> <td>25 ± 10.3</td> <td>12 ± 2.5</td> <td>8 ± 4.3</td> <td>20</td> <td>55</td> </tr> <tr> <td> <i>Salmonella typhimurium</i> </td> <td>2 ± 0.10</td> <td>18 ± 1.36</td> <td>36 ± 5.69</td> <td>53 ± 0.3</td> <td>62 ± 1.38</td> <td>59 ± 2.6</td> <td>52 ± 4.6</td> <td>41 ± 3.6</td> <td>39 ± 1.5</td> <td>35 ± 15.3</td> <td>30 ± 4.9</td> <td>25</td> <td>50</td> </tr> <tr> <td> <i>*Shigella flexneri</i> </td> <td>4 ± 1.3</td> <td>10 ± 2.3</td> <td>15 ± 1.3</td> <td>25 ± 5.3</td> <td>108 ± 10.53*</td> <td>100 ± 12.3</td> <td>95 ± 1.3</td> <td>92 ± 10.3</td> <td>89 ± 3.6</td> <td>82 ± 0.05</td> <td>75 ± 15.3</td> <td>25</td> <td>40</td> </tr> <tr> <td> <i>Klebsiella pneumonia</i> </td> <td>1 ± 0.03</td> <td>36 ± 4.5</td> <td>72 ± 3.69</td> <td>85 ± 6.2</td> <td>63 ± 3.21</td> <td>58 ± 2.3</td> <td>45 ± 3.5</td> <td>30 ± 12.4</td> <td>21 ± 12.3</td> <td>10 ± 2.4</td> <td>7 ± 2.1</td> <td>20</td> <td>45</td> </tr> <tr> <td> <i>Staphylococcus aureus</i> </td> <td>2 ± 1.3</td> <td>52 ± 10.5</td> <td>102 ± 10.3</td> <td>95 ± 4.5</td> <td>81 ± 9.14</td> <td>75 ± 1.3</td> <td>74 ± 2.3</td> <td>69 ± 9.6</td> <td>62 ± 2.3</td> <td>25 ± 10.8</td> <td>15 ± 5.2</td> <td>15</td> <td>35</td> </tr> <tr> <td> <i>Streptococcus pneumonia</i> </td> <td>3 ± 0.2</td> <td>12 ± 0.05</td> <td>25 ± 8.3</td> <td>35 ± 2.21</td> <td>40 ± 2.32</td> <td>55 ± 4.3</td> <td>65 ± 4.6</td> <td>89 ± 7.4</td> <td>78 ± 2.5</td> <td>71 ± 12.6</td> <td>62 ± 3.2</td> <td>40</td> <td>50</td> </tr> <tr> <td> <i>Bacillus cereus</i> </td> <td>3 ± 0.5</td> <td>15 ± 0.03</td> <td>25 ± 0.05</td> <td>45 ± 3.6</td> <td>75 ± 6.3</td> <td>62 ± 6.3</td> <td>45 ± 0.8</td> <td>21 ± 5.2</td> <td>15 ± 4.6</td> <td>9 ± 3.9</td> <td>7 ± 1.3</td> <td>25</td> <td>45</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>*The maximum inhibition zone.</p> </div>  <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 12</span> </a> </div>   <p>The comparative study among the 25 µg/ml nano-MgO solution (average of MICs) and 100 mg/ml ampicilin/sulbactam solution as a standard antibiotic with negative control (native microbial culturing) by calculating the antibacterial activity through Log10 CFU/ml. </p>   </div> <p /> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>The traditional methods of synthesis of nano-metal oxide are chemical and physical techniques that are usually expensive and potentially hazardous to the environment and human health. During the past decade, many biological systems (bacteria, yeast, and fungi) has been demonstrated and developed for biosynthesis of nano-metal oxide by using extracted proteins and metabolites <sup> <a href="#CR23">23</a>– <a href="#CR25">25</a>, <a href="#CR61">61</a> </sup>. For industrial production of nano-metal oxide; the safety, the low cost of microbial cultivation, short biosynthesis reaction time, and the ability to scale-up production; make <i>Streptomyces sp</i>., an attractive factory for a biogenic synthesis of nanoparticles that used for therapeutic applications <sup> <a href="#CR17">17</a>, <a href="#CR50">50</a>, <a href="#CR60">60</a>, <a href="#CR62">62</a> </sup>. Recently, biogenic synthesis of nanoparticles has taken an important stage in the modern industrial pharmaceutical sector, so exploring novel eco-friendly bioactive metabolites extracted from safe potential bio-factories that possess low toxicity was studied in this work. </p> <p>There are organic (bacteriocins, enzymes, etc) and inorganic (metal and metal oxide, etc) compounds were reported as proficient antibacterial agents <sup> <a href="#CR10">10</a>, <a href="#CR40">40</a> </sup>. But inorganic antibacterial agents have attracted for bacterial control due to their stability under harsh industrial processing conditions <sup> <a href="#CR8">8</a> </sup>. Metal oxide nanoparticles have considerable potential as a new class of antimicrobial agents that have been increasingly studied for their potential industrial applications in food safety and health care. Inorganic metal oxides nanoparticles that have high stability and discipline sizes, shapes, surface properties, &amp;amp; chemical compositions were used recently as antimicrobial agents. And due to their large surface area that interact with pathogens cells; these nanoparticles have a broad spectrum antimicrobial activity. Since some studies suggested that higher antimicrobial activities depends upon not only the nanoparticles sizes, shapes, chemical composition, and surface properties but also varies with pathogens species. Nano-MgO (nano-metal oxide) is very interesting due to its strong antimicrobial activities, furthermore its structure, surface properties, high thermal stability and the simplicity of its preparation. </p> <p>It is known that the medicinal plants have a novel natural compounds and endophytes population (microbial cells has been found to be associated with the internal tissues of a plant without causing detectable disease) which used for industrial biotechnological applications <sup> <a href="#CR13">13</a>, <a href="#CR28">28</a>, <a href="#CR38">38</a> </sup>. Endophytes possess the potential to produce eco-friendly novel primary and/or secondary metabolites, which can be attracted an attention in pharmaceutical industries. The biosynthesis of silver nanoparticles as an antimicrobial agent by using the epiphytic <i>Streptomyces coelicolor</i> klmp33 pigment was reported by Manikprabhu <sup> <a href="#CR15">15</a>, <a href="#CR16">16</a> </sup>. By gathering sufficient knowledge, there are no studies on isolation of the endophytic <i>Streptomyces</i> from the medicinal plants and used it for nano-MgO biosynthesis. So, reliable biogenic synthesis of nano-MgO by using safe metabolites that extracted from the less explored endophytic <i>Streptomyces sp</i>. which isolated from the healthy medicinal plant ( <i>Ocimum sanctum)</i> was characterized, evaluated, and developed through this study for environmental, and health safety therapeutic applications (antimicrobial agent). In the present investigations, different bioprocessing strategies used to maximize the <i>Streptomyces</i> biomass and nano-MgO yield in short reaction time. Through this work; endophytic <i>Streptomyces coelicolor</i> strain E72 selected as a proficient factory for the biogenic synthesis of nano-MgO that used as a promising antibacterial agent. </p> <p>There are few reports on the synthesis of MgO nanoparticles as antimicrobial agent alone, mixed with other metal oxide nanoparticles or mixed with antibiotics against pathogenic bacteria and fungi <sup> <a href="#CR5">5</a>, <a href="#CR9">9</a> </sup>. In nano-medicine, the biogenic synthesis method that used to fabricate the inorganic metal oxide nanoparticles is a new safe method compared to the electrochemical fabrication methods <sup> <a href="#CR6">6</a> </sup>. The physicochemical properties of nanoparticles (particle size, surface area, and chemical composition/structure) are important factors that contribute significantly to the toxicity of these nanoparticles. Sulfur-containing proteins, carbohydrates, saponins, phenolic compounds, quinones free amine groups and NADH-dependent nitrate reductase are reported as bio-reduction and/or bio-stabilizing agent (microbial cell-free extracts) that found to be responsible for the biogenic synthesis of metal oxide nanoparticles from metal salts <sup> <a href="#CR3">3</a>, <a href="#CR43">43</a>, <a href="#CR63">63</a> </sup>. In this report, the intracellular fraction that extracted from endophytic <i>Streptomyces coelicolor</i> strain E72 used as nano-MgO synthesizers. This fraction that contains 7.2 μg/μl of carbohydrate, 6.3 g/l of protein and 5.2 nmol/hr/ml of nitrate reductase used to produce multi-surface shaped nano-MgO (spherical and ellipsoidal) with diameter ~25 nm that have surface plasmon resonance at 448 nm. All of these results are strongly supported by former reports <sup> <a href="#CR5">5</a>, <a href="#CR6">6</a>, <a href="#CR40">40</a>, <a href="#CR43">43</a> </sup>. </p> <p>A killing of multidrug-resistant human pathogens considered as a crucial challenge for public health and required a new noble biotechnological strategy. Nano-antimicrobial agents are useful tools used for a wide variety of multidrug-resistant human pathogens. Several mechanisms have been reported previously for the antimicrobial therapeutic agent’s activity <sup> <a href="#CR4">4</a> </sup>. Nanoparticle toxicity may be caused on the cellular surface (interacts with membrane lipids by electrostatic interaction) or/and inside the cellular organelles (the production of toxic oxygen radicals) which loss of cell membranes integrity, damage DNA, or cell proteins, and finally inhibits the bacterial growth <sup> <a href="#CR13">13</a>, <a href="#CR22">22</a> </sup>. Others suggested that nanoparticles toxicity caused by increasing intracellular reactive oxygen species (ROS) levels and/or the levels of inflammatory mediators followed by molecular and cellular events that cause oxidative stress, severe genotoxicity (frame shift mutation) and then cell death <sup> <a href="#CR7">7</a> </sup>. The mechanism for ROS generation (irreversible damage) is different for each nanoparticle and the exact mechanism for ROS generation is incompletely understood <sup> <a href="#CR14">14</a> </sup>. As a general, the exact mechanisms of toxicity that caused by nanoparticles treatment are very complicated and are not understood completely <sup> <a href="#CR12">12</a>, <a href="#CR64">64</a> </sup>. </p> <p>The endophytic <i>Streptomyces sp</i>. has a proficient capacity to produce a large variety of different low toxicity bioactive metabolites that have a wide spectrum of important industrial activities and continue to be screened for new low-cost commercial bioactive substances. Metabolic activities inside the cell are extremely sensitive to the ambient environment (nutritional media and culturing conditions) and regulated at various levels, so it is very important to study and understand the nutritional and environmental factors affecting cell metabolism <sup> <a href="#CR45">45</a> </sup>. The microbial nutrition environment plays a significant role in the biomass production and hence in different metabolites production via the fermentation process. Evaluation, selection, and optimization of the culturing media are essential steps that prior the semi-pilot or pilot production plans for biomass production and hence for bioactive metabolite production <sup> <a href="#CR65">65</a> </sup>. Advanced statistical medium optimization technique has been proved as the effective optimization strategies that might be useful for the overproduction of many bioactive metabolites in the large scale fermentation process <sup> <a href="#CR1">1</a> </sup>. The successful industrial bioprocessing strategies depend on the product cost that could be reduced by using cheaper carbon/nitrogen sources (growth inducer) and/or by increasing the bioactive metabolites productivity and reduce the loss of unused substrate <sup> <a href="#CR66">66</a> </sup>. So the advanced statistical medium optimization strategies help in reducing the desired bioactive metabolites cost <sup> <a href="#CR2">2</a> </sup>. </p> <p>The extremely low biomass yields, the activity of the metabolites and long incubation period have restricted its further large-scale production and final marketing strategies <sup> <a href="#CR15">15</a>, <a href="#CR16">16</a>, <a href="#CR30">30</a>, <a href="#CR67">67</a> </sup>. That is mean; there is a relationship between the medium compositions &amp;amp; process parameters and maximum bioactive metabolites &amp;amp; biomass production. The microbial growth and accumulation of bioactive metabolites are strongly influenced by carbon sources, nitrogen sources, various inorganic salts and trace elements. So, to achieve the high-value of the bioactive metabolic compounds, it is necessary to evaluate an efficient industrial fermentation medium <sup> <a href="#CR20">20</a>, <a href="#CR34">34</a>, <a href="#CR42">42</a>, <a href="#CR55">55</a> </sup>. Important attention is paid to the selection of carbon source and nitrogen source because of their role in the microbial biomass yield and hence the formation of bioactive metabolites. There are some carbon/nitrogen sources have the better influence on the biomass growth but others reduced production of many bioactive metabolites <sup> <a href="#CR21">21</a>, <a href="#CR68">68</a>– <a href="#CR70">70</a> </sup>. For potential industrial large-scale fermentation, the choice of the carbon/nitrogen sources also depends on their availability on the market and prices. An excellent carbon source (the monosaccharide, disaccharide, or polysaccharide) and nitrogen sources (organic, inorganic or mix) have been added as an energy source to influence the microbial growth and the formation of several metabolic compounds. For <i>Streptomyces</i> species there is diversity in carbon/nitrogen sources can be used <sup> <a href="#CR10">10</a>, <a href="#CR27">27</a>, <a href="#CR41">41</a>, <a href="#CR66">66</a>, <a href="#CR71">71</a>, <a href="#CR72">72</a> </sup>. So in the present report, the primary screening studies including two influenced parameters selection the suitable and cheaper cultivation medium for <i>Streptomyces coelicolor</i> strain E72 mass production and the most potent extracted metabolites that used for the biogenic synthesis of nano-MgO. The final results showed that the <i>Streptomyces coelicolor</i> strain E72 was able to grow in all the tested carbon/nitrogen sources but modified Czapek-dox medium (M1) was selected as the original fermentation medium among the tested media (that contained different carbon/nitrogen sources). Since sucrose and (NH <sub>4</sub>) <sub>2</sub>SO <sub>4</sub>/NaNO <sub>3</sub> were the carbon/nitrogen sources that supported the maximum growth rate as well as the highest bioactive metabolites production at the same time. These results indicated that the highest bioactive metabolites that used for the biogenic synthesis of nano-MgO depend on the <i>Streptomyces coelicolor</i> strain E72 growth medium constituents. Different statistical experimental designs used in this stage to optimize the basic fermentation medium that selected via this work. Combination of the first-order model PBD (screening of the variables) and the second-order model BBD (estimate the optimal value and the interactions effects among a set of controlled experimental factors) were applied for optimization of different medium components for biomass and bioactive metabolites (that used for biogenic synthesis of nano-MgO) production by <i>Streptomyces coelicolor</i> strain E72. The final results indicated that the biomass production can be increased more than 7times when eliminating NaNO <sub>3</sub> and KCl from the optimized medium and the high-yield new medium formulation consisted of (g/l) 48.2 Sucrose, 4.8 (NH <sub>4</sub>) <sub>2</sub>SO <sub>4</sub>, 3.5 K <sub>2</sub>HPO <sub>4</sub>, 0.5 MgSO <sub>4</sub>·7H <sub>2</sub>O, and 0.01 FeSO <sub>4</sub>·7H <sub>2</sub>O. In this work, the inorganic nitrogen source ((NH <sub>4</sub>) <sub>2</sub>SO <sub>4</sub>) and disaccharide carbon source (sucrose) are preferred for the highest endophytic <i>Streptomyces coelicolor</i> strain E72 biomass and bioactive metabolites production compared to organic nitrogen and polysaccharide carbon sources this conclusion opposite to the earlier reports <sup> <a href="#CR41">41</a>, <a href="#CR45">45</a>, <a href="#CR65">65</a>, <a href="#CR73">73</a> </sup>. There are studies recorded that the specific metabolites production doubled when nitrate was replaced with ammonium, and the biomass yield increased many times <sup> <a href="#CR46">46</a>, <a href="#CR48">48</a>, <a href="#CR68">68</a>, <a href="#CR70">70</a> </sup>. Most of the growth media that reported earlier contained a mix of complex organic and/or inorganic nitrogen sources, viz the proficient <i>Streptomyces</i> medium cannot be generalized <sup> <a href="#CR1">1</a>, <a href="#CR47">47</a>, <a href="#CR67">67</a>, <a href="#CR74">74</a> </sup>. Finally, the endophytic <i>Streptomyces coelicolor</i> strain E72 cultivated by using cost-effective industrial fermentation medium (inexpensive substrates). Until now, <i>Streptomyces coelicolor</i> is a proficient applied strain of the genus <i>Streptomyces</i> that extracts different bioactive metabolite compounds in clinical field <sup> <a href="#CR71">71</a> </sup>. The promising strain was characterized by a high-yield biomass production by using inexpensive nutrients <sup> <a href="#CR44">44</a> </sup>. Wentzel <sup> <a href="#CR71">71</a> </sup> reported that the presence of D-glucose and/or L-glutamate was preferred as carbon sources used in <i>Streptomyces coelicolor</i> cultivation strategy. Dzhavakhiya <sup> <a href="#CR44">44</a> </sup> concluded that the highest-biomass yield of <i>Streptomyces virginiae</i> and hence the virginiamycin yield was recorded by using sucrose as a carbon source instead of more expensive glucose or D-maltose but complex carbohydrates (starch) shown a lower effect. </p> <p>Realize biomass productivity via scaling-up strategies strongly depends on ideal fermentation conditions (dissolved oxygen, gas flow, agitation, temperature, pH, etc.) and fermentation modes (batch, substrate fed-batch, whole nutrient fed-batch, etc.) <sup> <a href="#CR51">51</a>, <a href="#CR75">75</a>, <a href="#CR76">76</a> </sup>. Carbon catabolite repression the biosynthesis of many bioactive metabolites is a big problem caused by the carbon source (sucrose) during the <i>Streptomyces virginiae</i> cultivation. To solve this problem, a fed-batch fermentation mode was used by maintaining the substrate concentration at a very low level (below a critical level) to improve the highest biomass and metabolites yields <sup> <a href="#CR44">44</a>, <a href="#CR71">71</a> </sup>. There are several publications and patents using <i>Streptomyces</i> strain as antibiotics bio-factory that industrial cultivated by using different carbon/nitrogen sources and minor elements (the medium components) via different optimized fermentations modes (optimization of fermentation conditions and/or addition of precursors or regulators) <sup> <a href="#CR46">46</a>, <a href="#CR48">48</a>, <a href="#CR51">51</a>, <a href="#CR68">68</a>, <a href="#CR70">70</a>, <a href="#CR75">75</a>, <a href="#CR76">76</a> </sup>. To the best of our knowledge, this is the first report to evaluate endophytic <i>Streptomyces coelicolor</i> biomass production (123.3 g/l) to extract the highest bioactive metabolites that used for the biogenic synthesis of nano-MgO (135.3 g/l) using exponential sucrose pulses feeding strategy after 192 hr in semi industrial scale bioreactor (7 L). </p> <p>The reduction process of metal ions to produce metal/metal oxide nanoparticles is affected by various factors such as exogenous biometrics, the reaction mixture pH, incubation temperature, reaction time, and electrochemical potential of a metal ion <sup> <a href="#CR39">39</a>, <a href="#CR62">62</a> </sup>. So, the factors that affect the morphology, size, and yield of metal nanoparticles studied and optimized for the proficient fabrication of metal oxide nanoparticles. Terpenoids, polyphenols, sugars, alkaloids, phenolic acids, peptides, an aldehyde group, proteins, hydroxyl groups and carbonyl group reported as major stronger reductant metabolites that used for a bioreduction of metal ions <sup> <a href="#CR5">5</a>, <a href="#CR26">26</a>, <a href="#CR77">77</a> </sup>. Also, the reaction pH value affecting the fabrication of metal oxide nanoparticles; since a natural phytochemicals charge was changed and hence the bioreduction ability was affected, in turn, may affect the shape, size, and yield of metal oxide nanoparticles. There are some metal/metal oxides fabricated at alkaline pH mixture, whereas others prefer very acidic pH mixture values; depends on the nature of phytochemicals and fabricated nanoparticles <sup> <a href="#CR8">8</a>, <a href="#CR56">56</a> </sup>. Temperature is another important factor affecting the biogenic synthesis of metal oxide nanoparticles; because it affects the morphological nanoparticles form and increases the fabrication reaction rate and hence the efficiency of this process. Some reports concluded that high temperatures preferred than room temperature in fabricated some crystal nanoparticles. Furthermore, in some cases, high temperature may destroy the bioreductants that used in nanoparticles reduction process <sup> <a href="#CR9">9</a>, <a href="#CR26">26</a>, <a href="#CR54">54</a> </sup>. So the biogenic synthesis of metal oxide nanoparticles should be developed by using nanobiotechnological strategies to study the nanoparticles reduction parameters. Statistical experimental designs recently used to optimize a nanoparticles biofabrication process. There are few publications studied a nanoparticles biofabrication process and recorded the biosynthesis reaction factors (pH, precursor concentration, temperature, reaction time period and stability of biosynthesized nanoparticles) of the metal oxide nanoparticles (Copper/Copper oxide and Silver nanoparticles) that bio-fabricated by using extracted bioactive metabolites from plant, algae and endophytic fungi <sup> <a href="#CR23">23</a>– <a href="#CR25">25</a> </sup>. In this report, the biosynthesis reaction factors improved through the Taguchi experimental design method approach and the nano-MgO biosynthesis increased about 2.4 times larger than basal condition by using the optimized reaction condition that contained: Metabolites conc. (100%), Mg (NO <sub>3</sub>) <sub>2</sub>·6H <sub>2</sub>O (1 M), Temperature (100 °C) at pH (7.0). Finally, this is the first report using nanobiotechnological strategies to study the nano-MgO (320 g/l) biogenic synthesis parameters by using endophytic <i>Streptomyces coelicolor</i> intracellular extract that cultivated in semi industrial scale bioreactor. </p> <p>The nanoparticle sizes considered as the main factor that affects on the antimicrobial efficiency. The smaller sized MgO nanoparticles have proficient antimicrobial activities has been investigated against both gram positive and gram negative while bigger inhibition sizes were recorded against gram negative only <sup> <a href="#CR5">5</a>, <a href="#CR17">17</a>, <a href="#CR59">59</a>, <a href="#CR61">61</a>, <a href="#CR78">78</a>, <a href="#CR79">79</a> </sup>. There are few reports on the antimicrobial activity of chemical synthetic MgO nanoparticles in the literature to inhibit the growth of human pathogens especially gram positive bacteria than gram negative bacteria <sup> <a href="#CR5">5</a>, <a href="#CR78">78</a>, <a href="#CR79">79</a> </sup>. In this work, nano-MgO (25 nm) applied <i>in vitro</i> against multi-drug resident human pathogens ( <i>Shigella flexneri</i>, <i>Staphylococcus aureus</i>, <i>Streptococcus pneumonia</i>, and <i>Klebsiella pneumonia</i>) and the large inhibition zone recorded against <i>Shigella flexneri</i>. Other researchers also reported the minimum inhibitory concentration (MIC) of MgO nanoparticles (&amp;gt;20 nm) was recorded as 0.5, 4 and 8 mg/ml for <i>S</i>. <i>aureus</i>, <i>E</i>. <i>coli</i>, <i>Campylobacter jejuni</i>, <i>Staphylococcus epidermidis</i>, <i>Bacillus cereus</i>, <i>Klebsiella pneumonia</i>, <i>Proteus vulgaris</i> and <i>Salmonella</i> <sup> <a href="#CR17">17</a>, <a href="#CR59">59</a>, <a href="#CR61">61</a>, <a href="#CR78">78</a> </sup>. The average of MICs was recorded as 25 µg/ml that inhibited 90% of the pathogenic living cells and compared with 100 mg/ml ampicilin/sulbactam solution that killed 40% of the same pathogenic living cells. Generally, nanoparticles attacked easily microbial cells via their vital components such as peptidoglycan cell wall, cytoplasmic membrane, and bio-molecules to disturb the cell membrane permeability and pore the cytoplasmic membrane so the microbial membranes morphology were severity damaged <sup> <a href="#CR78">78</a>– <a href="#CR81">81</a> </sup>. Subsequently, the nanoparticles reacted with the sulfur-containing proteins as well as a phosphorus-containing compound such as DNA thereby inhibiting protein function and resulting in microbial cell death <sup> <a href="#CR61">61</a>, <a href="#CR78">78</a>, <a href="#CR82">82</a>, <a href="#CR83">83</a> </sup>. Recently, the scientists suggested that the uptake of the nano-MgO may be affected by the cell membrane binding process and internalization process (endocytosis pathway). So the antibacterial efficiency of nanoparticles may associate with the chemical structure of the microbial membranes (gram-positive or gram-negative bacteria) <sup> <a href="#CR78">78</a>, <a href="#CR84">84</a>, <a href="#CR85">85</a> </sup>. Since the nano-MgO may oxidize the lipoproteins on the plasma membrane and disturb the metabolic functions such as respiration resulting in cell decomposition and death eventually <sup> <a href="#CR86">86</a> </sup>. So the using of nano-MgO as an antibacterial agent was active significantly against gram-negative bacteria more than gram-positive bacteria <sup> <a href="#CR17">17</a>, <a href="#CR59">59</a>, <a href="#CR84">84</a> </sup>. Until now, the nano-toxicologists have concluded that the specific mechanisms of the accurate interactions between nanoparticles and pathogenic cell membrane (the first contact point) have been poorly explained <sup> <a href="#CR78">78</a>, <a href="#CR82">82</a>, <a href="#CR83">83</a> </sup>. So the precise bactericidal effect of nanoparticles is not yet clearly detected. Basically, there are many physicochemical parameters of nanoparticles influence their toxicities such as size, shape, surface charge and chemical composition among others <sup> <a href="#CR80">80</a>– <a href="#CR86">86</a> </sup>. Previously, it has been concluded that the highest concentrations of nanoparticles are mainly accumulated as aggregates in the microbial cells without manifesting any acute toxic effects <sup> <a href="#CR80">80</a> </sup>. So the aggregation states of nanoparticles have undesirable effects than the well-dispersed nanoparticles <sup> <a href="#CR81">81</a>, <a href="#CR82">82</a> </sup>. Some reports revealed that the bactericidal effect of nanoparticles depended on the dispersion of the smallest nanoparticles because they can penetrate easily into the pathogen cells, affecting the DNA and the enzymes leading to cell death, but nanoparticles at aggregations statue accumulated on the cell surface and compromise cellular permeability <sup> <a href="#CR83">83</a>, <a href="#CR85">85</a>, <a href="#CR86">86</a> </sup>. Finally, in this work, the low nano-MgO concentration has the lowest affinity for aggregation so its toxicity increased. In this respect, the reported data revealed that smaller nano-MgO particles at low concentration are found to have the greatest antibacterial activity that provided stronger bactericidal interactions, especially in gram-negative strains. The results of this report are expected to gather sufficient knowledge to discover and develop a new simple, cheap and eco-friendly nano-MgO as an extremely strong antimicrobial agent used in biomedical applications. </p> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p> <b>Publisher's note:</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. </p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>The author acknowledges the Bioprocess Development Department, Genetic Engineering and Biotechnology Research Institute (GEBRI), City of Scientific Research and Technological Applications (SRTA-City), New Borg El-Arab, Alexandria, Egypt for funding the activities presented in this study. </p> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>S.H.E.-M. proposed the research concept, designed the experiments, providing necessary tools for experiments, experimental instructions, conducted most of the experiments, analyzed and interpreted the data and wrote the manuscript. </p> <div class="competinginterests" title="sec"> <div class="title" tagx="title" title="title">
Competing Interests</div> <p>The author declares no competing interests.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="CR1" /> <span class="label" tagx="label" title="label">1.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Falzone</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Nutritional control of antibiotic production by <i>Streptomyces platensis</i> MA7327: importance of L-aspartic acid </span> <span class="source" tagx="source" title="source">J. Antibiotics.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">70</span> <span class="fpage" tagx="fpage" title="fpage">828</span> <span class="lpage" tagx="lpage" title="lpage">831</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/ja.2017.49">10.1038/ja.2017.49</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28465627">28465627</a> </span> </span> </li> <li tag="ref"> <a name="CR2" /> <span class="label" tagx="label" title="label">2.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Epigenetic activation of antibacterial property of an Endophytic <i>Streptomyces coelicolor</i> Strain AZRA 37 and Identification of the Induced Protein Using MALDI TOF MS/MS </span> <span class="source" tagx="source" title="source">PLoS ONE.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">e0147876</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0147876">10.1371/journal.pone.0147876</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26844762">26844762</a> </span> </span> </li> <li tag="ref"> <a name="CR3" /> <span class="label" tagx="label" title="label">3.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar</span> <span class="given-names" tagx="given-names" title="given-names">SP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Duraipandiyan</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ignacimuthu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Isolation, screening and partial purification of antimicrobial antibiotics from soil <i>Streptomyces sp</i>. SCA 7 </span> <span class="source" tagx="source" title="source">Kaohsiung J. Med. Sci.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">30</span> <span class="fpage" tagx="fpage" title="fpage">435</span> <span class="lpage" tagx="lpage" title="lpage">446</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.kjms.2014.05.006">10.1016/j.kjms.2014.05.006</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25224766">25224766</a> </span> </span> </li> <li tag="ref"> <a name="CR4" /> <span class="label" tagx="label" title="label">4.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salem</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Antibacterial activity of silver and zinc nanoparticles against <i>Vibrio cholerae</i> and enterotoxic <i>Escherichia coli</i> </span> <span class="source" tagx="source" title="source">Int J Med Microbiol.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">305</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">85</span> <span class="lpage" tagx="lpage" title="lpage">95</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ijmm.2014.11.005">10.1016/j.ijmm.2014.11.005</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25466205">25466205</a> </span> </span> </li> <li tag="ref"> <a name="CR5" /> <span class="label" tagx="label" title="label">5.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharma</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Soni</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jasuja</span> <span class="given-names" tagx="given-names" title="given-names">DN</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Phytoassisted synthesis of magnesium oxide nanoparticles with <i>Swertia chirayaita</i> </span> <span class="source" tagx="source" title="source">JTUSCI</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">471</span> <span class="lpage" tagx="lpage" title="lpage">477</span> </span> </li> <li tag="ref"> <a name="CR6" /> <span class="label" tagx="label" title="label">6.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Mohanta, Y. K., Panda, S. K., Bastia, A. K. &amp;amp; Mohanta, T. K. Biosynthesis of Silver Nanoparticles from <i>Protium serratum</i> and investigation of their potential impacts on food safety and control. <i>Front</i>. <i>Microbiol</i>. <b>8</b>(626), (2017). </span> </li> <li tag="ref"> <a name="CR7" /> <span class="label" tagx="label" title="label">7.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Durán</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Silver nanoparticles: A new view on mechanistic aspects on antimicrobial activity</span> <span class="source" tagx="source" title="source">Nanomedicine: Nanotechnology, Biology, and Medicine</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="fpage" tagx="fpage" title="fpage">789</span> <span class="lpage" tagx="lpage" title="lpage">799</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.nano.2015.11.016">10.1016/j.nano.2015.11.016</a> </span> </span> </li> <li tag="ref"> <a name="CR8" /> <span class="label" tagx="label" title="label">8.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suresh</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yuvakkumar</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sundrarajan</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hong</span> <span class="given-names" tagx="given-names" title="given-names">IS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Green synthesis of magnesium oxide nanoparticles</span> <span class="source" tagx="source" title="source">Advanced Materials Research.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">952</span> <span class="fpage" tagx="fpage" title="fpage">141</span> <span class="lpage" tagx="lpage" title="lpage">144</span> <span class="pub-id"> <a href="https://dx.doi.org/10.4028/www.scientific.net/AMR.952.141">10.4028/www.scientific.net/AMR.952.141</a> </span> </span> </li> <li tag="ref"> <a name="CR9" /> <span class="label" tagx="label" title="label">9.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meenakshi.</span> <span class="given-names" tagx="given-names" title="given-names">DS</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Synthesis and characterization of magnesium oxide nanoparticles</span> <span class="source" tagx="source" title="source">Elixir Nanotechnology.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">50</span> <span class="fpage" tagx="fpage" title="fpage">10618</span> <span class="lpage" tagx="lpage" title="lpage">10620</span> </span> </li> <li tag="ref"> <a name="CR10" /> <span class="label" tagx="label" title="label">10.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zonooz</span> <span class="given-names" tagx="given-names" title="given-names">FN</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salouti</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Extracellular biosynthesis of silver nanoparticles using cell filtrate of <i>Streptomyces sp</i>. <i>ERI-3</i> </span> <span class="source" tagx="source" title="source">Scientia Iranica F.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">18</span> <span class="issue" tagx="issue" title="issue">6</span> <span class="fpage" tagx="fpage" title="fpage">1631</span> <span class="lpage" tagx="lpage" title="lpage">1635</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.scient.2011.11.029">10.1016/j.scient.2011.11.029</a> </span> </span> </li> <li tag="ref"> <a name="CR11" /> <span class="label" tagx="label" title="label">11.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saliani</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jalal</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goharshadi</span> <span class="given-names" tagx="given-names" title="given-names">EK</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Mechanism of oxidative stress involved in the toxicity of ZnO nanoparticles against eukaryotic cells </span> <span class="source" tagx="source" title="source">Nanomed. J.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">14</span> </span> </li> <li tag="ref"> <a name="CR12" /> <span class="label" tagx="label" title="label">12.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tee</span> <span class="given-names" tagx="given-names" title="given-names">KJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ong</span> <span class="given-names" tagx="given-names" title="given-names">NC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bay</span> <span class="given-names" tagx="given-names" title="given-names">HB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ho</span> <span class="given-names" tagx="given-names" title="given-names">KH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leong</span> <span class="given-names" tagx="given-names" title="given-names">TD</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Oxidative stress by inorganic nanoparticles</span> <span class="source" tagx="source" title="source">WIREs Nanomed Nanobiotechnol</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">414</span> <span class="lpage" tagx="lpage" title="lpage">438</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/wnan.1374">10.1002/wnan.1374</a> </span> </span> </li> <li tag="ref"> <a name="CR13" /> <span class="label" tagx="label" title="label">13.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khanna</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ong</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bay</span> <span class="given-names" tagx="given-names" title="given-names">HB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baeg</span> <span class="given-names" tagx="given-names" title="given-names">HG</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Nanotoxicity: An Inter plays of Oxidative Stress, Inflammation and Cell Death</span> <span class="source" tagx="source" title="source">Nanomaterials.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">1163</span> <span class="lpage" tagx="lpage" title="lpage">1180</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3390/nano5031163">10.3390/nano5031163</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28347058">28347058</a> </span> </span> </li> <li tag="ref"> <a name="CR14" /> <span class="label" tagx="label" title="label">14.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Manke, A., Wang, L. &amp;amp; Rojanasakul, Y. Mechanisms of Nanoparticle-Induced Oxidative Stress and Toxicity. <i>BioMed Research International</i>. Article ID <b>942916</b> (2013). </span> </li> <li tag="ref"> <a name="CR15" /> <span class="label" tagx="label" title="label">15.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Manikprabhu, D. &amp;amp; Lingappa, K. Microwave assisted rapid and green synthesis of silver nanoparticles using a pigment produced by <i>Streptomyces coelicolor</i> klmp33. <i>Bioinorganic Chemistry and Applications</i>. Article ID <b>341798</b> (2013). </span> </li> <li tag="ref"> <a name="CR16" /> <span class="label" tagx="label" title="label">16.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Manikprabhu</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lingappa</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Synthesis of silver nanoparticles using the <i>Streptomyces coelicolor</i> klmp33 pigment: An antimicrobial agent against extended-spectrum beta-lactamase (ESBL) producing <i>Escherichia coli</i> </span> <span class="source" tagx="source" title="source">Materials Science and Engineering C.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">45</span> <span class="fpage" tagx="fpage" title="fpage">434</span> <span class="lpage" tagx="lpage" title="lpage">437</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.msec.2014.09.034">10.1016/j.msec.2014.09.034</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25491848">25491848</a> </span> </span> </li> <li tag="ref"> <a name="CR17" /> <span class="label" tagx="label" title="label">17.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jin</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">He</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Antibacterial activities of magnesium oxide (MgO) nanoparticles against foodborne pathogens </span> <span class="source" tagx="source" title="source">J Nanopart Res.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="fpage" tagx="fpage" title="fpage">6877</span> <span class="lpage" tagx="lpage" title="lpage">6885</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11051-011-0595-5">10.1007/s11051-011-0595-5</a> </span> </span> </li> <li tag="ref"> <a name="CR18" /> <span class="label" tagx="label" title="label">18.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Parizi</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Evaluation of the antifungal effect of magnesium oxide nanoparticles on <i>Fusarium oxysporum F</i>. <i>Sp</i>. <i>lycopersici</i>, pathogenic agent of tomato, <i>European</i> </span> <span class="source" tagx="source" title="source">Journal of Experimental Biology.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">151</span> <span class="lpage" tagx="lpage" title="lpage">156</span> </span> </li> <li tag="ref"> <a name="CR19" /> <span class="label" tagx="label" title="label">19.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Najafzadeh, H., Ghorbanpour, M., Hekmati, M. S. H. &amp;amp; Karimiyan, A. Antifungal effect of magnesium oxide, zinc oxide, silicon oxide and copper oxide nanoparticles against <i>candida albicans</i>. <i>Zahedan J Res Med Sci</i>. ( <b>2015</b>). </span> </li> <li tag="ref"> <a name="CR20" /> <span class="label" tagx="label" title="label">20.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rao</span> <span class="given-names" tagx="given-names" title="given-names">GK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ashok</span> <span class="given-names" tagx="given-names" title="given-names">CH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rao</span> <span class="given-names" tagx="given-names" title="given-names">VK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chakra</span> <span class="given-names" tagx="given-names" title="given-names">SCH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Akshaykrant</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Eco-Friendly Synthesis of MgO Nanoparticles from Orange Fruit Waste, <i>International</i> </span> <span class="source" tagx="source" title="source">Journal of Advanced Research in Physical Science.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">6</span> </span> </li> <li tag="ref"> <a name="CR21" /> <span class="label" tagx="label" title="label">21.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Raliya, R. &amp;amp; Tarafdar, C. J. Biosynthesis and characterization of zinc, magnesium and titanium nanoparticles: an eco-friendly approach. <i>Int Nano Lett</i>. <b>4</b>(93), (2014). </span> </li> <li tag="ref"> <a name="CR22" /> <span class="label" tagx="label" title="label">22.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">ZX</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bin-Feng</span> <span class="given-names" tagx="given-names" title="given-names">LV</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">MgO nanoparticles antibacterial agent: preparation and activity</span> <span class="source" tagx="source" title="source">Brazilian Journal of Chemical Engineering.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">31</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">591</span> <span class="lpage" tagx="lpage" title="lpage">601</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1590/0104-6632.20140313s00002813">10.1590/0104-6632.20140313s00002813</a> </span> </span> </li> <li tag="ref"> <a name="CR23" /> <span class="label" tagx="label" title="label">23.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">EL-Moslamy, H. S., Shokry, H. H., Rezk, H. A. &amp;amp; Abdel-Fattah R. Y. B S and Characterization of Anti- Multidrug Resistant Human Pathogens Copper/Copper Oxide Nanoparticles from Citrus Peel Waste Extracts. <i>J Nanomater Mol Nanotechnol</i>. <b>6</b>( <b>1</b>), (2017). </span> </li> <li tag="ref"> <a name="CR24" /> <span class="label" tagx="label" title="label">24.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">EL-Moslamy, H. S., Kabeil, S. A. S. &amp;amp; Hafez, E. E. Bioprocess development for <i>Chlorella vulgaris</i> cultivation and biosynthesis of anti-phytopathogens silver nanoparticles. <i>J Nanomater Mol Nanotechnol</i>. <b>5</b>(1), (2016). </span> </li> <li tag="ref"> <a name="CR25" /> <span class="label" tagx="label" title="label">25.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">EL-Moslamy</span> <span class="given-names" tagx="given-names" title="given-names">HS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Elkady</span> <span class="given-names" tagx="given-names" title="given-names">FM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rezk</span> <span class="given-names" tagx="given-names" title="given-names">HA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Abdel-Fattah</span> <span class="given-names" tagx="given-names" title="given-names">RY</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Applying Taguchi design and large-scale strategy for mycosynthesis of nano-silver from endophytic <i>Trichoderma harzianum</i> SYA.F4 and its application against phytopathogens </span> <span class="source" tagx="source" title="source">Sci. Rep.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">45297</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/srep45297">10.1038/srep45297</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28349997">28349997</a> </span> </span> </li> <li tag="ref"> <a name="CR26" /> <span class="label" tagx="label" title="label">26.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sandhu</span> <span class="given-names" tagx="given-names" title="given-names">SS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shukla</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shukla</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Biosynthesis of silver nanoparticles by endophytic fungi: Its mechanism, characterization techniques and antimicrobial potential </span> <span class="source" tagx="source" title="source">African Journal of Biotechnology.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="issue" tagx="issue" title="issue">14</span> <span class="fpage" tagx="fpage" title="fpage">683</span> <span class="lpage" tagx="lpage" title="lpage">698</span> <span class="pub-id"> <a href="https://dx.doi.org/10.5897/AJB2017.15873">10.5897/AJB2017.15873</a> </span> </span> </li> <li tag="ref"> <a name="CR27" /> <span class="label" tagx="label" title="label">27.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Zothanpuia <i>et al</i>. Production of potent antimicrobial compounds from <i>Streptomyces cyaneofuscatus</i> Associated with Fresh Water Sediment. <i>Front</i>. <i>Microbiol</i>. <b>8</b>(68), (2017). </span> </li> <li tag="ref"> <a name="CR28" /> <span class="label" tagx="label" title="label">28.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khushboo</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gangwar</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Antifungal activity of endophytic <i>actinomyetes</i> against <i>Fusarium</i> wilt ( <i>Fusarium oxysporum</i>) of Banana Trees (Musa acuminata) </span> <span class="source" tagx="source" title="source">Int. J. Curr. Microbiol. App. Sci.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="issue" tagx="issue" title="issue">6</span> <span class="fpage" tagx="fpage" title="fpage">328</span> <span class="lpage" tagx="lpage" title="lpage">337</span> <span class="pub-id"> <a href="https://dx.doi.org/10.20546/ijcmas.2017.606.039">10.20546/ijcmas.2017.606.039</a> </span> </span> </li> <li tag="ref"> <a name="CR29" /> <span class="label" tagx="label" title="label">29.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hassan</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Biocontrol potential of an endophytic <i>Streptomyces</i> sp. strain MBCN152-1 against <i>Alternaria brassicicola</i> on Cabbage Plug Seedlings </span> <span class="source" tagx="source" title="source">Microbes Environ.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">32</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">133</span> <span class="lpage" tagx="lpage" title="lpage">141</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1264/jsme2.ME17014">10.1264/jsme2.ME17014</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28515390">28515390</a> </span> </span> </li> <li tag="ref"> <a name="CR30" /> <span class="label" tagx="label" title="label">30.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Matsumoto</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takahashi</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Endophytic actinomycetes: promising source of novel bioactive compounds</span> <span class="source" tagx="source" title="source">The Journal of Antibiotics.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">70</span> <span class="fpage" tagx="fpage" title="fpage">514</span> <span class="lpage" tagx="lpage" title="lpage">519</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/ja.2017.20">10.1038/ja.2017.20</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28270688">28270688</a> </span> </span> </li> <li tag="ref"> <a name="CR31" /> <span class="label" tagx="label" title="label">31.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Golinska</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Endophytic actinobacteria of medicinal plants: diversity and bioactivity</span> <span class="source" tagx="source" title="source">Antonie van Leeuwenhoek.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">108</span> <span class="fpage" tagx="fpage" title="fpage">267</span> <span class="lpage" tagx="lpage" title="lpage">289</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s10482-015-0502-7">10.1007/s10482-015-0502-7</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26093915">26093915</a> </span> </span> </li> <li tag="ref"> <a name="CR32" /> <span class="label" tagx="label" title="label">32.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gangwar</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dogra</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gupta</span> <span class="given-names" tagx="given-names" title="given-names">PU</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kharwar</span> <span class="given-names" tagx="given-names" title="given-names">NR</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Diversity and biopotential of endophytic actinomycetes from three medicinal plants in India. <i>African</i> </span> <span class="source" tagx="source" title="source">Journal of Microbiology Research.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">184</span> <span class="lpage" tagx="lpage" title="lpage">191</span> </span> </li> <li tag="ref"> <a name="CR33" /> <span class="label" tagx="label" title="label">33.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ying</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bai</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Advance in endophytic microbes from medicinal plants. <i>Int J Pharm</i> </span> <span class="source" tagx="source" title="source">Sci Res.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="issue" tagx="issue" title="issue">5</span> <span class="fpage" tagx="fpage" title="fpage">1589</span> <span class="lpage" tagx="lpage" title="lpage">1600</span> </span> </li> <li tag="ref"> <a name="CR34" /> <span class="label" tagx="label" title="label">34.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Sivalingama, P., Arjunana, A., Deepikaa, T., Nivetha, C. &amp;amp; Muthuselvam, M. Endophytic <i>Streptomyces sp</i>. BDUSMP R07 from <i>Solanum trilobatum</i> ( <i>Thuthuvalai</i>) for the biosynthesis of bactericidal silver nanoparticles against multi resistant <i>Enterococcus</i> spp. <i>Online International Interdisciplinary Research Journal</i>. <b>4</b>(4), (2014). </span> </li> <li tag="ref"> <a name="CR35" /> <span class="label" tagx="label" title="label">35.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Machavariani</span> <span class="given-names" tagx="given-names" title="given-names">GN</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ivankova</span> <span class="given-names" tagx="given-names" title="given-names">DT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sineva</span> <span class="given-names" tagx="given-names" title="given-names">NO</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Terekhova</span> <span class="given-names" tagx="given-names" title="given-names">PL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Isolation of endophytic <i>Actinomycetes</i> from medicinal plants of the Moscow region, Russia </span> <span class="source" tagx="source" title="source">World Applied Sciences Journal.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">30</span> <span class="issue" tagx="issue" title="issue">11</span> <span class="fpage" tagx="fpage" title="fpage">1599</span> <span class="lpage" tagx="lpage" title="lpage">1604</span> </span> </li> <li tag="ref"> <a name="CR36" /> <span class="label" tagx="label" title="label">36.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title"> <i>Streptomyces endophyticus</i> sp. nov., an endophytic actinomycete isolated from <i>Artemisia annua L</i> </span> <span class="source" tagx="source" title="source">International Journal of Systematic and Evolutionary Microbiology.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">63</span> <span class="fpage" tagx="fpage" title="fpage">224</span> <span class="lpage" tagx="lpage" title="lpage">229</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1099/ijs.0.035725-0">10.1099/ijs.0.035725-0</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22389285">22389285</a> </span> </span> </li> <li tag="ref"> <a name="CR37" /> <span class="label" tagx="label" title="label">37.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Sunkar, S. &amp;amp; Valli, N. Endophytes as potential nanofactories. <i>Internat</i>ional Journal of Chemical, Environmental &amp;amp; Biological Sciences. <b>1</b>(3), (2013). </span> </li> <li tag="ref"> <a name="CR38" /> <span class="label" tagx="label" title="label">38.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kandpal</span> <span class="given-names" tagx="given-names" title="given-names">CK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jain</span> <span class="given-names" tagx="given-names" title="given-names">AD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tripathi</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar</span> <span class="given-names" tagx="given-names" title="given-names">ST</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Isolation and screening of endophytic <i>actinomycetes</i> producing antibacterial compound from <i>Citrus aurantifolia</i> Fruit. <i>European</i> </span> <span class="source" tagx="source" title="source">Journal of Experimental Biology.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="issue" tagx="issue" title="issue">5</span> <span class="fpage" tagx="fpage" title="fpage">1733</span> <span class="lpage" tagx="lpage" title="lpage">1737</span> </span> </li> <li tag="ref"> <a name="CR39" /> <span class="label" tagx="label" title="label">39.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wahab</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ansari</span> <span class="given-names" tagx="given-names" title="given-names">GS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dar</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">SY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shin</span> <span class="given-names" tagx="given-names" title="given-names">SH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Synthesis of magnesium oxide nanoparticles by sol-gel process</span> <span class="source" tagx="source" title="source">Materials Science Forum</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">558-559</span> <span class="fpage" tagx="fpage" title="fpage">983</span> <span class="lpage" tagx="lpage" title="lpage">986</span> <span class="pub-id"> <a href="https://dx.doi.org/10.4028/www.scientific.net/MSF.558-559.983">10.4028/www.scientific.net/MSF.558-559.983</a> </span> </span> </li> <li tag="ref"> <a name="CR40" /> <span class="label" tagx="label" title="label">40.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sadr</span> <span class="given-names" tagx="given-names" title="given-names">KM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mirhosseini</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rahimi</span> <span class="given-names" tagx="given-names" title="given-names">GH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Effects of combination of magnesium and zinc oxide nanoparticles and heat on <i>Escherichia coli</i> and <i>Staphylococcus aureus</i> bacteria in milk </span> <span class="source" tagx="source" title="source">Nanomed. J.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">49</span> <span class="lpage" tagx="lpage" title="lpage">56</span> </span> </li> <li tag="ref"> <a name="CR41" /> <span class="label" tagx="label" title="label">41.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tripathi</span> <span class="given-names" tagx="given-names" title="given-names">CKM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Praveen</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singh</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">&amp;amp; Bihari, V. Production of antibacterial and antifungal metabolites <i>Streptomyces violaceusniger</i> and media optimization studies for the maximum metabolite production </span> <span class="source" tagx="source" title="source">Medicinal Chemistry Research.</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="fpage" tagx="fpage" title="fpage">790</span> <span class="lpage" tagx="lpage" title="lpage">799</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00044-004-0118-3">10.1007/s00044-004-0118-3</a> </span> </span> </li> <li tag="ref"> <a name="CR42" /> <span class="label" tagx="label" title="label">42.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rajan</span> <span class="given-names" tagx="given-names" title="given-names">MB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kannabiran</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Extraction and identification of antibacterial secondary metabolites from marine <i>Streptomyces</i> sp. VITBRK2 </span> <span class="source" tagx="source" title="source">Int J Mol Cell Med.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">130</span> <span class="lpage" tagx="lpage" title="lpage">137</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25317399">25317399</a> </span> </span> </li> <li tag="ref"> <a name="CR43" /> <span class="label" tagx="label" title="label">43.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Ahsan, T., Chen, J., Zhao, X., Irfan, M. &amp;amp; Wu, Y. Extraction and identification of bioactive compounds (eicosane and dibutyl phthalate) produced by <i>Streptomyces</i> strain KX852460 for the biological control of <i>Rhizoctonia solani</i> AG-3 strain KX852461 to control target spot disease in tobacco leaf. <i>AMB Expr</i>. <b>7</b>(54), (2017). </span> </li> <li tag="ref"> <a name="CR44" /> <span class="label" tagx="label" title="label">44.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Dzhavakhiya, V. <i>et al</i>. Scaling up a <i>virginiamycin</i> production by a high-yield <i>Streptomyces virginiae</i> VKM Ac-2738D strain using adsorbing resin addition and fed-batch fermentation under controlled conditions. <i>Biotech</i>. <b>6</b>(240), (2016). </span> </li> <li tag="ref"> <a name="CR45" /> <span class="label" tagx="label" title="label">45.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baoxin</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiangjing</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wensheng</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Optimization of fermentation medium for enhanced production of milbemycin by a mutant of <i>Streptomyces bingchenggensis</i> BC-X-1 using response surface methodology </span> <span class="source" tagx="source" title="source">African Journal of Biotechnology.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="issue" tagx="issue" title="issue">37</span> <span class="fpage" tagx="fpage" title="fpage">7225</span> <span class="lpage" tagx="lpage" title="lpage">7235</span> </span> </li> <li tag="ref"> <a name="CR46" /> <span class="label" tagx="label" title="label">46.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bruheim</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sletta</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bibb</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">White</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Levine</span> <span class="given-names" tagx="given-names" title="given-names">WD</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">High-yield actinorhodin production in fed-batch culture by a <i>Streptomyces lividans strain overexpressing the pathway-specific activator</i> gene actII-ORF4 </span> <span class="source" tagx="source" title="source">Journal of Industrial Microbiology &amp;amp; Biotechnology.</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">28</span> <span class="fpage" tagx="fpage" title="fpage">103</span> <span class="lpage" tagx="lpage" title="lpage">111</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/sj/jim/7000219">10.1038/sj/jim/7000219</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12074050">12074050</a> </span> </span> </li> <li tag="ref"> <a name="CR47" /> <span class="label" tagx="label" title="label">47.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Abbas</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Edwards</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Effects of metals on <i>Streptomyces coelicolor</i> growth and actinorhodin production. <i>Applied</i> </span> <span class="source" tagx="source" title="source">Environmental Microbiology.</span> <span class="year" tagx="year" title="year">1990</span> <span class="volume" tagx="volume" title="volume">56</span> <span class="fpage" tagx="fpage" title="fpage">675</span> <span class="lpage" tagx="lpage" title="lpage">680</span> </span> </li> <li tag="ref"> <a name="CR48" /> <span class="label" tagx="label" title="label">48.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Basak</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Majumdar</span> <span class="given-names" tagx="given-names" title="given-names">KS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Utilization of Carbon and Nitrogen Sources by <i>Streptomyces kanamyceticus</i> for Kanamycin Production </span> <span class="source" tagx="source" title="source">Antimicrobial agents and chemotherapy.</span> <span class="year" tagx="year" title="year">1973</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">6</span> <span class="lpage" tagx="lpage" title="lpage">10</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/AAC.4.1.6">10.1128/AAC.4.1.6</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/4791483">4791483</a> </span> </span> </li> <li tag="ref"> <a name="CR49" /> <span class="label" tagx="label" title="label">49.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sheetal</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Prachi</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pinkey</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Modi</span> <span class="given-names" tagx="given-names" title="given-names">HA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Comparative study of extraction methods for intracellularly produced glucose isomerase by <i>Streptomyces sp</i> </span> <span class="source" tagx="source" title="source">SB – AII4. Int. Res. J. Biological Sci.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="issue" tagx="issue" title="issue">10</span> <span class="fpage" tagx="fpage" title="fpage">43</span> <span class="lpage" tagx="lpage" title="lpage">50</span> </span> </li> <li tag="ref"> <a name="CR50" /> <span class="label" tagx="label" title="label">50.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Abd-Elsalam</span> <span class="given-names" tagx="given-names" title="given-names">KA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alghuthaymi</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Nanobiofungicides: are they the Next-Generation of Fungicides? <i>J Nanotech</i> </span> <span class="source" tagx="source" title="source">Mater Sci.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">3</span> </span> </li> <li tag="ref"> <a name="CR51" /> <span class="label" tagx="label" title="label">51.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Elsayed</span> <span class="given-names" tagx="given-names" title="given-names">AE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Omar</span> <span class="given-names" tagx="given-names" title="given-names">GH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">El-Enshasy</span> <span class="given-names" tagx="given-names" title="given-names">AH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Development of Fed-Batch cultivation strategy for efficient Oxytetracycline Production by <i>Streptomyces rimosus</i> at Semi-Industrial Scale </span> <span class="source" tagx="source" title="source">Braz. Arch. Biol. Technol.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">58</span> <span class="issue" tagx="issue" title="issue">5</span> <span class="fpage" tagx="fpage" title="fpage">676</span> <span class="lpage" tagx="lpage" title="lpage">685</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1590/S1516-89132015050184">10.1590/S1516-89132015050184</a> </span> </span> </li> <li tag="ref"> <a name="CR52" /> <span class="label" tagx="label" title="label">52.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hata</span> <span class="given-names" tagx="given-names" title="given-names">ME</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sijam</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ahmad</span> <span class="given-names" tagx="given-names" title="given-names">AMZ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yusof</span> <span class="given-names" tagx="given-names" title="given-names">TM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Azman</span> <span class="given-names" tagx="given-names" title="given-names">AN</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title"> <i>In vitro</i> antimicrobial assay of actinomycetes in Rice against Xanthomonas oryzae pv. oryzicolaand as potential plant growth promoter </span> <span class="source" tagx="source" title="source">Braz. Arch. Biol. Technol.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">58</span> <span class="issue" tagx="issue" title="issue">6</span> <span class="fpage" tagx="fpage" title="fpage">821</span> <span class="lpage" tagx="lpage" title="lpage">832</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1590/S1516-89132015060263">10.1590/S1516-89132015060263</a> </span> </span> </li> <li tag="ref"> <a name="CR53" /> <span class="label" tagx="label" title="label">53.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khodair</span> <span class="given-names" tagx="given-names" title="given-names">TD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Abed</span> <span class="given-names" tagx="given-names" title="given-names">HA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Majeed</span> <span class="given-names" tagx="given-names" title="given-names">GS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Synthesis and structural characterization of MgO nanoparticles</span> <span class="source" tagx="source" title="source">IJARSET.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="issue" tagx="issue" title="issue">7</span> <span class="fpage" tagx="fpage" title="fpage">2400</span> <span class="lpage" tagx="lpage" title="lpage">2406</span> </span> </li> <li tag="ref"> <a name="CR54" /> <span class="label" tagx="label" title="label">54.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Makarov</span> <span class="given-names" tagx="given-names" title="given-names">VV</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Green nanotechnologies: synthesis of metal nanoparticles using plants</span> <span class="source" tagx="source" title="source">Acta naturae.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="issue" tagx="issue" title="issue">20</span> <span class="fpage" tagx="fpage" title="fpage">35</span> <span class="lpage" tagx="lpage" title="lpage">44</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24772325">24772325</a> </span> </span> </li> <li tag="ref"> <a name="CR55" /> <span class="label" tagx="label" title="label">55.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Singh, V. <i>et al</i>. Strategies for Fermentation Medium Optimization: An In-Depth Review. Front. <i>Microbiol</i>. <b>7</b>(2087), (2017). </span> </li> <li tag="ref"> <a name="CR56" /> <span class="label" tagx="label" title="label">56.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sabourimanesh</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rashidi</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marjani</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Motamedara</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Valizadeh</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Effects of Process Parameters on the size of nanostructure magnesium oxide synthesized by a surfactant and ligand assisted wet chemical method </span> <span class="source" tagx="source" title="source">Crystal structure theory and applications.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">28</span> <span class="lpage" tagx="lpage" title="lpage">34</span> <span class="pub-id"> <a href="https://dx.doi.org/10.4236/csta.2015.42004">10.4236/csta.2015.42004</a> </span> </span> </li> <li tag="ref"> <a name="CR57" /> <span class="label" tagx="label" title="label">57.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sundrarajan</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suresh</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rajivgandhi</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A comparative study on antibacterial properties of MgO nanoparticles prepared under different calcinations temperature </span> <span class="source" tagx="source" title="source">Digest Journal of Nanomaterials and Biostructures.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">983</span> <span class="lpage" tagx="lpage" title="lpage">989</span> </span> </li> <li tag="ref"> <a name="CR58" /> <span class="label" tagx="label" title="label">58.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oktiawati</span> <span class="given-names" tagx="given-names" title="given-names">YU</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mohamed</span> <span class="given-names" tagx="given-names" title="given-names">NM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burhanudin</span> <span class="given-names" tagx="given-names" title="given-names">ZA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Applications of Taguchi method for optimization of dye solar cell design (Aplikasi Kaedah Taguchi untuk Pengoptimuman Reka Bentuk Sel Pewarna Suria) </span> <span class="source" tagx="source" title="source">Sains Malaysiana.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">46</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">503</span> <span class="lpage" tagx="lpage" title="lpage">508</span> <span class="pub-id"> <a href="https://dx.doi.org/10.17576/jsm-2017-4603-19">10.17576/jsm-2017-4603-19</a> </span> </span> </li> <li tag="ref"> <a name="CR59" /> <span class="label" tagx="label" title="label">59.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Balouiri</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sadiki</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ibnsouda</span> <span class="given-names" tagx="given-names" title="given-names">KS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Methods for <i>in vitro</i> evaluating antimicrobial activity: A review </span> <span class="source" tagx="source" title="source">Journal of Pharmaceutical Analysis.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">71</span> <span class="lpage" tagx="lpage" title="lpage">79</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.jpha.2015.11.005">10.1016/j.jpha.2015.11.005</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29403965">29403965</a> </span> </span> </li> <li tag="ref"> <a name="CR60" /> <span class="label" tagx="label" title="label">60.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ansari</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Evaluation of antibacterial activity of silver nanoparticles against MSSA and MRSA on isolates from skin infections </span> <span class="source" tagx="source" title="source">Biology and Medicine.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">141</span> <span class="lpage" tagx="lpage" title="lpage">146</span> </span> </li> <li tag="ref"> <a name="CR61" /> <span class="label" tagx="label" title="label">61.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">He, Y. <i>et al</i>. Study on the mechanism of antibacterial action of magnesium oxide nanoparticles against foodborne pathogens. <i>J Nanobiotechnol</i>. <b>14</b>( <b>54</b>), (2016). </span> </li> <li tag="ref"> <a name="CR62" /> <span class="label" tagx="label" title="label">62.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Avanzato, P. C. Follieri, M. G. &amp;amp; Banerjee, A. I.Biomimetic Synthesis and Antibacterial Characteristics of Magnesium Oxide–Germanium dioxide Nanocomposite Powders. J <i>ournal of composite materials</i>. <b>43</b>(8), (2009). </span> </li> <li tag="ref"> <a name="CR63" /> <span class="label" tagx="label" title="label">63.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Taechowisan</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lu</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shen</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lumyong</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Secondary metabolites from endophytic <i>Streptomyces aureofaciens</i> CMUAc130 and their antifungal activity </span> <span class="source" tagx="source" title="source">Microbiology.</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">151</span> <span class="fpage" tagx="fpage" title="fpage">1691</span> <span class="lpage" tagx="lpage" title="lpage">1695</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1099/mic.0.27758-0">10.1099/mic.0.27758-0</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15870476">15870476</a> </span> </span> </li> <li tag="ref"> <a name="CR64" /> <span class="label" tagx="label" title="label">64.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Ikechi–Nwogu, G. &amp;amp; Elenwo, N. E. Comparative evaluation of growth media for the cultivation of fungal cultures <i>Chinyerum</i>, <i>J Plant Pathol Microb</i>. <b>3</b>( <b>6</b>), (2012). </span> </li> <li tag="ref"> <a name="CR65" /> <span class="label" tagx="label" title="label">65.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Kontroab, H. M. <i>et al</i>. Nutrient effects on ten <i>Streptomyces</i> spp. Sporulation. Communicating Current Research and Educational Topics and Trends in Applied Microbiology. Vol. <b>1</b> (A. Méndez-Vilas Ed) Ch. Environmental Microbiology, 137–142. (FORMATEX, 2007). </span> </li> <li tag="ref"> <a name="CR66" /> <span class="label" tagx="label" title="label">66.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Voelker</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Altaba</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Nitrogen source governs the patterns of growth and pristinamycin production in <i>Streptomyces pristinaespiralis</i> </span> <span class="source" tagx="source" title="source">Microbiology.</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">147</span> <span class="fpage" tagx="fpage" title="fpage">2447</span> <span class="lpage" tagx="lpage" title="lpage">2459</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1099/00221287-147-9-2447">10.1099/00221287-147-9-2447</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11535785">11535785</a> </span> </span> </li> <li tag="ref"> <a name="CR67" /> <span class="label" tagx="label" title="label">67.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">MS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kojima</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Demain</span> <span class="given-names" tagx="given-names" title="given-names">AL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Effect of nitrogen source on biosynthesis of rapamycin by Streptomyces hygroscopicus</span> <span class="source" tagx="source" title="source">Journal of Industrial Microbiology and Biotechnology.</span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">83</span> <span class="lpage" tagx="lpage" title="lpage">86</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/sj.jim.2900434">10.1038/sj.jim.2900434</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9366089">9366089</a> </span> </span> </li> <li tag="ref"> <a name="CR68" /> <span class="label" tagx="label" title="label">68.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Radwan</span> <span class="given-names" tagx="given-names" title="given-names">HH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moussa</span> <span class="given-names" tagx="given-names" title="given-names">MI</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alsarra</span> <span class="given-names" tagx="given-names" title="given-names">AI</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Optimization of a fed-batch fermentation process for production of bleomycin by <i>Streptomyces mobaraensis</i> ATCC 15003 </span> <span class="source" tagx="source" title="source">African Journal of Biotechnology.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="issue" tagx="issue" title="issue">9</span> <span class="fpage" tagx="fpage" title="fpage">1690</span> <span class="lpage" tagx="lpage" title="lpage">1695</span> </span> </li> <li tag="ref"> <a name="CR69" /> <span class="label" tagx="label" title="label">69.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thenmozhi</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sindhura</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kannabiran</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Characterization of antioxidant activity of <i>Streptomyces</i> species VITTK3 isolated from Puducherry Coast, India </span> <span class="source" tagx="source" title="source">J. Adv. Sci. Res</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">1</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">46</span> <span class="lpage" tagx="lpage" title="lpage">52</span> </span> </li> <li tag="ref"> <a name="CR70" /> <span class="label" tagx="label" title="label">70.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pandey</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shukla</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Majumdar</span> <span class="given-names" tagx="given-names" title="given-names">KS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Utilization of carbon and nitrogen sources by <i>Streptomyces kanamyceticus</i> M 27 for the production of an Anti bacterial antibiotic </span> <span class="source" tagx="source" title="source">African Journal of Biotechnology.</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="issue" tagx="issue" title="issue">9</span> <span class="fpage" tagx="fpage" title="fpage">909</span> <span class="lpage" tagx="lpage" title="lpage">910</span> </span> </li> <li tag="ref"> <a name="CR71" /> <span class="label" tagx="label" title="label">71.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Wentzel, A. <i>et al</i>. Optimized submerged batch fermentation strategy for systems scale studies of metabolic switching in <i>Streptomyces coelicolor</i> A3(2). <i>BMC Systems Biology</i>. <b>6</b>(59), (2012). </span> </li> <li tag="ref"> <a name="CR72" /> <span class="label" tagx="label" title="label">72.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Zhang, L., Zheng, X., Jin, Z., Hu, S. &amp;amp; He, M. Optimization of fermentation conditions for pristinamycin production by immobilized <i>Streptomyces pristinaespiralis</i> using response surface methodology. <i>Electronic Journal of Biotechnology North America</i>. <b>15</b>, (2012). </span> </li> <li tag="ref"> <a name="CR73" /> <span class="label" tagx="label" title="label">73.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hee-Seok</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Statistical optimization of growth medium for the production of the Entomopathogenic and Phytotoxic Cyclic Depsipeptide Beauvericin from <i>Fusarium oxysporum</i> KFCC 11363P </span> <span class="source" tagx="source" title="source">J. Microbiol. Biotechnol.</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">18</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">138</span> <span class="lpage" tagx="lpage" title="lpage">144</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18239431">18239431</a> </span> </span> </li> <li tag="ref"> <a name="CR74" /> <span class="label" tagx="label" title="label">74.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Ilić, S. B. The impact of composition and rheological characteristics of nutritive medium on kinetic of antibiotics production by <i>Streptomyces hygroscopicus</i> CH-7. University of Niš, Niš. Serbia, <i>Ph</i>. <i>Thesis</i>, (2010). </span> </li> <li tag="ref"> <a name="CR75" /> <span class="label" tagx="label" title="label">75.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tadijan</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Effect of carbon sources on the production of the biofungicide by <i>Streptomyces hygroscopicus</i>. <i>Acta Univ</i>. <i>Sapientiae</i> </span> <span class="source" tagx="source" title="source">Alimentaria.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">54</span> <span class="lpage" tagx="lpage" title="lpage">62</span> </span> </li> <li tag="ref"> <a name="CR76" /> <span class="label" tagx="label" title="label">76.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jahan</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maeda</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Matsuoka</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sugimoto</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kurata</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Development of an accurate kinetic model for the central carbon metabolism of <i>Escherichia coli</i> </span> <span class="source" tagx="source" title="source">Microb Cell Fact.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">15</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">112</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s12934-016-0511-x">10.1186/s12934-016-0511-x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27329289">27329289</a> </span> </span> </li> <li tag="ref"> <a name="CR77" /> <span class="label" tagx="label" title="label">77.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Siddiqi, S. K. &amp;amp; Husen, A. Fabrication of Metal Nanoparticles from Fungi and metal salts: scope and application. <i>Nanoscale Research Letters</i>. <b>11</b>(98), (2016). </span> </li> <li tag="ref"> <a name="CR78" /> <span class="label" tagx="label" title="label">78.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mirhosseini</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Evaluation of antibacterial effect of magnesium oxide nanoparticles with nisin and heat in milk </span> <span class="source" tagx="source" title="source">Nanomed. J.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">135</span> <span class="lpage" tagx="lpage" title="lpage">142</span> </span> </li> <li tag="ref"> <a name="CR79" /> <span class="label" tagx="label" title="label">79.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Karimiyan</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Najafzadeh</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ghorbanpour</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hekmati-Moghaddam</span> <span class="given-names" tagx="given-names" title="given-names">HS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Antifungal Effect of magnesium oxide, zinc oxide, silicon oxide and copper oxide nanoparticles against <i>Candida albicans</i>. <i>Zahedan</i> </span> <span class="source" tagx="source" title="source">J Res Med Sci.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">17</span> <span class="issue" tagx="issue" title="issue">10</span> <span class="fpage" tagx="fpage" title="fpage">2179</span> </span> </li> <li tag="ref"> <a name="CR80" /> <span class="label" tagx="label" title="label">80.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Gatoo, A.M. <i>et al</i>. Physicochemical Properties of Nanomaterials: Implication in Associated Toxic Manifestations. <i>BioMed Research International</i>. ( <b>2014</b>). </span> </li> <li tag="ref"> <a name="CR81" /> <span class="label" tagx="label" title="label">81.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Gurunathan, S. Rapid biological synthesis of silver nanoparticles and their enhanced antibacterial effects against <i>Escherichia fergusonii</i> and <i>Streptococcus mutans</i>. <i>Arabian Journal of Chemistry</i>. ( <b>2014</b>). </span> </li> <li tag="ref"> <a name="CR82" /> <span class="label" tagx="label" title="label">82.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Das</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Green synthesized silver nanoparticles destroy multidrug resistant bacteria via reactive oxygen species mediated membrane damage </span> <span class="source" tagx="source" title="source">Arabian Journal of Chemistry.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">862</span> <span class="lpage" tagx="lpage" title="lpage">876</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.arabjc.2015.08.008">10.1016/j.arabjc.2015.08.008</a> </span> </span> </li> <li tag="ref"> <a name="CR83" /> <span class="label" tagx="label" title="label">83.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cavassin</span> <span class="given-names" tagx="given-names" title="given-names">DE</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Comparison of methods to detect the <i>in vitro</i> activity of silver nanoparticles (AgNP) against multidrug resistant bacteria </span> <span class="source" tagx="source" title="source">J Nanobiotechnol.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="fpage" tagx="fpage" title="fpage">64</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s12951-015-0120-6">10.1186/s12951-015-0120-6</a> </span> </span> </li> <li tag="ref"> <a name="CR84" /> <span class="label" tagx="label" title="label">84.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Balakumaran</span> <span class="given-names" tagx="given-names" title="given-names">DM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ramachandran</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Balashanmugam</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mukeshkumar</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kalaichelvan</span> <span class="given-names" tagx="given-names" title="given-names">TP</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Mycosynthesis of silver and gold nanoparticles: Optimization, characterization and antimicrobial activity against human pathogens </span> <span class="source" tagx="source" title="source">Microbiological Research.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">182</span> <span class="fpage" tagx="fpage" title="fpage">8</span> <span class="lpage" tagx="lpage" title="lpage">20</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.micres.2015.09.009">10.1016/j.micres.2015.09.009</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26686609">26686609</a> </span> </span> </li> <li tag="ref"> <a name="CR85" /> <span class="label" tagx="label" title="label">85.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shaker</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shaaban</span> <span class="given-names" tagx="given-names" title="given-names">MI</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Synthesis of silver nanoparticles with antimicrobial and anti-adherence activities against multidrug-resistant isolates from <i>Acinetobacter baumannii</i> </span> <span class="source" tagx="source" title="source">J Taibah Univ Med Sc.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="issue" tagx="issue" title="issue">4</span> <span class="fpage" tagx="fpage" title="fpage">291</span> <span class="lpage" tagx="lpage" title="lpage">297</span> </span> </li> <li tag="ref"> <a name="CR86" /> <span class="label" tagx="label" title="label">86.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Abbaszadegan, A. <i>et al</i>. The Effect of Charge at the Surface of Silver Nanoparticles on Antimicrobial Activity against Gram-Positive and Gram-Negative Bacteria: A Preliminary Study. <i>Journal of Nanomaterials</i>. (2015). </span> </li> </ul> </div> </div>  </body></html>