<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Sci Rep</span> <span class="iso-abbrev" title="iso-abbrev">Sci Rep</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Scientific Reports</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">2045-2322</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5847565">5847565</a> </span> <span class="publisher-id" title="publisher-id">22631</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41598-018-22631-z">10.1038/s41598-018-22631-z</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
IMPPAT: A curated database of <div class="u_UNKNOWN">I</div>ndian <div class="u_UNKNOWN">M</div>edicinal <div class="u_UNKNOWN">P</div>lants, <div class="u_UNKNOWN">P</div>hytochemistry <div class="u_UNKNOWN">A</div>nd <div class="u_UNKNOWN">T</div>herapeutics </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mohanraj</span> <span class="given-names" tagx="given-names" title="given-names">Karthikeyan</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Karthikeyan</span> <span class="given-names" tagx="given-names" title="given-names">Bagavathy Shanmugam</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vivek-Ananth</span> <span class="given-names" tagx="given-names" title="given-names">R. P.</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chand</span> <span class="given-names" tagx="given-names" title="given-names">R. P. Bharath</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aparna</span> <span class="given-names" tagx="given-names" title="given-names">S. R.</span> </span> <a href="#Aff2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mangalapandi</span> <span class="given-names" tagx="given-names" title="given-names">Pattulingam</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-6796-9604</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Samal</span> <span class="given-names" tagx="given-names" title="given-names">Areejit</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">asamal@imsc.res.in</span> </span> <a href="#Aff1">1</a> </span> <span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0004 1775 9822</span> <span class="institution-id" title="institution-id">grid.450257.1</span> <span class="institution" title="institution">The Institute of Mathematical Sciences (IMSc), </span> <span class="institution" title="institution">Homi Bhabha National Institute (HBNI), </span> </div> </span> <span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0004 0505 215X</span> <span class="institution-id" title="institution-id">grid.413015.2</span> <span class="institution" title="institution">Stella Maris College, </span> </div> </span> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-3-3</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2018-3-3</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2018</span> </span> <span class="volume" tagx="volume" title="volume">8</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">4329</span> <span class="history" title="history"> <span class="received" title="received">received: 2017-10-31</span> <span class="accepted" title="accepted">accepted: 2018-2-27</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , </span> <span class="license" title="license"> <span class="license-p" title="license-p"> <b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Phytochemicals of medicinal plants encompass a diverse chemical space for drug discovery. India is rich with a flora of indigenous medicinal plants that have been used for centuries in traditional Indian medicine to treat human maladies. A comprehensive online database on the phytochemistry of Indian medicinal plants will enable computational approaches towards natural product based drug discovery. In this direction, we present, IMPPAT, a manually curated database of 1742 <div class="u_UNKNOWN">I</div>ndian <div class="u_UNKNOWN">M</div>edicinal <div class="u_UNKNOWN">P</div>lants, 9596 <div class="u_UNKNOWN">P</div>hytochemicals, <div class="u_UNKNOWN">A</div>nd 1124 <div class="u_UNKNOWN">T</div>herapeutic uses spanning 27074 plant-phytochemical associations and 11514 plant-therapeutic associations. Notably, the curation effort led to a non-redundant <i>in silico</i> library of 9596 phytochemicals with standard chemical identifiers and structure information. Using cheminformatic approaches, we have computed the physicochemical, ADMET (absorption, distribution, metabolism, excretion, toxicity) and drug-likeliness properties of the IMPPAT phytochemicals. We show that the stereochemical complexity and shape complexity of IMPPAT phytochemicals differ from libraries of commercial compounds or diversity-oriented synthesis compounds while being similar to other libraries of natural products. Within IMPPAT, we have filtered a subset of 960 potential druggable phytochemicals, of which majority have no significant similarity to existing FDA approved drugs, and thus, rendering them as good candidates for prospective drugs. IMPPAT database is openly accessible at: <a href="https://cb.imsc.res.in/imppat">https://cb.imsc.res.in/imppat</a>. </p> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2018" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Natural products continue to play a significant role in pharmaceutical industry <sup> <a href="#CR1">1</a>– <a href="#CR4">4</a> </sup> as new sources of drugs. However, recently there has been a decline in the number of marketable drugs derived from natural sources <sup> <a href="#CR3">3</a>, <a href="#CR4">4</a> </sup>. Furthermore, the majority of these drugs fall into already known structural scaffolds as due importance has not been given to unexplored sources of natural products for drug discovery <sup> <a href="#CR4">4</a> </sup>. As a result, lately, there has been significant interest in applying interdisciplinary approaches <sup> <a href="#CR5">5</a> </sup> to expand the novel chemical scaffold libraries for drug discovery. </p> <p>India is well known for its practice of traditional medicine and ethnopharmacology <sup> <a href="#CR6">6</a> </sup>. It is noteworthy that traditional Indian medicinal formulations are multi-component mixtures whose therapeutic use is based on empirical knowledge rather than a mechanistic understanding of the active ingredients in the mixture <sup> <a href="#CR6">6</a> </sup>. Until recently, knowledge of traditional Indian medicine including important medicinal plants and their formulations were buried within books such as Indian Materia Medica <sup> <a href="#CR7">7</a> </sup> and Ayurveda Materia Medica <sup> <a href="#CR8">8</a> </sup>. The nondigital nature of this information limited their effective use towards new drug discovery <sup> <a href="#CR5">5</a> </sup>. Hence, digitization of this knowledge into a comprehensive database on Indian medicinal plants, phytochemistry and ethnopharmacology will enable researchers to apply computational approaches towards drug discovery. </p> <p>Availability of a curated database of information on plants, their associated natural products and a repository of their chemical structures, can help in <i>in silico</i> drug discovery. In this direction, there has been significant recent progress in the development of databases <sup> <a href="#CR9">9</a>– <a href="#CR17">17</a> </sup> on natural products with a focus on phytochemistry of edible and herbaceous plants. Examples of such databases include CVDHD <sup> <a href="#CR12">12</a> </sup>, KNAPSACK <sup> <a href="#CR13">13</a> </sup>, Nutrichem <sup> <a href="#CR9">9</a>, <a href="#CR10">10</a> </sup>, Phytochemica <sup> <a href="#CR11">11</a> </sup>, TCMID <sup> <a href="#CR15">15</a> </sup>, TCM@Taiwan <sup> <a href="#CR14">14</a> </sup> and TCM-Mesh <sup> <a href="#CR16">16</a> </sup> which can facilitate virtual screening of prospective drug compounds or aid in the investigation of plant-disease associations. However, from the perspective of traditional Indian medicine, there have been relatively few efforts to build online databases that include Indian medicinal plants, their phytochemicals and therapeutic uses. Previously, Polur <i>et al</i>. <sup> <a href="#CR18">18</a> </sup> compiled information on 295 ayurvedic Indian medicinal plants, their 1829 phytochemicals and therapeutic uses. Subsequently, Polur <i>et al</i>. <sup> <a href="#CR18">18</a> </sup> studied the structural similarity between their library of 1829 phytochemicals and drugs in the DrugBank <sup> <a href="#CR19">19</a> </sup> database to predict pharmacologically active natural compounds. Recently, the Phytochemica <sup> <a href="#CR11">11</a> </sup> database gathered information on 5 Indian medicinal plants and their 963 phytochemicals. In addition, Phytochemica <sup> <a href="#CR11">11</a> </sup> provided chemical structures and pharmacological properties of the phytochemicals within their database. Other efforts to build online databases for traditional Indian medicine has largely been limited to cataloguing medicinal plants and their therapeutic uses rather than capturing the phytochemicals that are vital for drug discovery. On the other hand, in contrast to the above mentioned online databases, more comprehensive databases are available for Chinese medicinal plants <sup> <a href="#CR14">14</a>– <a href="#CR16">16</a> </sup>. For example, TCM-MeSH <sup> <a href="#CR16">16</a> </sup> is an online database for traditional Chinese medicine which captures phytochemical compositions and therapeutic uses for more than 6000 Chinese medicinal plants. </p> <p>We therefore have built a manually curated database, IMPPAT, containing 1742 <div class="u_UNKNOWN">I</div>ndian <div class="u_UNKNOWN">M</div>edicinal <div class="u_UNKNOWN">P</div>lants, 9596 <div class="u_UNKNOWN">P</div>hytochemicals, <div class="u_UNKNOWN">A</div>nd 1124 <div class="u_UNKNOWN">T</div> herapeutic uses. In addition, the IMPPAT database has linked Indian medicinal plants to 974 openly accessible traditional Indian medicinal formulations. Importantly, our curation efforts have led to a non-redundant <i>in silico</i> chemical library of 9596 phytochemicals with two-dimensional (2D) and three-dimensional (3D) chemical structures. For the 9596 phytochemicals in our database, we have computed physicochemical properties and predicted Absorption, distribution, metabolism, excretion and toxicity (ADMET) properties using cheminformatic tools <sup> <a href="#CR20">20</a>– <a href="#CR22">22</a> </sup>. We then employed cheminformatic approaches to evaluate the drug-likeliness of the phytochemicals in our <i>in silico</i> chemical library using multiple scoring schemes such as Lipinski’s rule of five (RO5) <sup> <a href="#CR23">23</a> </sup>, Oral PhysChem Score (Traffic Lights) <sup> <a href="#CR24">24</a> </sup>, GlaxoSmithKline’s (GSK’s) 4/400 <sup> <a href="#CR25">25</a> </sup>, Pfizer’s 3/75 <sup> <a href="#CR26">26</a> </sup>, Veber rule <sup> <a href="#CR27">27</a> </sup> and Egan rule <sup> <a href="#CR28">28</a> </sup>. We found a subset of 960 phytochemicals of Indian medicinal plants that are potentially druggable in our chemical library of 9596 phytochemicals based on multiple scoring schemes. We also provide predicted interactions between phytochemicals in our database and human target proteins from STITCH <sup> <a href="#CR29">29</a> </sup> database. Table <a href="#Tab1">1</a> provides a comparison of the IMPPAT database with previous efforts by Polur <i>et al</i>. <sup> <a href="#CR18">18</a> </sup> and Phytochemica <sup> <a href="#CR11">11</a> </sup> to build dedicated digital resource on phytochemical composition of Indian medicinal plants. In summary, IMPPAT is the largest database on phytochemicals of Indian medicinal plants to date, and this resource is a culmination of our efforts to digitize the wealth of information contained within traditional Indian medicine. IMPPAT provides an integrated platform to apply cheminformatic <sup> <a href="#CR30">30</a> </sup> approaches to accelerate natural product based drug discovery. IMPPAT is openly accessible at: <a href="https://cb.imsc.res.in/imppat">https://cb.imsc.res.in/imppat</a>. <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  </div></p><p>Comparison of IMPPAT with earlier databases on phytochemical composition of Indian medicinal plants. </p>  <table> <thead> <tr> <th>Database</th> <th>IMPPAT</th> <th>Phytochemica <sup> <a href="#CR11">11</a> </sup> </th> <th>Polur <i>et al</i>. <sup> <a href="#CR18">18</a> </sup> </th> </tr> </thead> <tbody> <tr> <td> <b>Basic statistics</b> </td> </tr> <tr> <td>Number of Indian medicinal plants</td> <td>1742</td> <td>5</td> <td>295</td> </tr> <tr> <td>Number of phytochemicals</td> <td>9596</td> <td>963</td> <td>1829</td> </tr> <tr> <td> <b>Type of associations</b> </td> </tr> <tr> <td>Plant-phytochemical associations</td> <td>Yes</td> <td>Yes</td> <td>Yes</td> </tr> <tr> <td>Plant-therapeutic use associations</td> <td>Yes</td> <td>No</td> <td>Yes</td> </tr> <tr> <td>Plant-medicinal formulation associations</td> <td>Yes</td> <td>No</td> <td>No</td> </tr> <tr> <td>Phytochemical-human target protein associations</td> <td>Yes</td> <td>No</td> <td>Yes</td> </tr> <tr> <td>Plant part-phytochemical associations</td> <td>No</td> <td>Yes</td> <td>No</td> </tr> <tr> <td> <b>Additional Features</b> </td> </tr> <tr> <td>Web interface</td> <td>Yes</td> <td>Yes</td> <td>No</td> </tr> <tr> <td>Availability of 2D structure of phytochemicals</td> <td>Yes</td> <td>No</td> <td>No</td> </tr> <tr> <td>Availability of 3D structure of phytochemicals</td> <td>Yes</td> <td>Yes</td> <td>No</td> </tr> <tr> <td>Downloadable structure file formats</td> <td>MOL, MOL2, SDF, <br /><br />PDB &amp;amp; PDBQT </td> <td>MOL2</td> <td>No</td> </tr> <tr> <td>Chemical classification</td> <td>Yes</td> <td>Yes</td> <td>No</td> </tr> <tr> <td>Physicochemical properties</td> <td>Yes</td> <td>Yes</td> <td>No</td> </tr> <tr> <td>ADMET properties</td> <td>Yes</td> <td>Yes</td> <td>No</td> </tr> <tr> <td>Druggability properties</td> <td>Yes</td> <td>No</td> <td>No</td> </tr> <tr> <td>Cytoscape network visualization of associations</td> <td>Yes</td> <td>No</td> <td>No</td> </tr> <tr> <td>Filter phytochemicals based on physicochemical properties</td> <td>Yes</td> <td>Yes</td> <td>No</td> </tr> <tr> <td>Filter phytochemicals based on druggability properties</td> <td>Yes</td> <td>No</td> <td>No</td> </tr> <tr> <td>Chemical similarity search within database</td> <td>Yes</td> <td>No</td> <td>No</td> </tr> </tbody> </table>  <p /> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Methods</div> <div class="curatedlistofindianmedicinalplants" title="sec"> <div class="title" tagx="title" title="title">
Curated list of Indian medicinal plants</div> <p>In the preliminary phase of the database construction (Fig. <a href="#Fig1">1</a>), we compiled a comprehensive list of more than 5000 Indian medicinal plants based on information contained in the Indian medicinal plants database ( <a href="http://www.medicinalplants.in/">http://www.medicinalplants.in/</a>) of the Foundation for Revitalisation of Local Health Traditions (FRLHT), Bengaluru. In addition to the comprehensive list from FRLHT, the AYUSH priority list was compiled from two sources, namely, the list prepared by the National Mission on Medicinal Plants of Ministry of AYUSH, Government of India which is available at <a href="http://ayush.gov.in/sites/default/files/IV.pdf">http://ayush.gov.in/sites/default/files/IV.pdf</a>, and the list jointly prepared by the National Medicinal Plants Board (NMPB), Directorate of Medicinal and Aromatic Plants Research (DMAPR), Department of Agriculture and Cooperation, and Central Institute of Medicinal and Aromatic Plants (CIMAP), Government of India which is available at <a href="http://pib.nic.in/newsite/PrintRelease.aspx?relid=67277">http://pib.nic.in/newsite/PrintRelease.aspx?relid = 67277</a>. We remark that the AYUSH priority list was prepared based on several criteria including the medicinal use, conservation status and herbal industry demand of Indian medicinal plants. Due to the usage of multiple synonyms for Indian medicinal plants across different sources, the common names of plants were manually mapped to their scientific species names using The Plant List database <sup> <a href="#CR31">31</a> </sup> ( <a href="http://www.theplantlist.org/">http://www.theplantlist.org/</a>), and the compiled list was manually curated to remove redundancies. Furthermore, the Indian medicinal plants in our database were manually classified into their respective taxonomic families within the kingdom plantae using The Plant List database <sup> <a href="#CR31">31</a> </sup> and Tropicos database ( <a href="http://www.tropicos.org/">http://www.tropicos.org/</a>). We have also linked the Indian medicinal plants in IMPPAT database to their corresponding page in The Plant List database, Tropicos database and the FRLHT digital herbarium ( <a href="http://envis.frlht.org">http://envis.frlht.org</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <p>Schematic overview of the IMPPAT database construction pipeline. Briefly, we first compiled a comprehensive list of Indian medicinal plants from various sources. We next mined specialized books on Indian traditional medicine, existing databases and PubMed abstracts of journal articles to gather information on phytochemicals of Indian medicinal plants. We then manually annotated, curated and indexed names of identified phytochemicals with standard identifiers to build a non-redundant library of phytochemicals. This manual curation effort led to a unique list of plant-phytochemical associations. We also classified the Indian medicinal plants into taxonomic families and phytochemicals into chemical classes. Subsequently, we gathered ethnopharmacological information from books on traditional Indian medicine to build a unique list of plant-therapeutic use associations. We also extracted publicly accessible information on traditional medicine formulations from TKDL database to build a list of plant-formulation associations. Lastly, we have used cheminformatic tools to obtain the 3D structures, physicochemical properties, druggability scores, predicted ADMET properties and predicted target human proteins of phytochemicals. </p>   </div> <p /> </div> <div class="phytochemicalcompositionofindianmedicinalplants" title="sec"> <div class="title" tagx="title" title="title">
Phytochemical composition of Indian medicinal plants</div> <p>After compiling a comprehensive list of more than 5000 Indian medicinal plants, we mined literature to gather information on their phytochemicals (Fig. <a href="#Fig1">1</a>). In the first stage of data mining, we focussed on specialized traditional Indian medicine books <sup> <a href="#CR32">32</a>– <a href="#CR41">41</a> </sup>. From these books <sup> <a href="#CR32">32</a>– <a href="#CR41">41</a> </sup>, we gathered phytochemical composition for more than 1600 Indian medicinal plants. In the second stage, we gathered information from published databases of Indian medicinal plants. Phytochemica <sup> <a href="#CR11">11</a> </sup> database contains information on 963 phytochemicals of 5 Indian medicinal plants. Another database described in Polur <i>et al</i>. <sup> <a href="#CR18">18</a> </sup> had compiled information on 1829 phytochemicals of 295 ayurvedic Indian medicinal plants <sup> <a href="#CR18">18</a> </sup>. While this list is no longer publicly available, the Nutrichem <sup> <a href="#CR9">9</a>, <a href="#CR10">10</a> </sup> database on phytochemical composition and therapeutic uses of plant-based food products has incorporated the information compiled by Polur <i>et al</i>. <sup> <a href="#CR18">18</a> </sup>. From the Phytochemica <sup> <a href="#CR11">11</a> </sup> and Nutrichem <sup> <a href="#CR9">9</a>, <a href="#CR10">10</a> </sup> databases, we gathered information on the phytochemical composition of more than 400 Indian medicinal plants. Note that our comprehensive list covers a wide spectrum of Indian medicinal plants which includes apart from Ayurveda, other systems of traditional Indian medicine such as Siddha and Unani. In the third stage of data mining for phytochemical composition, we performed text mining of abstracts from published research articles in PubMed <sup> <a href="#CR42">42</a> </sup> using natural language processing (NLP) <sup> <a href="#CR43">43</a> </sup>. Using in-house Python scripts and a dataset of known plant-phytochemical associations, we identified keywords in PubMed abstracts which imply plant-phytochemical associations (Supplementary Table <a href="#MOESM1">S1</a>). We then used the selected keywords listed in Supplementary Table <a href="#MOESM1">S1</a> to mine PubMed abstracts to identify and incorporate additional references for plant-phytochemical associations in our database. In total, our database captures the phytochemical composition of 1742 Indian medicinal plants (Supplementary Table <a href="#MOESM1">S2</a>). The literature references for plant-phytochemical associations are listed in our database in the form of ISBN or DOI identifiers for books and PubMed identifiers (PMIDs) for journal articles. </p> <p>We would like to mention a potential bias in the list of plant-phytochemical associations compiled from scientific literature. Our database most-likely contains high-quality yet incomplete information on phytochemical composition of Indian medicinal plants. That is, phytochemicals listed are most-likely produced by the corresponding Indian medicinal plant but other phytochemicals not listed in our database cannot be ruled out from being also produced by the same plant due to possible lack of scientific literature. Moreover, the scientific literature will most probably have more information on phytochemical composition of well-studied or sequenced Indian medicinal plants such as <i>Catharanthus roseus</i>. Thus, future updates of this database will be needed to capture additional information on phytochemical composition of Indian medicinal plants. Nevertheless, one can argue that for the discovery of novel molecules it is more important to know the list of phytochemicals produced by an herb rather than the list of phytochemicals not produced by an herb. </p> </div> <div class="annotation,curationandfilteringofidentifiedphytochemicals" title="sec"> <div class="title" tagx="title" title="title">
Annotation, curation and filtering of identified phytochemicals</div> <p>An overarching goal of this work is to create a platform for exploring the chemistry of the phytochemicals of Indian medicinal plants. Evaluation of the phytochemicals of Indian medicinal plants for their druggability or drug-likeliness will facilitate the identification of molecules for drug discovery. We would like to emphasize that synonymous chemical names are pervasive across the literature on traditional Indian medicine which were mined to construct this database. In order to remove redundancy, we manually annotated the common names of phytochemicals of Indian medicinal plants compiled from literature sources with documented synonyms and standard chemical identifiers (Fig. <a href="#Fig1">1</a>) from Pubchem <sup> <a href="#CR44">44</a> </sup>, CHEBI <sup> <a href="#CR45">45</a> </sup>, CAS ( <a href="https://www.cas.org/">https://www.cas.org/</a>), CHEMSPIDER <sup> <a href="#CR46">46</a> </sup>, KNAPSACK <sup> <a href="#CR47">47</a> </sup>, CHEMFACES ( <a href="http://www.chemfaces.com">http://www.chemfaces.com</a>), FOODB ( <a href="http://foodb.ca/">http://foodb.ca/</a>), NIST Chemistry webbook <sup> <a href="#CR48">48</a> </sup> and Human Metabolome database (HMDB) <sup> <a href="#CR49">49</a> </sup>. While assigning standard identifiers to phytochemicals in our database, we have chosen the following priority order: Pubchem <sup> <a href="#CR44">44</a> </sup>, CHEBI <sup> <a href="#CR45">45</a> </sup>, CAS, CHEMSPIDER <sup> <a href="#CR46">46</a> </sup>, KNAPSACK <sup> <a href="#CR47">47</a> </sup>, CHEMFACES, FOODB, NIST Chemistry webbook <sup> <a href="#CR48">48</a> </sup> and HMDB <sup> <a href="#CR49">49</a> </sup>. We highlight that this extensive manual curation effort led to the mapping of more than 15000 common names of phytochemicals used across literature sources to a unique set of 9596 standard chemical identifiers. Phytochemicals which could not be mapped to standard chemical identifiers were excluded from our finalized database. Our choice to include only phytochemicals with standard identifiers and structure information was dictated by our goal to investigate the chemistry and druggability of phytochemicals of Indian medicinal plants. We remark that the 2D structure information for the unique set of 9596 IMPPAT phytochemicals was obtained using the standard chemical identifiers from the respective databases. We have also determined the chemical classification of the IMPPAT phytochemicals using ClassyFire <sup> <a href="#CR50">50</a> </sup> ( <a href="http://classyfire.wishartlab.com/">http://classyfire.wishartlab.com/</a>). ClassyFire <sup> <a href="#CR50">50</a> </sup> gives a hierarchical classification for each chemical compound into kingdom (organic or inorganic), followed by super-class, followed by class, followed by sub-class. Note that ClassyFire classifies organic compounds into 26 super-classes. In a nutshell, this largely manual effort to compile a non-redundant chemical library of 9596 phytochemicals of Indian medicinal plants with standard identifiers and structure information will serve as valuable resource for natural product-based drug discovery in future. Moreover, the use of standard chemical identifiers will enable effortless integration of our IMPPAT database with other data sources. </p> </div> <div class="therapeuticusesofindianmedicinalplants" title="sec"> <div class="title" tagx="title" title="title">
Therapeutic uses of Indian medicinal plants</div> <p>Another goal of our database is to compile ethnopharmacological information on Indian medicinal plants. Towards this goal, we manually compiled the medicinal (therapeutic) uses of Indian medicinal plants from books on Indian traditional medicine <sup> <a href="#CR32">32</a>– <a href="#CR41">41</a>, <a href="#CR48">48</a>, <a href="#CR51">51</a>– <a href="#CR67">67</a> </sup>. Apart from books, Polur <i>et al</i>. <sup> <a href="#CR18">18</a> </sup> had previously compiled a list of therapeutic uses for 295 ayurvedic Indian medicinal plants, and this information was extracted from the Nutrichem <sup> <a href="#CR9">9</a>, <a href="#CR10">10</a> </sup> database. To ensure high quality, we manually curated information on therapeutic uses of Indian medicinal plants and consciously avoided automated text mining to retrieve additional information on plant-therapeutic associations. We remark that our database has manually compiled therapeutic uses of Indian medicinal plants from standard books on traditional Indian medicine which contain accumulated experience-based knowledge on treating human diseases. Furthermore, we manually annotated and standardized the compiled therapeutic uses of Indian medicinal plants from the above sources with identifiers from the Disease Ontology <sup> <a href="#CR68">68</a> </sup>, Online Mendelian Inheritance in Man (OMIM) <sup> <a href="#CR69">69</a> </sup>, Unified Medical Language System (UMLS) <sup> <a href="#CR70">70</a> </sup> and Medical Subject Headings (MeSH) <sup> <a href="#CR71">71</a> </sup> databases. To the best of our knowledge, this is the first large-scale attempt to link the ethnopharmacological information on Indian medicinal plants with standardized vocabulary in modern medicine. Note that databases of gene-disease associations <sup> <a href="#CR72">72</a> </sup> and disease-symptom associations <sup> <a href="#CR73">73</a> </sup> usually provide disease information in form of identifiers from OMIM, UMLS and MeSH databases, and in future, information from such databases can be effortlessly integrated into IMPPAT database. </p> </div> <div class="traditionalformulationsofindianmedicinalplants" title="sec"> <div class="title" tagx="title" title="title">
Traditional formulations of Indian medicinal plants</div> <p>Traditional knowledge digital library (TKDL) ( <a href="http://www.tkdl.res.in">http://www.tkdl.res.in</a>) is a knowledgebase of traditional Indian medicinal formulations. A traditional medicinal formulation is often a multi-component mixture derived from plant, animal and other sources which is used for treating disease based on specific indication. For example, Thinavu Sori Soolaiku Ennai (TKDL Identifier: HM02/36) is a medicinal formulation in traditional Indian system of medicine, Siddha, which is used to treat allergic rashes, and this formulation mainly consists of extracts of medicinal plants, <i>Plumbago zeylanica</i>, <i>Sesamum orientale</i> (also called <i>Sesamum indicum</i>) and <i>Cuminum cyminum</i>. According to TKDL, there are more than 250000 formulations of Ayurveda, Siddha and Unani of which 1200 representative formulations are openly accessible via their database. To exhibit the broader utility of our database to phytopharmacology, we have also compiled and curated the subset of 1200 openly accessible formulations in TKDL which contain at least one of the 1742 Indian medicinal plants in our database. This process led to associations between 321 Indian medicinal plants in our database and 974 traditional Indian medicinal formulations which are openly accessible through TKDL database (Fig. <a href="#Fig1">1</a>). We emphasize that our database has only incorporated open digital information on traditional Indian medicinal formulations from TKDL database. However, we are aware of the vast literature <sup> <a href="#CR7">7</a>, <a href="#CR8">8</a>, <a href="#CR74">74</a> </sup> on traditional Indian medicinal formulations, especially in books, and in the future, a significant effort will be needed to digitize and integrate such information into our database. </p> </div> <div class="3dstructureofphytochemicals" title="sec"> <div class="title" tagx="title" title="title">
3D structure of phytochemicals</div> <p>We have generated lowest energy 3D conformational structure of IMPPAT phytochemicals using Balloon <sup> <a href="#CR75">75</a> </sup> ( <a href="http://users.abo.fi/mivainio/balloon/">http://users.abo.fi/mivainio/balloon/</a>) and Open Babel <sup> <a href="#CR76">76</a> </sup> ( <a href="http://openbabel.org/wiki/Main_Page">http://openbabel.org/wiki/Main_Page</a>). Balloon generates 3D conformers of input 2D structures from scratch and optimizes them using Merck Molecular Force Field (MMFF94). The lowest energy 3D conformer was selected from 20 generated conformations for a given phytochemical. Of the 9596 IMPPAT phytochemicals, Balloon successfully generated 3D structures for 8021 phytochemicals. For the remaining 1575 phytochemicals, the lowest energy 3D conformer was generated using Open Babel with MMFF94 force field. We remark that our preferred choice of Balloon to generate lowest energy 3D structures of IMPPAT phytochemicals was motivated by similar choice made by two other databases of 3D structures of natural products, namely, KNApSAcK-3D <sup> <a href="#CR47">47</a> </sup> and TIPdb-3D <sup> <a href="#CR77">77</a> </sup>. </p> </div> <div class="physicochemicalpropertiesofphytochemicals" title="sec"> <div class="title" tagx="title" title="title">
Physicochemical properties of phytochemicals</div> <p>We used FAF-Drugs4 webserver <sup> <a href="#CR20">20</a> </sup> and RDKit <sup> <a href="#CR21">21</a> </sup> to compute the following physicochemical properties of the IMPPAT phytochemicals: molecular weight, octanol-water partition coefficient (logP), logP at physiological pH of 7.4 (logD), logarithm of water solubility (logSw), number of stereocenters, stereochemical complexity <sup> <a href="#CR78">78</a> </sup> which is the fraction of carbon atoms which are stereogenic, Fsp <sup>3</sup> which is the fraction of carbon atoms that are sp <sup>3</sup> hybridized <sup> <a href="#CR79">79</a> </sup>, topological polar surface area (TPSA), charge of the compound, number of hydrogen bond donors and acceptors, number of smallest set of smallest rings (SSSR) which is the number of smallest ring building blocks required for forming other ring systems, size of the biggest system ring which is the number of atoms present in the biggest ring system, number of rotatable and rigid bonds, number of charged groups, total charge of the compound, number of carbon, hetero- and heavy atoms, and ratio between the number of non-carbon atoms and the number of carbon atoms. </p> </div> <div class="admetpropertiesofphytochemicals" title="sec"> <div class="title" tagx="title" title="title">
ADMET properties of phytochemicals</div> <p>Absorption, distribution, metabolism, excretion and toxicity (ADMET) properties have been implicated as one of the reasons for high attrition rate of candidates from drug development pipeline. Thus, we used admetSAR <sup> <a href="#CR22">22</a> </sup> webserver to predict the ADMET properties of the phytochemicals. The predicted properties which influence absorption include Human Intestinal Absorption (HIA) <sup> <a href="#CR80">80</a> </sup>, Blood Brain Barrier (BBB) permeability <sup> <a href="#CR80">80</a> </sup>, Caco-2 permeability <sup> <a href="#CR81">81</a> </sup> and likeliness of being P-glycoprotein substrate <sup> <a href="#CR82">82</a> </sup>. The predicted properties which affect phytochemical metabolism include the ability to inhibit several CYP450 enzymes or likeliness of being a substrate to CYP450 enzymes <sup> <a href="#CR83">83</a>– <a href="#CR85">85</a> </sup>. Lastly, toxicity predictions are based on computational models for Ames test for mutagenicity <sup> <a href="#CR86">86</a> </sup>, carcinogenicity, biodegradability <sup> <a href="#CR87">87</a> </sup>, rat acute toxicity <sup> <a href="#CR88">88</a> </sup> and hERG inhibition <sup> <a href="#CR89">89</a>, <a href="#CR90">90</a> </sup>. Note that our choice of admetSAR <sup> <a href="#CR22">22</a> </sup> was motivated by the same choice made by DrugBank <sup> <a href="#CR19">19</a> </sup> database ( <a href="https://www.drugbank.ca/">https://www.drugbank.ca/</a>) which is the widely-used repository of approved and experimental drugs. </p> </div> <div class="druggabilityscoresofphytochemicals" title="sec"> <div class="title" tagx="title" title="title">
Druggability scores of phytochemicals</div> <p>We used FAF-Drugs4 webserver <sup> <a href="#CR20">20</a> </sup> to test the druggability of the phytochemicals based on multiple scoring schemes, namely, Lipinski’s rule of five (RO5) <sup> <a href="#CR23">23</a> </sup>, Oral PhysChem <sup> <a href="#CR24">24</a> </sup> score (Traffic Lights), GlaxoSmithKline’s (GSK’s) 4/400 <sup> <a href="#CR25">25</a> </sup>, Pfizer’s 3/75 <sup> <a href="#CR26">26</a> </sup>, Veber rule <sup> <a href="#CR27">27</a> </sup> and Egan rule <sup> <a href="#CR28">28</a> </sup>. Lipinski’s RO5 <sup> <a href="#CR23">23</a> </sup> is a classical rule of thumb to filter druggable small molecules based on four physiochemical properties. RO5 considers a small molecule to be druggable if it has ≤5 hydrogen bond donors, ≤10 hydrogen bond acceptors, molecular weight &amp;lt;500 Daltons and logP ≤5. If a small molecule violates none of the above rules it is assigned a RO5 value of 0, and on the other extreme if it violates all the above rules it is assigned a RO5 value of 4. OralPhysChem <sup> <a href="#CR24">24</a> </sup> score is another method for filtering druggable small molecules which is based on five physiochemical properties, namely, aqueous solubility, logP, corrected molecular weight for presence of halogen atoms, TPSA and number of rotatable bonds. OralPhysChem score ranges from 0 to 10 whereby 0 signifies high druggability while 10 signifies low druggability of the small molecule. GSK’s 4/400 <sup> <a href="#CR25">25</a> </sup> is another filter based on a number of ADMET assays carried out in GSK. Briefly, a small molecule is considered more druggable and labelled ‘Good’ by GSK’s 4/400 score if it has both molecular weight &amp;lt;400 Daltons and logP &amp;lt;4 while it is considered less druggable and labelled ‘Bad’ if at least one of the rules is not satisfied. Pfizer’s 3/75 <sup> <a href="#CR26">26</a> </sup> rule is used to filter small molecules which are more prone to be toxic, and hence, less likely to be druggable. Pfizer’s 3/75 rule considers small molecules with logP &amp;lt;3 and TPSA &amp;gt; 75 Å <sup>2</sup> to be ‘Good’ as they are likely to be less toxic, and hence, more likely to be druggable, and small molecules which do not satisfy one of the two rules are labelled as ‘Warning’, and small molecules which violate both rules are labelled as ‘Bad’. Veber rule <sup> <a href="#CR27">27</a> </sup> considers small molecules to have good oral bioavailability if they satisfy number of rotatable bonds ≤10 and TPSA ≤140 Å <sup>2</sup>, and small molecules which fail these criteria are considered to have low bioavailability. Similarly, Egan rule <sup> <a href="#CR28">28</a> </sup> considers small molecules to have good oral bioavailability if they satisfy −1.0 ≤logP ≤5.8 and TPSA ≤130 Å <sup>2</sup>, and small molecules which fail these criteria are considered to have low bioavailability. We filtered phytochemicals with no RO5 violation, net Traffic Lights value of zero and satisfying GSK’s 4/400, Pfizer’s 3/75, Veber rule and Egan rule as <i>druggable</i>. We further computed the weighted quantitative estimate of drug-likeness (QEDw) <sup> <a href="#CR91">91</a> </sup> score using FAF-QED webserver <sup> <a href="#CR20">20</a> </sup> for the filtered list of druggable phytochemicals within IMPPAT and TCM-Mesh <sup> <a href="#CR16">16</a> </sup>. QEDw is a druggability score for small molecules proposed by Bickerton <i>et al</i>. <sup> <a href="#CR91">91</a> </sup> which is the weighted geometric mean of molecular weight, logP, number of hydrogen bond donors, number of hydrogen bond acceptors, TPSA, number of rotatable bonds, number of aromatic rings and number of structural alerts. Note that QEDw is a continuous score between 0 and 1 where 0 signifies low druggability and 1 signifies high druggability. </p> </div> <div class="predictedhumantargetproteinsofphytochemicals" title="sec"> <div class="title" tagx="title" title="title">
Predicted human target proteins of phytochemicals</div> <p>We have extracted the predicted human target proteins of IMPPAT phytochemicals from STITCH <sup> <a href="#CR29">29</a> </sup> database ( <a href="http://stitch1.embl.de/">http://stitch1.embl.de/</a>). Note that STITCH <sup> <a href="#CR29">29</a> </sup> database is the largest resource on predicted interactions between chemicals and their target proteins. From the STITCH <sup> <a href="#CR29">29</a> </sup> database, we have extracted and reported only high confidence interactions between phytochemicals and target human proteins that have a combined STITCH score ≥700. Note that our choice of STITCH database to predict interactions between IMPPAT phytochemicals and target human proteins was based on similar choice made by the traditional Chinese medicine database TCM-Mesh <sup> <a href="#CR16">16</a> </sup>. </p> </div> <div class="smallmoleculecollectionsofcommercialcompounds,diversecompounds,naturalproductsandphytochemicalsfromchinesemedicinalplants" title="sec"> <div class="title" tagx="title" title="title">
Small molecule collections of commercial compounds, diverse compounds, natural products and phytochemicals from Chinese medicinal plants </div> <p>We have compared the physicochemical properties of 9596 IMPPAT phytochemicals from Indian medicinal plants with other collections of small molecules. Clemons <i>et al</i>. <sup> <a href="#CR78">78</a> </sup> have compiled small molecule collections from three different sources, namely, commercial compounds (CC), diversity-oriented synthesis compounds (DC’) and natural products (NP). CC contains 6152 representative small molecules from commercial sources. DC’ contains 5963 small molecules synthesized by academic community using methods like diversity-oriented synthesis. NP contains 2477 small molecules from natural products. We remark that 11, 3 and 147 small molecules in CC, DC’ and NP collections, respectively, are also contained in the set of 9596 IMPPAT phytochemicals. Note that the computation of physicochemical properties failed for 3 small molecules in CC and 3 small molecules in DC’, and we omitted these small molecules from subsequent analysis. </p> <p>In addition, we have also extracted the set of 10140 phytochemicals produced by 6235 Chinese medicinal plants or herbs from TCM-Mesh <sup> <a href="#CR16">16</a> </sup> ( <a href="http://mesh.tcm.microbioinformatics.org/">http://mesh.tcm.microbioinformatics.org/</a>) database. TCM-Mesh has compiled information on phytochemicals of Chinese herbs from two other extensive databases on traditional Chinese medicine, TCMID <sup> <a href="#CR15">15</a> </sup> ( <a href="http://www.megabionet.org/tcmid/">http://www.megabionet.org/tcmid/</a>) and TCM@Taiwan <sup> <a href="#CR14">14</a> </sup> ( <a href="http://tcm.cmu.edu.tw/">http://tcm.cmu.edu.tw/</a>). Note that TCM-Mesh contains a large set of 383840 chemical compounds but only a subset of 10140 phytochemicals are ingredients of 6235 Chinese herbs in the database. </p> </div> <div class="similarityofphytochemicals" title="sec"> <div class="title" tagx="title" title="title">
Similarity of phytochemicals</div> <p>Tanimoto coefficient (Tc) <sup> <a href="#CR92">92</a> </sup> is a widely used measure to compute structural similarity between chemicals <sup> <a href="#CR93">93</a> </sup>. To evaluate the structural similarity of chemicals within our database to known drugs using Tc, we employed two molecular fingerprints: (a) Extended Circular Fingerprints (ECFP4) <sup> <a href="#CR94">94</a> </sup> applying Morgan algorithm <sup> <a href="#CR95">95</a> </sup> with radius value of 2 as implemented in RDKit <sup> <a href="#CR21">21</a> </sup>, and (b) MACCS keys based fingerprint. We employed the open source package, RDKit <sup> <a href="#CR21">21</a> </sup>, to compute molecular fingerprints and Tc between pairs of chemical structures. To identify structural similarity between chemicals, a stringent cut-off of Tc ≥0.5 was used while employing ECFP4 and a cut-off of Tc ≥0.85 was used while employing MACCS keys. Our selection of Tc cut-offs for ECFP4 and MACCS keys based computations was motivated by the recent work of Jasial <i>et al</i>. <sup> <a href="#CR96">96</a> </sup>. </p> <p>We obtained a list of 2069 FDA approved drugs from DrugBank <sup> <a href="#CR19">19</a> </sup> database and computed their structural similarity with druggable IMPPAT phytochemicals using both ECFP4 and MACCS keys based molecular fingerprints. Note that ECFP4 molecular fingerprints were used to create the chemical similarity network of the druggable phytochemicals with QEDw score ≥0.9. Besides quantifying the structural similarity based on the Tc of phytochemicals, we have employed principal component analysis (PCA) to explore possible relationships between druggable phytochemicals with QEDw score ≥0.9 based on their physicochemical properties. </p> </div> <div class="databasemanagementandnetworkvisualization" title="sec"> <div class="title" tagx="title" title="title">
Database management and network visualization</div> <p>To construct this database, the compiled and curated data was integrated using MySQL ( <a href="https://www.mysql.com/">https://www.mysql.com/</a>), a relational database management system which serves as a back-end for our resource. The web interface for the database was built using Drupal ( <a href="https://www.drupal.org/">https://www.drupal.org/</a>), a PHP-based content management system hosted on Apache server with the MySQL database in the back-end. Users can browse or query our database using the scientific names of Indian medicinal plants, standard identifiers for phytochemicals, or associated therapeutic uses (Fig. <a href="#Fig2">2</a>). Further we have integrated the Cytoscape.js application <sup> <a href="#CR97">97</a> </sup> ( <a href="http://js.cytoscape.org/">http://js.cytoscape.org/</a>) into our web interface which enables visualization of plant-phytochemical associations, plant-therapeutic associations, and plant-formulation associations in the form of a network. The Cytoscape network visualization displays different types of nodes such as plant, phytochemical, therapeutic use and traditional medicinal formulations in different shapes and colours. Finally, the association network can be downloaded as a tab-separated list using the available export option in our database (Fig. <a href="#Fig2">2</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <p>Web-interface of the IMPPAT database. ( <b>a</b>) Snapshot of the result of a standard query for phytochemicals of an Indian medicinal plant. In this example, we show the plant-phytochemical association for <i>Ocimum tenuiflorum</i>, commonly known as Tulsi, from IMPPAT database. ( <b>b</b>) Snapshot of the dedicated page containing detailed information on 2D and 3D chemical structure, physicochemical properties, druggability scores, predicted ADMET properties and predicted target human proteins for a chosen phytochemical. From the dedicated page for each phytochemical, users can download the 2D and 3D structure of the phytochemical in the form of a SDF or MOL or MOL2 or PDB or PDBQT file. ( <b>c</b>) Snapshot of the result of a standard query for therapeutic uses of an Indian medicinal plant. In this example, we show the therapeutic uses of <i>Ocimum tenuiflorum</i> from IMPPAT database. ( <b>d</b>) Snapshot of the advanced search options which enable users to filter phytochemicals based on their physiochemical properties or druggability scores or chemical similarity with a query compound. </p>   </div> <p /> </div> <div class="dataavailability" title="sec"> <div class="title" tagx="title" title="title">
Data availability</div> <p>The datasets generated and analysed in this study are openly accessible at: <a href="https://cb.imsc.res.in/imppat">https://cb.imsc.res.in/imppat</a>. </p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="web-interfaceofthedatabase" title="sec"> <div class="title" tagx="title" title="title">
Web-interface of the database</div> <p>The IMPPAT database captures information on three types of associations for Indian medicinal plants: phytochemical composition, therapeutic uses, and traditional medicinal formulations (Fig. <a href="#Fig1">1</a>). The web-interface of the database enables users to query for each of these associations using (a) scientific names of plants, (b) standard chemical identifiers of phytochemicals, (c) therapeutic uses, or (d) formulation identifiers (Fig. <a href="#Fig2">2</a>). The web-interface displays the result of user queries for these associations in two ways: (a) A table of associations with references to literature sources, and (b) A network visualization of the associations which is powered by Cytoscape.js <sup> <a href="#CR97">97</a> </sup> (Fig. <a href="#Fig2">2</a>). In addition, users can also download the result of their queries for different associations of medicinal plants as a tab-separated list using the available export option in the web-interface. In the results page of queries for plant-phytochemical associations, users can click each phytochemical name or identifier to navigate to a separate page containing detailed information such as chemical structure, alternate chemical names or identifiers, computed physicochemical properties, computed druggability scores, predicted ADMET properties, predicted human target proteins and the option to download the 2D or 3D chemical structure file in several formats (Fig. <a href="#Fig2">2</a>; Methods). Queries for plant-therapeutic associations leads to a page where users can also obtain the disease ontology identifiers corresponding to therapeutic uses (Fig. <a href="#Fig2">2</a>; Methods). In the results page of queries for plant-formulation associations, users can click the medicinal formulation identifiers to navigate to the corresponding page in the TKDL database. Moreover, in the advanced search page of IMPPAT database (Fig. <a href="#Fig2">2</a>), users can filter phytochemicals based on physicochemical properties (e.g., molecular weight, number of hydrogen bond acceptors), or filter phytochemicals satisfying various druggability scores (e.g. RO5, Traffic Lights), or search for phytochemicals similar to query chemical compound. To run the similarity filter, users will have to provide the query compound in the form of Canonical SMILES and choose a molecular fingerprint (ECFP4 or MACCS keys) to compute Tc between the query compound and IMPPAT phytochemicals. The chemical similarity filter will list top 10 IMPPAT phytochemicals which are similar to the input query compound based on Tc. </p> </div> <div class="networkofplant-phytochemicalassociations,plant-therapeuticuseassociations,andplant-traditionalmedicinalformulationassociations" title="sec"> <div class="title" tagx="title" title="title">
Network of plant-phytochemical associations, plant-therapeutic use associations, and plant-traditional medicinal formulation associations </div> <p>IMPPAT database contains information on the phytochemical composition and therapeutic uses of 1742 Indian medicinal plants (Supplementary Table <a href="#MOESM1">S2</a>). The 1742 Indian medicinal plants in our database are distributed across 215 different taxonomic families (Fig. <a href="#Fig3">3a</a>; Methods). Among the 215 taxonomic families, Leguminosae contains the maximum number (131) of Indian medicinal plants in our database (Fig. <a href="#Fig3">3a</a>). Of the 134 Indian medicinal plants in the priority list of Ministry of AYUSH, Government of India, 116 Indian medicinal plants are contained in our database (Supplementary Table <a href="#MOESM1">S2</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <p>Basic statistics for Indian medicinal plants and associated phytochemicals in IMPPAT database. ( <b>a</b>) Pie chart shows the distribution of the 1742 Indian medicinal plants in IMPPAT database across different taxonomic families. ( <b>b</b>) Pie chart shows the distribution of the 9596 IMPPAT phytochemicals across different chemical super-classes obtained from ClassyFire <sup> <a href="#CR50">50</a> </sup>. ( <b>c</b>) Histogram of the number of Indian medicinal plants which produce a given phytochemical in our database. ( <b>d</b>) Histogram of the number of therapeutic uses per Indian medicinal plant in our database. ( <b>e</b>– <b>j</b>) Histogram of the molecular weight (in g/mol), logP, TPSA (in Å <sup>2</sup>), number of hydrogen bond donors, number of hydrogen bond acceptors and number of rotatable bonds of the phytochemicals in our database. </p>   </div> <p /> <p>IMPPAT captures information on 27074 plant-phytochemical associations which encompasses 1742 Indian medicinal plants and their 9596 phytochemicals. We used ClassyFire <sup> <a href="#CR50">50</a> </sup> webserver for chemical classification of the 9596 IMPPAT phytochemicals (Methods). The 9596 IMPPAT phytochemicals are distributed across 24 super-classes, 260 classes and 415 sub-classes of ClassyFire <sup> <a href="#CR50">50</a> </sup>. Among the 24 super-classes, lipids and lipid-like molecules, phenylpropanoids and polyketides, and organoheterocyclic compounds are the top three super-classes with 3190, 1793 and 1184 phytochemicals, respectively (Fig. <a href="#Fig3">3b</a>). Among 260 chemical classes, prenol lipids, organooxygen compounds, and flavonoids are the top three classes with 2005, 868 and 818 phytochemicals, respectively. Among the 1742 Indian medicinal plants in our database, <i>Catharanthus roseus</i> has the highest number of phytochemical associations. In Fig. <a href="#Fig3">3c</a>, we show a histogram of the occurrence of phytochemicals across 1742 Indian medicinal plants in our database. From this figure, it is seen that the majority of (8838) phytochemicals are found in less than 5 Indian medicinal plants while only a handful of (3) phytochemicals are found in more than 200 Indian medicinal plants. IMPPAT also captures 48632 predicted interactions between phytochemicals and their human target proteins from STITCH <sup> <a href="#CR29">29</a> </sup> database which encompasses 1477 IMPPAT phytochemicals and 8128 human proteins (Methods). </p> <p>IMPPAT also captures information on 11514 plant-therapeutic use associations which encompasses 1742 Indian medicinal plants and 1124 therapeutic uses. In Fig. <a href="#Fig3">3d</a>, we show a histogram of the number of therapeutic uses per Indian medicinal plant in our database. From this figure, it is seen that a majority of 1409 Indian medicinal plants have less than 10 documented therapeutic uses while a small fraction of 90 Indian medicinal plants have more than 20 therapeutic uses in our database. Among the 1742 Indian medicinal plants in our database, <i>Ginkgo biloba</i> (136), <i>Panax ginseng</i> (135) and <i>Allium sativum</i> (86) have the largest number of documented therapeutic uses. Lastly, IMPPAT also captures information on 5069 plant-formulation associations which encompasses 321 Indian medicinal plants in our database and 974 traditional Indian medicinal formulations which are openly accessible from the TKDL database (Methods). </p> </div> <div class="comparativeanalysisofthephysicochemicalpropertiesofimppatphytochemicalswithothersmallmoleculecollections" title="sec"> <div class="title" tagx="title" title="title">
Comparative analysis of the physicochemical properties of IMPPAT phytochemicals with other small molecule collections </div> <p>We have computed several physicochemical properties for the 9596 IMPPAT phytochemicals (Methods). Figure <a href="#Fig3">3e–j</a> shows the distribution and the table in Fig. <a href="#Fig4">4c</a> gives the mean and median of the distribution for six physicochemical properties, namely, molecular weight, logP, TPSA, number of hydrogen bond donors, number of hydrogen bond acceptors and number of rotatable bonds for the 9596 IMPPAT phytochemicals. Moreover, we have predicted several ADMET properties for the 9596 IMPPAT phytochemicals (Methods). For example, HIA model predicts 89% of IMPPAT phytochemicals have good intestinal absorption and carcinogenicity model predicts 94% of IMPPAT phytochemicals are non-carcinogenic. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <p>Comparison of the physicochemical properties of IMPPAT phytochemicals with other small molecule collections. ( <b>a</b>) Box plot shows the distribution of the stereochemical complexity of the small molecule collections CC, DC’, NP, IMPPAT phytochemicals and TCM-Mesh phytochemicals. The median, mean and standard deviation (SD) of the stereochemical complexity for each small molecule collection is shown below the box plot. ( <b>b</b>) Box plot shows the distribution of the Fsp <sup>3</sup> for the small molecule collections CC, DC’, NP, IMPPAT phytochemicals and TCM-Mesh phytochemicals. The median, mean and SD of the Fsp <sup>3</sup> for each small molecule collection is shown below the box plot. Note the lower end of the box shows the first quartile, upper end of the box shows the third quartile, brown line shows the median and green line shows the mean of the distribution of stereochemical complexity or Fsp <sup>3</sup> in the two box plots. ( <b>c</b>) Median, mean and SD of six physicochemical properties, namely, molecular weight, logP, TPSA, number of hydrogen bond donors, number of hydrogen bond acceptors and number of rotatable bonds for the small molecule collections CC, DC’, NP, IMPPAT phytochemicals and TCM-Mesh phytochemicals. </p>   </div> <p /> <p>Small molecules which are specific protein binders in screening assays are more favourable candidates for drug discovery pipeline than promiscuous binders which might interact with many proteins in a screening assay. Clemons <i>et al</i>. <sup> <a href="#CR78">78</a> </sup> have correlated two simple size-independent metrics, namely, stereochemical complexity and shape complexity (Fsp <sup>3</sup>) <sup> <a href="#CR79">79</a> </sup> with the binding specificity of representative compound collections, CC, DC’ and NP (Methods). Clemons <i>et al</i>. <sup> <a href="#CR78">78</a> </sup> found that DC’ and NP collections have more stereochemical complexity and shape complexity in comparison to CC collection, and interestingly, small molecules in DC’ and NP collections were shown to be more specific binders with less fraction of promiscuous binders in comparison to small molecules in CC collection. We have compared the distribution of stereochemical complexity and Fsp <sup>3</sup> across 9596 IMPPAT phytochemicals with CC, DC’ and NP collections from Clemons <i>et al</i>. <sup> <a href="#CR78">78</a> </sup> and 10140 TCM-Mesh <sup> <a href="#CR16">16</a> </sup> phytochemicals from Chinese medicinal plants (Fig. <a href="#Fig4">4a-b</a>; Methods). Interestingly, we find the mean and median of stereochemical complexity of IMPPAT phytochemicals is higher than CC and DC’ collections while being closer to NP collection (Fig. <a href="#Fig4">4a</a>). Furthermore, the mean and median of stereochemical complexity of IMPPAT phytochemicals was found to be much closer to TCM-Mesh phytochemicals in comparison to DC’, CC or even NP collection (Fig. <a href="#Fig4">4a</a>). We also obtain similar trends for the mean and median of shape complexity (Fsp <sup>3</sup>) of IMPPAT phytochemicals (Fig. <a href="#Fig4">4b</a>). These observations underscore that the IMPPAT phytochemicals are closer to small molecule libraries of natural products or phytochemicals from Chinese medicinal plants in terms of stereochemical complexity and Fsp <sup>3</sup>, and thus, are more likely enriched with specific binders than promiscuous binders. </p> <p>In a later study, Clemons <i>et al</i>. <sup> <a href="#CR98">98</a> </sup> have also shown that the small molecules in CC, DC’ and NP occupy different regions in the physicochemical space. By considering six physicochemical properties studied by Clemons <i>et al</i>. <sup> <a href="#CR98">98</a> </sup>, namely, molecular weight, logP, TPSA, number of hydrogen bond donors, number of hydrogen bond acceptors and number of rotatable bonds, we have compared the physicochemical properties of CC, DC’, NP and TCM-Mesh phytochemicals with IMPPAT phytochemicals (Fig. <a href="#Fig4">4c</a>). In terms of the six physicochemical properties, the IMPPAT phytochemicals are found to be more similar to TCM-Mesh phytochemicals in comparison to NP, DC’ or CC (Fig. <a href="#Fig4">4c</a>). The above results underscore the importance of our curated collection of 9596 IMPPAT phytochemicals from Indian medicinal plants which will be a valuable addition to natural product-based screening collections. </p> </div> <div class="druggabilityanalysisofphytochemicalsofindianmedicinalplants" title="sec"> <div class="title" tagx="title" title="title">
Druggability analysis of phytochemicals of Indian medicinal plants</div> <p>We evaluated the druggability of 9596 IMPPAT phytochemicals based on multiple rules or scoring schemes, namely, RO5 <sup> <a href="#CR23">23</a> </sup>, Traffic Lights <sup> <a href="#CR24">24</a> </sup>, GSK’s 4/400 <sup> <a href="#CR25">25</a> </sup>, Pfizer’s 3/75 <sup> <a href="#CR26">26</a> </sup>, Veber rule <sup> <a href="#CR27">27</a> </sup> and Egan rule <sup> <a href="#CR28">28</a> </sup> which were computed using FAF-Drugs4 webserver <sup> <a href="#CR20">20</a> </sup> (Methods). The horizontal bar plot in Fig. <a href="#Fig5">5a</a> gives the number of IMPPAT phytochemicals that satisfy different druggability scores. From this figure, it is seen that the majority of IMPPAT phytochemicals satisfy Veber rule or Egan rule in comparison to Pfizer’s 3/75 rule or net Traffic Lights value of zero. Furthermore, we find that the same set of 8712 IMPPAT phytochemicals satisfy both the Veber rule and Egan rule for drug-likeliness. The vertical bar plot of Fig. <a href="#Fig5">5a</a> shows the overlap between sets of phytochemicals that satisfy different druggability scores. We found that 960 out of 9596 IMPPAT phytochemicals satisfy all evaluated druggability scores (Fig. <a href="#Fig5">5a</a>). Subsequently, we designated this filtered list of 960 IMPPAT phytochemicals as <i>druggable</i>. Among the 1742 Indian medicinal plants in our database, <i>Brassica oleracea, Catharanthus roseus, Zea mays, Oryza sativa, Vigna radiate, Pisum sativum, Anethum sowa, Allium cepa, Cassia obtusifolia </i> and <i>Camellia sinensis</i> produce the highest number of druggable phytochemicals, and Supplementary Table <a href="#MOESM1">S3</a> gives the number of druggable phytochemicals for each plant in IMPPAT database. In Fig. <a href="#Fig5">5b</a>, we show the distribution of the 960 druggable IMPPAT phytochemicals across different chemical super-classes obtained using ClassyFire <sup> <a href="#CR50">50</a> </sup>. Among the chemical super-classes, phenylpropanoids and polyketides, organoheterocyclic compounds, and lipids and lipid-like molecules are the top three with 218, 182 and 137 phytochemicals, respectively (Fig. <a href="#Fig5">5b</a>; Methods). Among the chemical classes, organooxygen compounds, prenol lipids, and flavonoids are the top three with 111, 97 and 96 phytochemicals, respectively. Moreover, organooxygen class includes many carbohydrates, carbonyl compounds and alcohols while the prenol lipids class mainly comprises different types of terpenes and terpenoids. Figure <a href="#Fig5">5c</a> shows the distribution of QEDw <sup> <a href="#CR91">91</a> </sup> scores for the 960 druggable IMPPAT phytochemicals with mean score of 0.57 and standard deviation of 0.17 (Methods). From this figure, it is seen that 14 druggable phytochemicals have a QEDw score ≥0.9 and 98 druggable phytochemicals have a QEDw score ≥0.8. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 5</span> </a> </div>   <p>Druggability analysis of phytochemicals in IMPPAT database. ( <b>a</b>) Evaluation of drug-likeliness of phytochemicals based on multiple scores. The horizontal bar plot shows the number of phytochemicals in the IMPPAT database that satisfy different druggability scores (Methods). The vertical bar plot shows the overlap between sets of phytochemicals that satisfy different druggability scores. The pink bar in the vertical plot gives the 960 phytochemicals which satisfy all druggability scores. This plot was generated using UpSetR <sup> <a href="#CR105">105</a> </sup> package. ( <b>b</b>) Classification of the 960 druggable phytochemicals into chemical super-classes obtained from ClassyFire <sup> <a href="#CR50">50</a> </sup>. ( <b>c</b>) Distribution of QEDw <sup> <a href="#CR91">91</a> </sup> scores for the 960 IMPPAT phytochemicals which satisfy all druggability scores. ( <b>d</b>) Venn diagrams summarizing structural similarity analysis of 960 druggable phytochemicals in IMPPAT database and FDA approved drugs. Based on ECFP4 and MACCS keys molecular fingerprints, 249 and 302 druggable phytochemicals, respectively, were found to be similar to FDA approved drugs. </p>   </div> <p /> <p>By comparing the 2069 FDA approved drugs from the DrugBank <sup> <a href="#CR19">19</a> </sup> with the 960 druggable IMPPAT phytochemicals, we found that only 32 FDA approved drugs are among the 960 phytochemicals while the remaining 928 phytochemicals are potential new hits. By investigating the structural similarity between 2069 FDA approved drugs and 960 druggable IMPPAT phytochemicals, we found that 249 and 302 druggable phytochemicals are similar to FDA approved drugs based on ECFP4 or MACCS keys molecular fingerprints, respectively (Fig. <a href="#Fig5">5d</a>; Methods). Combined, ECFP4 and MACCS keys based fingerprints identified 369 out of 960 druggable IMPPAT phytochemicals that are similar to FDA approved drugs (Methods). Thus, almost 40% of the druggable IMPPAT phytochemicals are similar to at least one FDA approved drug which testifies to our systemic approach to identify potential druggable phytochemicals of Indian medicinal plants. Importantly, the remaining 591 druggable IMPPAT phytochemicals which have no similarity with any of the FDA approved drugs are novel candidates for designing new drugs based on natural products from Indian medicinal plants. </p> <p>For subsequent analysis, we selected 14 druggable phytochemicals with QEDw score <sup> <a href="#CR91">91</a> </sup> ≥0.9 which were designated as the <i>most-druggable</i> phytochemicals. Of these 14 phytochemicals, 12 were found to be similar to at least one of the FDA approved drugs based on either ECFP4 or MACCS keys based molecular fingerprint. The remaining 2 most-druggable phytochemicals, Onosmone (CID:102212116) and Truxillic acid (CID:78213), were found to have no similarity with any of the FDA approved drugs. Supplementary Figure <a href="#MOESM1">S1a</a> displays the similarity matrix based on Tc obtained using ECFP4 molecular fingerprint for the 14 most-druggable phytochemicals (Methods). From the similarity matrix, it is seen that 75 of the 91 Tc values between the 14 most-druggable phytochemicals are &amp;lt;0.5 implying high structural diversity. Moreover, the similarity matrix can be transformed into a similarity network for the 14 most-druggable phytochemicals by using a stringent threshold value of Tc ≥0.5 to determine edges in the graph (Supplementary Figure <a href="#MOESM1">S1b</a>; Methods). The similarity network for the 14 most-druggable phytochemicals has 16 edges and can be partitioned into a large connected component (cluster) of 7 phytochemicals, a smaller connected component of 2 phytochemicals and 5 remaining isolated phytochemicals. We highlight that the 2 phytochemicals, Onosmone and Truxillic acid, that have no similarity with any of the FDA approved drugs are among the isolated nodes in the similarity network (Supplementary Figure <a href="#MOESM1">S1b</a>). Based on plant-phytochemical associations in our database, Onosmone and Truxillic acid are phytochemicals of Indian medicinal plants, <i>Onosma echioides</i> and <i>Erythroxylum coca</i>, respectively, and a survey of the literature shows that these phytochemicals are under active investigation for their therapeutic uses <sup> <a href="#CR99">99</a>– <a href="#CR103">103</a> </sup>. We also highlight that none of the 14 most-druggable phytochemicals are captured by Phytochemica <sup> <a href="#CR11">11</a> </sup> database while 6 of the 14 phytochemicals are captured by Nutrichem <sup> <a href="#CR9">9</a>, <a href="#CR10">10</a> </sup> database. </p> <p>We also investigated the physicochemical properties of the 14 most-druggable phytochemicals. A principal component analysis (PCA) of the 14 most-druggable phytochemicals based on their physiochemical properties revealed that the first and second principal components together explained 69% of the total variance in the dataset (Supplementary Figure <a href="#MOESM1">S1c</a>). We find that some of the 7 most-druggable phytochemicals which are clustered together in the structural similarity space (Supplementary Figure <a href="#MOESM1">S1b</a>) are not clustered together in the physicochemical space (Supplementary Figure <a href="#MOESM1">S1c</a>). These observations based on limited analysis of 14 most-druggable phytochemicals suggest that a combined exploration of structural similarity space, physicochemical space and biological activity space of IMPPAT phytochemicals will facilitate identification and design of novel drugs. Thus, in future, it will be also worthwhile to compile biological activity profiles for phytochemicals of Indian medicinal plants. </p> </div> <div class="comparisonwithphytochemicalspaceofchinesemedicinalplants" title="sec"> <div class="title" tagx="title" title="title">
Comparison with phytochemical space of Chinese medicinal plants</div> <p>We have compared the set of 9596 IMPPAT phytochemicals produced by Indian medicinal plants with the set of 10140 TCM-Mesh <sup> <a href="#CR16">16</a> </sup> phytochemicals produced by Chinese medicinal plants (Methods). By comparing the 9596 IMPPAT phytochemicals with 10140 TCM-Mesh phytochemicals, we find that less than 25%, specifically 2305 phytochemicals, are common between the two databases (Fig. <a href="#Fig6">6a</a>). Among the 9596 IMPPAT phytochemicals, a subset of 960 phytochemicals were found to be druggable based on multiple druggability scores, namely, RO5 <sup> <a href="#CR23">23</a> </sup>, Traffic Lights <sup> <a href="#CR24">24</a> </sup>, GSK’s 4/400 <sup> <a href="#CR25">25</a> </sup>, Pfizer’s 3/75 <sup> <a href="#CR26">26</a> </sup>, Veber rule <sup> <a href="#CR27">27</a> </sup> and Egan rule <sup> <a href="#CR28">28</a> </sup> (Fig. <a href="#Fig5">5a</a>). Among the 10140 TCM-Mesh phytochemicals, we found a subset of 972 phytochemicals to be druggable based on multiple druggability scores listed above (Fig. <a href="#Fig6">6b</a>). Thus, the relative size of the filtered subset of druggable phytochemicals is very similar for IMPPAT database (10%) on Indian medicinal plants and TCM-Mesh database (9.6%) on Chinese medicinal plants. Figure <a href="#Fig6">6c</a> shows the distribution of QEDw <sup> <a href="#CR91">91</a> </sup> scores for the 972 druggable TCM-Mesh phytochemicals with mean score of 0.58 and standard deviation of 0.16, and thus, this distribution is similar to that for 960 druggable IMPPAT phytochemicals shown in Fig. <a href="#Fig5">5c</a>. By comparing the set of FDA approved drugs with the subset of druggable phytochemicals in IMPPAT and TCM-Mesh, we find that 32 approved drugs are contained in 960 IMPPAT phytochemicals while 19 approved drugs are contained in 972 TCM-Mesh phytochemicals. Thus, we find that majority of druggable phytochemicals in both IMPPAT (928 phytochemicals) and TCM-Mesh (953 phytochemicals) are potential hits for future drug discovery. Lastly, we find only a small overlap of 242 phytochemicals between the set of 960 druggable IMPPAT phytochemicals and 972 druggable TCM-Mesh phytochemicals (Fig. <a href="#Fig6">6d</a>), and thus, phytochemicals from both Indian herbs and Chinese herbs offer extensive opportunity for novel drug discovery. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 6</span> </a> </div>   <p>Comparison of the phytochemical space of Indian and Chinese medicinal plants. ( <b>a</b>) Venn diagram shows the overlap of the phytochemicals in IMPPAT and TCM-Mesh database. ( <b>b</b>) Evaluation of the drug-likeliness of TCM-Mesh phytochemicals based on multiple scores. The horizontal bar plot shows the number of phytochemicals in the TCM-Mesh database that satisfy different druggability scores (Methods). The vertical bar plot shows the overlap between sets of TCM-Mesh phytochemicals that satisfy different druggability scores. The pink bar in the vertical plot gives the 972 phytochemicals in TCM-Mesh database which satisfy all druggability scores. ( <b>c</b>) Distribution of QEDw <sup> <a href="#CR91">91</a> </sup> scores for the 972 TCM-Mesh phytochemicals which satisfy all druggability scores. ( <b>d</b>) Venn diagram shows the overlap between the druggable phytochemicals in IMPPAT database and TCM-Mesh database. </p>   </div> <p /> </div> </div> <div class="discussionandfuturedirections" title="sec"> <div class="title" tagx="title" title="title">
Discussion and future directions</div> <p>Cheminformatics can accelerate drug discovery from diverse natural sources <sup> <a href="#CR5">5</a> </sup>. We here incorporate cheminformatic principles to build an extensive resource on phytochemistry and ethnopharmacology of Indian medicinal plants. Here we present, IMPPAT, a curated database of 1742 <div class="u_UNKNOWN">I</div>ndian <div class="u_UNKNOWN">M</div>edicinal <div class="u_UNKNOWN">P</div>lants, 9596 <div class="u_UNKNOWN">P</div>hytochemicals, <div class="u_UNKNOWN">A</div>nd 1124 <div class="u_UNKNOWN">T</div> herapeutic uses which is the largest, freely accessible, digital resource on natural products from Indian herbs to date. IMPPAT provides chemical classification, 2D and 3D chemical structure, physicochemical properties, predicted ADMET properties, drug-likeliness scores and predicted human target proteins for phytochemicals in the database, and the available information in the database can be used for virtual screening. IMPPAT also captures limited information on the associations between Indian medicinal plants and their use in traditional Indian medicinal formulations. Thus, IMPPAT provides a unifying platform for the application of computational approaches to elucidate mechanistic links between phytochemicals of Indian medicinal plants and their therapeutic action. </p> <p>Following Clemons <i>et al</i>. <sup> <a href="#CR78">78</a> </sup>, we have compared the distributions of stereochemical complexity and shape complexity (Fsp <sup>3</sup>) across 9596 IMPPAT phytochemicals with small molecule collections, CC, DC’ and NP and 10140 TCM-Mesh <sup> <a href="#CR16">16</a> </sup> phytochemicals from Chinese medicinal plants (Fig. <a href="#Fig4">4a–b</a>). Interestingly, we show that the mean and median of stereochemical complexity or shape complexity of IMPPAT phytochemicals is closer to NP or TCM-Mesh phytochemicals than CC or DC’ collections. Following Clemons <i>et al</i>. <sup> <a href="#CR98">98</a> </sup>, we have also compared six physicochemical properties of CC, DC’, NP and TCM-Mesh phytochemicals with IMPPAT phytochemicals (Fig. <a href="#Fig4">4c</a>) to show that IMPPAT phytochemicals are closer to TCM-Mesh phytochemicals in physicochemical space (Fig. <a href="#Fig4">4c</a>). These results suggest that the IMPPAT library of phytochemicals is more likely to be enriched for specific protein binders rather than promiscuous binders <sup> <a href="#CR78">78</a> </sup>, and thus, our phytochemical library is expected to be a valuable addition to natural product-based screening collections. </p> <p>Using cheminformatic approaches, we found that 960 of the 9596 IMPPAT phytochemicals of Indian medicinal plants are potentially druggable based on multiple scoring schemes. Of the 960 IMPPAT phytochemicals which satisfy all druggability scores evaluated here, a subset of 14 phytochemicals were found to have a QEDw <sup> <a href="#CR91">91</a> </sup> score ≥0.9 (Supplementary Figure <a href="#MOESM1">S1</a>). Interestingly, the occurrence of these 14 most-druggable phytochemicals across 1742 Indian medicinal plants in our database is very rare with none of the 14 phytochemicals being found in more than 3 Indian medicinal plants. Specifically, the 14 most-druggable phytochemicals are constituents of only 17 Indian medicinal plants in our database. Also, 4 of the 14 most-druggable phytochemicals are constituents of 3 phylogenetically close Indian medicinal plants, <i>Iris germanica, Iris nepalensis</i> and <i>Iris kemaonensis</i>, which are from the same genus. However, we find that only 2 out of 17 Indian medicinal plants that produce the 14 most-druggable phytochemicals are in the priority list of Ministry of AYUSH, Government of India. This analysis suggests a possible revision in the AYUSH priority list to include the remaining 15 Indian medicinal plants that produced the majority of the most-druggable phytochemicals in our database. Thus, our resource will also aid in future expansion of the chemotaxonomy <sup> <a href="#CR104">104</a> </sup> of Indian medicinal plants. </p> <p>Lastly, we have also compared the IMPPAT phytochemicals from Indian herbs with the TCM-Mesh <sup> <a href="#CR16">16</a> </sup> phytochemicals from Chinese herbs to show that roughly 75% of the phytochemicals are unique to each database (Fig. <a href="#Fig6">6</a>). Moreover, we found that the filtered subsets of druggable phytochemicals in IMPPAT and TCM-Mesh are similar in size with roughly 75% of druggable phytochemicals unique to each database (Fig. <a href="#Fig6">6</a>). Furthermore, among the 960 and 972 druggable phytochemicals in IMPPAT and TCM-Mesh, respectively, a small fraction of 32 and 19 approved FDA drugs are contained in IMPPAT and TCM-Mesh, respectively. In sum, our results underline the vast potential of both Indian and Chinese herbs for future drug discovery. </p> <p>In the future, we hope to update IMPPAT database with the following additional information. Firstly, it will be important to update our database with more detailed information on the parts of the Indian medicinal plants such as leaves, stem or root, that produce the different phytochemicals along with relative composition of phytochemicals in different parts of the plants. Such detailed information on the relative phytochemical composition of parts of Indian medicinal plants will be crucial for evaluating and developing traditional Indian medicine formulations <sup> <a href="#CR74">74</a> </sup>. However, significant manual curation and literature mining will be needed to expand our database to include the relative phytochemical composition of the different parts of 1742 Indian medicinal plants which is beyond the scope of the present work. Secondly, it will be important to enrich our database by incorporating more traditional Indian medicinal formulations. For example, TKDL ( <a href="http://www.tkdl.res.in">http://www.tkdl.res.in</a>) has made only 1200 of their documented 250000 traditional Indian medicinal formulations openly accessible, and future efforts to associate this wealth of information to our database will shed mechanistic information on the therapeutic action of traditional formulations. Thirdly, it will be important to compile known biological activity information for phytochemicals of Indian medicinal plants. In conclusion, IMPPAT database will serve as a valuable resource in herbal drug discovery. </p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41598_2018_22631_MOESM1_ESM.docx">LINK</a>  <p>Supplementary Figure 1</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41598_2018_22631_MOESM2_ESM.xlsx">LINK</a>  <p>Supplementary Tables S1-S3</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p>Karthikeyan Mohanraj, Bagavathy Shanmugam Karthikeyan and R.P. Vivek-Ananth contributed equally to this work. </p> </div> <div class="fn-type-" title=""> <p> <b>Electronic supplementary material</b> </p> <p> <b>Supplementary information</b> accompanies this paper at 10.1038/s41598-018-22631-z. </p> </div> <div class="fn-type-" title=""> <p> <b>Publisher's note:</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. </p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We would like to thank Gopal C. Nanda and the staff of Achanta Lakshmipathi Research Centre for Ayurveda, Chennai for discussions and help in accessing relevant literature. We also thank S. Krishnaswamy and James Craig for discussions and the library staff of The Institute of Mathematical Sciences, Chennai for facilitating access to scientific literature. We thank the anonymous reviewers for their constructive comments which helped improve the manuscript. AS acknowledges financial support from Department of Science and Technology (DST) India through the award of a start-up grant (YSS/2015/000060) and Ramanujan fellowship (SB/S2/RJN-006/2014), Max Planck Society Germany through the award of a Max Planck Partner Group, and Department of Atomic Energy (DAE) India. The funders have no role in study design, data collection, data analysis, manuscript preparation or decision to publish. </p> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>K.M., B.S.K. and R.P.V. contributed equally to this work. A.S., R.P.V., B.S.K. and K.M. designed research. K.M. designed the database platform and visual interface. R.P.V. performed the cheminformatic analysis. K.M., B.S.K., R.P.V., R.P.B., S.R.A. and P.M. compiled and curated data from various sources. R.P.V., B.S.K. and A.S. wrote the manuscript. A.S. and R.P.V. are the lead-writers of the manuscript. All authors have read and approved the manuscript. </p> <div class="competinginterests" title="sec"> <div class="title" tagx="title" title="title">
Competing Interests</div> <p>The authors declare no competing interests.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="CR1" /> <span class="label" tagx="label" title="label">1.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koehn</span> <span class="given-names" tagx="given-names" title="given-names">FE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carter</span> <span class="given-names" tagx="given-names" title="given-names">GT</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The evolving role of natural products in drug discovery</span> <span class="source" tagx="source" title="source">Nat Rev Drug Discov</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">206</span> <span class="lpage" tagx="lpage" title="lpage">220</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nrd1657">10.1038/nrd1657</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15729362">15729362</a> </span> </span> </li> <li tag="ref"> <a name="CR2" /> <span class="label" tagx="label" title="label">2.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Newman</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cragg</span> <span class="given-names" tagx="given-names" title="given-names">GM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Natural Products as Sources of New Drugs from 1981 to 2014</span> <span class="source" tagx="source" title="source">J Nat Prod</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">79</span> <span class="fpage" tagx="fpage" title="fpage">629</span> <span class="lpage" tagx="lpage" title="lpage">661</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/acs.jnatprod.5b01055">10.1021/acs.jnatprod.5b01055</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26852623">26852623</a> </span> </span> </li> <li tag="ref"> <a name="CR3" /> <span class="label" tagx="label" title="label">3.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">JW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vederas</span> <span class="given-names" tagx="given-names" title="given-names">JC</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Drug discovery and natural products: end of an era or an endless frontier?</span> <span class="source" tagx="source" title="source">Science</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">325</span> <span class="fpage" tagx="fpage" title="fpage">161</span> <span class="lpage" tagx="lpage" title="lpage">165</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.1168243">10.1126/science.1168243</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19589993">19589993</a> </span> </span> </li> <li tag="ref"> <a name="CR4" /> <span class="label" tagx="label" title="label">4.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pye</span> <span class="given-names" tagx="given-names" title="given-names">CR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bertin</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lokey</span> <span class="given-names" tagx="given-names" title="given-names">RS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gerwick</span> <span class="given-names" tagx="given-names" title="given-names">WH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Linington</span> <span class="given-names" tagx="given-names" title="given-names">RG</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Retrospective analysis of natural products provides insights for future discovery trends </span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">114</span> <span class="fpage" tagx="fpage" title="fpage">5601</span> <span class="lpage" tagx="lpage" title="lpage">5606</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.1614680114">10.1073/pnas.1614680114</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28461474">28461474</a> </span> </span> </li> <li tag="ref"> <a name="CR5" /> <span class="label" tagx="label" title="label">5.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lagunin</span> <span class="given-names" tagx="given-names" title="given-names">AA</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Chemo- and bioinformatics resources for in silico drug discovery from medicinal plants beyond their traditional use: a critical review </span> <span class="source" tagx="source" title="source">Nat Prod Rep</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">31</span> <span class="fpage" tagx="fpage" title="fpage">1585</span> <span class="lpage" tagx="lpage" title="lpage">1611</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1039/C4NP00068D">10.1039/C4NP00068D</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25051191">25051191</a> </span> </span> </li> <li tag="ref"> <a name="CR6" /> <span class="label" tagx="label" title="label">6.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Pandey, M. M., Rastogi, S. &amp;amp; Rawat, A. K. Indian traditional ayurvedic system of medicine and nutritional supplementation. <i>Evid Based Complement Alternat Med</i>, 376327 (2013). </span> </li> <li tag="ref"> <a name="CR7" /> <span class="label" tagx="label" title="label">7.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Nadkarni, M. K. &amp;amp; Nadkarni, A. K. <i>Indian materia medica</i>. (Popular Book Depot, 1955). </span> </li> <li tag="ref"> <a name="CR8" /> <span class="label" tagx="label" title="label">8.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Dash, V. B. &amp;amp; Kashyap, V. L. <i>Ayurveda Materia Medica</i>. 711 (Concept Publishing Company, 1999). </span> </li> <li tag="ref"> <a name="CR9" /> <span class="label" tagx="label" title="label">9.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jensen</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Panagiotou</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kouskoumvekaki</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Integrated text mining and chemoinformatics analysis associates diet to health benefit at molecular level </span> <span class="source" tagx="source" title="source">PLoS Comput Biol</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">e1003432</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pcbi.1003432">10.1371/journal.pcbi.1003432</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24453957">24453957</a> </span> </span> </li> <li tag="ref"> <a name="CR10" /> <span class="label" tagx="label" title="label">10.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jensen</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Panagiotou</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kouskoumvekaki</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">NutriChem: a systems chemical biology resource to explore the medicinal value of plant-based foods </span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">43</span> <span class="fpage" tagx="fpage" title="fpage">D940</span> <span class="lpage" tagx="lpage" title="lpage">945</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gku724">10.1093/nar/gku724</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25106869">25106869</a> </span> </span> </li> <li tag="ref"> <a name="CR11" /> <span class="label" tagx="label" title="label">11.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Pathania, S., Ramakrishnan, S. M. &amp;amp; Bagler, G. Phytochemica: a platform to explore phytochemicals of medicinal plants. <i>Database (Oxford)</i> <b>2015</b> (2015). </span> </li> <li tag="ref"> <a name="CR12" /> <span class="label" tagx="label" title="label">12.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gu</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gui</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yuan</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">CVDHD: a cardiovascular disease herbal database for drug discovery and network pharmacology</span> <span class="source" tagx="source" title="source">J Cheminform</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">51</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1758-2946-5-51">10.1186/1758-2946-5-51</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24344970">24344970</a> </span> </span> </li> <li tag="ref"> <a name="CR13" /> <span class="label" tagx="label" title="label">13.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Afendi</span> <span class="given-names" tagx="given-names" title="given-names">FM</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">KNApSAcK family databases: integrated metabolite-plant species databases for multifaceted plant research </span> <span class="source" tagx="source" title="source">Plant Cell Physiol</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">53</span> <span class="fpage" tagx="fpage" title="fpage">e1</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/pcp/pcr165">10.1093/pcp/pcr165</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22123792">22123792</a> </span> </span> </li> <li tag="ref"> <a name="CR14" /> <span class="label" tagx="label" title="label">14.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">CY</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">TCM Database@Taiwan: the world’s largest traditional Chinese medicine database for drug screening in silico </span> <span class="source" tagx="source" title="source">PLoS One</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">e15939</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0015939">10.1371/journal.pone.0015939</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21253603">21253603</a> </span> </span> </li> <li tag="ref"> <a name="CR15" /> <span class="label" tagx="label" title="label">15.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xue</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">TCMID: Traditional Chinese Medicine integrative database for herb molecular mechanism analysis </span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">41</span> <span class="fpage" tagx="fpage" title="fpage">D1089</span> <span class="lpage" tagx="lpage" title="lpage">1095</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gks1100">10.1093/nar/gks1100</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23203875">23203875</a> </span> </span> </li> <li tag="ref"> <a name="CR16" /> <span class="label" tagx="label" title="label">16.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">RZ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">SJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bai</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ning</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">TCM-Mesh: The database and analytical system for network pharmacology analysis for TCM preparations </span> <span class="source" tagx="source" title="source">Sci Rep</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">2821</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/s41598-017-03039-7">10.1038/s41598-017-03039-7</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28588237">28588237</a> </span> </span> </li> <li tag="ref"> <a name="CR17" /> <span class="label" tagx="label" title="label">17.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mangal</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sagar</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singh</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raghava</span> <span class="given-names" tagx="given-names" title="given-names">GP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Agarwal</span> <span class="given-names" tagx="given-names" title="given-names">SM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">NPACT: Naturally Occurring Plant-based Anti-cancer Compound-Activity-Target database</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">41</span> <span class="fpage" tagx="fpage" title="fpage">D1124</span> <span class="lpage" tagx="lpage" title="lpage">1129</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gks1047">10.1093/nar/gks1047</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23203877">23203877</a> </span> </span> </li> <li tag="ref"> <a name="CR18" /> <span class="label" tagx="label" title="label">18.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Polur</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Joshi</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Workman</span> <span class="given-names" tagx="given-names" title="given-names">CT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lavekar</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kouskoumvekaki</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Back to the Roots: Prediction of Biologically Active Natural Products from Ayurveda Traditional Medicine </span> <span class="source" tagx="source" title="source">Mol Inform</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">30</span> <span class="fpage" tagx="fpage" title="fpage">181</span> <span class="lpage" tagx="lpage" title="lpage">187</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/minf.201000163">10.1002/minf.201000163</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27466772">27466772</a> </span> </span> </li> <li tag="ref"> <a name="CR19" /> <span class="label" tagx="label" title="label">19.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Law</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">DrugBank 4.0: shedding new light on drug metabolism</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">42</span> <span class="fpage" tagx="fpage" title="fpage">D1091</span> <span class="lpage" tagx="lpage" title="lpage">1097</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkt1068">10.1093/nar/gkt1068</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24203711">24203711</a> </span> </span> </li> <li tag="ref"> <a name="CR20" /> <span class="label" tagx="label" title="label">20.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Lagorce, D., Bouslama, L., Becot, J., Miteva, M. A. &amp;amp; Villoutreix, B. O. FAF-Drugs4: free ADME-tox filtering computations for chemical biology and early stages drug discovery. <i>Bioinformatics</i> (2017). </span> </li> <li tag="ref"> <a name="CR21" /> <span class="label" tagx="label" title="label">21.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">RDKit: Open-source cheminformatics. http://www.rdkit.org.</span> </li> <li tag="ref"> <a name="CR22" /> <span class="label" tagx="label" title="label">22.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheng</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">admetSAR: a comprehensive source and free tool for assessment of chemical ADMET properties</span> <span class="source" tagx="source" title="source">J Chem Inf Model</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">52</span> <span class="fpage" tagx="fpage" title="fpage">3099</span> <span class="lpage" tagx="lpage" title="lpage">3105</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/ci300367a">10.1021/ci300367a</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23092397">23092397</a> </span> </span> </li> <li tag="ref"> <a name="CR23" /> <span class="label" tagx="label" title="label">23.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lipinski</span> <span class="given-names" tagx="given-names" title="given-names">CA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lombardo</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dominy</span> <span class="given-names" tagx="given-names" title="given-names">BW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feeney</span> <span class="given-names" tagx="given-names" title="given-names">PJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings1PII of original article: S0169-409X(96)00423-1. The article was originally published in Advanced Drug Delivery Reviews 23 (1997) 3–25.1 </span> <span class="source" tagx="source" title="source">Advanced Drug Delivery Reviews</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">46</span> <span class="fpage" tagx="fpage" title="fpage">3</span> <span class="lpage" tagx="lpage" title="lpage">26</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0169-409X(00)00129-0">10.1016/S0169-409X(00)00129-0</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11259830">11259830</a> </span> </span> </li> <li tag="ref"> <a name="CR24" /> <span class="label" tagx="label" title="label">24.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lobell</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title"> <i>In Silico</i> ADMET Traffic Lights as a Tool for the Prioritization of HTS Hits </span> <span class="source" tagx="source" title="source">ChemMedChem</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">1</span> <span class="fpage" tagx="fpage" title="fpage">1229</span> <span class="lpage" tagx="lpage" title="lpage">1236</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/cmdc.200600168">10.1002/cmdc.200600168</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16991174">16991174</a> </span> </span> </li> <li tag="ref"> <a name="CR25" /> <span class="label" tagx="label" title="label">25.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gleeson</span> <span class="given-names" tagx="given-names" title="given-names">MP</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Generation of a Set of Simple, Interpretable ADMET Rules of Thumb</span> <span class="source" tagx="source" title="source">Journal of Medicinal Chemistry</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">51</span> <span class="fpage" tagx="fpage" title="fpage">817</span> <span class="lpage" tagx="lpage" title="lpage">834</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/jm701122q">10.1021/jm701122q</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18232648">18232648</a> </span> </span> </li> <li tag="ref"> <a name="CR26" /> <span class="label" tagx="label" title="label">26.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hughes</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Physiochemical drug properties associated with <i>in vivo</i> toxicological outcomes </span> <span class="source" tagx="source" title="source">Bioorganic &amp;amp; Medicinal Chemistry Letters</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">18</span> <span class="fpage" tagx="fpage" title="fpage">4872</span> <span class="lpage" tagx="lpage" title="lpage">4875</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.bmcl.2008.07.071">10.1016/j.bmcl.2008.07.071</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18691886">18691886</a> </span> </span> </li> <li tag="ref"> <a name="CR27" /> <span class="label" tagx="label" title="label">27.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Veber</span> <span class="given-names" tagx="given-names" title="given-names">DF</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Molecular properties that influence the oral bioavailability of drug candidates</span> <span class="source" tagx="source" title="source">Journal of medicinal chemistry</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">45</span> <span class="fpage" tagx="fpage" title="fpage">2615</span> <span class="lpage" tagx="lpage" title="lpage">2623</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/jm020017n">10.1021/jm020017n</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12036371">12036371</a> </span> </span> </li> <li tag="ref"> <a name="CR28" /> <span class="label" tagx="label" title="label">28.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Egan</span> <span class="given-names" tagx="given-names" title="given-names">WJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Merz</span> <span class="given-names" tagx="given-names" title="given-names">KM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baldwin</span> <span class="given-names" tagx="given-names" title="given-names">JJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Prediction of Drug Absorption Using Multivariate Statistics</span> <span class="source" tagx="source" title="source">Journal of Medicinal Chemistry</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">43</span> <span class="fpage" tagx="fpage" title="fpage">3867</span> <span class="lpage" tagx="lpage" title="lpage">3877</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/jm000292e">10.1021/jm000292e</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11052792">11052792</a> </span> </span> </li> <li tag="ref"> <a name="CR29" /> <span class="label" tagx="label" title="label">29.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Szklarczyk</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">STITCH 5: augmenting protein–chemical interaction networks with tissue and affinity data </span> <span class="source" tagx="source" title="source">Nucleic acids research</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">44</span> <span class="fpage" tagx="fpage" title="fpage">D380</span> <span class="lpage" tagx="lpage" title="lpage">D384</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkv1277">10.1093/nar/gkv1277</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26590256">26590256</a> </span> </span> </li> <li tag="ref"> <a name="CR30" /> <span class="label" tagx="label" title="label">30.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Olsson</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oprea</span> <span class="given-names" tagx="given-names" title="given-names">TI</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Cheminformatics: a tool for decision-makers in drug discovery</span> <span class="source" tagx="source" title="source">Curr Opin Drug Discov Devel</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">308</span> <span class="lpage" tagx="lpage" title="lpage">313</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11560063">11560063</a> </span> </span> </li> <li tag="ref"> <a name="CR31" /> <span class="label" tagx="label" title="label">31.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kalwij</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Review of ‘The Plant List, a working list of all plant species’</span> <span class="source" tagx="source" title="source">Journal of Vegetation Science</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">23</span> <span class="fpage" tagx="fpage" title="fpage">998</span> <span class="lpage" tagx="lpage" title="lpage">1002</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1654-1103.2012.01407.x">10.1111/j.1654-1103.2012.01407.x</a> </span> </span> </li> <li tag="ref"> <a name="CR32" /> <span class="label" tagx="label" title="label">32.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Khare, C. P. <i>Indian medicinal plants: an illustrated dictionary</i>. (Springer, 2007). </span> </li> <li tag="ref"> <a name="CR33" /> <span class="label" tagx="label" title="label">33.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Sharma, R. &amp;amp; Gupta, T. <i>Encyclopaedia of Medicinal Plants</i>. Vol. I (Campus Books International, 2009). </span> </li> <li tag="ref"> <a name="CR34" /> <span class="label" tagx="label" title="label">34.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Sharma, R. &amp;amp; Gupta, T. <i>Encyclopaedia of Medicinal Plants</i>. Vol. II (Campus Books International, 2009). </span> </li> <li tag="ref"> <a name="CR35" /> <span class="label" tagx="label" title="label">35.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Sharma, R. &amp;amp; Gupta, T. Encyclopaedia of Medicinal Plants. Vol. III (Campus Books International, 2009). </span> </li> <li tag="ref"> <a name="CR36" /> <span class="label" tagx="label" title="label">36.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Sharma, R. &amp;amp; Gupta, T. <i>Encyclopaedia of Medicinal Plants</i>. Vol. IV (Campus Books International, 2009). </span> </li> <li tag="ref"> <a name="CR37" /> <span class="label" tagx="label" title="label">37.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Sharma, R. &amp;amp; Gupta, T. <i>Encyclopaedia of Medicinal Plants</i>. Vol. V (Campus Books International, 2009). </span> </li> <li tag="ref"> <a name="CR38" /> <span class="label" tagx="label" title="label">38.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Sharma, R. &amp;amp; Gupta, T. <i>Encyclopaedia of Medicinal Plants</i>. Vol. VI (Campus Books International, 2009). </span> </li> <li tag="ref"> <a name="CR39" /> <span class="label" tagx="label" title="label">39.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Sharma, R. &amp;amp; Gupta, T. <i>Encyclopaedia of Medicinal Plants</i>. Vol. VII (Campus Books International, 2009). </span> </li> <li tag="ref"> <a name="CR40" /> <span class="label" tagx="label" title="label">40.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Sharma, R. &amp;amp; Gupta, T. <i>Encyclopaedia of Medicinal Plants</i>. Vol. VIII (Campus Books International, 2009). </span> </li> <li tag="ref"> <a name="CR41" /> <span class="label" tagx="label" title="label">41.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Duke, J. A. <i>Handbook of phytochemical constituents of GRAS herbs and other economic plants</i>. (CRC Press, 1992). </span> </li> <li tag="ref"> <a name="CR42" /> <span class="label" tagx="label" title="label">42.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coordinators</span> <span class="given-names" tagx="given-names" title="given-names">NR</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Database Resources of the National Center for Biotechnology Information</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">45</span> <span class="fpage" tagx="fpage" title="fpage">D12</span> <span class="lpage" tagx="lpage" title="lpage">D17</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkw1071">10.1093/nar/gkw1071</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27899561">27899561</a> </span> </span> </li> <li tag="ref"> <a name="CR43" /> <span class="label" tagx="label" title="label">43.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Bird, S., Klein, E. &amp;amp; Loper, E. <i>Natural Language Processing with Python</i>. (O’Reilly Media, Inc., 2009). </span> </li> <li tag="ref"> <a name="CR44" /> <span class="label" tagx="label" title="label">44.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">PubChem Substance and Compound databases</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">44</span> <span class="fpage" tagx="fpage" title="fpage">D1202</span> <span class="lpage" tagx="lpage" title="lpage">1213</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkv951">10.1093/nar/gkv951</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26400175">26400175</a> </span> </span> </li> <li tag="ref"> <a name="CR45" /> <span class="label" tagx="label" title="label">45.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hastings</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The ChEBI reference database and ontology for biologically relevant chemistry: enhancements for 2013 </span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">41</span> <span class="fpage" tagx="fpage" title="fpage">D456</span> <span class="lpage" tagx="lpage" title="lpage">463</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gks1146">10.1093/nar/gks1146</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23180789">23180789</a> </span> </span> </li> <li tag="ref"> <a name="CR46" /> <span class="label" tagx="label" title="label">46.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Editorial: ChemSpider–a tool for Natural Products research. <i>Nat Prod Rep</i> <b>32</b>, 1163-1164 (2015). </span> </li> <li tag="ref"> <a name="CR47" /> <span class="label" tagx="label" title="label">47.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakamura</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">KNApSAcK-3D: a three-dimensional structure database of plant metabolites</span> <span class="source" tagx="source" title="source">Plant Cell Physiol</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">54</span> <span class="fpage" tagx="fpage" title="fpage">e4</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/pcp/pcs186">10.1093/pcp/pcs186</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23292603">23292603</a> </span> </span> </li> <li tag="ref"> <a name="CR48" /> <span class="label" tagx="label" title="label">48.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">P.J. Linstrom &amp;amp; W.G. Mallard, E. <i>NIST Chemistry WebBook, NIST Standard Reference Database Number</i> 69. (National Institute of Standards and Technology, GaithersburgMD, 20899). </span> </li> <li tag="ref"> <a name="CR49" /> <span class="label" tagx="label" title="label">49.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wishart</span> <span class="given-names" tagx="given-names" title="given-names">DS</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">HMDB 3.0–The Human Metabolome Database in 2013</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">41</span> <span class="fpage" tagx="fpage" title="fpage">D801</span> <span class="lpage" tagx="lpage" title="lpage">807</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gks1065">10.1093/nar/gks1065</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23161693">23161693</a> </span> </span> </li> <li tag="ref"> <a name="CR50" /> <span class="label" tagx="label" title="label">50.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feunang</span> <span class="given-names" tagx="given-names" title="given-names">YD</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">ClassyFire: automated chemical classification with a comprehensive, computable taxonomy</span> <span class="source" tagx="source" title="source">Journal of cheminformatics</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">61</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s13321-016-0174-y">10.1186/s13321-016-0174-y</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27867422">27867422</a> </span> </span> </li> <li tag="ref"> <a name="CR51" /> <span class="label" tagx="label" title="label">51.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Deorani, S. C. &amp;amp; Sharma, G. D. <i>Medicinal plants of Nagaland</i>. (Bishen Singh Mahendra Pal Singh, 2007). </span> </li> <li tag="ref"> <a name="CR52" /> <span class="label" tagx="label" title="label">52.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Dhiman, A. K. &amp;amp; Hora, S. L. <i>Wild Medicinal Plants of India</i>. (BSMPS, 2005). </span> </li> <li tag="ref"> <a name="CR53" /> <span class="label" tagx="label" title="label">53.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Govil, J. N. &amp;amp; Singh, V. K. <i>Recent progress in medicinal plants-Ethnomedicine and pharmacognosy</i>. Vol. I (Studium Press LLC, U.S.A., 2002). </span> </li> <li tag="ref"> <a name="CR54" /> <span class="label" tagx="label" title="label">54.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Govil, J. N. &amp;amp; Singh, V. K. <i>Recent progress in medicinal plants-Ethnomedicine and pharmacognosy</i> II. Vol. VII (Studium Press LLC, U.S.A., 2003). </span> </li> <li tag="ref"> <a name="CR55" /> <span class="label" tagx="label" title="label">55.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Govil, J. N. &amp;amp; Singh, V. K. <i>Recent progress in medicinal plants-Ethnomedicine and pharmacognosy IV</i>. Vol. XXII (Studium Press LLC, U.S.A., 2008). </span> </li> <li tag="ref"> <a name="CR56" /> <span class="label" tagx="label" title="label">56.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Govil, J. N. &amp;amp; Singh, V. K. <i>Recent progress in medicinal plants- Phytopharmacology and Therapeurtic values II</i>. (Studium Press LLC, U.S.A., 2008). </span> </li> <li tag="ref"> <a name="CR57" /> <span class="label" tagx="label" title="label">57.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Gupta, A. K., Tandon, N. &amp;amp; Sharma, M. <i>Quality Standards of Indian Medicinal Plants</i>. (Indian Council of Medical Research, 2006). </span> </li> <li tag="ref"> <a name="CR58" /> <span class="label" tagx="label" title="label">58.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Kaushik, P. &amp;amp; Dhiman, A. K. <i>Medicinal plants and raw drugs of India</i>. (Bishen Singh Mahendra Pal Singh, 2000). </span> </li> <li tag="ref"> <a name="CR59" /> <span class="label" tagx="label" title="label">59.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Kirtikar, K. R. &amp;amp; Basu, B. D. <i>Indian Medicinal Plants</i>. Vol. I (Periodical Experts Book Agency, 2006). </span> </li> <li tag="ref"> <a name="CR60" /> <span class="label" tagx="label" title="label">60.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Kirtikar, K. R. &amp;amp; Basu, B. D. <i>Indian Medicinal Plants</i>. Vol. II (Periodical Experts Book Agency, 2006). </span> </li> <li tag="ref"> <a name="CR61" /> <span class="label" tagx="label" title="label">61.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Kirtikar, K. R. &amp;amp; Basu, B. D. <i>Indian Medicinal Plants</i>. Vol. III (Periodical Experts Book Agency, 2012). </span> </li> <li tag="ref"> <a name="CR62" /> <span class="label" tagx="label" title="label">62.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Kshirsagar, R. D. &amp;amp; Singh, N. P. <i>Ethnobotany of Mysore and Coorg, Karnataka State</i>. (Bishen Singh Mahendra Pal Singh, 2007). </span> </li> <li tag="ref"> <a name="CR63" /> <span class="label" tagx="label" title="label">63.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">P., K. C. <i>Medicinal Plants of Indian Trans Himalaya: Focus on Tibetan use of Medicinal Resources</i>. (Bishen Singh Mahendra Pal Singh, 2003). </span> </li> <li tag="ref"> <a name="CR64" /> <span class="label" tagx="label" title="label">64.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Pande, P. C., Tiwari, L. &amp;amp; Pande, H. C. <i>Folk-medicine and aromatic plants of Uttaranchal</i>. (Bishen Singh Mahendra Pal Singh, 2006). </span> </li> <li tag="ref"> <a name="CR65" /> <span class="label" tagx="label" title="label">65.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Sharma, U. K. <i>Medicinal plants of Assa</i>m. (2004). </span> </li> <li tag="ref"> <a name="CR66" /> <span class="label" tagx="label" title="label">66.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Singh, K. K. &amp;amp; Kushal, K. <i>Ethnobotanical wisdom of Gaddi tribe in Western Himalaya</i>. (Bishen Singh Mahendra Pal Singh, 2000). </span> </li> <li tag="ref"> <a name="CR67" /> <span class="label" tagx="label" title="label">67.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Viswanathan, M. B., Prem Kumar, E. H. &amp;amp; Ramesh, N. <i>Ethnobotany of the Kanis: Kalakkad-Mundanthurai Tiger Reserve in Tirunelveli district, Tamilnadu, India </i>. (Bishen Singh Mahendra Pal Singh, 2006). </span> </li> <li tag="ref"> <a name="CR68" /> <span class="label" tagx="label" title="label">68.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schriml</span> <span class="given-names" tagx="given-names" title="given-names">LM</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Disease Ontology: a backbone for disease semantic integration</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">40</span> <span class="fpage" tagx="fpage" title="fpage">D940</span> <span class="lpage" tagx="lpage" title="lpage">946</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkr972">10.1093/nar/gkr972</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22080554">22080554</a> </span> </span> </li> <li tag="ref"> <a name="CR69" /> <span class="label" tagx="label" title="label">69.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hamosh</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Scott</span> <span class="given-names" tagx="given-names" title="given-names">AF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Amberger</span> <span class="given-names" tagx="given-names" title="given-names">JS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bocchini</span> <span class="given-names" tagx="given-names" title="given-names">CA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McKusick</span> <span class="given-names" tagx="given-names" title="given-names">VA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Online Mendelian Inheritance in Man (OMIM), a knowledgebase of human genes and genetic disorders </span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">33</span> <span class="fpage" tagx="fpage" title="fpage">D514</span> <span class="lpage" tagx="lpage" title="lpage">517</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gki033">10.1093/nar/gki033</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15608251">15608251</a> </span> </span> </li> <li tag="ref"> <a name="CR70" /> <span class="label" tagx="label" title="label">70.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bodenreider</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The Unified Medical Language System (UMLS): integrating biomedical terminology</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">32</span> <span class="fpage" tagx="fpage" title="fpage">D267</span> <span class="lpage" tagx="lpage" title="lpage">270</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkh061">10.1093/nar/gkh061</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14681409">14681409</a> </span> </span> </li> <li tag="ref"> <a name="CR71" /> <span class="label" tagx="label" title="label">71.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rogers</span> <span class="given-names" tagx="given-names" title="given-names">FB</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Medical subject headings</span> <span class="source" tagx="source" title="source">Bull Med Libr Assoc</span> <span class="year" tagx="year" title="year">1963</span> <span class="volume" tagx="volume" title="volume">51</span> <span class="fpage" tagx="fpage" title="fpage">114</span> <span class="lpage" tagx="lpage" title="lpage">116</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/13982385">13982385</a> </span> </span> </li> <li tag="ref"> <a name="CR72" /> <span class="label" tagx="label" title="label">72.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pinero</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">DisGeNET: a comprehensive platform integrating information on human disease-associated genes and variants </span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">45</span> <span class="fpage" tagx="fpage" title="fpage">D833</span> <span class="lpage" tagx="lpage" title="lpage">D839</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkw943">10.1093/nar/gkw943</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27924018">27924018</a> </span> </span> </li> <li tag="ref"> <a name="CR73" /> <span class="label" tagx="label" title="label">73.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Menche</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barabasi</span> <span class="given-names" tagx="given-names" title="given-names">AL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharma</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Human symptoms-disease network</span> <span class="source" tagx="source" title="source">Nat Commun</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">4212</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24967666">24967666</a> </span> </span> </li> <li tag="ref"> <a name="CR74" /> <span class="label" tagx="label" title="label">74.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dev</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Ancient-modern concordance in Ayurvedic plants: some examples</span> <span class="source" tagx="source" title="source">Environ Health Perspect</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">107</span> <span class="fpage" tagx="fpage" title="fpage">783</span> <span class="lpage" tagx="lpage" title="lpage">789</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1289/ehp.99107783">10.1289/ehp.99107783</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10504143">10504143</a> </span> </span> </li> <li tag="ref"> <a name="CR75" /> <span class="label" tagx="label" title="label">75.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vainio</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnson</span> <span class="given-names" tagx="given-names" title="given-names">MS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Generating conformer ensembles using a multiobjective genetic algorithm</span> <span class="source" tagx="source" title="source">Journal of chemical information and modeling</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">47</span> <span class="fpage" tagx="fpage" title="fpage">2462</span> <span class="lpage" tagx="lpage" title="lpage">2474</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/ci6005646">10.1021/ci6005646</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17892278">17892278</a> </span> </span> </li> <li tag="ref"> <a name="CR76" /> <span class="label" tagx="label" title="label">76.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">O’Boyle</span> <span class="given-names" tagx="given-names" title="given-names">NM</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Open Babel: An open chemical toolbox</span> <span class="source" tagx="source" title="source">Journal of cheminformatics</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="fpage" tagx="fpage" title="fpage">33</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1758-2946-3-33">10.1186/1758-2946-3-33</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21982300">21982300</a> </span> </span> </li> <li tag="ref"> <a name="CR77" /> <span class="label" tagx="label" title="label">77.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Tung, C. W. <i>et al</i>. TIPdb-3D: the three-dimensional structure database of phytochemicals from Taiwan indigenous plants. <i>Database (Oxford)</i> <b>2014</b> (2014). </span> </li> <li tag="ref"> <a name="CR78" /> <span class="label" tagx="label" title="label">78.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Clemons</span> <span class="given-names" tagx="given-names" title="given-names">PA</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Small molecules of different origins have distinct distributions of structural complexity that correlate with protein-binding profiles </span> <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">107</span> <span class="fpage" tagx="fpage" title="fpage">18787</span> <span class="lpage" tagx="lpage" title="lpage">18792</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.1012741107">10.1073/pnas.1012741107</a> </span> </span> </li> <li tag="ref"> <a name="CR79" /> <span class="label" tagx="label" title="label">79.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lovering</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bikker</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Humblet</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Escape from flatland: increasing saturation as an approach to improving clinical success</span> <span class="source" tagx="source" title="source">Journal of medicinal chemistry</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">52</span> <span class="fpage" tagx="fpage" title="fpage">6752</span> <span class="lpage" tagx="lpage" title="lpage">6756</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/jm901241e">10.1021/jm901241e</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19827778">19827778</a> </span> </span> </li> <li tag="ref"> <a name="CR80" /> <span class="label" tagx="label" title="label">80.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shen</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheng</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Estimation of ADME properties with substructure pattern recognition</span> <span class="source" tagx="source" title="source">J Chem Inf Model</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">50</span> <span class="fpage" tagx="fpage" title="fpage">1034</span> <span class="lpage" tagx="lpage" title="lpage">1041</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/ci100104j">10.1021/ci100104j</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20578727">20578727</a> </span> </span> </li> <li tag="ref"> <a name="CR81" /> <span class="label" tagx="label" title="label">81.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pham The</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">In Silico Prediction of Caco‐2 Cell Permeability by a Classification QSAR Approach</span> <span class="source" tagx="source" title="source">Molecular informatics</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">30</span> <span class="fpage" tagx="fpage" title="fpage">376</span> <span class="lpage" tagx="lpage" title="lpage">385</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/minf.201000118">10.1002/minf.201000118</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27466954">27466954</a> </span> </span> </li> <li tag="ref"> <a name="CR82" /> <span class="label" tagx="label" title="label">82.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">P-glycoprotein substrate models using support vector machines based on a comprehensive data set </span> <span class="source" tagx="source" title="source">Journal of chemical information and modeling</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">51</span> <span class="fpage" tagx="fpage" title="fpage">1447</span> <span class="lpage" tagx="lpage" title="lpage">1456</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/ci2001583">10.1021/ci2001583</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21604677">21604677</a> </span> </span> </li> <li tag="ref"> <a name="CR83" /> <span class="label" tagx="label" title="label">83.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheng</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Insights into molecular basis of cytochrome p450 inhibitory promiscuity of compounds</span> <span class="source" tagx="source" title="source">Journal of chemical information and modeling</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">51</span> <span class="fpage" tagx="fpage" title="fpage">2482</span> <span class="lpage" tagx="lpage" title="lpage">2495</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/ci200317s">10.1021/ci200317s</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21875141">21875141</a> </span> </span> </li> <li tag="ref"> <a name="CR84" /> <span class="label" tagx="label" title="label">84.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheng</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Classification of cytochrome P450 inhibitors and noninhibitors using combined classifiers</span> <span class="source" tagx="source" title="source">Journal of chemical information and modeling</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">51</span> <span class="fpage" tagx="fpage" title="fpage">996</span> <span class="lpage" tagx="lpage" title="lpage">1011</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/ci200028n">10.1021/ci200028n</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21491913">21491913</a> </span> </span> </li> <li tag="ref"> <a name="CR85" /> <span class="label" tagx="label" title="label">85.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Carbon‐Mangels, M. &amp;amp; Hutter, M. C. Selecting relevant descriptors for classification by bayesian estimates: a comparison with decision trees and support vector machines approaches for disparate data sets. <i>Molecular informatics</i> <b>30</b>, 885-895 (2011). </span> </li> <li tag="ref"> <a name="CR86" /> <span class="label" tagx="label" title="label">86.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hansen</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Benchmark data set for in silico prediction of Ames mutagenicity</span> <span class="source" tagx="source" title="source">Journal of chemical information and modeling</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">49</span> <span class="fpage" tagx="fpage" title="fpage">2077</span> <span class="lpage" tagx="lpage" title="lpage">2081</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/ci900161g">10.1021/ci900161g</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19702240">19702240</a> </span> </span> </li> <li tag="ref"> <a name="CR87" /> <span class="label" tagx="label" title="label">87.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheng</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">In silico assessment of chemical biodegradability</span> <span class="source" tagx="source" title="source">Journal of chemical information and modeling</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">52</span> <span class="fpage" tagx="fpage" title="fpage">655</span> <span class="lpage" tagx="lpage" title="lpage">669</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/ci200622d">10.1021/ci200622d</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22332973">22332973</a> </span> </span> </li> <li tag="ref"> <a name="CR88" /> <span class="label" tagx="label" title="label">88.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Quantitative structure− activity relationship modeling of rat acute toxicity by oral exposure </span> <span class="source" tagx="source" title="source">Chemical research in toxicology</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">22</span> <span class="fpage" tagx="fpage" title="fpage">1913</span> <span class="lpage" tagx="lpage" title="lpage">1921</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/tx900189p">10.1021/tx900189p</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19845371">19845371</a> </span> </span> </li> <li tag="ref"> <a name="CR89" /> <span class="label" tagx="label" title="label">89.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marchese Robinson</span> <span class="given-names" tagx="given-names" title="given-names">RL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Glen</span> <span class="given-names" tagx="given-names" title="given-names">RC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mitchell</span> <span class="given-names" tagx="given-names" title="given-names">JB</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Development and comparison of hERG Blocker classifiers: assessment on different datasets yields markedly different results </span> <span class="source" tagx="source" title="source">Molecular informatics</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">30</span> <span class="fpage" tagx="fpage" title="fpage">443</span> <span class="lpage" tagx="lpage" title="lpage">458</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/minf.201000159">10.1002/minf.201000159</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27467090">27467090</a> </span> </span> </li> <li tag="ref"> <a name="CR90" /> <span class="label" tagx="label" title="label">90.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">ADMET evaluation in drug discovery. 12. Development of binary classification models for prediction of hERG potassium channel blockage </span> <span class="source" tagx="source" title="source">Molecular pharmaceutics</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">996</span> <span class="lpage" tagx="lpage" title="lpage">1010</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/mp300023x">10.1021/mp300023x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22380484">22380484</a> </span> </span> </li> <li tag="ref"> <a name="CR91" /> <span class="label" tagx="label" title="label">91.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bickerton</span> <span class="given-names" tagx="given-names" title="given-names">GR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paolini</span> <span class="given-names" tagx="given-names" title="given-names">GV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Besnard</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Muresan</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hopkins</span> <span class="given-names" tagx="given-names" title="given-names">AL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Quantifying the chemical beauty of drugs</span> <span class="source" tagx="source" title="source">Nature chemistry</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">90</span> <span class="lpage" tagx="lpage" title="lpage">98</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nchem.1243">10.1038/nchem.1243</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22270643">22270643</a> </span> </span> </li> <li tag="ref"> <a name="CR92" /> <span class="label" tagx="label" title="label">92.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Tanimoto, T. T. IBM Internal Report 17th Nov. (1957).</span> </li> <li tag="ref"> <a name="CR93" /> <span class="label" tagx="label" title="label">93.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bajusz</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Racz</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Heberger</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Why is Tanimoto index an appropriate choice for fingerprint-based similarity calculations?</span> <span class="source" tagx="source" title="source">J Cheminform</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">20</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s13321-015-0069-3">10.1186/s13321-015-0069-3</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26052348">26052348</a> </span> </span> </li> <li tag="ref"> <a name="CR94" /> <span class="label" tagx="label" title="label">94.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rogers</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hahn</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Extended-Connectivity Fingerprints</span> <span class="source" tagx="source" title="source">Journal of Chemical Information and Modeling</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">50</span> <span class="fpage" tagx="fpage" title="fpage">742</span> <span class="lpage" tagx="lpage" title="lpage">754</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/ci100050t">10.1021/ci100050t</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20426451">20426451</a> </span> </span> </li> <li tag="ref"> <a name="CR95" /> <span class="label" tagx="label" title="label">95.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morgan</span> <span class="given-names" tagx="given-names" title="given-names">HL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The Generation of a Unique Machine Description for Chemical Structures-A Technique Developed at Chemical Abstracts Service </span> <span class="source" tagx="source" title="source">Journal of Chemical Documentation</span> <span class="year" tagx="year" title="year">1965</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">107</span> <span class="lpage" tagx="lpage" title="lpage">113</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/c160017a018">10.1021/c160017a018</a> </span> </span> </li> <li tag="ref"> <a name="CR96" /> <span class="label" tagx="label" title="label">96.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Jasial, S., Hu, Y., Vogt, M. &amp;amp; Bajorath, J. Activity-relevant similarity values for fingerprints and implications for similarity searching. <i>F1000Research</i> <b> <i>5</i> </b>, <i>Chem Inf Sci-591</i> (2016). </span> </li> <li tag="ref"> <a name="CR97" /> <span class="label" tagx="label" title="label">97.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Franz</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Cytoscape.js: a graph theory library for visualisation and analysis</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">32</span> <span class="fpage" tagx="fpage" title="fpage">309</span> <span class="lpage" tagx="lpage" title="lpage">311</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26415722">26415722</a> </span> </span> </li> <li tag="ref"> <a name="CR98" /> <span class="label" tagx="label" title="label">98.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Clemons</span> <span class="given-names" tagx="given-names" title="given-names">PA</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Quantifying structure and performance diversity for sets of small molecules comprising small-molecule screening collections </span> <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">108</span> <span class="fpage" tagx="fpage" title="fpage">6817</span> <span class="lpage" tagx="lpage" title="lpage">6822</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.1015024108">10.1073/pnas.1015024108</a> </span> </span> </li> <li tag="ref"> <a name="CR99" /> <span class="label" tagx="label" title="label">99.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ahmad</span> <span class="given-names" tagx="given-names" title="given-names">VU</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A New Ketone and a Known Anticancer Triterpenoid from the Leaves of Onosma limitaneum</span> <span class="source" tagx="source" title="source">Helvetica Chimica Acta</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">88</span> <span class="fpage" tagx="fpage" title="fpage">309</span> <span class="lpage" tagx="lpage" title="lpage">311</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/hlca.200590013">10.1002/hlca.200590013</a> </span> </span> </li> <li tag="ref"> <a name="CR100" /> <span class="label" tagx="label" title="label">100.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kishore</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Onosma L.: A review of phytochemistry and ethnopharmacology</span> <span class="source" tagx="source" title="source">Pharmacogn Rev</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">140</span> <span class="lpage" tagx="lpage" title="lpage">151</span> <span class="pub-id"> <a href="https://dx.doi.org/10.4103/0973-7847.120513">10.4103/0973-7847.120513</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24347922">24347922</a> </span> </span> </li> <li tag="ref"> <a name="CR101" /> <span class="label" tagx="label" title="label">101.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marson</span> <span class="given-names" tagx="given-names" title="given-names">CM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">New and unusual scaffolds in medicinal chemistry</span> <span class="source" tagx="source" title="source">Chem Soc Rev</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">40</span> <span class="fpage" tagx="fpage" title="fpage">5514</span> <span class="lpage" tagx="lpage" title="lpage">5533</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1039/c1cs15119c">10.1039/c1cs15119c</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21837344">21837344</a> </span> </span> </li> <li tag="ref"> <a name="CR102" /> <span class="label" tagx="label" title="label">102.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rupp</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">From machine learning to natural product derivatives that selectively activate transcription factor PPARgamma </span> <span class="source" tagx="source" title="source">ChemMedChem</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">191</span> <span class="lpage" tagx="lpage" title="lpage">194</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/cmdc.200900469">10.1002/cmdc.200900469</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20043315">20043315</a> </span> </span> </li> <li tag="ref"> <a name="CR103" /> <span class="label" tagx="label" title="label">103.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sokolova</span> <span class="given-names" tagx="given-names" title="given-names">AP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alla</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ardashov</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shernyukov</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gatilov</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yarovaya</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tolstikova</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salakhutdinov</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Synthesis and analgesic activity of new α-truxillic acid derivatives with monoterpenoid fragments </span> <span class="source" tagx="source" title="source">Medicinal Chemistry Research</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">1608</span> <span class="lpage" tagx="lpage" title="lpage">1615</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00044-016-1593-z">10.1007/s00044-016-1593-z</a> </span> </span> </li> <li tag="ref"> <a name="CR104" /> <span class="label" tagx="label" title="label">104.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Verpoorte</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Exploration of nature’s chemodiversity: the role of secondary metabolites as leads in drug development </span> <span class="source" tagx="source" title="source">Drug Discovery Today</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="fpage" tagx="fpage" title="fpage">232</span> <span class="lpage" tagx="lpage" title="lpage">238</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S1359-6446(97)01167-7">10.1016/S1359-6446(97)01167-7</a> </span> </span> </li> <li tag="ref"> <a name="CR105" /> <span class="label" tagx="label" title="label">105.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Conway, J. R., Lex, A. &amp;amp; Gehlenborg, N. UpSetR: an R package for the visualization of intersecting sets and their properties. <i>Bioinformatics</i> <b>33</b>, 2938-2940 (2017). </span> </li> </ul> </div> </div>  </body></html>