<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Sci Rep</span> <span class="iso-abbrev" title="iso-abbrev">Sci Rep</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Scientific Reports</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">2045-2322</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5824918">5824918</a> </span> <span class="publisher-id" title="publisher-id">17153</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41598-017-17153-z">10.1038/s41598-017-17153-z</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
HMG-CoA reductase from Camphor Tulsi ( <i>Ocimum kilimandscharicum</i>) regulated MVA dependent biosynthesis of diverse terpenoids in homologous and heterologous plant systems </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bansal</span> <span class="given-names" tagx="given-names" title="given-names">Shilpi</span> </span> <a href="#Aff1">1</a> <a href="#Aff2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Narnoliya</span> <span class="given-names" tagx="given-names" title="given-names">Lokesh Kumar</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mishra</span> <span class="given-names" tagx="given-names" title="given-names">Bhawana</span> </span> <a href="#Aff1">1</a> <a href="#Aff2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chandra</span> <span class="given-names" tagx="given-names" title="given-names">Muktesh</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yadav</span> <span class="given-names" tagx="given-names" title="given-names">Ritesh Kumar</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sangwan</span> <span class="given-names" tagx="given-names" title="given-names">Neelam Singh</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">nsangwan5@gmail.com</span> </span> <a href="#Aff1">1</a> <a href="#Aff2">2</a> </span> <span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0001 2299 2571</span> <span class="institution-id" title="institution-id">grid.417631.6</span> <span class="institution" title="institution">Department of Metabolic and Structural Biology, CSIR-Central Institute of Medicinal and Aromatic Plants, </span> </div> </span> <span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">grid.469887.c</span> <span class="institution" title="institution">Academy of Scientific and Innovative Research (AcSIR), </span> </div> </span> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-2-2</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2018-2-2</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2018</span> </span> <span class="volume" tagx="volume" title="volume">8</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">3547</span> <span class="history" title="history"> <span class="received" title="received">received: 2017-4-11</span> <span class="accepted" title="accepted">accepted: 2017-11-2</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , </span> <span class="license" title="license"> <span class="license-p" title="license-p"> <b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p> <i>Ocimum kilimandscharicum</i> is unique in possessing terpenoids whereas other <i>Ocimum</i> species are renowned for phenylpropanoids as major constituents of essential oil. The key enzyme of MVA/terpenoid metabolic pathway <i>viz</i> 3-hydroxy-3-methylglutaryl Co-A reductase ( <i>OkHMGR</i>) of 1.7-Kb ORF encoding ~60-kDa protein was cloned from <i>O</i>. <i>kilimandscharicum</i> and its kinetic characteristics revealed the availability of HMG-CoA as a control point of MVA-pathway. Transcript profiling of the <i>OkHMGR</i> elucidated tissue-specific functions of the gene in flower and leaf tissues in accumulation of terpenoidal essential oil. <i>OkHMGR</i> was differentially regulated in response to exposure to methyl-jasmonate, salicylic-acid, and stress conditions such-as salt and temperature stress, demonstrating its key role in managing signaling and stress-responses. To elucidate its functional role, <i>OkHMGR</i> was transiently over-expressed in homologous and heterologous plants such as <i>O</i>. <i>sanctum</i>, <i>O</i>. <i>basilicum</i>, <i>O</i>. <i>gratissimum</i>, <i>Withania somnifera</i> and <i>Artemisia annua</i>. The over-expression and inhibition dual strategy revealed that the additional <i>OkHMGR in-planta</i> could afford endogenous flow of isoprenoid units towards synthesis of terpenoids. The present study provides in-depth insight of <i>OkHMGR</i> in regulation of biosynthesis of non-plastidal isoprenoids. This is first report on any gene of MVA/isoprenoid pathway from under-explored Camphor Tulsi belonging to genus Ocimum. Studies also suggested that <i>OkHMGR</i> could be a potential tool for attempting metabolic engineering for enhancing medicinally important terpenoidal metabolites in plants. </p> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2018" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p> <i>Ocimum</i> (Tulsi), an important member of Lamiaceae family comprises of about 150 species that are the inhabitants of tropical and warm temperate regions of Asia, Central and South Africa <sup> <a href="#CR1">1</a> </sup>. The Tulsi has been used as a source of medicinal preparations since ages, mentioned in various ancient literatures such as Charak Samhita, Rigveda etc. In Sanskrit the meaning of Tulsi is "one that is incomparable". It is classified as a "Rasayana," the herb which improves the health. The prominently used species of genus Ocimum are <i>O</i>. <i>basilicum</i>, <i>O</i>. <i>sanctum</i>, whereas <i>O</i>. <i>gratissimum</i> and <i>O</i>. <i>kilimandscharicum</i> are less explored <sup> <a href="#CR2">2</a> </sup>. <i>O</i>. <i>kilimandscharicum</i> is known for camphor as major constituent in its essential oil and is a perennial herb, native of East Africa and is also found in India, Turkey and Thailand <sup> <a href="#CR3">3</a>, <a href="#CR4">4</a> </sup>. The plant is rich in aromatic essential oil that makes it economically and medically important. <i>O</i>. <i>kilimandscharicum</i> contains pool of various secondary metabolites with a major proportion of monoterpenoids (≥ 90.0%) out of which oxygenated monoterpenoids may constitute upto 70% while hydrocarbons may constitute about upto 25% <sup> <a href="#CR5">5</a> </sup>. It has been reported that the essential oil content in Camphor Tulsi varied between 0.70–2.0% on dry weight basis <sup> <a href="#CR6">6</a> </sup>. Seed and leaf essential oil of <i>O</i>. <i>kilimandscharicum</i> also contains minor proportions of other monoterpenoids such as (α-pinene, camphene, β-myrcene along with sesquiterpenoids s such as trans-caryophyllene, germarcrene‐D <sup> <a href="#CR4">4</a> </sup>. Presence of a wide range of these secondary metabolites makes <i>O</i>. <i>kilimandscharicum</i> a valuable material to be used as medicinal bio-resource <sup> <a href="#CR7">7</a> </sup>. Phytomolecules such as methyl chavicol, 1,8-cineole, eugenol, (E)-bisabolene, terpineol, linalool, (Z)-cinnamic acid methyl ester, camphor etc are reported to possess various antimicrobial, antispasmodic, bactericidal, carminative, anti-helminthic, hepatoprotective, antiviral and larvicidal activities <sup> <a href="#CR6">6</a>– <a href="#CR8">8</a> </sup>. Camphor is the most distinct compound of the plant which may serve various medicinal and industrial purposes <sup> <a href="#CR5">5</a>– <a href="#CR8">8</a> </sup>. The terpenoid metabolites are produced by the plant through MEP (2C-methyl-D-erythritol 4-phosphate) and MVA (mevalonic acid) pathway in plastids and cytosol, respectively <sup> <a href="#CR9">9</a> </sup>. Both the pathways are actively involved in the biosynthesis of isoprenoid moieties operative from their respective compartments <sup> <a href="#CR10">10</a>, <a href="#CR11">11</a> </sup>. Mono, di-, tetra- terpenoids are mainly biosynthesized by MEP pathway whereas MVA pathway is reported for the biosynthesis of sesquiterpenoids and triterpenoids <sup> <a href="#CR10">10</a>, <a href="#CR11">11</a> </sup>. Our earlier studies on <i>W</i>. <i>somnifera</i> showed that the mevalonate pathway contributes dominantly to the synthesis of IPP than non-mevalonate pathway and HMG-CoA to mevalonic acid <i>via</i> HMGR enzyme is the rate-determining step <sup> <a href="#CR12">12</a>, <a href="#CR13">13</a> </sup>. Undoubtedly, HMGR gene governs the MVA pathway derived isoprenoids and plays a key role in isoprenoid biosynthesis. HMGR gene has been isolated from number of plant species including <i>W</i>. <i>somnifera</i>, <i>S</i>. <i>miltiorrhiza</i>, <i>A. indica</i> and other plants <sup> <a href="#CR12">12</a>– <a href="#CR20">20</a> </sup>. Different isoforms of HMGR have also been reported in cotton and tomato, which might have different roles to play under the influence of different environmental and developmental conditions <sup> <a href="#CR15">15</a>, <a href="#CR16">16</a> </sup>. HMGR is responsible for regulating the production of diverse terpenoids virtually in all plants and holds central regulatory position in terpenoid bearing plants (Fig. <a href="#Fig1">1</a>). Considering the key dominance of HMGR in terpenoidal essential oil biosynthesis, in the present study, we report the cloning and characterization of HMGR from <i>O</i>. <i>kilimandscharicum</i>, functional expression in <i>E</i>. <i>coli</i> and plants along with its association with secondary metabolite biosynthesis. To the best of our knowledge, this is the first report of kinetic and <i>in planta</i> study of HMGR from <i>Ocimum</i> species. The study provides a novel <i>OkHMGR</i> which can be used as a powerful sequence to drive MVA led, isoprenoids synthesis in medicinally and aromatically important plants. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <p>Schematic diagram of terpenoid biosynthetic pathway through mevalonate (MVA) as well as methyl-erythritol-phosphate (MEP) pathway. </p>   </div> <p /> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="cloningofokhmgr" title="sec"> <div class="title" tagx="title" title="title">
Cloning of <i>OkHMGR</i> </div> <p>A 500 bp amplicon was obtained from <i>O</i>. <i>kilimandscharicum</i> leaf tissues using degenerate primers (Supplementary Fig. <a href="#MOESM1">1A</a>). This partial amplicon was used for designing RACE primers. PCR amplification with 5′ RACE and 3′ RACE primers gave amplicon of 700 bp and 1000 bp respectively (Supplementary Fig. <a href="#MOESM1">1B,C</a>). Assembly of all these partial and RACE amplified products resulted in a full length <i>OkHMGR</i>. Once the full length sequence was obtained gene specific full length primers were designed by incorporating restriction enzymes <i>Sal</i>I and <i>Hind</i>III at 5′ and 3′ terminus. The ORF search confirmed 1698 bp coding region encoding a polypeptide of 565 amino acids. The theoretical pI of the protein is 7.10 and molecular weight is 60.53 kDa (Supplementary Table <a href="#MOESM1">1</a>). </p> </div> <div class="sequenceretrievalandphylogeneticstudies" title="sec"> <div class="title" tagx="title" title="title">
Sequence retrieval and phylogenetic studies</div> <p>Full length <i>OkHMGR</i> exhibited maximum similarity with HMGR of <i>Salvia</i> <i>miltiorrhiza</i> (94%) followed by <i>Andrographis paniculata</i> (86%), <i>Picrorhiza kurrooa</i> (86%), <i>Panax ginseng</i> (83%), <i>W. somnifera</i> (78%), <i>Arabidopsis thaliana</i> (78%) and <i>Nicotiana tabacum</i> (75%). Plants HMGR possess HMG-CoA binding motifs (EMPIGYVQIP) &amp;amp; (TTEGCLVA) and NADPH-binding motifs (DAMGMNM) &amp;amp; (GTVGGGT) which were also found in the OkHMGR (Fig. <a href="#Fig2">2A</a>). Likewise motifs (GQDPAQN) &amp;amp; (VLAGELS) were found to be highly conserved and appear to be representative motifs for plant HMGR as these were found to be missing in human HMGRs (Fig. <a href="#Fig2">2A</a>). Multiple sequence alignment analysis of 108 sequences of 55 different plant species coding for HMGR retrieved from database and their phylogeny along with the sequence from <i>O</i>. <i>kilimandscharicum</i> (OkHMGR) exhibits closeness to <i>S</i>. <i>miltiorrhiza</i> and the functional motifs were very similar to other plant HMGRs (Fig. <a href="#Fig2">2B</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <p>Multiple sequence alignment and phylogenetic analysis of OkHMGR. ( <b>A</b>) Multiple sequence alignment of OkHMGR and visualization of different motifs. Three sets of motifs, each having two motifs were detected. NADP(H) binding motifs (green), HMG-CoA binding motifs (red), and plant specific motifs(blue). The consensus sequences of NADPH binding motif are [DAMGMNM] &amp;amp; [GTVGGGT], HMG-CoA binding motifs are [EMPVGYVQIP] &amp;amp; [TTEGCLVA] and plant specific motifs are [GQDPAQN] &amp;amp; [VLAGELS] ( <b>B</b>) Phylogenetic analysis of 108 plant HMGRs from 55 different species analysed by neighbor joining approach, with 1000 bootstrap replicates. </p>   </div> <p /> <p>Secondary structure prediction study revealed presence of 20 alpha helices, 12 β-strands intervened by a number of coils (Supplementary Fig. <a href="#MOESM1">2</a>). Target P1.1 server observed that OkHMGR is a cytosolic protein and theoretical pI determined was 7.10. The incidence of both aspartic acid and glutamic acid (negatively charged amino acids) and arginine and lysine (positively charged amino acid) were 53 each. The instability index and aliphatic index were 43.45 and 95.46 respectively (Supplementary Table <a href="#MOESM1">1</a>). These features are key determinant of regulating catalytic activities of HMGR protein. </p> </div> <div class="heterologousexpression,purificationandkineticcharacterizationofokhmgr" title="sec"> <div class="title" tagx="title" title="title">
Heterologous expression, purification and kinetic characterization of OkHMGR</div> <p>Full length <i>OkHMGR</i> (approximately 1.7 kb; Fig. <a href="#Fig3">3A</a>) was cloned in pET28a expression vector. Recombinant OkHMGR (60 kDa) protein (Fig. <a href="#Fig3">3B</a>) was biochemically characterized for various catalytic and kinetic properties. OkHMGR activity was found maximum in potassium phosphate buffer followed by sodium phosphate and Tris buffer (Supplementary Fig. <a href="#MOESM1">3</a>). OkHMGR activity increased with the increase in pH from 5.0 to 7.0 with maximum activity at about 7.0, thereafter a drastic reduction in activity was noticed (Supplementary Fig. <a href="#MOESM1">4</a>). The OkHMGR enzyme showed a hyperbolic curve both for substrate and cofactor as revealed by substrate saturation studies. Various cations (Mn <sup>2+</sup>, Li <sup>2+</sup>, K <sup>+</sup>, Na <sup>+</sup>, Ca <sup>2+</sup> and Mg <sup>2+</sup>) introduced in the enzyme assay influenced the catalytic activity of the enzyme. It was observed that addition of K <sup>+</sup> increased the activity of enzyme (Supplementary Fig. <a href="#MOESM1">5</a>). Using Lineweaver-Burk plot, Michaelis constant (K <sub>m</sub>) and the maximum reaction velocity (V <sub>max</sub>) were determined (Fig. <a href="#Fig3">3C,D</a>). The K <sub>m</sub>, V <sub>max</sub>, K <sub>cat</sub> and K <sub>cat</sub>/K <sub>m</sub> values for HMG-CoA were 0.088 mM, 0.909µmole min <sup>−1</sup> mg <sup>−1</sup> enzyme protein, 31.13 s <sup>−1</sup> and 3.53 × 10 <sup>5</sup> M <sup>−1</sup> s <sup>−1</sup> respectively while for NADPH values were 0.903 mM (through LB plot), 0.425µmole min <sup>−1</sup> mg <sup>−1</sup> enzyme protein, 14.53 s <sup>−1</sup> and 1.6 × 10 <sup>4</sup> M <sup>−1</sup> s <sup>−1</sup> respectively. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <p>Molecular and biochemical analysis of <i>OkHMGR</i> gene and protein. ( <b>A</b>) PCR amplification product (1.7Kb) amplified with gene specific primers. (M-1Kb ladder, OK-amplified product), ( <b>B</b>) SDS-PAGE analysis of His-tagged recombinant OkHMGR (lane 1: protein molecular weight marker; lane 1–4 OkHMGR enzyme), ( <b>C</b>) Substrate saturation curves for HMG-CoA and ( <b>D</b>) Substrate saturation curves for NADPH. </p>   </div> <p /> </div> <div class="modulationofokhmgr" title="sec"> <div class="title" tagx="title" title="title">
Modulation of <i>OkHMGR</i> </div> <p>The abundance of <i>OkHMGR</i> transcripts in <i>O</i>. <i>kilimandscharicum</i> was found to be highest in the flower followed by other tissues (Fig. <a href="#Fig4">4A</a>). <i>OkHMGR</i> was found to be regulated by methyl jasmonate (MeJA) and salicylic acid (SA) as exhibited by the exogenous supplementation studies. A substantial increase in transcript abundance was observed with MeJA (Fig. <a href="#Fig4">4B</a>), and with SA (Fig. <a href="#Fig4">4C</a>). Whereas, mechanical injury of plants, also modulated the <i>OkHMGR</i> expression by 1.6 fold (Fig. <a href="#Fig4">4D</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <p>Expression profiles of <i>OkHMGR</i> in different tissues of plant as well as under the influence of elicitors and various stress conditions. Relative expression profiling of <i>OkHMGR</i> ( <b>A</b>) in different tissues of <i>O</i>. <i>kilimandscharicum</i>,(YL-Young leaf, MML-Middle mature leaf, FML-Fully mature leaf, ST-Stem, BD-Bud, FL-Flower), ( <b>B</b>) in response to different strength of methyl jasmonate (0, 10, 50, 100 and 500 µM), ( <b>C</b>) in response to different strength of salicylic acid (0, 10, 50, 100 and 500 µM), ( <b>D</b>) on wounding (0, 0.5, 3 and 24 hrs), ( <b>E</b>) under the influence of gibberellic acid (0, 1, 10 and 50 μM), ( <b>F</b>) after NaCl treatments (0, 50, 100 and 250 mM), ( <b>G</b>) under heat stress for different time period (0, 0.5, 1, 2 and 4 hrs), ( <b>H</b>) and after exposure to cold stress (0, 0.5, 1, 2 and 4 hrs). </p>   </div> <p /> <p>Similarly, the presence of GA <sub>3</sub> (gibberellic acid) hormone for 3 h, elevated the expression of <i>OkHMGR</i> transcripts (Fig. <a href="#Fig4">4E</a>). In addition, salt stress showed substantial increase in the expression of <i>OkHMGR</i>. After 3 h of treatment, transcript levels increased by 4 to 24 folds at 50 and 100 mM, with a decrease to 20 folds at 250 mM concentration (Fig. <a href="#Fig4">4F</a>). Interestingly, exposure to heat stress at 55 °C for different time intervals resulted in the substantial up-regulation of gene (Fig. <a href="#Fig4">4G</a>). Likewise, the cold stress treatment showed that the enzyme is under regulation by cold condition (Fig. <a href="#Fig4">4H</a>). </p> </div> <div class="geneexpressionandsecondarymetabolitecontentindifferentocimumspecies" title="sec"> <div class="title" tagx="title" title="title">
Gene expression and secondary metabolite content in different <i>Ocimum</i> species </div> <p>We observed that the transient over-expression of <i>OkHMGR</i> in different species of <i>Ocimum</i> namely; <i>O</i>. <i>sanctum</i>, <i>O</i>. <i>basilicum</i> and <i>O</i>. <i>gratissimum</i> resulted in increased transcript abundance of <i>OkHMGR</i>, with about 2.1, 1.8 and 3 folds respectively, compared to their respective controls (Fig. <a href="#Fig5">5A</a>). The values of RQ were correlated well with total essential oil content, isolated from transformed and control plants (Fig. <a href="#Fig5">5B</a>). Increased essential oil content in transiently transformed tissues suggests that <i>HMGR</i> may have a direct role in essential oil biosynthesis. In <i>O</i>. <i>sanctum</i> remarkable increase in quantity of sesquiterpenoids such as beta-elemene (7.4%), caryophyllene (37.87%) and germacrene D (19.1%) were observed (Fig. <a href="#Fig5">5C</a>). Interestingly, another species <i>O</i>. <i>gratissimum</i> showed tremendous increase in both sesquiterpenoids and monoterpenoid constituents. The compounds with increased content were beta-ocimene, germacrene A, caryophyllene, and germacrene D (Fig. <a href="#Fig5">5D</a>), whereas in <i>O</i>. <i>basilicum</i> limonene was detected as major monoterpenoid constituent (Fig. <a href="#Fig5">5E</a>). In <i>O</i>. <i>kilimandscharicum</i>, transformation of the plant with <i>OkHMGR</i>, triggered the expression of <i>HMGR</i> and thereby the production of terpenoidal essential oil was also enhanced. Transcript abundance of <i>OkHMGR</i> in over-expressed lines displayed about 5.36 folds increment when compared with untransformed control (Fig. <a href="#Fig5">5A,B</a>). When the individual constituents of essential oil were analysed by GC, marked increase in sesquiterpenoids and some of the monoterpenoids was observed. Sesquiterpenoids which were found increased after transformation event were germacrene A (75%), β-caryophyllene (18.43%), germacrene D (9.71%) (Fig. <a href="#Fig5">5F</a>). The mRNA transcript levels and essential oil content on comparison with controls (untransformed and vector control) lines did not show significant deviation from the results obtained, i.e both the controls showed almost similar response (Fig. <a href="#Fig5">5</a>) thereby clearly demonstrating that increase in gene expression and secondary metabolite content was due to <i>OkHMGR</i> over-expression. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 5</span> </a> </div>   <p>Relative expression profiles of <i>OkHMGR</i> transcript and secondary metabolite content in control, vector control and over-expressed lines of different species of <i>Ocimum</i> (OS: <i>O</i>. <i>sanctum</i>, OG: <i>O</i>. <i>gratissimum</i>, OB: <i>O</i>. <i>basilicum</i>, OK: <i>O</i>. <i>kilimandscharicum</i>). ( <b>A</b>) <i>OkHMGR</i> mRNA transcript levels, ( <b>B</b>) Total essential oil content (mgg <sup>−1</sup>) in above mentioned four species of <i>Ocimum</i>. Change in essential oil constituent’s percentage (%) in ( <b>C</b>) <i>O</i>. <i>sanctum</i>, ( <b>D</b>) <i>O</i>. <i>gratissimum</i>, ( <b>E</b>) <i>O</i>. <i>basilicum</i> and ( <b>F</b>) <i>O</i>. <i>kilimandscharicum</i>. ELE- β-elemene, CAR- caryophyllene, GER- germacrene D, OCE-β-ocimene, GER A- germacrene A, LIM- limonene, PIN- α-pinene, CAM- camphene, MYR- myrecrene, TER- gamma-terpinene, SAB- cis-sabinene hydrate. Control represents the untransformed tissues, vector control represents the tissues transformed with pCAMBIA1303 vector and HMGRox represents tissues transformed with <i>OkHMGR</i>. </p>   </div> <p /> </div> <div class="geneexpressionandsecondarymetabolitecontentinartemisiaannuaandwithaniasomnifera" title="sec"> <div class="title" tagx="title" title="title">
Gene expression and secondary metabolite content in <i>Artemisia annua</i> and <i>Withania somnifera</i> </div> <p> <i>A. annua</i> was selected as it contains important sesquiterpenoidal artemisinin as well as essential oil. Expression study revealed an increase of 2.4 folds in transformed tissues, than both untransformed control and vector control (Fig. <a href="#Fig6">6A</a>). In correlation to RT-PCR, total essential oil content was also high with a concentration of 0.64 mg/g which is 0.10 mg/g more than that of untransformed control and 0.08 mg/g more than vector control (Fig. <a href="#Fig6">6B</a>). HPLC analysis of untransformed control and over-expressed tissues showed an increase in artemisinin content by 7.4% (Fig. <a href="#Fig6">6C</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 6</span> </a> </div>   <p>Expression profiles of <i>OkHMGR</i> transcript and secondary metabolite content in control and over-expressed lines of <i>A</i>. <i>annua</i> ( <b>A</b>) <i>OkHMGR</i> mRNA transcript levels, ( <b>B</b>) total essential oil content (mg/g FW g <sup>−1</sup>), ( <b>C</b>) artemisinin content (µg g <sup>−1</sup>); and <i>W</i>. <i>somnifera</i> ( <b>D</b>) mRNA transcript levels in leaf and berry, ( <b>E</b>) total withanolide content (%), ( <b>F</b>) total sterol content in leaf and berries (µg g <sup>−1</sup>). Control represents the untransformed tissues, vector control represents the tissues transformed with pCAMBIA1303 vector and <i>HMGR</i> ox represents tissues transformed with <i>OkHMGR</i>. </p>   </div> <p /> <p>In order to unravel the role of HMGR in synthesis of more complex higher phytomolecules of triterpenoid ancestry, <i>W</i>. <i>somnifera</i> was selected being rich in withanolides and sterols. Consistent with the results noted in <i>O</i>. <i>kilimandscharicum</i> and <i>A</i>. <i>annua</i>, the expression levels of <i>HMGR</i> was higher in over-expressed <i>W</i>. <i>somnifera</i> tissues than control tissues by 3.5 and 2.8 folds in leaves and berry respectively (Fig. <a href="#Fig6">6D</a>). To understand the linkage between <i>HMGR</i> expression and secondary metabolite biosynthesis, withanolides and sterols were extracted from the over-expressed as well as control tissues of the plant. Over-expression of <i>HMGR</i> is likely to drive the MVA isoprenoid biosynthesis in withanolides and sterols pathway; however we observed that the responses were tissue dependent. In both the tissues, over-expression led to increase in total withanolide content by upto 44% with a minor increase in total sterol content i.e. 0.5% (Fig. <a href="#Fig6">6E, F</a>). Transformation with vector control did not produce significant alterations in gene expression levels and withanolide and sterol content. </p> </div> <div class="totalcarotenoidcontentinocimumspecies,a.annuaandw.somnifera" title="sec"> <div class="title" tagx="title" title="title">
Total carotenoid content in Ocimum species, <i>A. annua</i> and <i>W. somnifera</i> </div> <p>To determine the effect of <i>HMGR</i> on carotenoids, the leaves of the plants were transiently transformed with <i>OkHMGR</i>. It was observed that similar to other metabolites of terpenoid pathway, carotenoids contents were increased after over-expression suggesting role of HMGR. The increase in carotenoid content was in the range of 3–11% in different <i>Ocimum</i> species. Increase in carotenoids was more in other plants in comparison to <i>Ocimum</i> species such as <i>A</i>. <i>annua</i> where upregulation was highest among all the plants studied, followed by <i>W</i>. <i>somnifera</i> leaf, with 33% increment. This may be because in <i>Ocimum</i> species major flux is diverted for the synthesis of monoterpenes and sesquiterpenes rather than carotenoids (Supplementary Fig. <a href="#MOESM1">6</a>). </p> </div> <div class="effectofinhibitorsongeneexpressionandsecondarymetabolitecontent" title="sec"> <div class="title" tagx="title" title="title">
Effect of inhibitors on gene expression and secondary metabolite content</div> <p>Inhibitors mevastatin and pravastatin were used to evaluate the blocking of the pathway and its impact on transformed and untransformed tissues. In <i>O</i>. <i>kilimandscharicum</i>, addition of statins resulted in down regulation of gene expression levels and total essential oil content in comparison to control untreated tissues (Fig. <a href="#Fig7">7A,B</a>). Treatment of over-expressed lines with these inhibitors showed dual action in the way that though addition of statins lowered the gene expression from 5.3 folds to 2.8–3.3 folds and total essential oil content by 18–25% in comparison to untreated over-expressed lines, but they still retained high expression to an extent, as the transcript abundance and total essential oil content was higher in over-expressed tissues than that of treated control (Fig. <a href="#Fig7">7A,B</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 7</span> </a> </div>   <p>Analysis of secondary metabolites content and <i>HMGR</i> transcript expression levels in transiently transformed tissues of <i>O</i>. <i>kilimandscharicum</i> in presence and absence of inhibitors. ( <b>A</b>) mRNA transcript levels and ( <b>B</b>) Total essential oil content (μg g <sup>−1</sup>) in <i>O</i>. <i>kilimandscharicum</i>. Control- untransformed tissue without inhibitor, Vector control- pCAMBIA1303 vector transformed tissue without inhibitor, <i>HMGR</i> ox- transformed tissue without inhibitor treatment, Control + P- untransformed tissue with pravastatin treatment, Vector control + P- vector transformed tissue with pravastatin treatment, <i>HMGR</i> ox + P- transformed tissue with pravastatin treatment, Control + M- untransformed tissue with mevastatin treatment, Vector control + M- vector transformed tissue with mevastatin treatment and <i>HMGR</i> ox + M- transformed tissue with mevastatin treatment. </p>   </div> <p /> <p>Similar pattern was observed in <i>A</i>. <i>annua</i>, where inhibitor treatment lowered the relative expression to 0.3 and 0.4 fold for pravastatin and mevastatin treated and control lines, respectively (Fig. <a href="#Fig8">8A</a>) and a reduction in total essential oil content and artemisinin was also observed (Fig. <a href="#Fig8">8B,C</a>). Comparative analysis of treated transformed tissues with transformed control tissues, showed a decrease in mRNA transcript levels, total essential oil content as well as artemisinin. Compared to non-transgenic control, inhibitor treated over expressed lines retained their over-expression partly when gene transcript levels were determined which was found to be 1.8 folds higher than that of untransformed control. However, the secondary metabolites i.e. essential oil and artemisinin content declined, revealing the alterations in artemisinin were more prominent indicating preference of MVA pathway participation in biosynthesis (Fig. <a href="#Fig8">8A,B,C</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 8</span> </a> </div>   <p> <i>OkHMGR</i> transcript expression and secondary metabolite analysis in transiently transformed tissues of <i>A</i>. <i>annua</i> and <i>W</i>. <i>somnifera</i> with and without inhibitors. ( <b>A</b>) <i>HMGR</i> transcript levels, ( <b>B</b>) Total essential oil content (mg g <sup>−1</sup>) and ( <b>C</b>) Artemisinin content (µg g <sup>−1</sup>) in <i>A</i>. <i>annua</i>; ( <b>D</b>) <i>HMGR</i> transcripts in tissues of <i>W</i>. <i>somnifera</i>, ( <b>E</b>) total withanolide content (%) and ( <b>F</b>) total sterol content (µg g <sup>−1</sup>). Control- untransformed tissue without inhibitor, Vector control- pCAMBIA1303 vector transformed tissue without inhibitor treament, <i>HMGR</i> ox- transformed tissue without inhibitor treatment, Control + P- untransformed tissue with pravastatin treatment, Vector control + P- vector transformed tissue with pravastatin treatment, <i>HMGR</i> ox + P- transformed tissue with pravastatin treatment, Control + M- untransformed tissue with mevastatin treatment, Vector control + M- vector transformed tissue with mevastatin treatment and <i>HMGR</i> ox + M- transformed tissue with mevastatin treatment. </p>   </div> <p /> <p> <i>W</i>. <i>somnifera</i> was also influenced by the inhibitory action of inhibitors. Pravastatin and mevastatin treatment reduced the gene expression to 0.3–0.6 folds in untransformed treated control tissues. Overall decrease in both withanolides and sterol was observed in both tissues of <i>W</i>. <i>somnifera</i> after inhibitor treatment. In case of control leaves, there was prominent (about 96%) reduction in total withanolide content and 39–43% reduction in total sterol content compared to untreated untransformed tissues. Over-expressed lines treated with inhibitors showed lowering in gene expression levels, withanolide and sterol content to varying extents in comparison to untreated but transformed tissues (Fig. <a href="#Fig8">8D,E,F</a>). </p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Plants are rich source of secondary metabolites exhibiting important medicinal properties. Secondary metabolites of terpenoid origin are synthesized by mevalonate and non-mevalonate pathways and many studies have shown their involvement in the biosynthetic pathway <sup> <a href="#CR17">17</a>– <a href="#CR20">20</a> </sup>. Present study focusses on conversion of 3-hydroxy-3 methyl glutaryl Co-A into mevalonic acid by <i>OkHMGR</i> into an array of terpenoidal secondary metabolites such as monoterpenes, sesquiterpenes, triterpenoids and other isoprenoid compounds. These secondary metabolites are distinct in terms of number of carbon atoms, structure, functions, their targets and mode of action <sup> <a href="#CR21">21</a> </sup>. <i>HMGR</i> has been cloned and characterized in a number of plants such as <i>W</i>. <i>somnifera</i> <sup> <a href="#CR12">12</a> </sup>, <i>A</i>. <i>thaliana</i> <sup> <a href="#CR17">17</a> </sup>, <i>Andrographis paniculata</i> <sup> <a href="#CR19">19</a> </sup> etc, however no reports are available for its cloning, kinetic and functional characterization in any of the <i>Ocimum</i> species. Realizing its importance and lack of knowledge about its role in <i>Ocimum</i> species, we isolated <i>HMGR</i> from <i>O</i>. <i>kilimandscharicum</i>, the Camphor Tulsi (the least explored species of <i>Ocimum</i>). Among various species, we primarily focused on <i>O</i>. <i>kilimandscharicum</i> as it contains more terpenoids than any other <i>Ocimum</i> species, which are known for the presence of phenylpropanoids. The isolated OkHMGR with a molecular mass of 60 kDa (Supplementary Table <a href="#MOESM1">1</a>) showed evolutionarily relatedness to <i>S</i>. <i>miltiorrhiza</i> with highest similarity. The characterstic motifs such as HMG-CoA binding motif (TTEGCLVA and EMPVGYVQIP) and NADPH binding motifs (DAMGMNM and GTVGGGT) in OkHMGR sequence revealed that OkHMGR shares the conserved motifs with other plant species (Fig. <a href="#Fig2">2A</a>). </p> <p>HMGRs have been broadly classified under two categories: class I and class II. Class I are found in eukaryotes and, are membrane bound while class II in prokaryotes and are soluble in nature <sup> <a href="#CR22">22</a> </sup>. OkHMGR consisted of two transmembrane domains at the N-terminal region and these transmembrane regions are believed to be responsible for anchoring the protein to the endoplasmic reticulum <sup> <a href="#CR23">23</a> </sup>. The N terminal region of OkHMGR is highly diverse while C terminal is conserved; these findings are in accordance to the other plant species <i>viz</i>. <i>Corylus avellana</i>, <i>Centella asiatica</i> <sup> <a href="#CR24">24</a>, <a href="#CR25">25</a> </sup>. HMGR protein exists as a tetramer where four monomers are arranged into two set of dimers, each dimer contains two active sites formed by residues from both monomers. Each monomer is further divided into three domains: N, L and S-domain. N domain is small consisting of amino helix and L-domain is unique for binding of HMG CoA. This domain is central and large consisting of regular expression of (TTEGCLVA) and (EMPVGYVQIP) while the S-domain (DAMGMNM), forms the binding site for NADPH <sup> <a href="#CR9">9</a> </sup>. </p> <p>Biochemical characterization of the HMGR through its catalytic properties provided a detailed account of the kinetic nature of the protein as revealed by K <sub>m</sub>, V <sub>max</sub>, optimum pH, optimum temperature etc. Variation in the catalytic activity due to change in pH is a consequence in pattern of ionic bonding between the substrate and enzyme under acidic or alkaline conditions (Supplementary Fig. <a href="#MOESM1">4</a>). Deviation in pH from optimal value leads to breakage of these bond and hence loss of active site and subsequently loss in enzymatic activity. pH optima was close to effective pH for <i>A</i>. <i>thaliana</i> i. e. 7.5 <sup> <a href="#CR17">17</a> </sup>, whereas <i>Trypanosoma cruzi</i>, a parasite had optimal pH in the range of 5.7–6.3 <sup> <a href="#CR26">26</a> </sup>. Lower value of K <sub>m</sub> of HMG-CoA (88 µM) than NADPH (903 µM) showed that the enzyme has more affinity for its substrate rather than its cofactor. Other reports on HMGR where the enzyme had higher affinity for HMG-CoA than NADPH such as in <i>A</i>. <i>thaliana</i>, <i>Raphanus raphanistrum</i>, <i>Solanum tuberosum</i> with lower K <sub>m</sub> for HMG-CoA as compared to Km for co-factor NADPH <sup> <a href="#CR17">17</a> </sup>. This clearly suggests that the availability of the substrate could be a rate controlling factor. </p> <p>In <i>O</i>. <i>kilimandscharicum</i>, HMGR exhibited differential pattern of gene expression in different tissues of the plant. Similarly, in other plants such as in <i>C</i>. <i>arachnoids</i>, highest expression in stem tissues and lowest in leaves was recorded <sup> <a href="#CR27">27</a> </sup>. In <i>S</i>. <i>miltiorrhiza</i>, <i>HMGR</i> expression was detected in roots, stems and leaves but highest expression was found in roots, followed by stem and leaf tissues <sup> <a href="#CR28">28</a> </sup>. In, this study, high expression levels in flower tissues indicate that <i>OkHMGR</i> may have role in growth and development of flower or metabolite accumulation in inflorescence <sup> <a href="#CR29">29</a> </sup> (Fig. <a href="#Fig4">4A</a>) as <i>HMGR</i> is involved in the synthesis of diverse metabolites <i>viz</i> sesquiterpenes, triterpenes and polyterpenes in the cytoplasm <sup> <a href="#CR30">30</a> </sup>. This varied response of <i>HMGR</i> in different plant tissues may be interpreted as the need of plant adjustments for various tissue specific roles, developmental and environmental conditions. </p> <p>MeJA and SA are the plant signaling molecules which apart from having major role in defense responses are also involved in seed germination, fruit ripening and senescence which is a part of plant growth and development responses <sup> <a href="#CR31">31</a>, <a href="#CR32">32</a> </sup>. These molecules induce the expression of genes involved in secondary metabolism, cell-wall formation, stress protection and defense related processes <sup> <a href="#CR32">32</a> </sup>. Upregulation of <i>OkHMGR</i> in the presence of these signaling molecules (Fig. <a href="#Fig4">4B,C</a>) indicats that <i>HMGR</i> has a modulating function in response to various biotic and abiotic stresses and also key participation in physiological activities. These results were in agreement with the study done in <i>W</i>. <i>somnifera</i> where <i>HMGR</i> was responsive to MeJA and SA treatments <sup> <a href="#CR12">12</a> </sup>. Similarly, in <i>A</i>. <i>paniculata</i>, exogenous application of JA increased HMGR activity and secondary metabolite andrographolide content was also enhanced in jasmonic acid treated tissues <sup> <a href="#CR19">19</a> </sup>. Unlike other reports on drastic increase in HMGR activity on wounding, no significant changes in <i>OkHMGR</i> levels were observed. Apart from these elicitors certain hormones are known to influence the activity of enzymes. Application of GA <sub>3</sub> increased the expression of <i>OkHMGR</i> transcripts (Fig. <a href="#Fig4">4E</a>). These results were consistent with the findings in <i>Pisum sativum</i> seedlings and <i>Cannabis sativa</i> where GA <sub>3</sub> increased HMGR activity <sup> <a href="#CR33">33</a> </sup>. Thus it indicates that <i>OkHMGR</i> has hormonal control as well in the regulation of isoprenoids. </p> <p>Stress induces a number of physiological and metabolic changes in the plant at primary level such as production of reactive oxygen species, changes in structure of chromatin, altering membrane permeability, modulation at transcriptional level, and hormone release etc <sup> <a href="#CR34">34</a> </sup>. Nature has evolved mechanisms so as to sustain itself under various stress conditions. Increase in transcription levels and hence enhanced production of secondary metabolites is one such possibility. Phytomolecules are reported to minimize the detrimental effects of stress by increasing the tolerance of the plant against stress and reducing reactive oxygen species production. Elevated terpenoids accumulation and higher expression of <i>HMGR</i> in <i>A</i>. <i>annua</i> was reported under prolonged water deficit stress and NaCl salinity stress in relation to adaptive response <sup> <a href="#CR35">35</a>, <a href="#CR36">36</a> </sup>. Temperature related stress modulates the synthesis of secondary metabolites such as phenolics, terpenoids, flavonoids <sup> <a href="#CR37">37</a> </sup>. These isoprene units such as α-pinene, have been known to withstand the harmful effect of temperature related stress. Monoterpenes and sesquiterpenes content increased in <i>S</i>. <i>lycopersicum</i> when the plant underwent heat and cold stress thereby suggesting that plants exhibit a definite set of cellular and metabolic responses necessary for the survival of the plant under stress conditions <sup> <a href="#CR38">38</a>, <a href="#CR39">39</a> </sup>. <i>HMGR</i> is an important step which is rate limiting in the biosynthesis of secondary metabolites. Production of secondary metabolites, influential in combating stress, is governed by expression of <i>HMGR</i> of mevalonate pathway. Varying pattern of <i>HMGR</i> expression manifests that <i>HMGR</i> has a shielding effect on plants in response to various stress by synthesizing various isoprenoids including sterols under stress conditions. </p> <p>Transient transformation of a gene in plants offers various advantages due to which it has become a method of choice for determination of function of a gene. This approach is specifically beneficial for recalcitrant plants for regeneration, as it facilitates assaying gene transfer independent of regeneration <sup> <a href="#CR40">40</a> </sup>. Transient transformation of amorpha-3, 11-diene synthase (ADS) and epicedrol synthase (ECS) of <i>A</i>. <i>annua</i> in <i>N</i>. <i>benthamiana</i>, intermediate enzymes of artemisinin biosynthesis pathway was reported to show remarkable increase in the artemisinin production <sup> <a href="#CR41">41</a> </sup>. The percentage of the essential oil constituents in over expressed tissues was higher <sup> <a href="#CR4">4</a>, <a href="#CR42">42</a> </sup> along with 5 folds higher transcripts in over-expressed tissues. Another report from <i>Parthenium argentatum</i> (guayule) also showed that the over-expression of <i>HMGR</i> led to 65% increase in polyisoprenoids rubber <sup> <a href="#CR43">43</a> </sup>. Interestingly, when change in individual constituents was being studied we noted that not only sesquiterpenes like caryophyllene, germacrene D but also some of the monoterpenes such as beta-myrcene, limonene, beta ocimene, camphene also increased <sup> <a href="#CR44">44</a>, <a href="#CR30">30</a> </sup>. The stable transformed plants of <i>O</i>. <i>kilimandscharicum</i> confirm the above results (data not shown). The moderate enhancement in monoterpenoid may account probably to the observations that an immediate conversion of the C5 moieties might have been influenced, although major impact was shed on MVA dependent metabolites. To further affirm our findings, the gene was transformed in different species of <i>Ocimum</i> namely, <i>O</i>. <i>basilicum</i>, <i>O</i>. <i>sanctum</i> and <i>O</i>. <i>gratissimum</i>, which represent another class in terms of phytochemical moieties. In all the three species over-expression led to the increase in total essential oil content, individual essential oil constituents and mRNA transcript levels. It is reported earlier that the monoterpenoid essential oils specifically ocimene are derived from MEP pathway as confirmed by <sup>13</sup>C labelling and inhibitor treatment studies <sup> <a href="#CR45">45</a> </sup>. Interestingly, it was reported that in ripen strawberry, both monoterpenoid (linalool) and sesquiterpenoid (nerolidol) are biosynthesized in the cytosol as the gene FaNES1 which is capable of synthesizing both (linalool) and (nerolidol) depending upon the presence of its precursors GPP and FPP lacks plastid targeting peptide sequence at N-terminal. The observations were further confirmed by feeding studies where labeling of mevalonic acid led to labeling of linalool <sup> <a href="#CR46">46</a> </sup>. It was shown that the two isoforms of <i>A</i>. <i>thaliana</i> GPP synthase are involved in organelle specific manner, one being targeted to the plastids while the other was functionally linked to cytosol <sup> <a href="#CR47">47</a> </sup>. Shikonin, a monoterpene derivative from <i>L</i>. <i>erythrorhizon</i> is also produced by MVA pathway and negatively regulated by mevalonin treatment <sup> <a href="#CR48">48</a> </sup>. The reason which can be anticipated for this is loss of plastid targeting peptide due to some splicing event. Another reason is presence of dual isoforms of a synthase targeted to different cellular compartments which arises due to some mutation or difference in selection of first or second methionine group for initiation of protein synthesis or other post translational modifications <sup> <a href="#CR49">49</a> </sup>. The importance of <i>HMGR</i> is not restricted to triterpenoids only but carotenoids were also affected by its expression. Over-expression of <i>HMGR</i> increased the total carotenoids by up to 43% (Supplementary Fig. <a href="#MOESM1">6</a>). As with monoterpenes, carotenoids are also assumed to be solely derived by MEP pathway but change in carotenoids content with change in <i>HMGR</i> expression shows an association between <i>HMGR</i> and carotenoids. Differential increment among various plants can be attributed to the physiology of the plant which varies considerably. <i>HMGR</i> synthesizes a pool of IPP which is acted by a number of enzymes of downstream pathway leading to production of various metabolites. It depends on the flux of these downstream enzymes which determine the metabolite to be synthesized and their levels. </p> <p>Results in <i>O</i>. <i>kilimandscharicum</i> and different <i>Ocimum</i> species suggested to explore the <i>HMGR</i> function in plants of other family. <i>A</i>. <i>annua</i> is rich in essential oils and artemisinin content and over-expression of <i>HMGR</i> increased the total essential oil content by 15% while artemisinin content by 7.4%. Our result was in consensus with what has been observed earlier where transformation of plant with <i>Catharanthus roseus</i> (L.) <i>HMGR</i> could drive the artemisinin content <sup> <a href="#CR50">50</a> </sup>. These findings further proved that <i>HMGR</i> is involved in synthesis of sesqui-terpenes, especially artemisinin. Similarly, <i>W</i>. <i>somnifera</i> contains withanolides and sterols as an important phytoconstituents and <i>HMGR</i> is known to be involved in the production of various triterpenoids and sterols. In <i>W</i>. <i>somnifera</i>, over-expression of <i>OkHMGR</i>, transiently led to increase in mRNA transcripts and hence withanolides and sterols as well. </p> <p>The application of inhibitor strategy onto untransformed control, vector control and over-expressed lines exhibited decrease in <i>HMGR</i> transcripts as well as terpenoid production. In transformed lines, over-expression and inhibition both the phenomenon were acting together as inhibition in transformed lines was less than that of control lines. These results were common in all the plants studied thereby confirming that <i>HMGR</i> has a major function in the synthesis of terpenoids. However, in case of <i>W</i>. <i>somnifera</i>, inhibitor treatment was more pronounced than over-expression while in some change in withanolide content was more than that of sterol content. The variation may arise due to the fact that triterpenoids and sterols are not the immediate products of <i>HMGR</i>. They are intervened by a series of enzymes and their intermediates which are finally converted to product. The plant tissues are acted upon by different downstream enzymes which drive the metabolite pool differentially towards the biosynthesis of terpenoids depending upon the conditions such as over-expression, silencing, inhibitor treatments etc. It has already been established that different tissues and chemotypes of <i>W</i>. <i>somnifera</i> have different withanolide content which to a great extent is governed by <i>HMGR</i> thereby imparting its role in withanogenesis <sup> <a href="#CR12">12</a>, <a href="#CR51">51</a> </sup>. Increased <i>HMGR</i> expression with diversion in pathway it selects i.e. either withanolides or sterols may also be due to post-transcriptional modifications of <i>HMGR</i>. Involvement of <i>HMGR</i> in triterpenoid synthesis was earlier reported when over-expression of <i>P</i>. <i>ginseng HMGR</i> in <i>A</i>. <i>thaliana</i> increased the content of sterols and triterpenoids, and in <i>P</i>. <i>ginseng</i> increased ginsenosides <sup> <a href="#CR52">52</a> </sup>. Although MVA dependent pathway has major participation in sterol and triterpenoid biosynthesis <sup> <a href="#CR51">51</a>, <a href="#CR52">52</a> </sup>, yet MEP pathway too has a role with extent, depending on the class of terpenoids <sup> <a href="#CR51">51</a> </sup>. Our earlier report on <i>W</i>. <i>somnifera</i> has established that major contribution was from MVA pathway rather than MEP pathway <sup> <a href="#CR25">25</a> </sup>. </p> <p>In conclusion, a novel regulatory <i>HMGR</i> from less explored <i>O</i>. <i>kilimandscharicum</i>, is reported which has active participation and regulatory role in MVA derived phytomolecules not only in native plant which has terpenoid rich essential oil but in other medicinally important plants such as <i>W</i>. <i>somnifera</i> and <i>A</i>. <i>annua</i>. This is the first report of transformation of a gene in <i>O</i>. <i>kilimandscharicum</i> and hence opens-up new avenues for determination of function of more pathway genes downstream <i>HMGR</i> thereby providing a better understanding of biosynthetic pathway. The study also provides a novel <i>OkHMGR</i> which can be used as a powerful sequence to drive MVA led isoprenoids in medicinally and aromatically important plants. </p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="isolationofputativeokhmgrfromleaftissue" title="sec"> <div class="title" tagx="title" title="title">
Isolation of putative <i>OkHMGR</i> from leaf tissue </div> <p>Total RNA was extracted from <i>O</i>. <i>kilimandscharicum</i> plants growing at CSIR-Central Institute of Medicinal and Aromatic Plant, Lucknow <sup> <a href="#CR53">53</a> </sup>. RNA was analyzed quantitatively and qualitatively and used for cDNA preparation using RevertAid First Strand cDNA Synthesis Kit (Thermo Fisher Scientific, Waltham, MA) according to manufacturer’s protocol. Degenerate primer HMGRDGF and HMGRDGR were designed (Supplementary Table <a href="#MOESM1">2</a>) and used to amplify the partial <i>HMGR</i> fragment. The conditions used for amplification were: initial denaturation at 94 °C for 2.30 min followed by 35 cycles of 94 °C for 30 s, 54 °C for 40 s and 72 °C for 2 min followed by final extension for 7 min at 72°C in a thermal cycler (Eppendorf). Partial <i>OkHMGR</i> sequence was used to design 3′ and 5′ RACE primers. To amplify putative downstream region, PCR reaction was performed with 3′ RACE primer pair of <i>OkHMGR</i>DF1 and oligodT (for primary reaction) and <i>OkHMGR</i>DF2 and oligodT (for nested PCR reaction) under similar conditions as used for partial fragment with change in annealing temperature i.e. 52 °C. For amplification of upstream region, 5′ RACE primer pairs <i>OkHMGR</i>UR1 and AAP (for primary reaction) and <i>OkHMGR</i>UR2 and AUAP (for nested PCR reaction) were used with difference in annealing conditions i.e. 58 °C for 45 s. 50 µl reaction volume was set up for PCR reactions containing 100 ng template, 20 picomole of primers, 25 µl master mix and remaining volume was maintained using nuclease free water. Amplified fragments of both upstream and downstream regions were analyzed, eluted from gel, cloned in pJET1.2 cloning vector and transformed in DH5α <i>E</i>. <i>coli</i> cells. Transformed colonies were screened <i>via</i> colony PCR, plasmids were sequenced and confirmed as part of HMGR by BLAST analysis. </p> <p>All the three sequences were assembled to make full length sequence of <i>OkHMGR</i>. Using this assembled full length sequence forward <i>OkHMGR</i>FLF and reverse <i>OkHMGR</i>FLR primers were designed containing <i>Sal</i>I and <i>Hind</i>III restrictions sites respectively for full length amplification of <i>OkHMGR</i>. The PCR conditions were as follows: 94 °C for 3.0 min, 35 cycles of 94 °C for 30 s, 55 °C for 1 min and 72 °C for 2.0 min followed by a final extension of 72 °C for 7 min. The resultant product was cloned in pJET1.2 cloning vector and transformed into DH5α competent cells. By colony PCR the positive clones of <i>OkHMGR</i> cDNA were screened. Full length <i>OkHMGR</i> was further confirmed by restriction digestion with <i>Sal</i>I and <i>Hind</i>III restriction enzymes. </p> </div> <div class="sequenceretrieval,multiplesequencealignmentandphylogeneticstudies" title="sec"> <div class="title" tagx="title" title="title">
Sequence retrieval, multiple sequence alignment and phylogenetic studies</div> <p>A total of 108 HMGR sequences from 55 plants along with <i>OkHMGR</i> were taken and multiple sequence alignment was done by the Mega 6.06 <sup> <a href="#CR54">54</a> </sup>. Motifs and domains were assigned by the online web databases as MEME server <sup> <a href="#CR55">55</a> </sup>. Physiochemical properties were carried out, by using protpram tool from expasy server ( <a href="https://web.expasy.org/protparam/">https://web.expasy.org/protparam/</a>). Secondary structure prediction was carried by using GORIV sever, widely used for modeling purposes ( <a href="https://npsa-prabi.ibcp.fr/cgi-bin/npsa_automat.pl?page=/NPSA/npsa_gor4.html">https://npsa-prabi.ibcp.fr/cgi-bin/npsa_automat.pl?page = /NPSA/npsa_gor4.html</a>). </p> </div> <div class="heterologousexpressionofokhmgrine.coliandpurificationofrecombinantokhmgrprotein" title="sec"> <div class="title" tagx="title" title="title">
Heterologous expression of <i>OkHMGR</i> in <i>E</i>. <i>coli</i> and purification of recombinant OkHMGR protein </div> <p>Plasmid DNA of pJET cloned full length <i>OkHMGR</i> and pET28a plasmid were digested by <i>Sal</i>I and <i>Hind</i>III restriction enzymes and both were ligated by T4 DNA ligase. Ligated product, pET- <i>OkHMGR</i>, was transformed in BL21 (DE3) competent cells and confirmed by colony PCR and restriction digestion. For over-expression of pET- <i>OkHMGR</i> in <i>E</i>. <i>coli</i> (BL21) the cells were induced by 0.8 mM isopropyl β-D-1-thiogalactopyranoside (IPTG) and allowed to grow at 18 °C for overnight. The cell lysate was prepared by sonication and analysed by12% SDS gel. For purification of recombinant OkHMGR enzyme, one liter recombinant <i>E</i>. <i>coli</i> BL21 (DE3) culture was induced and purified through Ni-NTA affinity column chromatography. Fractions obtained after elution were screened for protein concentration (A <sub>280</sub>) as well as for its catalytic activity. </p> </div> <div class="enzymeassay" title="sec"> <div class="title" tagx="title" title="title">
Enzyme assay</div> <p>HMG-CoA-dependent oxidation of NADPH <i>via</i> HMGR enzyme was carried out at 30 °C using spectrophotometer at 340 nm essentially by using the method of Dale <i>et al</i>. <sup> <a href="#CR17">17</a> </sup>. Assay mixture consisted of 0.05 mM HMG-CoA, 3.0 mM NADPH, 7.5 mM dithiothreitol (DTT) and 150 mM potassium phosphate buffer (pH 7.0). Absorption coefficient of 6220 M <sup>−1</sup> cm <sup>−1</sup> at 340 nm for NADPH was used for computation of activity. Further, various catalytic properties of OkHMGR enzyme were estimated by altering required parameters. </p> </div> <div class="expressionprofilingofokhmgr" title="sec"> <div class="title" tagx="title" title="title">
Expression profiling of <i>OkHMGR</i> </div> <p>Relative level of <i>OkHMGR</i> transcripts was determined in six different tissues of the plant i.e. young leaf, middle/mature leaf, large/fully mature leaf, stem, bud and flower as described earlier <sup> <a href="#CR56">56</a> </sup>. Gene specific qRT (quantitative real time) primers were designed using Beacon Designer Software (Supplementary Table <a href="#MOESM1">2</a>). PCR reaction mixture comprised of ~50 ng cDNA, 5 pmole of each forward and reverse primers, 5 µl of Power SYBR Green PCR master mix 2X (Applied Biosystems, USA) and the final volume was maintained with nuclease free water to 10 µl. Actin gene was taken as an endogenous control <sup> <a href="#CR52">52</a> </sup>. All the reactions were carried out in triplicates. The reactions were carried out on StepOne <sup>TM</sup> Real Time PCR system with 48-well block module. The relative gene expression levels were calculated by ΔΔCT method and expressed as relative quantification (RQ) values. For different stresses and elicitor treatments leaves were treated for 4 hours. In case of wounding the leaves were injured with scalpel blade at different sites and treated samples were then collected after 0.5 h, 3 h and 24 h. Heat and cold stress were given at 55 °C and 4 °C respectively for different time durations i.e. 0.5, 1, 2, 4 hrs. Leaves were also treated with NaCl (0, 50, 100 and 250 mM), salicylic acid and methyl jasmonate (0, 10, 50, 100 and 500 µM), and gibberellic acid (0, 1, 10 and 50 µM) for 3 hrs. All the tissues were subjected to RNA isolation, cDNA preparation and transcript profiling. </p> </div> <div class="over-expressionofokhmgr" title="sec"> <div class="title" tagx="title" title="title">
Over-expression of <i>OkHMGR</i> </div> <p>For transient expression of <i>OkHMGR</i> in plants, the coding sequence of gene was cloned in plant binary vector pCAMBIA1303. Coding sequence of <i>OkHMGR</i> cloned in pJET 1.2 cloning vector, was digested with <i>Bgl</i>II enzyme. The resulting fragment was cloned at <i>Bgl</i>II site of pCAMBIA1303 under the control of CaMV35S promoter. After confirmation of ligation event through colony PCR and digestion, the resulting pCAMBIA: <i>OkHMGR</i> construct was transformed in <i>A</i>. <i>tumefaciens</i> strain GV3010 <i>via</i> freeze thaw method <sup> <a href="#CR57">57</a> </sup>. Transformation of pCAMBIA: <i>OkHMGR</i> construct in <i>A</i>. <i>tumefaciens</i> was confirmed by colony PCR with <i>npt</i>II primers (Supplementary Table <a href="#MOESM1">2</a>). To check the activity of <i>OkHMGR</i> at transient level, <i>O</i>. <i>basilicum</i>, <i>O</i>. <i>sanctum</i>, <i>O</i>. <i>gratissimum</i>, <i>O</i>. <i>kilimandscharicum</i>, <i>W</i>. <i>somnifera</i> and <i>A</i>. <i>annua</i> were used. The leaves from these plants were processed for transformation as reported earlier <sup> <a href="#CR18">18</a> </sup>. For inhibitor studies, tissues of control and transiently transformed plants were treated with both mevastatin and pravastatin individually for 24 hrs followed by various analysis. </p> </div> <div class="secondarymetabolitedetermination" title="sec"> <div class="title" tagx="title" title="title">
Secondary metabolite determination</div> <p>Essential oil from control as well as transformed tissues was isolated by the method described in Bose <i>et al</i>. <sup> <a href="#CR58">58</a> </sup>. Essential oil constituents were analyzed by using GC and GC-MS analysis as reported earlier for <i>A</i>. <i>annua</i> <sup> <a href="#CR35">35</a>, <a href="#CR36">36</a> </sup>. Artemisinin content was determined by the protocol as mentioned in our previous publications <sup> <a href="#CR27">27</a> </sup>. Withanolide content was determined by our earlier protocol established for <i>W</i>. <i>somnifera</i> <sup> <a href="#CR59">59</a>, <a href="#CR60">60</a> </sup>. </p> </div> <div class="experimentaldesignandstatisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Experimental design and statistical analysis</div> <p>Each experiment was set in triplicates. To test statistical significance of difference among mean responses at a significance of P = 0.05 one way Analysis Of Variance (ANOVA) was used. </p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41598_2017_17153_MOESM1_ESM.docx">LINK</a>  <p>Supplementry Figures</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41598_2017_17153_MOESM2_ESM.pdf">LINK</a>  <p>Supplementary Tables</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p> <b>Electronic supplementary material</b> </p> <p> <b>Supplementary information</b> accompanies this paper at 10.1038/s41598-017-17153-z. </p> </div> <div class="fn-type-" title=""> <p> <b>Publisher's note:</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. </p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>NSS thanks CSIR for BSC-203, BSC-107 and HCP007 project grants for providing financial assistance. SB is thankful to CSIR, New Delhi for fellowship and AcSIR for providing Ph. D registration. LKN, RKY and MC thanks to UGC and BM thanks to ICMR for fellowships. Authors also thank Dr Saudan Singh and Dr AK Gupta for plant materials and Dr. C.S. Chanotiya and Anju Gupta of central facility for providing GC and GC-MS instrumentation facility. </p> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>N.S.S. conceived the project, designed the experiments and arranged the funds. S.B. performed most of the experiments. S.B., L.K.N., B.M., M.C. and R.K.Y. were involved in experiments, analysis and compilation of data. S.B. and N.S.S. wrote and edited the manuscript. All the authors have critically read the manuscript. </p> <div class="competinginterests" title="sec"> <div class="title" tagx="title" title="title">
Competing Interests</div> <p>The authors declare no competing interests.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="CR1" /> <span class="label" tagx="label" title="label">1.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tewari</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pandey</span> <span class="given-names" tagx="given-names" title="given-names">HK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sah</span> <span class="given-names" tagx="given-names" title="given-names">AN</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meena</span> <span class="given-names" tagx="given-names" title="given-names">HS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Manchanda</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Pharmacognostical and biochemical investigation of <i>Ocimum kilimandscharicum</i> plants available in western Himalayan region </span> <span class="source" tagx="source" title="source">Asian J. Plant Sci. Res</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="fpage" tagx="fpage" title="fpage">446</span> <span class="lpage" tagx="lpage" title="lpage">451</span> </span> </li> <li tag="ref"> <a name="CR2" /> <span class="label" tagx="label" title="label">2.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Labra</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Morphological characterization, essential oil composition and DNA genotyping of <i>Ocimum basilicum</i> L. cultivars </span> <span class="source" tagx="source" title="source">Plant Sci.</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">167</span> <span class="fpage" tagx="fpage" title="fpage">725</span> <span class="lpage" tagx="lpage" title="lpage">731</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.plantsci.2004.04.026">10.1016/j.plantsci.2004.04.026</a> </span> </span> </li> <li tag="ref"> <a name="CR3" /> <span class="label" tagx="label" title="label">3.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Varier, vaidhyarathnam P. S. Indian Medicinal plants a compendium of 500 species in <i>Indian Medicinal plants a compendium of 500 species</i> 357 (1996). </span> </li> <li tag="ref"> <a name="CR4" /> <span class="label" tagx="label" title="label">4.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Narwal</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rana</span> <span class="given-names" tagx="given-names" title="given-names">AC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tiwari</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gangwani</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharma</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Review on chemical constituents and pharmacological action of <i>Ocimum kilimandscharicum</i> </span> <span class="source" tagx="source" title="source">Indo Glob. J. Pharm. Sci.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">1</span> <span class="fpage" tagx="fpage" title="fpage">287</span> <span class="lpage" tagx="lpage" title="lpage">293</span> </span> </li> <li tag="ref"> <a name="CR5" /> <span class="label" tagx="label" title="label">5.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Padalia</span> <span class="given-names" tagx="given-names" title="given-names">RC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Verma</span> <span class="given-names" tagx="given-names" title="given-names">RS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Comparative volatile oil composition of four <i>Ocimum</i> species from northern India </span> <span class="source" tagx="source" title="source">Nat. Prod. Res.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">569</span> <span class="lpage" tagx="lpage" title="lpage">575</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1080/14786419.2010.482936">10.1080/14786419.2010.482936</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21409717">21409717</a> </span> </span> </li> <li tag="ref"> <a name="CR6" /> <span class="label" tagx="label" title="label">6.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kashyap</span> <span class="given-names" tagx="given-names" title="given-names">CP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ranjeet</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vikrant</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vipin</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Therapeutic potency of <i>Ocimum kilimandscharicum</i> Guerke-A review </span> <span class="source" tagx="source" title="source">Glob. J. Pharmacol</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">191</span> <span class="lpage" tagx="lpage" title="lpage">200</span> </span> </li> <li tag="ref"> <a name="CR7" /> <span class="label" tagx="label" title="label">7.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Obeng-Ofori</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reichmuth</span> <span class="given-names" tagx="given-names" title="given-names">CH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bekele</span> <span class="given-names" tagx="given-names" title="given-names">AJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hassanali</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Toxicity and protectant potential of camphor, a major component of essential oil of <i>Ocimum kilimandscharicum</i>, against four stored product beetles </span> <span class="source" tagx="source" title="source">Int. J. Pest Manag</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">44</span> <span class="fpage" tagx="fpage" title="fpage">203</span> <span class="lpage" tagx="lpage" title="lpage">209</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1080/096708798228112">10.1080/096708798228112</a> </span> </span> </li> <li tag="ref"> <a name="CR8" /> <span class="label" tagx="label" title="label">8.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hakkim</span> <span class="given-names" tagx="given-names" title="given-names">FL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arivazhagan</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boopathy</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Antioxidant property of selected <i>Ocimum</i> species and their secondary metabolite content </span> <span class="source" tagx="source" title="source">J. Med. Plants Res</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="fpage" tagx="fpage" title="fpage">250</span> <span class="lpage" tagx="lpage" title="lpage">257</span> </span> </li> <li tag="ref"> <a name="CR9" /> <span class="label" tagx="label" title="label">9.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Darabi</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Masoudi-Nejad</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nemat-Zadeh</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Bioinformatics study of the 3-hydroxy-3-methylglutaryl-coenzyme A reductase (HMGR) gene in Gramineae </span> <span class="source" tagx="source" title="source">Mol. Biol. Rep.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">39</span> <span class="fpage" tagx="fpage" title="fpage">8925</span> <span class="lpage" tagx="lpage" title="lpage">8935</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11033-012-1761-2">10.1007/s11033-012-1761-2</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22722993">22722993</a> </span> </span> </li> <li tag="ref"> <a name="CR10" /> <span class="label" tagx="label" title="label">10.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rodriguez-Concepcion</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Elucidation of the methylerythritol phosphate pathway for isoprenoid biosynthesis in bacteria and plastids. A metabolic milestone achieved through genomics </span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">130</span> <span class="fpage" tagx="fpage" title="fpage">1079</span> <span class="lpage" tagx="lpage" title="lpage">1089</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.007138">10.1104/pp.007138</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12427975">12427975</a> </span> </span> </li> <li tag="ref"> <a name="CR11" /> <span class="label" tagx="label" title="label">11.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lange</span> <span class="given-names" tagx="given-names" title="given-names">BM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rujan</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martin</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Croteau</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Isoprenoid biosynthesis: The evolution of two ancient and distinct pathways across genomes </span> <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">97</span> <span class="fpage" tagx="fpage" title="fpage">13172</span> <span class="lpage" tagx="lpage" title="lpage">13177</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.240454797">10.1073/pnas.240454797</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11078528">11078528</a> </span> </span> </li> <li tag="ref"> <a name="CR12" /> <span class="label" tagx="label" title="label">12.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Akhtar</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gupta</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sangwan</span> <span class="given-names" tagx="given-names" title="given-names">NS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sangwan</span> <span class="given-names" tagx="given-names" title="given-names">RS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Trivedi</span> <span class="given-names" tagx="given-names" title="given-names">PK</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Cloning and functional characterization of 3-hydroxy-3-methylglutaryl coenzyme A reductase gene from <i>Withania somnifera</i>: An important medicinal plant </span> <span class="source" tagx="source" title="source">Protoplasma</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">250</span> <span class="fpage" tagx="fpage" title="fpage">613</span> <span class="lpage" tagx="lpage" title="lpage">622</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00709-012-0450-2">10.1007/s00709-012-0450-2</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22936023">22936023</a> </span> </span> </li> <li tag="ref"> <a name="CR13" /> <span class="label" tagx="label" title="label">13.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Narnoliya</span> <span class="given-names" tagx="given-names" title="given-names">LK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rajakani</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sangwan</span> <span class="given-names" tagx="given-names" title="given-names">NS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gupta</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sangwan</span> <span class="given-names" tagx="given-names" title="given-names">RS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Comparative transcripts profiling of fruit mesocarp and endocarp relevant to secondary metabolism by suppression subtractive hybridization in <i>Azadirachta indica</i> (neem) </span> <span class="source" tagx="source" title="source">Mol. Biol. Rep.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">41</span> <span class="fpage" tagx="fpage" title="fpage">3147</span> <span class="lpage" tagx="lpage" title="lpage">3162</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11033-014-3174-x">10.1007/s11033-014-3174-x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24477588">24477588</a> </span> </span> </li> <li tag="ref"> <a name="CR14" /> <span class="label" tagx="label" title="label">14.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dai</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cui</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">SF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Cloning and characterization of a novel 3-hydroxy-3-methylglutaryl coenzyme A reductase gene from <i>Salvia miltiorrhiza</i> involved in diterpenoid tanshinone accumulation </span> <span class="source" tagx="source" title="source">J. Plant Physiol.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">168</span> <span class="fpage" tagx="fpage" title="fpage">148</span> <span class="lpage" tagx="lpage" title="lpage">157</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.jplph.2010.06.008">10.1016/j.jplph.2010.06.008</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20637524">20637524</a> </span> </span> </li> <li tag="ref"> <a name="CR15" /> <span class="label" tagx="label" title="label">15.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Enjuto</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title"> <i>Arabidopsis thaliana</i> contains two differentially expressed 3-hydroxy-3-methylglutaryl-CoA reductase genes, which encode microsomal forms of the enzyme </span> <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U. S. A.</span> <span class="year" tagx="year" title="year">1994</span> <span class="volume" tagx="volume" title="volume">91</span> <span class="fpage" tagx="fpage" title="fpage">927</span> <span class="lpage" tagx="lpage" title="lpage">931</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.91.3.927">10.1073/pnas.91.3.927</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8302869">8302869</a> </span> </span> </li> <li tag="ref"> <a name="CR16" /> <span class="label" tagx="label" title="label">16.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Loguercio</span> <span class="given-names" tagx="given-names" title="given-names">LL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Scott</span> <span class="given-names" tagx="given-names" title="given-names">HC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Trolinder</span> <span class="given-names" tagx="given-names" title="given-names">NL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wilkins</span> <span class="given-names" tagx="given-names" title="given-names">TA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">HMG-CoA reductase gene family in cotton ( <i>Gossypium hirsutum</i> L.): Unique structural features and differential expression of HMG2 potentially associated with synthesis of specific isoprenoids in developing embryos </span> <span class="source" tagx="source" title="source">Plant Cell Physiol</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">40</span> <span class="fpage" tagx="fpage" title="fpage">750</span> <span class="lpage" tagx="lpage" title="lpage">761</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/oxfordjournals.pcp.a029602">10.1093/oxfordjournals.pcp.a029602</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10501034">10501034</a> </span> </span> </li> <li tag="ref"> <a name="CR17" /> <span class="label" tagx="label" title="label">17.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dale</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Bacterial expression of the catalytic domain of 3–hydroxy‐3–methylglutaryl‐coa reductase (isoform HMGR1) from <i>Arabidopsis thaliana</i>, and Its inactivation by phosphorylation at Ser577 by <i>Brassica oleracea</i> 3‐hydroxy‐3‐methylglutaryl‐CoA reductase kinase </span> <span class="source" tagx="source" title="source">Eur. J. Biochem.</span> <span class="year" tagx="year" title="year">1995</span> <span class="volume" tagx="volume" title="volume">233</span> <span class="fpage" tagx="fpage" title="fpage">506</span> <span class="lpage" tagx="lpage" title="lpage">513</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1432-1033.1995.506_2.x">10.1111/j.1432-1033.1995.506_2.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7588795">7588795</a> </span> </span> </li> <li tag="ref"> <a name="CR18" /> <span class="label" tagx="label" title="label">18.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jadaun</span> <span class="given-names" tagx="given-names" title="given-names">JS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sangwan</span> <span class="given-names" tagx="given-names" title="given-names">NS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Narnoliya</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singh</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Over-expression of DXS gene enhances terpenoidal secondary metabolite accumulation in rose-scented geranium and <i>Withania somnifera</i> : active involvement of plastid isoprenogenic pathway in their biosynthesis </span> <span class="source" tagx="source" title="source">Physiol. Plant.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">159</span> <span class="fpage" tagx="fpage" title="fpage">381</span> <span class="lpage" tagx="lpage" title="lpage">400</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/ppl.12507">10.1111/ppl.12507</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27580641">27580641</a> </span> </span> </li> <li tag="ref"> <a name="CR19" /> <span class="label" tagx="label" title="label">19.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jha</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharam</span> <span class="given-names" tagx="given-names" title="given-names">SN</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharma</span> <span class="given-names" tagx="given-names" title="given-names">DK</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Differential expression of 3-hydroxy-3-methylglutaryl-coenzyme a reductase of <i>Andrographis paniculata</i> in andrographolide accumulation </span> <span class="source" tagx="source" title="source">J. Chem. Pharm. Res.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="fpage" tagx="fpage" title="fpage">499</span> <span class="lpage" tagx="lpage" title="lpage">504</span> </span> </li> <li tag="ref"> <a name="CR20" /> <span class="label" tagx="label" title="label">20.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rodwell</span> <span class="given-names" tagx="given-names" title="given-names">VW</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">3-Hydroxy-3-methylglutaryl-CoA reductase</span> <span class="source" tagx="source" title="source">Methods Enzymol</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">324</span> <span class="fpage" tagx="fpage" title="fpage">259</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0076-6879(00)24237-7">10.1016/S0076-6879(00)24237-7</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10989436">10989436</a> </span> </span> </li> <li tag="ref"> <a name="CR21" /> <span class="label" tagx="label" title="label">21.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Sangwan, N. S.&amp;amp; Sangwan, R. S. Secondary metabolites of traditional medical plants: a case study of ashwagandha ( <i>Withania Somnifera</i>) in: <i>Applied Plant Cell Biology</i> 325–367 (Springer, 2014). </span> </li> <li tag="ref"> <a name="CR22" /> <span class="label" tagx="label" title="label">22.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Istvan</span> <span class="given-names" tagx="given-names" title="given-names">ES</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deisenhofer</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The structure of the catalytic portion of human HMG-CoA reductase</span> <span class="source" tagx="source" title="source">Biochimica et Biophysica Acta; Molecular and Cell Biology of Lipids</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">1529</span> <span class="fpage" tagx="fpage" title="fpage">9</span> <span class="lpage" tagx="lpage" title="lpage">18</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S1388-1981(00)00134-7">10.1016/S1388-1981(00)00134-7</a> </span> </span> </li> <li tag="ref"> <a name="CR23" /> <span class="label" tagx="label" title="label">23.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Olender</span> <span class="given-names" tagx="given-names" title="given-names">EH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Simon</span> <span class="given-names" tagx="given-names" title="given-names">RD</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The intracellular targeting and membrane topology of 3-hydroxy-3-methylglutaryl-CoA reductase </span> <span class="source" tagx="source" title="source">J Biol Chem</span> <span class="year" tagx="year" title="year">1992</span> <span class="volume" tagx="volume" title="volume">267</span> <span class="fpage" tagx="fpage" title="fpage">4223</span> <span class="lpage" tagx="lpage" title="lpage">4235</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/1740462">1740462</a> </span> </span> </li> <li tag="ref"> <a name="CR24" /> <span class="label" tagx="label" title="label">24.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Molecular cloning and functional analysis of the gene encoding 3-Hydroxy-3-Methylglutaryl coenzyme A reductase from hazel ( <i>Corylus avellana</i> L. Gasaway) </span> <span class="source" tagx="source" title="source">BMB Rep</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">40</span> <span class="fpage" tagx="fpage" title="fpage">861</span> <span class="lpage" tagx="lpage" title="lpage">869</span> <span class="pub-id"> <a href="https://dx.doi.org/10.5483/BMBRep.2007.40.6.861">10.5483/BMBRep.2007.40.6.861</a> </span> </span> </li> <li tag="ref"> <a name="CR25" /> <span class="label" tagx="label" title="label">25.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kalita</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Patar</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shasany</span> <span class="given-names" tagx="given-names" title="given-names">AK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Modi</span> <span class="given-names" tagx="given-names" title="given-names">MK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sen</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Molecular cloning, characterization and expression analysis of 3-Hydroxy-3-Methylglutaryl coenzyme A reductase gene from <i>Centella asiatica</i> L </span> <span class="source" tagx="source" title="source">Mol. Biol. Rep.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">42</span> <span class="fpage" tagx="fpage" title="fpage">1431</span> <span class="lpage" tagx="lpage" title="lpage">1439</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11033-015-3922-6">10.1007/s11033-015-3922-6</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26311154">26311154</a> </span> </span> </li> <li tag="ref"> <a name="CR26" /> <span class="label" tagx="label" title="label">26.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hurtado-Guerrrero</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pea-Díaz</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Montalvetti</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ruiz-Pérez</span> <span class="given-names" tagx="given-names" title="given-names">LM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">González-Pacanowska</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Kinetic properties and inhibition of <i>Trypanosoma cruzi</i> 3-hydroxy-3-methylglutaryl CoA reductase </span> <span class="source" tagx="source" title="source">FEBS Lett</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">510</span> <span class="fpage" tagx="fpage" title="fpage">141</span> <span class="lpage" tagx="lpage" title="lpage">144</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0014-5793(01)03238-0">10.1016/S0014-5793(01)03238-0</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11801242">11801242</a> </span> </span> </li> <li tag="ref"> <a name="CR27" /> <span class="label" tagx="label" title="label">27.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">QJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zheng</span> <span class="given-names" tagx="given-names" title="given-names">LP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">PF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">YL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">JW</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Cloning and characterization of an elicitor-responsive gene encoding 3-hydroxy-3-methylglutaryl coenzyme A reductase involved in 20-hydroxyecdysone production in cell cultures of <i>Cyanotis arachnoidea</i> </span> <span class="source" tagx="source" title="source">Plant Physiol. Biochem.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">84</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">9</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.plaphy.2014.08.021">10.1016/j.plaphy.2014.08.021</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25232679">25232679</a> </span> </span> </li> <li tag="ref"> <a name="CR28" /> <span class="label" tagx="label" title="label">28.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liao</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Molecular cloning, characterization and expression analysis of a new gene encoding 3-hydroxy-3-methylglutaryl coenzyme A reductase from <i>Salvia miltiorrhiza</i> </span> <span class="source" tagx="source" title="source">Acta Physiol. Plant.</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">31</span> <span class="fpage" tagx="fpage" title="fpage">565</span> <span class="lpage" tagx="lpage" title="lpage">572</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11738-008-0266-z">10.1007/s11738-008-0266-z</a> </span> </span> </li> <li tag="ref"> <a name="CR29" /> <span class="label" tagx="label" title="label">29.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Narváez</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Canto Canché</span> <span class="given-names" tagx="given-names" title="given-names">BB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pérez</span> <span class="given-names" tagx="given-names" title="given-names">PF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Madrid</span> <span class="given-names" tagx="given-names" title="given-names">RR</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Differential expression of 3-Hydroxy-3-Methylglutaryl-CoA reductase (HMGR) during flower and fruit development of <i>Bixa orellana</i> </span> <span class="source" tagx="source" title="source">J. Plant Physiol.</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">158</span> <span class="fpage" tagx="fpage" title="fpage">1471</span> <span class="lpage" tagx="lpage" title="lpage">1477</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1078/0176-1617-00614">10.1078/0176-1617-00614</a> </span> </span> </li> <li tag="ref"> <a name="CR30" /> <span class="label" tagx="label" title="label">30.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lichtenthaler</span> <span class="given-names" tagx="given-names" title="given-names">HK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schwender</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Disch</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rohmer</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Biosynthesis of isoprenoids in higher plant chloroplasts proceeds <i>via</i> a mevalonate-independent pathway </span> <span class="source" tagx="source" title="source">FEBS Lett</span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">400</span> <span class="fpage" tagx="fpage" title="fpage">271</span> <span class="lpage" tagx="lpage" title="lpage">274</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0014-5793(96)01404-4">10.1016/S0014-5793(96)01404-4</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9009212">9009212</a> </span> </span> </li> <li tag="ref"> <a name="CR31" /> <span class="label" tagx="label" title="label">31.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rivas-San Vicente</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Plasencia</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Salicylic acid beyond defence: Its role in plant growth and development</span> <span class="source" tagx="source" title="source">J Exp Bot</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">62</span> <span class="fpage" tagx="fpage" title="fpage">3321</span> <span class="lpage" tagx="lpage" title="lpage">3338</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/jxb/err031">10.1093/jxb/err031</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21357767">21357767</a> </span> </span> </li> <li tag="ref"> <a name="CR32" /> <span class="label" tagx="label" title="label">32.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheong</span> <span class="given-names" tagx="given-names" title="given-names">JJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Choi</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Do. Methyl jasmonate as a vital substance in plants</span> <span class="source" tagx="source" title="source">Trends in Genetics</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">409</span> <span class="lpage" tagx="lpage" title="lpage">413</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0168-9525(03)00138-0">10.1016/S0168-9525(03)00138-0</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12850447">12850447</a> </span> </span> </li> <li tag="ref"> <a name="CR33" /> <span class="label" tagx="label" title="label">33.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Asrar, Z. Terpenoids and gibberellic acids interaction in plants in <i>Advances in Selected Plant Physiology Aspects</i> Eds: Montanaro G. &amp;amp; Bartolomeo D. (InTech, 2012). </span> </li> <li tag="ref"> <a name="CR34" /> <span class="label" tagx="label" title="label">34.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Bita, C. E. &amp;amp; Gerats, T. Plant tolerance to high temperature in a changing environment: Scientific fundamentals and production of heat stress-tolerant crops. <i>Front. Plant Sci</i>. <b>4</b> (2013). </span> </li> <li tag="ref"> <a name="CR35" /> <span class="label" tagx="label" title="label">35.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yadav</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sangwan</span> <span class="given-names" tagx="given-names" title="given-names">RS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sabir</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Srivastava</span> <span class="given-names" tagx="given-names" title="given-names">AK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sangwan</span> <span class="given-names" tagx="given-names" title="given-names">NS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Effect of prolonged water stress on specialized secondary metabolites, peltate glandular trichomes, and pathway gene expression in <i>Artemisia annua</i> L </span> <span class="source" tagx="source" title="source">Plant Physiol. Biochem.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">74</span> <span class="fpage" tagx="fpage" title="fpage">70</span> <span class="lpage" tagx="lpage" title="lpage">83</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.plaphy.2013.10.023">10.1016/j.plaphy.2013.10.023</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24269871">24269871</a> </span> </span> </li> <li tag="ref"> <a name="CR36" /> <span class="label" tagx="label" title="label">36.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yadav</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sangwan</span> <span class="given-names" tagx="given-names" title="given-names">RS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Srivastava</span> <span class="given-names" tagx="given-names" title="given-names">AK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sangwan</span> <span class="given-names" tagx="given-names" title="given-names">NS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Prolonged exposure to salt stress affects specialized metabolites-artemisinin and essential oil accumulation in <i>Artemisia annua</i> L.: metabolic acclimation in preferential favour of enhanced terpenoid accumulation accompanying vegetative to reproductive pha </span> <span class="source" tagx="source" title="source">Protoplasma</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">254</span> <span class="fpage" tagx="fpage" title="fpage">505</span> <span class="lpage" tagx="lpage" title="lpage">522</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00709-016-0971-1">10.1007/s00709-016-0971-1</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27263081">27263081</a> </span> </span> </li> <li tag="ref"> <a name="CR37" /> <span class="label" tagx="label" title="label">37.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pateraki</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kanellis</span> <span class="given-names" tagx="given-names" title="given-names">AK</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Stress and developmental responses of terpenoid biosynthetic genes in <i>Cistus creticus</i> subsp. creticus </span> <span class="source" tagx="source" title="source">Plant Cell Rep</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">29</span> <span class="fpage" tagx="fpage" title="fpage">629</span> <span class="lpage" tagx="lpage" title="lpage">641</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00299-010-0849-1">10.1007/s00299-010-0849-1</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20364257">20364257</a> </span> </span> </li> <li tag="ref"> <a name="CR38" /> <span class="label" tagx="label" title="label">38.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Copolovici</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kännaste</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pazouki</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Niinemets</span> <span class="given-names" tagx="given-names" title="given-names">Ü</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Emissions of green leaf volatiles and terpenoids from <i>Solanum lycopersicum</i> are quantitatively related to the severity of cold and heat shock treatments </span> <span class="source" tagx="source" title="source">J. Plant Physiol.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">169</span> <span class="fpage" tagx="fpage" title="fpage">664</span> <span class="lpage" tagx="lpage" title="lpage">672</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.jplph.2011.12.019">10.1016/j.jplph.2011.12.019</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22341571">22341571</a> </span> </span> </li> <li tag="ref"> <a name="CR39" /> <span class="label" tagx="label" title="label">39.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guy</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Molecular responses of plants to cold shock and cold acclimation</span> <span class="source" tagx="source" title="source">J. Mol. Microbiol. Biotechnol.</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">1</span> <span class="fpage" tagx="fpage" title="fpage">231</span> <span class="lpage" tagx="lpage" title="lpage">42</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10943554">10943554</a> </span> </span> </li> <li tag="ref"> <a name="CR40" /> <span class="label" tagx="label" title="label">40.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kapila</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">De Rycke</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van Montagu</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Angenon</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">An <i>Agrobacterium</i>-mediated transient gene expression system for intact leaves </span> <span class="source" tagx="source" title="source">Plant Sci.</span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">122</span> <span class="fpage" tagx="fpage" title="fpage">101</span> <span class="lpage" tagx="lpage" title="lpage">108</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0168-9452(96)04541-4">10.1016/S0168-9452(96)04541-4</a> </span> </span> </li> <li tag="ref"> <a name="CR41" /> <span class="label" tagx="label" title="label">41.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kanagarajan</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Muthusamy</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gliszczynska</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lundgren</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brodelius</span> <span class="given-names" tagx="given-names" title="given-names">PE</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Functional expression and characterization of sesquiterpene synthases from <i>Artemisia annua</i> L. using transient expression system in <i>Nicotiana benthamiana</i> </span> <span class="source" tagx="source" title="source">Plant Cell Rep</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">31</span> <span class="fpage" tagx="fpage" title="fpage">1309</span> <span class="lpage" tagx="lpage" title="lpage">1319</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00299-012-1250-z">10.1007/s00299-012-1250-z</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22565787">22565787</a> </span> </span> </li> <li tag="ref"> <a name="CR42" /> <span class="label" tagx="label" title="label">42.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pragadheesh</span> <span class="given-names" tagx="given-names" title="given-names">VS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saroj</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yadav</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Samad</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chanotiya</span> <span class="given-names" tagx="given-names" title="given-names">CS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Compositions, enantiomer characterization and antifungal activity of two <i>Ocimum</i> essential oils </span> <span class="source" tagx="source" title="source">Ind. Crops Prod</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">50</span> <span class="fpage" tagx="fpage" title="fpage">333</span> <span class="lpage" tagx="lpage" title="lpage">337</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.indcrop.2013.08.009">10.1016/j.indcrop.2013.08.009</a> </span> </span> </li> <li tag="ref"> <a name="CR43" /> <span class="label" tagx="label" title="label">43.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dong</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Overexpression of 3-Hydroxy-3-Methylglutaryl coenzyme A reductase in <i>Parthenium argentatum</i> (guayule) </span> <span class="source" tagx="source" title="source">Ind. Crops Prod</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">46</span> <span class="fpage" tagx="fpage" title="fpage">15</span> <span class="lpage" tagx="lpage" title="lpage">24</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.indcrop.2012.12.044">10.1016/j.indcrop.2012.12.044</a> </span> </span> </li> <li tag="ref"> <a name="CR44" /> <span class="label" tagx="label" title="label">44.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eisenreich</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sagner</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zenk</span> <span class="given-names" tagx="given-names" title="given-names">MH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bacher</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Monoterpenoid essential oils are not mevalonoid origin</span> <span class="source" tagx="source" title="source">Tetrahedron Lett.</span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">38</span> <span class="fpage" tagx="fpage" title="fpage">3889</span> <span class="lpage" tagx="lpage" title="lpage">3892</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0040-4039(97)00761-2">10.1016/S0040-4039(97)00761-2</a> </span> </span> </li> <li tag="ref"> <a name="CR45" /> <span class="label" tagx="label" title="label">45.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Piel</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Donath</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bandemer</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boland</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Mevalonate-independent biosynthesis of terpenoid volatiles in plants: Induced and constitutive emission of volatiles </span> <span class="source" tagx="source" title="source">Angew. Chemie - Int. Ed.</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">37</span> <span class="fpage" tagx="fpage" title="fpage">2478</span> <span class="lpage" tagx="lpage" title="lpage">2481</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/(SICI)1521-3773(19981002)37:18&lt;2478::AID-ANIE2478&gt;3.0.CO;2-Q">10.1002/(SICI)1521-3773(19981002)37:18&amp;lt;2478::AID-ANIE2478&amp;gt;3.0.CO;2-Q</a> </span> </span> </li> <li tag="ref"> <a name="CR46" /> <span class="label" tagx="label" title="label">46.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aharoni</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Gain and loss of fruit flavor compounds produced by wild and cultivated strawberry species </span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">3110</span> <span class="lpage" tagx="lpage" title="lpage">3131</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.104.023895">10.1105/tpc.104.023895</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15522848">15522848</a> </span> </span> </li> <li tag="ref"> <a name="CR47" /> <span class="label" tagx="label" title="label">47.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bouvier</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suire</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">D’Harlingue</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Backhaus</span> <span class="given-names" tagx="given-names" title="given-names">RA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Camara</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Molecular cloning of geranyl diphosphate synthase and compartmentation of monoterpene synthesis in plant cells </span> <span class="source" tagx="source" title="source">Plant J.</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">24</span> <span class="fpage" tagx="fpage" title="fpage">241</span> <span class="lpage" tagx="lpage" title="lpage">252</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1046/j.1365-313x.2000.00875.x">10.1046/j.1365-313x.2000.00875.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11069698">11069698</a> </span> </span> </li> <li tag="ref"> <a name="CR48" /> <span class="label" tagx="label" title="label">48.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Markus Lange</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Severin</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bechthold</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Heide</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Regulatory role of microsomal 3-Hydroxy-3-Methylglutaryl-coenzyme A reductase for shikonin biosynthesis in <i>Lithospermum erythrorhizon</i> cell suspension cultures </span> <span class="source" tagx="source" title="source">Planta</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">204</span> <span class="fpage" tagx="fpage" title="fpage">234</span> <span class="lpage" tagx="lpage" title="lpage">241</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s004250050252">10.1007/s004250050252</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9487727">9487727</a> </span> </span> </li> <li tag="ref"> <a name="CR49" /> <span class="label" tagx="label" title="label">49.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Danpure</span> <span class="given-names" tagx="given-names" title="given-names">CJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">How can the products of a single gene be localized to more than one intracellular compartment? </span> <span class="source" tagx="source" title="source">Trends in Cell Biol</span> <span class="year" tagx="year" title="year">1995</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">230</span> <span class="lpage" tagx="lpage" title="lpage">238</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0962-8924(00)89016-9">10.1016/S0962-8924(00)89016-9</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14732127">14732127</a> </span> </span> </li> <li tag="ref"> <a name="CR50" /> <span class="label" tagx="label" title="label">50.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aquil</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Husaini</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Abdin</span> <span class="given-names" tagx="given-names" title="given-names">MZ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rather</span> <span class="given-names" tagx="given-names" title="given-names">GM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Overexpression of the HMG-CoA reductase gene leads to enhanced artemisinin biosynthesis in transgenic <i>Artemisia annua</i> plants </span> <span class="source" tagx="source" title="source">Planta Med.</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">75</span> <span class="fpage" tagx="fpage" title="fpage">1453</span> <span class="lpage" tagx="lpage" title="lpage">1458</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1055/s-0029-1185775">10.1055/s-0029-1185775</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19551613">19551613</a> </span> </span> </li> <li tag="ref"> <a name="CR51" /> <span class="label" tagx="label" title="label">51.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chaurasiya</span> <span class="given-names" tagx="given-names" title="given-names">ND</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sangwan</span> <span class="given-names" tagx="given-names" title="given-names">NS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sabir</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Misra</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sangwan</span> <span class="given-names" tagx="given-names" title="given-names">RS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Withanolide biosynthesis recruits both mevalonate and DOXP pathways of isoprenogenesis in Ashwagandha <i>Withania somnifera</i> L.(Dunal) </span> <span class="source" tagx="source" title="source">Plant Cell Rep</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">31</span> <span class="fpage" tagx="fpage" title="fpage">1889</span> <span class="lpage" tagx="lpage" title="lpage">1897</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00299-012-1302-4">10.1007/s00299-012-1302-4</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22733207">22733207</a> </span> </span> </li> <li tag="ref"> <a name="CR52" /> <span class="label" tagx="label" title="label">52.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">Y-J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">OR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oh</span> <span class="given-names" tagx="given-names" title="given-names">JY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jang</span> <span class="given-names" tagx="given-names" title="given-names">M-G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">D-C</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Functional analysis of HMGR encoding genes in triterpene saponin-producing <i>Panax ginseng</i> Meyer </span> <span class="source" tagx="source" title="source">Plant Physiol.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">165</span> <span class="fpage" tagx="fpage" title="fpage">373</span> <span class="lpage" tagx="lpage" title="lpage">387</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.113.222596">10.1104/pp.113.222596</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24569845">24569845</a> </span> </span> </li> <li tag="ref"> <a name="CR53" /> <span class="label" tagx="label" title="label">53.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rajakani</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Narnoliya</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sangwan</span> <span class="given-names" tagx="given-names" title="given-names">NS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sangwan</span> <span class="given-names" tagx="given-names" title="given-names">RS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gupta</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Activated charcoal-mediated RNA extraction method for <i>Azadirachta indica</i> and plants highly rich in polyphenolics, polysaccharides and other complex secondary compounds </span> <span class="source" tagx="source" title="source">BMC Res. Notes</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">125</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1756-0500-6-125">10.1186/1756-0500-6-125</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23537338">23537338</a> </span> </span> </li> <li tag="ref"> <a name="CR54" /> <span class="label" tagx="label" title="label">54.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tamura</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stecher</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peterson</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Filipski</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">MEGA6: Molecular evolutionary genetics analysis version 6.0</span> <span class="source" tagx="source" title="source">Mol. Biol. Evol</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">30</span> <span class="fpage" tagx="fpage" title="fpage">2725</span> <span class="lpage" tagx="lpage" title="lpage">2729</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/molbev/mst197">10.1093/molbev/mst197</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24132122">24132122</a> </span> </span> </li> <li tag="ref"> <a name="CR55" /> <span class="label" tagx="label" title="label">55.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Bailey, T. L. &amp;amp; Elkan, C. Fitting a Mixture model by expectation maximization to discover motifs in bipolymers. <i>Proc. Second Int. Conf. Intell. Syst. Mol. Biol</i>. 28–36 (1994). </span> </li> <li tag="ref"> <a name="CR56" /> <span class="label" tagx="label" title="label">56.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Kushwaha, A. K. <i>et al</i>. Tropine forming tropinone reductase gene from <i>Withania somnifera</i> (Ashwagandha): Biochemical characteristics of the recombinant enzyme and novel physiological overtones of tissue-wide gene expression patterns. <i>PLoS One</i> <b>8</b> (2013). </span> </li> <li tag="ref"> <a name="CR57" /> <span class="label" tagx="label" title="label">57.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Chen, H., Nelson, R. S. &amp;amp; Sherwood, J. L. Enhanced recovery of transformants of <i>Agrobacterium tumefaciens</i> after freeze-thaw transformation and drug selection. <i>Biotechniques</i> <b>16</b> (1994). </span> </li> <li tag="ref"> <a name="CR58" /> <span class="label" tagx="label" title="label">58.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bose</span> <span class="given-names" tagx="given-names" title="given-names">SK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yadav</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sangwan</span> <span class="given-names" tagx="given-names" title="given-names">RS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singh</span> <span class="given-names" tagx="given-names" title="given-names">AK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mishra</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Srivastava</span> <span class="given-names" tagx="given-names" title="given-names">AK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sangwan</span> <span class="given-names" tagx="given-names" title="given-names">NS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Effect of gibberellic acid and calliterpenone on plant growth attributes, trichomes, essential oil biosynthesis and pathway gene expression in differential manner in <i>Mentha arvensis</i> L </span> <span class="source" tagx="source" title="source">Plant Physiol. Biochem.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">66</span> <span class="fpage" tagx="fpage" title="fpage">150</span> <span class="lpage" tagx="lpage" title="lpage">158</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.plaphy.2013.02.011">10.1016/j.plaphy.2013.02.011</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23514759">23514759</a> </span> </span> </li> <li tag="ref"> <a name="CR59" /> <span class="label" tagx="label" title="label">59.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sangwan</span> <span class="given-names" tagx="given-names" title="given-names">RS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chaurasiya</span> <span class="given-names" tagx="given-names" title="given-names">ND</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Misra</span> <span class="given-names" tagx="given-names" title="given-names">LN</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lal</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Uniyal</span> <span class="given-names" tagx="given-names" title="given-names">GC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharma</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sangwan</span> <span class="given-names" tagx="given-names" title="given-names">NS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suri</span> <span class="given-names" tagx="given-names" title="given-names">KA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qazi</span> <span class="given-names" tagx="given-names" title="given-names">GN</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuli</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Phytochemical variability in commercial herbal products and preparations of <i>Withania somnifera</i> (Ashwagandha) </span> <span class="source" tagx="source" title="source">Curr. Sci.</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">86</span> <span class="fpage" tagx="fpage" title="fpage">461</span> <span class="lpage" tagx="lpage" title="lpage">465</span> </span> </li> <li tag="ref"> <a name="CR60" /> <span class="label" tagx="label" title="label">60.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sabir</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mishra</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sangwan</span> <span class="given-names" tagx="given-names" title="given-names">RS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jadaun</span> <span class="given-names" tagx="given-names" title="given-names">JS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sangwan</span> <span class="given-names" tagx="given-names" title="given-names">NS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Qualitative and quantitative variations in withanolides and expression of some pathway genes during different stages of morphogenesis in <i>Withania somnifera</i> Dunal </span> <span class="source" tagx="source" title="source">Protoplasma</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">250</span> <span class="fpage" tagx="fpage" title="fpage">539</span> <span class="lpage" tagx="lpage" title="lpage">549</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00709-012-0438-y">10.1007/s00709-012-0438-y</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22878597">22878597</a> </span> </span> </li> </ul> </div> </div>  </body></html>