<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">PLoS One</span> <span class="iso-abbrev" title="iso-abbrev">PLoS ONE</span> <span class="publisher-id" title="publisher-id">plos</span> <span class="pmc" title="pmc">plosone</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">PLoS ONE</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1932-6203</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA USA</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5973616">5973616</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pone.0198030">10.1371/journal.pone.0198030</a> </span> <span class="publisher-id" title="publisher-id">PONE-D-18-02247</span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Physical Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Chemistry</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Phytochemistry</span>: <span class="subject" title="subject">Phytochemicals</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biochemistry</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Biochemistry</span>: <span class="subject" title="subject">Phytochemicals</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Science</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Biochemistry</span>: <span class="subject" title="subject">Phytochemicals</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Eukaryota</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plants</span>: <span class="subject" title="subject">Herbs</span> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Endocrinology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Endocrine Disorders</span>: <span class="subject" title="subject">Diabetes Mellitus</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Metabolic Disorders</span>: <span class="subject" title="subject">Diabetes Mellitus</span> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Eukaryota</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plants</span>: <span class="subject" title="subject">Garlic</span> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computer and Information Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Information Technology</span>: <span class="subject" title="subject">Text Mining</span> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biochemistry</span>: <span class="subject" title="subject">Antioxidants</span> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computer and Information Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Information Technology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Natural Language Processing</span>: <span class="subject" title="subject">Named Entity Recognition</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computer and Information Sciences</span>: <span class="subject" title="subject">Neural Networks</span> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Neuroscience</span>: <span class="subject" title="subject">Neural Networks</span> </div> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Data-driven analysis of biomedical literature suggests broad-spectrum benefits of culinary herbs and spices </div> <div class="alt-title" title="alt-title">
Broad-spectrum benefits of culinary herbs and spices</div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rakhi</span> <span class="given-names" tagx="given-names" title="given-names">N. K.</span> </span> <span class="role" tagx="role" title="role">Data curation</span> <span class="role" tagx="role" title="role">Formal analysis</span> <span class="role" tagx="role" title="role">Investigation</span> <span class="role" tagx="role" title="role">Methodology</span> <span class="role" tagx="role" title="role">Visualization</span> <span class="role" tagx="role" title="role">Writing – original draft</span> <span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span> <a href="#aff001"> <sup>1</sup> </a> <a href="#aff002"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuwani</span> <span class="given-names" tagx="given-names" title="given-names">Rudraksh</span> </span> <span class="role" tagx="role" title="role">Formal analysis</span> <span class="role" tagx="role" title="role">Investigation</span> <span class="role" tagx="role" title="role">Methodology</span> <span class="role" tagx="role" title="role">Validation</span> <span class="role" tagx="role" title="role">Visualization</span> <span class="role" tagx="role" title="role">Writing – original draft</span> <span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span> <a href="#aff001"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mukherjee</span> <span class="given-names" tagx="given-names" title="given-names">Jagriti</span> </span> <span class="role" tagx="role" title="role">Data curation</span> <a href="#aff001"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-1924-6070</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bagler</span> <span class="given-names" tagx="given-names" title="given-names">Ganesh</span> </span> <span class="role" tagx="role" title="role">Conceptualization</span> <span class="role" tagx="role" title="role">Data curation</span> <span class="role" tagx="role" title="role">Formal analysis</span> <span class="role" tagx="role" title="role">Investigation</span> <span class="role" tagx="role" title="role">Methodology</span> <span class="role" tagx="role" title="role">Project administration</span> <span class="role" tagx="role" title="role">Resources</span> <span class="role" tagx="role" title="role">Supervision</span> <span class="role" tagx="role" title="role">Validation</span> <span class="role" tagx="role" title="role">Visualization</span> <span class="role" tagx="role" title="role">Writing – original draft</span> <span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span> <a href="#aff001"> <sup>1</sup> </a> <a href="#cor001">*</a> </span> </div> <span class="citation_author_institution" id="aff001">[1], <span class="addr-line" title="addr-line">Center for Computational Biology, Indraprastha Institute of Information Technology (IIIT-Delhi), New Delhi, India </span> </span> <span class="citation_author_institution" id="aff002">[2], <span class="addr-line" title="addr-line">Department of Bioscience and Bioengineering, Indian Institute of Technology Jodhpur, Jodhpur, India </span> </span> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Narayan</span> <span class="given-names" tagx="given-names" title="given-names">Mahesh</span> </span> <span class="role" tagx="role" title="role">Editor</span> <a href="#edit1" /> </span> </div> <span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">The University of Texas at El Paso, UNITED STATES</span> </span> <div class="author-notes" title="author-notes"> <div class="fn-type-COI-statement" title="COI-statement"> <p> <b>Competing Interests: </b>The authors have declared that no competing interests exist. </p> </div> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">ganesh.bagler@gmail.com</span>, <span class="email" tagx="email" title="email">bagler@iiitd.ac.in</span> </div> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-5-5</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2018</span> </span> <span class="volume" tagx="volume" title="volume">13</span> <span class="issue" tagx="issue" title="issue">5</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">e0198030</span> <span class="history" title="history"> <span class="received" title="received">received: 2018-1-22</span> <span class="accepted" title="accepted">accepted: 2018-5-11</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Rakhi et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2018</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited. </span> </span> </div> <a class="self-uri" href="pone.0198030.pdf" title="self-uri">pone.0198030.pdf</a> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Spices and herbs are key dietary ingredients used across cultures worldwide. Beyond their use as flavoring and coloring agents, the popularity of these aromatic plant products in culinary preparations has been attributed to their antimicrobial properties. Last few decades have witnessed an exponential growth of biomedical literature investigating the impact of spices and herbs on health, presenting an opportunity to mine for patterns from empirical evidence. Systematic investigation of empirical evidence to enumerate the health consequences of culinary herbs and spices can provide valuable insights into their therapeutic utility. We implemented a text mining protocol to assess the health impact of spices by assimilating, both, their positive and negative effects. We conclude that spices show broad-spectrum benevolence across a range of disease categories in contrast to negative effects that are comparatively narrow-spectrum. We also implement a strategy for disease-specific culinary recommendations of spices based on their therapeutic tradeoff against adverse effects. Further by integrating spice-phytochemical-disease associations, we identify bioactive spice phytochemicals potentially involved in their therapeutic effects. Our study provides a systems perspective on health effects of culinary spices and herbs with applications for dietary recommendations as well as identification of phytochemicals potentially involved in underlying molecular mechanisms. </p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="funding-statement" title="funding-statement">
This work was supported by a senior research fellowship from the Ministry of Human Resource Development, Government of India and Indian Institute of Technology Jodhpur to NKR. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript. </div> </div> <div class="counts" title="counts"> <span class="fig-count" title="fig-count">fig-count: </span> <span class="table-count" title="table-count">table-count: </span> <span class="page-count" title="page-count">page-count: </span> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Data Availability" value="All relevant data are within the paper and its Supporting Information files." /> </div> </div> <div class="title" tagx="title" title="title">
Data Availability</div> <p>All relevant data are within the paper and its Supporting Information files.</p> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Culinary practices across cultures around the world have evolved to incorporate spices and herbs in them. The potential utility of these aromatic plant products in recipes has received a lot of attention leading to multiple rationales for their wide-spread use in food preparations [ <a href="#pone.0198030.ref001">1</a>, <a href="#pone.0198030.ref002">2</a>]. Apart from their use as flavoring agents, spices have been suggested to be of value for their ability to inhibit or kill food-spoilage microorganisms [ <a href="#pone.0198030.ref002">2</a>]. Beyond their antimicrobial properties, the diverse therapeutic values of spices have been highlighted through <i>in vivo</i> and <i>in vitro</i> studies. Spices have been reported to possess therapeutic potential for their hypolipidemic [ <a href="#pone.0198030.ref003">3</a>], anti-diabetic [ <a href="#pone.0198030.ref004">4</a>], anti-lithogenic [ <a href="#pone.0198030.ref005">5</a>], antioxidant [ <a href="#pone.0198030.ref006">6</a>], anti-inflammatory and anticarcinogenic [ <a href="#pone.0198030.ref007">7</a>] activity. </p> <p>Scientific investigations into the health effects of spices have resulted in a large body of biomedical literature mentioning their direct or indirect connections to health and diseases. With focus on a specific spice/herb, such studies have discussed their health consequences to report heterogeneous results. While some of the surveys have attempted to collate and summarize this knowledge [ <a href="#pone.0198030.ref003">3</a>, <a href="#pone.0198030.ref006">6</a>, <a href="#pone.0198030.ref008">8</a>], a comprehensive picture of health impacts of culinary herbs and spices based on empirical evidence still evades us. Data from MEDLINE suggests an exponential increase in scientific reports associating culinary spices and herbs with diseases since 1990’s. Given their importance in food preparations, it is imperative to systematically investigate these empirical data to investigate health consequences of culinary herbs and spices. </p> <p>Beyond their culinary use, traditional medicinal systems have also advocated the role of spices as therapeutic agents [ <a href="#pone.0198030.ref008">8</a>, <a href="#pone.0198030.ref009">9</a>]. Apart from obtaining a coherent picture of the impact of these exceptional culinary ingredients on health, it would also be of value to probe the molecular mechanisms behind their action which remain largely unknown. A framework that integrates data on spice-disease associations and their phytochemicals to explore their underlying connections will help unravel molecular mechanisms behind the health impact of culinary spices and herbs ( <a href="#pone.0198030.g001">Fig 1</a>). Towards this end, we set out to find associations between spices and diseases from biomedical abstracts available from MEDLINE using a text mining approach. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0198030.g001"> <span class="label" tagx="label" title="label">Fig 1</span> </a> </div>   <div class="title" tagx="title" title="title">
Workflow implemented for data-driven analysis of biomedical literature associating culinary spices and herbs to diseases. </div> <p>Starting with compilation of an exhaustive dictionary of culinary spices and herbs, towards identification of spice-disease associations, one thread of investigation involved implementation of a computational protocol for text mining of biomedical literature including named entity recognition of herbs/spices as well as diseases, pre-processing, extraction of candidate sentences, manual annotations followed by predictions of associations with a machine learning based model. The other thread involved identification of bioactive spice phytochemicals and linking them to diseases. By integrating tripartite information of spices-phytochemicals-diseases, this study establishes the broad-spectrum benevolence of spices, suggests ways for their disease-specific culinary recommendations and probes potential molecular mechanisms underlying their therapeutic properties. Thus it provides a systems perspective to health effects of spices with potential culinary and medicinal applications. </p>   </div> <p>One of the earliest attempts in linking diet and diseases from literature was by Swanson who suggested the utility of dietary fish oil for the treatment of Reynaud’s syndrome from indirect associations manually inferred from literature survey [ <a href="#pone.0198030.ref010">10</a>]. Biomedical literature has expanded by many folds since this pilot study making it impossible to manually concatenate the information available from research articles to infer relationships between different entities or to formulate a hypothesis. Computational approaches to text mining and natural language processing are potent tools in this pursuit [ <a href="#pone.0198030.ref011">11</a>] and many studies in recent years have contributed to efforts in this direction [ <a href="#pone.0198030.ref012">12</a>– <a href="#pone.0198030.ref015">15</a>]. NutriChem [ <a href="#pone.0198030.ref015">15</a>, <a href="#pone.0198030.ref016">16</a>] database relates plant-based foods, their phytochemicals, and diseases by using a text mining approach. HerDing [ <a href="#pone.0198030.ref017">17</a>] is another resource which links herbs to diseases by indirectly connecting constituent chemicals of the former to genes associated with the latter. </p> <p>We investigated the impact of culinary spices and herbs for their role as regulators of health by text mining biomedical literature to assimilate, both, positive and negative associations. We observed that in general, the benevolent effects of spices span a broader spectrum of disorders than their adverse effects. Thus by exhaustively integrating evidence for beneficial and harmful effects of spices, we provide a framework for identification of spices whose benefits far outweigh their harms. We also suggest ways for their informed culinary use as well as for identification of phytochemicals with potential therapeutic value. In summary, our study offers a systems perspective of health effects of spices and herbs to provide informed culinary recommendations and insights into underlying molecular mechanisms behind their therapeutic utility. </p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="protocolforintegrationofspice-phytochemical-diseasedata" title="sec"> <div class="title" tagx="title" title="title">
Protocol for integration of spice-phytochemical-disease data</div> <p>We text mined spice-disease associations from abstracts available in MEDLINE, the largest database of biomedical literature containing more than 28 million references to research articles in biomedicine. First, a comprehensive dictionary of 188 species of culinary spices and herbs was manually compiled from various sources such as FooDB ( <a href="http://foodb.ca/">http://foodb.ca</a>), Wikipedia ( <a href="https://en.wikipedia.org/wiki/List_of_culinary_herbs_and_spices">https://en.wikipedia.org/wiki/List_of_culinary_herbs_and_spices</a>), PFAF (Plants For A Future, <a href="http://www.pfaf.org/user/Default.aspx">http://www.pfaf.org/user/Default.aspx</a>), FPI (Food Plants International, <a href="http://foodplantsinternational.com/">http://foodplantsinternational.com</a>) and FlavorDB [ <a href="#pone.0198030.ref018">18</a>] ( <a href="http://cosylab.iiitd.edu.in/flavordb">http://cosylab.iiitd.edu.in/flavordb</a>). This dictionary was then used to retrieve relevant abstracts from MEDLINE database. We then carried out Named Entity Recognition (NER) and normalization of spice and disease entities using a dictionary matching approach for the former and NCBI’s TaggerOne [ <a href="#pone.0198030.ref019">19</a>] tool the latter. For extracting relations, we only considered sentences that mention at least one spice/herb and disease and manually labeled a subset of these for positive, negative and neutral associations between the spice-disease pairs. The labeled sentences were then used to train a machine learning classifier to categorize the associations between the spice-disease pairs in the unlabeled sentences. To further probe putative molecular mechanisms for benevolent effects of spices, we identified spice phytochemicals from PhenolExplorer [ <a href="#pone.0198030.ref020">20</a>] and KNApSAcK [ <a href="#pone.0198030.ref021">21</a>] and found their therapeutic associations with diseases using Comparative Toxicogenomic Database [ <a href="#pone.0198030.ref022">22</a>] (CTD). <a href="#pone.0198030.g001">Fig 1</a> depicts the computational framework implemented for integrating and extracting tripartite spice-phytochemical-disease associations. These information are made available through an interactive resource, SpiceRx [ <a href="#pone.0198030.ref023">23</a>]. </p> </div> <div class="spicesdiseaseassociations" title="sec"> <div class="title" tagx="title" title="title">
Spices disease associations</div> <p>By combining manually annotated and predicted associations, we obtained a total of 8957 spice-disease associations from 5769 abstracts. Among these 8172 were positive spice-disease associations and 783 were negative. Out of 188 spices present in the dictionary, we obtained associations for 152 spices linking them to 848 unique disease-specific MeSH [ <a href="#pone.0198030.ref024">24</a>] (Medical Subject Headings) IDs ( <a href="#pone.0198030.s007">S1 Dataset</a>). We used a Convolutional Neural Network (CNN) classifier with word, position, part of speech and chunk embedding [ <a href="#pone.0198030.ref025">25</a>– <a href="#pone.0198030.ref027">27</a>] to predict positive, negative or neutral association in a spice-disease pair. It was evaluated on an external test set and found to have an accuracy of 86.7% and macro-averaged precision, recall and F1 score of 90.7%, 80% and 84.2% respectively. The class-wise performance metrics for the model are provided in <a href="#pone.0198030.t001">Table 1</a>. </p> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0198030.t001">10.1371/journal.pone.0198030.t001</a> <span class="label" tagx="label" title="label">Table 1</span>  <div class="title" tagx="title" title="title">
The class-wise performance metrics for the best CNN model, implementing word, position, part of speech and chunk embedding features, used for spice-disease relationship extraction. </div> <p>All negative associations were cleaned manually.</p>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th>Class</th> <th>Precision</th> <th>Recall</th> <th>F1-Score</th> </tr> </thead> <tbody> <tr> <td>No-Association</td> <td>88.06%</td> <td>89.83%</td> <td>88.9%</td> </tr> <tr> <td>Negative</td> <td>1 <a href="#sec009">*</a> </td> <td>65.96%</td> <td>79.49%</td> </tr> <tr> <td>Positive</td> <td>83.98%</td> <td>84.32%</td> <td>84.15%</td> </tr> <tr> <td>Macro averaged</td> <td>90.68%</td> <td>80.03%</td> <td>84.20%</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>*cleaned manually.</p> </div> </div> </div> <p>Disease entities were recognized and normalized to their corresponding MeSH IDs using TaggerOne [ <a href="#pone.0198030.ref019">19</a>]. MeSH [ <a href="#pone.0198030.ref024">24</a>] is a controlled vocabulary of biomedical terms curated and developed by National Library of Medicine. It organizes terms hierarchically from general to more specific ( <a href="#pone.0198030.s001">S1 Fig</a>). In this hierarchical structure, a spice may have associations with a disease at multiple levels of specificity. For example, Endocrine System Diseases (C19) present at the first level of MeSH hierarchy constitutes disease sub-categories such as Adrenal Gland Diseases (C19.053), Diabetes Mellitus (C19.246) at the second level. Further, specific types of Diabetes Mellitus such as ‘Diabetes Mellitus, Type 1 (C19.246.267)’, ‘Diabetes Mellitus, Type 2 (C19.246.300)’ appear at the third level. To conduct a multi-level analysis, we associated spices with disease terms at three levels of MeSH hierarchy labeled as ‘category’, ‘sub-category’ and a ‘disease’. </p> <p>We observed an exponential increase in articles reporting therapeutic properties of spices after 1995 ( <a href="#pone.0198030.g002">Fig 2</a>), with the number of abstracts reporting positive associations of spices with diseases far out-numbering those reporting negative associations ( <a href="#pone.0198030.g003">Fig 3</a>). A large number of spices such as ginger ( <i>Zingiber officinale)</i> and turmeric ( <i>Curcuma longa)</i> have very few negative associations reported in MEDLINE whereas a few others like liquorice ( <i>Glycyrrhiza glabra</i>) and celery ( <i>Apium graveolens)</i>, have almost an equal number of abstracts reporting positive and negative associations. The complete list of associations for spices is provided in <a href="#pone.0198030.s008">S2 Dataset</a>. These data suggest that, in general, beneficial effects of spices have been reported more widely than their adverse effects in biomedical literature. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0198030.g002"> <span class="label" tagx="label" title="label">Fig 2</span> </a> </div>   <div class="title" tagx="title" title="title">
Statistics of spice-disease associations.</div> <p>Historical trend in biomedical literature reporting spice-disease associations. There is an exponential increase in articles reporting the therapeutic effects of spices in last few decades. Data of research articles archived in MEDLINE till July 2017 is represented in the illustration. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0198030.g003"> <span class="label" tagx="label" title="label">Fig 3</span> </a> </div>   <div class="title" tagx="title" title="title">
Statistics of positive and negative disease associations for the top 50 spices with most number of associations. </div> <p>Notice that certain spices like liquorice ( <i>Glycyrrhiza glabra</i>) and celery <i>(Apium graveolens</i>) had equal number of positive as well as negative associations. The bias in number of associations may also indicate the inherent biases in scientific literature suggesting that certain spices are studied more than others. </p>   </div> <p>On analyzing individual diseases (third level of MeSH hierarchy) associated with spices, we found that diabetes mellitus, inflammation, and carcinogenesis have the highest number of positive associations ( <a href="#pone.0198030.g004">Fig 4</a>) ( <a href="#pone.0198030.s009">S3 Dataset</a>). Spices were also shown to have a preventive role in various cancers including breast, colorectal, prostatic and liver neoplasms. Among the diseases adversely affected by spices were hypersensitivity, dermatitis, rhinitis, hypertension and allergic rhinitis, ( <a href="#pone.0198030.g005">Fig 5</a>) ( <a href="#pone.0198030.s009">S3 Dataset</a>). It is worth noting that majority of these diseases are autoimmune in nature and are subjective to certain individuals sensitive to that spice. In such cases, spices may act as triggering factors rather than causal agents. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0198030.g004"> <span class="label" tagx="label" title="label">Fig 4</span> </a> </div>   <div class="title" tagx="title" title="title">
Top diseases (Third level of MeSH hierarchy) ranked according to their total number of positive associations. </div> <p>Numbers shown against the bars indicate the ‘number of spices’ involved in the associations. The number of positive disease associations for spices outnumber the number of negative associations ( <a href="#pone.0198030.g005">Fig 5</a>) indicating that spices, in general, have been reported with beneficial health effects. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0198030.g005"> <span class="label" tagx="label" title="label">Fig 5</span> </a> </div>   <div class="title" tagx="title" title="title">
Top diseases (Third level of MeSH hierarchy) ranked according to their total number of negative associations. </div> <p>The numbers mentioned on the bars indicate the number of spices associated negatively with each disease. </p>   </div> </div> <div class="broad-spectrumbenevolenceofherbs/spices" title="sec"> <div class="title" tagx="title" title="title">
Broad-spectrum benevolence of herbs/spices</div> <p>To probe for the effects of spice/herb across a spectrum of disorders, we analyzed its associations with disease ‘sub-categories’ at the second level of MeSH hierarchy ( <a href="#pone.0198030.s001">S1 Fig</a>). Analyzing associations at this level provides a balance between specificity and generality of disease terms. Among the disease sub-categories positively associated with spices, pathologic processes, signs and symptoms, metabolic diseases, diabetes mellitus, vascular diseases as well as central nervous system diseases were found to be dominant ( <a href="#pone.0198030.g006">Fig 6</a>), <a href="#pone.0198030.s010">S4 Dataset</a>). Top disease categories which were negatively associated with spices included vascular diseases, skin diseases, hypersensitivity and respiratory hypersensitivity ( <a href="#pone.0198030.g007">Fig 7</a>), <a href="#pone.0198030.s010">S4 Dataset</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0198030.g006"> <span class="label" tagx="label" title="label">Fig 6</span> </a> </div>   <div class="title" tagx="title" title="title">
Disease categories (First level of MeSH hierarchy) ranked according to the number of positive associations with spices. </div> <p>Numbers shown against the bars indicate the ‘number of spices’ linked with each of the associations. The number of positive disease category associations for spices outnumber those with negative associations ( <a href="#pone.0198030.g007">Fig 7</a>) further confirming the benevolent health effects of spices. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0198030.g007"> <span class="label" tagx="label" title="label">Fig 7</span> </a> </div>   <div class="title" tagx="title" title="title">
Disease categories (First level of MeSH hierarchy) ranked according to the number of negative associations with spices. </div> <p>Numbers shown against the bars indicate the ‘number of spices’ linked with each of the associations. </p>   </div> <p>To quantify the broad impact a spice may have across diverse disease categories as well as sub-categories, we devised a ‘spectrum score ( <i>Ω</i> <sub> <i>s</i> </sub>)’. This metric computes the sum of proportion of disease terms associated with a spice at the second level of MeSH hierarchy (sub-categories), multiplied by the number of disease terms associated at the first level (categories). (See <a href="#sec009">Materials and Methods</a>). With 27 disease categories, the lower and upper bound for the spectrum score is 0 and 729 respectively. To elucidate further, let us consider a spice that is associated with all diseases in exactly half of the MeSH disease categories versus another spice that has associations with half of the diseases in every disease category. In such a case, the latter would have a higher spectrum score than the former. We computed the spectrum score for both positive (benevolence spectrum score, <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
Ω</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
+</div> </div> </div> </div> </div> ) as well as negative associations (adverse spectrum score, <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
Ω</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> </div> </div> </div> </div> </span>). <p /> <p>The spices with highest ‘benevolence spectrum score’ according to our analysis were garlic ( <i>Allium sativum</i>), ginger ( <i>Zingiber officinale</i>), turmeric ( <i>Curcuma longa</i>), liquorice ( <i>Glycyrrhiza glabra</i>), ginkgo ( <i>Ginkgo biloba</i>), black cumin ( <i>Nigella sativa</i>), cinnamon ( <i>Cinnamomum verum</i>) and saffron ( <i>Crocus sativus</i>) whereas the top adverse spectrum spices were liquorice <i>(Glycyrrhiza glabra)</i>, ginger <i>(Zingiber officinale)</i>, fenugreek <i>(Trigonella foenum-graecum)</i>, ginkgo <i>(Ginkgo biloba)</i>, sunflower <i>(Helianthus annuus)</i> and <i>Celery (Apium graveolens)</i>. Spices such as garlic, liquorice and ginkgo have a high benevolence as well as adverse spectrum scores. </p> <p>We found that for 150 out of 152 spices, the ‘benevolence spectrum score’ exceeded the ‘adverse spectrum score’, with almost 50 spices having ‘relative benevolence’ ( <i>ΔΩ</i> <sub> <i>s</i> </sub>) greater than 50 ( <a href="#pone.0198030.g008">Fig 8</a>). Hence, it may be concluded that in general spices have positive effects with a broad spectrum of diseases in contrast to their negative effects which are comparatively narrow-spectrum. In line with our analysis, spices have been reported to be effective against a range of disorders [ <a href="#pone.0198030.ref003">3</a>– <a href="#pone.0198030.ref005">5</a>, <a href="#pone.0198030.ref007">7</a>]. Details of benevolent, adverse as well as relative benevolence scores for all spices are provided in the <a href="#pone.0198030.s011">S5 Dataset</a>. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0198030.g008"> <span class="label" tagx="label" title="label">Fig 8</span> </a> </div>   <div class="title" tagx="title" title="title">
Spices ranked according to their ‘relative benevolence score’ highlighting their broad-spectrum benevolence. </div> <p>This score enumerates the relative health benefits as reflected in the difference between ‘benevolence spectrum’ and ‘adverse spectrum’ scores. Barring two, all spices had positive scores with a large number of them showing significantly larger therapeutic effects compared to their adverse effects. </p>   </div> </div> <div class="culinaryrecommendations" title="sec"> <div class="title" tagx="title" title="title">
Culinary recommendations</div> <p>Each of the MeSH disease categories refers to a class of disorders such as nutritional and metabolic disorders, cardiovascular diseases, nervous systems diseases, digestive system diseases, immune system diseases, neoplasms, bacterial infections and mycoses, virus diseases and such. The spectrum score forms the basis to prioritize spices for culinary intervention against a MeSH disease category. We computed the category-specific ‘benevolence spectrum’ and ‘adverse spectrum’ scores to enumerate the ‘trade-off score’ that represents the therapeutic value of a spice against a class of disorders. <a href="#pone.0198030.s012">S6 Dataset</a> provides a list of culinary recommendations intended as a dietary intervention against various disease categories. </p> <p>There is ample amount of empirical evidence for the recommendations provided by our study. Our data suggest that spices show therapeutic effects against most of the viral diseases. Among them, turmeric ( <i>Curcuma longa</i>) is the most broad-spectrum antiviral spice and is reported with inhibitory properties against various viruses including HIV, influenza, and coxsackievirus [ <a href="#pone.0198030.ref028">28</a>]. Studies in human and animal models have shown that dietary spices significantly stimulate the activities of digestive enzymes of the pancreas and small intestines such as pancreatic lipase, amylase and proteases thereby acting as digestive stimulants. Spices like ginger and garlic stimulate TRPV1, a sensor in the digestive system which has implications for gastrointestinal tract pathology and physiology [ <a href="#pone.0198030.ref029">29</a>, <a href="#pone.0198030.ref030">30</a>]. Prominent spices recommended for cardiovascular diseases, such as tulsi ( <i>Ocimum tenuiflorum</i>), mint ( <i>Mentha X piperita</i>), ginkgo ( <i>Ginkgo biloba</i>) and ginger ( <i>Zingiber officinale</i>), have been reported with beneficial effects against cardiovascular disorders. Epidemiological studies suggest that these spices lower cholesterol level, decrease platelet aggregation, reduce blood pressure, and increases antioxidant status which in turn decreases the progression of cardiovascular diseases [ <a href="#pone.0198030.ref031">31</a>]. Black cumin ( <i>Nigella sativa</i>), turmeric ( <i>Curcuma longa</i>) and garlic ( <i>Allium sativum</i>) are prominent spices recommended for diabetes, a major metabolic disorder. Evidence from animal studies and human trials have indicated that these spices modulate hyperglycemia and lipid profile function. Their antioxidant characteristics and effects on insulin secretion, glucose absorption, and gluconeogenesis make them potent candidates towards treating diabetes [ <a href="#pone.0198030.ref032">32</a>, <a href="#pone.0198030.ref033">33</a>]. Similarly, the anti-diabetic property of ginkgo ( <i>Ginkgo Biloba)</i> may be linked to the ability of its extract to reduce insulin resistance. </p> <p>Incidentally, the spices that frequent in the culinary recommendations are among those used for culinary and medicinal preparations across cultures. Curcumin ( <i>Curcuma longa</i>) and tulsi ( <i>Ocimum tenufloreum</i>), widely used in Indian culinary and medicinal preparations, were present across recommendations made throughout the spectrum of MeSH disease categories. Similarly garlic, used in Southern European especially Italian cuisine, also appeared in culinary recommendations across all categories of diseases. Some of the other most potent spices include ginger ( <i>Zingiber officinale</i>), black cumin ( <i>Nigella sativa</i>) and ginkgo ( <i>Ginkgo biloba</i>) (see <a href="#pone.0198030.s005">S1 Table</a>). </p> </div> <div class="linkingspicestodiseasesthroughphytochemicals" title="sec"> <div class="title" tagx="title" title="title">
Linking spices to diseases through phytochemicals</div> <p>Our analysis suggests that beyond their utility as flavoring, coloring, and food preserving (antimicrobial [ <a href="#pone.0198030.ref002">2</a>]) agents, spices may have been incorporated in traditional culinary practices due to their beneficial health effects across a spectrum of disorders. Given that the therapeutic properties of plants are mediated by their phytochemicals [ <a href="#pone.0198030.ref034">34</a>– <a href="#pone.0198030.ref036">36</a>] we hypothesize that the broad spectrum benevolence of spices can be attributed to the presence of bioactive phytochemicals such as polyphenols [ <a href="#pone.0198030.ref036">36</a>]. For example, curcumin, a polyphenol from turmeric is known to have a wide range of health benefits including antioxidant, anti-inflammatory, and anticancer effects [ <a href="#pone.0198030.ref037">37</a>]. Ajoene, a polyphenol compound derived from garlic, has been shown to induce apoptosis in leukemic cells [ <a href="#pone.0198030.ref038">38</a>]. Similarly, eugenol present in clove is reported to have antifungal property [ <a href="#pone.0198030.ref039">39</a>]. The antioxidant activity of black pepper has been attributed to the presence of β-caryophyllene, limonene, β-pinene, piperine and piperolein in its essential oil and oleoresins [ <a href="#pone.0198030.ref039">39</a>]. The anticancer properties of ginger are attributed to the presence of certain pungent vallinoids, gingerol, and paradol, as well as some other constituents like shogaols, zingerone, amongst others [ <a href="#pone.0198030.ref039">39</a>]. Going beyond the investigation of spice-disease associations, we linked spices to their constituent bioactive molecules and further connected them to diseases to obtain potential evidence of therapeutic associations ( <a href="#pone.0198030.g001">Fig 1</a>). </p> <p>We obtained 866 chemical compounds corresponding to 142 culinary spices in our dictionary from PhenolExplorer [ <a href="#pone.0198030.ref020">20</a>] and KNApSAcK [ <a href="#pone.0198030.ref021">21</a>], and consisted of 2042 spice-phytochemical associations. These data were filtered using PubChem [ <a href="#pone.0198030.ref040">40</a>] to keep only 570 bioactive phytochemicals, as they are known to react with tissues or cells. Further, we associated spice phytochemicals to diseases with the help of CTD [ <a href="#pone.0198030.ref022">22</a>], a public database of curated and inferred chemical-disease associations from the literature. CTD [ <a href="#pone.0198030.ref022">22</a>] classifies chemical-disease associations into therapeutic, inferred or marker associations. Therapeutic and marker associations are directly curated from the literature, whereas inferred relations are obtained from indirect associations. Therapeutic associations between a phytochemical and disease imply the presence of direct evidence of that phytochemical in alleviating the disease. For our further analysis, we focused only on 211 bioactive chemicals from the spices which were reported to have therapeutic associations. </p> <p>We integrated the data of spice-disease associations with spice-phytochemical and phytochemical-disease mappings. This tripartite data of spice-phytochemical-disease associations can form the basis for finding putative molecular mechanisms behind the beneficial effects of spices against diseases. Using data of curated phytochemical-disease associations from CTD, we found that out of 4380 positive spice–disease associations (where disease terms were mapped to third level of MeSH), 37% (1619) could be explained through evidence of phytochemical-disease associations. To elucidate, we found empirical evidences supporting anti-carcinogenic effects of garlic ( <i>Allium sativum</i>) against liver neoplasms. With the help of CTD [ <a href="#pone.0198030.ref022">22</a>], we found allyl sulfide, a compound in garlic, to be therapeutically associated with liver neoplasms. It can therefore be hypothesized that the anti-carcinogenic effects of garlic can be attributed to the presence of allyl sulfide. Incidentally, this hypothesis is independently supported by the literature [ <a href="#pone.0198030.ref041">41</a>]. The 63% spice-disease associations which could not be explained through evidence of phytochemical-disease relations may serve as hypotheses for unearthing the putative molecular mechanisms by utilizing the data of spice-phytochemical associations. </p> <p> <a href="#pone.0198030.s013">S7 Dataset</a> provides an exhaustive list of spice-disease associations and phytochemicals identified from the integration of tripartite data of diseases, spices and their phytochemicals ( <a href="#pone.0198030.g001">Fig 1</a>) and <a href="#pone.0198030.s014">S8 Dataset</a> provides the list of positive spice-disease associations for which no specific therapeutic phytochemical from a spice could be obtained. </p> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Humans are unique in having developed the ability to cook, which has been argued to be critical for the emergence of their large brains [ <a href="#pone.0198030.ref042">42</a>, <a href="#pone.0198030.ref043">43</a>]. While cooked food must have provided with much-needed energy supply, it is intriguing that they flavor the food with nutritionally insignificant quantities of herbs and spices. Going beyond the ability of spices to act as flavoring and antimicrobial agents [ <a href="#pone.0198030.ref002">2</a>], our analysis of spice-disease associations text-mined from biomedical literature shows the broad-spectrum benefits of spices. Recent studies have shown the potential benefits of consumption of spices such as chillies through cohort studies [ <a href="#pone.0198030.ref044">44</a>] as well as the role of specific spice phytochemicals in their health effects [ <a href="#pone.0198030.ref045">45</a>]. Interestingly, the broad-spectrum benevolence score of a spice was not positively correlated with its phytochemical repertoire ( <a href="#pone.0198030.s002">S2 Fig</a>) suggesting that richness in the phytochemical content itself does not explain its therapeutic value. </p> <p>We also point out negative health effects of spices, largely reflected in allergies, immune system, and skin-related disorders. Few of the negative effects of spices have been linked with their excessive use. For example, licorice, a beneficial herb for hypertension can cause weight loss, hypokalemia and other related adverse effects if consumed in large doses. Beyond probing the molecular basis of positive associations, it would also be of interest to identify toxic phytochemicals present in spices and assess their effect on specific diseases so as to provide an advisory against their consumption. Negative associations for spices projected by our study can serve as a basis for such investigations. </p> <p>As opposed to a previous attempt in this direction [ <a href="#pone.0198030.ref015">15</a>, <a href="#pone.0198030.ref016">16</a>] that linked all plant-based foods with diseases and phytochemicals from literature, our study focused on culinary spices and herbs. We investigated an exhaustive dictionary of 188 culinary herbs and spices with far better coverage (99 additional) than that of NutriChem [ <a href="#pone.0198030.ref015">15</a>, <a href="#pone.0198030.ref016">16</a>]. Overall, in terms of the number of disease associations, the depth of our analysis was better than that of NutriChem [ <a href="#pone.0198030.ref015">15</a>, <a href="#pone.0198030.ref016">16</a>] ( <a href="#pone.0198030.s003">S3 Fig</a>) and our data comprised a larger set of associations for most spices ( <a href="#pone.0198030.s004">S4 Fig</a>). NutriChem [ <a href="#pone.0198030.ref015">15</a>, <a href="#pone.0198030.ref016">16</a>] used dictionary based string matching approach for named entity recognition and normalization of diseases as well as plants. In case of diseases, it is empirically shown that depending on the disease dictionary used, the string matching approach typically leads to a low precision and recall [ <a href="#pone.0198030.ref046">46</a>]. We used TaggerOne [ <a href="#pone.0198030.ref019">19</a>], a machine learning based named entity recognition tool which yields state of the art performance. Even though the performance of our relationship extraction model was evaluated on a dataset consisting of positive, negative and neutral associations in contrast to previous studies which evaluated on only positive and negative associations, our model achieves a comparative F1 score. In addition to this, we provide an accurate information of adverse effect of spices by manually correcting all predicted negative associations. Despite our best efforts to ensure accurate extraction of spice-disease associations, our method is constrained by shortcomings inherent to text mining approaches and use of limited information pertaining to biomedical literature, namely, title and abstract. Overall, our analysis serves as a precursor to systematic reviews including meta-analysis as well as hypothesis-driven investigations into the health effects of spices and herbs. The data compiled as part of our study are made available through an interactive resource, SpiceRx [ <a href="#pone.0198030.ref023">23</a>]. </p> <p>Similar to languages where words are synthesized from the same phonetic repertoire, cuisines around the world have concocted their own unique ingredient combinations, especially those made from spices [ <a href="#pone.0198030.ref047">47</a>, <a href="#pone.0198030.ref048">48</a>]. Interestingly, many cuisines around the world such as those from the Indian subcontinent ( <i>paanch phoron</i>, <i>garam masala</i>, <i>sambar masala</i> among a host of others referred to as <i>masala</i>), Ethiopia ( <i>berbere</i>) and Middle East ( <i>baharat</i>) to mention a few, have ended up developing unique spice combinations of their own. It remains to be critically examined whether these have been deliberately composed with an appreciation of therapeutic properties of spices and herbs, or are accidentally emerged constructs. Spices are frequently used as part of functional foods, for example, the Indian dish <i>rasam</i> is a concoction of different spices and has been reported to be hypoglycemic, anti-anemic and antipyretic [ <a href="#pone.0198030.ref049">49</a>]. <i>Sambar</i>, another predominantly spice-based recipe has been shown to work against prostrate cancer [ <a href="#pone.0198030.ref050">50</a>]. Traditional medicinal systems are also known to recommend spices as part of their prescriptions. <i>Trikatu</i> [ <a href="#pone.0198030.ref051">51</a>], a spice concoction made with black pepper, long pepper, and dried ginger has been advised to be of value against rheumatoid arthritis by Ayurveda, a classical traditional medicinal system from India. In Chinese traditional medicine, <i>Xiaoyao-san</i>, a combination of various spices, has been recommended for management of stress and depression-related disorders [ <a href="#pone.0198030.ref052">52</a>]. </p> <p>Cooking typically involves high-temperature processing via heating, boiling, frying and such. It could be argued [ <a href="#pone.0198030.ref053">53</a>] that heating is a simpler and more effective means of killing microbes, thereby refuting the antimicrobial hypothesis [ <a href="#pone.0198030.ref002">2</a>]. Other beneficial effects of spices (such as anti-diabetic, anti-carcinogenic and antioxidant and inflammatory), unearthed in this study, could not be argued against with this logic. Ironically, this argument raises another critical question: Whether the therapeutic properties and bioactivity of spice phytochemicals can sustain the intense heating processes typically involved in cooking [ <a href="#pone.0198030.ref054">54</a>]? Besides that, one of the ambiguous factors in appreciating the benevolence of spices is the distinction between the effectiveness of individual compounds vis-à-vis their synergistic actions. Apart from these aspects, there is ample scope for improvising the strategy for culinary recommendations as well as for identifications of molecular mechanisms involved in health impact of spices by including the data of quantity and disease-specific potency of their constituent phytochemicals. While raising a host of such critical questions related to dietary intake of herbs and spices, by investigating evidence from biomedical literature reporting health effects of culinary herbs and spices our data-driven analysis suggests their broad-spectrum benevolence. </p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="compilationofspicesandherbsdictionary" title="sec"> <div class="title" tagx="title" title="title">
Compilation of spices and herbs dictionary</div> <p>We compiled a dictionary of 188 species of culinary spices and herbs. Scientific names and common names were obtained from Foodb ( <a href="http://foodb.ca/">http://foodb.ca/</a>) and Wikipedia ( <a href="https://en.wikipedia.org/wiki/List_of_culinary_herbs_and_spices">https://en.wikipedia.org/wiki/List_of_culinary_herbs_and_spices</a>). Varieties in scientific names, wherever available, were standardized to their respective species name. For example, <i>Capsicum baccatum var</i>. <i>pendulum</i>, the scientific name of Peruvian pepper, was standardized to <i>Capsicum baccatum</i>. All scientific names were then mapped to their respective NCBI Taxonomy IDs. This dictionary was further enriched by adding common names from FPI (Food Plants International, <a href="http://foodplantsinternational.com/plants/">http://foodplantsinternational.com/plants/</a>), NCBI Taxonomy ( <a href="https://www.ncbi.nlm.nih.gov/taxonomy">https://www.ncbi.nlm.nih.gov/taxonomy</a>) and PFAF (Plants for a Future, <a href="http://www.pfaf.org/">http://www.pfaf.org</a>). Singular and plural forms of common names of the spices and herbs were also included. Common names that did not exclusively map to an NCBI Taxonomy ID were removed. </p> </div> <div class="biomedicalliterature" title="sec"> <div class="title" tagx="title" title="title">
Biomedical literature</div> <p>We used MEDLINE (Medical Literature Analysis and Retrieval System Online, <a href="https://www.nlm.nih.gov/bsd/mms/medlineelements.html">https://www.nlm.nih.gov/bsd/mms/medlineelements.html</a>) as our source of biomedical literature. It includes citations from more than 5600 scholarly journals with over 24 million references to peer-reviewed biomedical and life science research articles from as early as 1946. The data was downloaded in bulk from the FTP server of NCBI ( <a href="https://www.nlm.nih.gov/databases/download/pubmed_medline.html">https://www.nlm.nih.gov/databases/download/pubmed_medline.html</a>). A modified version of PubMed parser ( <a href="https://github.com/titipata/pubmed_parser">https://github.com/titipata/pubmed_parser</a>) was used to extract information of PMID, Date, Title, Abstract, Journal, and Authors from the XML files. Articles for which no abstract text was available were not considered. The modified parser is available at <a href="https://github.com/cosylabiiit/pubmed_parser">https://github.com/cosylabiiit/pubmed_parser</a>. </p> </div> <div class="namedentityrecognition" title="sec"> <div class="title" tagx="title" title="title">
Named entity recognition</div> <p>We adopted a dictionary matching approach for Named Entity Recognition (NER) of spices and herbs. With a large dictionary, the process of dictionary matching becomes a computational bottleneck. Therefore, we used a modified implementation of Aho-Corasick algorithm (NoAho, <a href="https://github.com/JDonner/NoAho">https://github.com/JDonner/NoAho</a>) to efficiently obtain non-overlapping and longest matches at the token level. For disease NER (DNER) and normalization, we used TaggerOne [ <a href="#pone.0198030.ref019">19</a>] which utilizes semi-Markov models with a rich feature set. It was reported to have a precision of 85% and a recall of 80% on the Biocreative V Chemical Disease Relation test set [ <a href="#pone.0198030.ref046">46</a>]. We used the pre-trained disease-only model available with TaggerOne [ <a href="#pone.0198030.ref019">19</a>] on our data. </p> </div> <div class="preprocessing" title="sec"> <div class="title" tagx="title" title="title">
Preprocessing</div> <p>Sentence segmentation was carried out on the retrieved abstracts using Stanford CoreNLP package [ <a href="#pone.0198030.ref055">55</a>]. Only sentences with mention of at least one herb/spice and one disease were considered for extracting relations. Those with mentions of multiple herbs/spices and/or diseases were simplified by duplicating the sentence while iteratively masking all except a specific spice-disease pair. In all sentences, numbers were replaced by a standard identifier token and, barring some punctuation characters (!,.:;), all special characters were removed. The preprocessed sentences were then tokenized using GENIA [ <a href="#pone.0198030.ref056">56</a>] and the part-of-speech (PoS) tag, as well as the chunk tag of each token were obtained. Further, we also computed the distance of each token from the candidate spice-disease pair and used them as position features. </p> </div> <div class="labellingassociations" title="sec"> <div class="title" tagx="title" title="title">
Labelling associations</div> <p>Hitherto, to the best of our knowledge, no labeled corpus for associations between plant-based foods and diseases is publicly available. We thus manually annotated a total of 6712 spice-disease pairs to tag positive, negative and neutral associations. Out of all the annotated pairs, 2669 had positive associations, 301 had negative associations, and 3742 had neutral or no associations. This data was used for training as well as evaluating our relationship extraction models. </p> </div> <div class="relationextractionmodel" title="sec"> <div class="title" tagx="title" title="title">
Relation extraction model</div> <p>We developed a Machine Learning Classifier to categorize tagged spice-disease pair(s) in a sentence as having positive, negative or neutral associations. The following models were tested: (i) Linear Support Vector Machine (SVM) with unigram and bigram word features; (ii) Convolutional Neural Network (CNN) with word embedding [ <a href="#pone.0198030.ref027">27</a>] features; and (iii) CNN with word, position, PoS and chunk embedding features. </p> <p>For the Linear SVM model, we obtained the unigram and bigram word features and scaled their respective weights using Term Frequency-Inverse Document Frequency (TF-IDF) approach. This model was trained using one-versus-all strategy. Following are the equations describing the method for computing TF-IDF weights of features: ( <i>i</i>) <i>tf</i>( <i>t</i>, <i>s</i>) = <i>f</i> <sub> <i>t</i>, <i>s</i> </sub>; <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <div class="math_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
)</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> </div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
l</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
g</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> </div> </div> </div> </div> </div> </div> ; and ( <i>iii</i>) <i>tfidf</i>( <i>t</i>, <i>s</i>) = <i>tf</i>( <i>t</i>, <i>s</i>) * <i>idf</i>( <i>t</i>), where <i>f</i> <sub> <i>t</i>, <i>s</i> </sub> denotes the number of times feature <i>t</i> appears in sentence <i>s</i>, <i>n</i> <sub> <i>t</i> </sub> is the number of sentences in which the feature <i>t</i> appears and <i>N</i> is the total number of sentences. <p /> <p>The architecture for our CNN models is based on state-of-the-art models for sentence classification and relation extraction [ <a href="#pone.0198030.ref025">25</a>– <a href="#pone.0198030.ref027">27</a>]. As input, we fed mini-batches of sentence sequences to the models. The two CNNs differ in the representation of the tokens or words present in input sequences. For the first model, we only used the word embedding as the token representation, whereas for the second model we used the PoS, chunk and position embedding in addition to word embedding. The word embedding was initialized using pre-trained weights from Chiu <i>et</i>. <i>al</i> [ <a href="#pone.0198030.ref057">57</a>], with the embedding for unknown words initialized from a uniform (− <i>α</i>, <i>α</i>) distribution. The parameter <i>α</i> was determined on the basis of the variance of the known words [ <a href="#pone.0198030.ref058">58</a>]. Further, CNN requires all input sequences to have consistent size, thus sentences were zero-padded to equalize their lengths to that of the longest sentence. The input to the CNNs was a <i>b</i> × <i>d</i> × <i>n</i> × 1 tensor, where <i>b</i> is the size of the mini-batch, <i>d</i> is the length of the ‘token’ vector of the sentence and <i>n</i> is the length of the longest sentence in the corpus. The architecture of the second CNN is depicted in <a href="#pone.0198030.g009">Fig 9</a>. The first layer is a Convolutional layer with <i>n</i> <sub> <i>f</i> </sub> filters of different filter sizes <i>f</i> and rectified linear unit (ReLU) activation. The respective maximum activations of all the filters are then concatenated into a single vector of size <i>n</i> <sub> <i>f</i> </sub> and fed to a Dropout layer [ <a href="#pone.0198030.ref059">59</a>], which randomly sets an activation to zero with probability <i>p</i>. This is followed by a dense layer of <i>h</i> hidden units with ReLU activation and a softmax layer with 3 units. For both the networks, we used categorical cross entropy as our objective function and applied <i>l</i> <sub>2</sub> regularization of 3 on the dense layers only. The networks were trained using mini-batch gradient descent with shuffled batches of size 50 and Adam [ <a href="#pone.0198030.ref060">60</a>] optimizer. We adopted an early stopping criterion for the training process and stopped model training if the validation loss did not decrease for 5 epochs. To address the class imbalance problem, we over-sampled the negative class and the positive class by a factor of 12 and 1.35 respectively. The hyper-parameters of both the neural networks were determined using 5-fold cross validation and are available in <a href="#pone.0198030.s006">S2 Table</a>. The code as well as the data used for the CNNs is available at the Complex System Laboratory, IIIT-Delhi’s GitHub page: <a href="https://github.com/cosylabiiit/spice-disease-associations">https://github.com/cosylabiiit/spice-disease-associations</a>. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0198030.g009"> <span class="label" tagx="label" title="label">Fig 9</span> </a> </div>   <div class="title" tagx="title" title="title">
Architecture of the Convolutional Neural Network.</div> <p>Illustration of the convolutional neural network model utilizing word, position, part of speech and chunk embeddings. </p>   </div> </div> <div class="evaluationmetrics" title="sec"> <div class="title" tagx="title" title="title">
Evaluation metrics</div> <p>We evaluated the performance of our model based on its precision, recall, F1 score and accuracy: <i>Precision</i> = <i>TP</i>/( <i>TP</i> + <i>FP</i>); <i>Recall</i> = <i>TP</i>/( <i>TP</i> + <i>FN</i>); <i>F</i>1 − <i>score</i> = 2 ∙ <i>Precision</i> ∙ <i>Recall</i>/( <i>Precision</i> + <i>Recall</i>); <i>Accuracy</i> = <i>TP</i> + <i>TN</i>/( <i>TP</i> + <i>TN</i> + <i>FP</i> + <i>FN</i>), where <i>TP</i>, <i>FP</i>, <i>TN</i>, <i>FN</i> are True Positives, False Positives, True Negatives and False Negatives respectively. </p> </div> <div class="meshhierarchy" title="sec"> <div class="title" tagx="title" title="title">
MeSH hierarchy</div> <p>MeSH is a controlled vocabulary of biomedical terms curated and developed by National Library of Medicine. The terms are hierarchically organized from generic to more specific. The DNER tool used in this study (TaggerOne [ <a href="#pone.0198030.ref019">19</a>]) normalizes the tagged entities to MeSH IDs. The hierarchical structure of MeSH results in situations where a spice is typically associated with a disease at multiple levels of specificity. For example, in the first level of MeSH hierarchy a spice may be linked with the disease category Endocrine System Diseases (C19) and at the second level C19 may be associated with sub-categories such as Adrenal Gland Diseases (C19.053) or Diabetes Mellitus (C19.246). Further, it may be linked to the specific type of Diabetes Mellitus, say, ‘Diabetes Mellitus, Type 1 (C19.246.267)’ or ‘Diabetes Mellitus, Type 2 (C19.246.300)’ appearing at the third level. We conducted a multi-level analysis by associating spices with disease terms at top three levels of MeSH hierarchy which were referred to as category, sub-category, and a disease ( <a href="#pone.0198030.s001">S1 Fig</a>). </p> </div> <div class="adverseandbenevolentspectrumscores" title="sec"> <div class="title" tagx="title" title="title">
Adverse and benevolent spectrum scores</div> <p>The ‘spectrum score of a spice ( <i>Ω</i> <sub> <i>s</i> </sub>)’ encodes diversity of adverse ( <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
Ω</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> </div> </div> </div> </div> ) or therapeutic ( <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
Ω</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
+</div> </div> </div> </div> </div> </span>) effects of a spice <i>s</i> across the MeSH disease categories as well as their constituent subcategories, and is defined as <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Ω</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> </div> <div class="mo_UNKNOWN">
^</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> </div> </div> <div class="mo_UNKNOWN">
∙</div> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mo_UNKNOWN">
^</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> <div class="mo_UNKNOWN">
/</div> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div> </div> </div> </div> </div> </span>. Here, <i>D</i> is total number of MeSH disease categories, <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> </div> <div class="mo_UNKNOWN">
^</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> </div> </div> </div> </div> </span> represents the number of disease categories with which spice <i>s</i> has therapeutic association with, <i>d</i> <sup> <i>i</i> </sup> represents the total number of disease sub-categories in the <i>i</i>th disease category, and <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mo_UNKNOWN">
^</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div> </span> represents the number of disease subcategories in the <i>i</i>th disease category with which the spice <i>s</i> is associated. When calculating the ‘spectrum scores’ across all 27 categories, the ‘adverse spectrum score’ and ‘benevolent spectrum score’ vary between 0 and 729. Further, for each spice the ‘relative benevolence’ ( <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Δ</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Ω</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
Ω</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
+</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
Ω</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> </div> </div> </div> </div> </span>) that encodes its residual therapeutic benefit was computed. <p /> </div> <div class="‘therapeutictradeoffscore’forculinaryrecommendations" title="sec"> <div class="title" tagx="title" title="title">
‘Therapeutic tradeoff score’ for culinary recommendations</div> <p>Category-specific (benevolence and adverse) spectrum score was defined as <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
Ω</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mo_UNKNOWN">
^</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mo_UNKNOWN">
∙</div> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> </div> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
α</div> </div> <div class="mo_UNKNOWN">
^</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> </div> </div> </div> <div class="mo_UNKNOWN">
/</div> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
α</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> </div> </div> </div> </div> </div> </div> </div> </div> . Here, <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mo_UNKNOWN">
^</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div> </span> represents the number of disease sub-categories in the <i>i</i>th disease category with which spice <i>s</i> has therapeutic association with, <i>α</i> <sup> <i>k</i> </sup> represents the total number of diseases in the <i>k</i>th disease sub-category, and <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
α</div> </div> <div class="mo_UNKNOWN">
^</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> </div> </div> </div> </div> </span> represents the number of disease subcategories in the <i>k</i>th disease sub-category with which the spice <i>s</i> is associated. The ‘therapeutic tradeoff score’, <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Δ</div> </div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
Ω</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div> </span>, represents the difference between the ‘benevolence spectrum’ and ‘adverse spectrum’ of spice <i>s</i> for category <i>i</i>; the higher the tradeoff score of a spice the better is its therapeutic value against the spectrum of diseases represented by this category. Thus, tradeoff score of a spice serves as a basis for its recommendation against a MeSH disease category. <p /> </div> <div class="linkingphytochemicalsfromspices/herbstodiseases" title="sec"> <div class="title" tagx="title" title="title">
Linking phytochemicals from spices/herbs to diseases</div> <p>We obtained the phytochemical information for spices/herbs using KNApSAcK [ <a href="#pone.0198030.ref021">21</a>] and CTD [ <a href="#pone.0198030.ref022">22</a>]. The different compound identifiers were standardized to PubChem IDs and further PubChem BioAssay [ <a href="#pone.0198030.ref040">40</a>] was used for ascertaining their bioactive status. Therapeutic associations of a compound were obtained from CTD [ <a href="#pone.0198030.ref022">22</a>] after mapping its PubChem ID to corresponding MeSH ID. </p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting information</div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S1 Fig</span>  <div class="title" tagx="title" title="title">
Hierarchical structure of MeSH disease headers.</div> <p>For the purpose of multi-level analysis, spices were associated with disease terms at three levels of MeSH hierarchy—‘category’, ‘sub-category’ and a ‘disease’. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0198030.s001.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S2 Fig</span>  <div class="title" tagx="title" title="title">
Correlation between the number of phytochemicals in spices and their broad-spectrum benevolence. </div> <p>The data indicate that the broad-spectrum benevolence score of spices and their phytochemical repertoire are not correlated. </p> <p>(TIFF)</p>  <div class="media" title="media"> <a href="pone.0198030.s002.tiff">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S3 Fig</span>  <div class="title" tagx="title" title="title">
Comparison of the number of associations obtained for spices reported by our study with that of NutriChem [ <a href="#pone.0198030.ref015">15</a>, <a href="#pone.0198030.ref016">16</a>] indicating richer associations in our data. </div> <p>(TIFF)</p>  <div class="media" title="media"> <a href="pone.0198030.s003.tiff">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S4 Fig</span>  <div class="title" tagx="title" title="title">
Comparison of associations retrieved for ‘individual spices’ by NutriChem[ <a href="#pone.0198030.ref015">15</a>, <a href="#pone.0198030.ref016">16</a>] to those from our study, suggesting better depth/coverage in the latter. </div> <p>(TIFF)</p>  <div class="media" title="media"> <a href="pone.0198030.s004.tiff">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S1 Table</span>  <div class="title" tagx="title" title="title">
Top ten broad spectrum spices and number of MeSH disease categories and subcategories with which they are positively associated. </div> <p>(DOCX)</p>  <div class="media" title="media"> <a href="pone.0198030.s005.docx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S2 Table</span>  <div class="title" tagx="title" title="title">
Hyper-parameters selected for the convolutional neural network Model 2 and Model 3.</div> <p>(DOCX)</p>  <div class="media" title="media"> <a href="pone.0198030.s006.docx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S1 Dataset</span>  <div class="title" tagx="title" title="title">
Statistics of positive and negative spice-disease associations for each spice.</div> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0198030.s007.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S2 Dataset</span>  <div class="title" tagx="title" title="title">
Statistics of positive and negative associations as well as number of spices, at the third level of MeSH. </div> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0198030.s008.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S3 Dataset</span>  <div class="title" tagx="title" title="title">
Statistics of positive and negative associations as well as the number of spices at the third level of MeSH disease hierarchy. </div> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0198030.s009.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S4 Dataset</span>  <div class="title" tagx="title" title="title">
Statistics of positive and negative associations as well as the number of spices at the second level (sub-category) of MeSH disease hierarchy. </div> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0198030.s010.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S5 Dataset</span>  <div class="title" tagx="title" title="title">
Benevolent, adverse as well as relative benevolence scores for all spices.</div> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0198030.s011.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S6 Dataset</span>  <div class="title" tagx="title" title="title">
List of culinary recommendations against various disease categories.</div> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0198030.s012.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S7 Dataset</span>  <div class="title" tagx="title" title="title">
Tripartite associations for a spice and a disease along with specific phytochemicals reported to be involved in the therapeutic action. </div> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0198030.s013.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S8 Dataset</span>  <div class="title" tagx="title" title="title">
Statistics of spice-disease associations for which bo specific phytochemicals were ascertained. </div> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0198030.s014.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We thank the Indraprastha Institute of Information Technology (IIIT-Delhi) for providing computational facilities to GB and RT. This work was supported by a senior research fellowship from the Ministry of Human Resource Development, Government of India and Indian Institute of Technology Jodhpur to NKR. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript. </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="pone.0198030.ref001" /> <span class="label" tagx="label" title="label">1</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sherman</span> <span class="given-names" tagx="given-names" title="given-names">PW</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Billing</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Darwinian Gastronomy: Spices taste good because they are good for us</span>. <span class="source" tagx="source" title="source">Bioscience</span>. <span class="year" tagx="year" title="year">1999</span>; <span class="volume" tagx="volume" title="volume">49</span>: <span class="fpage" tagx="fpage" title="fpage">453</span>– <span class="lpage" tagx="lpage" title="lpage">463</span>. <span class="comment">doi: <a href="https://doi.org/10.2307/1313553">10.2307/1313553</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref002" /> <span class="label" tagx="label" title="label">2</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Billing</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sherman</span> <span class="given-names" tagx="given-names" title="given-names">PW</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Antimicrobial functions of spices: why some like it hot</span>. <span class="source" tagx="source" title="source">Q Rev Biol</span>. <span class="year" tagx="year" title="year">1998</span>; <span class="volume" tagx="volume" title="volume">73</span>: <span class="fpage" tagx="fpage" title="fpage">3</span>– <span class="lpage" tagx="lpage" title="lpage">49</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9586227">9586227</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref003" /> <span class="label" tagx="label" title="label">3</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Srinivasan</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Spices as influencers of body metabolism: An overview of three decades of research</span>. <span class="source" tagx="source" title="source">Food Res Int. Elsevier</span>; <span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">38</span>: <span class="fpage" tagx="fpage" title="fpage">77</span>– <span class="lpage" tagx="lpage" title="lpage">86</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.foodres.2004.09.001">10.1016/j.foodres.2004.09.001</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref004" /> <span class="label" tagx="label" title="label">4</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Srinivasan</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Plant foods in the management of diabetes mellitus: Spices as beneficial antidiabetic food adjuncts </span>. <span class="source" tagx="source" title="source">Int J Food Sci Nutr</span>. Taylor &amp;amp; Francis; <span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">56</span>: <span class="fpage" tagx="fpage" title="fpage">399</span>– <span class="lpage" tagx="lpage" title="lpage">414</span>. <span class="comment">doi: <a href="https://doi.org/10.1080/09637480500512872">10.1080/09637480500512872</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16361181">16361181</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref005" /> <span class="label" tagx="label" title="label">5</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Srinivasan</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Anti-cholelithogenic potential of dietary spices and their bioactives</span>. <span class="source" tagx="source" title="source">Crit Rev Food Sci Nutr</span>. <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">57</span>: <span class="fpage" tagx="fpage" title="fpage">1749</span>– <span class="lpage" tagx="lpage" title="lpage">1758</span>. <span class="comment">doi: <a href="https://doi.org/10.1080/10408398.2014.1003783">10.1080/10408398.2014.1003783</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26147513">26147513</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref006" /> <span class="label" tagx="label" title="label">6</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yashin</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yashin</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xia</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nemzer</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Antioxidant Activity of Spices and Their Impact on Human Health: A Review</span>. <span class="source" tagx="source" title="source">Antioxidants</span>. Multidisciplinary Digital Publishing Institute (MDPI); <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">70</span> <span class="comment">doi: <a href="https://doi.org/10.3390/antiox6030070">10.3390/antiox6030070</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28914764">28914764</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref007" /> <span class="label" tagx="label" title="label">7</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kaefer</span> <span class="given-names" tagx="given-names" title="given-names">CM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Milner</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span>. <span class="mixed-article-title" title="mixed-article-title">The role of herbs and spices in cancer prevention</span>. <span class="source" tagx="source" title="source">Journal of Nutritional Biochemistry</span>. Elsevier; <span class="year" tagx="year" title="year">2008</span> pp. <span class="fpage" tagx="fpage" title="fpage">347</span>– <span class="lpage" tagx="lpage" title="lpage">361</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.jnutbio.2007.11.003">10.1016/j.jnutbio.2007.11.003</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18499033">18499033</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref008" /> <span class="label" tagx="label" title="label">8</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Srinivasan</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Role of Spices Beyond Food Flavoring: Nutraceuticals with Multiple Health Effects</span>. <span class="source" tagx="source" title="source">Food Rev Int</span>. <span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">21</span>: <span class="fpage" tagx="fpage" title="fpage">167</span>– <span class="lpage" tagx="lpage" title="lpage">188</span>. <span class="comment">doi: <a href="https://doi.org/10.1081/FRI-200051872">10.1081/FRI-200051872</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref009" /> <span class="label" tagx="label" title="label">9</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johri</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zutshi</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span>. <span class="mixed-article-title" title="mixed-article-title">An Ayurvedic formulation "Trikatu" and its constituents</span>. <span class="source" tagx="source" title="source">Journal of Ethnopharmacology</span>. Elsevier; <span class="year" tagx="year" title="year">1992</span> pp. <span class="fpage" tagx="fpage" title="fpage">85</span>– <span class="lpage" tagx="lpage" title="lpage">91</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/0378-8741(92)90067-2">10.1016/0378-8741(92)90067-2</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/1434692">1434692</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref010" /> <span class="label" tagx="label" title="label">10</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Swanson</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Fish Oil, Raynaud’s Syndrom, and Undicovered Public Knowledge</span>. <span class="source" tagx="source" title="source">Perspectives in Biology and Medicine</span>. <span class="year" tagx="year" title="year">1986</span> pp. <span class="fpage" tagx="fpage" title="fpage">7</span>– <span class="lpage" tagx="lpage" title="lpage">18</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/3797213">3797213</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref011" /> <span class="label" tagx="label" title="label">11</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jensen</span> <span class="given-names" tagx="given-names" title="given-names">LJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saric</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bork</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Literature mining for the biologist: from information retrieval to biological discovery</span>. <span class="source" tagx="source" title="source">Nat Rev Genet</span>. Nature Publishing Group; <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">119</span>– <span class="lpage" tagx="lpage" title="lpage">129</span>. <span class="comment">doi: <a href="https://doi.org/10.1038/nrg1768">10.1038/nrg1768</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16418747">16418747</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref012" /> <span class="label" tagx="label" title="label">12</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Choi</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Park</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Song</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Discovering context-specific relationships from biological literature by using multi-level context terms </span>. <span class="source" tagx="source" title="source">BMC Med Inform Decis Mak</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">12</span> <span class="comment">doi: <a href="https://doi.org/10.1186/1472-6947-12-S1-S1">10.1186/1472-6947-12-S1-S1</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22595086">22595086</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref013" /> <span class="label" tagx="label" title="label">13</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Swaminathan</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharma</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ketkar</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">D’Silva</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Mining biomedical text towards building a quantitative food-disease-gene network</span>. <span class="source" tagx="source" title="source">Stud Comput Intell</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">375</span>: <span class="fpage" tagx="fpage" title="fpage">205</span>– <span class="lpage" tagx="lpage" title="lpage">225</span>. <span class="comment">doi: <a href="https://doi.org/10.1007/978-3-642-22913-8_10">10.1007/978-3-642-22913-8_10</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref014" /> <span class="label" tagx="label" title="label">14</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Srinivasan</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Libbus</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Mining MEDLINE for implicit links between dietary substances and diseases</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">20</span>: <span class="fpage" tagx="fpage" title="fpage">290</span>– <span class="lpage" tagx="lpage" title="lpage">296</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/bioinformatics/bth914">10.1093/bioinformatics/bth914</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15262811">15262811</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref015" /> <span class="label" tagx="label" title="label">15</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jensen</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Panagiotou</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kouskoumvekaki</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Integrated text mining and chemoinformatics analysis associates diet to health benefit at molecular level </span>. <span class="source" tagx="source" title="source">PLoS Comput Biol</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">e1003432</span> <span class="comment">doi: <a href="https://doi.org/10.1371/journal.pcbi.1003432">10.1371/journal.pcbi.1003432</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24453957">24453957</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref016" /> <span class="label" tagx="label" title="label">16</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jensen</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Panagiotou</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kouskoumvekaki</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span>. <span class="mixed-article-title" title="mixed-article-title">NutriChem: A systems chemical biology resource to explore the medicinal value of plant-based foods </span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">43</span>: <span class="fpage" tagx="fpage" title="fpage">D940</span>– <span class="lpage" tagx="lpage" title="lpage">D945</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/nar/gku724">10.1093/nar/gku724</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25106869">25106869</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref017" /> <span class="label" tagx="label" title="label">17</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Choi</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Choi</span> <span class="given-names" tagx="given-names" title="given-names">C-H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">YR</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">S-J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Na</span> <span class="given-names" tagx="given-names" title="given-names">C-S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>. <span class="mixed-article-title" title="mixed-article-title">HerDing: herb recommendation system to treat diseases using genes and chemicals</span>. <span class="source" tagx="source" title="source">Database</span>. Oxford University Press; <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">baw011</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">7</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/database/baw011">10.1093/database/baw011</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26980517">26980517</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref018" /> <span class="label" tagx="label" title="label">18</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garg</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sethupathy</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuwani</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">NK</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dokania</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Iyer</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">FlavorDB: a database of flavor molecules</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. Oxford University Press; <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">46</span>: <span class="fpage" tagx="fpage" title="fpage">D1210</span>– <span class="lpage" tagx="lpage" title="lpage">D1216</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/nar/gkx957">10.1093/nar/gkx957</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29059383">29059383</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref019" /> <span class="label" tagx="label" title="label">19</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leaman</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lu</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>. <span class="mixed-article-title" title="mixed-article-title">TaggerOne: Joint named entity recognition and normalization with semi-Markov Models</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">32</span>: <span class="fpage" tagx="fpage" title="fpage">2839</span>– <span class="lpage" tagx="lpage" title="lpage">2846</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/bioinformatics/btw343">10.1093/bioinformatics/btw343</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27283952">27283952</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref020" /> <span class="label" tagx="label" title="label">20</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Neveu</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Perez-Jimenez</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vos</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Crespy</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">du Chaffaut</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mennen</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Phenol-Explorer: an online comprehensive database on polyphenol contents in foods</span>. <span class="source" tagx="source" title="source">Database</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">2010</span>: <span class="fpage" tagx="fpage" title="fpage">bap024</span> <span class="comment">doi: <a href="https://doi.org/10.1093/database/bap024">10.1093/database/bap024</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20428313">20428313</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref021" /> <span class="label" tagx="label" title="label">21</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Afendi</span> <span class="given-names" tagx="given-names" title="given-names">FM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Okada</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamazaki</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hirai-Morita</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakamura</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakamura</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">KNApSAcK family databases: Integrated metabolite-plant species databases for multifaceted plant research </span>. <span class="source" tagx="source" title="source">Plant Cell Physiol</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">53</span>: <span class="fpage" tagx="fpage" title="fpage">e1</span> <span class="comment">doi: <a href="https://doi.org/10.1093/pcp/pcr165">10.1093/pcp/pcr165</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22123792">22123792</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref022" /> <span class="label" tagx="label" title="label">22</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Davis</span> <span class="given-names" tagx="given-names" title="given-names">AP</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grondin</span> <span class="given-names" tagx="given-names" title="given-names">CJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnson</span> <span class="given-names" tagx="given-names" title="given-names">RJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sciaky</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">King</span> <span class="given-names" tagx="given-names" title="given-names">BL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McMorran</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">The Comparative Toxicogenomics Database: Update 2017</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">45</span>: <span class="fpage" tagx="fpage" title="fpage">D972</span>– <span class="lpage" tagx="lpage" title="lpage">D978</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/nar/gkw838">10.1093/nar/gkw838</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27651457">27651457</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref023" /> <span class="label" tagx="label" title="label">23</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rakhi</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuwani</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garg</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mukherjee</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bagler</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>. <span class="mixed-article-title" title="mixed-article-title">SpiceRx: an integrated resource for the health impacts of culinary spices and herbs</span>. <span class="source" tagx="source" title="source">bioRxiv 273599</span>. Cold Spring Harbor Laboratory; <span class="year" tagx="year" title="year">2018</span>; <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">24</span>. <span class="comment">doi: <a href="https://doi.org/10.1101/273599">10.1101/273599</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref024" /> <span class="label" tagx="label" title="label">24</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lipscomb</span> <span class="given-names" tagx="given-names" title="given-names">CE</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Medical Subject Headings (MeSH)</span>. <span class="source" tagx="source" title="source">Bull Med Libr Assoc</span>. <span class="year" tagx="year" title="year">2000</span>; <span class="volume" tagx="volume" title="volume">88</span>: <span class="fpage" tagx="fpage" title="fpage">265</span>– <span class="lpage" tagx="lpage" title="lpage">266</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10928714">10928714</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref025" /> <span class="label" tagx="label" title="label">25</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar Sahu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Anand</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oruganty</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gattu</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Relation extraction from clinical texts using domain invariant convolutional neural network </span>. <span class="source" tagx="source" title="source">Proc 15th Work Biomed Nat Lang Process</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">71</span> <span class="comment">doi: <a href="https://doi.org/10.18653/v1/W16-2928">10.18653/v1/W16-2928</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref026" /> <span class="label" tagx="label" title="label">26</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nguyen</span> <span class="given-names" tagx="given-names" title="given-names">TH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grishman</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Relation Extraction: Perspective from Convolutional Neural Networks</span>. <span class="source" tagx="source" title="source">Work Vector Model NLP</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="fpage" tagx="fpage" title="fpage">39</span>– <span class="lpage" tagx="lpage" title="lpage">48</span>. <span class="comment">doi: <a href="https://doi.org/10.3115/v1/W15-1506">10.3115/v1/W15-1506</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref027" /> <span class="label" tagx="label" title="label">27</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mikolov</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Corrado</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dean</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Efficient estimation of word representations in vector space</span>. <span class="source" tagx="source" title="source">arXiv Prepr arXiv13013781</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">12</span>. <span class="comment">doi: <a href="https://doi.org/10.1162/153244303322533223">10.1162/153244303322533223</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref028" /> <span class="label" tagx="label" title="label">28</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zorofchian Moghadamtousi</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Abdul Kadir</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hassandarvish</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tajik</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Abubakar</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zandi</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>. <span class="chapter-title" tagx="chapter-title" title="chapter-title">A review on antibacterial, antiviral, and antifungal activity of curcumin</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Research International</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Hindawi</span>; <span class="year" tagx="year" title="year">2014</span> p. <span class="fpage" tagx="fpage" title="fpage">186864</span>. </span> </li> <li tag="ref"> <a name="pone.0198030.ref029" /> <span class="label" tagx="label" title="label">29</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Holzer</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Transient receptor potential (TRP) channels as drug targets for diseases of the digestive system </span>. <span class="source" tagx="source" title="source">Pharmacol Ther</span>. Pergamon; <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">131</span>: <span class="fpage" tagx="fpage" title="fpage">142</span>– <span class="lpage" tagx="lpage" title="lpage">170</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.pharmthera.2011.03.006">10.1016/j.pharmthera.2011.03.006</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21420431">21420431</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref030" /> <span class="label" tagx="label" title="label">30</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Platel</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Srinivasan</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Digestive stimulant action of spices: a myth or reality?</span> <span class="source" tagx="source" title="source">Indian J Med Res</span>. <span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">119</span>: <span class="fpage" tagx="fpage" title="fpage">167</span>– <span class="lpage" tagx="lpage" title="lpage">79</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15218978">15218978</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref031" /> <span class="label" tagx="label" title="label">31</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rahman</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lowe</span> <span class="given-names" tagx="given-names" title="given-names">GM</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Garlic and Cardiovascular Disease: A Critical Review</span>. <span class="source" tagx="source" title="source">J Nutr</span>. American Society for Nutrition; <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">136</span>: <span class="fpage" tagx="fpage" title="fpage">736</span>– <span class="lpage" tagx="lpage" title="lpage">740</span>. </span> </li> <li tag="ref"> <a name="pone.0198030.ref032" /> <span class="label" tagx="label" title="label">32</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bi</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lim</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jeyakumar Henry</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Spices in the management of diabetes mellitus</span>. <span class="source" tagx="source" title="source">Food Chem</span>. Elsevier; <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">217</span>: <span class="fpage" tagx="fpage" title="fpage">281</span>– <span class="lpage" tagx="lpage" title="lpage">293</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.foodchem.2016.08.111">10.1016/j.foodchem.2016.08.111</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27664636">27664636</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref033" /> <span class="label" tagx="label" title="label">33</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Heshmati</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Namazi</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Effects of black seed (Nigella sativa) on metabolic parameters in diabetes mellitus: A systematic review </span>. <span class="source" tagx="source" title="source">Complement Ther Med</span>. Churchill Livingstone; <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">23</span>: <span class="fpage" tagx="fpage" title="fpage">275</span>– <span class="lpage" tagx="lpage" title="lpage">282</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.ctim.2015.01.013">10.1016/j.ctim.2015.01.013</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25847566">25847566</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref034" /> <span class="label" tagx="label" title="label">34</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Visioli</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Borsani</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Galli</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Diet and prevention of coronary heart disease: The potential role of phytochemicals</span>. <span class="source" tagx="source" title="source">Cardiovascular Research</span>. Oxford University Press; <span class="year" tagx="year" title="year">2000</span> pp. <span class="fpage" tagx="fpage" title="fpage">419</span>– <span class="lpage" tagx="lpage" title="lpage">425</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/S0008-6363(00)00053-5">10.1016/S0008-6363(00)00053-5</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10963715">10963715</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref035" /> <span class="label" tagx="label" title="label">35</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mishra</span> <span class="given-names" tagx="given-names" title="given-names">BB</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tiwari</span> <span class="given-names" tagx="given-names" title="given-names">VK</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Natural products: An evolving role in future drug discovery</span>. <span class="source" tagx="source" title="source">Eur J Med Chem</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">46</span>: <span class="fpage" tagx="fpage" title="fpage">4769</span>– <span class="lpage" tagx="lpage" title="lpage">4807</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.ejmech.2011.07.057">10.1016/j.ejmech.2011.07.057</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21889825">21889825</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref036" /> <span class="label" tagx="label" title="label">36</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Manach</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Scalbert</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morand</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rémésy</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiménez</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Polyphenols: Food sources and bioavailability</span>. <span class="source" tagx="source" title="source">American Journal of Clinical Nutrition</span>. American Society for Nutrition; <span class="year" tagx="year" title="year">2004</span> pp. <span class="fpage" tagx="fpage" title="fpage">727</span>– <span class="lpage" tagx="lpage" title="lpage">747</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/ajcn/79.5.727">10.1093/ajcn/79.5.727</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15113710">15113710</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref037" /> <span class="label" tagx="label" title="label">37</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kocaadam</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Şanlier</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Curcumin, an active component of turmeric (Curcuma longa), and its effects on health</span>. <span class="source" tagx="source" title="source">Crit Rev Food Sci Nutr</span>. Taylor &amp;amp; Francis; <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">57</span>: <span class="fpage" tagx="fpage" title="fpage">2889</span>– <span class="lpage" tagx="lpage" title="lpage">2895</span>. <span class="comment">doi: <a href="https://doi.org/10.1080/10408398.2015.1077195">10.1080/10408398.2015.1077195</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26528921">26528921</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref038" /> <span class="label" tagx="label" title="label">38</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bayan</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koulivand</span> <span class="given-names" tagx="given-names" title="given-names">PH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gorji</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Garlic: a review of potential therapeutic effects</span>. <span class="source" tagx="source" title="source">Avicenna J phytomedicine</span>. Mashhad University of Medical Sciences; <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">4</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">14</span>. </span> </li> <li tag="ref"> <a name="pone.0198030.ref039" /> <span class="label" tagx="label" title="label">39</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shahidi</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ambigaipalan</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Phenolics and polyphenolics in foods, beverages and spices: Antioxidant activity and health effects—A review </span>. <span class="source" tagx="source" title="source">J Funct Foods</span>. Elsevier Ltd; <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">18</span>: <span class="fpage" tagx="fpage" title="fpage">820</span>– <span class="lpage" tagx="lpage" title="lpage">897</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.jff.2015.06.018">10.1016/j.jff.2015.06.018</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref040" /> <span class="label" tagx="label" title="label">40</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bryant</span> <span class="given-names" tagx="given-names" title="given-names">SH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheng</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gindulyte</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shoemaker</span> <span class="given-names" tagx="given-names" title="given-names">BA</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">PubChem BioAssay: 2017 update</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. Oxford University Press; <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">45</span>: <span class="fpage" tagx="fpage" title="fpage">D955</span>– <span class="lpage" tagx="lpage" title="lpage">D963</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/nar/gkw1118">10.1093/nar/gkw1118</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27899599">27899599</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref041" /> <span class="label" tagx="label" title="label">41</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">C-C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chung</span> <span class="given-names" tagx="given-names" title="given-names">JG</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsai</span> <span class="given-names" tagx="given-names" title="given-names">S-J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">JH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sheen</span> <span class="given-names" tagx="given-names" title="given-names">LY</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Differential effects of allyl sulfides from garlic essential oil on cell cycle regulation in human liver tumor cells </span>. <span class="source" tagx="source" title="source">Food Chem Toxicol</span>. Pergamon; <span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">42</span>: <span class="fpage" tagx="fpage" title="fpage">1937</span>– <span class="lpage" tagx="lpage" title="lpage">1947</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.fct.2004.07.008">10.1016/j.fct.2004.07.008</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15500931">15500931</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref042" /> <span class="label" tagx="label" title="label">42</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wrangham</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>. <span class="source" tagx="source" title="source">Catching Fire: How Cooking Made Us Human</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">Basic Books</span>; <span class="year" tagx="year" title="year">2009</span>. </span> </li> <li tag="ref"> <a name="pone.0198030.ref043" /> <span class="label" tagx="label" title="label">43</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fonseca-Azevedo</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Herculano-Houzel</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Metabolic constraint imposes tradeoff between body size and number of brain neurons in human evolution </span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">109</span>: <span class="fpage" tagx="fpage" title="fpage">18571</span>– <span class="lpage" tagx="lpage" title="lpage">18576</span>. <span class="comment">doi: <a href="https://doi.org/10.1073/pnas.1206390109">10.1073/pnas.1206390109</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23090991">23090991</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref044" /> <span class="label" tagx="label" title="label">44</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chopan</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Littenberg</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>. <span class="mixed-article-title" title="mixed-article-title">The association of hot red chili pepper consumption and mortality: A large population-based cohort study </span>. <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gualillo</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span>, editor. <span class="source" tagx="source" title="source">PLoS One</span>. Public Library of Science; <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">e0169876</span> <span class="comment">doi: <a href="https://doi.org/10.1371/journal.pone.0169876">10.1371/journal.pone.0169876</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28068423">28068423</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref045" /> <span class="label" tagx="label" title="label">45</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Emont</span> <span class="given-names" tagx="given-names" title="given-names">MP</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jun</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qiao</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liao</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">D il</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Cinnamaldehyde induces fat cell-autonomous thermogenesis and metabolic reprogramming</span>. <span class="source" tagx="source" title="source">Metabolism</span>. W.B. Saunders; <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">77</span>: <span class="fpage" tagx="fpage" title="fpage">58</span>– <span class="lpage" tagx="lpage" title="lpage">64</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.metabol.2017.08.006">10.1016/j.metabol.2017.08.006</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29046261">29046261</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref046" /> <span class="label" tagx="label" title="label">46</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnson</span> <span class="given-names" tagx="given-names" title="given-names">RJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sciaky</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wei</span> <span class="given-names" tagx="given-names" title="given-names">CH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leaman</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">BioCreative V CDR task corpus: a resource for chemical disease relation extraction</span>. <span class="source" tagx="source" title="source">Database (Oxford)</span>. Oxford University Press; <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">2016</span>: <span class="fpage" tagx="fpage" title="fpage">baw608</span> <span class="comment">doi: <a href="https://doi.org/10.1093/database/baw068">10.1093/database/baw068</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27161011">27161011</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref047" /> <span class="label" tagx="label" title="label">47</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jain</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rakhi</span> <span class="given-names" tagx="given-names" title="given-names">NK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bagler</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Analysis of food pairing in regional cuisines of India</span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">10</span> <span class="comment">doi: <a href="https://doi.org/10.1371/journal.pone.0139539">10.1371/journal.pone.0139539</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26430895">26430895</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref048" /> <span class="label" tagx="label" title="label">48</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Jain A, Rakhi NK, Bagler G. Spices form the basis of food pairing in Indian cuisine. arXiv:150203815. 2015; 30. </span> </li> <li tag="ref"> <a name="pone.0198030.ref049" /> <span class="label" tagx="label" title="label">49</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Devarajan</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mohanmarugaraja</span> <span class="given-names" tagx="given-names" title="given-names">MK</span> </span>. <span class="mixed-article-title" title="mixed-article-title">A Comprehensive Review on Rasam: A South Indian Traditional Functional Food</span>. <span class="source" tagx="source" title="source">Pharmacogn Rev</span>. Wolters Kluwer—Medknow Publications; <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">73</span>– <span class="lpage" tagx="lpage" title="lpage">82</span>. <span class="comment">doi: <a href="https://doi.org/10.4103/phrev.phrev_13_17">10.4103/phrev.phrev_13_17</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28989243">28989243</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref050" /> <span class="label" tagx="label" title="label">50</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Prasad</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reddy</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Francis</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nayak</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kishore</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nandakumar</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Sambar, an Indian dish prevents the development of dimethyl hydrazine-induced colon cancer: A preclinical study </span>. <span class="source" tagx="source" title="source">Pharmacogn Mag</span>. Wolters Kluwer—Medknow Publications; <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">441</span> <span class="comment">doi: <a href="https://doi.org/10.4103/0973-1296.191454">10.4103/0973-1296.191454</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27761072">27761072</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref051" /> <span class="label" tagx="label" title="label">51</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Doss</span> <span class="given-names" tagx="given-names" title="given-names">HM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ganesan</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rasool</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Trikatu, an herbal compound ameliorates rheumatoid arthritis by the suppression of inflammatory immune responses in rats with adjuvant-induced arthritis and on cultured fibroblast like synoviocytes via the inhibition of the NFκB signaling pathway </span>. <span class="source" tagx="source" title="source">Chem Biol Interact</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">258</span>: <span class="fpage" tagx="fpage" title="fpage">175</span>– <span class="lpage" tagx="lpage" title="lpage">186</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.cbi.2016.09.003">10.1016/j.cbi.2016.09.003</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27613480">27613480</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref052" /> <span class="label" tagx="label" title="label">52</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">C-T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">B-Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hung</span> <span class="given-names" tagx="given-names" title="given-names">Y-C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">L-Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">Y-Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">T-K</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Decreased risk of dementia in migraine patients with traditional Chinese medicine use: a population-based cohort study </span>. <span class="source" tagx="source" title="source">Oncotarget</span>. <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">79680</span>– <span class="lpage" tagx="lpage" title="lpage">79692</span>. <span class="comment">doi: <a href="https://doi.org/10.18632/oncotarget.19094">10.18632/oncotarget.19094</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29108348">29108348</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref053" /> <span class="label" tagx="label" title="label">53</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McGee</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>. <span class="mixed-article-title" title="mixed-article-title">In victu veritas</span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">1998</span>; <span class="volume" tagx="volume" title="volume">392</span>: <span class="fpage" tagx="fpage" title="fpage">649</span>– <span class="lpage" tagx="lpage" title="lpage">650</span>. <span class="comment">doi: <a href="https://doi.org/10.1038/33528">10.1038/33528</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9565025">9565025</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref054" /> <span class="label" tagx="label" title="label">54</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suresh</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gurudutt</span> <span class="given-names" tagx="given-names" title="given-names">KN</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Srinivasan</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Degradation of bioactive spice compound: Curcumin during domestic cooking</span>. <span class="source" tagx="source" title="source">Eur Food Res Technol</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">228</span>: <span class="fpage" tagx="fpage" title="fpage">807</span>– <span class="lpage" tagx="lpage" title="lpage">812</span>. <span class="comment">doi: <a href="https://doi.org/10.1007/s00217-008-0993-9">10.1007/s00217-008-0993-9</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref055" /> <span class="label" tagx="label" title="label">55</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Manning</span> <span class="given-names" tagx="given-names" title="given-names">CD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Surdeanu</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bauer</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Finkel</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bethard</span> <span class="given-names" tagx="given-names" title="given-names">SJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McClosky</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>. <span class="mixed-article-title" title="mixed-article-title">The Stanford CoreNLP Natural Language Processing Toolkit</span>. <span class="source" tagx="source" title="source">Association for Computational Linguistics (ACL) System Demonstrations</span>. <span class="year" tagx="year" title="year">2014</span> pp. <span class="fpage" tagx="fpage" title="fpage">55</span>– <span class="lpage" tagx="lpage" title="lpage">60</span>. </span> </li> <li tag="ref"> <a name="pone.0198030.ref056" /> <span class="label" tagx="label" title="label">56</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">J-D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ohta</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tateisi</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="mixed-article-title" title="mixed-article-title">Tsujii. J. GENIA corpus—a semantically annotated corpus for bio-textmining</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="year" tagx="year" title="year">2003</span>; <span class="volume" tagx="volume" title="volume">19</span>: <span class="fpage" tagx="fpage" title="fpage">i180</span>– <span class="lpage" tagx="lpage" title="lpage">i182</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12855455">12855455</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref057" /> <span class="label" tagx="label" title="label">57</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chiu</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Crichton</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Korhonen</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pyysalo</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>. <span class="mixed-article-title" title="mixed-article-title">How to Train good Word Embeddings for Biomedical NLP</span>. <span class="source" tagx="source" title="source">Proceedings of the 15th Workshop on Biomedical Natural Language Processing</span>. <span class="year" tagx="year" title="year">2016</span> pp. <span class="fpage" tagx="fpage" title="fpage">166</span>– <span class="lpage" tagx="lpage" title="lpage">174</span>. </span> </li> <li tag="ref"> <a name="pone.0198030.ref058" /> <span class="label" tagx="label" title="label">58</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Kim Y. Convolutional Neural Networks for Sentence Classification. arXiv:14085882. 2014; </span> </li> <li tag="ref"> <a name="pone.0198030.ref059" /> <span class="label" tagx="label" title="label">59</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Srivastava</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hinton</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Krizhevsky</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sutskever</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salakhutdinov</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Dropout: A Simple Way to Prevent Neural Networks from Overfitting</span>. <span class="source" tagx="source" title="source">J Mach Learn Res</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">1929</span>– <span class="lpage" tagx="lpage" title="lpage">1958</span>. <span class="comment">doi: <a href="https://doi.org/10.1214/12-AOS1000">10.1214/12-AOS1000</a> </span> </span> </li> <li tag="ref"> <a name="pone.0198030.ref060" /> <span class="label" tagx="label" title="label">60</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Kingma DP, Ba J. Adam: A Method for Stochastic Optimization. arXiv:14126980. 2014;</span> </li> </ul> </div> </div>  </body></html>